{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/AMIRMOHAMMAD-OSS/Bioinformatics/blob/main/Phaseek.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "7DkACTVHMVCm"
      },
      "source": [
        "#Phaseek a Liquid-Liquid Phase Separation (LLPS) Protein Scorer!\n",
        "\n",
        "This tool allows you to analyze protein sequences for their propensity to undergo phase separation:\n",
        "\n",
        "Input Options:\n",
        "Upload a single protein sequence directly into the designated box.\n",
        "Alternatively, load a FASTA or TXT file containing multiple protein sequences.\n",
        "\n",
        "To maintain efficiency, we advise against loading more than 100 sequences at once\n",
        "\n",
        "Specify the sequence ID or directory for your analysis.\n",
        "\n",
        "By setting the 'End_Sequence'  parameter,you can focus the scoring on sequences up to the specified endpoint.\n",
        "\n",
        "         \n",
        "\n",
        "\n",
        "\n",
        "#BLAST+ tool\n",
        "You can utilize the BLAST+ tool to run ncbi BLAST searches on either your protein sequence or your FASTA file against your dataset. This powerful tool allows you to compare sequences and identify homologous regions, aiding in various biological analyses.\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "id": "-c3Fkzg9MPNn",
        "cellView": "form"
      },
      "outputs": [],
      "source": [
        "#@title Importing libraries\n",
        "%%capture\n",
        "from google.colab import output\n",
        "!git clone https://github.com/AMIRMOHAMMAD-OSS/classi\n",
        "!pip install biopython\n",
        "!apt-get update\n",
        "!apt-get install -y ncbi-blast+\n",
        "!pip install streamlit\n",
        "\n",
        "!pip install py3Dmol\n",
        "!pip install requests\n",
        "!pip install biotite\n",
        "!pip install stmol\n",
        "!pip install command_runner\n",
        "from command_runner import command_runner\n",
        "!pip install plotly\n",
        "!pip install --upgrade \"kaleido==0.1.*\"\n",
        "import kaleido\n",
        "from Bio import SeqIO\n",
        "from Bio.Seq import Seq\n",
        "from Bio import pairwise2\n",
        "from google.colab import files\n",
        "import plotly.graph_objects as go\n",
        "from plotly.subplots import make_subplots\n",
        "import zipfile\n",
        "import shutil\n",
        "from transformers import AutoTokenizer\n",
        "from tqdm import tqdm\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "import numpy as np\n",
        "import time\n",
        "from collections import defaultdict\n",
        "import torch\n",
        "from torch.utils.data.dataloader import DataLoader\n",
        "import time\n",
        "import os\n",
        "import sys\n",
        "import json\n",
        "import random\n",
        "from ast import literal_eval\n",
        "from tokenizers import Tokenizer, decoders, models, normalizers, pre_tokenizers, trainers\n",
        "from transformers import CanineTokenizer, CanineModel\n",
        "tokenizer = Tokenizer.from_file(\"classi/Trained_BPE2.json\")\n",
        "tokenizer.model_max_length = 256\n",
        "tokenizer\n",
        "\n",
        "from sklearn.preprocessing import StandardScaler as Sc\n",
        "from sklearn.decomposition import PCA\n",
        "import numpy as np\n",
        "import pandas as pd\n",
        "\n",
        "import torch\n",
        "\n",
        "from sklearn.model_selection import train_test_split as tts\n",
        "\n",
        "\n",
        "def edit(x):\n",
        "    for i in x:\n",
        "      if i in \"BJOUX\\Z_n\\n\":\n",
        "        x = x.replace(i,\"\")\n",
        "      else:\n",
        "        pass\n",
        "    return x\n",
        "\n",
        "def encode(x):\n",
        "  l = []\n",
        "  chars = tokenizer.get_vocab()\n",
        "  for i in x:\n",
        "    l.append(chars[i])\n",
        "  return l\n",
        "import pickle\n",
        "\n",
        "with open('classi/xgb_model (1).pkl', 'rb') as f:\n",
        "    clf = pickle.load(f)\n",
        "\n",
        "with open(\"classi/training.txt\",\"r\",encoding=\"utf-8\") as f:\n",
        "  text = f.read()\n",
        "  k = \"#\".join([i for i in edit(text).split(\"<|edoftext|>\")])\n",
        "chars = tokenizer.get_vocab()\n",
        "vocab_size = len(tokenizer.get_vocab())\n",
        "\n",
        "with open(\"classi/validation.txt\",\"r\",encoding=\"utf-8\") as f:\n",
        "  text = f.read()\n",
        "  l = \"#\".join([i for i in edit(text).split(\"<|edoftext|>\")])\n",
        "\n",
        "def Padding(x,PAD = 0,max_len = 512):\n",
        "  l = []\n",
        "  max_len = max(512,len(max(x,key = lambda x: len(x))))\n",
        "  for i in x:\n",
        "    a = i\n",
        "    if len(a) < max_len:\n",
        "      a = a + [PAD for i in range(max_len-len(a))]\n",
        "    l.append(a)\n",
        "  return np.array(l).reshape(len(x),max_len)\n",
        "with open(\"classi/negative_dataset.txt\") as x:\n",
        "  c = x.read()\n",
        "Filtered_train  = [i for i in k.split(\"#\")][1:]\n",
        "Filtered_val = [i for i in l.split(\"#\")][1:]\n",
        "Filtered_neg = [i for i in c.split(\"\\n\")]\n",
        "pos = Filtered_train + Filtered_val\n",
        "pos2 = list(set([i for i in k.split(\"#\")][1:] +  [i for i in l.split(\"#\")][1:]))\n",
        "with open(\"DrLLPS.fasta\",\"w\") as w:\n",
        "  for i in range(len(pos2)):\n",
        "    w.write(\">seq{i}\\n\".format(i=i))\n",
        "    w.write(pos2[i]+\"\\n\")\n",
        "  w.close()\n",
        "print(\"Everything looks ok! ‚ú®üç∞‚ú®\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {
        "cellView": "form",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "cgxWphcfMe1N",
        "outputId": "9f618b82-4a5d-4fab-a3de-92f161774c21"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Cuda is available üéâ\n",
            "\n",
            "The allocated device is : Tesla T4\n"
          ]
        }
      ],
      "source": [
        "#@title Loading models and requierments\n",
        "if torch.cuda.is_available():\n",
        "  print(\"Cuda is available üéâ\\n\")\n",
        "  print(\"The allocated device is : \"+torch.cuda.get_device_name())\n",
        "  device = torch.device(\"cuda\")\n",
        "else:\n",
        "  print(\"Cuda is not available üòî\\n\")\n",
        "  print(\"The allocated device is : cpu\")\n",
        "  device = torch.device(\"cpu\")\n",
        "\n",
        "\n",
        "\n",
        "def set_seed(seed):\n",
        "    random.seed(seed)\n",
        "    np.random.seed(seed)\n",
        "    torch.manual_seed(seed)\n",
        "    torch.cuda.manual_seed_all(seed)\n",
        "\n",
        "def setup_logging(config):\n",
        "    work_dir = config.system.work_dir\n",
        "    os.makedirs(work_dir, exist_ok=True)\n",
        "    with open(os.path.join(work_dir, 'args.txt'), 'w') as f:\n",
        "        f.write(' '.join(sys.argv))\n",
        "    with open(os.path.join(work_dir, 'config.json'), 'w') as f:\n",
        "        f.write(json.dumps(config.to_dict(), indent=4))\n",
        "\n",
        "class CfgNode:\n",
        "    def __init__(self, **kwargs):\n",
        "        self.__dict__.update(kwargs)\n",
        "\n",
        "    def __str__(self):\n",
        "        return self._str_helper(0)\n",
        "\n",
        "    def _str_helper(self, indent):\n",
        "        parts = []\n",
        "        for k, v in self.__dict__.items():\n",
        "            if isinstance(v, CfgNode):\n",
        "                parts.append(\"%s:\\n\" % k)\n",
        "                parts.append(v._str_helper(indent + 1))\n",
        "            else:\n",
        "                parts.append(\"%s: %s\\n\" % (k, v))\n",
        "        parts = [' ' * (indent * 4) + p for p in parts]\n",
        "        return \"\".join(parts)\n",
        "\n",
        "    def to_dict(self):\n",
        "        return { k: v.to_dict() if isinstance(v, CfgNode) else v for k, v in self.__dict__.items() }\n",
        "\n",
        "    def merge_from_dict(self, d):\n",
        "        self.__dict__.update(d)\n",
        "\n",
        "    def merge_from_args(self, args):\n",
        "\n",
        "        for arg in args:\n",
        "            keyval = arg.split('=')\n",
        "            assert len(keyval) == 2, \"expecting each override arg to be of form --arg=value, got %s\" % arg\n",
        "            key, val = keyval\n",
        "            try:\n",
        "                val = literal_eval(val)\n",
        "            except ValueError:\n",
        "                pass\n",
        "\n",
        "            assert key[:2] == '--'\n",
        "            key = key[2:]\n",
        "            keys = key.split('.')\n",
        "            obj = self\n",
        "            for k in keys[:-1]:\n",
        "                obj = getattr(obj, k)\n",
        "            leaf_key = keys[-1]\n",
        "            assert hasattr(obj, leaf_key), f\"{key} is not an attribute that exists in the config\"\n",
        "            print(\"command line overwriting config attribute %s with %s\" % (key, val))\n",
        "            setattr(obj, leaf_key, val)\n",
        "\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "\n",
        "class SiLU(nn.Module):\n",
        "   def forward(self, x):\n",
        "        return x*F.sigmoid(x)\n",
        "\n",
        "class NY(nn.Module):\n",
        "  def forward(self,x):\n",
        "    return 3*torch.tanh(0.3*x)\n",
        "\n",
        "class NewGELU(nn.Module):\n",
        "    def forward(self, x):\n",
        "        return 0.5 * x * (1.0 + torch.tanh(math.sqrt(2.0 / math.pi) * (x + 0.044715 * torch.pow(x, 3.0))))\n",
        "\n",
        "class SelfAttention(nn.Module):\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        assert config.n_embd % config.n_head == 0\n",
        "        self.c_attn = nn.Linear(config.n_embd, 3 * config.n_embd)\n",
        "        self.c_proj = nn.Linear(config.n_embd, config.n_embd)\n",
        "        self.attn_dropout = nn.Dropout(config.attn_pdrop)\n",
        "        self.resid_dropout = nn.Dropout(config.resid_pdrop)\n",
        "        self.register_buffer(\"bias\", torch.tril(torch.ones(config.block_size, config.block_size))\n",
        "                                     .view(1, 1, config.block_size, config.block_size))\n",
        "        self.n_head = config.n_head\n",
        "        self.n_embd = config.n_embd\n",
        "\n",
        "    def forward(self, x):\n",
        "        B, T, C = x.size()\n",
        "        q, k ,v  = self.c_attn(x).split(self.n_embd, dim=2)\n",
        "        k = k.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        q = q.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        v = v.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        att = (q @ k.transpose(-2, -1)) * (1.0 / math.sqrt(k.size(-1)))\n",
        "        att = att.masked_fill(self.bias[:,:,:T,:T] == 0, float('-inf'))\n",
        "        att = F.softmax(att, dim=-1)\n",
        "        att = self.attn_dropout(att)\n",
        "        y = att @ v\n",
        "        y = y.transpose(1, 2).contiguous().view(B, T, C)\n",
        "        y = self.resid_dropout(self.c_proj(y))\n",
        "        return y\n",
        "\n",
        "class Block(nn.Module):\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.ln_1 = nn.LayerNorm(config.n_embd)\n",
        "        self.attn = SelfAttention(config)\n",
        "        self.ln_2 = nn.LayerNorm(config.n_embd)\n",
        "        self.mlp = nn.ModuleDict(dict(\n",
        "            c_fc    = nn.Linear(config.n_embd, 4 * config.n_embd),\n",
        "            c_proj  = nn.Linear(4 * config.n_embd, config.n_embd),\n",
        "            act     = NewGELU(),\n",
        "            dropout = nn.Dropout(config.resid_pdrop),\n",
        "        ))\n",
        "        m = self.mlp\n",
        "        self.mlpf = lambda x: m.dropout(m.c_proj(m.act(m.c_fc(x))))\n",
        "    def forward(self, x):\n",
        "        x = x + self.attn(self.ln_1(x))\n",
        "        x = x + self.mlpf(self.ln_2(x))\n",
        "        return x\n",
        "\n",
        "class ClassifierII(nn.Module):\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.model_type = 'gpt'\n",
        "        C.n_layer = None\n",
        "        C.n_head = None\n",
        "        C.n_embd =  None\n",
        "        C.vocab_size = len(chars)\n",
        "        C.max_length = 512\n",
        "        C.embd_pdrop = 0.1\n",
        "        C.resid_pdrop = 0.1\n",
        "        C.attn_pdrop = 0.1\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.max_length = config.block_size\n",
        "        self.soft = nn.Softmax(1)\n",
        "        self.config = self.get_default_config()\n",
        "        self.device = \"cuda\"\n",
        "        self.model_states = {'h':{'n_layer': 48, 'n_head': 25, 'n_embd': 1600},\n",
        "                                    'g':{'n_layer': 12, '': 12, 'n_embd': 768},\n",
        "                'f':   {'n_layer': 24, 'n_head': 16, 'n_embd': 1024},\n",
        "                'e':   {'n_layer': 36, 'n_head': 20, 'n_embd': 1280},\n",
        "                'd':{'n_layer': 8, 'n_head': 16, 'n_embd': 512},\n",
        "                'c':{'n_layer': 6, 'n_head': 6, 'n_embd': 192},\n",
        "                'b':{'n_layer': 4, 'n_head': 4, 'n_embd': 128},\n",
        "                'a':{'n_layer': 3, 'n_head': 3, 'n_embd': 48}}\n",
        "\n",
        "        type_ = config.model_type is not None\n",
        "        #assert type_ in \"abcdefgh\"\n",
        "        p = all([config.n_layer is not None, config.n_head is not None, config.n_embd is not None])\n",
        "        #assert type_ == True and p == True\n",
        "        if type_:\n",
        "            config.merge_from_dict(self.model_states[config.model_type])\n",
        "        self.closs = nn.BCELoss()\n",
        "        self.ny = NY()\n",
        "        self.l = nn.Linear(512,1,64)\n",
        "        self.transformer = nn.ModuleDict(dict(\n",
        "            wte = nn.Embedding(config.vocab_size, config.n_embd),\n",
        "            wpe = nn.Embedding(config.max_length, config.n_embd),\n",
        "            drop = nn.Dropout(config.embd_pdrop),\n",
        "            h = nn.ModuleList([Block(config) for _ in range(config.n_layer)]),\n",
        "            ln_f = nn.LayerNorm(config.n_embd),))\n",
        "        self.classifier_head = nn.Sequential(#nn.Tanh(),\n",
        "                                             nn.Linear(config.n_embd, 2)\n",
        "                                             )\n",
        "        self.apply(self._init_weights)\n",
        "        for pn, p in self.named_parameters():\n",
        "            if pn.endswith('c_proj.weight'):\n",
        "                torch.nn.init.normal_(p, mean=0.0, std=0.02/math.sqrt(2 * config.n_layer))\n",
        "\n",
        "        n_params = sum(p.numel() for p in self.transformer.parameters())\n",
        "        print(\"[ Number of trainable parameters: %.2fM ]\" % (n_params/1e6,))\n",
        "\n",
        "    def _init_weights(self, module):\n",
        "        if isinstance(module, nn.Linear):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "            if module.bias is not None:\n",
        "                torch.nn.init.zeros_(module.bias)\n",
        "        elif isinstance(module, nn.Embedding):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "        elif isinstance(module, nn.LayerNorm):\n",
        "            torch.nn.init.zeros_(module.bias)\n",
        "            torch.nn.init.ones_(module.weight)\n",
        "\n",
        "    @classmethod\n",
        "    def from_pretrained(cls, model_type):\n",
        "        from transformers import GPT2LMHeadModel\n",
        "        config = cls.get_default_config()\n",
        "        config.model_type = model_type\n",
        "        config.vocab_size = 25\n",
        "        config.max_length = 512\n",
        "        model = ClassifierII(config)\n",
        "        sd = model.state_dict()\n",
        "        model_hf = GPT2LMHeadModel.from_pretrained(model_type)\n",
        "        sd_hf = model_hf.state_dict()\n",
        "        keys = [k for k in sd_hf if not k.endswith('attn.masked_bias')]\n",
        "        transposed = ['attn.c_attn.weight', 'attn.c_proj.weight', 'mlp.c_fc.weight', 'mlp.c_proj.weight']\n",
        "        assert len(keys) == len(sd)\n",
        "        for k in keys:\n",
        "            if any(k.endswith(w) for w in transposed):\n",
        "                assert sd_hf[k].shape[::-1] == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k].t())\n",
        "            else:\n",
        "                assert sd_hf[k].shape == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k])\n",
        "        return model\n",
        "\n",
        "    def configure_optimizers(self, train_config):\n",
        "        decay = set()\n",
        "        no_decay = set()\n",
        "        whitelist_weight_modules = (torch.nn.Linear, )\n",
        "        blacklist_weight_modules = (torch.nn.LayerNorm, torch.nn.Embedding)\n",
        "        for mn, m in self.named_modules():\n",
        "            for pn, p in m.named_parameters():\n",
        "                fpn = '%s.%s' % (mn, pn) if mn else pn\n",
        "                if pn.endswith('bias'):\n",
        "                    no_decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, whitelist_weight_modules):\n",
        "                    decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, blacklist_weight_modules):\n",
        "                    no_decay.add(fpn)\n",
        "        param_dict = {pn: p for pn, p in self.named_parameters()}\n",
        "        inter_params = decay & no_decay\n",
        "        union_params = decay | no_decay\n",
        "        assert len(inter_params) == 0, \"parameters %s made it into both decay/no_decay sets!\" % (str(inter_params), )\n",
        "        assert len(param_dict.keys() - union_params) == 0, \"parameters %s were not separated into either decay/no_decay set!\" \\\n",
        "                                                    % (str(param_dict.keys() - union_params), )\n",
        "        optim_groups = [\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(decay))], \"weight_decay\": train_config.weight_decay},\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(no_decay))], \"weight_decay\": 0.0},]\n",
        "        optimizer = torch.optim.AdamW(optim_groups, lr=train_config.learning_rate, betas=train_config.betas)\n",
        "        return optimizer\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict_proba(self,idx,j =\"None\"):\n",
        "      sigmoid = nn.Sigmoid()\n",
        "      soft = nn.Softmax(dim=1)\n",
        "      ny = NY()\n",
        "      si = SiLU()\n",
        "      self.eval()\n",
        "      x,_ = self.forward(idx)\n",
        "      if j == \"None\":\n",
        "        return x[:,0:1]\n",
        "      elif j == \"soft\":\n",
        "        return self.soft(x)[:,0:1]\n",
        "      else:\n",
        "        return sigmoid(x)[:,0:1]\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict(self,idx):\n",
        "      g = torch.zeros((idx.shape[0],1))\n",
        "      e = -1\n",
        "      for i in self.predict_proba(idx):\n",
        "        e+=1\n",
        "        if i[0].item()>=0.5:\n",
        "          g[e] = 1\n",
        "      return g\n",
        "\n",
        "    def forward(self, idx, targets=None):\n",
        "        device = idx.device\n",
        "        b, t = idx.size()\n",
        "        pos = torch.arange(0, t, dtype=torch.long, device=device).unsqueeze(0)\n",
        "        tok_emb = self.transformer.wte(idx)\n",
        "        pos_emb = self.transformer.wpe(pos)\n",
        "        x = self.transformer.drop(tok_emb + pos_emb)\n",
        "        for block in self.transformer.h:\n",
        "            x = block(x)\n",
        "        x = self.transformer.ln_f(x)\n",
        "        logits = self.classifier_head(x).view(b,2,t).to(device)\n",
        "        logits = self.l(logits)\n",
        "        logits = F.sigmoid(self.ny(logits).view(b,2).mean(1).view(b,1))\n",
        "        loss = None\n",
        "        if targets is not None:\n",
        "            loss = F.binary_cross_entropy(logits,targets)\n",
        "        return logits, loss\n",
        "\n",
        "\n",
        "class Trainer:\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.device = 'cuda'\n",
        "        C.num_workers = 4\n",
        "        C.max_iters = None\n",
        "        C.batch_size = 128\n",
        "        C.max_length = 512\n",
        "        C.learning_rate = 8e-4\n",
        "        C.betas = (0.9, 0.95)\n",
        "        C.weight_decay = 0.1\n",
        "        C.grad_norm_clip = 1.0\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config, model, train_dataset,val_dataset):\n",
        "        self.config = config\n",
        "        self.model = model\n",
        "        self.optimizer = None\n",
        "        self.train_dataset = train_dataset\n",
        "        self.val_dataset = val_dataset\n",
        "        self.callbacks = defaultdict(list)\n",
        "        self.device ='cuda'\n",
        "        self.model = self.model.to(self.device)\n",
        "        self.iter_num = 0\n",
        "        self.iter_time = 0.0\n",
        "        self.iter_dt = 0.0\n",
        "\n",
        "    def add_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent].append(callback)\n",
        "\n",
        "    def set_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent] = [callback]\n",
        "\n",
        "    def trigger_callbacks(self, onevent: str):\n",
        "        for callback in self.callbacks.get(onevent, []):\n",
        "            callback(self)\n",
        "\n",
        "    def run(self):\n",
        "        model, config = self.model, self.config\n",
        "        self.optimizer = model.configure_optimizers(config)\n",
        "        batch_size = self.config.batch_size\n",
        "        def get_batch(mode):\n",
        "            batch_size = 64\n",
        "            if mode == \"train\":\n",
        "              data = train_dataset\n",
        "              pos_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]),513))\n",
        "              neg_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]),513))\n",
        "              N = np.random.randint(batch_size)\n",
        "              ix_pos = np.random.randint(pos_train.shape[0]-N)\n",
        "              ix_neg = np.random.randint(neg_train.shape[0]-batch_size-N)\n",
        "              pos_data = pos_train[ix_pos:ix_pos+N,:]\n",
        "              neg_data = neg_train[ix_neg:ix_neg+batch_size-N,:]\n",
        "              data = torch.concat((pos_data,neg_data))\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,1), device=\"cuda\")\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o] = 1\n",
        "                #else:\n",
        "                  #targets[o][1] = 1\n",
        "            else:\n",
        "              data = val_dataset\n",
        "              ix = np.random.randint(data.shape[0]-batch_size)\n",
        "              data = data[ix:ix+batch_size,:]\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,1), device=\"cuda\")\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o] = 1\n",
        "                #else:\n",
        "                 # targets[o][1] = 1\n",
        "            targets = targets.view(batch_size,1).to(\"cuda\")\n",
        "            seq = seq.view(batch_size,512).to(\"cuda\")\n",
        "            return seq, targets\n",
        "\n",
        "        @torch.no_grad\n",
        "        def cross_val():\n",
        "          model.eval()\n",
        "          out = []\n",
        "          for i in [\"train\",\"val\"]:\n",
        "            losses = torch.zeros(200+1)\n",
        "            for k in range(200):\n",
        "              X,Y = get_batch(i)\n",
        "              logits,loss = model(X,Y)\n",
        "              losses[k]=loss.item()\n",
        "              out1 = losses.mean()\n",
        "            out.append(out1)\n",
        "          model.train()\n",
        "          return out\n",
        "        losses = cross_val()\n",
        "        LOSS = [losses]\n",
        "        print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        model.train()\n",
        "        for epoch in range(20):\n",
        "          print(\"[epoch {o}] \\n\".format(o = epoch))\n",
        "          iters = 200\n",
        "          for i in range(iters):\n",
        "            if i == 0:\n",
        "              Y = \"| =\"\n",
        "            elif i == iters -1 :\n",
        "              Y = \"=> 100% |\"\n",
        "            else:\n",
        "              if i%(int(iters/50)) == 0 :\n",
        "                Y = \"=\"\n",
        "              else:\n",
        "                Y = \"\"\n",
        "            print(\"{y}\".format(y = Y),end=\"\")\n",
        "            x, y = get_batch(\"train\")\n",
        "            logits, self.loss = model(x, y)\n",
        "            model.zero_grad(set_to_none=True)\n",
        "            self.loss.backward()\n",
        "            torch.nn.utils.clip_grad_norm_(model.parameters(), config.grad_norm_clip)\n",
        "            self.optimizer.step()\n",
        "          losses = cross_val()\n",
        "          LOSS.append(losses)\n",
        "          print(LOSS)\n",
        "          print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        PATH = \"model {h}\".format(h = model_config.model_type)\n",
        "        torch.save(model.state_dict(), PATH)\n",
        "\n",
        "def set_seed(seed):\n",
        "    random.seed(seed)\n",
        "    np.random.seed(seed)\n",
        "    torch.manual_seed(seed)\n",
        "    torch.cuda.manual_seed_all(seed)\n",
        "\n",
        "def setup_logging(config):\n",
        "    work_dir = config.system.work_dir\n",
        "    os.makedirs(work_dir, exist_ok=True)\n",
        "    with open(os.path.join(work_dir, 'args.txt'), 'w') as f:\n",
        "        f.write(' '.join(sys.argv))\n",
        "    with open(os.path.join(work_dir, 'config.json'), 'w') as f:\n",
        "        f.write(json.dumps(config.to_dict(), indent=4))\n",
        "\n",
        "class CfgNode:\n",
        "    def __init__(self, **kwargs):\n",
        "        self.__dict__.update(kwargs)\n",
        "\n",
        "    def __str__(self):\n",
        "        return self._str_helper(0)\n",
        "\n",
        "    def _str_helper(self, indent):\n",
        "        parts = []\n",
        "        for k, v in self.__dict__.items():\n",
        "            if isinstance(v, CfgNode):\n",
        "                parts.append(\"%s:\\n\" % k)\n",
        "                parts.append(v._str_helper(indent + 1))\n",
        "            else:\n",
        "                parts.append(\"%s: %s\\n\" % (k, v))\n",
        "        parts = [' ' * (indent * 4) + p for p in parts]\n",
        "        return \"\".join(parts)\n",
        "\n",
        "    def to_dict(self):\n",
        "        return { k: v.to_dict() if isinstance(v, CfgNode) else v for k, v in self.__dict__.items() }\n",
        "\n",
        "    def merge_from_dict(self, d):\n",
        "        self.__dict__.update(d)\n",
        "\n",
        "    def merge_from_args(self, args):\n",
        "\n",
        "        for arg in args:\n",
        "            keyval = arg.split('=')\n",
        "            assert len(keyval) == 2, \"expecting each override arg to be of form --arg=value, got %s\" % arg\n",
        "            key, val = keyval\n",
        "            try:\n",
        "                val = literal_eval(val)\n",
        "            except ValueError:\n",
        "                pass\n",
        "\n",
        "            assert key[:2] == '--'\n",
        "            key = key[2:]\n",
        "            keys = key.split('.')\n",
        "            obj = self\n",
        "            for k in keys[:-1]:\n",
        "                obj = getattr(obj, k)\n",
        "            leaf_key = keys[-1]\n",
        "            assert hasattr(obj, leaf_key), f\"{key} is not an attribute that exists in the config\"\n",
        "            print(\"command line overwriting config attribute %s with %s\" % (key, val))\n",
        "            setattr(obj, leaf_key, val)\n",
        "\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "\n",
        "class SiLU(nn.Module):\n",
        "   def forward(self, x):\n",
        "        return x*F.sigmoid(x)\n",
        "\n",
        "class NY(nn.Module):\n",
        "  def forward(self,x):\n",
        "    return 3*torch.tanh(0.3*x)\n",
        "\n",
        "class NewGELU(nn.Module):\n",
        "    def forward(self, x):\n",
        "        return 0.5 * x * (1.0 + torch.tanh(math.sqrt(2.0 / math.pi) * (x + 0.044715 * torch.pow(x, 3.0))))\n",
        "\n",
        "class SelfAttention(nn.Module):\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        assert config.n_embd % config.n_head == 0\n",
        "        self.c_attn = nn.Linear(config.n_embd, 3 * config.n_embd)\n",
        "        self.c_proj = nn.Linear(config.n_embd, config.n_embd)\n",
        "        self.attn_dropout = nn.Dropout(config.attn_pdrop)\n",
        "        self.resid_dropout = nn.Dropout(config.resid_pdrop)\n",
        "        self.register_buffer(\"bias\", torch.tril(torch.ones(config.block_size, config.block_size))\n",
        "                                     .view(1, 1, config.block_size, config.block_size))\n",
        "        self.n_head = config.n_head\n",
        "        self.n_embd = config.n_embd\n",
        "\n",
        "    def forward(self, x):\n",
        "        B, T, C = x.size()\n",
        "        q, k ,v  = self.c_attn(x).split(self.n_embd, dim=2)\n",
        "        k = k.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        q = q.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        v = v.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        att = (q @ k.transpose(-2, -1)) * (1.0 / math.sqrt(k.size(-1)))\n",
        "        att = att.masked_fill(self.bias[:,:,:T,:T] == 0, float('-inf'))\n",
        "        att = F.softmax(att, dim=-1)\n",
        "        att = self.attn_dropout(att)\n",
        "        y = att @ v\n",
        "        y = y.transpose(1, 2).contiguous().view(B, T, C)\n",
        "        y = self.resid_dropout(self.c_proj(y))\n",
        "        return y\n",
        "\n",
        "class Block(nn.Module):\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.ln_1 = nn.LayerNorm(config.n_embd)\n",
        "        self.attn = SelfAttention(config)\n",
        "        self.ln_2 = nn.LayerNorm(config.n_embd)\n",
        "        self.mlp = nn.ModuleDict(dict(\n",
        "            c_fc    = nn.Linear(config.n_embd, 4 * config.n_embd),\n",
        "            c_proj  = nn.Linear(4 * config.n_embd, config.n_embd),\n",
        "            act     = NewGELU(),\n",
        "            dropout = nn.Dropout(config.resid_pdrop),\n",
        "        ))\n",
        "        m = self.mlp\n",
        "        self.mlpf = lambda x: m.dropout(m.c_proj(m.act(m.c_fc(x))))\n",
        "    def forward(self, x):\n",
        "        x = x + self.attn(self.ln_1(x))\n",
        "        x = x + self.mlpf(self.ln_2(x))\n",
        "        return x\n",
        "\n",
        "class ClassifierI(nn.Module):\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.model_type = 'gpt'\n",
        "        C.n_layer = None\n",
        "        C.n_head = None\n",
        "        C.n_embd =  None\n",
        "        C.vocab_size = len(chars)\n",
        "        C.max_length = 512\n",
        "        C.embd_pdrop = 0.1\n",
        "        C.resid_pdrop = 0.1\n",
        "        C.attn_pdrop = 0.1\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.max_length = config.block_size\n",
        "        self.soft = nn.Softmax(1)\n",
        "        self.config = self.get_default_config()\n",
        "        self.device = \"cpu\"\n",
        "        self.model_states = {'h':{'n_layer': 48, 'n_head': 25, 'n_embd': 1600},\n",
        "                                    'g':{'n_layer': 12, '': 12, 'n_embd': 768},\n",
        "                'f':   {'n_layer': 24, 'n_head': 16, 'n_embd': 1024},\n",
        "                'e':   {'n_layer': 36, 'n_head': 20, 'n_embd': 1280},\n",
        "                'd':{'n_layer': 8, 'n_head': 16, 'n_embd': 512},\n",
        "                'c':{'n_layer': 6, 'n_head': 6, 'n_embd': 192},\n",
        "                'b':{'n_layer': 4, 'n_head': 4, 'n_embd': 128},\n",
        "                'a':{'n_layer': 3, 'n_head': 3, 'n_embd': 48}}\n",
        "\n",
        "        type_ = config.model_type is not None\n",
        "        #assert type_ in \"abcdefgh\"\n",
        "        p = all([config.n_layer is not None, config.n_head is not None, config.n_embd is not None])\n",
        "        #assert type_ == True and p == True\n",
        "        if type_:\n",
        "            config.merge_from_dict(self.model_states[config.model_type])\n",
        "        self.closs = nn.BCELoss()\n",
        "        self.ny = NY()\n",
        "        #self.l = nn.Linear(512,1,64)\n",
        "        self.transformer = nn.ModuleDict(dict(\n",
        "            wte = nn.Embedding(config.vocab_size, config.n_embd),\n",
        "            wpe = nn.Embedding(config.max_length, config.n_embd),\n",
        "            drop = nn.Dropout(config.embd_pdrop),\n",
        "            h = nn.ModuleList([Block(config) for _ in range(config.n_layer)]),\n",
        "            ln_f = nn.LayerNorm(config.n_embd),))\n",
        "        self.classifier_head = nn.Sequential(#nn.Tanh(),\n",
        "                                             nn.Linear(config.n_embd, 2)\n",
        "                                             )\n",
        "        self.apply(self._init_weights)\n",
        "        for pn, p in self.named_parameters():\n",
        "            if pn.endswith('c_proj.weight'):\n",
        "                torch.nn.init.normal_(p, mean=0.0, std=0.02/math.sqrt(2 * config.n_layer))\n",
        "\n",
        "        n_params = sum(p.numel() for p in self.transformer.parameters())\n",
        "        #print(\"[ Number of trainable parameters: %.2fM ]\" % (n_params/1e6,))\n",
        "\n",
        "    def _init_weights(self, module):\n",
        "        if isinstance(module, nn.Linear):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "            if module.bias is not None:\n",
        "                torch.nn.init.zeros_(module.bias)\n",
        "        elif isinstance(module, nn.Embedding):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "        elif isinstance(module, nn.LayerNorm):\n",
        "            torch.nn.init.zeros_(module.bias)\n",
        "            torch.nn.init.ones_(module.weight)\n",
        "\n",
        "    @classmethod\n",
        "    def from_pretrained(cls, model_type):\n",
        "        from transformers import GPT2LMHeadModel\n",
        "        config = cls.get_default_config()\n",
        "        config.model_type = model_type\n",
        "        config.vocab_size = 25\n",
        "        config.max_length = 512\n",
        "        model = ClassifierI(config)\n",
        "        sd = model.state_dict()\n",
        "        model_hf = GPT2LMHeadModel.from_pretrained(model_type)\n",
        "        sd_hf = model_hf.state_dict()\n",
        "        keys = [k for k in sd_hf if not k.endswith('attn.masked_bias')]\n",
        "        transposed = ['attn.c_attn.weight', 'attn.c_proj.weight', 'mlp.c_fc.weight', 'mlp.c_proj.weight']\n",
        "        assert len(keys) == len(sd)\n",
        "        for k in keys:\n",
        "            if any(k.endswith(w) for w in transposed):\n",
        "                assert sd_hf[k].shape[::-1] == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k].t())\n",
        "            else:\n",
        "                assert sd_hf[k].shape == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k])\n",
        "        return model\n",
        "\n",
        "    def configure_optimizers(self, train_config):\n",
        "        decay = set()\n",
        "        no_decay = set()\n",
        "        whitelist_weight_modules = (torch.nn.Linear, )\n",
        "        blacklist_weight_modules = (torch.nn.LayerNorm, torch.nn.Embedding)\n",
        "        for mn, m in self.named_modules():\n",
        "            for pn, p in m.named_parameters():\n",
        "                fpn = '%s.%s' % (mn, pn) if mn else pn\n",
        "                if pn.endswith('bias'):\n",
        "                    no_decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, whitelist_weight_modules):\n",
        "                    decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, blacklist_weight_modules):\n",
        "                    no_decay.add(fpn)\n",
        "        param_dict = {pn: p for pn, p in self.named_parameters()}\n",
        "        inter_params = decay & no_decay\n",
        "        union_params = decay | no_decay\n",
        "        assert len(inter_params) == 0, \"parameters %s made it into both decay/no_decay sets!\" % (str(inter_params), )\n",
        "        assert len(param_dict.keys() - union_params) == 0, \"parameters %s were not separated into either decay/no_decay set!\" \\\n",
        "                                                    % (str(param_dict.keys() - union_params), )\n",
        "        optim_groups = [\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(decay))], \"weight_decay\": train_config.weight_decay},\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(no_decay))], \"weight_decay\": 0.0},]\n",
        "        optimizer = torch.optim.AdamW(optim_groups, lr=train_config.learning_rate, betas=train_config.betas)\n",
        "        return optimizer\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict_proba(self,idx,j =\"None\"):\n",
        "      sigmoid = nn.Sigmoid()\n",
        "      soft = nn.Softmax(dim=1)\n",
        "      ny = NY()\n",
        "      si = SiLU()\n",
        "      self.eval()\n",
        "      x,_ = self.forward(idx)\n",
        "      if j == \"None\":\n",
        "        return x[:,0:1]\n",
        "      elif j == \"soft\":\n",
        "        return self.soft(x)[:,0:1]\n",
        "      else:\n",
        "        return sigmoid(x)[:,0:1]\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict(self,idx):\n",
        "      g = torch.zeros((idx.shape[0],1))\n",
        "      e = -1\n",
        "      for i in self.predict_proba(idx):\n",
        "        e+=1\n",
        "        if i[0].item()>=0.5:\n",
        "          g[e] = 1\n",
        "      return g\n",
        "\n",
        "    def forward(self, idx, targets=None):\n",
        "        device = idx.device\n",
        "        b, t = idx.size()\n",
        "        pos = torch.arange(0, t, dtype=torch.long, device=device).unsqueeze(0)\n",
        "        tok_emb = self.transformer.wte(idx)\n",
        "        pos_emb = self.transformer.wpe(pos)\n",
        "        x = self.transformer.drop(tok_emb + pos_emb)\n",
        "        for block in self.transformer.h:\n",
        "            x = block(x)\n",
        "        x = self.transformer.ln_f(x)\n",
        "        logits = self.ny(self.classifier_head(x)).mean(1).to(device)\n",
        "        #logits = self.l(logits)\n",
        "        # = F.sigmoid(logits.view(b,2).mean(1).view(b,1))\n",
        "        loss = None\n",
        "        if targets is not None:\n",
        "            loss = F.cross_entropy(logits,targets,ignore_index = -1)\n",
        "        return logits, loss\n",
        "\n",
        "\n",
        "class Trainer:\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.device = 'cuda'\n",
        "        C.num_workers = 4\n",
        "        C.max_iters = None\n",
        "        C.batch_size = 64\n",
        "        C.max_length = 512\n",
        "        C.learning_rate = 8e-4\n",
        "        C.betas = (0.9, 0.95)\n",
        "        C.weight_decay = 0.1\n",
        "        C.grad_norm_clip = 1.0\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config, model, train_dataset,val_dataset):\n",
        "        self.config = config\n",
        "        self.model = model\n",
        "        self.optimizer = None\n",
        "        self.train_dataset = train_dataset\n",
        "        self.val_dataset = val_dataset\n",
        "        self.callbacks = defaultdict(list)\n",
        "        self.device ='cuda'\n",
        "        self.model = self.model.to(self.device)\n",
        "        self.iter_num = 0\n",
        "        self.iter_time = 0.0\n",
        "        self.iter_dt = 0.0\n",
        "\n",
        "    def add_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent].append(callback)\n",
        "\n",
        "    def set_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent] = [callback]\n",
        "\n",
        "    def trigger_callbacks(self, onevent: str):\n",
        "        for callback in self.callbacks.get(onevent, []):\n",
        "            callback(self)\n",
        "\n",
        "    def run(self):\n",
        "        model, config = self.model, self.config\n",
        "        self.optimizer = model.configure_optimizers(config)\n",
        "        batch_size = self.config.batch_size\n",
        "        def get_batch(mode):\n",
        "            batch_size = 64\n",
        "            if mode == \"train\":\n",
        "              data = train_dataset\n",
        "              pos_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]),513))\n",
        "              neg_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]),513))\n",
        "              N = np.random.randint(batch_size)\n",
        "              ix_pos = np.random.randint(pos_train.shape[0]-N)\n",
        "              ix_neg = np.random.randint(neg_train.shape[0]-batch_size-N)\n",
        "              pos_data = pos_train[ix_pos:ix_pos+N,:]\n",
        "              neg_data = neg_train[ix_neg:ix_neg+batch_size-N,:]\n",
        "              data = torch.concat((pos_data,neg_data))\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,2), device=self.device)\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o][0] = 1\n",
        "                else:\n",
        "                  targets[o][1] = 1\n",
        "            else:\n",
        "              data = val_dataset\n",
        "              ix = np.random.randint(data.shape[0]-batch_size)\n",
        "              data = data[ix:ix+batch_size,:]\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,2), device=self.device)\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o][0] = 1\n",
        "                else:\n",
        "                  targets[o][1] = 1\n",
        "            targets = targets.view(batch_size,2).to(self.device)\n",
        "            seq = seq.view(batch_size,512).to(self.device)\n",
        "            return seq, targets\n",
        "\n",
        "        @torch.no_grad\n",
        "        def cross_val():\n",
        "          model.eval()\n",
        "          out = []\n",
        "          for i in [\"train\",\"val\"]:\n",
        "            losses = torch.zeros(200+1)\n",
        "            for k in range(200):\n",
        "              X,Y = get_batch(i)\n",
        "              logits,loss = model(X,Y)\n",
        "              losses[k]=loss.item()\n",
        "              out1 = losses.mean()\n",
        "            out.append(out1)\n",
        "          model.train()\n",
        "          return out\n",
        "        losses = cross_val()\n",
        "        LOSS = [losses]\n",
        "        print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        model.train()\n",
        "        for epoch in range(20):\n",
        "          print(\"[epoch {o}] \\n\".format(o = epoch))\n",
        "          iters = 200\n",
        "          for i in range(iters):\n",
        "            if i == 0:\n",
        "              Y = \"| =\"\n",
        "            elif i == iters -1 :\n",
        "              Y = \"=> 100% |\"\n",
        "            else:\n",
        "              if i%(int(iters/50)) == 0 :\n",
        "                Y = \"=\"\n",
        "              else:\n",
        "                Y = \"\"\n",
        "            print(\"{y}\".format(y = Y),end=\"\")\n",
        "            x, y = get_batch(\"train\")\n",
        "            logits, self.loss = model(x, y)\n",
        "            model.zero_grad(set_to_none=True)\n",
        "            self.loss.backward()\n",
        "            torch.nn.utils.clip_grad_norm_(model.parameters(), config.grad_norm_clip)\n",
        "            self.optimizer.step()\n",
        "          losses = cross_val()\n",
        "          LOSS.append(losses)\n",
        "          print(LOSS)\n",
        "          print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        PATH = \"model {h}\".format(h = model_config.model_type)\n",
        "        torch.save(model.state_dict(), PATH)\n",
        "class Transformer():\n",
        "  def __init__(self,mode):\n",
        "    self.mode = mode\n",
        "    self.device = device\n",
        "    self.vocab_dict = tokenizer.get_vocab()\n",
        "\n",
        "  def classifier(self):\n",
        "    device = self.device\n",
        "    model_config = ClassifierI.get_default_config()\n",
        "    model_config.vocab_size = 25\n",
        "    model_config.block_size = 512\n",
        "    if self.mode == \"b\":\n",
        "      model_config.model_type = 'b'\n",
        "      model2 = ClassifierI(model_config)\n",
        "      model2.load_state_dict(torch.load(\"classi/model_b\",map_location=self.device,weights_only = True))\n",
        "    else:\n",
        "      model_config.model_type = 'c'\n",
        "      model2 = ClassifierI(model_config)\n",
        "      model2.load_state_dict(torch.load(\"classi/model_c\",map_location = self.device,weights_only = True))\n",
        "    model2.to(device)\n",
        "    return model2\n",
        "\n",
        "  def Encode(self, i):\n",
        "        def encode_char(x):\n",
        "            return [self.vocab_dict[char] for char in x]\n",
        "\n",
        "        def pad_sequences(x, PAD=0, max_len=512):\n",
        "            return np.array([seq + [PAD] * (max_len - len(seq)) for seq in x])\n",
        "\n",
        "        encoded = list(map(encode_char, i))\n",
        "        padded = pad_sequences(encoded)\n",
        "        return torch.tensor(padded).to(self.device)\n",
        "\n",
        "  def Decode(self,i):\n",
        "    def decode(k):\n",
        "      l = [j for j in k if j != 0 ]\n",
        "      seq = [chars[j] for j in l]\n",
        "      return \"\".join(seq)\n",
        "    H = list(map(decode,i))\n",
        "    G = []\n",
        "    for i in H:\n",
        "      if len(i)>512:\n",
        "        u = H.index(i)\n",
        "        for j in range(len(i)-512):\n",
        "          G.append(i[j:j+512])\n",
        "      else:\n",
        "        G.append(i)\n",
        "    return G,u\n",
        "  def Mean(self,i):\n",
        "    I = list(i.ravel())\n",
        "    sig = lambda x:np.e**(0*x)\n",
        "    Sum = [sig(np.abs(x-int(len(I)/2))) for x in range(len(I))]\n",
        "    wei = np.array([Sum[i]/sum(Sum) for i in range(len(I))]).reshape(len(I),1)\n",
        "    return np.sum(wei*i)\n",
        "\n",
        "  def enhancer(self,x):\n",
        "    wei = np.array([np.exp((i[0]-1)/(i[0])) for i in x])\n",
        "    return np.sum(np.array([i[0] for i in x])*wei)/np.sum(wei)\n",
        "\n",
        "  def predict_proba(self,i):\n",
        "    if len(i[0])>512:\n",
        "       l = [self.Encode([i[0][j:j+512]]) for j in range(len(i[0])-512)]\n",
        "       if len(l)>700:\n",
        "        L = [torch.concat(tuple(l[i*700:(i+1)*700])) for i in range(len(l)//700)] + [torch.concat(tuple(l[(len(l)//700)*700:(len(l)//700)*700+len(l)%700]))]\n",
        "        T = [self.classifier().predict_proba(i,\"sig\").tolist() for i in L ]\n",
        "        return self.enhancer(np.concatenate(tuple([i for i in T])))\n",
        "       else:\n",
        "         t = torch.concat(tuple(l))\n",
        "         return self.enhancer(self.classifier().predict_proba(t,\"sig\").tolist())\n",
        "    else:\n",
        "       seq = self.Encode(i)\n",
        "       U = np.array(self.classifier().predict_proba(seq,\"sig\").tolist())\n",
        "       out = np.array(U)\n",
        "       return out.reshape(len(i),1)\n",
        "\n",
        "model2 = Transformer(\"b\")\n",
        "model3 = Transformer(\"c\")\n",
        "\n",
        "\n",
        "import functools as FUNC\n",
        "from numba import njit\n",
        "\n",
        "class PPA():\n",
        "    def __init__(self, seq, model):\n",
        "        self.model = model\n",
        "        self.seq = seq\n",
        "        self.len = len(self.seq)\n",
        "        self.idx = None\n",
        "\n",
        "    def attention_mask(self):\n",
        "        idx2 = self.splicer(self.seq)[1]\n",
        "        k = self.T()\n",
        "        mask = [[True if len(k[0]) > idx2[j] - i > -1 else False for i in range(len(k[j]))] for j in range(len(k))]\n",
        "        return mask\n",
        "\n",
        "    def mean(self, mask, spliced):\n",
        "        u = [np.mean(self.model.predict_proba([spliced[i][j] for j in range(len(spliced[i])) if mask[i][j]])) for i in trange(len(spliced))]\n",
        "        return u\n",
        "\n",
        "    def splice_scorer(self, i):\n",
        "        lmin = len(i) // 2\n",
        "        return [i[j:j + lmin] for j in range(len(i) - lmin + 1)]\n",
        "\n",
        "    def splicer(self, i):\n",
        "        idx = self.idx\n",
        "        o, Before = [], []\n",
        "        for j in range(len(i)):\n",
        "            n_b, n_a = j, len(i) - j - 1\n",
        "            if n_b <= idx // 2:\n",
        "                before = i[max(0, j - idx // 2):j]\n",
        "                after = i[j + 1:j + idx - len(before)]\n",
        "            elif n_a <= idx // 2:\n",
        "                before = i[j - idx + n_a + 1:j]\n",
        "                after = i[j + 1:]\n",
        "            else:\n",
        "                before = i[j - idx // 2:j]\n",
        "                after = i[j + 1:j + idx // 2 + 1]\n",
        "            o.append(before + i[j] + after)\n",
        "            Before.append(len(before))\n",
        "        return o, Before\n",
        "\n",
        "    def T(self):\n",
        "        return list(map(self.splice_scorer, self.splicer(self.seq)[0]))\n",
        "\n",
        "    def Out(self):\n",
        "        s = self.s\n",
        "        def d(x):\n",
        "            if s > 0.7:\n",
        "                return x * np.exp(-1.2 * (x - s))\n",
        "            else:\n",
        "                return x if x > 0.7 else x\n",
        "\n",
        "        return list(map(d, self.mean(self.attention_mask(), self.T())))\n",
        "def show(i):\n",
        "  values = np.array(i).reshape(1, len(i))\n",
        "  y = range(len(i))\n",
        "  Y = [(j,1) for j in range(len(i)) if np.mean([i[j],i[min((j+1),len(i)-1)],i[max(0,(j-1))]])>0.6]\n",
        "  Y2 = [(j,1) for j in range(len(i)) if np.mean([i[j],i[min((j+1),len(i)-1)],i[min((j+2),len(i)-1)],i[min((j+3),len(i)-1)],i[min((j+4),len(i)-1)],i[min((j+5),len(i)-1)],i[max(0,(j-1))],i[max(0,(j-2))],i[max(0,(j-3))],i[max(0,(j-4))],i[max(0,(j-5))]])>0.85]\n",
        "  threshold = 0.5\n",
        "  above_threshold = np.maximum(values - threshold, 0)\n",
        "  below_threshold = np.minimum(values, threshold)\n",
        "  fig, (ax3 , ax4) = plt.subplots(2,1,sharex = True,sharey = True)\n",
        "  ax3.bar(y, below_threshold.ravel(), 0.9, color=\"lightblue\")\n",
        "  ax3.bar(y, above_threshold.ravel(), 0.9, color=\"lightgreen\",\n",
        "               bottom=below_threshold.ravel())\n",
        "  ax4.broken_barh(Y, (0.7, 0.03),\n",
        "               facecolors=( 'red'))\n",
        "  ax4.plot([0., y[-1]],[0.715,0.715],\"-b\")\n",
        "  for k in Y:\n",
        "    ax4.plot([k[0],k[0]+1],[0.715,0.715],\"-r\")\n",
        "\n",
        "  ax4.text(0,0.8 , \"LLPS-susceptible regions\",fontsize = 20)\n",
        "  ax3.plot([0., y[-1]], [threshold, threshold], \"k--\")\n",
        "  ax4.set_ylim([0,1])\n",
        "  ax3.set_ylim([0,1])\n",
        "\n",
        "  plt.subplots_adjust(\n",
        "                    wspace=0.4,\n",
        "                    hspace=0)\n",
        "  fig.set_figwidth(20)\n",
        "  fig.set_figheight(15)\n",
        "  plt.show()\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 33,
      "metadata": {
        "cellView": "form",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "YV1vl5J0Mn5W",
        "outputId": "bf64ba28-cfd2-4c50-ba96-b81f321d8154"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Score : 0.32609919454801944\n"
          ]
        }
      ],
      "source": [
        "#@title Loading sequence , then hit `Runtime` -> `Run all`\n",
        "#@markdown - Load the sequence of your protein :\n",
        "Sequence = \"NSDSECPLSHDGYCLHDGVCMYIEALDKYACNCVVGYIGERCQYRDLKWWELR\" #@param {type:\"string\"}\n",
        "ID = None #@param {type:\"string\"}\n",
        "Directory = None #@param {type:\"string\"}\n",
        "#@markdown  - Use `End_sequence` to focus the scoring on sequences up to the specified endpoint\n",
        "\n",
        "End_sequence = 1000 #@param {type:\"slider\" , min : 0 , max: 1000 , step : 1}\n",
        "Plot = True #@param {type: \"boolean\"}\n",
        "\n",
        "#@markdown - Specify the path of `FASTA format file` of your sequences.\n",
        "Fasta_file = \"synthestic.fasta\" #@param {type:\"string\"}\n",
        "import pandas as pd\n",
        "#assert [Sequence == None , Fasta_file == None].count(True) == 1\n",
        "model = model3\n",
        "def edit(i):\n",
        "  for j in i:\n",
        "    if j in \"\\t\\nBJOUXZ, _@#$%^& *n({}[])\\/\\\\\":\n",
        "      i = i.replace(j,\"\")\n",
        "  return i.upper()\n",
        "\n",
        "def rescale_scores(scores, original_threshold=0.454253, desired_threshold=0.5):\n",
        "    return (scores-original_threshold)*(1-desired_threshold)/(1-original_threshold)+desired_threshold\n",
        "def enh(x):\n",
        "  k = 50\n",
        "  wei = np.array([1 / (1 + np.exp(-k * (2 * i - 1))) for i in x])\n",
        "  return np.sum(np.array(x)*wei)/np.sum(wei)\n",
        "\n",
        "def att(x):\n",
        "  k = 100\n",
        "  wei = np.array([1 / (1 + np.exp(k * (2 * i - 1))) for i in x])\n",
        "  return np.sum(np.array(x)*wei)/np.sum(wei)\n",
        "\n",
        "\n",
        "\n",
        "def SCORE(i):\n",
        "  def scorer(i):\n",
        "    i = np.array(i).reshape(1,5538)\n",
        "    ii = clf.predict_proba(i)[:,1][0]\n",
        "    TT = [j for j in i.ravel() if j != 0]\n",
        "    S = rescale_scores(0.3*i[0][-1]+0.4*enh(TT[:-1])+0.3*ii)\n",
        "    return S\n",
        "  TT = [j for j in i if j != 0]\n",
        "  if len(TT)>5537:\n",
        "    truncated_p = [TT[i*5537:(i+1)*5537] for i in range(len(TT)//5537+1)]\n",
        "    S = enh([scorer(padd(p1)+[i[-1]]) for p1 in truncated_p])\n",
        "  else:\n",
        "    S = scorer(i)\n",
        "  return S\n",
        "\n",
        "def SW(s):\n",
        "  if len(s)> 700:\n",
        "    L= [i for i in  [s[i*700:(i+1)*700] for i in range(len(s)//700+1)] if len(i)>0]\n",
        "    H = []\n",
        "    for i in range(len(L)):\n",
        "      H.append(model.predict_proba(L[i]))\n",
        "    return H\n",
        "  else:\n",
        "    return model.predict_proba(s)\n",
        "\n",
        "\n",
        "def Score1(i,Sc):\n",
        "    i = i-1\n",
        "    if i<= L:\n",
        "      return sum([Sc[j] for j in range(i+1)])/(i+1)\n",
        "\n",
        "    elif L+1<=i<n-L:\n",
        "       return sum([Sc[j] for j in range(i-L,i+1)])/(L+1)\n",
        "\n",
        "    else:\n",
        "       u = (n-(i)+1)\n",
        "       return sum([Sc[j] for j in range(i-L,n-L+1)])/(u)\n",
        "Sequence = edit(Sequence)\n",
        "\n",
        "\n",
        "def d(x):\n",
        "    if u>0.7:\n",
        "        return x*np.exp(-1.2*(x-u))\n",
        "    else:\n",
        "        if x > 0.7 :\n",
        "            return x\n",
        "        else:\n",
        "            return x\n",
        "def padd(i):\n",
        "  i = i+[float(0)]*(5537-len(i))\n",
        "  return i\n",
        "\n",
        "if Sequence != None and Sequence !=\"\":\n",
        "  Sequence = edit(Sequence)\n",
        "  k = max(5,min(50,int(np.ceil(0.1*len(Sequence)))))\n",
        "  L = k//2\n",
        "  if Plot == True:\n",
        "    n = len(Sequence)\n",
        "    S = [Sequence[i:i+L] for i in range(len(Sequence)-L+1)]\n",
        "    Sc = SW(S)\n",
        "    Sc1 = Sc[0]\n",
        "    for i in Sc[1:]:\n",
        "      Sc1 = np.concatenate((Sc1,i))\n",
        "    u = model.predict_proba([Sequence])\n",
        "\n",
        "    p = list(map(d,[Score1(i,Sc1) for i in range(1,n+1)]))\n",
        "    if type(p[0]) != np.float64:\n",
        "      p = [i[0] for i in p]\n",
        "    if type(p[0]) != np.float64:\n",
        "      p = [i[0] for i in p]\n",
        "\n",
        "    i = p\n",
        "    values = np.array(i).reshape(1, len(i))\n",
        "    below_threshold = [k if k<= 0.5 else 0.5 for k in i]\n",
        "    above_threshold = [0 if k<= 0.5 else k-0.5 for k in i]\n",
        "    threshold = 0.5\n",
        "    y = list(range(len(i)))\n",
        "    Y = [(j, 1) for j in range(len(i)) if np.mean([i[j], i[min((j + 1), len(i) - 1)], i[max(0, (j - 1))]]) > 0.6]\n",
        "    Y2 = [(j, 1) for j in range(len(i)) if np.mean([i[j], i[min((j + 1), len(i) - 1)], i[min((j + 2), len(i) - 1)],\n",
        "                                               i[min((j + 3), len(i) - 1)], i[min((j + 4), len(i) - 1)],\n",
        "                                               i[min((j + 5), len(i) - 1)], i[max(0, (j - 1))],\n",
        "                                               i[max(0, (j - 2))], i[max(0, (j - 3))], i[max(0, (j - 4))],\n",
        "                                               i[max(0, (j - 5))]]) > 0.85]\n",
        "    #if n>= 1500:\n",
        "      #c1 = \"blue\"\n",
        "      #c2 = \"green\"\n",
        "    #else:\n",
        "    c1 = \"lightblue\"\n",
        "    c2 = \"lightgreen\"\n",
        "\n",
        "    fig1 = make_subplots(rows=1, cols=1, shared_xaxes=True, shared_yaxes=True, vertical_spacing=0.02)\n",
        "    fig1.add_trace(go.Bar(x=y, y=below_threshold, marker_color= c1, name='Below Threshold', width= 0.6), row=1, col=1)\n",
        "    fig1.add_trace(go.Bar(x=y, y=above_threshold, marker_color=c2, name='Above Threshold', base=below_threshold, width= 0.6), row=1, col=1)\n",
        "\n",
        "    fig1.add_trace(go.Scatter(x=[0, len(i) - 1], y=[threshold, threshold], mode='lines', line=dict(color='black', dash='dash'), showlegend=False), row=1, col=1)\n",
        "    for start, duration in Y:\n",
        "        fig1.add_trace(go.Scatter(x=[start, start + duration], y=[0.01, 0.01], mode='lines', line=dict(color='red', width=7), showlegend=False), row=1, col=1)\n",
        "    fig1.update_layout(height=800, width=1700, title_text=\"LLPS Prediction Plot\")\n",
        "    if type(u) != np.float64:\n",
        "      u = u[0][0]\n",
        "    daa = padd(p)+[u]\n",
        "    score = str(SCORE(daa))\n",
        "    print(\"Score : \"+score)\n",
        "\n",
        "\n",
        "else:\n",
        "  if os.path.exists(Fasta_file):\n",
        "    End_sequence = min(End_sequence,len(U))\n",
        "    D = {\"id\":[str(i.id) for i in SeqIO.parse(Fasta_file,\"fasta\")],\"seq\":[edit(str(i.seq)) for i in SeqIO.parse(Fasta_file,\"fasta\")],\"score\":[]}\n",
        "    for sequence in tqdm(D[\"seq\"]):\n",
        "      try:\n",
        "        k = max(5,min(50,int(np.ceil(0.1*len(sequence)))))\n",
        "        L = k//2\n",
        "        n = len(sequence)\n",
        "        S = [sequence[i:i+L] for i in range(len(sequence)-L+1)]\n",
        "        Sc = SW(S)\n",
        "        Sc1 = Sc[0]\n",
        "        for i in Sc[1:]:\n",
        "            Sc1 = np.concatenate((Sc1,i))\n",
        "        u = model.predict_proba([sequence])\n",
        "        p = list(map(d,[Score1(i,Sc1) for i in range(1,n+1)]))\n",
        "        if type(p[0]) != np.float64:\n",
        "            p = [i[0] for i in p]\n",
        "        if type(p[0]) != np.float64:\n",
        "            p = [i[0] for i in p]\n",
        "\n",
        "        if type(u) != np.float64:\n",
        "            u = u[0][0]\n",
        "        daa = padd(p)+[u]\n",
        "        score = str(SCORE(daa))\n",
        "        D[\"score\"].append(score)\n",
        "      except:\n",
        "        D[\"score\"].append(None)\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "Sequence[25:90]"
      ],
      "metadata": {
        "id": "QI4Zqlo_KUuE",
        "outputId": "28090770-3380-4ff8-b7ae-4403e332f5c7",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 36
        }
      },
      "execution_count": 30,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'STSPLSADPPVAAAVVSHFNDCPDSHTQFCFHGTCRFLVQEDKPACVCHSGYVGARCEHADLLAV'"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            }
          },
          "metadata": {},
          "execution_count": 30
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "\"NSDSECPLSHDGYCLHDGVCMYIEALDKYACNCVVGYIGERCQYRDLKWWELR\""
      ],
      "metadata": {
        "id": "6x0kvalxLqU3"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!wget http://zhanggroup.org/TM-align/TMalign.tar.gz\n",
        "!tar -xzf TMalign.tar.gz\n"
      ],
      "metadata": {
        "id": "S_fLgrVNMkJ3",
        "outputId": "496f6ba1-afcf-4324-e168-1300d6e203c8",
        "colab": {
          "base_uri": "https://localhost:8080/"
        }
      },
      "execution_count": 35,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2024-10-29 12:13:49--  http://zhanggroup.org/TM-align/TMalign.tar.gz\n",
            "Resolving zhanggroup.org (zhanggroup.org)... 141.213.137.249\n",
            "Connecting to zhanggroup.org (zhanggroup.org)|141.213.137.249|:80... connected.\n",
            "HTTP request sent, awaiting response... 301 Moved Permanently\n",
            "Location: https://zhanggroup.org/TM-align/TMalign.tar.gz [following]\n",
            "--2024-10-29 12:13:49--  https://zhanggroup.org/TM-align/TMalign.tar.gz\n",
            "Connecting to zhanggroup.org (zhanggroup.org)|141.213.137.249|:443... connected.\n",
            "HTTP request sent, awaiting response... 404 Not Found\n",
            "2024-10-29 12:13:50 ERROR 404: Not Found.\n",
            "\n",
            "tar (child): TMalign.tar.gz: Cannot open: No such file or directory\n",
            "tar (child): Error is not recoverable: exiting now\n",
            "tar: Child returned status 2\n",
            "tar: Error is not recoverable: exiting now\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from google.colab import files\n",
        "uploaded = files.upload()\n"
      ],
      "metadata": {
        "id": "Yh_H1bfqMnZ7"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@title plot\n",
        "import matplotlib.pyplot as plt\n",
        "import streamlit as st\n",
        "from stmol import showmol\n",
        "import py3Dmol\n",
        "import requests\n",
        "import biotite.structure.io as bsio\n",
        "import certifi\n",
        "import matplotlib.colors as mcolors\n",
        "import urllib3\n",
        "\n",
        "if Plot == True and Sequence != \"\" and Sequence != None:\n",
        "  fig1.show()\n",
        "\n",
        "\n",
        "  urllib3.disable_warnings()\n",
        "\n",
        "  st.sidebar.title('üéà ESMFold')\n",
        "  st.sidebar.write('[*ESMFold*](https://esmatlas.com/about) is an end-to-end single sequence protein structure predictor based on the ESM-2 language model. For more information, read the [research article](https://www.biorxiv.org/content/10.1101/2022.07.20.500902v2) and the [news article](https://www.nature.com/articles/d41586-022-03539-1) published in *Nature*.')\n",
        "\n",
        "  def render_mol(pdb):\n",
        "    pdbview = py3Dmol.view()\n",
        "    pdbview.addModel(pdb, 'pdb')\n",
        "    pdbview.setStyle({'cartoon': {'color': 'spectrum'}})\n",
        "    pdbview.setBackgroundColor('white')  # ('0xeeeeee')\n",
        "    pdbview.zoomTo()\n",
        "    pdbview.zoom(2, 800)\n",
        "    pdbview.spin(True)\n",
        "    showmol(pdbview, height=500, width=800)\n",
        "\n",
        "\n",
        "  DEFAULT_SEQ = Sequence\n",
        "  show_mainchains = True #@param {type: \"boolean\"}\n",
        "  show_sidechains = True #@param {type: \"boolean\"}\n",
        "  Color_Style =  \"red_blue_spectrum\" #@param [\"red_blue_spectrum\",\"green_magenta_spectrum\",\"yellow_blue_spectrum\",\"cyan_red_spectrum\"]\n",
        "\n",
        "  import streamlit as st\n",
        "  import requests\n",
        "  import biotite.structure.io as bsio\n",
        "  import py3Dmol\n",
        "\n",
        "  DEFAULT_SEQ = Sequence[:400]\n",
        "\n",
        "  txt = st.sidebar.text_area('Input sequence', DEFAULT_SEQ, height=275)\n",
        "\n",
        "  def update(sequence,name):\n",
        "    headers = {\n",
        "        'Content-Type': 'application/x-www-form-urlencoded',\n",
        "    }\n",
        "\n",
        "    def get_pdb(sequence):\n",
        "        response = requests.post('https://api.esmatlas.com/foldSequence/v1/pdb/', headers=headers, data=sequence, verify=False)\n",
        "        if response.status_code != 200:\n",
        "            st.error(f\"Failed to get prediction: {response.status_code}\")\n",
        "            return None\n",
        "        pdb_string = response.content.decode('utf-8')\n",
        "        if not pdb_string or \"ATOM\" not in pdb_string:\n",
        "            st.error(\"Received an invalid PDB file.\")\n",
        "            return None\n",
        "        return pdb_string\n",
        "\n",
        "    def save_and_render_pdb(pdb_string, file_name):\n",
        "        with open(file_name, 'w') as f:\n",
        "            f.write(pdb_string)\n",
        "        try:\n",
        "            struct = bsio.load_structure(file_name, extra_fields=[\"b_factor\"])\n",
        "            b_value = round(struct.b_factor.mean(), 4)\n",
        "        except Exception as e:\n",
        "            st.error(f\"Error loading structure: {e}\")\n",
        "            return\n",
        "        st.subheader('Visualization of predicted protein structure')\n",
        "        render_mol(pdb_string)\n",
        "        st.subheader('plDDT')\n",
        "        st.write('plDDT is a per-residue estimate of the confidence in prediction on a scale from 0-100.')\n",
        "        st.info(f'plDDT: {b_value}')\n",
        "        st.download_button(label=\"Download PDB\", data=pdb_string, file_name=file_name, mime='text/plain')\n",
        "\n",
        "    def render_mol(pdb_string,neme):\n",
        "        view = py3Dmol.view(width=1000, height=800)\n",
        "        view.addModel(pdb_string, 'pdb')\n",
        "        F1 = Get_color(p)\n",
        "        F1.i = F1.J.index(Color_Style)\n",
        "\n",
        "        custom_colors=F1.spectrum()\n",
        "\n",
        "        for i, color in enumerate(custom_colors[1:-1]):\n",
        "          view.setStyle({ 'resi': i+1}, {'cartoon': {'color': color}})\n",
        "        #view.setStyle({'cartoon': {'color': 'spectrum'}})\n",
        "        view.zoomTo()\n",
        "        view.show()\n",
        "        html_content = view._make_html()\n",
        "        with open(name+'.html', 'w') as file:\n",
        "          file.write(html_content)\n",
        "        files.download(name+'.html')\n",
        "\n",
        "\n",
        "    chunk_size = 400\n",
        "    chunks = [sequence[i:i + chunk_size] for i in range(0, len(sequence), chunk_size)]\n",
        "\n",
        "    combined_pdb_string = \"\"\n",
        "    for i, chunk in enumerate(chunks):\n",
        "        st.write(f\"Processing chunk {i + 1} of {len(chunks)}\")\n",
        "        pdb_string = get_pdb(chunk)\n",
        "        if pdb_string:\n",
        "            combined_pdb_string += pdb_string\n",
        "\n",
        "    if combined_pdb_string:\n",
        "        render_mol(combined_pdb_string,name)\n",
        "        with open('combined_predicted.pdb', 'w') as f:\n",
        "            f.write(combined_pdb_string)\n",
        "        st.download_button(label=\"Download Combined PDB\", data=combined_pdb_string, file_name='combined_predicted.pdb', mime='text/plain')\n",
        "\n",
        "  predict = st.sidebar.button('Predict', on_click=update)\n",
        "\n",
        "  if not predict:\n",
        "    st.warning('üëà Enter protein sequence data!')\n",
        "\n",
        "  class Get_color():\n",
        "    def __init__(self,values):\n",
        "      self.J = [\"red_blue_spectrum\",\"green_magenta_spectrum\",\"yellow_blue_spectrum\",\"cyan_red_spectrum\"]\n",
        "      self.i = None\n",
        "      self.values = values\n",
        "\n",
        "    def get_red_blue_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"RedBlue\", [\"blue\", \"red\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "    def get_green_magenta_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"GreenMagenta\", [\"green\", \"magenta\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "    def get_yellow_blue_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"YellowBlue\", [\"yellow\", \"blue\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "\n",
        "    def get_cyan_red_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"CyanRed\", [\"cyan\", \"red\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "    def spectrum(self):\n",
        "      if self.i == 0 : return self.get_red_blue_spectrum(self.values)\n",
        "      if self.i == 1 : return self.get_green_magenta_spectrum(self.values)\n",
        "      if self.i == 2 : return self.get_yellow_blue_spectrum(self.values)\n",
        "      if self.i == 3 : return self.get_cyan_red_spectrum(self.values)\n",
        "  update(txt,\"A\")\n",
        "\n",
        "\n",
        "# Create a color bar legend using matplotlib\n",
        "  fig, ax = plt.subplots(figsize=(6, 1))\n",
        "  fig.subplots_adjust(bottom=0.5)\n",
        "\n",
        "  cmap = plt.cm.Reds\n",
        "  norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "  fig.colorbar(plt.cm.ScalarMappable(norm=norm, cmap=cmap), cax=ax, orientation='horizontal', label='Predicted LLPS Score')\n",
        "  plt.show()\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n"
      ],
      "metadata": {
        "cellView": "form",
        "id": "1il0zXHeK7Q5",
        "outputId": "031f422e-667e-422e-b4b0-d5efc1db238f",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        }
      },
      "execution_count": 32,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<html>\n",
              "<head><meta charset=\"utf-8\" /></head>\n",
              "<body>\n",
              "    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax && window.MathJax.Hub && window.MathJax.Hub.Config) {window.MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n",
              "        <script charset=\"utf-8\" src=\"https://cdn.plot.ly/plotly-2.35.2.min.js\"></script>                <div id=\"cdf919e4-1c9e-4d8a-aaaa-47848ae0504d\" class=\"plotly-graph-div\" style=\"height:800px; width:1700px;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"cdf919e4-1c9e-4d8a-aaaa-47848ae0504d\")) {                    Plotly.newPlot(                        \"cdf919e4-1c9e-4d8a-aaaa-47848ae0504d\",                        [{\"marker\":{\"color\":\"lightblue\"},\"name\":\"Below Threshold\",\"width\":0.6,\"x\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64],\"y\":[0.04936041682958603,0.0499713234603405,0.04981006930271784,0.04957994353026152,0.124652368016541,0.12437185645103455,0.12441925331950188,0.12982815597206354,0.277734094299376,0.4841757770627737,0.5,0.5,0.35242687352001667,0.15428153425455093,0.19661789201200008,0.1735351625829935,0.3469053450971842,0.33845852222293615,0.2796736415475607,0.44022304005920887,0.45895680971443653,0.46112387627363205,0.5,0.5,0.2827881360426545,0.49990879744291306,0.27696017920970917,0.3873866256326437,0.5,0.39292434975504875,0.39240504056215286,0.49411914497613907,0.27286455780267715,0.49079519882798195,0.5,0.36104258988052607,0.36261555552482605,0.1396849974989891,0.051056843250989914,0.27556199114769697,0.49521923437714577,0.5,0.5,0.29604776948690414,0.2967839017510414,0.2915316205471754,0.5,0.4994291178882122,0.3028389997780323,0.5,0.5,0.5,0.5,0.5,0.4972520638257265,0.5,0.5,0.5,0.33177609369158745,0.3014937825500965,0.297627161256969,0.07662972528487444,0.05766122601926327,0.05485443025827408,0.05439087562263012],\"type\":\"bar\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"base\":[0.04936041682958603,0.0499713234603405,0.04981006930271784,0.04957994353026152,0.124652368016541,0.12437185645103455,0.12441925331950188,0.12982815597206354,0.277734094299376,0.4841757770627737,0.5,0.5,0.35242687352001667,0.15428153425455093,0.19661789201200008,0.1735351625829935,0.3469053450971842,0.33845852222293615,0.2796736415475607,0.44022304005920887,0.45895680971443653,0.46112387627363205,0.5,0.5,0.2827881360426545,0.49990879744291306,0.27696017920970917,0.3873866256326437,0.5,0.39292434975504875,0.39240504056215286,0.49411914497613907,0.27286455780267715,0.49079519882798195,0.5,0.36104258988052607,0.36261555552482605,0.1396849974989891,0.051056843250989914,0.27556199114769697,0.49521923437714577,0.5,0.5,0.29604776948690414,0.2967839017510414,0.2915316205471754,0.5,0.4994291178882122,0.3028389997780323,0.5,0.5,0.5,0.5,0.5,0.4972520638257265,0.5,0.5,0.5,0.33177609369158745,0.3014937825500965,0.297627161256969,0.07662972528487444,0.05766122601926327,0.05485443025827408,0.05439087562263012],\"marker\":{\"color\":\"lightgreen\"},\"name\":\"Above Threshold\",\"width\":0.6,\"x\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64],\"y\":[0,0,0,0,0,0,0,0,0,0,0.00016165152192115784,0.04431295581161976,0,0,0,0,0,0,0,0,0,0,0.1845971206203103,0.004408801905810833,0,0,0,0,0.10720998793840408,0,0,0,0,0,0.07967639714479446,0,0,0,0,0,0,0.003633236512541771,0.0037952233105897903,0,0,0,0.013474499806761742,0,0,0.02406986430287361,0.026885051280260086,0.028658894822001457,0.003233032301068306,0.0035668183118104935,0,0.027000930160284042,0.03237812966108322,0.030173562467098236,0,0,0,0,0,0,0],\"type\":\"bar\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"black\",\"dash\":\"dash\"},\"mode\":\"lines\",\"showlegend\":false,\"x\":[0,64],\"y\":[0.5,0.5],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"}],                        {\"template\":{\"data\":{\"candlestick\":[{\"decreasing\":{\"line\":{\"color\":\"#000033\"}},\"increasing\":{\"line\":{\"color\":\"#000032\"}},\"type\":\"candlestick\"}],\"contourcarpet\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"contourcarpet\"}],\"contour\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"contour\"}],\"heatmap\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"heatmap\"}],\"histogram2d\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"histogram2d\"}],\"icicle\":[{\"textfont\":{\"color\":\"white\"},\"type\":\"icicle\"}],\"sankey\":[{\"textfont\":{\"color\":\"#000036\"},\"type\":\"sankey\"}],\"scatter\":[{\"marker\":{\"line\":{\"width\":0}},\"type\":\"scatter\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#000038\"},\"font\":{\"color\":\"#000037\"},\"line\":{\"color\":\"#000039\"}},\"header\":{\"fill\":{\"color\":\"#000040\"},\"font\":{\"color\":\"#000036\"},\"line\":{\"color\":\"#000039\"}},\"type\":\"table\"}],\"waterfall\":[{\"connector\":{\"line\":{\"color\":\"#000036\",\"width\":2}},\"decreasing\":{\"marker\":{\"color\":\"#000033\"}},\"increasing\":{\"marker\":{\"color\":\"#000032\"}},\"totals\":{\"marker\":{\"color\":\"#000034\"}},\"type\":\"waterfall\"}]},\"layout\":{\"coloraxis\":{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]]},\"colorscale\":{\"diverging\":[[0.0,\"#000021\"],[0.1,\"#000022\"],[0.2,\"#000023\"],[0.3,\"#000024\"],[0.4,\"#000025\"],[0.5,\"#000026\"],[0.6,\"#000027\"],[0.7,\"#000028\"],[0.8,\"#000029\"],[0.9,\"#000030\"],[1.0,\"#000031\"]],\"sequential\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"sequentialminus\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]]},\"colorway\":[\"#000001\",\"#000002\",\"#000003\",\"#000004\",\"#000005\",\"#000006\",\"#000007\",\"#000008\",\"#000009\",\"#000010\"]}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0]},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0]},\"title\":{\"text\":\"LLPS Prediction Plot\"},\"height\":800,\"width\":1700},                        {\"responsive\": true}                    ).then(function(){\n",
              "                            \n",
              "var gd = document.getElementById('cdf919e4-1c9e-4d8a-aaaa-47848ae0504d');\n",
              "var x = new MutationObserver(function (mutations, observer) {{\n",
              "        var display = window.getComputedStyle(gd).display;\n",
              "        if (!display || display === 'none') {{\n",
              "            console.log([gd, 'removed!']);\n",
              "            Plotly.purge(gd);\n",
              "            observer.disconnect();\n",
              "        }}\n",
              "}});\n",
              "\n",
              "// Listen for the removal of the full notebook cells\n",
              "var notebookContainer = gd.closest('#notebook-container');\n",
              "if (notebookContainer) {{\n",
              "    x.observe(notebookContainer, {childList: true});\n",
              "}}\n",
              "\n",
              "// Listen for the clearing of the current output cell\n",
              "var outputEl = gd.closest('.output');\n",
              "if (outputEl) {{\n",
              "    x.observe(outputEl, {childList: true});\n",
              "}}\n",
              "\n",
              "                        })                };                            </script>        </div>\n",
              "</body>\n",
              "</html>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "2024-10-29 12:09:17.304 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.306 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.308 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.310 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.310 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.312 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.313 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.314 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.315 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.317 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.319 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.320 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.322 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.323 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.324 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.325 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.327 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.328 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.329 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.330 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.331 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.332 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:17.333 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17302037596177619\"  style=\"position: relative; width: 1000px; height: 800px;\">\n        <p id=\"3dmolwarning_17302037596177619\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.0/3Dmol-min.js');\n}\n\nvar viewer_17302037596177619 = null;\nvar warn = document.getElementById(\"3dmolwarning_17302037596177619\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17302037596177619 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17302037596177619\"),{backgroundColor:\"white\"});\nviewer_17302037596177619.zoomTo();\n\tviewer_17302037596177619.addModel(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   SER A   1      -2.038  12.334 -43.299  1.00  0.47           N  \\nATOM      2  CA  SER A   1      -2.858  11.164 -42.998  1.00  0.53           C  \\nATOM      3  C   SER A   1      -3.169  11.075 -41.508  1.00  0.50           C  \\nATOM      4  CB  SER A   1      -2.157   9.885 -43.459  1.00  0.44           C  \\nATOM      5  O   SER A   1      -2.258  11.077 -40.677  1.00  0.48           O  \\nATOM      6  OG  SER A   1      -3.034   8.775 -43.376  1.00  0.47           O  \\nATOM      7  N   THR A   2      -3.907  12.144 -41.043  1.00  0.54           N  \\nATOM      8  CA  THR A   2      -4.608  12.316 -39.776  1.00  0.56           C  \\nATOM      9  C   THR A   2      -5.167  10.985 -39.282  1.00  0.55           C  \\nATOM     10  CB  THR A   2      -5.752  13.339 -39.904  1.00  0.50           C  \\nATOM     11  O   THR A   2      -5.981  10.357 -39.961  1.00  0.53           O  \\nATOM     12  CG2 THR A   2      -5.210  14.764 -39.973  1.00  0.44           C  \\nATOM     13  OG1 THR A   2      -6.498  13.066 -41.096  1.00  0.50           O  \\nATOM     14  N   SER A   3      -4.215   9.946 -38.896  1.00  0.56           N  \\nATOM     15  CA  SER A   3      -4.893   8.792 -38.315  1.00  0.56           C  \\nATOM     16  C   SER A   3      -5.906   9.221 -37.258  1.00  0.56           C  \\nATOM     17  CB  SER A   3      -3.879   7.827 -37.699  1.00  0.50           C  \\nATOM     18  O   SER A   3      -5.628  10.108 -36.449  1.00  0.54           O  \\nATOM     19  OG  SER A   3      -2.975   7.353 -38.682  1.00  0.50           O  \\nATOM     20  N   PRO A   4      -7.327   9.272 -37.567  1.00  0.54           N  \\nATOM     21  CA  PRO A   4      -8.238   9.442 -36.433  1.00  0.54           C  \\nATOM     22  C   PRO A   4      -7.634   8.967 -35.114  1.00  0.54           C  \\nATOM     23  CB  PRO A   4      -9.446   8.584 -36.821  1.00  0.50           C  \\nATOM     24  O   PRO A   4      -6.969   7.928 -35.074  1.00  0.52           O  \\nATOM     25  CG  PRO A   4      -8.987   7.789 -38.000  1.00  0.48           C  \\nATOM     26  CD  PRO A   4      -7.673   8.344 -38.468  1.00  0.49           C  \\nATOM     27  N   LEU A   5      -6.824   9.851 -34.347  1.00  0.51           N  \\nATOM     28  CA  LEU A   5      -6.752   9.455 -32.945  1.00  0.51           C  \\nATOM     29  C   LEU A   5      -7.472   8.131 -32.716  1.00  0.51           C  \\nATOM     30  CB  LEU A   5      -7.356  10.540 -32.050  1.00  0.47           C  \\nATOM     31  O   LEU A   5      -8.675   8.022 -32.968  1.00  0.49           O  \\nATOM     32  CG  LEU A   5      -6.447  11.720 -31.706  1.00  0.46           C  \\nATOM     33  CD1 LEU A   5      -7.259  13.009 -31.624  1.00  0.44           C  \\nATOM     34  CD2 LEU A   5      -5.708  11.463 -30.397  1.00  0.44           C  \\nATOM     35  N   SER A   6      -7.329   7.026 -33.488  1.00  0.49           N  \\nATOM     36  CA  SER A   6      -7.822   5.743 -32.998  1.00  0.52           C  \\nATOM     37  C   SER A   6      -8.137   5.804 -31.507  1.00  0.50           C  \\nATOM     38  CB  SER A   6      -6.800   4.638 -33.267  1.00  0.46           C  \\nATOM     39  O   SER A   6      -7.293   6.208 -30.705  1.00  0.48           O  \\nATOM     40  OG  SER A   6      -7.088   3.487 -32.491  1.00  0.45           O  \\nATOM     41  N   ALA A   7      -9.098   6.608 -31.113  1.00  0.47           N  \\nATOM     42  CA  ALA A   7      -9.959   6.490 -29.939  1.00  0.49           C  \\nATOM     43  C   ALA A   7      -9.888   5.085 -29.347  1.00  0.48           C  \\nATOM     44  CB  ALA A   7     -11.401   6.841 -30.298  1.00  0.44           C  \\nATOM     45  O   ALA A   7     -10.527   4.159 -29.852  1.00  0.46           O  \\nATOM     46  N   ASP A   8      -8.815   4.312 -29.207  1.00  0.47           N  \\nATOM     47  CA  ASP A   8      -8.867   3.410 -28.061  1.00  0.48           C  \\nATOM     48  C   ASP A   8      -9.067   4.185 -26.761  1.00  0.47           C  \\nATOM     49  CB  ASP A   8      -7.591   2.570 -27.982  1.00  0.44           C  \\nATOM     50  O   ASP A   8      -8.453   5.234 -26.557  1.00  0.45           O  \\nATOM     51  CG  ASP A   8      -7.496   1.529 -29.083  1.00  0.43           C  \\nATOM     52  OD1 ASP A   8      -8.545   1.012 -29.526  1.00  0.43           O  \\nATOM     53  OD2 ASP A   8      -6.363   1.220 -29.511  1.00  0.48           O  \\nATOM     54  N   PRO A   9     -10.359   4.479 -26.370  1.00  0.48           N  \\nATOM     55  CA  PRO A   9     -10.291   5.081 -25.037  1.00  0.48           C  \\nATOM     56  C   PRO A   9      -8.929   4.895 -24.373  1.00  0.49           C  \\nATOM     57  CB  PRO A   9     -11.383   4.341 -24.260  1.00  0.46           C  \\nATOM     58  O   PRO A   9      -8.201   3.954 -24.703  1.00  0.47           O  \\nATOM     59  CG  PRO A   9     -11.689   3.135 -25.087  1.00  0.44           C  \\nATOM     60  CD  PRO A   9     -11.136   3.349 -26.467  1.00  0.45           C  \\nATOM     61  N   PRO A  10      -8.115   6.032 -24.138  1.00  0.47           N  \\nATOM     62  CA  PRO A  10      -7.130   5.621 -23.135  1.00  0.47           C  \\nATOM     63  C   PRO A  10      -7.594   4.425 -22.308  1.00  0.47           C  \\nATOM     64  CB  PRO A  10      -6.980   6.867 -22.257  1.00  0.45           C  \\nATOM     65  O   PRO A  10      -8.777   4.320 -21.976  1.00  0.45           O  \\nATOM     66  CG  PRO A  10      -8.116   7.753 -22.655  1.00  0.42           C  \\nATOM     67  CD  PRO A  10      -8.720   7.216 -23.921  1.00  0.43           C  \\nATOM     68  N   VAL A  11      -7.696   3.073 -22.725  1.00  0.44           N  \\nATOM     69  CA  VAL A  11      -7.764   2.172 -21.579  1.00  0.46           C  \\nATOM     70  C   VAL A  11      -8.191   2.948 -20.335  1.00  0.44           C  \\nATOM     71  CB  VAL A  11      -6.412   1.467 -21.329  1.00  0.43           C  \\nATOM     72  O   VAL A  11      -7.566   3.949 -19.976  1.00  0.43           O  \\nATOM     73  CG1 VAL A  11      -6.545   0.418 -20.227  1.00  0.41           C  \\nATOM     74  CG2 VAL A  11      -5.897   0.830 -22.619  1.00  0.41           C  \\nATOM     75  N   ALA A  12      -9.421   3.665 -20.370  1.00  0.43           N  \\nATOM     76  CA  ALA A  12      -9.817   3.924 -18.988  1.00  0.44           C  \\nATOM     77  C   ALA A  12      -8.853   3.260 -18.008  1.00  0.43           C  \\nATOM     78  CB  ALA A  12     -11.242   3.435 -18.742  1.00  0.41           C  \\nATOM     79  O   ALA A  12      -8.557   2.069 -18.130  1.00  0.41           O  \\nATOM     80  N   ALA A  13      -7.543   3.638 -18.005  1.00  0.44           N  \\nATOM     81  CA  ALA A  13      -6.895   3.360 -16.726  1.00  0.45           C  \\nATOM     82  C   ALA A  13      -7.847   2.643 -15.773  1.00  0.44           C  \\nATOM     83  CB  ALA A  13      -6.387   4.654 -16.094  1.00  0.42           C  \\nATOM     84  O   ALA A  13      -8.978   3.089 -15.564  1.00  0.42           O  \\nATOM     85  N   ALA A  14      -8.522   1.429 -16.199  1.00  0.46           N  \\nATOM     86  CA  ALA A  14      -9.099   0.649 -15.108  1.00  0.46           C  \\nATOM     87  C   ALA A  14      -9.154   1.464 -13.819  1.00  0.46           C  \\nATOM     88  CB  ALA A  14      -8.298  -0.633 -14.889  1.00  0.43           C  \\nATOM     89  O   ALA A  14      -8.140   2.012 -13.380  1.00  0.44           O  \\nATOM     90  N   VAL A  15      -9.878   2.615 -13.856  1.00  0.46           N  \\nATOM     91  CA  VAL A  15     -10.245   3.079 -12.522  1.00  0.46           C  \\nATOM     92  C   VAL A  15      -9.975   1.978 -11.499  1.00  0.46           C  \\nATOM     93  CB  VAL A  15     -11.727   3.512 -12.460  1.00  0.44           C  \\nATOM     94  O   VAL A  15     -10.504   0.871 -11.614  1.00  0.44           O  \\nATOM     95  CG1 VAL A  15     -12.095   3.979 -11.052  1.00  0.40           C  \\nATOM     96  CG2 VAL A  15     -12.004   4.614 -13.481  1.00  0.42           C  \\nATOM     97  N   VAL A  16      -8.781   1.491 -11.315  1.00  0.50           N  \\nATOM     98  CA  VAL A  16      -8.406   0.826 -10.072  1.00  0.49           C  \\nATOM     99  C   VAL A  16      -9.306   1.306  -8.936  1.00  0.50           C  \\nATOM    100  CB  VAL A  16      -6.923   1.077  -9.719  1.00  0.48           C  \\nATOM    101  O   VAL A  16      -9.343   2.499  -8.626  1.00  0.49           O  \\nATOM    102  CG1 VAL A  16      -6.512   0.256  -8.498  1.00  0.45           C  \\nATOM    103  CG2 VAL A  16      -6.028   0.750 -10.913  1.00  0.46           C  \\nATOM    104  N   SER A  17     -10.683   1.068  -8.984  1.00  0.52           N  \\nATOM    105  CA  SER A  17     -11.628   1.272  -7.892  1.00  0.51           C  \\nATOM    106  C   SER A  17     -10.988   0.968  -6.542  1.00  0.52           C  \\nATOM    107  CB  SER A  17     -12.869   0.399  -8.085  1.00  0.49           C  \\nATOM    108  O   SER A  17     -10.264  -0.021  -6.401  1.00  0.50           O  \\nATOM    109  OG  SER A  17     -13.488   0.123  -6.841  1.00  0.47           O  \\nATOM    110  N   HIS A  18     -10.316   1.933  -5.765  1.00  0.52           N  \\nATOM    111  CA  HIS A  18     -10.082   2.300  -4.373  1.00  0.52           C  \\nATOM    112  C   HIS A  18     -11.002   1.523  -3.436  1.00  0.51           C  \\nATOM    113  CB  HIS A  18     -10.279   3.804  -4.175  1.00  0.49           C  \\nATOM    114  O   HIS A  18     -12.224   1.555  -3.593  1.00  0.50           O  \\nATOM    115  CG  HIS A  18      -9.153   4.632  -4.707  1.00  0.48           C  \\nATOM    116  CD2 HIS A  18      -9.056   5.388  -5.826  1.00  0.47           C  \\nATOM    117  ND1 HIS A  18      -7.941   4.741  -4.060  1.00  0.45           N  \\nATOM    118  CE1 HIS A  18      -7.145   5.532  -4.761  1.00  0.44           C  \\nATOM    119  NE2 HIS A  18      -7.798   5.938  -5.837  1.00  0.41           N  \\nATOM    120  N   PHE A  19     -11.021  -0.064  -3.367  1.00  0.55           N  \\nATOM    121  CA  PHE A  19     -11.475  -0.896  -2.260  1.00  0.54           C  \\nATOM    122  C   PHE A  19     -11.050  -2.345  -2.462  1.00  0.55           C  \\nATOM    123  CB  PHE A  19     -12.997  -0.811  -2.111  1.00  0.50           C  \\nATOM    124  O   PHE A  19     -11.842  -3.173  -2.915  1.00  0.52           O  \\nATOM    125  CG  PHE A  19     -13.483   0.508  -1.575  1.00  0.48           C  \\nATOM    126  CD1 PHE A  19     -13.453   0.774  -0.212  1.00  0.45           C  \\nATOM    127  CD2 PHE A  19     -13.969   1.484  -2.436  1.00  0.46           C  \\nATOM    128  CE1 PHE A  19     -13.901   1.995   0.287  1.00  0.46           C  \\nATOM    129  CE2 PHE A  19     -14.419   2.706  -1.945  1.00  0.46           C  \\nATOM    130  CZ  PHE A  19     -14.385   2.959  -0.583  1.00  0.46           C  \\nATOM    131  N   ASN A  20      -9.963  -2.745  -3.148  1.00  0.63           N  \\nATOM    132  CA  ASN A  20      -9.502  -4.129  -3.097  1.00  0.62           C  \\nATOM    133  C   ASN A  20      -8.753  -4.423  -1.800  1.00  0.63           C  \\nATOM    134  CB  ASN A  20      -8.616  -4.444  -4.305  1.00  0.57           C  \\nATOM    135  O   ASN A  20      -7.555  -4.157  -1.696  1.00  0.60           O  \\nATOM    136  CG  ASN A  20      -9.377  -4.396  -5.615  1.00  0.54           C  \\nATOM    137  ND2 ASN A  20      -8.661  -4.154  -6.707  1.00  0.50           N  \\nATOM    138  OD1 ASN A  20     -10.597  -4.573  -5.646  1.00  0.53           O  \\nATOM    139  N   ASP A  21      -9.440  -4.148  -0.739  1.00  0.66           N  \\nATOM    140  CA  ASP A  21      -9.031  -4.698   0.550  1.00  0.65           C  \\nATOM    141  C   ASP A  21      -8.386  -6.071   0.382  1.00  0.66           C  \\nATOM    142  CB  ASP A  21     -10.229  -4.792   1.498  1.00  0.62           C  \\nATOM    143  O   ASP A  21      -8.746  -6.826  -0.524  1.00  0.64           O  \\nATOM    144  CG  ASP A  21     -10.659  -3.443   2.046  1.00  0.59           C  \\nATOM    145  OD1 ASP A  21      -9.860  -2.483   2.001  1.00  0.58           O  \\nATOM    146  OD2 ASP A  21     -11.807  -3.340   2.530  1.00  0.60           O  \\nATOM    147  N   CYS A  22      -7.281  -6.172   0.742  1.00  0.71           N  \\nATOM    148  CA  CYS A  22      -6.613  -7.468   0.769  1.00  0.71           C  \\nATOM    149  C   CYS A  22      -7.535  -8.545   1.327  1.00  0.71           C  \\nATOM    150  CB  CYS A  22      -5.334  -7.395   1.603  1.00  0.68           C  \\nATOM    151  O   CYS A  22      -8.367  -8.268   2.192  1.00  0.68           O  \\nATOM    152  SG  CYS A  22      -3.928  -6.685   0.720  1.00  0.67           S  \\nATOM    153  N   PRO A  23      -7.667  -9.698   0.678  1.00  0.69           N  \\nATOM    154  CA  PRO A  23      -8.330 -10.814   1.358  1.00  0.68           C  \\nATOM    155  C   PRO A  23      -7.867 -10.984   2.803  1.00  0.69           C  \\nATOM    156  CB  PRO A  23      -7.935 -12.027   0.511  1.00  0.66           C  \\nATOM    157  O   PRO A  23      -6.781 -10.524   3.165  1.00  0.68           O  \\nATOM    158  CG  PRO A  23      -7.162 -11.458  -0.634  1.00  0.62           C  \\nATOM    159  CD  PRO A  23      -6.847 -10.021  -0.331  1.00  0.61           C  \\nATOM    160  N   ASP A  24      -8.773 -11.366   3.788  1.00  0.66           N  \\nATOM    161  CA  ASP A  24      -8.495 -11.597   5.203  1.00  0.65           C  \\nATOM    162  C   ASP A  24      -7.178 -12.348   5.388  1.00  0.67           C  \\nATOM    163  CB  ASP A  24      -9.640 -12.374   5.855  1.00  0.61           C  \\nATOM    164  O   ASP A  24      -6.500 -12.182   6.404  1.00  0.66           O  \\nATOM    165  CG  ASP A  24     -10.903 -11.546   6.015  1.00  0.57           C  \\nATOM    166  OD1 ASP A  24     -10.835 -10.303   5.907  1.00  0.58           O  \\nATOM    167  OD2 ASP A  24     -11.976 -12.142   6.253  1.00  0.60           O  \\nATOM    168  N   SER A  25      -6.972 -13.265   4.502  1.00  0.68           N  \\nATOM    169  CA  SER A  25      -5.745 -14.042   4.643  1.00  0.67           C  \\nATOM    170  C   SER A  25      -4.515 -13.141   4.638  1.00  0.67           C  \\nATOM    171  CB  SER A  25      -5.634 -15.076   3.521  1.00  0.62           C  \\nATOM    172  O   SER A  25      -3.449 -13.534   5.116  1.00  0.63           O  \\nATOM    173  OG  SER A  25      -5.648 -14.445   2.252  1.00  0.58           O  \\nATOM    174  N   HIS A  26      -4.727 -11.922   4.235  1.00  0.69           N  \\nATOM    175  CA  HIS A  26      -3.524 -11.120   4.047  1.00  0.69           C  \\nATOM    176  C   HIS A  26      -3.530  -9.896   4.958  1.00  0.70           C  \\nATOM    177  CB  HIS A  26      -3.391 -10.686   2.586  1.00  0.64           C  \\nATOM    178  O   HIS A  26      -2.749  -8.963   4.755  1.00  0.66           O  \\nATOM    179  CG  HIS A  26      -3.032 -11.804   1.659  1.00  0.60           C  \\nATOM    180  CD2 HIS A  26      -1.871 -12.098   1.029  1.00  0.58           C  \\nATOM    181  ND1 HIS A  26      -3.933 -12.778   1.285  1.00  0.54           N  \\nATOM    182  CE1 HIS A  26      -3.338 -13.626   0.463  1.00  0.53           C  \\nATOM    183  NE2 HIS A  26      -2.086 -13.236   0.291  1.00  0.51           N  \\nATOM    184  N   THR A  27      -4.513  -9.842   5.949  1.00  0.68           N  \\nATOM    185  CA  THR A  27      -4.648  -8.710   6.859  1.00  0.67           C  \\nATOM    186  C   THR A  27      -3.349  -8.473   7.623  1.00  0.69           C  \\nATOM    187  CB  THR A  27      -5.801  -8.930   7.857  1.00  0.64           C  \\nATOM    188  O   THR A  27      -3.046  -7.341   8.006  1.00  0.67           O  \\nATOM    189  CG2 THR A  27      -7.148  -8.958   7.143  1.00  0.56           C  \\nATOM    190  OG1 THR A  27      -5.606 -10.176   8.536  1.00  0.58           O  \\nATOM    191  N   GLN A  28      -2.458  -9.617   7.710  1.00  0.75           N  \\nATOM    192  CA  GLN A  28      -1.205  -9.433   8.435  1.00  0.75           C  \\nATOM    193  C   GLN A  28      -0.043  -9.200   7.474  1.00  0.77           C  \\nATOM    194  CB  GLN A  28      -0.916 -10.643   9.325  1.00  0.71           C  \\nATOM    195  O   GLN A  28       1.122  -9.271   7.871  1.00  0.73           O  \\nATOM    196  CG  GLN A  28      -1.933 -10.843  10.440  1.00  0.65           C  \\nATOM    197  CD  GLN A  28      -1.350 -11.557  11.645  1.00  0.62           C  \\nATOM    198  NE2 GLN A  28      -2.051 -11.488  12.772  1.00  0.53           N  \\nATOM    199  OE1 GLN A  28      -0.278 -12.166  11.564  1.00  0.61           O  \\nATOM    200  N   PHE A  29      -0.376  -8.975   6.274  1.00  0.81           N  \\nATOM    201  CA  PHE A  29       0.702  -8.872   5.298  1.00  0.82           C  \\nATOM    202  C   PHE A  29       1.533  -7.618   5.540  1.00  0.82           C  \\nATOM    203  CB  PHE A  29       0.138  -8.861   3.873  1.00  0.79           C  \\nATOM    204  O   PHE A  29       2.764  -7.662   5.482  1.00  0.80           O  \\nATOM    205  CG  PHE A  29       1.189  -8.706   2.806  1.00  0.74           C  \\nATOM    206  CD1 PHE A  29       2.028  -9.764   2.478  1.00  0.69           C  \\nATOM    207  CD2 PHE A  29       1.337  -7.502   2.131  1.00  0.71           C  \\nATOM    208  CE1 PHE A  29       3.001  -9.624   1.492  1.00  0.70           C  \\nATOM    209  CE2 PHE A  29       2.307  -7.354   1.144  1.00  0.69           C  \\nATOM    210  CZ  PHE A  29       3.138  -8.416   0.826  1.00  0.68           C  \\nATOM    211  N   CYS A  30       0.913  -6.533   5.878  1.00  0.85           N  \\nATOM    212  CA  CYS A  30       1.588  -5.261   6.109  1.00  0.85           C  \\nATOM    213  C   CYS A  30       1.849  -5.045   7.595  1.00  0.85           C  \\nATOM    214  CB  CYS A  30       0.757  -4.105   5.554  1.00  0.83           C  \\nATOM    215  O   CYS A  30       0.951  -5.221   8.420  1.00  0.83           O  \\nATOM    216  SG  CYS A  30       0.398  -4.247   3.790  1.00  0.80           S  \\nATOM    217  N   PHE A  31       3.179  -4.735   8.028  1.00  0.86           N  \\nATOM    218  CA  PHE A  31       3.524  -4.531   9.429  1.00  0.86           C  \\nATOM    219  C   PHE A  31       3.069  -3.157   9.905  1.00  0.85           C  \\nATOM    220  CB  PHE A  31       5.034  -4.686   9.640  1.00  0.84           C  \\nATOM    221  O   PHE A  31       2.453  -3.034  10.966  1.00  0.82           O  \\nATOM    222  CG  PHE A  31       5.515  -6.110   9.564  1.00  0.79           C  \\nATOM    223  CD1 PHE A  31       5.385  -6.961  10.655  1.00  0.73           C  \\nATOM    224  CD2 PHE A  31       6.097  -6.597   8.401  1.00  0.75           C  \\nATOM    225  CE1 PHE A  31       5.829  -8.279  10.587  1.00  0.75           C  \\nATOM    226  CE2 PHE A  31       6.543  -7.914   8.326  1.00  0.75           C  \\nATOM    227  CZ  PHE A  31       6.409  -8.753   9.421  1.00  0.74           C  \\nATOM    228  N   HIS A  32       3.346  -2.012   9.192  1.00  0.86           N  \\nATOM    229  CA  HIS A  32       3.026  -0.627   9.519  1.00  0.86           C  \\nATOM    230  C   HIS A  32       2.377   0.083   8.336  1.00  0.85           C  \\nATOM    231  CB  HIS A  32       4.285   0.125   9.956  1.00  0.84           C  \\nATOM    232  O   HIS A  32       2.858   1.128   7.893  1.00  0.82           O  \\nATOM    233  CG  HIS A  32       4.987  -0.504  11.117  1.00  0.79           C  \\nATOM    234  CD2 HIS A  32       4.811  -0.354  12.450  1.00  0.77           C  \\nATOM    235  ND1 HIS A  32       6.013  -1.412  10.962  1.00  0.74           N  \\nATOM    236  CE1 HIS A  32       6.438  -1.793  12.155  1.00  0.78           C  \\nATOM    237  NE2 HIS A  32       5.726  -1.166  13.075  1.00  0.76           N  \\nATOM    238  N   GLY A  33       1.339  -0.432   7.743  1.00  0.85           N  \\nATOM    239  CA  GLY A  33       0.648   0.112   6.585  1.00  0.85           C  \\nATOM    240  C   GLY A  33      -0.676  -0.573   6.303  1.00  0.84           C  \\nATOM    241  O   GLY A  33      -1.129  -1.407   7.090  1.00  0.82           O  \\nATOM    242  N   THR A  34      -1.396  -0.087   5.317  1.00  0.83           N  \\nATOM    243  CA  THR A  34      -2.677  -0.644   4.895  1.00  0.82           C  \\nATOM    244  C   THR A  34      -2.506  -1.514   3.653  1.00  0.83           C  \\nATOM    245  CB  THR A  34      -3.702   0.469   4.608  1.00  0.81           C  \\nATOM    246  O   THR A  34      -1.846  -1.111   2.693  1.00  0.81           O  \\nATOM    247  CG2 THR A  34      -5.060  -0.117   4.237  1.00  0.73           C  \\nATOM    248  OG1 THR A  34      -3.850   1.287   5.775  1.00  0.73           O  \\nATOM    249  N   CYS A  35      -2.976  -2.745   3.764  1.00  0.82           N  \\nATOM    250  CA  CYS A  35      -2.860  -3.707   2.674  1.00  0.82           C  \\nATOM    251  C   CYS A  35      -3.838  -3.381   1.553  1.00  0.82           C  \\nATOM    252  CB  CYS A  35      -3.108  -5.126   3.184  1.00  0.80           C  \\nATOM    253  O   CYS A  35      -4.999  -3.057   1.810  1.00  0.79           O  \\nATOM    254  SG  CYS A  35      -2.773  -6.409   1.957  1.00  0.79           S  \\nATOM    255  N   ARG A  36      -3.329  -3.494   0.381  1.00  0.80           N  \\nATOM    256  CA  ARG A  36      -4.123  -3.356  -0.835  1.00  0.80           C  \\nATOM    257  C   ARG A  36      -3.834  -4.493  -1.810  1.00  0.80           C  \\nATOM    258  CB  ARG A  36      -3.849  -2.008  -1.506  1.00  0.78           C  \\nATOM    259  O   ARG A  36      -2.719  -5.019  -1.846  1.00  0.78           O  \\nATOM    260  CG  ARG A  36      -4.211  -0.807  -0.647  1.00  0.72           C  \\nATOM    261  CD  ARG A  36      -4.398   0.450  -1.486  1.00  0.69           C  \\nATOM    262  NE  ARG A  36      -4.868   1.571  -0.678  1.00  0.62           N  \\nATOM    263  NH1 ARG A  36      -5.277   2.963  -2.476  1.00  0.50           N  \\nATOM    264  NH2 ARG A  36      -5.684   3.691  -0.341  1.00  0.46           N  \\nATOM    265  CZ  ARG A  36      -5.276   2.739  -1.167  1.00  0.62           C  \\nATOM    266  N   PHE A  37      -4.825  -4.874  -2.550  1.00  0.78           N  \\nATOM    267  CA  PHE A  37      -4.645  -5.989  -3.471  1.00  0.78           C  \\nATOM    268  C   PHE A  37      -4.633  -5.502  -4.916  1.00  0.78           C  \\nATOM    269  CB  PHE A  37      -5.750  -7.032  -3.278  1.00  0.76           C  \\nATOM    270  O   PHE A  37      -5.560  -4.818  -5.353  1.00  0.76           O  \\nATOM    271  CG  PHE A  37      -5.437  -8.370  -3.893  1.00  0.73           C  \\nATOM    272  CD1 PHE A  37      -4.603  -9.271  -3.243  1.00  0.70           C  \\nATOM    273  CD2 PHE A  37      -5.977  -8.725  -5.122  1.00  0.71           C  \\nATOM    274  CE1 PHE A  37      -4.312 -10.510  -3.810  1.00  0.69           C  \\nATOM    275  CE2 PHE A  37      -5.691  -9.961  -5.695  1.00  0.69           C  \\nATOM    276  CZ  PHE A  37      -4.857 -10.852  -5.037  1.00  0.66           C  \\nATOM    277  N   LEU A  38      -3.510  -5.639  -5.603  1.00  0.76           N  \\nATOM    278  CA  LEU A  38      -3.353  -5.252  -7.001  1.00  0.76           C  \\nATOM    279  C   LEU A  38      -3.954  -6.304  -7.927  1.00  0.76           C  \\nATOM    280  CB  LEU A  38      -1.873  -5.045  -7.337  1.00  0.75           C  \\nATOM    281  O   LEU A  38      -3.339  -7.343  -8.176  1.00  0.75           O  \\nATOM    282  CG  LEU A  38      -1.119  -4.029  -6.479  1.00  0.72           C  \\nATOM    283  CD1 LEU A  38       0.369  -4.053  -6.814  1.00  0.69           C  \\nATOM    284  CD2 LEU A  38      -1.694  -2.631  -6.676  1.00  0.69           C  \\nATOM    285  N   VAL A  39      -5.095  -6.185  -8.422  1.00  0.76           N  \\nATOM    286  CA  VAL A  39      -5.896  -7.146  -9.174  1.00  0.75           C  \\nATOM    287  C   VAL A  39      -5.197  -7.485 -10.488  1.00  0.76           C  \\nATOM    288  CB  VAL A  39      -7.318  -6.606  -9.450  1.00  0.73           C  \\nATOM    289  O   VAL A  39      -5.197  -8.641 -10.918  1.00  0.74           O  \\nATOM    290  CG1 VAL A  39      -8.116  -7.596 -10.296  1.00  0.68           C  \\nATOM    291  CG2 VAL A  39      -8.041  -6.315  -8.136  1.00  0.68           C  \\nATOM    292  N   GLN A  40      -4.590  -6.510 -11.031  1.00  0.75           N  \\nATOM    293  CA  GLN A  40      -3.929  -6.758 -12.307  1.00  0.75           C  \\nATOM    294  C   GLN A  40      -2.724  -7.679 -12.134  1.00  0.75           C  \\nATOM    295  CB  GLN A  40      -3.494  -5.441 -12.952  1.00  0.72           C  \\nATOM    296  O   GLN A  40      -2.474  -8.547 -12.972  1.00  0.74           O  \\nATOM    297  CG  GLN A  40      -4.650  -4.604 -13.482  1.00  0.67           C  \\nATOM    298  CD  GLN A  40      -4.187  -3.348 -14.197  1.00  0.65           C  \\nATOM    299  NE2 GLN A  40      -5.135  -2.589 -14.735  1.00  0.55           N  \\nATOM    300  OE1 GLN A  40      -2.987  -3.062 -14.263  1.00  0.61           O  \\nATOM    301  N   GLU A  41      -1.974  -7.588 -11.103  1.00  0.77           N  \\nATOM    302  CA  GLU A  41      -0.783  -8.384 -10.822  1.00  0.77           C  \\nATOM    303  C   GLU A  41      -1.112  -9.574  -9.927  1.00  0.77           C  \\nATOM    304  CB  GLU A  41       0.300  -7.519 -10.171  1.00  0.75           C  \\nATOM    305  O   GLU A  41      -0.281 -10.465  -9.738  1.00  0.76           O  \\nATOM    306  CG  GLU A  41       0.830  -6.414 -11.072  1.00  0.70           C  \\nATOM    307  CD  GLU A  41       1.835  -5.506 -10.381  1.00  0.68           C  \\nATOM    308  OE1 GLU A  41       2.282  -5.838  -9.260  1.00  0.66           O  \\nATOM    309  OE2 GLU A  41       2.177  -4.453 -10.965  1.00  0.61           O  \\nATOM    310  N   ASP A  42      -2.371  -9.531  -9.454  1.00  0.77           N  \\nATOM    311  CA  ASP A  42      -2.742 -10.555  -8.481  1.00  0.77           C  \\nATOM    312  C   ASP A  42      -1.748 -10.599  -7.323  1.00  0.77           C  \\nATOM    313  CB  ASP A  42      -2.828 -11.927  -9.153  1.00  0.74           C  \\nATOM    314  O   ASP A  42      -1.226 -11.664  -6.987  1.00  0.76           O  \\nATOM    315  CG  ASP A  42      -3.522 -12.967  -8.291  1.00  0.72           C  \\nATOM    316  OD1 ASP A  42      -4.369 -12.595  -7.450  1.00  0.70           O  \\nATOM    317  OD2 ASP A  42      -3.217 -14.168  -8.452  1.00  0.71           O  \\nATOM    318  N   LYS A  43      -1.324  -9.604  -6.708  1.00  0.80           N  \\nATOM    319  CA  LYS A  43      -0.380  -9.539  -5.596  1.00  0.80           C  \\nATOM    320  C   LYS A  43      -0.801  -8.483  -4.579  1.00  0.80           C  \\nATOM    321  CB  LYS A  43       1.031  -9.242  -6.106  1.00  0.78           C  \\nATOM    322  O   LYS A  43      -1.363  -7.449  -4.946  1.00  0.78           O  \\nATOM    323  CG  LYS A  43       1.655 -10.380  -6.900  1.00  0.74           C  \\nATOM    324  CD  LYS A  43       3.097 -10.072  -7.279  1.00  0.72           C  \\nATOM    325  CE  LYS A  43       3.725 -11.214  -8.068  1.00  0.64           C  \\nATOM    326  NZ  LYS A  43       5.142 -10.918  -8.438  1.00  0.59           N  \\nATOM    327  N   PRO A  44      -0.580  -8.819  -3.334  1.00  0.80           N  \\nATOM    328  CA  PRO A  44      -0.829  -7.792  -2.319  1.00  0.80           C  \\nATOM    329  C   PRO A  44       0.247  -6.708  -2.301  1.00  0.81           C  \\nATOM    330  CB  PRO A  44      -0.829  -8.583  -1.009  1.00  0.79           C  \\nATOM    331  O   PRO A  44       1.385  -6.956  -2.708  1.00  0.80           O  \\nATOM    332  CG  PRO A  44       0.051  -9.762  -1.275  1.00  0.77           C  \\nATOM    333  CD  PRO A  44      -0.074 -10.134  -2.724  1.00  0.76           C  \\nATOM    334  N   ALA A  45      -0.122  -5.502  -1.968  1.00  0.82           N  \\nATOM    335  CA  ALA A  45       0.790  -4.375  -1.792  1.00  0.82           C  \\nATOM    336  C   ALA A  45       0.439  -3.576  -0.541  1.00  0.83           C  \\nATOM    337  CB  ALA A  45       0.763  -3.471  -3.022  1.00  0.81           C  \\nATOM    338  O   ALA A  45      -0.701  -3.614  -0.071  1.00  0.82           O  \\nATOM    339  N   CYS A  46       1.460  -2.945   0.127  1.00  0.85           N  \\nATOM    340  CA  CYS A  46       1.230  -2.178   1.347  1.00  0.85           C  \\nATOM    341  C   CYS A  46       1.322  -0.681   1.076  1.00  0.85           C  \\nATOM    342  CB  CYS A  46       2.238  -2.574   2.426  1.00  0.83           C  \\nATOM    343  O   CYS A  46       2.156  -0.240   0.284  1.00  0.83           O  \\nATOM    344  SG  CYS A  46       2.008  -4.251   3.057  1.00  0.80           S  \\nATOM    345  N   VAL A  47       0.378   0.083   1.491  1.00  0.84           N  \\nATOM    346  CA  VAL A  47       0.504   1.529   1.635  1.00  0.84           C  \\nATOM    347  C   VAL A  47       1.048   1.866   3.021  1.00  0.85           C  \\nATOM    348  CB  VAL A  47      -0.848   2.242   1.406  1.00  0.82           C  \\nATOM    349  O   VAL A  47       0.360   1.679   4.027  1.00  0.84           O  \\nATOM    350  CG1 VAL A  47      -0.693   3.755   1.554  1.00  0.77           C  \\nATOM    351  CG2 VAL A  47      -1.409   1.891   0.029  1.00  0.77           C  \\nATOM    352  N   CYS A  48       2.192   2.309   3.101  1.00  0.86           N  \\nATOM    353  CA  CYS A  48       2.890   2.450   4.374  1.00  0.87           C  \\nATOM    354  C   CYS A  48       2.419   3.693   5.120  1.00  0.87           C  \\nATOM    355  CB  CYS A  48       4.401   2.519   4.152  1.00  0.85           C  \\nATOM    356  O   CYS A  48       2.108   4.712   4.502  1.00  0.85           O  \\nATOM    357  SG  CYS A  48       5.104   1.009   3.454  1.00  0.83           S  \\nATOM    358  N   HIS A  49       2.289   3.652   6.409  1.00  0.87           N  \\nATOM    359  CA  HIS A  49       2.103   4.818   7.265  1.00  0.87           C  \\nATOM    360  C   HIS A  49       3.323   5.732   7.223  1.00  0.87           C  \\nATOM    361  CB  HIS A  49       1.819   4.386   8.705  1.00  0.85           C  \\nATOM    362  O   HIS A  49       4.419   5.295   6.865  1.00  0.85           O  \\nATOM    363  CG  HIS A  49       0.549   3.612   8.860  1.00  0.78           C  \\nATOM    364  CD2 HIS A  49      -0.504   3.455   8.024  1.00  0.74           C  \\nATOM    365  ND1 HIS A  49       0.256   2.883   9.992  1.00  0.70           N  \\nATOM    366  CE1 HIS A  49      -0.927   2.310   9.845  1.00  0.73           C  \\nATOM    367  NE2 HIS A  49      -1.410   2.641   8.660  1.00  0.70           N  \\nATOM    368  N   SER A  50       3.078   6.941   7.467  1.00  0.89           N  \\nATOM    369  CA  SER A  50       4.186   7.890   7.436  1.00  0.88           C  \\nATOM    370  C   SER A  50       5.354   7.405   8.288  1.00  0.89           C  \\nATOM    371  CB  SER A  50       3.727   9.266   7.920  1.00  0.87           C  \\nATOM    372  O   SER A  50       5.151   6.841   9.365  1.00  0.87           O  \\nATOM    373  OG  SER A  50       3.361   9.221   9.289  1.00  0.78           O  \\nATOM    374  N   GLY A  51       6.595   7.433   7.763  1.00  0.87           N  \\nATOM    375  CA  GLY A  51       7.813   7.094   8.482  1.00  0.88           C  \\nATOM    376  C   GLY A  51       8.273   5.669   8.237  1.00  0.87           C  \\nATOM    377  O   GLY A  51       9.289   5.237   8.785  1.00  0.85           O  \\nATOM    378  N   TYR A  52       7.596   4.915   7.551  1.00  0.88           N  \\nATOM    379  CA  TYR A  52       7.927   3.525   7.254  1.00  0.88           C  \\nATOM    380  C   TYR A  52       8.006   3.293   5.750  1.00  0.87           C  \\nATOM    381  CB  TYR A  52       6.893   2.581   7.875  1.00  0.87           C  \\nATOM    382  O   TYR A  52       7.346   3.987   4.973  1.00  0.85           O  \\nATOM    383  CG  TYR A  52       6.905   2.576   9.385  1.00  0.84           C  \\nATOM    384  CD1 TYR A  52       7.746   1.719  10.091  1.00  0.81           C  \\nATOM    385  CD2 TYR A  52       6.076   3.427  10.108  1.00  0.82           C  \\nATOM    386  CE1 TYR A  52       7.760   1.709  11.482  1.00  0.83           C  \\nATOM    387  CE2 TYR A  52       6.081   3.425  11.499  1.00  0.82           C  \\nATOM    388  OH  TYR A  52       6.935   2.559  13.553  1.00  0.75           O  \\nATOM    389  CZ  TYR A  52       6.926   2.564  12.176  1.00  0.83           C  \\nATOM    390  N   VAL A  53       8.934   2.429   5.340  1.00  0.88           N  \\nATOM    391  CA  VAL A  53       9.180   2.056   3.951  1.00  0.88           C  \\nATOM    392  C   VAL A  53       9.358   0.543   3.846  1.00  0.88           C  \\nATOM    393  CB  VAL A  53      10.420   2.780   3.380  1.00  0.86           C  \\nATOM    394  O   VAL A  53       9.512  -0.142   4.859  1.00  0.87           O  \\nATOM    395  CG1 VAL A  53      10.191   4.289   3.334  1.00  0.75           C  \\nATOM    396  CG2 VAL A  53      11.659   2.449   4.211  1.00  0.75           C  \\nATOM    397  N   GLY A  54       9.345  -0.026   2.484  1.00  0.88           N  \\nATOM    398  CA  GLY A  54       9.516  -1.437   2.173  1.00  0.87           C  \\nATOM    399  C   GLY A  54       8.284  -2.065   1.550  1.00  0.87           C  \\nATOM    400  O   GLY A  54       7.216  -1.449   1.518  1.00  0.86           O  \\nATOM    401  N   ALA A  55       8.335  -3.279   1.124  1.00  0.87           N  \\nATOM    402  CA  ALA A  55       7.236  -4.000   0.487  1.00  0.87           C  \\nATOM    403  C   ALA A  55       6.136  -4.323   1.493  1.00  0.87           C  \\nATOM    404  CB  ALA A  55       7.749  -5.282  -0.166  1.00  0.84           C  \\nATOM    405  O   ALA A  55       4.953  -4.341   1.145  1.00  0.85           O  \\nATOM    406  N   ARG A  56       6.490  -4.544   2.739  1.00  0.88           N  \\nATOM    407  CA  ARG A  56       5.530  -4.823   3.802  1.00  0.88           C  \\nATOM    408  C   ARG A  56       5.534  -3.712   4.848  1.00  0.89           C  \\nATOM    409  CB  ARG A  56       5.837  -6.167   4.465  1.00  0.87           C  \\nATOM    410  O   ARG A  56       5.073  -3.911   5.974  1.00  0.88           O  \\nATOM    411  CG  ARG A  56       5.733  -7.356   3.523  1.00  0.81           C  \\nATOM    412  CD  ARG A  56       5.949  -8.675   4.253  1.00  0.79           C  \\nATOM    413  NE  ARG A  56       4.897  -8.926   5.234  1.00  0.72           N  \\nATOM    414  NH1 ARG A  56       6.022 -10.633   6.310  1.00  0.56           N  \\nATOM    415  NH2 ARG A  56       3.941  -9.995   7.028  1.00  0.54           N  \\nATOM    416  CZ  ARG A  56       4.956  -9.851   6.188  1.00  0.76           C  \\nATOM    417  N   CYS A  57       6.207  -2.525   4.478  1.00  0.88           N  \\nATOM    418  CA  CYS A  57       6.315  -1.439   5.445  1.00  0.89           C  \\nATOM    419  C   CYS A  57       7.023  -1.904   6.711  1.00  0.88           C  \\nATOM    420  CB  CYS A  57       4.930  -0.894   5.795  1.00  0.86           C  \\nATOM    421  O   CYS A  57       6.551  -1.651   7.821  1.00  0.85           O  \\nATOM    422  SG  CYS A  57       3.978  -0.350   4.360  1.00  0.80           S  \\nATOM    423  N   GLU A  58       8.179  -2.658   6.525  1.00  0.89           N  \\nATOM    424  CA  GLU A  58       8.862  -3.273   7.660  1.00  0.89           C  \\nATOM    425  C   GLU A  58      10.025  -2.409   8.138  1.00  0.89           C  \\nATOM    426  CB  GLU A  58       9.361  -4.672   7.293  1.00  0.87           C  \\nATOM    427  O   GLU A  58      10.556  -2.623   9.230  1.00  0.87           O  \\nATOM    428  CG  GLU A  58      10.477  -4.675   6.257  1.00  0.81           C  \\nATOM    429  CD  GLU A  58       9.967  -4.760   4.827  1.00  0.79           C  \\nATOM    430  OE1 GLU A  58       8.872  -4.226   4.540  1.00  0.77           O  \\nATOM    431  OE2 GLU A  58      10.668  -5.367   3.987  1.00  0.75           O  \\nATOM    432  N   HIS A  59      10.386  -1.405   7.414  1.00  0.90           N  \\nATOM    433  CA  HIS A  59      11.550  -0.604   7.774  1.00  0.90           C  \\nATOM    434  C   HIS A  59      11.146   0.821   8.141  1.00  0.89           C  \\nATOM    435  CB  HIS A  59      12.563  -0.584   6.628  1.00  0.88           C  \\nATOM    436  O   HIS A  59      10.156   1.342   7.624  1.00  0.88           O  \\nATOM    437  CG  HIS A  59      13.011  -1.945   6.200  1.00  0.81           C  \\nATOM    438  CD2 HIS A  59      13.681  -2.912   6.869  1.00  0.75           C  \\nATOM    439  ND1 HIS A  59      12.777  -2.442   4.936  1.00  0.71           N  \\nATOM    440  CE1 HIS A  59      13.286  -3.659   4.847  1.00  0.76           C  \\nATOM    441  NE2 HIS A  59      13.840  -3.968   6.007  1.00  0.74           N  \\nATOM    442  N   ALA A  60      11.894   1.412   9.087  1.00  0.87           N  \\nATOM    443  CA  ALA A  60      11.732   2.831   9.396  1.00  0.87           C  \\nATOM    444  C   ALA A  60      12.386   3.703   8.328  1.00  0.87           C  \\nATOM    445  CB  ALA A  60      12.320   3.146  10.769  1.00  0.84           C  \\nATOM    446  O   ALA A  60      13.467   3.379   7.829  1.00  0.84           O  \\nATOM    447  N   ASP A  61      11.562   4.580   7.614  1.00  0.85           N  \\nATOM    448  CA  ASP A  61      12.133   5.508   6.643  1.00  0.85           C  \\nATOM    449  C   ASP A  61      13.145   6.440   7.304  1.00  0.83           C  \\nATOM    450  CB  ASP A  61      11.030   6.324   5.967  1.00  0.82           C  \\nATOM    451  O   ASP A  61      12.766   7.378   8.008  1.00  0.79           O  \\nATOM    452  CG  ASP A  61      11.525   7.111   4.766  1.00  0.75           C  \\nATOM    453  OD1 ASP A  61      12.753   7.165   4.537  1.00  0.74           O  \\nATOM    454  OD2 ASP A  61      10.681   7.684   4.045  1.00  0.76           O  \\nATOM    455  N   LEU A  62      14.420   6.132   7.317  1.00  0.81           N  \\nATOM    456  CA  LEU A  62      15.479   6.887   7.979  1.00  0.80           C  \\nATOM    457  C   LEU A  62      15.554   8.309   7.433  1.00  0.79           C  \\nATOM    458  CB  LEU A  62      16.829   6.187   7.801  1.00  0.75           C  \\nATOM    459  O   LEU A  62      16.033   9.216   8.117  1.00  0.74           O  \\nATOM    460  CG  LEU A  62      16.970   4.811   8.453  1.00  0.70           C  \\nATOM    461  CD1 LEU A  62      18.297   4.172   8.059  1.00  0.65           C  \\nATOM    462  CD2 LEU A  62      16.854   4.923   9.969  1.00  0.66           C  \\nATOM    463  N   LEU A  63      15.136   8.507   6.139  1.00  0.76           N  \\nATOM    464  CA  LEU A  63      15.200   9.852   5.578  1.00  0.76           C  \\nATOM    465  C   LEU A  63      14.020  10.695   6.051  1.00  0.74           C  \\nATOM    466  CB  LEU A  63      15.220   9.795   4.049  1.00  0.71           C  \\nATOM    467  O   LEU A  63      14.007  11.913   5.865  1.00  0.70           O  \\nATOM    468  CG  LEU A  63      16.446   9.138   3.411  1.00  0.65           C  \\nATOM    469  CD1 LEU A  63      16.247   8.994   1.906  1.00  0.61           C  \\nATOM    470  CD2 LEU A  63      17.704   9.944   3.715  1.00  0.62           C  \\nATOM    471  N   ALA A  64      12.871  10.157   6.394  1.00  0.65           N  \\nATOM    472  CA  ALA A  64      11.705  10.897   6.871  1.00  0.64           C  \\nATOM    473  C   ALA A  64      12.030  11.670   8.145  1.00  0.63           C  \\nATOM    474  CB  ALA A  64      10.535   9.946   7.112  1.00  0.59           C  \\nATOM    475  O   ALA A  64      11.338  12.632   8.488  1.00  0.60           O  \\nATOM    476  N   VAL A  65      13.127  11.375   8.769  1.00  0.54           N  \\nATOM    477  CA  VAL A  65      13.475  12.163   9.947  1.00  0.57           C  \\nATOM    478  C   VAL A  65      14.396  13.313   9.546  1.00  0.54           C  \\nATOM    479  CB  VAL A  65      14.150  11.293  11.031  1.00  0.51           C  \\nATOM    480  O   VAL A  65      14.255  14.432  10.045  1.00  0.53           O  \\nATOM    481  CG1 VAL A  65      14.153  12.015  12.378  1.00  0.45           C  \\nATOM    482  CG2 VAL A  65      13.445   9.943  11.147  1.00  0.46           C  \",\"pdb\");\n\tviewer_17302037596177619.setStyle({\"resi\": 1},{\"cartoon\": {\"color\": \"#0c00f3\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 2},{\"cartoon\": {\"color\": \"#0c00f3\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 3},{\"cartoon\": {\"color\": \"#0c00f3\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 4},{\"cartoon\": {\"color\": \"#1f00e0\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 5},{\"cartoon\": {\"color\": \"#1f00e0\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 6},{\"cartoon\": {\"color\": \"#1f00e0\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 7},{\"cartoon\": {\"color\": \"#2100de\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 8},{\"cartoon\": {\"color\": \"#4700b8\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 9},{\"cartoon\": {\"color\": \"#7b0084\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 10},{\"cartoon\": {\"color\": \"#80007f\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 11},{\"cartoon\": {\"color\": \"#8b0074\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 12},{\"cartoon\": {\"color\": \"#5a00a5\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 13},{\"cartoon\": {\"color\": \"#2700d8\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 14},{\"cartoon\": {\"color\": \"#3200cd\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 15},{\"cartoon\": {\"color\": \"#2c00d3\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 16},{\"cartoon\": {\"color\": \"#5800a7\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 17},{\"cartoon\": {\"color\": \"#5600a9\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 18},{\"cartoon\": {\"color\": \"#4700b8\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 19},{\"cartoon\": {\"color\": \"#70008f\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 20},{\"cartoon\": {\"color\": \"#75008a\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 21},{\"cartoon\": {\"color\": \"#760089\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 22},{\"cartoon\": {\"color\": \"#af0050\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 23},{\"cartoon\": {\"color\": \"#81007e\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 24},{\"cartoon\": {\"color\": \"#4800b7\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 25},{\"cartoon\": {\"color\": \"#7f0080\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 26},{\"cartoon\": {\"color\": \"#4600b9\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 27},{\"cartoon\": {\"color\": \"#63009c\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 28},{\"cartoon\": {\"color\": \"#9b0064\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 29},{\"cartoon\": {\"color\": \"#64009b\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 30},{\"cartoon\": {\"color\": \"#64009b\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 31},{\"cartoon\": {\"color\": \"#7e0081\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 32},{\"cartoon\": {\"color\": \"#4500ba\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 33},{\"cartoon\": {\"color\": \"#7d0082\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 34},{\"cartoon\": {\"color\": \"#94006b\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 35},{\"cartoon\": {\"color\": \"#5c00a3\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 36},{\"cartoon\": {\"color\": \"#5c00a3\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 37},{\"cartoon\": {\"color\": \"#2300dc\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 38},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 39},{\"cartoon\": {\"color\": \"#4600b9\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 40},{\"cartoon\": {\"color\": \"#7e0081\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 41},{\"cartoon\": {\"color\": \"#80007f\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 42},{\"cartoon\": {\"color\": \"#80007f\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 43},{\"cartoon\": {\"color\": \"#4b00b4\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 44},{\"cartoon\": {\"color\": \"#4b00b4\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 45},{\"cartoon\": {\"color\": \"#4a00b5\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 46},{\"cartoon\": {\"color\": \"#83007c\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 47},{\"cartoon\": {\"color\": \"#7f0080\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 48},{\"cartoon\": {\"color\": \"#4d00b2\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 49},{\"cartoon\": {\"color\": \"#860079\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 50},{\"cartoon\": {\"color\": \"#860079\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 51},{\"cartoon\": {\"color\": \"#870078\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 52},{\"cartoon\": {\"color\": \"#80007f\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 53},{\"cartoon\": {\"color\": \"#80007f\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 54},{\"cartoon\": {\"color\": \"#7f0080\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 55},{\"cartoon\": {\"color\": \"#860079\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 56},{\"cartoon\": {\"color\": \"#880077\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 57},{\"cartoon\": {\"color\": \"#870078\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 58},{\"cartoon\": {\"color\": \"#5400ab\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 59},{\"cartoon\": {\"color\": \"#4d00b2\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 60},{\"cartoon\": {\"color\": \"#4c00b3\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 61},{\"cartoon\": {\"color\": \"#1300ec\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 62},{\"cartoon\": {\"color\": \"#0e00f1\"}});\n\tviewer_17302037596177619.setStyle({\"resi\": 63},{\"cartoon\": {\"color\": \"#0e00f1\"}});\n\tviewer_17302037596177619.zoomTo();\nviewer_17302037596177619.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_17302037596177619\"  style=\"position: relative; width: 1000px; height: 800px;\">\n",
              "        <p id=\"3dmolwarning_17302037596177619\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.0/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_17302037596177619 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_17302037596177619\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_17302037596177619 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17302037596177619\"),{backgroundColor:\"white\"});\n",
              "viewer_17302037596177619.zoomTo();\n",
              "\tviewer_17302037596177619.addModel(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   SER A   1      -2.038  12.334 -43.299  1.00  0.47           N  \\nATOM      2  CA  SER A   1      -2.858  11.164 -42.998  1.00  0.53           C  \\nATOM      3  C   SER A   1      -3.169  11.075 -41.508  1.00  0.50           C  \\nATOM      4  CB  SER A   1      -2.157   9.885 -43.459  1.00  0.44           C  \\nATOM      5  O   SER A   1      -2.258  11.077 -40.677  1.00  0.48           O  \\nATOM      6  OG  SER A   1      -3.034   8.775 -43.376  1.00  0.47           O  \\nATOM      7  N   THR A   2      -3.907  12.144 -41.043  1.00  0.54           N  \\nATOM      8  CA  THR A   2      -4.608  12.316 -39.776  1.00  0.56           C  \\nATOM      9  C   THR A   2      -5.167  10.985 -39.282  1.00  0.55           C  \\nATOM     10  CB  THR A   2      -5.752  13.339 -39.904  1.00  0.50           C  \\nATOM     11  O   THR A   2      -5.981  10.357 -39.961  1.00  0.53           O  \\nATOM     12  CG2 THR A   2      -5.210  14.764 -39.973  1.00  0.44           C  \\nATOM     13  OG1 THR A   2      -6.498  13.066 -41.096  1.00  0.50           O  \\nATOM     14  N   SER A   3      -4.215   9.946 -38.896  1.00  0.56           N  \\nATOM     15  CA  SER A   3      -4.893   8.792 -38.315  1.00  0.56           C  \\nATOM     16  C   SER A   3      -5.906   9.221 -37.258  1.00  0.56           C  \\nATOM     17  CB  SER A   3      -3.879   7.827 -37.699  1.00  0.50           C  \\nATOM     18  O   SER A   3      -5.628  10.108 -36.449  1.00  0.54           O  \\nATOM     19  OG  SER A   3      -2.975   7.353 -38.682  1.00  0.50           O  \\nATOM     20  N   PRO A   4      -7.327   9.272 -37.567  1.00  0.54           N  \\nATOM     21  CA  PRO A   4      -8.238   9.442 -36.433  1.00  0.54           C  \\nATOM     22  C   PRO A   4      -7.634   8.967 -35.114  1.00  0.54           C  \\nATOM     23  CB  PRO A   4      -9.446   8.584 -36.821  1.00  0.50           C  \\nATOM     24  O   PRO A   4      -6.969   7.928 -35.074  1.00  0.52           O  \\nATOM     25  CG  PRO A   4      -8.987   7.789 -38.000  1.00  0.48           C  \\nATOM     26  CD  PRO A   4      -7.673   8.344 -38.468  1.00  0.49           C  \\nATOM     27  N   LEU A   5      -6.824   9.851 -34.347  1.00  0.51           N  \\nATOM     28  CA  LEU A   5      -6.752   9.455 -32.945  1.00  0.51           C  \\nATOM     29  C   LEU A   5      -7.472   8.131 -32.716  1.00  0.51           C  \\nATOM     30  CB  LEU A   5      -7.356  10.540 -32.050  1.00  0.47           C  \\nATOM     31  O   LEU A   5      -8.675   8.022 -32.968  1.00  0.49           O  \\nATOM     32  CG  LEU A   5      -6.447  11.720 -31.706  1.00  0.46           C  \\nATOM     33  CD1 LEU A   5      -7.259  13.009 -31.624  1.00  0.44           C  \\nATOM     34  CD2 LEU A   5      -5.708  11.463 -30.397  1.00  0.44           C  \\nATOM     35  N   SER A   6      -7.329   7.026 -33.488  1.00  0.49           N  \\nATOM     36  CA  SER A   6      -7.822   5.743 -32.998  1.00  0.52           C  \\nATOM     37  C   SER A   6      -8.137   5.804 -31.507  1.00  0.50           C  \\nATOM     38  CB  SER A   6      -6.800   4.638 -33.267  1.00  0.46           C  \\nATOM     39  O   SER A   6      -7.293   6.208 -30.705  1.00  0.48           O  \\nATOM     40  OG  SER A   6      -7.088   3.487 -32.491  1.00  0.45           O  \\nATOM     41  N   ALA A   7      -9.098   6.608 -31.113  1.00  0.47           N  \\nATOM     42  CA  ALA A   7      -9.959   6.490 -29.939  1.00  0.49           C  \\nATOM     43  C   ALA A   7      -9.888   5.085 -29.347  1.00  0.48           C  \\nATOM     44  CB  ALA A   7     -11.401   6.841 -30.298  1.00  0.44           C  \\nATOM     45  O   ALA A   7     -10.527   4.159 -29.852  1.00  0.46           O  \\nATOM     46  N   ASP A   8      -8.815   4.312 -29.207  1.00  0.47           N  \\nATOM     47  CA  ASP A   8      -8.867   3.410 -28.061  1.00  0.48           C  \\nATOM     48  C   ASP A   8      -9.067   4.185 -26.761  1.00  0.47           C  \\nATOM     49  CB  ASP A   8      -7.591   2.570 -27.982  1.00  0.44           C  \\nATOM     50  O   ASP A   8      -8.453   5.234 -26.557  1.00  0.45           O  \\nATOM     51  CG  ASP A   8      -7.496   1.529 -29.083  1.00  0.43           C  \\nATOM     52  OD1 ASP A   8      -8.545   1.012 -29.526  1.00  0.43           O  \\nATOM     53  OD2 ASP A   8      -6.363   1.220 -29.511  1.00  0.48           O  \\nATOM     54  N   PRO A   9     -10.359   4.479 -26.370  1.00  0.48           N  \\nATOM     55  CA  PRO A   9     -10.291   5.081 -25.037  1.00  0.48           C  \\nATOM     56  C   PRO A   9      -8.929   4.895 -24.373  1.00  0.49           C  \\nATOM     57  CB  PRO A   9     -11.383   4.341 -24.260  1.00  0.46           C  \\nATOM     58  O   PRO A   9      -8.201   3.954 -24.703  1.00  0.47           O  \\nATOM     59  CG  PRO A   9     -11.689   3.135 -25.087  1.00  0.44           C  \\nATOM     60  CD  PRO A   9     -11.136   3.349 -26.467  1.00  0.45           C  \\nATOM     61  N   PRO A  10      -8.115   6.032 -24.138  1.00  0.47           N  \\nATOM     62  CA  PRO A  10      -7.130   5.621 -23.135  1.00  0.47           C  \\nATOM     63  C   PRO A  10      -7.594   4.425 -22.308  1.00  0.47           C  \\nATOM     64  CB  PRO A  10      -6.980   6.867 -22.257  1.00  0.45           C  \\nATOM     65  O   PRO A  10      -8.777   4.320 -21.976  1.00  0.45           O  \\nATOM     66  CG  PRO A  10      -8.116   7.753 -22.655  1.00  0.42           C  \\nATOM     67  CD  PRO A  10      -8.720   7.216 -23.921  1.00  0.43           C  \\nATOM     68  N   VAL A  11      -7.696   3.073 -22.725  1.00  0.44           N  \\nATOM     69  CA  VAL A  11      -7.764   2.172 -21.579  1.00  0.46           C  \\nATOM     70  C   VAL A  11      -8.191   2.948 -20.335  1.00  0.44           C  \\nATOM     71  CB  VAL A  11      -6.412   1.467 -21.329  1.00  0.43           C  \\nATOM     72  O   VAL A  11      -7.566   3.949 -19.976  1.00  0.43           O  \\nATOM     73  CG1 VAL A  11      -6.545   0.418 -20.227  1.00  0.41           C  \\nATOM     74  CG2 VAL A  11      -5.897   0.830 -22.619  1.00  0.41           C  \\nATOM     75  N   ALA A  12      -9.421   3.665 -20.370  1.00  0.43           N  \\nATOM     76  CA  ALA A  12      -9.817   3.924 -18.988  1.00  0.44           C  \\nATOM     77  C   ALA A  12      -8.853   3.260 -18.008  1.00  0.43           C  \\nATOM     78  CB  ALA A  12     -11.242   3.435 -18.742  1.00  0.41           C  \\nATOM     79  O   ALA A  12      -8.557   2.069 -18.130  1.00  0.41           O  \\nATOM     80  N   ALA A  13      -7.543   3.638 -18.005  1.00  0.44           N  \\nATOM     81  CA  ALA A  13      -6.895   3.360 -16.726  1.00  0.45           C  \\nATOM     82  C   ALA A  13      -7.847   2.643 -15.773  1.00  0.44           C  \\nATOM     83  CB  ALA A  13      -6.387   4.654 -16.094  1.00  0.42           C  \\nATOM     84  O   ALA A  13      -8.978   3.089 -15.564  1.00  0.42           O  \\nATOM     85  N   ALA A  14      -8.522   1.429 -16.199  1.00  0.46           N  \\nATOM     86  CA  ALA A  14      -9.099   0.649 -15.108  1.00  0.46           C  \\nATOM     87  C   ALA A  14      -9.154   1.464 -13.819  1.00  0.46           C  \\nATOM     88  CB  ALA A  14      -8.298  -0.633 -14.889  1.00  0.43           C  \\nATOM     89  O   ALA A  14      -8.140   2.012 -13.380  1.00  0.44           O  \\nATOM     90  N   VAL A  15      -9.878   2.615 -13.856  1.00  0.46           N  \\nATOM     91  CA  VAL A  15     -10.245   3.079 -12.522  1.00  0.46           C  \\nATOM     92  C   VAL A  15      -9.975   1.978 -11.499  1.00  0.46           C  \\nATOM     93  CB  VAL A  15     -11.727   3.512 -12.460  1.00  0.44           C  \\nATOM     94  O   VAL A  15     -10.504   0.871 -11.614  1.00  0.44           O  \\nATOM     95  CG1 VAL A  15     -12.095   3.979 -11.052  1.00  0.40           C  \\nATOM     96  CG2 VAL A  15     -12.004   4.614 -13.481  1.00  0.42           C  \\nATOM     97  N   VAL A  16      -8.781   1.491 -11.315  1.00  0.50           N  \\nATOM     98  CA  VAL A  16      -8.406   0.826 -10.072  1.00  0.49           C  \\nATOM     99  C   VAL A  16      -9.306   1.306  -8.936  1.00  0.50           C  \\nATOM    100  CB  VAL A  16      -6.923   1.077  -9.719  1.00  0.48           C  \\nATOM    101  O   VAL A  16      -9.343   2.499  -8.626  1.00  0.49           O  \\nATOM    102  CG1 VAL A  16      -6.512   0.256  -8.498  1.00  0.45           C  \\nATOM    103  CG2 VAL A  16      -6.028   0.750 -10.913  1.00  0.46           C  \\nATOM    104  N   SER A  17     -10.683   1.068  -8.984  1.00  0.52           N  \\nATOM    105  CA  SER A  17     -11.628   1.272  -7.892  1.00  0.51           C  \\nATOM    106  C   SER A  17     -10.988   0.968  -6.542  1.00  0.52           C  \\nATOM    107  CB  SER A  17     -12.869   0.399  -8.085  1.00  0.49           C  \\nATOM    108  O   SER A  17     -10.264  -0.021  -6.401  1.00  0.50           O  \\nATOM    109  OG  SER A  17     -13.488   0.123  -6.841  1.00  0.47           O  \\nATOM    110  N   HIS A  18     -10.316   1.933  -5.765  1.00  0.52           N  \\nATOM    111  CA  HIS A  18     -10.082   2.300  -4.373  1.00  0.52           C  \\nATOM    112  C   HIS A  18     -11.002   1.523  -3.436  1.00  0.51           C  \\nATOM    113  CB  HIS A  18     -10.279   3.804  -4.175  1.00  0.49           C  \\nATOM    114  O   HIS A  18     -12.224   1.555  -3.593  1.00  0.50           O  \\nATOM    115  CG  HIS A  18      -9.153   4.632  -4.707  1.00  0.48           C  \\nATOM    116  CD2 HIS A  18      -9.056   5.388  -5.826  1.00  0.47           C  \\nATOM    117  ND1 HIS A  18      -7.941   4.741  -4.060  1.00  0.45           N  \\nATOM    118  CE1 HIS A  18      -7.145   5.532  -4.761  1.00  0.44           C  \\nATOM    119  NE2 HIS A  18      -7.798   5.938  -5.837  1.00  0.41           N  \\nATOM    120  N   PHE A  19     -11.021  -0.064  -3.367  1.00  0.55           N  \\nATOM    121  CA  PHE A  19     -11.475  -0.896  -2.260  1.00  0.54           C  \\nATOM    122  C   PHE A  19     -11.050  -2.345  -2.462  1.00  0.55           C  \\nATOM    123  CB  PHE A  19     -12.997  -0.811  -2.111  1.00  0.50           C  \\nATOM    124  O   PHE A  19     -11.842  -3.173  -2.915  1.00  0.52           O  \\nATOM    125  CG  PHE A  19     -13.483   0.508  -1.575  1.00  0.48           C  \\nATOM    126  CD1 PHE A  19     -13.453   0.774  -0.212  1.00  0.45           C  \\nATOM    127  CD2 PHE A  19     -13.969   1.484  -2.436  1.00  0.46           C  \\nATOM    128  CE1 PHE A  19     -13.901   1.995   0.287  1.00  0.46           C  \\nATOM    129  CE2 PHE A  19     -14.419   2.706  -1.945  1.00  0.46           C  \\nATOM    130  CZ  PHE A  19     -14.385   2.959  -0.583  1.00  0.46           C  \\nATOM    131  N   ASN A  20      -9.963  -2.745  -3.148  1.00  0.63           N  \\nATOM    132  CA  ASN A  20      -9.502  -4.129  -3.097  1.00  0.62           C  \\nATOM    133  C   ASN A  20      -8.753  -4.423  -1.800  1.00  0.63           C  \\nATOM    134  CB  ASN A  20      -8.616  -4.444  -4.305  1.00  0.57           C  \\nATOM    135  O   ASN A  20      -7.555  -4.157  -1.696  1.00  0.60           O  \\nATOM    136  CG  ASN A  20      -9.377  -4.396  -5.615  1.00  0.54           C  \\nATOM    137  ND2 ASN A  20      -8.661  -4.154  -6.707  1.00  0.50           N  \\nATOM    138  OD1 ASN A  20     -10.597  -4.573  -5.646  1.00  0.53           O  \\nATOM    139  N   ASP A  21      -9.440  -4.148  -0.739  1.00  0.66           N  \\nATOM    140  CA  ASP A  21      -9.031  -4.698   0.550  1.00  0.65           C  \\nATOM    141  C   ASP A  21      -8.386  -6.071   0.382  1.00  0.66           C  \\nATOM    142  CB  ASP A  21     -10.229  -4.792   1.498  1.00  0.62           C  \\nATOM    143  O   ASP A  21      -8.746  -6.826  -0.524  1.00  0.64           O  \\nATOM    144  CG  ASP A  21     -10.659  -3.443   2.046  1.00  0.59           C  \\nATOM    145  OD1 ASP A  21      -9.860  -2.483   2.001  1.00  0.58           O  \\nATOM    146  OD2 ASP A  21     -11.807  -3.340   2.530  1.00  0.60           O  \\nATOM    147  N   CYS A  22      -7.281  -6.172   0.742  1.00  0.71           N  \\nATOM    148  CA  CYS A  22      -6.613  -7.468   0.769  1.00  0.71           C  \\nATOM    149  C   CYS A  22      -7.535  -8.545   1.327  1.00  0.71           C  \\nATOM    150  CB  CYS A  22      -5.334  -7.395   1.603  1.00  0.68           C  \\nATOM    151  O   CYS A  22      -8.367  -8.268   2.192  1.00  0.68           O  \\nATOM    152  SG  CYS A  22      -3.928  -6.685   0.720  1.00  0.67           S  \\nATOM    153  N   PRO A  23      -7.667  -9.698   0.678  1.00  0.69           N  \\nATOM    154  CA  PRO A  23      -8.330 -10.814   1.358  1.00  0.68           C  \\nATOM    155  C   PRO A  23      -7.867 -10.984   2.803  1.00  0.69           C  \\nATOM    156  CB  PRO A  23      -7.935 -12.027   0.511  1.00  0.66           C  \\nATOM    157  O   PRO A  23      -6.781 -10.524   3.165  1.00  0.68           O  \\nATOM    158  CG  PRO A  23      -7.162 -11.458  -0.634  1.00  0.62           C  \\nATOM    159  CD  PRO A  23      -6.847 -10.021  -0.331  1.00  0.61           C  \\nATOM    160  N   ASP A  24      -8.773 -11.366   3.788  1.00  0.66           N  \\nATOM    161  CA  ASP A  24      -8.495 -11.597   5.203  1.00  0.65           C  \\nATOM    162  C   ASP A  24      -7.178 -12.348   5.388  1.00  0.67           C  \\nATOM    163  CB  ASP A  24      -9.640 -12.374   5.855  1.00  0.61           C  \\nATOM    164  O   ASP A  24      -6.500 -12.182   6.404  1.00  0.66           O  \\nATOM    165  CG  ASP A  24     -10.903 -11.546   6.015  1.00  0.57           C  \\nATOM    166  OD1 ASP A  24     -10.835 -10.303   5.907  1.00  0.58           O  \\nATOM    167  OD2 ASP A  24     -11.976 -12.142   6.253  1.00  0.60           O  \\nATOM    168  N   SER A  25      -6.972 -13.265   4.502  1.00  0.68           N  \\nATOM    169  CA  SER A  25      -5.745 -14.042   4.643  1.00  0.67           C  \\nATOM    170  C   SER A  25      -4.515 -13.141   4.638  1.00  0.67           C  \\nATOM    171  CB  SER A  25      -5.634 -15.076   3.521  1.00  0.62           C  \\nATOM    172  O   SER A  25      -3.449 -13.534   5.116  1.00  0.63           O  \\nATOM    173  OG  SER A  25      -5.648 -14.445   2.252  1.00  0.58           O  \\nATOM    174  N   HIS A  26      -4.727 -11.922   4.235  1.00  0.69           N  \\nATOM    175  CA  HIS A  26      -3.524 -11.120   4.047  1.00  0.69           C  \\nATOM    176  C   HIS A  26      -3.530  -9.896   4.958  1.00  0.70           C  \\nATOM    177  CB  HIS A  26      -3.391 -10.686   2.586  1.00  0.64           C  \\nATOM    178  O   HIS A  26      -2.749  -8.963   4.755  1.00  0.66           O  \\nATOM    179  CG  HIS A  26      -3.032 -11.804   1.659  1.00  0.60           C  \\nATOM    180  CD2 HIS A  26      -1.871 -12.098   1.029  1.00  0.58           C  \\nATOM    181  ND1 HIS A  26      -3.933 -12.778   1.285  1.00  0.54           N  \\nATOM    182  CE1 HIS A  26      -3.338 -13.626   0.463  1.00  0.53           C  \\nATOM    183  NE2 HIS A  26      -2.086 -13.236   0.291  1.00  0.51           N  \\nATOM    184  N   THR A  27      -4.513  -9.842   5.949  1.00  0.68           N  \\nATOM    185  CA  THR A  27      -4.648  -8.710   6.859  1.00  0.67           C  \\nATOM    186  C   THR A  27      -3.349  -8.473   7.623  1.00  0.69           C  \\nATOM    187  CB  THR A  27      -5.801  -8.930   7.857  1.00  0.64           C  \\nATOM    188  O   THR A  27      -3.046  -7.341   8.006  1.00  0.67           O  \\nATOM    189  CG2 THR A  27      -7.148  -8.958   7.143  1.00  0.56           C  \\nATOM    190  OG1 THR A  27      -5.606 -10.176   8.536  1.00  0.58           O  \\nATOM    191  N   GLN A  28      -2.458  -9.617   7.710  1.00  0.75           N  \\nATOM    192  CA  GLN A  28      -1.205  -9.433   8.435  1.00  0.75           C  \\nATOM    193  C   GLN A  28      -0.043  -9.200   7.474  1.00  0.77           C  \\nATOM    194  CB  GLN A  28      -0.916 -10.643   9.325  1.00  0.71           C  \\nATOM    195  O   GLN A  28       1.122  -9.271   7.871  1.00  0.73           O  \\nATOM    196  CG  GLN A  28      -1.933 -10.843  10.440  1.00  0.65           C  \\nATOM    197  CD  GLN A  28      -1.350 -11.557  11.645  1.00  0.62           C  \\nATOM    198  NE2 GLN A  28      -2.051 -11.488  12.772  1.00  0.53           N  \\nATOM    199  OE1 GLN A  28      -0.278 -12.166  11.564  1.00  0.61           O  \\nATOM    200  N   PHE A  29      -0.376  -8.975   6.274  1.00  0.81           N  \\nATOM    201  CA  PHE A  29       0.702  -8.872   5.298  1.00  0.82           C  \\nATOM    202  C   PHE A  29       1.533  -7.618   5.540  1.00  0.82           C  \\nATOM    203  CB  PHE A  29       0.138  -8.861   3.873  1.00  0.79           C  \\nATOM    204  O   PHE A  29       2.764  -7.662   5.482  1.00  0.80           O  \\nATOM    205  CG  PHE A  29       1.189  -8.706   2.806  1.00  0.74           C  \\nATOM    206  CD1 PHE A  29       2.028  -9.764   2.478  1.00  0.69           C  \\nATOM    207  CD2 PHE A  29       1.337  -7.502   2.131  1.00  0.71           C  \\nATOM    208  CE1 PHE A  29       3.001  -9.624   1.492  1.00  0.70           C  \\nATOM    209  CE2 PHE A  29       2.307  -7.354   1.144  1.00  0.69           C  \\nATOM    210  CZ  PHE A  29       3.138  -8.416   0.826  1.00  0.68           C  \\nATOM    211  N   CYS A  30       0.913  -6.533   5.878  1.00  0.85           N  \\nATOM    212  CA  CYS A  30       1.588  -5.261   6.109  1.00  0.85           C  \\nATOM    213  C   CYS A  30       1.849  -5.045   7.595  1.00  0.85           C  \\nATOM    214  CB  CYS A  30       0.757  -4.105   5.554  1.00  0.83           C  \\nATOM    215  O   CYS A  30       0.951  -5.221   8.420  1.00  0.83           O  \\nATOM    216  SG  CYS A  30       0.398  -4.247   3.790  1.00  0.80           S  \\nATOM    217  N   PHE A  31       3.179  -4.735   8.028  1.00  0.86           N  \\nATOM    218  CA  PHE A  31       3.524  -4.531   9.429  1.00  0.86           C  \\nATOM    219  C   PHE A  31       3.069  -3.157   9.905  1.00  0.85           C  \\nATOM    220  CB  PHE A  31       5.034  -4.686   9.640  1.00  0.84           C  \\nATOM    221  O   PHE A  31       2.453  -3.034  10.966  1.00  0.82           O  \\nATOM    222  CG  PHE A  31       5.515  -6.110   9.564  1.00  0.79           C  \\nATOM    223  CD1 PHE A  31       5.385  -6.961  10.655  1.00  0.73           C  \\nATOM    224  CD2 PHE A  31       6.097  -6.597   8.401  1.00  0.75           C  \\nATOM    225  CE1 PHE A  31       5.829  -8.279  10.587  1.00  0.75           C  \\nATOM    226  CE2 PHE A  31       6.543  -7.914   8.326  1.00  0.75           C  \\nATOM    227  CZ  PHE A  31       6.409  -8.753   9.421  1.00  0.74           C  \\nATOM    228  N   HIS A  32       3.346  -2.012   9.192  1.00  0.86           N  \\nATOM    229  CA  HIS A  32       3.026  -0.627   9.519  1.00  0.86           C  \\nATOM    230  C   HIS A  32       2.377   0.083   8.336  1.00  0.85           C  \\nATOM    231  CB  HIS A  32       4.285   0.125   9.956  1.00  0.84           C  \\nATOM    232  O   HIS A  32       2.858   1.128   7.893  1.00  0.82           O  \\nATOM    233  CG  HIS A  32       4.987  -0.504  11.117  1.00  0.79           C  \\nATOM    234  CD2 HIS A  32       4.811  -0.354  12.450  1.00  0.77           C  \\nATOM    235  ND1 HIS A  32       6.013  -1.412  10.962  1.00  0.74           N  \\nATOM    236  CE1 HIS A  32       6.438  -1.793  12.155  1.00  0.78           C  \\nATOM    237  NE2 HIS A  32       5.726  -1.166  13.075  1.00  0.76           N  \\nATOM    238  N   GLY A  33       1.339  -0.432   7.743  1.00  0.85           N  \\nATOM    239  CA  GLY A  33       0.648   0.112   6.585  1.00  0.85           C  \\nATOM    240  C   GLY A  33      -0.676  -0.573   6.303  1.00  0.84           C  \\nATOM    241  O   GLY A  33      -1.129  -1.407   7.090  1.00  0.82           O  \\nATOM    242  N   THR A  34      -1.396  -0.087   5.317  1.00  0.83           N  \\nATOM    243  CA  THR A  34      -2.677  -0.644   4.895  1.00  0.82           C  \\nATOM    244  C   THR A  34      -2.506  -1.514   3.653  1.00  0.83           C  \\nATOM    245  CB  THR A  34      -3.702   0.469   4.608  1.00  0.81           C  \\nATOM    246  O   THR A  34      -1.846  -1.111   2.693  1.00  0.81           O  \\nATOM    247  CG2 THR A  34      -5.060  -0.117   4.237  1.00  0.73           C  \\nATOM    248  OG1 THR A  34      -3.850   1.287   5.775  1.00  0.73           O  \\nATOM    249  N   CYS A  35      -2.976  -2.745   3.764  1.00  0.82           N  \\nATOM    250  CA  CYS A  35      -2.860  -3.707   2.674  1.00  0.82           C  \\nATOM    251  C   CYS A  35      -3.838  -3.381   1.553  1.00  0.82           C  \\nATOM    252  CB  CYS A  35      -3.108  -5.126   3.184  1.00  0.80           C  \\nATOM    253  O   CYS A  35      -4.999  -3.057   1.810  1.00  0.79           O  \\nATOM    254  SG  CYS A  35      -2.773  -6.409   1.957  1.00  0.79           S  \\nATOM    255  N   ARG A  36      -3.329  -3.494   0.381  1.00  0.80           N  \\nATOM    256  CA  ARG A  36      -4.123  -3.356  -0.835  1.00  0.80           C  \\nATOM    257  C   ARG A  36      -3.834  -4.493  -1.810  1.00  0.80           C  \\nATOM    258  CB  ARG A  36      -3.849  -2.008  -1.506  1.00  0.78           C  \\nATOM    259  O   ARG A  36      -2.719  -5.019  -1.846  1.00  0.78           O  \\nATOM    260  CG  ARG A  36      -4.211  -0.807  -0.647  1.00  0.72           C  \\nATOM    261  CD  ARG A  36      -4.398   0.450  -1.486  1.00  0.69           C  \\nATOM    262  NE  ARG A  36      -4.868   1.571  -0.678  1.00  0.62           N  \\nATOM    263  NH1 ARG A  36      -5.277   2.963  -2.476  1.00  0.50           N  \\nATOM    264  NH2 ARG A  36      -5.684   3.691  -0.341  1.00  0.46           N  \\nATOM    265  CZ  ARG A  36      -5.276   2.739  -1.167  1.00  0.62           C  \\nATOM    266  N   PHE A  37      -4.825  -4.874  -2.550  1.00  0.78           N  \\nATOM    267  CA  PHE A  37      -4.645  -5.989  -3.471  1.00  0.78           C  \\nATOM    268  C   PHE A  37      -4.633  -5.502  -4.916  1.00  0.78           C  \\nATOM    269  CB  PHE A  37      -5.750  -7.032  -3.278  1.00  0.76           C  \\nATOM    270  O   PHE A  37      -5.560  -4.818  -5.353  1.00  0.76           O  \\nATOM    271  CG  PHE A  37      -5.437  -8.370  -3.893  1.00  0.73           C  \\nATOM    272  CD1 PHE A  37      -4.603  -9.271  -3.243  1.00  0.70           C  \\nATOM    273  CD2 PHE A  37      -5.977  -8.725  -5.122  1.00  0.71           C  \\nATOM    274  CE1 PHE A  37      -4.312 -10.510  -3.810  1.00  0.69           C  \\nATOM    275  CE2 PHE A  37      -5.691  -9.961  -5.695  1.00  0.69           C  \\nATOM    276  CZ  PHE A  37      -4.857 -10.852  -5.037  1.00  0.66           C  \\nATOM    277  N   LEU A  38      -3.510  -5.639  -5.603  1.00  0.76           N  \\nATOM    278  CA  LEU A  38      -3.353  -5.252  -7.001  1.00  0.76           C  \\nATOM    279  C   LEU A  38      -3.954  -6.304  -7.927  1.00  0.76           C  \\nATOM    280  CB  LEU A  38      -1.873  -5.045  -7.337  1.00  0.75           C  \\nATOM    281  O   LEU A  38      -3.339  -7.343  -8.176  1.00  0.75           O  \\nATOM    282  CG  LEU A  38      -1.119  -4.029  -6.479  1.00  0.72           C  \\nATOM    283  CD1 LEU A  38       0.369  -4.053  -6.814  1.00  0.69           C  \\nATOM    284  CD2 LEU A  38      -1.694  -2.631  -6.676  1.00  0.69           C  \\nATOM    285  N   VAL A  39      -5.095  -6.185  -8.422  1.00  0.76           N  \\nATOM    286  CA  VAL A  39      -5.896  -7.146  -9.174  1.00  0.75           C  \\nATOM    287  C   VAL A  39      -5.197  -7.485 -10.488  1.00  0.76           C  \\nATOM    288  CB  VAL A  39      -7.318  -6.606  -9.450  1.00  0.73           C  \\nATOM    289  O   VAL A  39      -5.197  -8.641 -10.918  1.00  0.74           O  \\nATOM    290  CG1 VAL A  39      -8.116  -7.596 -10.296  1.00  0.68           C  \\nATOM    291  CG2 VAL A  39      -8.041  -6.315  -8.136  1.00  0.68           C  \\nATOM    292  N   GLN A  40      -4.590  -6.510 -11.031  1.00  0.75           N  \\nATOM    293  CA  GLN A  40      -3.929  -6.758 -12.307  1.00  0.75           C  \\nATOM    294  C   GLN A  40      -2.724  -7.679 -12.134  1.00  0.75           C  \\nATOM    295  CB  GLN A  40      -3.494  -5.441 -12.952  1.00  0.72           C  \\nATOM    296  O   GLN A  40      -2.474  -8.547 -12.972  1.00  0.74           O  \\nATOM    297  CG  GLN A  40      -4.650  -4.604 -13.482  1.00  0.67           C  \\nATOM    298  CD  GLN A  40      -4.187  -3.348 -14.197  1.00  0.65           C  \\nATOM    299  NE2 GLN A  40      -5.135  -2.589 -14.735  1.00  0.55           N  \\nATOM    300  OE1 GLN A  40      -2.987  -3.062 -14.263  1.00  0.61           O  \\nATOM    301  N   GLU A  41      -1.974  -7.588 -11.103  1.00  0.77           N  \\nATOM    302  CA  GLU A  41      -0.783  -8.384 -10.822  1.00  0.77           C  \\nATOM    303  C   GLU A  41      -1.112  -9.574  -9.927  1.00  0.77           C  \\nATOM    304  CB  GLU A  41       0.300  -7.519 -10.171  1.00  0.75           C  \\nATOM    305  O   GLU A  41      -0.281 -10.465  -9.738  1.00  0.76           O  \\nATOM    306  CG  GLU A  41       0.830  -6.414 -11.072  1.00  0.70           C  \\nATOM    307  CD  GLU A  41       1.835  -5.506 -10.381  1.00  0.68           C  \\nATOM    308  OE1 GLU A  41       2.282  -5.838  -9.260  1.00  0.66           O  \\nATOM    309  OE2 GLU A  41       2.177  -4.453 -10.965  1.00  0.61           O  \\nATOM    310  N   ASP A  42      -2.371  -9.531  -9.454  1.00  0.77           N  \\nATOM    311  CA  ASP A  42      -2.742 -10.555  -8.481  1.00  0.77           C  \\nATOM    312  C   ASP A  42      -1.748 -10.599  -7.323  1.00  0.77           C  \\nATOM    313  CB  ASP A  42      -2.828 -11.927  -9.153  1.00  0.74           C  \\nATOM    314  O   ASP A  42      -1.226 -11.664  -6.987  1.00  0.76           O  \\nATOM    315  CG  ASP A  42      -3.522 -12.967  -8.291  1.00  0.72           C  \\nATOM    316  OD1 ASP A  42      -4.369 -12.595  -7.450  1.00  0.70           O  \\nATOM    317  OD2 ASP A  42      -3.217 -14.168  -8.452  1.00  0.71           O  \\nATOM    318  N   LYS A  43      -1.324  -9.604  -6.708  1.00  0.80           N  \\nATOM    319  CA  LYS A  43      -0.380  -9.539  -5.596  1.00  0.80           C  \\nATOM    320  C   LYS A  43      -0.801  -8.483  -4.579  1.00  0.80           C  \\nATOM    321  CB  LYS A  43       1.031  -9.242  -6.106  1.00  0.78           C  \\nATOM    322  O   LYS A  43      -1.363  -7.449  -4.946  1.00  0.78           O  \\nATOM    323  CG  LYS A  43       1.655 -10.380  -6.900  1.00  0.74           C  \\nATOM    324  CD  LYS A  43       3.097 -10.072  -7.279  1.00  0.72           C  \\nATOM    325  CE  LYS A  43       3.725 -11.214  -8.068  1.00  0.64           C  \\nATOM    326  NZ  LYS A  43       5.142 -10.918  -8.438  1.00  0.59           N  \\nATOM    327  N   PRO A  44      -0.580  -8.819  -3.334  1.00  0.80           N  \\nATOM    328  CA  PRO A  44      -0.829  -7.792  -2.319  1.00  0.80           C  \\nATOM    329  C   PRO A  44       0.247  -6.708  -2.301  1.00  0.81           C  \\nATOM    330  CB  PRO A  44      -0.829  -8.583  -1.009  1.00  0.79           C  \\nATOM    331  O   PRO A  44       1.385  -6.956  -2.708  1.00  0.80           O  \\nATOM    332  CG  PRO A  44       0.051  -9.762  -1.275  1.00  0.77           C  \\nATOM    333  CD  PRO A  44      -0.074 -10.134  -2.724  1.00  0.76           C  \\nATOM    334  N   ALA A  45      -0.122  -5.502  -1.968  1.00  0.82           N  \\nATOM    335  CA  ALA A  45       0.790  -4.375  -1.792  1.00  0.82           C  \\nATOM    336  C   ALA A  45       0.439  -3.576  -0.541  1.00  0.83           C  \\nATOM    337  CB  ALA A  45       0.763  -3.471  -3.022  1.00  0.81           C  \\nATOM    338  O   ALA A  45      -0.701  -3.614  -0.071  1.00  0.82           O  \\nATOM    339  N   CYS A  46       1.460  -2.945   0.127  1.00  0.85           N  \\nATOM    340  CA  CYS A  46       1.230  -2.178   1.347  1.00  0.85           C  \\nATOM    341  C   CYS A  46       1.322  -0.681   1.076  1.00  0.85           C  \\nATOM    342  CB  CYS A  46       2.238  -2.574   2.426  1.00  0.83           C  \\nATOM    343  O   CYS A  46       2.156  -0.240   0.284  1.00  0.83           O  \\nATOM    344  SG  CYS A  46       2.008  -4.251   3.057  1.00  0.80           S  \\nATOM    345  N   VAL A  47       0.378   0.083   1.491  1.00  0.84           N  \\nATOM    346  CA  VAL A  47       0.504   1.529   1.635  1.00  0.84           C  \\nATOM    347  C   VAL A  47       1.048   1.866   3.021  1.00  0.85           C  \\nATOM    348  CB  VAL A  47      -0.848   2.242   1.406  1.00  0.82           C  \\nATOM    349  O   VAL A  47       0.360   1.679   4.027  1.00  0.84           O  \\nATOM    350  CG1 VAL A  47      -0.693   3.755   1.554  1.00  0.77           C  \\nATOM    351  CG2 VAL A  47      -1.409   1.891   0.029  1.00  0.77           C  \\nATOM    352  N   CYS A  48       2.192   2.309   3.101  1.00  0.86           N  \\nATOM    353  CA  CYS A  48       2.890   2.450   4.374  1.00  0.87           C  \\nATOM    354  C   CYS A  48       2.419   3.693   5.120  1.00  0.87           C  \\nATOM    355  CB  CYS A  48       4.401   2.519   4.152  1.00  0.85           C  \\nATOM    356  O   CYS A  48       2.108   4.712   4.502  1.00  0.85           O  \\nATOM    357  SG  CYS A  48       5.104   1.009   3.454  1.00  0.83           S  \\nATOM    358  N   HIS A  49       2.289   3.652   6.409  1.00  0.87           N  \\nATOM    359  CA  HIS A  49       2.103   4.818   7.265  1.00  0.87           C  \\nATOM    360  C   HIS A  49       3.323   5.732   7.223  1.00  0.87           C  \\nATOM    361  CB  HIS A  49       1.819   4.386   8.705  1.00  0.85           C  \\nATOM    362  O   HIS A  49       4.419   5.295   6.865  1.00  0.85           O  \\nATOM    363  CG  HIS A  49       0.549   3.612   8.860  1.00  0.78           C  \\nATOM    364  CD2 HIS A  49      -0.504   3.455   8.024  1.00  0.74           C  \\nATOM    365  ND1 HIS A  49       0.256   2.883   9.992  1.00  0.70           N  \\nATOM    366  CE1 HIS A  49      -0.927   2.310   9.845  1.00  0.73           C  \\nATOM    367  NE2 HIS A  49      -1.410   2.641   8.660  1.00  0.70           N  \\nATOM    368  N   SER A  50       3.078   6.941   7.467  1.00  0.89           N  \\nATOM    369  CA  SER A  50       4.186   7.890   7.436  1.00  0.88           C  \\nATOM    370  C   SER A  50       5.354   7.405   8.288  1.00  0.89           C  \\nATOM    371  CB  SER A  50       3.727   9.266   7.920  1.00  0.87           C  \\nATOM    372  O   SER A  50       5.151   6.841   9.365  1.00  0.87           O  \\nATOM    373  OG  SER A  50       3.361   9.221   9.289  1.00  0.78           O  \\nATOM    374  N   GLY A  51       6.595   7.433   7.763  1.00  0.87           N  \\nATOM    375  CA  GLY A  51       7.813   7.094   8.482  1.00  0.88           C  \\nATOM    376  C   GLY A  51       8.273   5.669   8.237  1.00  0.87           C  \\nATOM    377  O   GLY A  51       9.289   5.237   8.785  1.00  0.85           O  \\nATOM    378  N   TYR A  52       7.596   4.915   7.551  1.00  0.88           N  \\nATOM    379  CA  TYR A  52       7.927   3.525   7.254  1.00  0.88           C  \\nATOM    380  C   TYR A  52       8.006   3.293   5.750  1.00  0.87           C  \\nATOM    381  CB  TYR A  52       6.893   2.581   7.875  1.00  0.87           C  \\nATOM    382  O   TYR A  52       7.346   3.987   4.973  1.00  0.85           O  \\nATOM    383  CG  TYR A  52       6.905   2.576   9.385  1.00  0.84           C  \\nATOM    384  CD1 TYR A  52       7.746   1.719  10.091  1.00  0.81           C  \\nATOM    385  CD2 TYR A  52       6.076   3.427  10.108  1.00  0.82           C  \\nATOM    386  CE1 TYR A  52       7.760   1.709  11.482  1.00  0.83           C  \\nATOM    387  CE2 TYR A  52       6.081   3.425  11.499  1.00  0.82           C  \\nATOM    388  OH  TYR A  52       6.935   2.559  13.553  1.00  0.75           O  \\nATOM    389  CZ  TYR A  52       6.926   2.564  12.176  1.00  0.83           C  \\nATOM    390  N   VAL A  53       8.934   2.429   5.340  1.00  0.88           N  \\nATOM    391  CA  VAL A  53       9.180   2.056   3.951  1.00  0.88           C  \\nATOM    392  C   VAL A  53       9.358   0.543   3.846  1.00  0.88           C  \\nATOM    393  CB  VAL A  53      10.420   2.780   3.380  1.00  0.86           C  \\nATOM    394  O   VAL A  53       9.512  -0.142   4.859  1.00  0.87           O  \\nATOM    395  CG1 VAL A  53      10.191   4.289   3.334  1.00  0.75           C  \\nATOM    396  CG2 VAL A  53      11.659   2.449   4.211  1.00  0.75           C  \\nATOM    397  N   GLY A  54       9.345  -0.026   2.484  1.00  0.88           N  \\nATOM    398  CA  GLY A  54       9.516  -1.437   2.173  1.00  0.87           C  \\nATOM    399  C   GLY A  54       8.284  -2.065   1.550  1.00  0.87           C  \\nATOM    400  O   GLY A  54       7.216  -1.449   1.518  1.00  0.86           O  \\nATOM    401  N   ALA A  55       8.335  -3.279   1.124  1.00  0.87           N  \\nATOM    402  CA  ALA A  55       7.236  -4.000   0.487  1.00  0.87           C  \\nATOM    403  C   ALA A  55       6.136  -4.323   1.493  1.00  0.87           C  \\nATOM    404  CB  ALA A  55       7.749  -5.282  -0.166  1.00  0.84           C  \\nATOM    405  O   ALA A  55       4.953  -4.341   1.145  1.00  0.85           O  \\nATOM    406  N   ARG A  56       6.490  -4.544   2.739  1.00  0.88           N  \\nATOM    407  CA  ARG A  56       5.530  -4.823   3.802  1.00  0.88           C  \\nATOM    408  C   ARG A  56       5.534  -3.712   4.848  1.00  0.89           C  \\nATOM    409  CB  ARG A  56       5.837  -6.167   4.465  1.00  0.87           C  \\nATOM    410  O   ARG A  56       5.073  -3.911   5.974  1.00  0.88           O  \\nATOM    411  CG  ARG A  56       5.733  -7.356   3.523  1.00  0.81           C  \\nATOM    412  CD  ARG A  56       5.949  -8.675   4.253  1.00  0.79           C  \\nATOM    413  NE  ARG A  56       4.897  -8.926   5.234  1.00  0.72           N  \\nATOM    414  NH1 ARG A  56       6.022 -10.633   6.310  1.00  0.56           N  \\nATOM    415  NH2 ARG A  56       3.941  -9.995   7.028  1.00  0.54           N  \\nATOM    416  CZ  ARG A  56       4.956  -9.851   6.188  1.00  0.76           C  \\nATOM    417  N   CYS A  57       6.207  -2.525   4.478  1.00  0.88           N  \\nATOM    418  CA  CYS A  57       6.315  -1.439   5.445  1.00  0.89           C  \\nATOM    419  C   CYS A  57       7.023  -1.904   6.711  1.00  0.88           C  \\nATOM    420  CB  CYS A  57       4.930  -0.894   5.795  1.00  0.86           C  \\nATOM    421  O   CYS A  57       6.551  -1.651   7.821  1.00  0.85           O  \\nATOM    422  SG  CYS A  57       3.978  -0.350   4.360  1.00  0.80           S  \\nATOM    423  N   GLU A  58       8.179  -2.658   6.525  1.00  0.89           N  \\nATOM    424  CA  GLU A  58       8.862  -3.273   7.660  1.00  0.89           C  \\nATOM    425  C   GLU A  58      10.025  -2.409   8.138  1.00  0.89           C  \\nATOM    426  CB  GLU A  58       9.361  -4.672   7.293  1.00  0.87           C  \\nATOM    427  O   GLU A  58      10.556  -2.623   9.230  1.00  0.87           O  \\nATOM    428  CG  GLU A  58      10.477  -4.675   6.257  1.00  0.81           C  \\nATOM    429  CD  GLU A  58       9.967  -4.760   4.827  1.00  0.79           C  \\nATOM    430  OE1 GLU A  58       8.872  -4.226   4.540  1.00  0.77           O  \\nATOM    431  OE2 GLU A  58      10.668  -5.367   3.987  1.00  0.75           O  \\nATOM    432  N   HIS A  59      10.386  -1.405   7.414  1.00  0.90           N  \\nATOM    433  CA  HIS A  59      11.550  -0.604   7.774  1.00  0.90           C  \\nATOM    434  C   HIS A  59      11.146   0.821   8.141  1.00  0.89           C  \\nATOM    435  CB  HIS A  59      12.563  -0.584   6.628  1.00  0.88           C  \\nATOM    436  O   HIS A  59      10.156   1.342   7.624  1.00  0.88           O  \\nATOM    437  CG  HIS A  59      13.011  -1.945   6.200  1.00  0.81           C  \\nATOM    438  CD2 HIS A  59      13.681  -2.912   6.869  1.00  0.75           C  \\nATOM    439  ND1 HIS A  59      12.777  -2.442   4.936  1.00  0.71           N  \\nATOM    440  CE1 HIS A  59      13.286  -3.659   4.847  1.00  0.76           C  \\nATOM    441  NE2 HIS A  59      13.840  -3.968   6.007  1.00  0.74           N  \\nATOM    442  N   ALA A  60      11.894   1.412   9.087  1.00  0.87           N  \\nATOM    443  CA  ALA A  60      11.732   2.831   9.396  1.00  0.87           C  \\nATOM    444  C   ALA A  60      12.386   3.703   8.328  1.00  0.87           C  \\nATOM    445  CB  ALA A  60      12.320   3.146  10.769  1.00  0.84           C  \\nATOM    446  O   ALA A  60      13.467   3.379   7.829  1.00  0.84           O  \\nATOM    447  N   ASP A  61      11.562   4.580   7.614  1.00  0.85           N  \\nATOM    448  CA  ASP A  61      12.133   5.508   6.643  1.00  0.85           C  \\nATOM    449  C   ASP A  61      13.145   6.440   7.304  1.00  0.83           C  \\nATOM    450  CB  ASP A  61      11.030   6.324   5.967  1.00  0.82           C  \\nATOM    451  O   ASP A  61      12.766   7.378   8.008  1.00  0.79           O  \\nATOM    452  CG  ASP A  61      11.525   7.111   4.766  1.00  0.75           C  \\nATOM    453  OD1 ASP A  61      12.753   7.165   4.537  1.00  0.74           O  \\nATOM    454  OD2 ASP A  61      10.681   7.684   4.045  1.00  0.76           O  \\nATOM    455  N   LEU A  62      14.420   6.132   7.317  1.00  0.81           N  \\nATOM    456  CA  LEU A  62      15.479   6.887   7.979  1.00  0.80           C  \\nATOM    457  C   LEU A  62      15.554   8.309   7.433  1.00  0.79           C  \\nATOM    458  CB  LEU A  62      16.829   6.187   7.801  1.00  0.75           C  \\nATOM    459  O   LEU A  62      16.033   9.216   8.117  1.00  0.74           O  \\nATOM    460  CG  LEU A  62      16.970   4.811   8.453  1.00  0.70           C  \\nATOM    461  CD1 LEU A  62      18.297   4.172   8.059  1.00  0.65           C  \\nATOM    462  CD2 LEU A  62      16.854   4.923   9.969  1.00  0.66           C  \\nATOM    463  N   LEU A  63      15.136   8.507   6.139  1.00  0.76           N  \\nATOM    464  CA  LEU A  63      15.200   9.852   5.578  1.00  0.76           C  \\nATOM    465  C   LEU A  63      14.020  10.695   6.051  1.00  0.74           C  \\nATOM    466  CB  LEU A  63      15.220   9.795   4.049  1.00  0.71           C  \\nATOM    467  O   LEU A  63      14.007  11.913   5.865  1.00  0.70           O  \\nATOM    468  CG  LEU A  63      16.446   9.138   3.411  1.00  0.65           C  \\nATOM    469  CD1 LEU A  63      16.247   8.994   1.906  1.00  0.61           C  \\nATOM    470  CD2 LEU A  63      17.704   9.944   3.715  1.00  0.62           C  \\nATOM    471  N   ALA A  64      12.871  10.157   6.394  1.00  0.65           N  \\nATOM    472  CA  ALA A  64      11.705  10.897   6.871  1.00  0.64           C  \\nATOM    473  C   ALA A  64      12.030  11.670   8.145  1.00  0.63           C  \\nATOM    474  CB  ALA A  64      10.535   9.946   7.112  1.00  0.59           C  \\nATOM    475  O   ALA A  64      11.338  12.632   8.488  1.00  0.60           O  \\nATOM    476  N   VAL A  65      13.127  11.375   8.769  1.00  0.54           N  \\nATOM    477  CA  VAL A  65      13.475  12.163   9.947  1.00  0.57           C  \\nATOM    478  C   VAL A  65      14.396  13.313   9.546  1.00  0.54           C  \\nATOM    479  CB  VAL A  65      14.150  11.293  11.031  1.00  0.51           C  \\nATOM    480  O   VAL A  65      14.255  14.432  10.045  1.00  0.53           O  \\nATOM    481  CG1 VAL A  65      14.153  12.015  12.378  1.00  0.45           C  \\nATOM    482  CG2 VAL A  65      13.445   9.943  11.147  1.00  0.46           C  \",\"pdb\");\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 1},{\"cartoon\": {\"color\": \"#0c00f3\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 2},{\"cartoon\": {\"color\": \"#0c00f3\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 3},{\"cartoon\": {\"color\": \"#0c00f3\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 4},{\"cartoon\": {\"color\": \"#1f00e0\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 5},{\"cartoon\": {\"color\": \"#1f00e0\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 6},{\"cartoon\": {\"color\": \"#1f00e0\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 7},{\"cartoon\": {\"color\": \"#2100de\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 8},{\"cartoon\": {\"color\": \"#4700b8\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 9},{\"cartoon\": {\"color\": \"#7b0084\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 10},{\"cartoon\": {\"color\": \"#80007f\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 11},{\"cartoon\": {\"color\": \"#8b0074\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 12},{\"cartoon\": {\"color\": \"#5a00a5\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 13},{\"cartoon\": {\"color\": \"#2700d8\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 14},{\"cartoon\": {\"color\": \"#3200cd\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 15},{\"cartoon\": {\"color\": \"#2c00d3\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 16},{\"cartoon\": {\"color\": \"#5800a7\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 17},{\"cartoon\": {\"color\": \"#5600a9\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 18},{\"cartoon\": {\"color\": \"#4700b8\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 19},{\"cartoon\": {\"color\": \"#70008f\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 20},{\"cartoon\": {\"color\": \"#75008a\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 21},{\"cartoon\": {\"color\": \"#760089\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 22},{\"cartoon\": {\"color\": \"#af0050\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 23},{\"cartoon\": {\"color\": \"#81007e\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 24},{\"cartoon\": {\"color\": \"#4800b7\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 25},{\"cartoon\": {\"color\": \"#7f0080\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 26},{\"cartoon\": {\"color\": \"#4600b9\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 27},{\"cartoon\": {\"color\": \"#63009c\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 28},{\"cartoon\": {\"color\": \"#9b0064\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 29},{\"cartoon\": {\"color\": \"#64009b\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 30},{\"cartoon\": {\"color\": \"#64009b\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 31},{\"cartoon\": {\"color\": \"#7e0081\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 32},{\"cartoon\": {\"color\": \"#4500ba\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 33},{\"cartoon\": {\"color\": \"#7d0082\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 34},{\"cartoon\": {\"color\": \"#94006b\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 35},{\"cartoon\": {\"color\": \"#5c00a3\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 36},{\"cartoon\": {\"color\": \"#5c00a3\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 37},{\"cartoon\": {\"color\": \"#2300dc\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 38},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 39},{\"cartoon\": {\"color\": \"#4600b9\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 40},{\"cartoon\": {\"color\": \"#7e0081\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 41},{\"cartoon\": {\"color\": \"#80007f\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 42},{\"cartoon\": {\"color\": \"#80007f\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 43},{\"cartoon\": {\"color\": \"#4b00b4\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 44},{\"cartoon\": {\"color\": \"#4b00b4\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 45},{\"cartoon\": {\"color\": \"#4a00b5\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 46},{\"cartoon\": {\"color\": \"#83007c\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 47},{\"cartoon\": {\"color\": \"#7f0080\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 48},{\"cartoon\": {\"color\": \"#4d00b2\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 49},{\"cartoon\": {\"color\": \"#860079\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 50},{\"cartoon\": {\"color\": \"#860079\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 51},{\"cartoon\": {\"color\": \"#870078\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 52},{\"cartoon\": {\"color\": \"#80007f\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 53},{\"cartoon\": {\"color\": \"#80007f\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 54},{\"cartoon\": {\"color\": \"#7f0080\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 55},{\"cartoon\": {\"color\": \"#860079\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 56},{\"cartoon\": {\"color\": \"#880077\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 57},{\"cartoon\": {\"color\": \"#870078\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 58},{\"cartoon\": {\"color\": \"#5400ab\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 59},{\"cartoon\": {\"color\": \"#4d00b2\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 60},{\"cartoon\": {\"color\": \"#4c00b3\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 61},{\"cartoon\": {\"color\": \"#1300ec\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 62},{\"cartoon\": {\"color\": \"#0e00f1\"}});\n",
              "\tviewer_17302037596177619.setStyle({\"resi\": 63},{\"cartoon\": {\"color\": \"#0e00f1\"}});\n",
              "\tviewer_17302037596177619.zoomTo();\n",
              "viewer_17302037596177619.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_7a7f9e7c-64e5-4247-8bd7-9ab639a7cd40\", \"A.html\", 47987)"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "2024-10-29 12:09:19.625 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:19.627 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:19.629 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:19.630 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:19.631 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:09:19.633 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 600x100 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAfoAAABlCAYAAAClUiFrAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAAAWWElEQVR4nO3dfVTV9R0H8Pfl4V4gLgIaXECGCAFGpinpIUNGY4PcfNjOSVuGsNTWtJOD5UO5hmmgh7QsjkXDBO2wbGu4Y0ia2dRJuZ0hOI8yTEDTBrbOCh95vJ/9ofdyH34XuQRX+fl+ncO5/H7fp8/3w7184PrDn0ZEBERERKRKbjc7ACIiIho8LPREREQqxkJPRESkYiz0REREKsZCT0REpGIs9ERERCrGQk9ERKRiLPREREQq5jEYk7a1taGjo2MwpiYiIlI1rVYLLy+vAZtvwAt9W1sbIkeNQsv58wM9NRERkeoZDAY0NTUNWLEf8ELf0dGBlvPncfbkcfj5+l4/K4Dpf9o1/4e7pmPLR5s+5v+dVxSOrc+JUt9ex9uu04e1Hc0nRutjo9Gir02bZbxGo/J4y2PT50YHa1k+OpzH2LOmoz5K89uuYVQa21ubzbx28yms6XCfCsd2cUFhvn7E5TDHSvtU2P8N89bzPBK7NtiPVfpaO5rXtk3pdWa0ec2Yu14/b+ytr8LrzUEfpfnEZi3FWGxf032Jy8F817681rk1t6HnuGe4aTys1rROsfU5uzFW3yp64ri2pv1YsY1L4dg2HqOjvViuef2cbcrFHIVFH4s203nbp6QRNmuZPrEYZ/Oss3i0yLFtm8IeeuJS2N/1eE3xGG3jNK8jFn1t92nbV2GcQh9TTErjLfdidDDOuq9Fm8WaHRCUtbSgo6Pj1i30Jn56Pfz0+utHCt/8vkuht+qr8I3HdP6Ghd7JtftT6B0UrIEr9H3p299Cr7Av27G9/hBgM29f1rzRPpXmc7R2fwu9o8KsOJ/CGCfmsy/0CmO/S6FX3IPNa6aXwnzDQu/kfDcu9KIcx43i6kOh762gOizaGtu+9uPtC71C8dYor+10obctbkpF0sWFXpTigu2j2Mdj0Wa3BwdrGi0ebQuyfR+x6Ou4zXEfx4+O2qQPfRTnEeu2gcaL8YiIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxTwGa+ILFy8CItePpOdz0ynTJ2L5aNPHcrzdsfU5Uerb63jbdfqwtqP5xGh9bDRa9LVps4zXaFQeb3ls+tzoYC3LR4fzGHvWdNRHaX7bNYxKY3trs5nXbj6FNR3uU+HYLi4ozNePuBzmWGmfCvu/Yd56nkdi1wb7sUpfa0fz2rYpvc6MNq8Zc9fr54299VV4vTnoozSf2KylGIvta7ovcTmY79qX1zq35jb0HPcMN42H1ZrWKbY+ZzfG6ltFTxzX1rQfK7ZxKRzbxmN0tBfLNa+fs025mKOw6GPRZjpv+5Q0wmYt0ycW42yedRaPFjm2bVPYQ09cCvu7Hq8pHqNtnOZ1xKKv7T5t+yqMU+hjiklpvOVejA7GWfe1aLNYs6OnSA6YAS/0IgJfX1+Ex8QP9NRERESq5+vra/HD9Hc34IVeo9Hg0qVLOHv2LPz8/AZ6elJw4cIFhIeHM+cuxJy7HnPuesy565lyrtFoBmzOQXvr3s/Pj08MF2POXY85dz3m3PWY86GNF+MRERGpGAs9ERGRig14odfpdMjNzYVOpxvoqckB5tz1mHPXY85djzl3vcHIuUYG8tI+IiIiuqXwrXsiIiIVY6EnIiJSMRZ6IiIiFWOhJyIiUrF+FfpNmzZh1KhR8PLywuTJk/GPf/yj1/5/+tOfEBcXBy8vL4wdOxaVlZX9CvZ25kzOi4uLkZSUhICAAAQEBCA1NfWGXyOy5+zz3GT79u3QaDSYNWvW4AaoQs7m/Ntvv8XixYsREhICnU6HmJgYfn9xkrM537hxI2JjY+Ht7Y3w8HBkZ2ejra3NRdEObQcPHsT06dMRGhoKjUaDv/zlLzccs3//fkyYMAE6nQ7R0dEoLS11fmFx0vbt20Wr1cqWLVvk+PHjsnDhQvH395fz588r9q+qqhJ3d3cpKCiQEydOyG9/+1vx9PSUY8eOObv0bcvZnD/22GOyadMmqampkbq6OsnKypJhw4bJuXPnXBz50OVszk2ampokLCxMkpKSZObMma4JViWczXl7e7skJCTItGnT5NChQ9LU1CT79++X2tpaF0c+dDmb87KyMtHpdFJWViZNTU2yZ88eCQkJkezsbBdHPjRVVlbKypUrpby8XADIjh07eu3f2NgoPj4+kpOTIydOnJDCwkJxd3eX3bt3O7Wu04V+0qRJsnjxYvNxd3e3hIaGytq1axX7z549W3784x9bnZs8ebL88pe/dHbp25azObfV1dUler1etm7dOlghqk5/ct7V1SUPPPCAbN68WTIzM1noneRszt98800ZPXq0dHR0uCpE1XE254sXL5aHHnrI6lxOTo5MmTJlUONUo74U+mXLlkl8fLzVuTlz5khaWppTazn11n1HRweqq6uRmppqPufm5obU1FR89tlnimM+++wzq/4AkJaW5rA/WetPzm1duXIFnZ2dCAwMHKwwVaW/OV+9ejWCgoIwf/58V4SpKv3J+c6dO5GYmIjFixcjODgY99xzD/Lz89Hd3e2qsIe0/uT8gQceQHV1tfnt/cbGRlRWVmLatGkuifl2M1D106mb2nz99dfo7u5GcHCw1fng4GD8+9//VhzT0tKi2L+lpcWpQG9X/cm5reXLlyM0NNTuCUPK+pPzQ4cO4e2330Ztba0LIlSf/uS8sbERn3zyCebOnYvKykqcOnUKixYtQmdnJ3Jzc10R9pDWn5w/9thj+Prrr/Hggw9CRNDV1YWnnnoKzz//vCtCvu04qp8XLlzA1atX4e3t3ad5eNW9yq1btw7bt2/Hjh074OXldbPDUaWLFy8iIyMDxcXFGDFixM0O57ZhNBoRFBSE3//+95g4cSLmzJmDlStXoqio6GaHplr79+9Hfn4+3njjDRw5cgTl5eXYtWsX1qxZc7NDo1449Rv9iBEj4O7ujvPnz1udP3/+PAwGg+IYg8HgVH+y1p+cm6xfvx7r1q3Dxx9/jHvvvXcww1QVZ3Pe0NCA06dPY/r06eZzRqMRAODh4YH6+npERUUNbtBDXH+e5yEhIfD09IS7u7v53JgxY9DS0oKOjg5otdpBjXmo60/OX3jhBWRkZGDBggUAgLFjx+Ly5ct48sknsXLlSri58XfHgeSofvr5+fX5t3nAyd/otVotJk6ciH379pnPGY1G7Nu3D4mJiYpjEhMTrfoDwN69ex32J2v9yTkAFBQUYM2aNdi9ezcSEhJcEapqOJvzuLg4HDt2DLW1teaPGTNmICUlBbW1tQgPD3dl+ENSf57nU6ZMwalTp8w/VAHAyZMnERISwiLfB/3J+ZUrV+yKuekHLeFtUwbcgNVP564TvPbnGDqdTkpLS+XEiRPy5JNPir+/v7S0tIiISEZGhqxYscLcv6qqSjw8PGT9+vVSV1cnubm5/PM6Jzmb83Xr1olWq5X3339fmpubzR8XL168WVsYcpzNuS1ede88Z3P+xRdfiF6vl6efflrq6+uloqJCgoKC5KWXXrpZWxhynM15bm6u6PV6effdd6WxsVE++ugjiYqKktmzZ9+sLQwpFy9elJqaGqmpqREA8sorr0hNTY2cOXNGRERWrFghGRkZ5v6mP69bunSp1NXVyaZNm1zz53UiIoWFhfK9731PtFqtTJo0SQ4fPmxuS05OlszMTKv+f/zjHyUmJka0Wq3Ex8fLrl27+rPsbc2ZnEdERAgAu4/c3FzXBz6EOfs8t8RC3z/O5vzTTz+VyZMni06nk9GjR0teXp50dXW5OOqhzZmcd3Z2yqpVqyQqKkq8vLwkPDxcFi1aJN98843rAx+C/vrXvyp+bzblODMzU5KTk+3GjB8/XrRarYwePVpKSkqcXpe3qSUiIlIxXjlBRESkYiz0REREKsZCT0REpGIs9ERERCrGQk9ERKRiLPREREQqxkJPRESkYiz0RN9RVlYWZs2aZT7+/ve/j1//+tcuj2P//v3QaDT49ttvXb42Ed26WOhJlbKysqDRaKDRaKDVahEdHY3Vq1ejq6tr0NcuLy/v8928XF2cR40ahY0bNyq2nT59GhqNxuGtdktLS805dXNzw8iRI/GLX/wCX331lbnPgQMH8NBDDyEwMBA+Pj646667kJmZiY6ODocxHT16FDNmzEBQUBC8vLwwatQozJkzx2peIuo/FnpSrfT0dDQ3N+Pzzz/Hb37zG6xatQovv/yyYt/eCpGzAgMDodfrB2y+W4mfnx+am5tx7tw5FBcX48MPP0RGRgYA4MSJE0hPT0dCQgIOHjyIY8eOobCwEFqtFt3d3Yrz/fe//8UPfvADBAYGYs+ePairq0NJSQlCQ0Nx+fLlQdtHZ2fnoM1NdKthoSfV0ul0MBgMiIiIwK9+9SukpqZi586dAHrebs/Ly0NoaChiY2MBAGfPnsXs2bPh7++PwMBAzJw5E6dPnzbP2d3djZycHPj7+2P48OFYtmyZ3V27bN+6b29vx/LlyxEeHg6dTofo6Gi8/fbbOH36NFJSUgAAAQEB0Gg0yMrKAnDtLmJr165FZGQkvL29MW7cOLz//vtW61RWViImJgbe3t5ISUmxinOwaDQaGAwGhIaG4uGHH8YzzzyDjz/+GFevXsVHH30Eg8GAgoIC3HPPPYiKikJ6ejqKi4sd3lKzqqoKra2t2Lx5M+677z5ERkYiJSUFr776KiIjI839jh8/jp/85Cfw8/ODXq9HUlISGhoaAFzL1erVqzFy5EjodDqMHz8eu3fvNo81vVPx3nvvITk5GV5eXigrKwMAbN68GWPGjIGXlxfi4uLwxhtvDGL2iG4OFnq6bXh7e1v95r5v3z7U19dj7969qKioQGdnJ9LS0qDX6/G3v/0NVVVV8PX1RXp6unnchg0bUFpaii1btuDQoUP43//+hx07dvS67rx58/Duu+/i9ddfR11dHd566y34+voiPDwcf/7znwEA9fX1aG5uxmuvvQYAWLt2LbZt24aioiIcP34c2dnZePzxx3HgwAEA134g+dnPfobp06ejtrYWCxYswIoVKwYjbb3y9vaG0WhEV1cXDAYDmpubcfDgwT6PNxgM6Orqwo4dOxze5vTLL7/E1KlTodPp8Mknn6C6uhpPPPGE+Z9hXnvtNWzYsAHr16/Hv/71L6SlpWHGjBn4/PPPreZZsWIFlixZgrq6OqSlpaGsrAy/+93vkJeXh7q6OuTn5+OFF17A1q1b+58QolvRd7sXD9GtyfLucUajUfbu3Ss6nU6effZZc3twcLC0t7ebx7zzzjsSGxsrRqPRfK69vV28vb1lz549IiISEhIiBQUF5vbOzk4ZOXKk1Z3qkpOTZcmSJSIiUl9fLwBk7969inGa7mZlefevtrY28fHxkU8//dSq7/z58+XnP/+5iIg899xzcvfdd1u1L1++3G4uWxEREfLqq68qtjU1NQkAqampUWwvKSmRYcOGmY9PnjwpMTExkpCQICIiXV1dkpWVJQDEYDDIrFmzpLCwUFpbWx3GIyLy/PPPi4eHhwQGBkp6eroUFBSYb5Nq2mtkZKR0dHQojg8NDZW8vDyrc/fff78sWrTIal8bN2606hMVFSV/+MMfrM6tWbNGEhMTe42XaKjhb/SkWhUVFfD19YWXlxcefvhhzJkzB6tWrTK3jx07Flqt1nx89OhRnDp1Cnq9Hr6+vvD19UVgYCDa2trQ0NCA1tZWNDc3Y/LkyeYxHh4eSEhIcBhDbW0t3N3dkZyc3Oe4T506hStXruCHP/yhOQ5fX19s27bN/HZ1XV2dVRwAkJiY2Oc1+qu1tRW+vr7w8fFBbGwsgoODzW+Du7u7o6SkBOfOnUNBQQHCwsKQn5+P+Ph4NDc3O5wzLy8PLS0tKCoqQnx8PIqKihAXF4djx44BuJbDpKQkeHp62o29cOEC/vOf/2DKlClW56dMmYK6ujqrc5Zfp8uXL6OhoQHz58+3yvFLL71kzjGRWnjc7ACIBktKSgrefPNNaLVahIaGwsPD+ul+xx13WB1funQJEydONBcuS3feeWe/YnD0b9O9uXTpEgBg165dCAsLs2rT6XT9imOg6PV6HDlyBG5ubggJCVHcX1hYGDIyMpCRkYE1a9YgJiYGRUVFePHFFx3OO3z4cDzyyCN45JFHkJ+fj/vuuw/r16/H1q1b+5VDJZZfb1OOi4uL7X5gcnd3H5D1iG4VLPSkWnfccQeio6P73H/ChAl47733EBQUBD8/P8U+ISEh+Pvf/46pU6cCALq6ulBdXY0JEyYo9h87diyMRiMOHDiA1NRUu3bTOwqWV6Xffffd0Ol0+OKLLxy+EzBmzBjzhYUmhw8fvvEmvyM3NzenchoQEICQkBCnrqDXarWIiooyj7n33nuxdetWdHZ22v1W7+fnh9DQUFRVVVnlqqqqCpMmTXK4RnBwMEJDQ9HY2Ii5c+f2OTaioYiFnui6uXPn4uWXX8bMmTPNV3GfOXMG5eXlWLZsGUaOHIklS5Zg3bp1uOuuuxAXF4dXXnml17+BHzVqFDIzM/HEE0/g9ddfx7hx43DmzBl89dVXmD17NiIiIqDRaFBRUYFp06bB29sber0ezz77LLKzs2E0GvHggw+itbUVVVVV8PPzQ2ZmJp566ils2LABS5cuxYIFC1BdXY3S0tI+7fPLL7+0+1v5iIgI8+f19fV2Y+Lj428471tvvYXa2lr89Kc/RVRUFNra2rBt2zYcP34chYWFimMqKiqwfft2PProo4iJiYGI4IMPPkBlZSVKSkoAAE8//TQKCwvx6KOP4rnnnsOwYcNw+PBhTJo0CbGxsVi6dClyc3MRFRWF8ePHo6SkBLW1tYrvzFh68cUX8cwzz2DYsGFIT09He3s7/vnPf+Kbb75BTk7ODfdLNGTc7IsEiAaD5cV4zrQ3NzfLvHnzZMSIEaLT6WT06NGycOFC8wVlnZ2dsmTJEvHz8xN/f3/JycmRefPmObwYT0Tk6tWrkp2dLSEhIaLVaiU6Olq2bNlibl+9erUYDAbRaDSSmZkpItcuINy4caPExsaKp6en3HnnnZKWliYHDhwwj/vggw8kOjpadDqdJCUlyZYtW/p0MR4Au4933nnHfNGa0sfZs2ftLsazdeTIEXn88cclMjJSdDqdDB8+XKZOnSo7d+50OKahoUEWLlwoMTEx4u3tLf7+/nL//fdLSUmJVb+jR4/Kj370I/Hx8RG9Xi9JSUnS0NAgIiLd3d2yatUqCQsLE09PTxk3bpx8+OGH5rG9XWRYVlYm48ePF61WKwEBATJ16lQpLy93GC/RUKQRcfA3LURERDTk8ap7IiIiFWOhJyIiUjEWeiIiIhVjoSciIlIxFnoiIiIVY6EnIiJSMRZ6IiIiFWOhJyIiUjEWeiIiIhVjoSciIlIxFnoiIiIVY6EnIiJSsf8DMIiDYEQUuXYAAAAASUVORK5CYII=\n"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 34,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "LSnHOep_Mvyy",
        "outputId": "d3c46d5e-d91c-476b-c8b7-9798157f0f37",
        "cellView": "form"
      },
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<html>\n",
              "<head><meta charset=\"utf-8\" /></head>\n",
              "<body>\n",
              "    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax && window.MathJax.Hub && window.MathJax.Hub.Config) {window.MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n",
              "        <script charset=\"utf-8\" src=\"https://cdn.plot.ly/plotly-2.35.2.min.js\"></script>                <div id=\"fd9e8ce8-d72a-4657-9b55-1c5e6d0c3e30\" class=\"plotly-graph-div\" style=\"height:800px; width:1700px;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"fd9e8ce8-d72a-4657-9b55-1c5e6d0c3e30\")) {                    Plotly.newPlot(                        \"fd9e8ce8-d72a-4657-9b55-1c5e6d0c3e30\",                        [{\"marker\":{\"color\":\"lightblue\"},\"name\":\"Below Threshold\",\"width\":0.6,\"x\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52],\"y\":[0.04982471093535423,0.05371448025107384,0.052452425161997475,0.05186203494668007,0.19051217008382082,0.18893855903297663,0.18867887184023857,0.25234416127204895,0.320746049284935,0.3199102645739913,0.5,0.5,0.5,0.5,0.4957986641675234,0.2713504023849964,0.2735261060297489,0.49177967943251133,0.5,0.5,0.5,0.35193790774792433,0.1307492358610034,0.12766801007092,0.14227838162332773,0.07116931676864624,0.07157211285084486,0.06892109010368586,0.271085393615067,0.492100877687335,0.4960729470476508,0.5,0.5,0.31640778575092554,0.3116879425942898,0.31252045556902885,0.41318424697965384,0.37728158477693796,0.5,0.5,0.5,0.5,0.5,0.5,0.45144943054765463,0.4524252722039819,0.22858523577451706,0.05343150720000267,0.06769882794469595,0.06613709311932325,0.0663314713165164,0.06894452249010403,0.0500356201082468],\"type\":\"bar\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"base\":[0.04982471093535423,0.05371448025107384,0.052452425161997475,0.05186203494668007,0.19051217008382082,0.18893855903297663,0.18867887184023857,0.25234416127204895,0.320746049284935,0.3199102645739913,0.5,0.5,0.5,0.5,0.4957986641675234,0.2713504023849964,0.2735261060297489,0.49177967943251133,0.5,0.5,0.5,0.35193790774792433,0.1307492358610034,0.12766801007092,0.14227838162332773,0.07116931676864624,0.07157211285084486,0.06892109010368586,0.271085393615067,0.492100877687335,0.4960729470476508,0.5,0.5,0.31640778575092554,0.3116879425942898,0.31252045556902885,0.41318424697965384,0.37728158477693796,0.5,0.5,0.5,0.5,0.5,0.5,0.45144943054765463,0.4524252722039819,0.22858523577451706,0.05343150720000267,0.06769882794469595,0.06613709311932325,0.0663314713165164,0.06894452249010403,0.0500356201082468],\"marker\":{\"color\":\"lightgreen\"},\"name\":\"Above Threshold\",\"width\":0.6,\"x\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52],\"y\":[0,0,0,0,0,0,0,0,0,0,0.04324487783014774,0.20349680073559284,0.2161730844527483,0.2192674670368433,0,0,0,0,0.2140508759766817,0.22029469721019268,0.0008126003667712212,0,0,0,0,0,0,0,0,0,0,0.21905973181128502,0.0026742154732346535,0,0,0,0,0,0.10165921039879322,0.09974663145840168,0.08170612342655659,0.08244670927524567,0.08154985308647156,0.03691670298576355,0,0,0,0,0,0,0,0,0],\"type\":\"bar\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"black\",\"dash\":\"dash\"},\"mode\":\"lines\",\"showlegend\":false,\"x\":[0,52],\"y\":[0.5,0.5],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[11,12],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[12,13],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[13,14],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[18,19],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[19,20],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"}],                        {\"template\":{\"data\":{\"candlestick\":[{\"decreasing\":{\"line\":{\"color\":\"#000033\"}},\"increasing\":{\"line\":{\"color\":\"#000032\"}},\"type\":\"candlestick\"}],\"contourcarpet\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"contourcarpet\"}],\"contour\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"contour\"}],\"heatmap\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"heatmap\"}],\"histogram2d\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"histogram2d\"}],\"icicle\":[{\"textfont\":{\"color\":\"white\"},\"type\":\"icicle\"}],\"sankey\":[{\"textfont\":{\"color\":\"#000036\"},\"type\":\"sankey\"}],\"scatter\":[{\"marker\":{\"line\":{\"width\":0}},\"type\":\"scatter\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#000038\"},\"font\":{\"color\":\"#000037\"},\"line\":{\"color\":\"#000039\"}},\"header\":{\"fill\":{\"color\":\"#000040\"},\"font\":{\"color\":\"#000036\"},\"line\":{\"color\":\"#000039\"}},\"type\":\"table\"}],\"waterfall\":[{\"connector\":{\"line\":{\"color\":\"#000036\",\"width\":2}},\"decreasing\":{\"marker\":{\"color\":\"#000033\"}},\"increasing\":{\"marker\":{\"color\":\"#000032\"}},\"totals\":{\"marker\":{\"color\":\"#000034\"}},\"type\":\"waterfall\"}]},\"layout\":{\"coloraxis\":{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]]},\"colorscale\":{\"diverging\":[[0.0,\"#000021\"],[0.1,\"#000022\"],[0.2,\"#000023\"],[0.3,\"#000024\"],[0.4,\"#000025\"],[0.5,\"#000026\"],[0.6,\"#000027\"],[0.7,\"#000028\"],[0.8,\"#000029\"],[0.9,\"#000030\"],[1.0,\"#000031\"]],\"sequential\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"sequentialminus\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]]},\"colorway\":[\"#000001\",\"#000002\",\"#000003\",\"#000004\",\"#000005\",\"#000006\",\"#000007\",\"#000008\",\"#000009\",\"#000010\"]}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0]},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0]},\"title\":{\"text\":\"LLPS Prediction Plot\"},\"height\":800,\"width\":1700},                        {\"responsive\": true}                    ).then(function(){\n",
              "                            \n",
              "var gd = document.getElementById('fd9e8ce8-d72a-4657-9b55-1c5e6d0c3e30');\n",
              "var x = new MutationObserver(function (mutations, observer) {{\n",
              "        var display = window.getComputedStyle(gd).display;\n",
              "        if (!display || display === 'none') {{\n",
              "            console.log([gd, 'removed!']);\n",
              "            Plotly.purge(gd);\n",
              "            observer.disconnect();\n",
              "        }}\n",
              "}});\n",
              "\n",
              "// Listen for the removal of the full notebook cells\n",
              "var notebookContainer = gd.closest('#notebook-container');\n",
              "if (notebookContainer) {{\n",
              "    x.observe(notebookContainer, {childList: true});\n",
              "}}\n",
              "\n",
              "// Listen for the clearing of the current output cell\n",
              "var outputEl = gd.closest('.output');\n",
              "if (outputEl) {{\n",
              "    x.observe(outputEl, {childList: true});\n",
              "}}\n",
              "\n",
              "                        })                };                            </script>        </div>\n",
              "</body>\n",
              "</html>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "2024-10-29 12:10:06.276 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.278 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.279 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.280 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.282 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.285 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.286 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.287 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.288 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.289 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.290 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.291 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.291 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.292 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.293 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.294 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.295 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.296 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.297 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.298 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.299 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.300 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:06.305 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_1730203807245728\"  style=\"position: relative; width: 1000px; height: 800px;\">\n        <p id=\"3dmolwarning_1730203807245728\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.0/3Dmol-min.js');\n}\n\nvar viewer_1730203807245728 = null;\nvar warn = document.getElementById(\"3dmolwarning_1730203807245728\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_1730203807245728 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1730203807245728\"),{backgroundColor:\"white\"});\nviewer_1730203807245728.zoomTo();\n\tviewer_1730203807245728.addModel(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   ASN A   1      -7.305   7.869 -18.652  1.00  0.36           N  \\nATOM      2  CA  ASN A   1      -8.435   7.903 -17.729  1.00  0.40           C  \\nATOM      3  C   ASN A   1      -9.107   6.538 -17.617  1.00  0.37           C  \\nATOM      4  CB  ASN A   1      -9.451   8.961 -18.164  1.00  0.35           C  \\nATOM      5  O   ASN A   1      -9.757   6.081 -18.559  1.00  0.37           O  \\nATOM      6  CG  ASN A   1      -9.843   9.893 -17.034  1.00  0.36           C  \\nATOM      7  ND2 ASN A   1     -10.634  10.911 -17.352  1.00  0.29           N  \\nATOM      8  OD1 ASN A   1      -9.438   9.699 -15.885  1.00  0.35           O  \\nATOM      9  N   SER A   2      -8.282   5.362 -17.381  1.00  0.46           N  \\nATOM     10  CA  SER A   2      -8.849   4.046 -17.655  1.00  0.46           C  \\nATOM     11  C   SER A   2      -8.929   3.204 -16.385  1.00  0.46           C  \\nATOM     12  CB  SER A   2      -8.020   3.313 -18.710  1.00  0.42           C  \\nATOM     13  O   SER A   2      -7.974   3.150 -15.608  1.00  0.44           O  \\nATOM     14  OG  SER A   2      -6.930   2.633 -18.111  1.00  0.39           O  \\nATOM     15  N   ASP A   3      -9.818   3.550 -15.357  1.00  0.49           N  \\nATOM     16  CA  ASP A   3     -10.231   2.490 -14.442  1.00  0.49           C  \\nATOM     17  C   ASP A   3      -9.516   1.179 -14.762  1.00  0.49           C  \\nATOM     18  CB  ASP A   3     -11.747   2.290 -14.501  1.00  0.46           C  \\nATOM     19  O   ASP A   3      -9.859   0.499 -15.731  1.00  0.48           O  \\nATOM     20  CG  ASP A   3     -12.520   3.418 -13.841  1.00  0.45           C  \\nATOM     21  OD1 ASP A   3     -11.944   4.141 -12.999  1.00  0.46           O  \\nATOM     22  OD2 ASP A   3     -13.717   3.584 -14.163  1.00  0.45           O  \\nATOM     23  N   SER A   4      -8.165   1.215 -15.150  1.00  0.53           N  \\nATOM     24  CA  SER A   4      -7.106   0.214 -15.074  1.00  0.53           C  \\nATOM     25  C   SER A   4      -7.394  -0.814 -13.985  1.00  0.54           C  \\nATOM     26  CB  SER A   4      -5.755   0.881 -14.812  1.00  0.48           C  \\nATOM     27  O   SER A   4      -7.257  -0.521 -12.795  1.00  0.51           O  \\nATOM     28  OG  SER A   4      -5.859   1.838 -13.772  1.00  0.43           O  \\nATOM     29  N   GLU A   5      -8.786  -1.207 -13.868  1.00  0.59           N  \\nATOM     30  CA  GLU A   5      -9.425  -2.219 -13.032  1.00  0.58           C  \\nATOM     31  C   GLU A   5      -8.531  -3.445 -12.871  1.00  0.59           C  \\nATOM     32  CB  GLU A   5     -10.777  -2.628 -13.622  1.00  0.53           C  \\nATOM     33  O   GLU A   5      -7.971  -3.945 -13.849  1.00  0.56           O  \\nATOM     34  CG  GLU A   5     -11.870  -1.586 -13.435  1.00  0.50           C  \\nATOM     35  CD  GLU A   5     -13.225  -2.037 -13.955  1.00  0.51           C  \\nATOM     36  OE1 GLU A   5     -13.331  -3.176 -14.465  1.00  0.50           O  \\nATOM     37  OE2 GLU A   5     -14.189  -1.246 -13.853  1.00  0.45           O  \\nATOM     38  N   CYS A   6      -7.805  -3.434 -11.874  1.00  0.68           N  \\nATOM     39  CA  CYS A   6      -7.276  -4.690 -11.355  1.00  0.68           C  \\nATOM     40  C   CYS A   6      -8.207  -5.851 -11.684  1.00  0.68           C  \\nATOM     41  CB  CYS A   6      -7.071  -4.601  -9.843  1.00  0.66           C  \\nATOM     42  O   CYS A   6      -9.425  -5.678 -11.745  1.00  0.66           O  \\nATOM     43  SG  CYS A   6      -5.621  -3.640  -9.360  1.00  0.66           S  \\nATOM     44  N   PRO A   7      -7.733  -6.781 -12.513  1.00  0.63           N  \\nATOM     45  CA  PRO A   7      -8.638  -7.901 -12.783  1.00  0.62           C  \\nATOM     46  C   PRO A   7      -9.494  -8.273 -11.574  1.00  0.64           C  \\nATOM     47  CB  PRO A   7      -7.686  -9.042 -13.150  1.00  0.59           C  \\nATOM     48  O   PRO A   7      -9.092  -8.035 -10.432  1.00  0.62           O  \\nATOM     49  CG  PRO A   7      -6.360  -8.383 -13.354  1.00  0.57           C  \\nATOM     50  CD  PRO A   7      -6.375  -7.055 -12.653  1.00  0.56           C  \\nATOM     51  N   LEU A   8     -10.850  -8.459 -11.712  1.00  0.60           N  \\nATOM     52  CA  LEU A   8     -11.733  -8.966 -10.668  1.00  0.58           C  \\nATOM     53  C   LEU A   8     -11.023 -10.013  -9.816  1.00  0.60           C  \\nATOM     54  CB  LEU A   8     -13.001  -9.565 -11.282  1.00  0.54           C  \\nATOM     55  O   LEU A   8     -11.283 -10.124  -8.616  1.00  0.58           O  \\nATOM     56  CG  LEU A   8     -14.000  -8.572 -11.878  1.00  0.51           C  \\nATOM     57  CD1 LEU A   8     -14.999  -9.297 -12.773  1.00  0.44           C  \\nATOM     58  CD2 LEU A   8     -14.721  -7.810 -10.771  1.00  0.46           C  \\nATOM     59  N   SER A   9     -10.284 -10.902 -10.463  1.00  0.60           N  \\nATOM     60  CA  SER A   9      -9.555 -11.936  -9.736  1.00  0.58           C  \\nATOM     61  C   SER A   9      -8.618 -11.325  -8.698  1.00  0.59           C  \\nATOM     62  CB  SER A   9      -8.756 -12.810 -10.703  1.00  0.54           C  \\nATOM     63  O   SER A   9      -8.105 -12.029  -7.827  1.00  0.56           O  \\nATOM     64  OG  SER A   9      -7.746 -12.054 -11.348  1.00  0.50           O  \\nATOM     65  N   HIS A  10      -8.393 -10.120  -8.767  1.00  0.63           N  \\nATOM     66  CA  HIS A  10      -7.507  -9.535  -7.767  1.00  0.61           C  \\nATOM     67  C   HIS A  10      -8.301  -8.932  -6.613  1.00  0.63           C  \\nATOM     68  CB  HIS A  10      -6.612  -8.469  -8.402  1.00  0.58           C  \\nATOM     69  O   HIS A  10      -7.826  -8.016  -5.938  1.00  0.61           O  \\nATOM     70  CG  HIS A  10      -5.432  -9.032  -9.128  1.00  0.55           C  \\nATOM     71  CD2 HIS A  10      -4.127  -9.110  -8.777  1.00  0.53           C  \\nATOM     72  ND1 HIS A  10      -5.530  -9.600 -10.379  1.00  0.52           N  \\nATOM     73  CE1 HIS A  10      -4.332 -10.006 -10.767  1.00  0.49           C  \\nATOM     74  NE2 HIS A  10      -3.463  -9.719  -9.813  1.00  0.46           N  \\nATOM     75  N   ASP A  11      -9.703  -9.372  -6.500  1.00  0.67           N  \\nATOM     76  CA  ASP A  11     -10.363  -9.201  -5.210  1.00  0.66           C  \\nATOM     77  C   ASP A  11      -9.570  -9.876  -4.093  1.00  0.67           C  \\nATOM     78  CB  ASP A  11     -11.787  -9.760  -5.256  1.00  0.62           C  \\nATOM     79  O   ASP A  11      -9.244 -11.062  -4.186  1.00  0.63           O  \\nATOM     80  CG  ASP A  11     -12.834  -8.753  -4.812  1.00  0.58           C  \\nATOM     81  OD1 ASP A  11     -12.473  -7.603  -4.481  1.00  0.57           O  \\nATOM     82  OD2 ASP A  11     -14.030  -9.114  -4.790  1.00  0.59           O  \\nATOM     83  N   GLY A  12      -8.604  -9.100  -3.629  1.00  0.74           N  \\nATOM     84  CA  GLY A  12      -7.784  -9.534  -2.509  1.00  0.74           C  \\nATOM     85  C   GLY A  12      -6.301  -9.302  -2.730  1.00  0.75           C  \\nATOM     86  O   GLY A  12      -5.474  -9.717  -1.916  1.00  0.73           O  \\nATOM     87  N   TYR A  13      -5.995  -8.843  -3.903  1.00  0.82           N  \\nATOM     88  CA  TYR A  13      -4.571  -8.632  -4.139  1.00  0.82           C  \\nATOM     89  C   TYR A  13      -3.982  -7.683  -3.102  1.00  0.82           C  \\nATOM     90  CB  TYR A  13      -4.335  -8.077  -5.547  1.00  0.80           C  \\nATOM     91  O   TYR A  13      -2.964  -7.991  -2.476  1.00  0.81           O  \\nATOM     92  CG  TYR A  13      -2.877  -7.896  -5.892  1.00  0.76           C  \\nATOM     93  CD1 TYR A  13      -2.038  -8.995  -6.055  1.00  0.72           C  \\nATOM     94  CD2 TYR A  13      -2.335  -6.625  -6.054  1.00  0.74           C  \\nATOM     95  CE1 TYR A  13      -0.693  -8.833  -6.371  1.00  0.71           C  \\nATOM     96  CE2 TYR A  13      -0.992  -6.451  -6.369  1.00  0.71           C  \\nATOM     97  OH  TYR A  13       1.151  -7.393  -6.839  1.00  0.64           O  \\nATOM     98  CZ  TYR A  13      -0.180  -7.559  -6.526  1.00  0.68           C  \\nATOM     99  N   CYS A  14      -4.652  -6.448  -2.907  1.00  0.85           N  \\nATOM    100  CA  CYS A  14      -4.239  -5.521  -1.858  1.00  0.86           C  \\nATOM    101  C   CYS A  14      -4.892  -5.877  -0.528  1.00  0.86           C  \\nATOM    102  CB  CYS A  14      -4.593  -4.085  -2.242  1.00  0.85           C  \\nATOM    103  O   CYS A  14      -6.107  -6.068  -0.459  1.00  0.85           O  \\nATOM    104  SG  CYS A  14      -3.871  -3.551  -3.810  1.00  0.84           S  \\nATOM    105  N   LEU A  15      -4.073  -5.961   0.460  1.00  0.87           N  \\nATOM    106  CA  LEU A  15      -4.552  -6.327   1.789  1.00  0.88           C  \\nATOM    107  C   LEU A  15      -4.987  -5.090   2.569  1.00  0.88           C  \\nATOM    108  CB  LEU A  15      -3.466  -7.077   2.563  1.00  0.86           C  \\nATOM    109  O   LEU A  15      -4.739  -3.961   2.139  1.00  0.87           O  \\nATOM    110  CG  LEU A  15      -2.936  -8.358   1.919  1.00  0.83           C  \\nATOM    111  CD1 LEU A  15      -1.789  -8.932   2.744  1.00  0.81           C  \\nATOM    112  CD2 LEU A  15      -4.055  -9.382   1.765  1.00  0.81           C  \\nATOM    113  N   HIS A  16      -5.851  -5.334   3.662  1.00  0.88           N  \\nATOM    114  CA  HIS A  16      -6.200  -4.338   4.669  1.00  0.89           C  \\nATOM    115  C   HIS A  16      -6.826  -3.103   4.029  1.00  0.88           C  \\nATOM    116  CB  HIS A  16      -4.965  -3.940   5.479  1.00  0.87           C  \\nATOM    117  O   HIS A  16      -6.429  -1.975   4.328  1.00  0.86           O  \\nATOM    118  CG  HIS A  16      -4.284  -5.096   6.141  1.00  0.84           C  \\nATOM    119  CD2 HIS A  16      -3.095  -5.691   5.886  1.00  0.82           C  \\nATOM    120  ND1 HIS A  16      -4.837  -5.777   7.203  1.00  0.78           N  \\nATOM    121  CE1 HIS A  16      -4.015  -6.744   7.575  1.00  0.83           C  \\nATOM    122  NE2 HIS A  16      -2.950  -6.713   6.792  1.00  0.83           N  \\nATOM    123  N   ASP A  17      -7.803  -3.279   3.146  1.00  0.86           N  \\nATOM    124  CA  ASP A  17      -8.627  -2.264   2.498  1.00  0.86           C  \\nATOM    125  C   ASP A  17      -7.803  -1.432   1.518  1.00  0.86           C  \\nATOM    126  CB  ASP A  17      -9.279  -1.355   3.542  1.00  0.85           C  \\nATOM    127  O   ASP A  17      -8.167  -0.297   1.201  1.00  0.84           O  \\nATOM    128  CG  ASP A  17     -10.312  -2.073   4.392  1.00  0.82           C  \\nATOM    129  OD1 ASP A  17     -10.918  -3.056   3.914  1.00  0.80           O  \\nATOM    130  OD2 ASP A  17     -10.525  -1.650   5.549  1.00  0.81           O  \\nATOM    131  N   GLY A  18      -6.648  -1.956   1.127  1.00  0.87           N  \\nATOM    132  CA  GLY A  18      -5.904  -1.344   0.037  1.00  0.87           C  \\nATOM    133  C   GLY A  18      -6.642  -1.388  -1.288  1.00  0.86           C  \\nATOM    134  O   GLY A  18      -7.519  -2.230  -1.488  1.00  0.84           O  \\nATOM    135  N   VAL A  19      -6.407  -0.364  -2.154  1.00  0.85           N  \\nATOM    136  CA  VAL A  19      -7.053  -0.262  -3.458  1.00  0.85           C  \\nATOM    137  C   VAL A  19      -6.062  -0.638  -4.557  1.00  0.85           C  \\nATOM    138  CB  VAL A  19      -7.610   1.158  -3.706  1.00  0.84           C  \\nATOM    139  O   VAL A  19      -4.963  -0.083  -4.627  1.00  0.83           O  \\nATOM    140  CG1 VAL A  19      -8.187   1.273  -5.116  1.00  0.80           C  \\nATOM    141  CG2 VAL A  19      -8.669   1.506  -2.661  1.00  0.79           C  \\nATOM    142  N   CYS A  20      -6.373  -1.610  -5.323  1.00  0.84           N  \\nATOM    143  CA  CYS A  20      -5.540  -2.077  -6.427  1.00  0.84           C  \\nATOM    144  C   CYS A  20      -5.621  -1.122  -7.612  1.00  0.84           C  \\nATOM    145  CB  CYS A  20      -5.963  -3.480  -6.860  1.00  0.83           C  \\nATOM    146  O   CYS A  20      -6.701  -0.632  -7.947  1.00  0.82           O  \\nATOM    147  SG  CYS A  20      -4.884  -4.210  -8.112  1.00  0.82           S  \\nATOM    148  N   MET A  21      -4.563  -0.934  -8.213  1.00  0.83           N  \\nATOM    149  CA  MET A  21      -4.470  -0.092  -9.401  1.00  0.83           C  \\nATOM    150  C   MET A  21      -3.542  -0.715 -10.439  1.00  0.83           C  \\nATOM    151  CB  MET A  21      -3.976   1.308  -9.032  1.00  0.82           C  \\nATOM    152  O   MET A  21      -2.582  -1.401 -10.086  1.00  0.82           O  \\nATOM    153  CG  MET A  21      -2.580   1.326  -8.428  1.00  0.77           C  \\nATOM    154  SD  MET A  21      -2.091   2.993  -7.839  1.00  0.76           S  \\nATOM    155  CE  MET A  21      -1.944   3.863  -9.425  1.00  0.65           C  \\nATOM    156  N   TYR A  22      -3.868  -0.655 -11.665  1.00  0.82           N  \\nATOM    157  CA  TYR A  22      -3.016  -1.101 -12.762  1.00  0.82           C  \\nATOM    158  C   TYR A  22      -2.256   0.069 -13.374  1.00  0.82           C  \\nATOM    159  CB  TYR A  22      -3.850  -1.802 -13.839  1.00  0.80           C  \\nATOM    160  O   TYR A  22      -2.855   1.083 -13.740  1.00  0.81           O  \\nATOM    161  CG  TYR A  22      -3.035  -2.308 -15.004  1.00  0.77           C  \\nATOM    162  CD1 TYR A  22      -2.158  -3.380 -14.853  1.00  0.75           C  \\nATOM    163  CD2 TYR A  22      -3.138  -1.716 -16.258  1.00  0.76           C  \\nATOM    164  CE1 TYR A  22      -1.403  -3.849 -15.923  1.00  0.74           C  \\nATOM    165  CE2 TYR A  22      -2.389  -2.176 -17.335  1.00  0.74           C  \\nATOM    166  OH  TYR A  22      -0.781  -3.701 -18.221  1.00  0.68           O  \\nATOM    167  CZ  TYR A  22      -1.525  -3.242 -17.158  1.00  0.70           C  \\nATOM    168  N   ILE A  23      -0.909   0.026 -13.379  1.00  0.81           N  \\nATOM    169  CA  ILE A  23      -0.030   1.046 -13.939  1.00  0.81           C  \\nATOM    170  C   ILE A  23       0.412   0.634 -15.341  1.00  0.82           C  \\nATOM    171  CB  ILE A  23       1.202   1.287 -13.037  1.00  0.80           C  \\nATOM    172  O   ILE A  23       1.288  -0.221 -15.496  1.00  0.81           O  \\nATOM    173  CG1 ILE A  23       0.762   1.580 -11.599  1.00  0.80           C  \\nATOM    174  CG2 ILE A  23       2.062   2.427 -13.591  1.00  0.80           C  \\nATOM    175  CD1 ILE A  23       1.914   1.695 -10.610  1.00  0.78           C  \\nATOM    176  N   GLU A  24      -0.160   1.169 -16.357  1.00  0.82           N  \\nATOM    177  CA  GLU A  24       0.044   0.808 -17.756  1.00  0.81           C  \\nATOM    178  C   GLU A  24       1.500   0.999 -18.169  1.00  0.82           C  \\nATOM    179  CB  GLU A  24      -0.873   1.632 -18.664  1.00  0.80           C  \\nATOM    180  O   GLU A  24       2.067   0.158 -18.871  1.00  0.81           O  \\nATOM    181  CG  GLU A  24      -0.892   1.161 -20.110  1.00  0.75           C  \\nATOM    182  CD  GLU A  24      -1.880   1.925 -20.977  1.00  0.72           C  \\nATOM    183  OE1 GLU A  24      -2.596   2.806 -20.450  1.00  0.67           O  \\nATOM    184  OE2 GLU A  24      -1.940   1.639 -22.194  1.00  0.64           O  \\nATOM    185  N   ALA A  25       2.013   2.100 -17.746  1.00  0.81           N  \\nATOM    186  CA  ALA A  25       3.369   2.431 -18.177  1.00  0.82           C  \\nATOM    187  C   ALA A  25       4.360   1.353 -17.748  1.00  0.82           C  \\nATOM    188  CB  ALA A  25       3.789   3.788 -17.616  1.00  0.80           C  \\nATOM    189  O   ALA A  25       5.363   1.117 -18.426  1.00  0.81           O  \\nATOM    190  N   LEU A  26       4.083   0.698 -16.649  1.00  0.82           N  \\nATOM    191  CA  LEU A  26       4.955  -0.329 -16.091  1.00  0.82           C  \\nATOM    192  C   LEU A  26       4.397  -1.722 -16.362  1.00  0.82           C  \\nATOM    193  CB  LEU A  26       5.133  -0.122 -14.585  1.00  0.80           C  \\nATOM    194  O   LEU A  26       5.097  -2.722 -16.181  1.00  0.80           O  \\nATOM    195  CG  LEU A  26       5.842   1.164 -14.156  1.00  0.76           C  \\nATOM    196  CD1 LEU A  26       5.848   1.284 -12.636  1.00  0.72           C  \\nATOM    197  CD2 LEU A  26       7.263   1.200 -14.707  1.00  0.74           C  \\nATOM    198  N   ASP A  27       3.124  -1.697 -16.831  1.00  0.83           N  \\nATOM    199  CA  ASP A  27       2.419  -2.970 -16.949  1.00  0.82           C  \\nATOM    200  C   ASP A  27       2.454  -3.741 -15.631  1.00  0.82           C  \\nATOM    201  CB  ASP A  27       3.025  -3.818 -18.070  1.00  0.81           C  \\nATOM    202  O   ASP A  27       2.861  -4.904 -15.596  1.00  0.80           O  \\nATOM    203  CG  ASP A  27       2.167  -5.016 -18.437  1.00  0.79           C  \\nATOM    204  OD1 ASP A  27       0.951  -5.006 -18.151  1.00  0.78           O  \\nATOM    205  OD2 ASP A  27       2.714  -5.980 -19.015  1.00  0.77           O  \\nATOM    206  N   LYS A  28       2.180  -3.218 -14.579  1.00  0.83           N  \\nATOM    207  CA  LYS A  28       2.192  -3.810 -13.245  1.00  0.83           C  \\nATOM    208  C   LYS A  28       0.988  -3.351 -12.428  1.00  0.83           C  \\nATOM    209  CB  LYS A  28       3.487  -3.455 -12.512  1.00  0.81           C  \\nATOM    210  O   LYS A  28       0.450  -2.266 -12.661  1.00  0.82           O  \\nATOM    211  CG  LYS A  28       4.729  -4.114 -13.096  1.00  0.78           C  \\nATOM    212  CD  LYS A  28       5.960  -3.841 -12.243  1.00  0.76           C  \\nATOM    213  CE  LYS A  28       7.188  -4.561 -12.784  1.00  0.68           C  \\nATOM    214  NZ  LYS A  28       8.384  -4.345 -11.916  1.00  0.65           N  \\nATOM    215  N   TYR A  29       0.624  -4.199 -11.528  1.00  0.83           N  \\nATOM    216  CA  TYR A  29      -0.385  -3.846 -10.536  1.00  0.83           C  \\nATOM    217  C   TYR A  29       0.263  -3.301  -9.269  1.00  0.83           C  \\nATOM    218  CB  TYR A  29      -1.254  -5.061 -10.197  1.00  0.81           C  \\nATOM    219  O   TYR A  29       1.357  -3.727  -8.892  1.00  0.82           O  \\nATOM    220  CG  TYR A  29      -2.008  -5.617 -11.381  1.00  0.79           C  \\nATOM    221  CD1 TYR A  29      -3.185  -5.019 -11.825  1.00  0.77           C  \\nATOM    222  CD2 TYR A  29      -1.546  -6.741 -12.056  1.00  0.78           C  \\nATOM    223  CE1 TYR A  29      -3.885  -5.529 -12.914  1.00  0.75           C  \\nATOM    224  CE2 TYR A  29      -2.237  -7.260 -13.146  1.00  0.76           C  \\nATOM    225  OH  TYR A  29      -4.091  -7.157 -14.646  1.00  0.70           O  \\nATOM    226  CZ  TYR A  29      -3.403  -6.648 -13.567  1.00  0.71           C  \\nATOM    227  N   ALA A  30      -0.345  -2.309  -8.674  1.00  0.85           N  \\nATOM    228  CA  ALA A  30       0.128  -1.721  -7.423  1.00  0.85           C  \\nATOM    229  C   ALA A  30      -1.030  -1.479  -6.459  1.00  0.85           C  \\nATOM    230  CB  ALA A  30       0.871  -0.416  -7.697  1.00  0.84           C  \\nATOM    231  O   ALA A  30      -2.192  -1.445  -6.871  1.00  0.85           O  \\nATOM    232  N   CYS A  31      -0.771  -1.457  -5.228  1.00  0.87           N  \\nATOM    233  CA  CYS A  31      -1.761  -1.201  -4.188  1.00  0.87           C  \\nATOM    234  C   CYS A  31      -1.593   0.198  -3.608  1.00  0.87           C  \\nATOM    235  CB  CYS A  31      -1.650  -2.242  -3.075  1.00  0.86           C  \\nATOM    236  O   CYS A  31      -0.469   0.653  -3.390  1.00  0.86           O  \\nATOM    237  SG  CYS A  31      -2.063  -3.920  -3.602  1.00  0.84           S  \\nATOM    238  N   ASN A  32      -2.667   0.955  -3.557  1.00  0.87           N  \\nATOM    239  CA  ASN A  32      -2.741   2.120  -2.682  1.00  0.87           C  \\nATOM    240  C   ASN A  32      -3.148   1.732  -1.264  1.00  0.88           C  \\nATOM    241  CB  ASN A  32      -3.715   3.156  -3.249  1.00  0.86           C  \\nATOM    242  O   ASN A  32      -4.287   1.321  -1.030  1.00  0.87           O  \\nATOM    243  CG  ASN A  32      -3.676   4.469  -2.493  1.00  0.79           C  \\nATOM    244  ND2 ASN A  32      -4.593   5.371  -2.823  1.00  0.75           N  \\nATOM    245  OD1 ASN A  32      -2.828   4.672  -1.621  1.00  0.76           O  \\nATOM    246  N   CYS A  33      -2.283   1.827  -0.373  1.00  0.90           N  \\nATOM    247  CA  CYS A  33      -2.528   1.356   0.986  1.00  0.91           C  \\nATOM    248  C   CYS A  33      -3.249   2.417   1.809  1.00  0.91           C  \\nATOM    249  CB  CYS A  33      -1.214   0.976   1.666  1.00  0.89           C  \\nATOM    250  O   CYS A  33      -3.026   3.614   1.619  1.00  0.89           O  \\nATOM    251  SG  CYS A  33      -0.271  -0.282   0.777  1.00  0.89           S  \\nATOM    252  N   VAL A  34      -4.150   2.077   2.645  1.00  0.90           N  \\nATOM    253  CA  VAL A  34      -4.735   2.997   3.614  1.00  0.90           C  \\nATOM    254  C   VAL A  34      -3.683   3.399   4.646  1.00  0.90           C  \\nATOM    255  CB  VAL A  34      -5.962   2.377   4.317  1.00  0.89           C  \\nATOM    256  O   VAL A  34      -2.683   2.700   4.826  1.00  0.90           O  \\nATOM    257  CG1 VAL A  34      -7.062   2.060   3.305  1.00  0.79           C  \\nATOM    258  CG2 VAL A  34      -5.558   1.118   5.083  1.00  0.79           C  \\nATOM    259  N   VAL A  35      -3.937   4.536   5.231  1.00  0.91           N  \\nATOM    260  CA  VAL A  35      -3.001   5.051   6.226  1.00  0.91           C  \\nATOM    261  C   VAL A  35      -2.778   4.004   7.315  1.00  0.91           C  \\nATOM    262  CB  VAL A  35      -3.506   6.371   6.850  1.00  0.89           C  \\nATOM    263  O   VAL A  35      -3.730   3.382   7.791  1.00  0.89           O  \\nATOM    264  CG1 VAL A  35      -2.537   6.867   7.922  1.00  0.81           C  \\nATOM    265  CG2 VAL A  35      -3.701   7.432   5.769  1.00  0.81           C  \\nATOM    266  N   GLY A  36      -1.488   3.610   7.611  1.00  0.91           N  \\nATOM    267  CA  GLY A  36      -1.118   2.655   8.644  1.00  0.91           C  \\nATOM    268  C   GLY A  36      -0.564   1.358   8.087  1.00  0.91           C  \\nATOM    269  O   GLY A  36      -0.048   0.525   8.835  1.00  0.90           O  \\nATOM    270  N   TYR A  37      -0.654   1.183   6.843  1.00  0.92           N  \\nATOM    271  CA  TYR A  37      -0.150  -0.009   6.171  1.00  0.93           C  \\nATOM    272  C   TYR A  37       0.792   0.364   5.033  1.00  0.92           C  \\nATOM    273  CB  TYR A  37      -1.310  -0.854   5.634  1.00  0.92           C  \\nATOM    274  O   TYR A  37       0.668   1.440   4.443  1.00  0.91           O  \\nATOM    275  CG  TYR A  37      -2.123  -1.524   6.714  1.00  0.90           C  \\nATOM    276  CD1 TYR A  37      -1.829  -2.820   7.132  1.00  0.88           C  \\nATOM    277  CD2 TYR A  37      -3.188  -0.864   7.318  1.00  0.89           C  \\nATOM    278  CE1 TYR A  37      -2.577  -3.443   8.126  1.00  0.89           C  \\nATOM    279  CE2 TYR A  37      -3.944  -1.477   8.312  1.00  0.89           C  \\nATOM    280  OH  TYR A  37      -4.375  -3.376   9.693  1.00  0.82           O  \\nATOM    281  CZ  TYR A  37      -3.631  -2.764   8.709  1.00  0.88           C  \\nATOM    282  N   ILE A  38       1.765  -0.605   4.902  1.00  0.92           N  \\nATOM    283  CA  ILE A  38       2.757  -0.433   3.846  1.00  0.92           C  \\nATOM    284  C   ILE A  38       3.060  -1.782   3.198  1.00  0.93           C  \\nATOM    285  CB  ILE A  38       4.055   0.205   4.390  1.00  0.92           C  \\nATOM    286  O   ILE A  38       2.548  -2.816   3.634  1.00  0.92           O  \\nATOM    287  CG1 ILE A  38       4.678  -0.689   5.468  1.00  0.89           C  \\nATOM    288  CG2 ILE A  38       3.780   1.609   4.935  1.00  0.89           C  \\nATOM    289  CD1 ILE A  38       6.082  -0.270   5.884  1.00  0.86           C  \\nATOM    290  N   GLY A  39       3.964  -1.780   2.045  1.00  0.91           N  \\nATOM    291  CA  GLY A  39       4.350  -2.984   1.327  1.00  0.91           C  \\nATOM    292  C   GLY A  39       3.720  -3.087  -0.049  1.00  0.90           C  \\nATOM    293  O   GLY A  39       2.854  -2.285  -0.402  1.00  0.89           O  \\nATOM    294  N   GLU A  40       4.067  -4.027  -0.773  1.00  0.89           N  \\nATOM    295  CA  GLU A  40       3.601  -4.210  -2.144  1.00  0.89           C  \\nATOM    296  C   GLU A  40       2.103  -4.501  -2.184  1.00  0.89           C  \\nATOM    297  CB  GLU A  40       4.374  -5.339  -2.829  1.00  0.87           C  \\nATOM    298  O   GLU A  40       1.400  -4.044  -3.087  1.00  0.88           O  \\nATOM    299  CG  GLU A  40       5.829  -5.000  -3.120  1.00  0.78           C  \\nATOM    300  CD  GLU A  40       6.556  -6.089  -3.894  1.00  0.75           C  \\nATOM    301  OE1 GLU A  40       5.976  -7.179  -4.097  1.00  0.71           O  \\nATOM    302  OE2 GLU A  40       7.715  -5.849  -4.299  1.00  0.67           O  \\nATOM    303  N   ARG A  41       1.609  -5.201  -1.159  1.00  0.89           N  \\nATOM    304  CA  ARG A  41       0.194  -5.550  -1.094  1.00  0.89           C  \\nATOM    305  C   ARG A  41      -0.463  -4.948   0.144  1.00  0.90           C  \\nATOM    306  CB  ARG A  41       0.014  -7.069  -1.097  1.00  0.88           C  \\nATOM    307  O   ARG A  41      -1.486  -5.448   0.615  1.00  0.89           O  \\nATOM    308  CG  ARG A  41       0.499  -7.745  -2.370  1.00  0.85           C  \\nATOM    309  CD  ARG A  41       0.225  -9.242  -2.354  1.00  0.83           C  \\nATOM    310  NE  ARG A  41      -1.204  -9.528  -2.269  1.00  0.78           N  \\nATOM    311  NH1 ARG A  41      -0.937 -11.717  -1.584  1.00  0.64           N  \\nATOM    312  NH2 ARG A  41      -3.039 -10.848  -1.863  1.00  0.63           N  \\nATOM    313  CZ  ARG A  41      -1.723 -10.697  -1.906  1.00  0.80           C  \\nATOM    314  N   CYS A  42       0.277  -3.945   0.723  1.00  0.90           N  \\nATOM    315  CA  CYS A  42      -0.213  -3.337   1.955  1.00  0.91           C  \\nATOM    316  C   CYS A  42      -0.206  -4.342   3.100  1.00  0.91           C  \\nATOM    317  CB  CYS A  42      -1.626  -2.788   1.756  1.00  0.90           C  \\nATOM    318  O   CYS A  42      -1.128  -4.366   3.917  1.00  0.90           O  \\nATOM    319  SG  CYS A  42      -1.791  -1.706   0.318  1.00  0.88           S  \\nATOM    320  N   GLN A  43       0.844  -5.243   3.151  1.00  0.92           N  \\nATOM    321  CA  GLN A  43       0.821  -6.399   4.040  1.00  0.92           C  \\nATOM    322  C   GLN A  43       1.448  -6.067   5.391  1.00  0.92           C  \\nATOM    323  CB  GLN A  43       1.549  -7.583   3.400  1.00  0.90           C  \\nATOM    324  O   GLN A  43       1.232  -6.780   6.373  1.00  0.91           O  \\nATOM    325  CG  GLN A  43       3.038  -7.347   3.187  1.00  0.86           C  \\nATOM    326  CD  GLN A  43       3.350  -6.754   1.826  1.00  0.84           C  \\nATOM    327  NE2 GLN A  43       4.459  -7.182   1.233  1.00  0.79           N  \\nATOM    328  OE1 GLN A  43       2.600  -5.918   1.312  1.00  0.82           O  \\nATOM    329  N   TYR A  44       2.093  -5.035   5.476  1.00  0.93           N  \\nATOM    330  CA  TYR A  44       2.773  -4.710   6.724  1.00  0.93           C  \\nATOM    331  C   TYR A  44       2.121  -3.513   7.406  1.00  0.92           C  \\nATOM    332  CB  TYR A  44       4.256  -4.420   6.469  1.00  0.92           C  \\nATOM    333  O   TYR A  44       1.588  -2.625   6.737  1.00  0.91           O  \\nATOM    334  CG  TYR A  44       4.989  -5.555   5.796  1.00  0.89           C  \\nATOM    335  CD1 TYR A  44       5.121  -6.793   6.422  1.00  0.86           C  \\nATOM    336  CD2 TYR A  44       5.551  -5.393   4.534  1.00  0.86           C  \\nATOM    337  CE1 TYR A  44       5.798  -7.841   5.807  1.00  0.87           C  \\nATOM    338  CE2 TYR A  44       6.229  -6.434   3.910  1.00  0.87           C  \\nATOM    339  OH  TYR A  44       7.017  -8.687   3.939  1.00  0.79           O  \\nATOM    340  CZ  TYR A  44       6.347  -7.653   4.552  1.00  0.86           C  \\nATOM    341  N   ARG A  45       2.158  -3.532   8.714  1.00  0.92           N  \\nATOM    342  CA  ARG A  45       1.771  -2.357   9.488  1.00  0.92           C  \\nATOM    343  C   ARG A  45       2.907  -1.340   9.545  1.00  0.92           C  \\nATOM    344  CB  ARG A  45       1.356  -2.758  10.905  1.00  0.91           C  \\nATOM    345  O   ARG A  45       4.073  -1.712   9.690  1.00  0.91           O  \\nATOM    346  CG  ARG A  45       0.050  -3.533  10.968  1.00  0.84           C  \\nATOM    347  CD  ARG A  45      -0.298  -3.937  12.394  1.00  0.82           C  \\nATOM    348  NE  ARG A  45      -1.576  -4.641  12.459  1.00  0.76           N  \\nATOM    349  NH1 ARG A  45      -1.550  -4.904  14.754  1.00  0.58           N  \\nATOM    350  NH2 ARG A  45      -3.306  -5.713  13.522  1.00  0.56           N  \\nATOM    351  CZ  ARG A  45      -2.141  -5.084  13.578  1.00  0.76           C  \\nATOM    352  N   ASP A  46       2.608  -0.114   9.094  1.00  0.92           N  \\nATOM    353  CA  ASP A  46       3.581   0.949   9.328  1.00  0.92           C  \\nATOM    354  C   ASP A  46       3.793   1.180  10.822  1.00  0.91           C  \\nATOM    355  CB  ASP A  46       3.130   2.246   8.653  1.00  0.91           C  \\nATOM    356  O   ASP A  46       2.943   1.771  11.491  1.00  0.90           O  \\nATOM    357  CG  ASP A  46       4.199   3.325   8.669  1.00  0.88           C  \\nATOM    358  OD1 ASP A  46       5.186   3.195   9.424  1.00  0.86           O  \\nATOM    359  OD2 ASP A  46       4.051   4.315   7.921  1.00  0.86           O  \\nATOM    360  N   LEU A  47       4.865   0.611  11.474  1.00  0.90           N  \\nATOM    361  CA  LEU A  47       5.079   0.563  12.916  1.00  0.90           C  \\nATOM    362  C   LEU A  47       5.090   1.967  13.510  1.00  0.89           C  \\nATOM    363  CB  LEU A  47       6.394  -0.151  13.240  1.00  0.88           C  \\nATOM    364  O   LEU A  47       4.657   2.168  14.648  1.00  0.88           O  \\nATOM    365  CG  LEU A  47       6.461  -1.639  12.893  1.00  0.83           C  \\nATOM    366  CD1 LEU A  47       7.838  -2.201  13.232  1.00  0.78           C  \\nATOM    367  CD2 LEU A  47       5.368  -2.409  13.626  1.00  0.78           C  \\nATOM    368  N   LYS A  48       5.663   2.934  12.806  1.00  0.88           N  \\nATOM    369  CA  LYS A  48       5.659   4.301  13.319  1.00  0.88           C  \\nATOM    370  C   LYS A  48       4.234   4.811  13.512  1.00  0.88           C  \\nATOM    371  CB  LYS A  48       6.425   5.230  12.375  1.00  0.85           C  \\nATOM    372  O   LYS A  48       3.924   5.431  14.532  1.00  0.86           O  \\nATOM    373  CG  LYS A  48       7.932   5.021  12.389  1.00  0.77           C  \\nATOM    374  CD  LYS A  48       8.645   6.036  11.506  1.00  0.74           C  \\nATOM    375  CE  LYS A  48      10.152   5.819  11.508  1.00  0.69           C  \\nATOM    376  NZ  LYS A  48      10.853   6.795  10.620  1.00  0.61           N  \\nATOM    377  N   TRP A  49       3.476   4.570  12.507  1.00  0.87           N  \\nATOM    378  CA  TRP A  49       2.080   4.989  12.567  1.00  0.88           C  \\nATOM    379  C   TRP A  49       1.353   4.297  13.716  1.00  0.86           C  \\nATOM    380  CB  TRP A  49       1.370   4.689  11.244  1.00  0.85           C  \\nATOM    381  O   TRP A  49       0.601   4.935  14.456  1.00  0.83           O  \\nATOM    382  CG  TRP A  49       0.008   5.305  11.129  1.00  0.74           C  \\nATOM    383  CD1 TRP A  49      -0.311   6.493  10.533  1.00  0.66           C  \\nATOM    384  CD2 TRP A  49      -1.218   4.765  11.630  1.00  0.69           C  \\nATOM    385  CE2 TRP A  49      -2.243   5.679  11.300  1.00  0.71           C  \\nATOM    386  CE3 TRP A  49      -1.551   3.594  12.324  1.00  0.74           C  \\nATOM    387  NE1 TRP A  49      -1.664   6.724  10.632  1.00  0.76           N  \\nATOM    388  CH2 TRP A  49      -3.877   4.305  12.322  1.00  0.71           C  \\nATOM    389  CZ2 TRP A  49      -3.579   5.458  11.643  1.00  0.74           C  \\nATOM    390  CZ3 TRP A  49      -2.881   3.376  12.664  1.00  0.72           C  \\nATOM    391  N   TRP A  50       1.416   3.062  13.899  1.00  0.89           N  \\nATOM    392  CA  TRP A  50       0.668   2.281  14.879  1.00  0.89           C  \\nATOM    393  C   TRP A  50       1.161   2.568  16.294  1.00  0.88           C  \\nATOM    394  CB  TRP A  50       0.785   0.785  14.577  1.00  0.88           C  \\nATOM    395  O   TRP A  50       0.392   2.485  17.254  1.00  0.85           O  \\nATOM    396  CG  TRP A  50      -0.088   0.320  13.451  1.00  0.85           C  \\nATOM    397  CD1 TRP A  50       0.222   0.307  12.120  1.00  0.81           C  \\nATOM    398  CD2 TRP A  50      -1.419  -0.194  13.559  1.00  0.82           C  \\nATOM    399  CE2 TRP A  50      -1.856  -0.501  12.251  1.00  0.85           C  \\nATOM    400  CE3 TRP A  50      -2.285  -0.427  14.635  1.00  0.79           C  \\nATOM    401  NE1 TRP A  50      -0.837  -0.186  11.392  1.00  0.79           N  \\nATOM    402  CH2 TRP A  50      -3.950  -1.246  13.062  1.00  0.78           C  \\nATOM    403  CZ2 TRP A  50      -3.123  -1.029  11.991  1.00  0.79           C  \\nATOM    404  CZ3 TRP A  50      -3.546  -0.952  14.375  1.00  0.79           C  \\nATOM    405  N   GLU A  51       2.410   2.889  16.502  1.00  0.89           N  \\nATOM    406  CA  GLU A  51       2.952   3.218  17.817  1.00  0.88           C  \\nATOM    407  C   GLU A  51       2.372   4.528  18.341  1.00  0.87           C  \\nATOM    408  CB  GLU A  51       4.480   3.303  17.763  1.00  0.86           C  \\nATOM    409  O   GLU A  51       2.248   4.717  19.553  1.00  0.85           O  \\nATOM    410  CG  GLU A  51       5.168   1.950  17.657  1.00  0.79           C  \\nATOM    411  CD  GLU A  51       6.682   2.053  17.561  1.00  0.75           C  \\nATOM    412  OE1 GLU A  51       7.211   3.186  17.506  1.00  0.72           O  \\nATOM    413  OE2 GLU A  51       7.344   0.992  17.541  1.00  0.70           O  \\nATOM    414  N   LEU A  52       2.083   5.414  17.516  1.00  0.87           N  \\nATOM    415  CA  LEU A  52       1.544   6.710  17.913  1.00  0.86           C  \\nATOM    416  C   LEU A  52       0.074   6.592  18.303  1.00  0.84           C  \\nATOM    417  CB  LEU A  52       1.703   7.727  16.780  1.00  0.84           C  \\nATOM    418  O   LEU A  52      -0.483   7.500  18.924  1.00  0.81           O  \\nATOM    419  CG  LEU A  52       3.135   8.132  16.428  1.00  0.79           C  \\nATOM    420  CD1 LEU A  52       3.155   8.942  15.136  1.00  0.73           C  \\nATOM    421  CD2 LEU A  52       3.764   8.922  17.571  1.00  0.74           C  \\nATOM    422  N   ARG A  53      -0.616   5.423  18.300  1.00  0.76           N  \\nATOM    423  CA  ARG A  53      -2.027   5.260  18.633  1.00  0.74           C  \\nATOM    424  C   ARG A  53      -2.212   4.248  19.758  1.00  0.73           C  \\nATOM    425  CB  ARG A  53      -2.823   4.825  17.400  1.00  0.71           C  \\nATOM    426  O   ARG A  53      -1.359   3.382  19.967  1.00  0.70           O  \\nATOM    427  CG  ARG A  53      -3.098   5.951  16.416  1.00  0.67           C  \\nATOM    428  CD  ARG A  53      -4.222   5.595  15.452  1.00  0.65           C  \\nATOM    429  NE  ARG A  53      -4.488   6.680  14.511  1.00  0.59           N  \\nATOM    430  NH1 ARG A  53      -6.382   5.710  13.610  1.00  0.52           N  \\nATOM    431  NH2 ARG A  53      -5.655   7.746  12.845  1.00  0.48           N  \\nATOM    432  CZ  ARG A  53      -5.508   6.709  13.658  1.00  0.63           C  \",\"pdb\");\n\tviewer_1730203807245728.setStyle({\"resi\": 1},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 2},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 3},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 4},{\"cartoon\": {\"color\": \"#3000cf\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 5},{\"cartoon\": {\"color\": \"#3000cf\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 6},{\"cartoon\": {\"color\": \"#3000cf\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 7},{\"cartoon\": {\"color\": \"#4000bf\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 8},{\"cartoon\": {\"color\": \"#5200ad\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 9},{\"cartoon\": {\"color\": \"#5100ae\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 10},{\"cartoon\": {\"color\": \"#8b0074\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 11},{\"cartoon\": {\"color\": \"#b4004b\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 12},{\"cartoon\": {\"color\": \"#b70048\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 13},{\"cartoon\": {\"color\": \"#b80047\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 14},{\"cartoon\": {\"color\": \"#7e0081\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 15},{\"cartoon\": {\"color\": \"#4500ba\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 16},{\"cartoon\": {\"color\": \"#4600b9\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 17},{\"cartoon\": {\"color\": \"#7d0082\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 18},{\"cartoon\": {\"color\": \"#b60049\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 19},{\"cartoon\": {\"color\": \"#b80047\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 20},{\"cartoon\": {\"color\": \"#80007f\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 21},{\"cartoon\": {\"color\": \"#5a00a5\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 22},{\"cartoon\": {\"color\": \"#2100de\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 23},{\"cartoon\": {\"color\": \"#2000df\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 24},{\"cartoon\": {\"color\": \"#2400db\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 25},{\"cartoon\": {\"color\": \"#1200ed\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 26},{\"cartoon\": {\"color\": \"#1200ed\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 27},{\"cartoon\": {\"color\": \"#1100ee\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 28},{\"cartoon\": {\"color\": \"#4500ba\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 29},{\"cartoon\": {\"color\": \"#7d0082\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 30},{\"cartoon\": {\"color\": \"#7e0081\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 31},{\"cartoon\": {\"color\": \"#b80047\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 32},{\"cartoon\": {\"color\": \"#80007f\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 33},{\"cartoon\": {\"color\": \"#5100ae\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 34},{\"cartoon\": {\"color\": \"#4f00b0\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 35},{\"cartoon\": {\"color\": \"#5000af\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 36},{\"cartoon\": {\"color\": \"#690096\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 37},{\"cartoon\": {\"color\": \"#60009f\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 38},{\"cartoon\": {\"color\": \"#9a0065\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 39},{\"cartoon\": {\"color\": \"#990066\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 40},{\"cartoon\": {\"color\": \"#94006b\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 41},{\"cartoon\": {\"color\": \"#95006a\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 42},{\"cartoon\": {\"color\": \"#94006b\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 43},{\"cartoon\": {\"color\": \"#890076\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 44},{\"cartoon\": {\"color\": \"#73008c\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 45},{\"cartoon\": {\"color\": \"#73008c\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 46},{\"cartoon\": {\"color\": \"#3a00c5\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 47},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 48},{\"cartoon\": {\"color\": \"#1100ee\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 49},{\"cartoon\": {\"color\": \"#1000ef\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 50},{\"cartoon\": {\"color\": \"#1000ef\"}});\n\tviewer_1730203807245728.setStyle({\"resi\": 51},{\"cartoon\": {\"color\": \"#1100ee\"}});\n\tviewer_1730203807245728.zoomTo();\nviewer_1730203807245728.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_1730203807245728\"  style=\"position: relative; width: 1000px; height: 800px;\">\n",
              "        <p id=\"3dmolwarning_1730203807245728\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.0/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_1730203807245728 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_1730203807245728\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_1730203807245728 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1730203807245728\"),{backgroundColor:\"white\"});\n",
              "viewer_1730203807245728.zoomTo();\n",
              "\tviewer_1730203807245728.addModel(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   ASN A   1      -7.305   7.869 -18.652  1.00  0.36           N  \\nATOM      2  CA  ASN A   1      -8.435   7.903 -17.729  1.00  0.40           C  \\nATOM      3  C   ASN A   1      -9.107   6.538 -17.617  1.00  0.37           C  \\nATOM      4  CB  ASN A   1      -9.451   8.961 -18.164  1.00  0.35           C  \\nATOM      5  O   ASN A   1      -9.757   6.081 -18.559  1.00  0.37           O  \\nATOM      6  CG  ASN A   1      -9.843   9.893 -17.034  1.00  0.36           C  \\nATOM      7  ND2 ASN A   1     -10.634  10.911 -17.352  1.00  0.29           N  \\nATOM      8  OD1 ASN A   1      -9.438   9.699 -15.885  1.00  0.35           O  \\nATOM      9  N   SER A   2      -8.282   5.362 -17.381  1.00  0.46           N  \\nATOM     10  CA  SER A   2      -8.849   4.046 -17.655  1.00  0.46           C  \\nATOM     11  C   SER A   2      -8.929   3.204 -16.385  1.00  0.46           C  \\nATOM     12  CB  SER A   2      -8.020   3.313 -18.710  1.00  0.42           C  \\nATOM     13  O   SER A   2      -7.974   3.150 -15.608  1.00  0.44           O  \\nATOM     14  OG  SER A   2      -6.930   2.633 -18.111  1.00  0.39           O  \\nATOM     15  N   ASP A   3      -9.818   3.550 -15.357  1.00  0.49           N  \\nATOM     16  CA  ASP A   3     -10.231   2.490 -14.442  1.00  0.49           C  \\nATOM     17  C   ASP A   3      -9.516   1.179 -14.762  1.00  0.49           C  \\nATOM     18  CB  ASP A   3     -11.747   2.290 -14.501  1.00  0.46           C  \\nATOM     19  O   ASP A   3      -9.859   0.499 -15.731  1.00  0.48           O  \\nATOM     20  CG  ASP A   3     -12.520   3.418 -13.841  1.00  0.45           C  \\nATOM     21  OD1 ASP A   3     -11.944   4.141 -12.999  1.00  0.46           O  \\nATOM     22  OD2 ASP A   3     -13.717   3.584 -14.163  1.00  0.45           O  \\nATOM     23  N   SER A   4      -8.165   1.215 -15.150  1.00  0.53           N  \\nATOM     24  CA  SER A   4      -7.106   0.214 -15.074  1.00  0.53           C  \\nATOM     25  C   SER A   4      -7.394  -0.814 -13.985  1.00  0.54           C  \\nATOM     26  CB  SER A   4      -5.755   0.881 -14.812  1.00  0.48           C  \\nATOM     27  O   SER A   4      -7.257  -0.521 -12.795  1.00  0.51           O  \\nATOM     28  OG  SER A   4      -5.859   1.838 -13.772  1.00  0.43           O  \\nATOM     29  N   GLU A   5      -8.786  -1.207 -13.868  1.00  0.59           N  \\nATOM     30  CA  GLU A   5      -9.425  -2.219 -13.032  1.00  0.58           C  \\nATOM     31  C   GLU A   5      -8.531  -3.445 -12.871  1.00  0.59           C  \\nATOM     32  CB  GLU A   5     -10.777  -2.628 -13.622  1.00  0.53           C  \\nATOM     33  O   GLU A   5      -7.971  -3.945 -13.849  1.00  0.56           O  \\nATOM     34  CG  GLU A   5     -11.870  -1.586 -13.435  1.00  0.50           C  \\nATOM     35  CD  GLU A   5     -13.225  -2.037 -13.955  1.00  0.51           C  \\nATOM     36  OE1 GLU A   5     -13.331  -3.176 -14.465  1.00  0.50           O  \\nATOM     37  OE2 GLU A   5     -14.189  -1.246 -13.853  1.00  0.45           O  \\nATOM     38  N   CYS A   6      -7.805  -3.434 -11.874  1.00  0.68           N  \\nATOM     39  CA  CYS A   6      -7.276  -4.690 -11.355  1.00  0.68           C  \\nATOM     40  C   CYS A   6      -8.207  -5.851 -11.684  1.00  0.68           C  \\nATOM     41  CB  CYS A   6      -7.071  -4.601  -9.843  1.00  0.66           C  \\nATOM     42  O   CYS A   6      -9.425  -5.678 -11.745  1.00  0.66           O  \\nATOM     43  SG  CYS A   6      -5.621  -3.640  -9.360  1.00  0.66           S  \\nATOM     44  N   PRO A   7      -7.733  -6.781 -12.513  1.00  0.63           N  \\nATOM     45  CA  PRO A   7      -8.638  -7.901 -12.783  1.00  0.62           C  \\nATOM     46  C   PRO A   7      -9.494  -8.273 -11.574  1.00  0.64           C  \\nATOM     47  CB  PRO A   7      -7.686  -9.042 -13.150  1.00  0.59           C  \\nATOM     48  O   PRO A   7      -9.092  -8.035 -10.432  1.00  0.62           O  \\nATOM     49  CG  PRO A   7      -6.360  -8.383 -13.354  1.00  0.57           C  \\nATOM     50  CD  PRO A   7      -6.375  -7.055 -12.653  1.00  0.56           C  \\nATOM     51  N   LEU A   8     -10.850  -8.459 -11.712  1.00  0.60           N  \\nATOM     52  CA  LEU A   8     -11.733  -8.966 -10.668  1.00  0.58           C  \\nATOM     53  C   LEU A   8     -11.023 -10.013  -9.816  1.00  0.60           C  \\nATOM     54  CB  LEU A   8     -13.001  -9.565 -11.282  1.00  0.54           C  \\nATOM     55  O   LEU A   8     -11.283 -10.124  -8.616  1.00  0.58           O  \\nATOM     56  CG  LEU A   8     -14.000  -8.572 -11.878  1.00  0.51           C  \\nATOM     57  CD1 LEU A   8     -14.999  -9.297 -12.773  1.00  0.44           C  \\nATOM     58  CD2 LEU A   8     -14.721  -7.810 -10.771  1.00  0.46           C  \\nATOM     59  N   SER A   9     -10.284 -10.902 -10.463  1.00  0.60           N  \\nATOM     60  CA  SER A   9      -9.555 -11.936  -9.736  1.00  0.58           C  \\nATOM     61  C   SER A   9      -8.618 -11.325  -8.698  1.00  0.59           C  \\nATOM     62  CB  SER A   9      -8.756 -12.810 -10.703  1.00  0.54           C  \\nATOM     63  O   SER A   9      -8.105 -12.029  -7.827  1.00  0.56           O  \\nATOM     64  OG  SER A   9      -7.746 -12.054 -11.348  1.00  0.50           O  \\nATOM     65  N   HIS A  10      -8.393 -10.120  -8.767  1.00  0.63           N  \\nATOM     66  CA  HIS A  10      -7.507  -9.535  -7.767  1.00  0.61           C  \\nATOM     67  C   HIS A  10      -8.301  -8.932  -6.613  1.00  0.63           C  \\nATOM     68  CB  HIS A  10      -6.612  -8.469  -8.402  1.00  0.58           C  \\nATOM     69  O   HIS A  10      -7.826  -8.016  -5.938  1.00  0.61           O  \\nATOM     70  CG  HIS A  10      -5.432  -9.032  -9.128  1.00  0.55           C  \\nATOM     71  CD2 HIS A  10      -4.127  -9.110  -8.777  1.00  0.53           C  \\nATOM     72  ND1 HIS A  10      -5.530  -9.600 -10.379  1.00  0.52           N  \\nATOM     73  CE1 HIS A  10      -4.332 -10.006 -10.767  1.00  0.49           C  \\nATOM     74  NE2 HIS A  10      -3.463  -9.719  -9.813  1.00  0.46           N  \\nATOM     75  N   ASP A  11      -9.703  -9.372  -6.500  1.00  0.67           N  \\nATOM     76  CA  ASP A  11     -10.363  -9.201  -5.210  1.00  0.66           C  \\nATOM     77  C   ASP A  11      -9.570  -9.876  -4.093  1.00  0.67           C  \\nATOM     78  CB  ASP A  11     -11.787  -9.760  -5.256  1.00  0.62           C  \\nATOM     79  O   ASP A  11      -9.244 -11.062  -4.186  1.00  0.63           O  \\nATOM     80  CG  ASP A  11     -12.834  -8.753  -4.812  1.00  0.58           C  \\nATOM     81  OD1 ASP A  11     -12.473  -7.603  -4.481  1.00  0.57           O  \\nATOM     82  OD2 ASP A  11     -14.030  -9.114  -4.790  1.00  0.59           O  \\nATOM     83  N   GLY A  12      -8.604  -9.100  -3.629  1.00  0.74           N  \\nATOM     84  CA  GLY A  12      -7.784  -9.534  -2.509  1.00  0.74           C  \\nATOM     85  C   GLY A  12      -6.301  -9.302  -2.730  1.00  0.75           C  \\nATOM     86  O   GLY A  12      -5.474  -9.717  -1.916  1.00  0.73           O  \\nATOM     87  N   TYR A  13      -5.995  -8.843  -3.903  1.00  0.82           N  \\nATOM     88  CA  TYR A  13      -4.571  -8.632  -4.139  1.00  0.82           C  \\nATOM     89  C   TYR A  13      -3.982  -7.683  -3.102  1.00  0.82           C  \\nATOM     90  CB  TYR A  13      -4.335  -8.077  -5.547  1.00  0.80           C  \\nATOM     91  O   TYR A  13      -2.964  -7.991  -2.476  1.00  0.81           O  \\nATOM     92  CG  TYR A  13      -2.877  -7.896  -5.892  1.00  0.76           C  \\nATOM     93  CD1 TYR A  13      -2.038  -8.995  -6.055  1.00  0.72           C  \\nATOM     94  CD2 TYR A  13      -2.335  -6.625  -6.054  1.00  0.74           C  \\nATOM     95  CE1 TYR A  13      -0.693  -8.833  -6.371  1.00  0.71           C  \\nATOM     96  CE2 TYR A  13      -0.992  -6.451  -6.369  1.00  0.71           C  \\nATOM     97  OH  TYR A  13       1.151  -7.393  -6.839  1.00  0.64           O  \\nATOM     98  CZ  TYR A  13      -0.180  -7.559  -6.526  1.00  0.68           C  \\nATOM     99  N   CYS A  14      -4.652  -6.448  -2.907  1.00  0.85           N  \\nATOM    100  CA  CYS A  14      -4.239  -5.521  -1.858  1.00  0.86           C  \\nATOM    101  C   CYS A  14      -4.892  -5.877  -0.528  1.00  0.86           C  \\nATOM    102  CB  CYS A  14      -4.593  -4.085  -2.242  1.00  0.85           C  \\nATOM    103  O   CYS A  14      -6.107  -6.068  -0.459  1.00  0.85           O  \\nATOM    104  SG  CYS A  14      -3.871  -3.551  -3.810  1.00  0.84           S  \\nATOM    105  N   LEU A  15      -4.073  -5.961   0.460  1.00  0.87           N  \\nATOM    106  CA  LEU A  15      -4.552  -6.327   1.789  1.00  0.88           C  \\nATOM    107  C   LEU A  15      -4.987  -5.090   2.569  1.00  0.88           C  \\nATOM    108  CB  LEU A  15      -3.466  -7.077   2.563  1.00  0.86           C  \\nATOM    109  O   LEU A  15      -4.739  -3.961   2.139  1.00  0.87           O  \\nATOM    110  CG  LEU A  15      -2.936  -8.358   1.919  1.00  0.83           C  \\nATOM    111  CD1 LEU A  15      -1.789  -8.932   2.744  1.00  0.81           C  \\nATOM    112  CD2 LEU A  15      -4.055  -9.382   1.765  1.00  0.81           C  \\nATOM    113  N   HIS A  16      -5.851  -5.334   3.662  1.00  0.88           N  \\nATOM    114  CA  HIS A  16      -6.200  -4.338   4.669  1.00  0.89           C  \\nATOM    115  C   HIS A  16      -6.826  -3.103   4.029  1.00  0.88           C  \\nATOM    116  CB  HIS A  16      -4.965  -3.940   5.479  1.00  0.87           C  \\nATOM    117  O   HIS A  16      -6.429  -1.975   4.328  1.00  0.86           O  \\nATOM    118  CG  HIS A  16      -4.284  -5.096   6.141  1.00  0.84           C  \\nATOM    119  CD2 HIS A  16      -3.095  -5.691   5.886  1.00  0.82           C  \\nATOM    120  ND1 HIS A  16      -4.837  -5.777   7.203  1.00  0.78           N  \\nATOM    121  CE1 HIS A  16      -4.015  -6.744   7.575  1.00  0.83           C  \\nATOM    122  NE2 HIS A  16      -2.950  -6.713   6.792  1.00  0.83           N  \\nATOM    123  N   ASP A  17      -7.803  -3.279   3.146  1.00  0.86           N  \\nATOM    124  CA  ASP A  17      -8.627  -2.264   2.498  1.00  0.86           C  \\nATOM    125  C   ASP A  17      -7.803  -1.432   1.518  1.00  0.86           C  \\nATOM    126  CB  ASP A  17      -9.279  -1.355   3.542  1.00  0.85           C  \\nATOM    127  O   ASP A  17      -8.167  -0.297   1.201  1.00  0.84           O  \\nATOM    128  CG  ASP A  17     -10.312  -2.073   4.392  1.00  0.82           C  \\nATOM    129  OD1 ASP A  17     -10.918  -3.056   3.914  1.00  0.80           O  \\nATOM    130  OD2 ASP A  17     -10.525  -1.650   5.549  1.00  0.81           O  \\nATOM    131  N   GLY A  18      -6.648  -1.956   1.127  1.00  0.87           N  \\nATOM    132  CA  GLY A  18      -5.904  -1.344   0.037  1.00  0.87           C  \\nATOM    133  C   GLY A  18      -6.642  -1.388  -1.288  1.00  0.86           C  \\nATOM    134  O   GLY A  18      -7.519  -2.230  -1.488  1.00  0.84           O  \\nATOM    135  N   VAL A  19      -6.407  -0.364  -2.154  1.00  0.85           N  \\nATOM    136  CA  VAL A  19      -7.053  -0.262  -3.458  1.00  0.85           C  \\nATOM    137  C   VAL A  19      -6.062  -0.638  -4.557  1.00  0.85           C  \\nATOM    138  CB  VAL A  19      -7.610   1.158  -3.706  1.00  0.84           C  \\nATOM    139  O   VAL A  19      -4.963  -0.083  -4.627  1.00  0.83           O  \\nATOM    140  CG1 VAL A  19      -8.187   1.273  -5.116  1.00  0.80           C  \\nATOM    141  CG2 VAL A  19      -8.669   1.506  -2.661  1.00  0.79           C  \\nATOM    142  N   CYS A  20      -6.373  -1.610  -5.323  1.00  0.84           N  \\nATOM    143  CA  CYS A  20      -5.540  -2.077  -6.427  1.00  0.84           C  \\nATOM    144  C   CYS A  20      -5.621  -1.122  -7.612  1.00  0.84           C  \\nATOM    145  CB  CYS A  20      -5.963  -3.480  -6.860  1.00  0.83           C  \\nATOM    146  O   CYS A  20      -6.701  -0.632  -7.947  1.00  0.82           O  \\nATOM    147  SG  CYS A  20      -4.884  -4.210  -8.112  1.00  0.82           S  \\nATOM    148  N   MET A  21      -4.563  -0.934  -8.213  1.00  0.83           N  \\nATOM    149  CA  MET A  21      -4.470  -0.092  -9.401  1.00  0.83           C  \\nATOM    150  C   MET A  21      -3.542  -0.715 -10.439  1.00  0.83           C  \\nATOM    151  CB  MET A  21      -3.976   1.308  -9.032  1.00  0.82           C  \\nATOM    152  O   MET A  21      -2.582  -1.401 -10.086  1.00  0.82           O  \\nATOM    153  CG  MET A  21      -2.580   1.326  -8.428  1.00  0.77           C  \\nATOM    154  SD  MET A  21      -2.091   2.993  -7.839  1.00  0.76           S  \\nATOM    155  CE  MET A  21      -1.944   3.863  -9.425  1.00  0.65           C  \\nATOM    156  N   TYR A  22      -3.868  -0.655 -11.665  1.00  0.82           N  \\nATOM    157  CA  TYR A  22      -3.016  -1.101 -12.762  1.00  0.82           C  \\nATOM    158  C   TYR A  22      -2.256   0.069 -13.374  1.00  0.82           C  \\nATOM    159  CB  TYR A  22      -3.850  -1.802 -13.839  1.00  0.80           C  \\nATOM    160  O   TYR A  22      -2.855   1.083 -13.740  1.00  0.81           O  \\nATOM    161  CG  TYR A  22      -3.035  -2.308 -15.004  1.00  0.77           C  \\nATOM    162  CD1 TYR A  22      -2.158  -3.380 -14.853  1.00  0.75           C  \\nATOM    163  CD2 TYR A  22      -3.138  -1.716 -16.258  1.00  0.76           C  \\nATOM    164  CE1 TYR A  22      -1.403  -3.849 -15.923  1.00  0.74           C  \\nATOM    165  CE2 TYR A  22      -2.389  -2.176 -17.335  1.00  0.74           C  \\nATOM    166  OH  TYR A  22      -0.781  -3.701 -18.221  1.00  0.68           O  \\nATOM    167  CZ  TYR A  22      -1.525  -3.242 -17.158  1.00  0.70           C  \\nATOM    168  N   ILE A  23      -0.909   0.026 -13.379  1.00  0.81           N  \\nATOM    169  CA  ILE A  23      -0.030   1.046 -13.939  1.00  0.81           C  \\nATOM    170  C   ILE A  23       0.412   0.634 -15.341  1.00  0.82           C  \\nATOM    171  CB  ILE A  23       1.202   1.287 -13.037  1.00  0.80           C  \\nATOM    172  O   ILE A  23       1.288  -0.221 -15.496  1.00  0.81           O  \\nATOM    173  CG1 ILE A  23       0.762   1.580 -11.599  1.00  0.80           C  \\nATOM    174  CG2 ILE A  23       2.062   2.427 -13.591  1.00  0.80           C  \\nATOM    175  CD1 ILE A  23       1.914   1.695 -10.610  1.00  0.78           C  \\nATOM    176  N   GLU A  24      -0.160   1.169 -16.357  1.00  0.82           N  \\nATOM    177  CA  GLU A  24       0.044   0.808 -17.756  1.00  0.81           C  \\nATOM    178  C   GLU A  24       1.500   0.999 -18.169  1.00  0.82           C  \\nATOM    179  CB  GLU A  24      -0.873   1.632 -18.664  1.00  0.80           C  \\nATOM    180  O   GLU A  24       2.067   0.158 -18.871  1.00  0.81           O  \\nATOM    181  CG  GLU A  24      -0.892   1.161 -20.110  1.00  0.75           C  \\nATOM    182  CD  GLU A  24      -1.880   1.925 -20.977  1.00  0.72           C  \\nATOM    183  OE1 GLU A  24      -2.596   2.806 -20.450  1.00  0.67           O  \\nATOM    184  OE2 GLU A  24      -1.940   1.639 -22.194  1.00  0.64           O  \\nATOM    185  N   ALA A  25       2.013   2.100 -17.746  1.00  0.81           N  \\nATOM    186  CA  ALA A  25       3.369   2.431 -18.177  1.00  0.82           C  \\nATOM    187  C   ALA A  25       4.360   1.353 -17.748  1.00  0.82           C  \\nATOM    188  CB  ALA A  25       3.789   3.788 -17.616  1.00  0.80           C  \\nATOM    189  O   ALA A  25       5.363   1.117 -18.426  1.00  0.81           O  \\nATOM    190  N   LEU A  26       4.083   0.698 -16.649  1.00  0.82           N  \\nATOM    191  CA  LEU A  26       4.955  -0.329 -16.091  1.00  0.82           C  \\nATOM    192  C   LEU A  26       4.397  -1.722 -16.362  1.00  0.82           C  \\nATOM    193  CB  LEU A  26       5.133  -0.122 -14.585  1.00  0.80           C  \\nATOM    194  O   LEU A  26       5.097  -2.722 -16.181  1.00  0.80           O  \\nATOM    195  CG  LEU A  26       5.842   1.164 -14.156  1.00  0.76           C  \\nATOM    196  CD1 LEU A  26       5.848   1.284 -12.636  1.00  0.72           C  \\nATOM    197  CD2 LEU A  26       7.263   1.200 -14.707  1.00  0.74           C  \\nATOM    198  N   ASP A  27       3.124  -1.697 -16.831  1.00  0.83           N  \\nATOM    199  CA  ASP A  27       2.419  -2.970 -16.949  1.00  0.82           C  \\nATOM    200  C   ASP A  27       2.454  -3.741 -15.631  1.00  0.82           C  \\nATOM    201  CB  ASP A  27       3.025  -3.818 -18.070  1.00  0.81           C  \\nATOM    202  O   ASP A  27       2.861  -4.904 -15.596  1.00  0.80           O  \\nATOM    203  CG  ASP A  27       2.167  -5.016 -18.437  1.00  0.79           C  \\nATOM    204  OD1 ASP A  27       0.951  -5.006 -18.151  1.00  0.78           O  \\nATOM    205  OD2 ASP A  27       2.714  -5.980 -19.015  1.00  0.77           O  \\nATOM    206  N   LYS A  28       2.180  -3.218 -14.579  1.00  0.83           N  \\nATOM    207  CA  LYS A  28       2.192  -3.810 -13.245  1.00  0.83           C  \\nATOM    208  C   LYS A  28       0.988  -3.351 -12.428  1.00  0.83           C  \\nATOM    209  CB  LYS A  28       3.487  -3.455 -12.512  1.00  0.81           C  \\nATOM    210  O   LYS A  28       0.450  -2.266 -12.661  1.00  0.82           O  \\nATOM    211  CG  LYS A  28       4.729  -4.114 -13.096  1.00  0.78           C  \\nATOM    212  CD  LYS A  28       5.960  -3.841 -12.243  1.00  0.76           C  \\nATOM    213  CE  LYS A  28       7.188  -4.561 -12.784  1.00  0.68           C  \\nATOM    214  NZ  LYS A  28       8.384  -4.345 -11.916  1.00  0.65           N  \\nATOM    215  N   TYR A  29       0.624  -4.199 -11.528  1.00  0.83           N  \\nATOM    216  CA  TYR A  29      -0.385  -3.846 -10.536  1.00  0.83           C  \\nATOM    217  C   TYR A  29       0.263  -3.301  -9.269  1.00  0.83           C  \\nATOM    218  CB  TYR A  29      -1.254  -5.061 -10.197  1.00  0.81           C  \\nATOM    219  O   TYR A  29       1.357  -3.727  -8.892  1.00  0.82           O  \\nATOM    220  CG  TYR A  29      -2.008  -5.617 -11.381  1.00  0.79           C  \\nATOM    221  CD1 TYR A  29      -3.185  -5.019 -11.825  1.00  0.77           C  \\nATOM    222  CD2 TYR A  29      -1.546  -6.741 -12.056  1.00  0.78           C  \\nATOM    223  CE1 TYR A  29      -3.885  -5.529 -12.914  1.00  0.75           C  \\nATOM    224  CE2 TYR A  29      -2.237  -7.260 -13.146  1.00  0.76           C  \\nATOM    225  OH  TYR A  29      -4.091  -7.157 -14.646  1.00  0.70           O  \\nATOM    226  CZ  TYR A  29      -3.403  -6.648 -13.567  1.00  0.71           C  \\nATOM    227  N   ALA A  30      -0.345  -2.309  -8.674  1.00  0.85           N  \\nATOM    228  CA  ALA A  30       0.128  -1.721  -7.423  1.00  0.85           C  \\nATOM    229  C   ALA A  30      -1.030  -1.479  -6.459  1.00  0.85           C  \\nATOM    230  CB  ALA A  30       0.871  -0.416  -7.697  1.00  0.84           C  \\nATOM    231  O   ALA A  30      -2.192  -1.445  -6.871  1.00  0.85           O  \\nATOM    232  N   CYS A  31      -0.771  -1.457  -5.228  1.00  0.87           N  \\nATOM    233  CA  CYS A  31      -1.761  -1.201  -4.188  1.00  0.87           C  \\nATOM    234  C   CYS A  31      -1.593   0.198  -3.608  1.00  0.87           C  \\nATOM    235  CB  CYS A  31      -1.650  -2.242  -3.075  1.00  0.86           C  \\nATOM    236  O   CYS A  31      -0.469   0.653  -3.390  1.00  0.86           O  \\nATOM    237  SG  CYS A  31      -2.063  -3.920  -3.602  1.00  0.84           S  \\nATOM    238  N   ASN A  32      -2.667   0.955  -3.557  1.00  0.87           N  \\nATOM    239  CA  ASN A  32      -2.741   2.120  -2.682  1.00  0.87           C  \\nATOM    240  C   ASN A  32      -3.148   1.732  -1.264  1.00  0.88           C  \\nATOM    241  CB  ASN A  32      -3.715   3.156  -3.249  1.00  0.86           C  \\nATOM    242  O   ASN A  32      -4.287   1.321  -1.030  1.00  0.87           O  \\nATOM    243  CG  ASN A  32      -3.676   4.469  -2.493  1.00  0.79           C  \\nATOM    244  ND2 ASN A  32      -4.593   5.371  -2.823  1.00  0.75           N  \\nATOM    245  OD1 ASN A  32      -2.828   4.672  -1.621  1.00  0.76           O  \\nATOM    246  N   CYS A  33      -2.283   1.827  -0.373  1.00  0.90           N  \\nATOM    247  CA  CYS A  33      -2.528   1.356   0.986  1.00  0.91           C  \\nATOM    248  C   CYS A  33      -3.249   2.417   1.809  1.00  0.91           C  \\nATOM    249  CB  CYS A  33      -1.214   0.976   1.666  1.00  0.89           C  \\nATOM    250  O   CYS A  33      -3.026   3.614   1.619  1.00  0.89           O  \\nATOM    251  SG  CYS A  33      -0.271  -0.282   0.777  1.00  0.89           S  \\nATOM    252  N   VAL A  34      -4.150   2.077   2.645  1.00  0.90           N  \\nATOM    253  CA  VAL A  34      -4.735   2.997   3.614  1.00  0.90           C  \\nATOM    254  C   VAL A  34      -3.683   3.399   4.646  1.00  0.90           C  \\nATOM    255  CB  VAL A  34      -5.962   2.377   4.317  1.00  0.89           C  \\nATOM    256  O   VAL A  34      -2.683   2.700   4.826  1.00  0.90           O  \\nATOM    257  CG1 VAL A  34      -7.062   2.060   3.305  1.00  0.79           C  \\nATOM    258  CG2 VAL A  34      -5.558   1.118   5.083  1.00  0.79           C  \\nATOM    259  N   VAL A  35      -3.937   4.536   5.231  1.00  0.91           N  \\nATOM    260  CA  VAL A  35      -3.001   5.051   6.226  1.00  0.91           C  \\nATOM    261  C   VAL A  35      -2.778   4.004   7.315  1.00  0.91           C  \\nATOM    262  CB  VAL A  35      -3.506   6.371   6.850  1.00  0.89           C  \\nATOM    263  O   VAL A  35      -3.730   3.382   7.791  1.00  0.89           O  \\nATOM    264  CG1 VAL A  35      -2.537   6.867   7.922  1.00  0.81           C  \\nATOM    265  CG2 VAL A  35      -3.701   7.432   5.769  1.00  0.81           C  \\nATOM    266  N   GLY A  36      -1.488   3.610   7.611  1.00  0.91           N  \\nATOM    267  CA  GLY A  36      -1.118   2.655   8.644  1.00  0.91           C  \\nATOM    268  C   GLY A  36      -0.564   1.358   8.087  1.00  0.91           C  \\nATOM    269  O   GLY A  36      -0.048   0.525   8.835  1.00  0.90           O  \\nATOM    270  N   TYR A  37      -0.654   1.183   6.843  1.00  0.92           N  \\nATOM    271  CA  TYR A  37      -0.150  -0.009   6.171  1.00  0.93           C  \\nATOM    272  C   TYR A  37       0.792   0.364   5.033  1.00  0.92           C  \\nATOM    273  CB  TYR A  37      -1.310  -0.854   5.634  1.00  0.92           C  \\nATOM    274  O   TYR A  37       0.668   1.440   4.443  1.00  0.91           O  \\nATOM    275  CG  TYR A  37      -2.123  -1.524   6.714  1.00  0.90           C  \\nATOM    276  CD1 TYR A  37      -1.829  -2.820   7.132  1.00  0.88           C  \\nATOM    277  CD2 TYR A  37      -3.188  -0.864   7.318  1.00  0.89           C  \\nATOM    278  CE1 TYR A  37      -2.577  -3.443   8.126  1.00  0.89           C  \\nATOM    279  CE2 TYR A  37      -3.944  -1.477   8.312  1.00  0.89           C  \\nATOM    280  OH  TYR A  37      -4.375  -3.376   9.693  1.00  0.82           O  \\nATOM    281  CZ  TYR A  37      -3.631  -2.764   8.709  1.00  0.88           C  \\nATOM    282  N   ILE A  38       1.765  -0.605   4.902  1.00  0.92           N  \\nATOM    283  CA  ILE A  38       2.757  -0.433   3.846  1.00  0.92           C  \\nATOM    284  C   ILE A  38       3.060  -1.782   3.198  1.00  0.93           C  \\nATOM    285  CB  ILE A  38       4.055   0.205   4.390  1.00  0.92           C  \\nATOM    286  O   ILE A  38       2.548  -2.816   3.634  1.00  0.92           O  \\nATOM    287  CG1 ILE A  38       4.678  -0.689   5.468  1.00  0.89           C  \\nATOM    288  CG2 ILE A  38       3.780   1.609   4.935  1.00  0.89           C  \\nATOM    289  CD1 ILE A  38       6.082  -0.270   5.884  1.00  0.86           C  \\nATOM    290  N   GLY A  39       3.964  -1.780   2.045  1.00  0.91           N  \\nATOM    291  CA  GLY A  39       4.350  -2.984   1.327  1.00  0.91           C  \\nATOM    292  C   GLY A  39       3.720  -3.087  -0.049  1.00  0.90           C  \\nATOM    293  O   GLY A  39       2.854  -2.285  -0.402  1.00  0.89           O  \\nATOM    294  N   GLU A  40       4.067  -4.027  -0.773  1.00  0.89           N  \\nATOM    295  CA  GLU A  40       3.601  -4.210  -2.144  1.00  0.89           C  \\nATOM    296  C   GLU A  40       2.103  -4.501  -2.184  1.00  0.89           C  \\nATOM    297  CB  GLU A  40       4.374  -5.339  -2.829  1.00  0.87           C  \\nATOM    298  O   GLU A  40       1.400  -4.044  -3.087  1.00  0.88           O  \\nATOM    299  CG  GLU A  40       5.829  -5.000  -3.120  1.00  0.78           C  \\nATOM    300  CD  GLU A  40       6.556  -6.089  -3.894  1.00  0.75           C  \\nATOM    301  OE1 GLU A  40       5.976  -7.179  -4.097  1.00  0.71           O  \\nATOM    302  OE2 GLU A  40       7.715  -5.849  -4.299  1.00  0.67           O  \\nATOM    303  N   ARG A  41       1.609  -5.201  -1.159  1.00  0.89           N  \\nATOM    304  CA  ARG A  41       0.194  -5.550  -1.094  1.00  0.89           C  \\nATOM    305  C   ARG A  41      -0.463  -4.948   0.144  1.00  0.90           C  \\nATOM    306  CB  ARG A  41       0.014  -7.069  -1.097  1.00  0.88           C  \\nATOM    307  O   ARG A  41      -1.486  -5.448   0.615  1.00  0.89           O  \\nATOM    308  CG  ARG A  41       0.499  -7.745  -2.370  1.00  0.85           C  \\nATOM    309  CD  ARG A  41       0.225  -9.242  -2.354  1.00  0.83           C  \\nATOM    310  NE  ARG A  41      -1.204  -9.528  -2.269  1.00  0.78           N  \\nATOM    311  NH1 ARG A  41      -0.937 -11.717  -1.584  1.00  0.64           N  \\nATOM    312  NH2 ARG A  41      -3.039 -10.848  -1.863  1.00  0.63           N  \\nATOM    313  CZ  ARG A  41      -1.723 -10.697  -1.906  1.00  0.80           C  \\nATOM    314  N   CYS A  42       0.277  -3.945   0.723  1.00  0.90           N  \\nATOM    315  CA  CYS A  42      -0.213  -3.337   1.955  1.00  0.91           C  \\nATOM    316  C   CYS A  42      -0.206  -4.342   3.100  1.00  0.91           C  \\nATOM    317  CB  CYS A  42      -1.626  -2.788   1.756  1.00  0.90           C  \\nATOM    318  O   CYS A  42      -1.128  -4.366   3.917  1.00  0.90           O  \\nATOM    319  SG  CYS A  42      -1.791  -1.706   0.318  1.00  0.88           S  \\nATOM    320  N   GLN A  43       0.844  -5.243   3.151  1.00  0.92           N  \\nATOM    321  CA  GLN A  43       0.821  -6.399   4.040  1.00  0.92           C  \\nATOM    322  C   GLN A  43       1.448  -6.067   5.391  1.00  0.92           C  \\nATOM    323  CB  GLN A  43       1.549  -7.583   3.400  1.00  0.90           C  \\nATOM    324  O   GLN A  43       1.232  -6.780   6.373  1.00  0.91           O  \\nATOM    325  CG  GLN A  43       3.038  -7.347   3.187  1.00  0.86           C  \\nATOM    326  CD  GLN A  43       3.350  -6.754   1.826  1.00  0.84           C  \\nATOM    327  NE2 GLN A  43       4.459  -7.182   1.233  1.00  0.79           N  \\nATOM    328  OE1 GLN A  43       2.600  -5.918   1.312  1.00  0.82           O  \\nATOM    329  N   TYR A  44       2.093  -5.035   5.476  1.00  0.93           N  \\nATOM    330  CA  TYR A  44       2.773  -4.710   6.724  1.00  0.93           C  \\nATOM    331  C   TYR A  44       2.121  -3.513   7.406  1.00  0.92           C  \\nATOM    332  CB  TYR A  44       4.256  -4.420   6.469  1.00  0.92           C  \\nATOM    333  O   TYR A  44       1.588  -2.625   6.737  1.00  0.91           O  \\nATOM    334  CG  TYR A  44       4.989  -5.555   5.796  1.00  0.89           C  \\nATOM    335  CD1 TYR A  44       5.121  -6.793   6.422  1.00  0.86           C  \\nATOM    336  CD2 TYR A  44       5.551  -5.393   4.534  1.00  0.86           C  \\nATOM    337  CE1 TYR A  44       5.798  -7.841   5.807  1.00  0.87           C  \\nATOM    338  CE2 TYR A  44       6.229  -6.434   3.910  1.00  0.87           C  \\nATOM    339  OH  TYR A  44       7.017  -8.687   3.939  1.00  0.79           O  \\nATOM    340  CZ  TYR A  44       6.347  -7.653   4.552  1.00  0.86           C  \\nATOM    341  N   ARG A  45       2.158  -3.532   8.714  1.00  0.92           N  \\nATOM    342  CA  ARG A  45       1.771  -2.357   9.488  1.00  0.92           C  \\nATOM    343  C   ARG A  45       2.907  -1.340   9.545  1.00  0.92           C  \\nATOM    344  CB  ARG A  45       1.356  -2.758  10.905  1.00  0.91           C  \\nATOM    345  O   ARG A  45       4.073  -1.712   9.690  1.00  0.91           O  \\nATOM    346  CG  ARG A  45       0.050  -3.533  10.968  1.00  0.84           C  \\nATOM    347  CD  ARG A  45      -0.298  -3.937  12.394  1.00  0.82           C  \\nATOM    348  NE  ARG A  45      -1.576  -4.641  12.459  1.00  0.76           N  \\nATOM    349  NH1 ARG A  45      -1.550  -4.904  14.754  1.00  0.58           N  \\nATOM    350  NH2 ARG A  45      -3.306  -5.713  13.522  1.00  0.56           N  \\nATOM    351  CZ  ARG A  45      -2.141  -5.084  13.578  1.00  0.76           C  \\nATOM    352  N   ASP A  46       2.608  -0.114   9.094  1.00  0.92           N  \\nATOM    353  CA  ASP A  46       3.581   0.949   9.328  1.00  0.92           C  \\nATOM    354  C   ASP A  46       3.793   1.180  10.822  1.00  0.91           C  \\nATOM    355  CB  ASP A  46       3.130   2.246   8.653  1.00  0.91           C  \\nATOM    356  O   ASP A  46       2.943   1.771  11.491  1.00  0.90           O  \\nATOM    357  CG  ASP A  46       4.199   3.325   8.669  1.00  0.88           C  \\nATOM    358  OD1 ASP A  46       5.186   3.195   9.424  1.00  0.86           O  \\nATOM    359  OD2 ASP A  46       4.051   4.315   7.921  1.00  0.86           O  \\nATOM    360  N   LEU A  47       4.865   0.611  11.474  1.00  0.90           N  \\nATOM    361  CA  LEU A  47       5.079   0.563  12.916  1.00  0.90           C  \\nATOM    362  C   LEU A  47       5.090   1.967  13.510  1.00  0.89           C  \\nATOM    363  CB  LEU A  47       6.394  -0.151  13.240  1.00  0.88           C  \\nATOM    364  O   LEU A  47       4.657   2.168  14.648  1.00  0.88           O  \\nATOM    365  CG  LEU A  47       6.461  -1.639  12.893  1.00  0.83           C  \\nATOM    366  CD1 LEU A  47       7.838  -2.201  13.232  1.00  0.78           C  \\nATOM    367  CD2 LEU A  47       5.368  -2.409  13.626  1.00  0.78           C  \\nATOM    368  N   LYS A  48       5.663   2.934  12.806  1.00  0.88           N  \\nATOM    369  CA  LYS A  48       5.659   4.301  13.319  1.00  0.88           C  \\nATOM    370  C   LYS A  48       4.234   4.811  13.512  1.00  0.88           C  \\nATOM    371  CB  LYS A  48       6.425   5.230  12.375  1.00  0.85           C  \\nATOM    372  O   LYS A  48       3.924   5.431  14.532  1.00  0.86           O  \\nATOM    373  CG  LYS A  48       7.932   5.021  12.389  1.00  0.77           C  \\nATOM    374  CD  LYS A  48       8.645   6.036  11.506  1.00  0.74           C  \\nATOM    375  CE  LYS A  48      10.152   5.819  11.508  1.00  0.69           C  \\nATOM    376  NZ  LYS A  48      10.853   6.795  10.620  1.00  0.61           N  \\nATOM    377  N   TRP A  49       3.476   4.570  12.507  1.00  0.87           N  \\nATOM    378  CA  TRP A  49       2.080   4.989  12.567  1.00  0.88           C  \\nATOM    379  C   TRP A  49       1.353   4.297  13.716  1.00  0.86           C  \\nATOM    380  CB  TRP A  49       1.370   4.689  11.244  1.00  0.85           C  \\nATOM    381  O   TRP A  49       0.601   4.935  14.456  1.00  0.83           O  \\nATOM    382  CG  TRP A  49       0.008   5.305  11.129  1.00  0.74           C  \\nATOM    383  CD1 TRP A  49      -0.311   6.493  10.533  1.00  0.66           C  \\nATOM    384  CD2 TRP A  49      -1.218   4.765  11.630  1.00  0.69           C  \\nATOM    385  CE2 TRP A  49      -2.243   5.679  11.300  1.00  0.71           C  \\nATOM    386  CE3 TRP A  49      -1.551   3.594  12.324  1.00  0.74           C  \\nATOM    387  NE1 TRP A  49      -1.664   6.724  10.632  1.00  0.76           N  \\nATOM    388  CH2 TRP A  49      -3.877   4.305  12.322  1.00  0.71           C  \\nATOM    389  CZ2 TRP A  49      -3.579   5.458  11.643  1.00  0.74           C  \\nATOM    390  CZ3 TRP A  49      -2.881   3.376  12.664  1.00  0.72           C  \\nATOM    391  N   TRP A  50       1.416   3.062  13.899  1.00  0.89           N  \\nATOM    392  CA  TRP A  50       0.668   2.281  14.879  1.00  0.89           C  \\nATOM    393  C   TRP A  50       1.161   2.568  16.294  1.00  0.88           C  \\nATOM    394  CB  TRP A  50       0.785   0.785  14.577  1.00  0.88           C  \\nATOM    395  O   TRP A  50       0.392   2.485  17.254  1.00  0.85           O  \\nATOM    396  CG  TRP A  50      -0.088   0.320  13.451  1.00  0.85           C  \\nATOM    397  CD1 TRP A  50       0.222   0.307  12.120  1.00  0.81           C  \\nATOM    398  CD2 TRP A  50      -1.419  -0.194  13.559  1.00  0.82           C  \\nATOM    399  CE2 TRP A  50      -1.856  -0.501  12.251  1.00  0.85           C  \\nATOM    400  CE3 TRP A  50      -2.285  -0.427  14.635  1.00  0.79           C  \\nATOM    401  NE1 TRP A  50      -0.837  -0.186  11.392  1.00  0.79           N  \\nATOM    402  CH2 TRP A  50      -3.950  -1.246  13.062  1.00  0.78           C  \\nATOM    403  CZ2 TRP A  50      -3.123  -1.029  11.991  1.00  0.79           C  \\nATOM    404  CZ3 TRP A  50      -3.546  -0.952  14.375  1.00  0.79           C  \\nATOM    405  N   GLU A  51       2.410   2.889  16.502  1.00  0.89           N  \\nATOM    406  CA  GLU A  51       2.952   3.218  17.817  1.00  0.88           C  \\nATOM    407  C   GLU A  51       2.372   4.528  18.341  1.00  0.87           C  \\nATOM    408  CB  GLU A  51       4.480   3.303  17.763  1.00  0.86           C  \\nATOM    409  O   GLU A  51       2.248   4.717  19.553  1.00  0.85           O  \\nATOM    410  CG  GLU A  51       5.168   1.950  17.657  1.00  0.79           C  \\nATOM    411  CD  GLU A  51       6.682   2.053  17.561  1.00  0.75           C  \\nATOM    412  OE1 GLU A  51       7.211   3.186  17.506  1.00  0.72           O  \\nATOM    413  OE2 GLU A  51       7.344   0.992  17.541  1.00  0.70           O  \\nATOM    414  N   LEU A  52       2.083   5.414  17.516  1.00  0.87           N  \\nATOM    415  CA  LEU A  52       1.544   6.710  17.913  1.00  0.86           C  \\nATOM    416  C   LEU A  52       0.074   6.592  18.303  1.00  0.84           C  \\nATOM    417  CB  LEU A  52       1.703   7.727  16.780  1.00  0.84           C  \\nATOM    418  O   LEU A  52      -0.483   7.500  18.924  1.00  0.81           O  \\nATOM    419  CG  LEU A  52       3.135   8.132  16.428  1.00  0.79           C  \\nATOM    420  CD1 LEU A  52       3.155   8.942  15.136  1.00  0.73           C  \\nATOM    421  CD2 LEU A  52       3.764   8.922  17.571  1.00  0.74           C  \\nATOM    422  N   ARG A  53      -0.616   5.423  18.300  1.00  0.76           N  \\nATOM    423  CA  ARG A  53      -2.027   5.260  18.633  1.00  0.74           C  \\nATOM    424  C   ARG A  53      -2.212   4.248  19.758  1.00  0.73           C  \\nATOM    425  CB  ARG A  53      -2.823   4.825  17.400  1.00  0.71           C  \\nATOM    426  O   ARG A  53      -1.359   3.382  19.967  1.00  0.70           O  \\nATOM    427  CG  ARG A  53      -3.098   5.951  16.416  1.00  0.67           C  \\nATOM    428  CD  ARG A  53      -4.222   5.595  15.452  1.00  0.65           C  \\nATOM    429  NE  ARG A  53      -4.488   6.680  14.511  1.00  0.59           N  \\nATOM    430  NH1 ARG A  53      -6.382   5.710  13.610  1.00  0.52           N  \\nATOM    431  NH2 ARG A  53      -5.655   7.746  12.845  1.00  0.48           N  \\nATOM    432  CZ  ARG A  53      -5.508   6.709  13.658  1.00  0.63           C  \",\"pdb\");\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 1},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 2},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 3},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 4},{\"cartoon\": {\"color\": \"#3000cf\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 5},{\"cartoon\": {\"color\": \"#3000cf\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 6},{\"cartoon\": {\"color\": \"#3000cf\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 7},{\"cartoon\": {\"color\": \"#4000bf\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 8},{\"cartoon\": {\"color\": \"#5200ad\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 9},{\"cartoon\": {\"color\": \"#5100ae\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 10},{\"cartoon\": {\"color\": \"#8b0074\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 11},{\"cartoon\": {\"color\": \"#b4004b\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 12},{\"cartoon\": {\"color\": \"#b70048\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 13},{\"cartoon\": {\"color\": \"#b80047\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 14},{\"cartoon\": {\"color\": \"#7e0081\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 15},{\"cartoon\": {\"color\": \"#4500ba\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 16},{\"cartoon\": {\"color\": \"#4600b9\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 17},{\"cartoon\": {\"color\": \"#7d0082\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 18},{\"cartoon\": {\"color\": \"#b60049\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 19},{\"cartoon\": {\"color\": \"#b80047\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 20},{\"cartoon\": {\"color\": \"#80007f\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 21},{\"cartoon\": {\"color\": \"#5a00a5\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 22},{\"cartoon\": {\"color\": \"#2100de\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 23},{\"cartoon\": {\"color\": \"#2000df\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 24},{\"cartoon\": {\"color\": \"#2400db\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 25},{\"cartoon\": {\"color\": \"#1200ed\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 26},{\"cartoon\": {\"color\": \"#1200ed\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 27},{\"cartoon\": {\"color\": \"#1100ee\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 28},{\"cartoon\": {\"color\": \"#4500ba\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 29},{\"cartoon\": {\"color\": \"#7d0082\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 30},{\"cartoon\": {\"color\": \"#7e0081\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 31},{\"cartoon\": {\"color\": \"#b80047\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 32},{\"cartoon\": {\"color\": \"#80007f\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 33},{\"cartoon\": {\"color\": \"#5100ae\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 34},{\"cartoon\": {\"color\": \"#4f00b0\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 35},{\"cartoon\": {\"color\": \"#5000af\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 36},{\"cartoon\": {\"color\": \"#690096\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 37},{\"cartoon\": {\"color\": \"#60009f\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 38},{\"cartoon\": {\"color\": \"#9a0065\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 39},{\"cartoon\": {\"color\": \"#990066\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 40},{\"cartoon\": {\"color\": \"#94006b\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 41},{\"cartoon\": {\"color\": \"#95006a\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 42},{\"cartoon\": {\"color\": \"#94006b\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 43},{\"cartoon\": {\"color\": \"#890076\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 44},{\"cartoon\": {\"color\": \"#73008c\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 45},{\"cartoon\": {\"color\": \"#73008c\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 46},{\"cartoon\": {\"color\": \"#3a00c5\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 47},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 48},{\"cartoon\": {\"color\": \"#1100ee\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 49},{\"cartoon\": {\"color\": \"#1000ef\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 50},{\"cartoon\": {\"color\": \"#1000ef\"}});\n",
              "\tviewer_1730203807245728.setStyle({\"resi\": 51},{\"cartoon\": {\"color\": \"#1100ee\"}});\n",
              "\tviewer_1730203807245728.zoomTo();\n",
              "viewer_1730203807245728.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_65d412ce-3983-428b-9124-142c88b1ec22\", \"A.html\", 42818)"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "2024-10-29 12:10:07.253 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:07.254 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:07.255 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:07.258 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:07.261 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-10-29 12:10:07.264 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 600x100 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAfoAAABlCAYAAAClUiFrAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAAAWWElEQVR4nO3dfVTV9R0H8Pfl4V4gLgIaXECGCAFGpinpIUNGY4PcfNjOSVuGsNTWtJOD5UO5hmmgh7QsjkXDBO2wbGu4Y0ia2dRJuZ0hOI8yTEDTBrbOCh95vJ/9ofdyH34XuQRX+fl+ncO5/H7fp8/3w7184PrDn0ZEBERERKRKbjc7ACIiIho8LPREREQqxkJPRESkYiz0REREKsZCT0REpGIs9ERERCrGQk9ERKRiLPREREQq5jEYk7a1taGjo2MwpiYiIlI1rVYLLy+vAZtvwAt9W1sbIkeNQsv58wM9NRERkeoZDAY0NTUNWLEf8ELf0dGBlvPncfbkcfj5+l4/K4Dpf9o1/4e7pmPLR5s+5v+dVxSOrc+JUt9ex9uu04e1Hc0nRutjo9Gir02bZbxGo/J4y2PT50YHa1k+OpzH2LOmoz5K89uuYVQa21ubzbx28yms6XCfCsd2cUFhvn7E5TDHSvtU2P8N89bzPBK7NtiPVfpaO5rXtk3pdWa0ec2Yu14/b+ytr8LrzUEfpfnEZi3FWGxf032Jy8F817681rk1t6HnuGe4aTys1rROsfU5uzFW3yp64ri2pv1YsY1L4dg2HqOjvViuef2cbcrFHIVFH4s203nbp6QRNmuZPrEYZ/Oss3i0yLFtm8IeeuJS2N/1eE3xGG3jNK8jFn1t92nbV2GcQh9TTErjLfdidDDOuq9Fm8WaHRCUtbSgo6Pj1i30Jn56Pfz0+utHCt/8vkuht+qr8I3HdP6Ghd7JtftT6B0UrIEr9H3p299Cr7Av27G9/hBgM29f1rzRPpXmc7R2fwu9o8KsOJ/CGCfmsy/0CmO/S6FX3IPNa6aXwnzDQu/kfDcu9KIcx43i6kOh762gOizaGtu+9uPtC71C8dYor+10obctbkpF0sWFXpTigu2j2Mdj0Wa3BwdrGi0ebQuyfR+x6Ou4zXEfx4+O2qQPfRTnEeu2gcaL8YiIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxTwGa+ILFy8CItePpOdz0ynTJ2L5aNPHcrzdsfU5Uerb63jbdfqwtqP5xGh9bDRa9LVps4zXaFQeb3ls+tzoYC3LR4fzGHvWdNRHaX7bNYxKY3trs5nXbj6FNR3uU+HYLi4ozNePuBzmWGmfCvu/Yd56nkdi1wb7sUpfa0fz2rYpvc6MNq8Zc9fr54299VV4vTnoozSf2KylGIvta7ovcTmY79qX1zq35jb0HPcMN42H1ZrWKbY+ZzfG6ltFTxzX1rQfK7ZxKRzbxmN0tBfLNa+fs025mKOw6GPRZjpv+5Q0wmYt0ycW42yedRaPFjm2bVPYQ09cCvu7Hq8pHqNtnOZ1xKKv7T5t+yqMU+hjiklpvOVejA7GWfe1aLNYs6OnSA6YAS/0IgJfX1+Ex8QP9NRERESq5+vra/HD9Hc34IVeo9Hg0qVLOHv2LPz8/AZ6elJw4cIFhIeHM+cuxJy7HnPuesy565lyrtFoBmzOQXvr3s/Pj08MF2POXY85dz3m3PWY86GNF+MRERGpGAs9ERGRig14odfpdMjNzYVOpxvoqckB5tz1mHPXY85djzl3vcHIuUYG8tI+IiIiuqXwrXsiIiIVY6EnIiJSMRZ6IiIiFWOhJyIiUrF+FfpNmzZh1KhR8PLywuTJk/GPf/yj1/5/+tOfEBcXBy8vL4wdOxaVlZX9CvZ25kzOi4uLkZSUhICAAAQEBCA1NfWGXyOy5+zz3GT79u3QaDSYNWvW4AaoQs7m/Ntvv8XixYsREhICnU6HmJgYfn9xkrM537hxI2JjY+Ht7Y3w8HBkZ2ejra3NRdEObQcPHsT06dMRGhoKjUaDv/zlLzccs3//fkyYMAE6nQ7R0dEoLS11fmFx0vbt20Wr1cqWLVvk+PHjsnDhQvH395fz588r9q+qqhJ3d3cpKCiQEydOyG9/+1vx9PSUY8eOObv0bcvZnD/22GOyadMmqampkbq6OsnKypJhw4bJuXPnXBz50OVszk2ampokLCxMkpKSZObMma4JViWczXl7e7skJCTItGnT5NChQ9LU1CT79++X2tpaF0c+dDmb87KyMtHpdFJWViZNTU2yZ88eCQkJkezsbBdHPjRVVlbKypUrpby8XADIjh07eu3f2NgoPj4+kpOTIydOnJDCwkJxd3eX3bt3O7Wu04V+0qRJsnjxYvNxd3e3hIaGytq1axX7z549W3784x9bnZs8ebL88pe/dHbp25azObfV1dUler1etm7dOlghqk5/ct7V1SUPPPCAbN68WTIzM1noneRszt98800ZPXq0dHR0uCpE1XE254sXL5aHHnrI6lxOTo5MmTJlUONUo74U+mXLlkl8fLzVuTlz5khaWppTazn11n1HRweqq6uRmppqPufm5obU1FR89tlnimM+++wzq/4AkJaW5rA/WetPzm1duXIFnZ2dCAwMHKwwVaW/OV+9ejWCgoIwf/58V4SpKv3J+c6dO5GYmIjFixcjODgY99xzD/Lz89Hd3e2qsIe0/uT8gQceQHV1tfnt/cbGRlRWVmLatGkuifl2M1D106mb2nz99dfo7u5GcHCw1fng4GD8+9//VhzT0tKi2L+lpcWpQG9X/cm5reXLlyM0NNTuCUPK+pPzQ4cO4e2330Ztba0LIlSf/uS8sbERn3zyCebOnYvKykqcOnUKixYtQmdnJ3Jzc10R9pDWn5w/9thj+Prrr/Hggw9CRNDV1YWnnnoKzz//vCtCvu04qp8XLlzA1atX4e3t3ad5eNW9yq1btw7bt2/Hjh074OXldbPDUaWLFy8iIyMDxcXFGDFixM0O57ZhNBoRFBSE3//+95g4cSLmzJmDlStXoqio6GaHplr79+9Hfn4+3njjDRw5cgTl5eXYtWsX1qxZc7NDo1449Rv9iBEj4O7ujvPnz1udP3/+PAwGg+IYg8HgVH+y1p+cm6xfvx7r1q3Dxx9/jHvvvXcww1QVZ3Pe0NCA06dPY/r06eZzRqMRAODh4YH6+npERUUNbtBDXH+e5yEhIfD09IS7u7v53JgxY9DS0oKOjg5otdpBjXmo60/OX3jhBWRkZGDBggUAgLFjx+Ly5ct48sknsXLlSri58XfHgeSofvr5+fX5t3nAyd/otVotJk6ciH379pnPGY1G7Nu3D4mJiYpjEhMTrfoDwN69ex32J2v9yTkAFBQUYM2aNdi9ezcSEhJcEapqOJvzuLg4HDt2DLW1teaPGTNmICUlBbW1tQgPD3dl+ENSf57nU6ZMwalTp8w/VAHAyZMnERISwiLfB/3J+ZUrV+yKuekHLeFtUwbcgNVP564TvPbnGDqdTkpLS+XEiRPy5JNPir+/v7S0tIiISEZGhqxYscLcv6qqSjw8PGT9+vVSV1cnubm5/PM6Jzmb83Xr1olWq5X3339fmpubzR8XL168WVsYcpzNuS1ede88Z3P+xRdfiF6vl6efflrq6+uloqJCgoKC5KWXXrpZWxhynM15bm6u6PV6effdd6WxsVE++ugjiYqKktmzZ9+sLQwpFy9elJqaGqmpqREA8sorr0hNTY2cOXNGRERWrFghGRkZ5v6mP69bunSp1NXVyaZNm1zz53UiIoWFhfK9731PtFqtTJo0SQ4fPmxuS05OlszMTKv+f/zjHyUmJka0Wq3Ex8fLrl27+rPsbc2ZnEdERAgAu4/c3FzXBz6EOfs8t8RC3z/O5vzTTz+VyZMni06nk9GjR0teXp50dXW5OOqhzZmcd3Z2yqpVqyQqKkq8vLwkPDxcFi1aJN98843rAx+C/vrXvyp+bzblODMzU5KTk+3GjB8/XrRarYwePVpKSkqcXpe3qSUiIlIxXjlBRESkYiz0REREKsZCT0REpGIs9ERERCrGQk9ERKRiLPREREQqxkJPRESkYiz0RN9RVlYWZs2aZT7+/ve/j1//+tcuj2P//v3QaDT49ttvXb42Ed26WOhJlbKysqDRaKDRaKDVahEdHY3Vq1ejq6tr0NcuLy/v8928XF2cR40ahY0bNyq2nT59GhqNxuGtdktLS805dXNzw8iRI/GLX/wCX331lbnPgQMH8NBDDyEwMBA+Pj646667kJmZiY6ODocxHT16FDNmzEBQUBC8vLwwatQozJkzx2peIuo/FnpSrfT0dDQ3N+Pzzz/Hb37zG6xatQovv/yyYt/eCpGzAgMDodfrB2y+W4mfnx+am5tx7tw5FBcX48MPP0RGRgYA4MSJE0hPT0dCQgIOHjyIY8eOobCwEFqtFt3d3Yrz/fe//8UPfvADBAYGYs+ePairq0NJSQlCQ0Nx+fLlQdtHZ2fnoM1NdKthoSfV0ul0MBgMiIiIwK9+9SukpqZi586dAHrebs/Ly0NoaChiY2MBAGfPnsXs2bPh7++PwMBAzJw5E6dPnzbP2d3djZycHPj7+2P48OFYtmyZ3V27bN+6b29vx/LlyxEeHg6dTofo6Gi8/fbbOH36NFJSUgAAAQEB0Gg0yMrKAnDtLmJr165FZGQkvL29MW7cOLz//vtW61RWViImJgbe3t5ISUmxinOwaDQaGAwGhIaG4uGHH8YzzzyDjz/+GFevXsVHH30Eg8GAgoIC3HPPPYiKikJ6ejqKi4sd3lKzqqoKra2t2Lx5M+677z5ERkYiJSUFr776KiIjI839jh8/jp/85Cfw8/ODXq9HUlISGhoaAFzL1erVqzFy5EjodDqMHz8eu3fvNo81vVPx3nvvITk5GV5eXigrKwMAbN68GWPGjIGXlxfi4uLwxhtvDGL2iG4OFnq6bXh7e1v95r5v3z7U19dj7969qKioQGdnJ9LS0qDX6/G3v/0NVVVV8PX1RXp6unnchg0bUFpaii1btuDQoUP43//+hx07dvS67rx58/Duu+/i9ddfR11dHd566y34+voiPDwcf/7znwEA9fX1aG5uxmuvvQYAWLt2LbZt24aioiIcP34c2dnZePzxx3HgwAEA134g+dnPfobp06ejtrYWCxYswIoVKwYjbb3y9vaG0WhEV1cXDAYDmpubcfDgwT6PNxgM6Orqwo4dOxze5vTLL7/E1KlTodPp8Mknn6C6uhpPPPGE+Z9hXnvtNWzYsAHr16/Hv/71L6SlpWHGjBn4/PPPreZZsWIFlixZgrq6OqSlpaGsrAy/+93vkJeXh7q6OuTn5+OFF17A1q1b+58QolvRd7sXD9GtyfLucUajUfbu3Ss6nU6effZZc3twcLC0t7ebx7zzzjsSGxsrRqPRfK69vV28vb1lz549IiISEhIiBQUF5vbOzk4ZOXKk1Z3qkpOTZcmSJSIiUl9fLwBk7969inGa7mZlefevtrY28fHxkU8//dSq7/z58+XnP/+5iIg899xzcvfdd1u1L1++3G4uWxEREfLqq68qtjU1NQkAqampUWwvKSmRYcOGmY9PnjwpMTExkpCQICIiXV1dkpWVJQDEYDDIrFmzpLCwUFpbWx3GIyLy/PPPi4eHhwQGBkp6eroUFBSYb5Nq2mtkZKR0dHQojg8NDZW8vDyrc/fff78sWrTIal8bN2606hMVFSV/+MMfrM6tWbNGEhMTe42XaKjhb/SkWhUVFfD19YWXlxcefvhhzJkzB6tWrTK3jx07Flqt1nx89OhRnDp1Cnq9Hr6+vvD19UVgYCDa2trQ0NCA1tZWNDc3Y/LkyeYxHh4eSEhIcBhDbW0t3N3dkZyc3Oe4T506hStXruCHP/yhOQ5fX19s27bN/HZ1XV2dVRwAkJiY2Oc1+qu1tRW+vr7w8fFBbGwsgoODzW+Du7u7o6SkBOfOnUNBQQHCwsKQn5+P+Ph4NDc3O5wzLy8PLS0tKCoqQnx8PIqKihAXF4djx44BuJbDpKQkeHp62o29cOEC/vOf/2DKlClW56dMmYK6ujqrc5Zfp8uXL6OhoQHz58+3yvFLL71kzjGRWnjc7ACIBktKSgrefPNNaLVahIaGwsPD+ul+xx13WB1funQJEydONBcuS3feeWe/YnD0b9O9uXTpEgBg165dCAsLs2rT6XT9imOg6PV6HDlyBG5ubggJCVHcX1hYGDIyMpCRkYE1a9YgJiYGRUVFePHFFx3OO3z4cDzyyCN45JFHkJ+fj/vuuw/r16/H1q1b+5VDJZZfb1OOi4uL7X5gcnd3H5D1iG4VLPSkWnfccQeio6P73H/ChAl47733EBQUBD8/P8U+ISEh+Pvf/46pU6cCALq6ulBdXY0JEyYo9h87diyMRiMOHDiA1NRUu3bTOwqWV6Xffffd0Ol0+OKLLxy+EzBmzBjzhYUmhw8fvvEmvyM3NzenchoQEICQkBCnrqDXarWIiooyj7n33nuxdetWdHZ22v1W7+fnh9DQUFRVVVnlqqqqCpMmTXK4RnBwMEJDQ9HY2Ii5c+f2OTaioYiFnui6uXPn4uWXX8bMmTPNV3GfOXMG5eXlWLZsGUaOHIklS5Zg3bp1uOuuuxAXF4dXXnml17+BHzVqFDIzM/HEE0/g9ddfx7hx43DmzBl89dVXmD17NiIiIqDRaFBRUYFp06bB29sber0ezz77LLKzs2E0GvHggw+itbUVVVVV8PPzQ2ZmJp566ils2LABS5cuxYIFC1BdXY3S0tI+7fPLL7+0+1v5iIgI8+f19fV2Y+Lj428471tvvYXa2lr89Kc/RVRUFNra2rBt2zYcP34chYWFimMqKiqwfft2PProo4iJiYGI4IMPPkBlZSVKSkoAAE8//TQKCwvx6KOP4rnnnsOwYcNw+PBhTJo0CbGxsVi6dClyc3MRFRWF8ePHo6SkBLW1tYrvzFh68cUX8cwzz2DYsGFIT09He3s7/vnPf+Kbb75BTk7ODfdLNGTc7IsEiAaD5cV4zrQ3NzfLvHnzZMSIEaLT6WT06NGycOFC8wVlnZ2dsmTJEvHz8xN/f3/JycmRefPmObwYT0Tk6tWrkp2dLSEhIaLVaiU6Olq2bNlibl+9erUYDAbRaDSSmZkpItcuINy4caPExsaKp6en3HnnnZKWliYHDhwwj/vggw8kOjpadDqdJCUlyZYtW/p0MR4Au4933nnHfNGa0sfZs2ftLsazdeTIEXn88cclMjJSdDqdDB8+XKZOnSo7d+50OKahoUEWLlwoMTEx4u3tLf7+/nL//fdLSUmJVb+jR4/Kj370I/Hx8RG9Xi9JSUnS0NAgIiLd3d2yatUqCQsLE09PTxk3bpx8+OGH5rG9XWRYVlYm48ePF61WKwEBATJ16lQpLy93GC/RUKQRcfA3LURERDTk8ap7IiIiFWOhJyIiUjEWeiIiIhVjoSciIlIxFnoiIiIVY6EnIiJSMRZ6IiIiFWOhJyIiUjEWeiIiIhVjoSciIlIxFnoiIiIVY6EnIiJSsf8DMIiDYEQUuXYAAAAASUVORK5CYII=\n"
          },
          "metadata": {}
        }
      ],
      "source": [
        "#@title plot\n",
        "import matplotlib.pyplot as plt\n",
        "import streamlit as st\n",
        "from stmol import showmol\n",
        "import py3Dmol\n",
        "import requests\n",
        "import biotite.structure.io as bsio\n",
        "import certifi\n",
        "import matplotlib.colors as mcolors\n",
        "import urllib3\n",
        "\n",
        "if Plot == True and Sequence != \"\" and Sequence != None:\n",
        "  fig1.show()\n",
        "\n",
        "\n",
        "  urllib3.disable_warnings()\n",
        "\n",
        "  st.sidebar.title('üéà ESMFold')\n",
        "  st.sidebar.write('[*ESMFold*](https://esmatlas.com/about) is an end-to-end single sequence protein structure predictor based on the ESM-2 language model. For more information, read the [research article](https://www.biorxiv.org/content/10.1101/2022.07.20.500902v2) and the [news article](https://www.nature.com/articles/d41586-022-03539-1) published in *Nature*.')\n",
        "\n",
        "  def render_mol(pdb):\n",
        "    pdbview = py3Dmol.view()\n",
        "    pdbview.addModel(pdb, 'pdb')\n",
        "    pdbview.setStyle({'cartoon': {'color': 'spectrum'}})\n",
        "    pdbview.setBackgroundColor('white')  # ('0xeeeeee')\n",
        "    pdbview.zoomTo()\n",
        "    pdbview.zoom(2, 800)\n",
        "    pdbview.spin(True)\n",
        "    showmol(pdbview, height=500, width=800)\n",
        "\n",
        "\n",
        "  DEFAULT_SEQ = Sequence\n",
        "  show_mainchains = True #@param {type: \"boolean\"}\n",
        "  show_sidechains = True #@param {type: \"boolean\"}\n",
        "  Color_Style =  \"red_blue_spectrum\" #@param [\"red_blue_spectrum\",\"green_magenta_spectrum\",\"yellow_blue_spectrum\",\"cyan_red_spectrum\"]\n",
        "\n",
        "  import streamlit as st\n",
        "  import requests\n",
        "  import biotite.structure.io as bsio\n",
        "  import py3Dmol\n",
        "\n",
        "  DEFAULT_SEQ = Sequence[:400]\n",
        "\n",
        "  txt = st.sidebar.text_area('Input sequence', DEFAULT_SEQ, height=275)\n",
        "\n",
        "  def update(sequence,name):\n",
        "    headers = {\n",
        "        'Content-Type': 'application/x-www-form-urlencoded',\n",
        "    }\n",
        "\n",
        "    def get_pdb(sequence):\n",
        "        response = requests.post('https://api.esmatlas.com/foldSequence/v1/pdb/', headers=headers, data=sequence, verify=False)\n",
        "        if response.status_code != 200:\n",
        "            st.error(f\"Failed to get prediction: {response.status_code}\")\n",
        "            return None\n",
        "        pdb_string = response.content.decode('utf-8')\n",
        "        if not pdb_string or \"ATOM\" not in pdb_string:\n",
        "            st.error(\"Received an invalid PDB file.\")\n",
        "            return None\n",
        "        return pdb_string\n",
        "\n",
        "    def save_and_render_pdb(pdb_string, file_name):\n",
        "        with open(file_name, 'w') as f:\n",
        "            f.write(pdb_string)\n",
        "        try:\n",
        "            struct = bsio.load_structure(file_name, extra_fields=[\"b_factor\"])\n",
        "            b_value = round(struct.b_factor.mean(), 4)\n",
        "        except Exception as e:\n",
        "            st.error(f\"Error loading structure: {e}\")\n",
        "            return\n",
        "        st.subheader('Visualization of predicted protein structure')\n",
        "        render_mol(pdb_string)\n",
        "        st.subheader('plDDT')\n",
        "        st.write('plDDT is a per-residue estimate of the confidence in prediction on a scale from 0-100.')\n",
        "        st.info(f'plDDT: {b_value}')\n",
        "        st.download_button(label=\"Download PDB\", data=pdb_string, file_name=file_name, mime='text/plain')\n",
        "\n",
        "    def render_mol(pdb_string,neme):\n",
        "        view = py3Dmol.view(width=1000, height=800)\n",
        "        view.addModel(pdb_string, 'pdb')\n",
        "        F1 = Get_color(p)\n",
        "        F1.i = F1.J.index(Color_Style)\n",
        "\n",
        "        custom_colors=F1.spectrum()\n",
        "\n",
        "        for i, color in enumerate(custom_colors[1:-1]):\n",
        "          view.setStyle({ 'resi': i+1}, {'cartoon': {'color': color}})\n",
        "        #view.setStyle({'cartoon': {'color': 'spectrum'}})\n",
        "        view.zoomTo()\n",
        "        view.show()\n",
        "        html_content = view._make_html()\n",
        "        with open(name+'.html', 'w') as file:\n",
        "          file.write(html_content)\n",
        "        files.download(name+'.html')\n",
        "\n",
        "\n",
        "    chunk_size = 400\n",
        "    chunks = [sequence[i:i + chunk_size] for i in range(0, len(sequence), chunk_size)]\n",
        "\n",
        "    combined_pdb_string = \"\"\n",
        "    for i, chunk in enumerate(chunks):\n",
        "        st.write(f\"Processing chunk {i + 1} of {len(chunks)}\")\n",
        "        pdb_string = get_pdb(chunk)\n",
        "        if pdb_string:\n",
        "            combined_pdb_string += pdb_string\n",
        "\n",
        "    if combined_pdb_string:\n",
        "        render_mol(combined_pdb_string,name)\n",
        "        with open('combined_predicted.pdb', 'w') as f:\n",
        "            f.write(combined_pdb_string)\n",
        "        st.download_button(label=\"Download Combined PDB\", data=combined_pdb_string, file_name='combined_predicted.pdb', mime='text/plain')\n",
        "\n",
        "  predict = st.sidebar.button('Predict', on_click=update)\n",
        "\n",
        "  if not predict:\n",
        "    st.warning('üëà Enter protein sequence data!')\n",
        "\n",
        "  class Get_color():\n",
        "    def __init__(self,values):\n",
        "      self.J = [\"red_blue_spectrum\",\"green_magenta_spectrum\",\"yellow_blue_spectrum\",\"cyan_red_spectrum\"]\n",
        "      self.i = None\n",
        "      self.values = values\n",
        "\n",
        "    def get_red_blue_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"RedBlue\", [\"blue\", \"red\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "    def get_green_magenta_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"GreenMagenta\", [\"green\", \"magenta\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "    def get_yellow_blue_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"YellowBlue\", [\"yellow\", \"blue\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "\n",
        "    def get_cyan_red_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"CyanRed\", [\"cyan\", \"red\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "    def spectrum(self):\n",
        "      if self.i == 0 : return self.get_red_blue_spectrum(self.values)\n",
        "      if self.i == 1 : return self.get_green_magenta_spectrum(self.values)\n",
        "      if self.i == 2 : return self.get_yellow_blue_spectrum(self.values)\n",
        "      if self.i == 3 : return self.get_cyan_red_spectrum(self.values)\n",
        "  update(txt,\"A\")\n",
        "\n",
        "\n",
        "# Create a color bar legend using matplotlib\n",
        "  fig, ax = plt.subplots(figsize=(6, 1))\n",
        "  fig.subplots_adjust(bottom=0.5)\n",
        "\n",
        "  cmap = plt.cm.Reds\n",
        "  norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "  fig.colorbar(plt.cm.ScalarMappable(norm=norm, cmap=cmap), cax=ax, orientation='horizontal', label='Predicted LLPS Score')\n",
        "  plt.show()\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "Sequence"
      ],
      "metadata": {
        "id": "72zztIXWJSkq",
        "outputId": "b1922f86-71bf-489c-b673-9b4547f9b85a",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 36
        }
      },
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'MVPSAGQLALFALGIVLAACQALENSTSPLSADPPVAAAVVSHFNDCPDSHTQFCFHGTCRFLVQEDKPACVCHSGYVGARCEHADLLAVVAASQKKQAITALVVVSIVALAVLIITCVLIHCCQVRKHCEWCRALICRHEKPSALLKGRTACCHSETVV'"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            }
          },
          "metadata": {},
          "execution_count": 6
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "3XzZgNVFM0G-"
      },
      "outputs": [],
      "source": [
        "#@title Downloading the results`\n",
        "\n",
        "if os.path.exists(\"d\"):\n",
        "    !rm -rf \"d\"\n",
        "if os.path.exists(\"text_dir\"):\n",
        "    !rm -rf \"text_dir\"\n",
        "\n",
        "!mkdir text_dir\n",
        "!mkdir d\n",
        "c = PPA(Sequence , model3)\n",
        "if Sequence != None and Sequence !=\"\" :\n",
        "  if ID == None:\n",
        "    c.name = \"Your peptide\"\n",
        "  with open(\"seq.txt\",\"w\") as w:\n",
        "          w.write(\"                                                 <<<<<<  LLPS prediction results  >>>>>>\\n\\n\\n\")\n",
        "          w.write(\"[Sequence id] : \" +c.name+\"\\n\\n\")\n",
        "          w.write(\"[Sequence length] : \"+str(len(c.seq))+\"\\n\\n\")\n",
        "          w.write(\"[Sequence_of_peptide] : \" +c.seq+\"\\n\\n\")\n",
        "          w.write(\"[Predicted_score] : \" +score +\"\\n\\n\")\n",
        "          w.close()\n",
        "\n",
        "\n",
        "\n",
        "  if c.name != None:\n",
        "    img_name = \"Info_of_sequenceID:\"+c.name+\".png\"\n",
        "    fig1.write_image(img_name)\n",
        "  else:\n",
        "    name = np.random.randint(1000000)\n",
        "    img_name = \"Info_of_sequenceID:\"+str(name)+\".png\"\n",
        "    fig1.write_image(img_name)\n",
        "\n",
        "  shutil.move(img_name, \"d\")\n",
        "  shutil.move(\"seq.txt\", \"text_dir\")\n",
        "  text_files_path = \"text_dir\"\n",
        "  png_files_path = \"d\"\n",
        "\n",
        "  zip_output_path = 'zipped_files.zip'\n",
        "\n",
        "  text_files = [f for f in os.listdir(text_files_path) if f.endswith('.txt')]\n",
        "  png_files = [f for f in os.listdir(png_files_path) if f.endswith('.png')]\n",
        "\n",
        "  with zipfile.ZipFile(zip_output_path, 'w') as zipf:\n",
        "    for text_file in text_files:\n",
        "      zipf.write(os.path.join(text_files_path, text_file), arcname=text_file)\n",
        "\n",
        "    for png_file in png_files:\n",
        "      zipf.write(os.path.join(png_files_path, png_file), arcname=png_file)\n",
        "\n",
        "  print(\"Files zipped successfully!\")\n",
        "\n",
        "\n",
        "else:\n",
        "  h = pd.DataFrame(D)\n",
        "  h.to_json(\"h.json\")\n",
        "  with open(\"results.txt\",\"w\") as w:\n",
        "    w.write(\"                                                 <<<<<<  LLPS prediction results  >>>>>>\\n\\n\\n\")\n",
        "    w.write(\"Number of sequences : \"+str(len(D[\"seq\"]))+\"\\n\\n\")\n",
        "    for i in range(len(D[\"id\"])):\n",
        "      w.write(\" [Sequence {i}]\".format(i = i)+\"\\n\\n\")\n",
        "      w.write(\"   Sequence id : \"+D[\"id\"][i]+\"\\n\\n\")\n",
        "      w.write(\"   Sequence length : \"+str(len(D[\"seq\"][i]))+\"\\n\\n\")\n",
        "      w.write(\"   Sequence : \" + D[\"seq\"][i]+\"\\n\\n\")\n",
        "      w.write(\"   Predicted Score : \" + str(D[\"score\"][i])+\"\\n\\n\")\n",
        "      w.write(\"-\"*150+\"\\n\\n\")\n",
        "    w.close()\n",
        "    shutil.move(\"h.json\", \"d\")\n",
        "    shutil.move(\"results.txt\", \"text_dir\")\n",
        "    text_files_path = \"text_dir\"\n",
        "    png_files_path = \"d\"\n",
        "    zip_output_path = 'zipped_files.zip'\n",
        "\n",
        "    text_files = [f for f in os.listdir(text_files_path) if f.endswith('.txt')]\n",
        "    png_files = [f for f in os.listdir(png_files_path) if f.endswith('.json')]\n",
        "\n",
        "    with zipfile.ZipFile(zip_output_path, 'w') as zipf:\n",
        "      for text_file in text_files:\n",
        "        zipf.write(os.path.join(text_files_path, text_file), arcname=text_file)\n",
        "\n",
        "      for png_file in png_files:\n",
        "        zipf.write(os.path.join(png_files_path, png_file), arcname=png_file)\n",
        "\n",
        "    print(\"Files zipped successfully!\")\n",
        "\n",
        "files.download(\"zipped_files.zip\")\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "tDV8YK1JM457"
      },
      "source": [
        "##NCBI BLAST+\n",
        "To determine the BLAST database, you can either provide a FASTA file path to create your own database or use the URL link to NCBI's protein databases available on the NCBI page: https://ftp.ncbi.nlm.nih.gov/blast/db/\n",
        "\n",
        "The DrLLPS database has already been provided. By setting the Blast_Database to DrLLPS, the BLAST search will be conducted against the DrLLPS database.\n",
        "\n",
        "To load your sequences, you can either input a single sequence or upload a FASTA format file containing your sequences into the designated box\n",
        "\n",
        "as example :\n",
        "  **MAVRVAKMTEREKLLLKKYGNRRLYDTKRSAYVTLSQVADIIKQGVDVEVVDAKTQEDVTAFILTQIVLEQARRKNALLPAELLHLIIRYGEDLLQEFFQKYLEQTIHGYIQFKKAMDEQFSKWIDMQLNYSDMAHKTLSGLTRFPNIFSPPPGSPGTKKDKG**\n",
        "\n",
        "or\n",
        "\n",
        "**Your_file.fasta**\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "zf3DjMfIM32y",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "df0066dd-09d0-4872-f35d-cc4d14ab4a60"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Cloning into 'bioinformatics_stockholm'...\n",
            "remote: Enumerating objects: 296, done.\u001b[K\n",
            "remote: Total 296 (delta 0), reused 0 (delta 0), pack-reused 296 (from 1)\u001b[K\n",
            "Receiving objects: 100% (296/296), 42.10 MiB | 13.85 MiB/s, done.\n",
            "Resolving deltas: 100% (127/127), done.\n",
            "\n",
            "\n",
            "Building a new DB, current time: 09/30/2024 08:13:12\n",
            "New DB name:   /content/blastdb\n",
            "New DB title:  DrLLPS.fasta\n",
            "Sequence type: Protein\n",
            "Keep MBits: T\n",
            "Maximum file size: 1000000000B\n",
            "Adding sequences from FASTA; added 1272 sequences in 0.0305941 seconds.\n",
            "\n",
            "\n",
            "-- Blast Database has been created -- \n",
            "\n"
          ]
        }
      ],
      "source": [
        "#@title BLAST+ Tool\n",
        "\n",
        "\n",
        "import os\n",
        "#@markdown - Specify the sequence or the path of your FASTA format file in this box : like  `MAVRVAKMTEREKLL`  or `Your_file.fasta`\n",
        "\n",
        "Sequence = \"MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGQQQQQQWQWQWWQQQQQQQWWWWWWWWWWWWWWPPPPPPPPPPPPPQQQQPPPPPPPPPPPPPQQPW\" #@param {type:\"string\"}\n",
        "#@markdown - Specify the fasta file or URL of your Database here :                    like  `https://ftp.ncbi.nlm.nih.gov/blast/db/swissprot.tar.gz`  or `Your_Database.fasta`\n",
        "\n",
        "Blast_Database = \"DrLLPS\" #@param [\"DrLLPS\",\"https://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/swissprot.gz\"] {allow-input : true}\n",
        "Blast_result_format = \"6\" #@param [1,2,3,4,5,6,7]\n",
        "Blast_result_file_name = \"Blast_result.txt\" #@param {type:\"string\"}\n",
        "E_value_threshold = \"1e-9\" #@param {type:\"string\"}\n",
        "Max_target_Sequences = 6 #@param {type:\"integer\"}\n",
        "\n",
        "if Blast_Database == \"DrLLPS\":\n",
        "  Blast_Database = \"DrLLPS.fasta\"\n",
        "\n",
        "import sys\n",
        "from os import path\n",
        "\n",
        "if path.exists(\"/content/bioinformatics_stockholm/\"):\n",
        "  pass\n",
        "else:\n",
        "  !git clone https://github.com/Intertangler/bioinformatics_stockholm\n",
        "sys.path.insert(0,'/content')\n",
        "\n",
        "import importlib.util\n",
        "spec = importlib.util.spec_from_file_location(\"msa\", \"/content/bioinformatics_stockholm/msa/msa.py\")\n",
        "msa = importlib.util.module_from_spec(spec)\n",
        "spec.loader.exec_module(msa)\n",
        "\n",
        "spec = importlib.util.spec_from_file_location(\"miniblast\", \"/content/bioinformatics_stockholm/Miniblast/miniblast.py\")\n",
        "miniblast = importlib.util.module_from_spec(spec)\n",
        "spec.loader.exec_module(miniblast)\n",
        "\n",
        "def Fasta(fasta_path):\n",
        "  from command_runner import command_runner\n",
        "  command = \"makeblastdb -in {k} -dbtype prot -out blastdb\".format(k =fasta_path )\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  print(\"-- Blast Database has been created -- \\n\")\n",
        "\n",
        "\n",
        "if \".fasta\" in Blast_Database or \".txt\" in Blast_Database:\n",
        "  if os.path.exists(Blast_Database):\n",
        "    Fasta(Blast_Database)\n",
        "  with open(Blast_Database) as w:\n",
        "    j = w.read().split(\">\")[1:]\n",
        "  Dict = {i[:i.index(\"\\n\")].split(\" \")[0] if \"\\n\" in i else i : edit(i[i.index(\"\\n\"):]) if \"\\n\" in i else None for i in j}\n",
        "\n",
        "else:\n",
        "  command = \"wget {k}\".format(k = Blast_Database)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  print(\"-- Blast Database has been downloaded -- \\n\")\n",
        "  exit_code, output = command_runner(\"gunzip {l}\".format(l = Blast_Database.split(\"/\")[-1]), shell=True, live_output=True)\n",
        "  Fasta(Blast_Database.split(\"/\")[-1][:-3])\n",
        "  db = miniblast.read_database_fasta(Blast_Database.split(\"/\")[-1][:-3])\n",
        "  Dict = {i.id :\"\".join(list(i.seq)) for i in db}\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "GWl1oETONCIj",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 766
        },
        "outputId": "aebf7c82-e950-470e-e355-26b15c962ada"
      },
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_2a0fd4c0-8727-4365-9dde-5df3c7311412\", \"Blast_result.txt\", 329)"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Subject_Id : seq1074\n",
            "\n",
            "-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGQQQQQQWQWQWWQQQQQQQWWWWWWWWWWWWWWPPPPPPPPPPPPPQQQQPPPPPPPPPPPPPQQPW------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                  ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||                                                                                                                                                 \n",
            "MKIEEGKLVIWINGDKGYNGLAEVGKKFEKDTGIKVTVEHPDKLEEKFPQVAATGDGPDIIFWAHDRFGGYAQSGLLAEITPDKAFQDKLYPFTWDAVRYNGKLIAYPIAVEALSLIYNKDLLPNPPKTWEEIPALDKELKAKGKSALMFNLQEPYFTWPLIAADGGYAFKYENGKYDIKDVGVDNAGAKAGLTFLVDLIKNKHMNADTDYSIAEAAFNKETAMTINGPWAWSNIDTSKVNYGVTVLPTFKGQPSKPFVGVLSAGINAASPNKELAKEFLENYLLTDEGLEAVNKDKPLGAVALKSYEEELAKDPRIAATMENAQKGEIMPNIPQMSAFWYAVRTAVINAASGRQTVDEALKDAQTNSSSNNNNNNNNNNLGIEGRASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGGGGGGGYNRSSGGYEPRGRGGGRGGRGGMGGSDRGGFNKFGGPRDQGSRHDSEQDNSDNNTIFVQGLGENVTIESVADYFKQIGIIKTNKKTGQPMINLYTDRETGKLKGEATVSFDDPPSAKAAIDWFDGKEFSGNPIKVSFAT\n",
            "  Score=1213\n",
            "\n",
            "Subject_Id : seq1213\n",
            "\n",
            "-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGQQQQQQWQWQWWQQQQQQQWWWWWWWWWWWWWWPPPPPPPPPPPPPQQQQPPPPPPPPPPPPPQQPW\n",
            "                                                                                                                                                                                      ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||                                                                   \n",
            "MDKDCEMKRTTLDSPLGKLELSGCEQGLHEIKLLGKGTSAADAVEVPAPAAVLGGPEPLMQATAWLNAYFHQPEAIEEFPVPALHHPVFQQESFTRQVLWKLLKVVKFGEVISYQQLAALAGNPAATAAVKTALSGNPVPILIPCHRVVSSSGAVGGYEGGLAVKEWLLAHEGHRLGKPGLGASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGGGGGGGYNRSSG-------------------------------------------------------\n",
            "  Score=1213\n",
            "\n",
            "Subject_Id : seq4\n",
            "\n",
            "MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGQQQQQQWQWQWWQQQQQQQWWWWWWWWWWWWWWPPPPPPPPPPPPPQQQQPPPPPPPPPPPPPQQPW----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           \n",
            "MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGGGGGGGYNRSSGGYEPRGRGGGRGGRGGMGGSDRGGFNKFGGPRDQGSRHDSEQDNSDNNTIFVQGLGENVTIESVADYFKQIGIIKTNKKTGQPMINLYTDRETGKLKGEATVSFDDPPSAKAAIDWFDGKEFSGNPIKVSFATRRADFNRGGGNGRGGRGRGGPMGRGGYGGGGSGGGGRGGFPSGGGGGGGQQRAGDWKCPNPTCENMNFSWRNECNQCKAPKPDGPGGGPGGSHMGGNYGDDRRGGRGGYDRGGYRGRGGDRGGFRGGRGGGDRGGFGPGKMDSRGEHRQDRRERLYVSKGEELFTGVVPILVELDGDVNGHKFSVSGEGEGDATYGKLTLKFICTTGKLPVPWPTLVTTLTYGVQCFSRYPDHMKQHDFFKSAMPEGYVQERTIFFKDDGNYKTRAEVKFEGDTLVNRIELKGIDFKEDGNILGHKLEYNYNSHNVYIMADKQKNGIKVNFKIRHNIEDGSVQLADHYQQNTPIGDGPVLLPDNHYLSTQSALSKDPNEKRDHMVLLEFVTAAGITLGMDELYK\n",
            "  Score=1218\n",
            "\n",
            "Subject_Id : seq1214\n",
            "\n",
            "MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGQQQQQQWQWQWWQQQQQQQWWWWWWWWWWWWWWPPPPPPPPPPPPPQQQQPPPPPPPPPPPPPQQPW----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           \n",
            "MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGGGGGGGYNRSSGGYEPRGRGGGRGGRGGMGGSDRGGFNKFGGPRDQGSRHDSEQDNSDNNTIFVQGLGENVTIESVADYFKQIGIIKTNKKTGQPMINLYTDRETGKLKGEATVSFDDPPSAKAAIDWFDGKEFSGNPIKVSFATRRADFNRGGGNGRGGRGRGGPMGRGGYGGGGSGGGGRGGFPSGGGGGGGQQRAGDWKCPNPTCENMNFSWRNECNQCKAPKPDGPGGGPGGSHMGGNYGDDRRGGRGGYDRGGYRGRGGDRGGFRGGRGGGDRGGFGPGKMDSRGEHRQDRRERPYVSKGEELFTGVVPILVELDGDVNGHKFSVSGEGEGDATYGKLTLKFICTTGKLPVPWPTLVTTLTYGVQCFSRYPDHMKQHDFFKSAMPEGYVQERTIFFKDDGNYKTRAEVKFEGDTLVNRIELKGIDFKEDGNILGHKLEYNYNSHNVYIMADKQKNGIKVNFKIRHNIEDGSVQLADHYQQNTPIGDGPVLLPDNHYLSTQSALSKDPNEKRDHMVLLEFVTAAGITLGMDELYK\n",
            "  Score=1218\n",
            "\n",
            "Subject_Id : seq559\n",
            "\n",
            "MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGQQQQQQWQWQWWQQQQQQQWWWWWWWWWWWWWWPPPPPPPPPPPPPQQQQPPPPPPPPPPPPPQQPW---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  \n",
            "MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGGGGGGGYNRSSGGYEPRGRGGGRGGRGGMGGSDRGGFNKFGGPRDQGSRHDSEQDNSDNNTIFVQGLGENVTIESVADYFKQIGIIKTNKKTGQPMINLYTDRETGKLKGEATVSFDDPPSAKAAIDWFDGKEFSGNPIKVSFATRRADFNRGGGNGRGGRGRGGPMGRGGYGGGGSGGGGRGGFPSGGGGGGGQQRAGDWKCPNPTCENMNFSWRNECNQCKAPKPDGPGGGPGGSHMGGNYGDDRRGGRGGYDRGGYRGRGGDRGGFRGGRGGGVSKGEELFTGVVPILVELDGDVNGHKFSVSGEGEGDATYGKLTLKFICTTGKLPVPWPTLVTTLTYGVQCFSRYPDHMKQHDFFKSAMPEGYVQERTIFFKDDGNYKTRAEVKFEGDTLVNRIELKGIDFKEDGNILGHKLEYNYNSHNVYIMADKQKNGIKVNFKIRHNIEDGSVQLADHYQQNTPIGDGPVLLPDNHYLSTQSALSKDPNEKRDHMVLLEFVTAAGITLGMDELYK\n",
            "  Score=1218\n",
            "\n",
            "Subject_Id : seq167\n",
            "\n",
            "MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGQQQQQQWQWQWWQQQQQQQWWWWWWWWWWWWWWPPPPPPPPPPPPPQQQQPPPPPPPPPPPPPQQPW----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           \n",
            "MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGGGGGGGYNRSSGGYEPRGCGGGRGGRGGMGGSDRGGFNKFGGPRDQGSRHDSEQDNSDNNTIFVQGLGENVTIESVADYFKQIGIIKTNKKTGQPMINLYTDRETGKLKGEATVSFDDPPSAKAAIDWFDGKEFSGNPIKVSFATRRADFNRGGGNGRGGRGRGGPMGRGGYGGGGSGGGGRGGFPSGGGGGGGQQRAGDWKCPNPTCENMNFSWRNECNQCKAPKPDGPGGGPGGSHMGGNYGDDRRGGRGGYDRGGYRGRGGDRGGFRGGRGGGDRGGFGPGKMDSRGEHRQDRRERPYVSKGEELFTGVVPILVELDGDVNGHKFSVSGEGEGDATYGKLTLKFICTTGKLPVPWPTLVTTLTYGVQCFSRYPDHMKQHDFFKSAMPEGYVQERTIFFKDDGNYKTRAEVKFEGDTLVNRIELKGIDFKEDGNILGHKLEYNYNSHNVYIMADKQKNGIKVNFKIRHNIEDGSVQLADHYQQNTPIGDGPVLLPDNHYLSTQSALSKDPNEKRDHMVLLEFVTAAGITLGMDELYK\n",
            "  Score=1218\n",
            "\n"
          ]
        }
      ],
      "source": [
        "#@title Blast Result\n",
        "\n",
        "blosum62 = {\n",
        "    ('W', 'F'): 1, ('L', 'R'): -2, ('S', 'P'): -1, ('V', 'T'): 0,\n",
        "    ('Q', 'Q'): 5, ('N', 'A'): -2, ('Z', 'Y'): -2, ('W', 'R'): -3,\n",
        "    ('Q', 'A'): -1, ('S', 'D'): 0, ('H', 'H'): 8, ('S', 'H'): -1,\n",
        "    ('H', 'D'): -1, ('L', 'N'): -3, ('W', 'A'): -3, ('Y', 'M'): -1,\n",
        "    ('G', 'R'): -2, ('Y', 'I'): -1, ('Y', 'E'): -2, ('B', 'Y'): -3,\n",
        "    ('Y', 'A'): -2, ('V', 'D'): -3, ('B', 'S'): 0, ('Y', 'Y'): 7,\n",
        "    ('G', 'N'): 0, ('E', 'C'): -4, ('Y', 'Q'): -1, ('Z', 'Z'): 4,\n",
        "    ('V', 'A'): 0, ('C', 'C'): 9, ('M', 'R'): -1, ('V', 'E'): -2,\n",
        "    ('T', 'N'): 0, ('P', 'P'): 7, ('V', 'I'): 3, ('V', 'S'): -2,\n",
        "    ('Z', 'P'): -1, ('V', 'M'): 1, ('T', 'F'): -2, ('V', 'Q'): -2,\n",
        "    ('K', 'K'): 5, ('P', 'D'): -1, ('I', 'H'): -3, ('I', 'D'): -3,\n",
        "    ('T', 'R'): -1, ('P', 'L'): -3, ('K', 'G'): -2, ('M', 'N'): -2,\n",
        "    ('P', 'H'): -2, ('F', 'Q'): -3, ('Z', 'G'): -2, ('X', 'L'): -1,\n",
        "    ('T', 'M'): -1, ('Z', 'C'): -3, ('X', 'H'): -1, ('D', 'R'): -2,\n",
        "    ('B', 'W'): -4, ('X', 'D'): -1, ('Z', 'K'): 1, ('F', 'A'): -2,\n",
        "    ('Z', 'W'): -3, ('F', 'E'): -3, ('D', 'N'): 1, ('B', 'K'): 0,\n",
        "    ('X', 'X'): -1, ('F', 'I'): 0, ('B', 'G'): -1, ('X', 'T'): 0,\n",
        "    ('F', 'M'): 0, ('B', 'C'): -3, ('Z', 'I'): -3, ('Z', 'V'): -2,\n",
        "    ('S', 'S'): 4, ('L', 'Q'): -2, ('W', 'E'): -3, ('Q', 'R'): 1,\n",
        "    ('N', 'N'): 6, ('W', 'M'): -1, ('Q', 'C'): -3, ('W', 'I'): -3,\n",
        "    ('S', 'C'): -1, ('L', 'A'): -1, ('S', 'G'): 0, ('L', 'E'): -3,\n",
        "    ('W', 'Q'): -2, ('H', 'G'): -2, ('S', 'K'): 0, ('Q', 'N'): 0,\n",
        "    ('N', 'R'): 0, ('H', 'C'): -3, ('Y', 'N'): -2, ('G', 'Q'): -2,\n",
        "    ('Y', 'F'): 3, ('C', 'A'): 0, ('V', 'L'): 1, ('G', 'E'): -2,\n",
        "    ('G', 'A'): 0, ('K', 'R'): 2, ('E', 'D'): 2, ('Y', 'R'): -2,\n",
        "    ('M', 'Q'): 0, ('T', 'I'): -1, ('C', 'D'): -3, ('V', 'F'): -1,\n",
        "    ('T', 'A'): 0, ('T', 'P'): -1, ('B', 'P'): -2, ('T', 'E'): -1,\n",
        "    ('V', 'N'): -3, ('P', 'G'): -2, ('M', 'A'): -1, ('K', 'H'): -1,\n",
        "    ('V', 'R'): -3, ('P', 'C'): -3, ('M', 'E'): -2, ('K', 'L'): -2,\n",
        "    ('V', 'V'): 4, ('M', 'I'): 1, ('T', 'Q'): -1, ('I', 'G'): -4,\n",
        "    ('P', 'K'): -1, ('M', 'M'): 5, ('K', 'D'): -1, ('I', 'C'): -1,\n",
        "    ('Z', 'D'): 1, ('F', 'R'): -3, ('X', 'K'): -1, ('Q', 'D'): 0,\n",
        "    ('X', 'G'): -1, ('Z', 'L'): -3, ('X', 'C'): -2, ('Z', 'H'): 0,\n",
        "    ('B', 'L'): -4, ('B', 'H'): 0, ('F', 'F'): 6, ('X', 'W'): -2,\n",
        "    ('B', 'D'): 4, ('D', 'A'): -2, ('S', 'L'): -2, ('X', 'S'): 0,\n",
        "    ('F', 'N'): -3, ('S', 'R'): -1, ('W', 'D'): -4, ('V', 'Y'): -1,\n",
        "    ('W', 'L'): -2, ('H', 'R'): 0, ('W', 'H'): -2, ('H', 'N'): 1,\n",
        "    ('W', 'T'): -2, ('T', 'T'): 5, ('S', 'F'): -2, ('W', 'P'): -4,\n",
        "    ('L', 'D'): -4, ('B', 'I'): -3, ('L', 'H'): -3, ('S', 'N'): 1,\n",
        "    ('B', 'T'): -1, ('L', 'L'): 4, ('Y', 'K'): -2, ('E', 'Q'): 2,\n",
        "    ('Y', 'G'): -3, ('Z', 'S'): 0, ('Y', 'C'): -2, ('G', 'D'): -1,\n",
        "    ('B', 'V'): -3, ('E', 'A'): -1, ('Y', 'W'): 2, ('E', 'E'): 5,\n",
        "    ('Y', 'S'): -2, ('C', 'N'): -3, ('V', 'C'): -1, ('T', 'H'): -2,\n",
        "    ('P', 'R'): -2, ('V', 'G'): -3, ('T', 'L'): -1, ('V', 'K'): -2,\n",
        "    ('K', 'Q'): 1, ('R', 'A'): -1, ('I', 'R'): -3, ('T', 'D'): -1,\n",
        "    ('P', 'F'): -4, ('I', 'N'): -3, ('K', 'I'): -3, ('M', 'D'): -3,\n",
        "    ('V', 'W'): -3, ('W', 'W'): 11, ('M', 'H'): -2, ('P', 'N'): -2,\n",
        "    ('K', 'A'): -1, ('M', 'L'): 2, ('K', 'E'): 1, ('Z', 'E'): 4,\n",
        "    ('X', 'N'): -1, ('Z', 'A'): -1, ('Z', 'M'): -1, ('X', 'F'): -1,\n",
        "    ('K', 'C'): -3, ('B', 'Q'): 0, ('X', 'B'): -1, ('B', 'M'): -3,\n",
        "    ('F', 'C'): -2, ('Z', 'Q'): 3, ('X', 'Z'): -1, ('F', 'G'): -3,\n",
        "    ('B', 'E'): 1, ('X', 'V'): -1, ('F', 'K'): -3, ('B', 'A'): -2,\n",
        "    ('X', 'R'): -1, ('D', 'D'): 6, ('W', 'G'): -2, ('Z', 'F'): -3,\n",
        "    ('S', 'Q'): 0, ('W', 'C'): -2, ('W', 'K'): -3, ('H', 'Q'): 0,\n",
        "    ('L', 'C'): -1, ('W', 'N'): -4, ('S', 'A'): 1, ('L', 'G'): -4,\n",
        "    ('W', 'S'): -3, ('S', 'E'): 0, ('H', 'E'): 0, ('S', 'I'): -2,\n",
        "    ('H', 'A'): -2, ('S', 'M'): -1, ('Y', 'L'): -1, ('Y', 'H'): 2,\n",
        "    ('Y', 'D'): -3, ('E', 'R'): 0, ('X', 'P'): -2, ('G', 'G'): 6,\n",
        "    ('G', 'C'): -3, ('E', 'N'): 0, ('Y', 'T'): -2, ('Y', 'P'): -3,\n",
        "    ('T', 'K'): -1, ('A', 'A'): 4, ('P', 'Q'): -1, ('T', 'C'): -1,\n",
        "    ('V', 'H'): -3, ('T', 'G'): -2, ('I', 'Q'): -3, ('Z', 'T'): -1,\n",
        "    ('C', 'R'): -3, ('V', 'P'): -2, ('P', 'E'): -1, ('M', 'C'): -1,\n",
        "    ('K', 'N'): 0, ('I', 'I'): 4, ('P', 'A'): -1, ('M', 'G'): -3,\n",
        "    ('T', 'S'): 1, ('I', 'E'): -3, ('P', 'M'): -2, ('M', 'K'): -1,\n",
        "    ('I', 'A'): -1, ('P', 'I'): -3, ('R', 'R'): 5, ('X', 'M'): -1,\n",
        "    ('L', 'I'): 2, ('X', 'I'): -1, ('Z', 'B'): 1, ('X', 'E'): -1,\n",
        "    ('Z', 'N'): 0, ('X', 'A'): 0, ('B', 'R'): -1, ('B', 'N'): 3,\n",
        "    ('F', 'D'): -3, ('X', 'Y'): -1, ('Z', 'R'): 0, ('F', 'H'): -1,\n",
        "    ('B', 'F'): -3, ('F', 'L'): 0, ('X', 'Q'): -1, ('B', 'B'): 4\n",
        "}\n",
        "\n",
        "Blosum62 = {}\n",
        "for i in blosum62:\n",
        "  Blosum62[i] = blosum62[i]\n",
        "  Blosum62[i[::-1]] = blosum62[i]\n",
        "if \".\" not in Sequence:\n",
        "  with open(\"sequence.txt\",\"w\") as w:\n",
        "    w.write(\">seq1\\n{o}\".format(o = Sequence))\n",
        "    w.close()\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = \"sequence.txt\",d = \"blastdb\"\n",
        "  ,f = Blast_result_format,o = Blast_result_file_name , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  files.download(Blast_result_file_name)\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = \"sequence.txt\",d = \"blastdb\"\n",
        "  ,f = 6 ,o = \"results_dict.txt\" , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "\n",
        "  with open(\"results_dict.txt\") as w:\n",
        "    y = w.read()\n",
        "  idx = [i.split(\"\\t\")[1] for i in y.split(\"\\nseq\")]\n",
        "  for i in idx:\n",
        "    target_sequence = Dict[i]\n",
        "    aln = pairwise2.align.localds(Sequence,target_sequence, match_dict = Blosum62, open = -3, extend = -1)[0]\n",
        "    print(\"Subject_Id : \"+i+\"\\n\")\n",
        "    print(pairwise2.format_alignment(align1=aln[0], align2=aln[1], score=aln[2], begin=aln[3], end=aln[4], full_sequences=True))\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "if \".\" in Sequence:\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = Sequence,d = \"blastdb\"\n",
        "  ,f = Blast_result_format,o = Blast_result_file_name , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  files.download(Blast_result_file_name)"
      ]
    }
  ],
  "metadata": {
    "accelerator": "GPU",
    "colab": {
      "gpuType": "T4",
      "provenance": [],
      "authorship_tag": "ABX9TyO3yz8bJF+CQ2p64q4vzIA0",
      "include_colab_link": true
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}