{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "authorship_tag": "ABX9TyN39Yg3oaN1utEBwI8xCGRJ",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/AMIRMOHAMMAD-OSS/Bioinformatics/blob/main/phase_seperation.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#Liquid-Liquid Phase Separation (LLPS) Protein Scorer!\n",
        "\n",
        "This tool allows you to analyze protein sequences for their propensity to undergo phase separation:\n",
        "\n",
        "Input Options:\n",
        "Upload a single protein sequence directly into the designated box.\n",
        "Alternatively, load a FASTA or TXT file containing multiple protein sequences.\n",
        "\n",
        "To maintain efficiency, we advise against loading more than 100 sequences at once\n",
        "\n",
        "Specify the sequence ID or directory for your analysis.\n",
        "\n",
        "By setting the 'End_Sequence'  parameter,you can focus the scoring on sequences up to the specified endpoint.\n",
        "\n",
        "         \n",
        "\n",
        "\n",
        "\n",
        "#BLAST+ tool\n",
        "You can utilize the BLAST+ tool to run ncbi BLAST searches on either your protein sequence or your FASTA file against your dataset. This powerful tool allows you to compare sequences and identify homologous regions, aiding in various biological analyses.\n"
      ],
      "metadata": {
        "id": "KUBBeMl67hBW"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "cellView": "form",
        "id": "rpsJRBHX7U3p",
        "outputId": "ab18eeec-f396-47be-9fef-b41a28f25922"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Everything looks ok! ‚ú®üç∞‚ú®\n"
          ]
        }
      ],
      "source": [
        "#@title Importing libraries\n",
        "from google.colab import output\n",
        "!git clone https://github.com/AMIRMOHAMMAD-OSS/classi\n",
        "output.clear()\n",
        "!pip install biopython\n",
        "output.clear()\n",
        "!apt-get update\n",
        "!apt-get install -y ncbi-blast+\n",
        "!pip install streamlit\n",
        "output.clear()\n",
        "\n",
        "!pip install py3Dmol\n",
        "output.clear()\n",
        "!pip install requests\n",
        "!pip install biotite\n",
        "!pip install stmol\n",
        "output.clear()\n",
        "!pip install command_runner\n",
        "output.clear()\n",
        "from command_runner import command_runner\n",
        "!pip install plotly\n",
        "!pip install --upgrade \"kaleido==0.1.*\"\n",
        "output.clear()\n",
        "from Bio import SeqIO\n",
        "from Bio.Seq import Seq\n",
        "from Bio import pairwise2\n",
        "from google.colab import files\n",
        "import plotly.graph_objects as go\n",
        "from plotly.subplots import make_subplots\n",
        "import zipfile\n",
        "import shutil\n",
        "from transformers import AutoTokenizer\n",
        "from tqdm import tqdm\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "import numpy as np\n",
        "import time\n",
        "from collections import defaultdict\n",
        "import torch\n",
        "from torch.utils.data.dataloader import DataLoader\n",
        "import time\n",
        "import os\n",
        "import sys\n",
        "import json\n",
        "import random\n",
        "from ast import literal_eval\n",
        "import numpy as np\n",
        "import torch\n",
        "from transformers import AutoTokenizer\n",
        "from transformers import AutoTokenizer\n",
        "from tokenizers import Tokenizer, decoders, models, normalizers, pre_tokenizers, trainers\n",
        "from transformers import CanineTokenizer, CanineModel\n",
        "tokenizer = Tokenizer.from_file(\"classi/Trained_BPE2.json\")\n",
        "tokenizer.model_max_length = 256\n",
        "tokenizer\n",
        "\n",
        "from sklearn.preprocessing import StandardScaler as Sc\n",
        "from sklearn.decomposition import PCA\n",
        "from sklearn.model_selection import train_test_split as TTS\n",
        "import numpy as np\n",
        "\n",
        "import torch\n",
        "\n",
        "from sklearn.model_selection import train_test_split as TTS\n",
        "\n",
        "\n",
        "def edit(x):\n",
        "    for i in x:\n",
        "      if i in \"BJOUX\\Z_n\\n\":\n",
        "        x = x.replace(i,\"\")\n",
        "      else:\n",
        "        pass\n",
        "    return x\n",
        "\n",
        "def encode(x):\n",
        "  l = []\n",
        "  chars = tokenizer.get_vocab()\n",
        "  for i in x:\n",
        "    l.append(chars[i])\n",
        "  return l\n",
        "\n",
        "with open(\"classi/training.txt\",\"r\",encoding=\"utf-8\") as f:\n",
        "  text = f.read()\n",
        "  k = \"#\".join([i for i in edit(text).split(\"<|edoftext|>\")])\n",
        "chars = tokenizer.get_vocab()\n",
        "vocab_size = len(tokenizer.get_vocab())\n",
        "\n",
        "with open(\"classi/validation.txt\",\"r\",encoding=\"utf-8\") as f:\n",
        "  text = f.read()\n",
        "  l = \"#\".join([i for i in edit(text).split(\"<|edoftext|>\")])\n",
        "\n",
        "def Padding(x,PAD = 0,max_len = 512):\n",
        "  l = []\n",
        "  max_len = max(512,len(max(x,key = lambda x: len(x))))\n",
        "  for i in x:\n",
        "    a = i\n",
        "    if len(a) < max_len:\n",
        "      a = a + [PAD for i in range(max_len-len(a))]\n",
        "    l.append(a)\n",
        "  return np.array(l).reshape(len(x),max_len)\n",
        "with open(\"classi/negative_dataset.txt\") as x:\n",
        "  c = x.read()\n",
        "Filtered_train  = [i for i in k.split(\"#\") if len(i)<= 512][1:]\n",
        "Filtered_val = [i for i in l.split(\"#\") if len(i)<= 512][1:]\n",
        "Filtered_neg = [i for i in c.split(\"\\n\") if len(i)<= 512]\n",
        "pos = Filtered_train + Filtered_val\n",
        "pos2 = list(set([i for i in k.split(\"#\")][1:] +  [i for i in l.split(\"#\")][1:]))\n",
        "with open(\"DrLLPS.fasta\",\"w\") as w:\n",
        "  for i in range(len(pos2)):\n",
        "    w.write(\">seq{i}\\n\".format(i=i))\n",
        "    w.write(pos2[i]+\"\\n\")\n",
        "  w.close()\n",
        "files.download(\"DrLLPS.fasta\")\n",
        "output.clear()\n",
        "#@title RR\n",
        "from tqdm import tqdm\n",
        "from multiprocessing import Pool\n",
        "from Bio import SeqIO\n",
        "from scipy.spatial.distance import pdist, squareform\n",
        "from scipy.sparse.linalg import eigs\n",
        "import pandas as pd\n",
        "import numpy as np\n",
        "from scipy.io import loadmat\n",
        "!unzip /content/classi/neg_feature_extracted_dataset.zip\n",
        "!unzip /content/classi/pos_feature_extracted_Oversampled_dataset.zip\n",
        "output.clear()\n",
        "\n",
        "n1 = pd.read_csv(\"classi/neg_feature_extracted_dataset(512_).csv\")\n",
        "n2 = pd.read_csv(\"/content/neg_feature_extracted_dataset.csv\")\n",
        "p1 = pd.read_csv(\"classi/pos_feature_extracted_dataset(512_).csv\")\n",
        "p2 = pd.read_csv(\"/content/pos_feature_extracted_Oversampled_dataset.csv\")\n",
        "\n",
        "def EDIT(s):\n",
        "  R = []\n",
        "  for i in s:\n",
        "    i = i[1:i.index(\"j\")-2]\n",
        "    if i == \"\":\n",
        "      i = float(0)\n",
        "    else:\n",
        "      i = float(i)\n",
        "    R.append(i)\n",
        "  return np.array(R).reshape(1,578)\n",
        "p1 = np.array([EDIT(i) for i in p1.iloc[:,1:].values.tolist()]).reshape(-1,578)\n",
        "p2 = np.array([EDIT(i) for i in p2.iloc[:,1:].values.tolist()]).reshape(-1,578)\n",
        "n1 = np.array([EDIT(i) for i in n1.iloc[:,1:].values.tolist()]).reshape(-1,578)\n",
        "n2 = np.array([EDIT(i) for i in n2.iloc[:,2:].values.tolist()]).reshape(-1,578)\n",
        "def coordinate():\n",
        "    pt = [np.array([np.cos(i * 2 * np.pi / 20), np.sin(i * 2 * np.pi / 20), 1]) for i in range(20)]\n",
        "    P = np.vstack(pt)\n",
        "    V = [[pt[i] + (1 / 20) * (pt[j] - pt[i]) for j in range(20)] for i in range(20)]\n",
        "    return P, V\n",
        "\n",
        "\n",
        "def FEGS(sequences):\n",
        "    P, V = coordinate()\n",
        "    #sequences = [str(record.seq) for record in SeqIO.parse(f\"{data}.fasta\", \"fasta\")]\n",
        "    l = len(sequences)\n",
        "\n",
        "    with Pool() as pool:\n",
        "        g_p = pool.starmap(GRS, [(seq, P, V) for seq in sequences])\n",
        "        EL = np.array([ME(i) for i in  [g for g_list in g_p for g in g_list]]).reshape(l,158)\n",
        "\n",
        "    char = 'ARNDCQEGHILKMFPSTWYV'\n",
        "    with Pool() as pool:\n",
        "        results = pool.map(SAD, [(seq, char) for seq in sequences])\n",
        "        FA = np.array([res[0] for res in results])\n",
        "        FD = np.array([res[1].flatten() for res in results])\n",
        "\n",
        "    FV = np.hstack((EL, FA, FD))\n",
        "    return FV\n",
        "\n",
        "\n",
        "def GRS(seq, P, V):\n",
        "    M = loadmat(\"/content/classi/M.mat\")\n",
        "    M = M[\"M\"].flatten()\n",
        "    l_seq = len(seq)\n",
        "    k = M.shape[0]\n",
        "    cha = 'ACDEFGHIKLMNPQRSTVWY'\n",
        "    g = []\n",
        "\n",
        "    for j in range(k):\n",
        "        DPC = np.zeros((20, 20))\n",
        "        c = [[0, 0, 0]]\n",
        "        d = np.zeros(3)\n",
        "        y = None\n",
        "\n",
        "        for i in range(l_seq):\n",
        "            x = np.array([seq[i] == aa for aa in M[j]]).astype(int)\n",
        "            if i == 0:\n",
        "                c.append(c[i] + np.dot(x, P))\n",
        "            else:\n",
        "                if not np.any(x):\n",
        "                    d = d * (i - 1) / i\n",
        "                    c.append(c[i] + [0, 0, 1] + d)\n",
        "                elif not np.any(y):\n",
        "                    d = d * (i - 1) / i\n",
        "                    c.append(c[i] + np.dot(x, P) + d)\n",
        "                else:\n",
        "                    d = d * (i - 1) / i + V[int(np.where(y)[0][0])][int(np.where(x)[0][0])] / i\n",
        "                    c.append(c[i] + np.dot(x, P) + d)\n",
        "            y = x\n",
        "\n",
        "        g.append(np.vstack(c))\n",
        "    return g\n",
        "\n",
        "def ME(W):\n",
        "    W = W[1:, :]\n",
        "    x = W.shape[0]\n",
        "    D = pdist(W)\n",
        "    E = squareform(D)\n",
        "    sdist = np.zeros((x, x))\n",
        "\n",
        "    for i in range(x):\n",
        "        for j in range(i, x):\n",
        "            if j - i == 1:\n",
        "                sdist[i, j] = E[i, j]\n",
        "            elif j - i > 1:\n",
        "                sdist[i, j] = sdist[i, j - 1] + E[j - 1, j]\n",
        "\n",
        "    sdist += sdist.T\n",
        "    sdd = sdist + np.diag(np.ones(x))\n",
        "    L = E / sdd\n",
        "    ME = eigs(L, k=1)[0][0] / x\n",
        "    return ME\n",
        "\n",
        "def SAD(args):\n",
        "    seq, a = args\n",
        "    len_seq = len(seq)\n",
        "    len_a = len(a)\n",
        "    AAC = np.zeros(len_a)\n",
        "    DPC = np.zeros((len_a, len_a))\n",
        "\n",
        "    c = [np.array([s == aa for s in seq]) for aa in a]\n",
        "    AAC = np.array([np.sum(c[i]) / len_seq for i in range(len_a)])\n",
        "\n",
        "    if len_seq != 1:\n",
        "        for i in range(len_a):\n",
        "            for j in range(len_a):\n",
        "                DPC[i, j] = np.sum((np.roll(c[j], -1) * 2 - c[i]) == 1) / (len_seq - 1)\n",
        "    return AAC, DPC\n",
        "\n",
        "from sklearn.model_selection import train_test_split\n",
        "Data = np.concatenate((p2,p1,n2,n1))\n",
        "labels = np.concatenate((np.ones(p2.shape[0]),np.ones(p1.shape[0]),np.zeros(n2.shape[0]),np.zeros(n1.shape[0])))\n",
        "train_data,test_data,train_l,test_l = train_test_split(Data,labels,test_size = 0.1,random_state = 48)\n",
        "from sklearn.ensemble import RandomForestClassifier\n",
        "from sklearn.metrics import accuracy_score,confusion_matrix\n",
        "\n",
        "r = RandomForestClassifier(n_estimators=100, random_state=48)\n",
        "r.fit(train_data,train_l)\n",
        "output.clear()\n",
        "\n",
        "class RF():\n",
        "  def __init__(self,seq):\n",
        "    self.model = r\n",
        "    self.seq = seq\n",
        "  def TR(self):\n",
        "    return FEGS(self.seq)\n",
        "  def Model(self):\n",
        "    return self.model\n",
        "  def predict_proba(self):\n",
        "    return self.model.predict_proba(self.TR().astype(np.float64))\n",
        "print(\"Everything looks ok! ‚ú®üç∞‚ú®\")"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Loading models and requierments\n",
        "if torch.cuda.is_available():\n",
        "  print(\"Cuda is available üéâ\\n\")\n",
        "  print(\"The allocated device is : \"+torch.cuda.get_device_name())\n",
        "  device = torch.device(\"cuda\")\n",
        "else:\n",
        "  print(\"Cuda is not available üòî\\n\")\n",
        "  print(\"The allocated device is : cpu\")\n",
        "  device = torch.device(\"cpu\")\n",
        "\n",
        "\n",
        "\n",
        "def set_seed(seed):\n",
        "    random.seed(seed)\n",
        "    np.random.seed(seed)\n",
        "    torch.manual_seed(seed)\n",
        "    torch.cuda.manual_seed_all(seed)\n",
        "\n",
        "def setup_logging(config):\n",
        "    work_dir = config.system.work_dir\n",
        "    os.makedirs(work_dir, exist_ok=True)\n",
        "    with open(os.path.join(work_dir, 'args.txt'), 'w') as f:\n",
        "        f.write(' '.join(sys.argv))\n",
        "    with open(os.path.join(work_dir, 'config.json'), 'w') as f:\n",
        "        f.write(json.dumps(config.to_dict(), indent=4))\n",
        "\n",
        "class CfgNode:\n",
        "    def __init__(self, **kwargs):\n",
        "        self.__dict__.update(kwargs)\n",
        "\n",
        "    def __str__(self):\n",
        "        return self._str_helper(0)\n",
        "\n",
        "    def _str_helper(self, indent):\n",
        "        parts = []\n",
        "        for k, v in self.__dict__.items():\n",
        "            if isinstance(v, CfgNode):\n",
        "                parts.append(\"%s:\\n\" % k)\n",
        "                parts.append(v._str_helper(indent + 1))\n",
        "            else:\n",
        "                parts.append(\"%s: %s\\n\" % (k, v))\n",
        "        parts = [' ' * (indent * 4) + p for p in parts]\n",
        "        return \"\".join(parts)\n",
        "\n",
        "    def to_dict(self):\n",
        "        return { k: v.to_dict() if isinstance(v, CfgNode) else v for k, v in self.__dict__.items() }\n",
        "\n",
        "    def merge_from_dict(self, d):\n",
        "        self.__dict__.update(d)\n",
        "\n",
        "    def merge_from_args(self, args):\n",
        "\n",
        "        for arg in args:\n",
        "            keyval = arg.split('=')\n",
        "            assert len(keyval) == 2, \"expecting each override arg to be of form --arg=value, got %s\" % arg\n",
        "            key, val = keyval\n",
        "            try:\n",
        "                val = literal_eval(val)\n",
        "            except ValueError:\n",
        "                pass\n",
        "\n",
        "            assert key[:2] == '--'\n",
        "            key = key[2:]\n",
        "            keys = key.split('.')\n",
        "            obj = self\n",
        "            for k in keys[:-1]:\n",
        "                obj = getattr(obj, k)\n",
        "            leaf_key = keys[-1]\n",
        "            assert hasattr(obj, leaf_key), f\"{key} is not an attribute that exists in the config\"\n",
        "            print(\"command line overwriting config attribute %s with %s\" % (key, val))\n",
        "            setattr(obj, leaf_key, val)\n",
        "\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "\n",
        "class SiLU(nn.Module):\n",
        "   def forward(self, x):\n",
        "        return x*F.sigmoid(x)\n",
        "\n",
        "class NY(nn.Module):\n",
        "  def forward(self,x):\n",
        "    return 3*torch.tanh(0.3*x)\n",
        "\n",
        "class NewGELU(nn.Module):\n",
        "    def forward(self, x):\n",
        "        return 0.5 * x * (1.0 + torch.tanh(math.sqrt(2.0 / math.pi) * (x + 0.044715 * torch.pow(x, 3.0))))\n",
        "\n",
        "class SelfAttention(nn.Module):\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        assert config.n_embd % config.n_head == 0\n",
        "        self.c_attn = nn.Linear(config.n_embd, 3 * config.n_embd)\n",
        "        self.c_proj = nn.Linear(config.n_embd, config.n_embd)\n",
        "        self.attn_dropout = nn.Dropout(config.attn_pdrop)\n",
        "        self.resid_dropout = nn.Dropout(config.resid_pdrop)\n",
        "        self.register_buffer(\"bias\", torch.tril(torch.ones(config.block_size, config.block_size))\n",
        "                                     .view(1, 1, config.block_size, config.block_size))\n",
        "        self.n_head = config.n_head\n",
        "        self.n_embd = config.n_embd\n",
        "\n",
        "    def forward(self, x):\n",
        "        B, T, C = x.size()\n",
        "        q, k ,v  = self.c_attn(x).split(self.n_embd, dim=2)\n",
        "        k = k.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        q = q.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        v = v.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        att = (q @ k.transpose(-2, -1)) * (1.0 / math.sqrt(k.size(-1)))\n",
        "        att = att.masked_fill(self.bias[:,:,:T,:T] == 0, float('-inf'))\n",
        "        att = F.softmax(att, dim=-1)\n",
        "        att = self.attn_dropout(att)\n",
        "        y = att @ v\n",
        "        y = y.transpose(1, 2).contiguous().view(B, T, C)\n",
        "        y = self.resid_dropout(self.c_proj(y))\n",
        "        return y\n",
        "\n",
        "class Block(nn.Module):\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.ln_1 = nn.LayerNorm(config.n_embd)\n",
        "        self.attn = SelfAttention(config)\n",
        "        self.ln_2 = nn.LayerNorm(config.n_embd)\n",
        "        self.mlp = nn.ModuleDict(dict(\n",
        "            c_fc    = nn.Linear(config.n_embd, 4 * config.n_embd),\n",
        "            c_proj  = nn.Linear(4 * config.n_embd, config.n_embd),\n",
        "            act     = NewGELU(),\n",
        "            dropout = nn.Dropout(config.resid_pdrop),\n",
        "        ))\n",
        "        m = self.mlp\n",
        "        self.mlpf = lambda x: m.dropout(m.c_proj(m.act(m.c_fc(x))))\n",
        "    def forward(self, x):\n",
        "        x = x + self.attn(self.ln_1(x))\n",
        "        x = x + self.mlpf(self.ln_2(x))\n",
        "        return x\n",
        "\n",
        "class ClassifierII(nn.Module):\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.model_type = 'gpt'\n",
        "        C.n_layer = None\n",
        "        C.n_head = None\n",
        "        C.n_embd =  None\n",
        "        C.vocab_size = len(chars)\n",
        "        C.max_length = 512\n",
        "        C.embd_pdrop = 0.1\n",
        "        C.resid_pdrop = 0.1\n",
        "        C.attn_pdrop = 0.1\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.max_length = config.block_size\n",
        "        self.soft = nn.Softmax(1)\n",
        "        self.config = self.get_default_config()\n",
        "        self.device = \"cuda\"\n",
        "        self.model_states = {'h':{'n_layer': 48, 'n_head': 25, 'n_embd': 1600},\n",
        "                                    'g':{'n_layer': 12, '': 12, 'n_embd': 768},\n",
        "                'f':   {'n_layer': 24, 'n_head': 16, 'n_embd': 1024},\n",
        "                'e':   {'n_layer': 36, 'n_head': 20, 'n_embd': 1280},\n",
        "                'd':{'n_layer': 8, 'n_head': 16, 'n_embd': 512},\n",
        "                'c':{'n_layer': 6, 'n_head': 6, 'n_embd': 192},\n",
        "                'b':{'n_layer': 4, 'n_head': 4, 'n_embd': 128},\n",
        "                'a':{'n_layer': 3, 'n_head': 3, 'n_embd': 48}}\n",
        "\n",
        "        type_ = config.model_type is not None\n",
        "        #assert type_ in \"abcdefgh\"\n",
        "        p = all([config.n_layer is not None, config.n_head is not None, config.n_embd is not None])\n",
        "        #assert type_ == True and p == True\n",
        "        if type_:\n",
        "            config.merge_from_dict(self.model_states[config.model_type])\n",
        "        self.closs = nn.BCELoss()\n",
        "        self.ny = NY()\n",
        "        self.l = nn.Linear(512,1,64)\n",
        "        self.transformer = nn.ModuleDict(dict(\n",
        "            wte = nn.Embedding(config.vocab_size, config.n_embd),\n",
        "            wpe = nn.Embedding(config.max_length, config.n_embd),\n",
        "            drop = nn.Dropout(config.embd_pdrop),\n",
        "            h = nn.ModuleList([Block(config) for _ in range(config.n_layer)]),\n",
        "            ln_f = nn.LayerNorm(config.n_embd),))\n",
        "        self.classifier_head = nn.Sequential(#nn.Tanh(),\n",
        "                                             nn.Linear(config.n_embd, 2)\n",
        "                                             )\n",
        "        self.apply(self._init_weights)\n",
        "        for pn, p in self.named_parameters():\n",
        "            if pn.endswith('c_proj.weight'):\n",
        "                torch.nn.init.normal_(p, mean=0.0, std=0.02/math.sqrt(2 * config.n_layer))\n",
        "\n",
        "        n_params = sum(p.numel() for p in self.transformer.parameters())\n",
        "        print(\"[ Number of trainable parameters: %.2fM ]\" % (n_params/1e6,))\n",
        "\n",
        "    def _init_weights(self, module):\n",
        "        if isinstance(module, nn.Linear):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "            if module.bias is not None:\n",
        "                torch.nn.init.zeros_(module.bias)\n",
        "        elif isinstance(module, nn.Embedding):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "        elif isinstance(module, nn.LayerNorm):\n",
        "            torch.nn.init.zeros_(module.bias)\n",
        "            torch.nn.init.ones_(module.weight)\n",
        "\n",
        "    @classmethod\n",
        "    def from_pretrained(cls, model_type):\n",
        "        from transformers import GPT2LMHeadModel\n",
        "        config = cls.get_default_config()\n",
        "        config.model_type = model_type\n",
        "        config.vocab_size = 25\n",
        "        config.max_length = 512\n",
        "        model = ClassifierII(config)\n",
        "        sd = model.state_dict()\n",
        "        model_hf = GPT2LMHeadModel.from_pretrained(model_type)\n",
        "        sd_hf = model_hf.state_dict()\n",
        "        keys = [k for k in sd_hf if not k.endswith('attn.masked_bias')]\n",
        "        transposed = ['attn.c_attn.weight', 'attn.c_proj.weight', 'mlp.c_fc.weight', 'mlp.c_proj.weight']\n",
        "        assert len(keys) == len(sd)\n",
        "        for k in keys:\n",
        "            if any(k.endswith(w) for w in transposed):\n",
        "                assert sd_hf[k].shape[::-1] == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k].t())\n",
        "            else:\n",
        "                assert sd_hf[k].shape == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k])\n",
        "        return model\n",
        "\n",
        "    def configure_optimizers(self, train_config):\n",
        "        decay = set()\n",
        "        no_decay = set()\n",
        "        whitelist_weight_modules = (torch.nn.Linear, )\n",
        "        blacklist_weight_modules = (torch.nn.LayerNorm, torch.nn.Embedding)\n",
        "        for mn, m in self.named_modules():\n",
        "            for pn, p in m.named_parameters():\n",
        "                fpn = '%s.%s' % (mn, pn) if mn else pn\n",
        "                if pn.endswith('bias'):\n",
        "                    no_decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, whitelist_weight_modules):\n",
        "                    decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, blacklist_weight_modules):\n",
        "                    no_decay.add(fpn)\n",
        "        param_dict = {pn: p for pn, p in self.named_parameters()}\n",
        "        inter_params = decay & no_decay\n",
        "        union_params = decay | no_decay\n",
        "        assert len(inter_params) == 0, \"parameters %s made it into both decay/no_decay sets!\" % (str(inter_params), )\n",
        "        assert len(param_dict.keys() - union_params) == 0, \"parameters %s were not separated into either decay/no_decay set!\" \\\n",
        "                                                    % (str(param_dict.keys() - union_params), )\n",
        "        optim_groups = [\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(decay))], \"weight_decay\": train_config.weight_decay},\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(no_decay))], \"weight_decay\": 0.0},]\n",
        "        optimizer = torch.optim.AdamW(optim_groups, lr=train_config.learning_rate, betas=train_config.betas)\n",
        "        return optimizer\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict_proba(self,idx,j =\"None\"):\n",
        "      sigmoid = nn.Sigmoid()\n",
        "      soft = nn.Softmax(dim=1)\n",
        "      ny = NY()\n",
        "      si = SiLU()\n",
        "      self.eval()\n",
        "      x,_ = self.forward(idx)\n",
        "      if j == \"None\":\n",
        "        return x[:,0:1]\n",
        "      elif j == \"soft\":\n",
        "        return self.soft(x)[:,0:1]\n",
        "      else:\n",
        "        return sigmoid(x)[:,0:1]\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict(self,idx):\n",
        "      g = torch.zeros((idx.shape[0],1))\n",
        "      e = -1\n",
        "      for i in self.predict_proba(idx):\n",
        "        e+=1\n",
        "        if i[0].item()>=0.5:\n",
        "          g[e] = 1\n",
        "      return g\n",
        "\n",
        "    def forward(self, idx, targets=None):\n",
        "        device = idx.device\n",
        "        b, t = idx.size()\n",
        "        pos = torch.arange(0, t, dtype=torch.long, device=device).unsqueeze(0)\n",
        "        tok_emb = self.transformer.wte(idx)\n",
        "        pos_emb = self.transformer.wpe(pos)\n",
        "        x = self.transformer.drop(tok_emb + pos_emb)\n",
        "        for block in self.transformer.h:\n",
        "            x = block(x)\n",
        "        x = self.transformer.ln_f(x)\n",
        "        logits = self.classifier_head(x).view(b,2,t).to(device)\n",
        "        logits = self.l(logits)\n",
        "        logits = F.sigmoid(self.ny(logits).view(b,2).mean(1).view(b,1))\n",
        "        loss = None\n",
        "        if targets is not None:\n",
        "            loss = F.binary_cross_entropy(logits,targets)\n",
        "        return logits, loss\n",
        "\n",
        "\n",
        "class Trainer:\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.device = 'cuda'\n",
        "        C.num_workers = 4\n",
        "        C.max_iters = None\n",
        "        C.batch_size = 128\n",
        "        C.max_length = 512\n",
        "        C.learning_rate = 8e-4\n",
        "        C.betas = (0.9, 0.95)\n",
        "        C.weight_decay = 0.1\n",
        "        C.grad_norm_clip = 1.0\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config, model, train_dataset,val_dataset):\n",
        "        self.config = config\n",
        "        self.model = model\n",
        "        self.optimizer = None\n",
        "        self.train_dataset = train_dataset\n",
        "        self.val_dataset = val_dataset\n",
        "        self.callbacks = defaultdict(list)\n",
        "        self.device ='cuda'\n",
        "        self.model = self.model.to(self.device)\n",
        "        self.iter_num = 0\n",
        "        self.iter_time = 0.0\n",
        "        self.iter_dt = 0.0\n",
        "\n",
        "    def add_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent].append(callback)\n",
        "\n",
        "    def set_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent] = [callback]\n",
        "\n",
        "    def trigger_callbacks(self, onevent: str):\n",
        "        for callback in self.callbacks.get(onevent, []):\n",
        "            callback(self)\n",
        "\n",
        "    def run(self):\n",
        "        model, config = self.model, self.config\n",
        "        self.optimizer = model.configure_optimizers(config)\n",
        "        batch_size = self.config.batch_size\n",
        "        def get_batch(mode):\n",
        "            batch_size = 64\n",
        "            if mode == \"train\":\n",
        "              data = train_dataset\n",
        "              pos_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]),513))\n",
        "              neg_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]),513))\n",
        "              N = np.random.randint(batch_size)\n",
        "              ix_pos = np.random.randint(pos_train.shape[0]-N)\n",
        "              ix_neg = np.random.randint(neg_train.shape[0]-batch_size-N)\n",
        "              pos_data = pos_train[ix_pos:ix_pos+N,:]\n",
        "              neg_data = neg_train[ix_neg:ix_neg+batch_size-N,:]\n",
        "              data = torch.concat((pos_data,neg_data))\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,1), device=\"cuda\")\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o] = 1\n",
        "                #else:\n",
        "                  #targets[o][1] = 1\n",
        "            else:\n",
        "              data = val_dataset\n",
        "              ix = np.random.randint(data.shape[0]-batch_size)\n",
        "              data = data[ix:ix+batch_size,:]\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,1), device=\"cuda\")\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o] = 1\n",
        "                #else:\n",
        "                 # targets[o][1] = 1\n",
        "            targets = targets.view(batch_size,1).to(\"cuda\")\n",
        "            seq = seq.view(batch_size,512).to(\"cuda\")\n",
        "            return seq, targets\n",
        "\n",
        "        @torch.no_grad\n",
        "        def cross_val():\n",
        "          model.eval()\n",
        "          out = []\n",
        "          for i in [\"train\",\"val\"]:\n",
        "            losses = torch.zeros(200+1)\n",
        "            for k in range(200):\n",
        "              X,Y = get_batch(i)\n",
        "              logits,loss = model(X,Y)\n",
        "              losses[k]=loss.item()\n",
        "              out1 = losses.mean()\n",
        "            out.append(out1)\n",
        "          model.train()\n",
        "          return out\n",
        "        losses = cross_val()\n",
        "        LOSS = [losses]\n",
        "        print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        model.train()\n",
        "        for epoch in range(20):\n",
        "          print(\"[epoch {o}] \\n\".format(o = epoch))\n",
        "          iters = 200\n",
        "          for i in range(iters):\n",
        "            if i == 0:\n",
        "              Y = \"| =\"\n",
        "            elif i == iters -1 :\n",
        "              Y = \"=> 100% |\"\n",
        "            else:\n",
        "              if i%(int(iters/50)) == 0 :\n",
        "                Y = \"=\"\n",
        "              else:\n",
        "                Y = \"\"\n",
        "            print(\"{y}\".format(y = Y),end=\"\")\n",
        "            x, y = get_batch(\"train\")\n",
        "            logits, self.loss = model(x, y)\n",
        "            model.zero_grad(set_to_none=True)\n",
        "            self.loss.backward()\n",
        "            torch.nn.utils.clip_grad_norm_(model.parameters(), config.grad_norm_clip)\n",
        "            self.optimizer.step()\n",
        "          losses = cross_val()\n",
        "          LOSS.append(losses)\n",
        "          print(LOSS)\n",
        "          print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        PATH = \"model {h}\".format(h = model_config.model_type)\n",
        "        torch.save(model.state_dict(), PATH)\n",
        "\n",
        "def set_seed(seed):\n",
        "    random.seed(seed)\n",
        "    np.random.seed(seed)\n",
        "    torch.manual_seed(seed)\n",
        "    torch.cuda.manual_seed_all(seed)\n",
        "\n",
        "def setup_logging(config):\n",
        "    work_dir = config.system.work_dir\n",
        "    os.makedirs(work_dir, exist_ok=True)\n",
        "    with open(os.path.join(work_dir, 'args.txt'), 'w') as f:\n",
        "        f.write(' '.join(sys.argv))\n",
        "    with open(os.path.join(work_dir, 'config.json'), 'w') as f:\n",
        "        f.write(json.dumps(config.to_dict(), indent=4))\n",
        "\n",
        "class CfgNode:\n",
        "    def __init__(self, **kwargs):\n",
        "        self.__dict__.update(kwargs)\n",
        "\n",
        "    def __str__(self):\n",
        "        return self._str_helper(0)\n",
        "\n",
        "    def _str_helper(self, indent):\n",
        "        parts = []\n",
        "        for k, v in self.__dict__.items():\n",
        "            if isinstance(v, CfgNode):\n",
        "                parts.append(\"%s:\\n\" % k)\n",
        "                parts.append(v._str_helper(indent + 1))\n",
        "            else:\n",
        "                parts.append(\"%s: %s\\n\" % (k, v))\n",
        "        parts = [' ' * (indent * 4) + p for p in parts]\n",
        "        return \"\".join(parts)\n",
        "\n",
        "    def to_dict(self):\n",
        "        return { k: v.to_dict() if isinstance(v, CfgNode) else v for k, v in self.__dict__.items() }\n",
        "\n",
        "    def merge_from_dict(self, d):\n",
        "        self.__dict__.update(d)\n",
        "\n",
        "    def merge_from_args(self, args):\n",
        "\n",
        "        for arg in args:\n",
        "            keyval = arg.split('=')\n",
        "            assert len(keyval) == 2, \"expecting each override arg to be of form --arg=value, got %s\" % arg\n",
        "            key, val = keyval\n",
        "            try:\n",
        "                val = literal_eval(val)\n",
        "            except ValueError:\n",
        "                pass\n",
        "\n",
        "            assert key[:2] == '--'\n",
        "            key = key[2:]\n",
        "            keys = key.split('.')\n",
        "            obj = self\n",
        "            for k in keys[:-1]:\n",
        "                obj = getattr(obj, k)\n",
        "            leaf_key = keys[-1]\n",
        "            assert hasattr(obj, leaf_key), f\"{key} is not an attribute that exists in the config\"\n",
        "            print(\"command line overwriting config attribute %s with %s\" % (key, val))\n",
        "            setattr(obj, leaf_key, val)\n",
        "\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "\n",
        "class SiLU(nn.Module):\n",
        "   def forward(self, x):\n",
        "        return x*F.sigmoid(x)\n",
        "\n",
        "class NY(nn.Module):\n",
        "  def forward(self,x):\n",
        "    return 3*torch.tanh(0.3*x)\n",
        "\n",
        "class NewGELU(nn.Module):\n",
        "    def forward(self, x):\n",
        "        return 0.5 * x * (1.0 + torch.tanh(math.sqrt(2.0 / math.pi) * (x + 0.044715 * torch.pow(x, 3.0))))\n",
        "\n",
        "class SelfAttention(nn.Module):\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        assert config.n_embd % config.n_head == 0\n",
        "        self.c_attn = nn.Linear(config.n_embd, 3 * config.n_embd)\n",
        "        self.c_proj = nn.Linear(config.n_embd, config.n_embd)\n",
        "        self.attn_dropout = nn.Dropout(config.attn_pdrop)\n",
        "        self.resid_dropout = nn.Dropout(config.resid_pdrop)\n",
        "        self.register_buffer(\"bias\", torch.tril(torch.ones(config.block_size, config.block_size))\n",
        "                                     .view(1, 1, config.block_size, config.block_size))\n",
        "        self.n_head = config.n_head\n",
        "        self.n_embd = config.n_embd\n",
        "\n",
        "    def forward(self, x):\n",
        "        B, T, C = x.size()\n",
        "        q, k ,v  = self.c_attn(x).split(self.n_embd, dim=2)\n",
        "        k = k.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        q = q.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        v = v.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        att = (q @ k.transpose(-2, -1)) * (1.0 / math.sqrt(k.size(-1)))\n",
        "        att = att.masked_fill(self.bias[:,:,:T,:T] == 0, float('-inf'))\n",
        "        att = F.softmax(att, dim=-1)\n",
        "        att = self.attn_dropout(att)\n",
        "        y = att @ v\n",
        "        y = y.transpose(1, 2).contiguous().view(B, T, C)\n",
        "        y = self.resid_dropout(self.c_proj(y))\n",
        "        return y\n",
        "\n",
        "class Block(nn.Module):\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.ln_1 = nn.LayerNorm(config.n_embd)\n",
        "        self.attn = SelfAttention(config)\n",
        "        self.ln_2 = nn.LayerNorm(config.n_embd)\n",
        "        self.mlp = nn.ModuleDict(dict(\n",
        "            c_fc    = nn.Linear(config.n_embd, 4 * config.n_embd),\n",
        "            c_proj  = nn.Linear(4 * config.n_embd, config.n_embd),\n",
        "            act     = NewGELU(),\n",
        "            dropout = nn.Dropout(config.resid_pdrop),\n",
        "        ))\n",
        "        m = self.mlp\n",
        "        self.mlpf = lambda x: m.dropout(m.c_proj(m.act(m.c_fc(x))))\n",
        "    def forward(self, x):\n",
        "        x = x + self.attn(self.ln_1(x))\n",
        "        x = x + self.mlpf(self.ln_2(x))\n",
        "        return x\n",
        "\n",
        "class ClassifierI(nn.Module):\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.model_type = 'gpt'\n",
        "        C.n_layer = None\n",
        "        C.n_head = None\n",
        "        C.n_embd =  None\n",
        "        C.vocab_size = len(chars)\n",
        "        C.max_length = 512\n",
        "        C.embd_pdrop = 0.1\n",
        "        C.resid_pdrop = 0.1\n",
        "        C.attn_pdrop = 0.1\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.max_length = config.block_size\n",
        "        self.soft = nn.Softmax(1)\n",
        "        self.config = self.get_default_config()\n",
        "        self.device = \"cpu\"\n",
        "        self.model_states = {'h':{'n_layer': 48, 'n_head': 25, 'n_embd': 1600},\n",
        "                                    'g':{'n_layer': 12, '': 12, 'n_embd': 768},\n",
        "                'f':   {'n_layer': 24, 'n_head': 16, 'n_embd': 1024},\n",
        "                'e':   {'n_layer': 36, 'n_head': 20, 'n_embd': 1280},\n",
        "                'd':{'n_layer': 8, 'n_head': 16, 'n_embd': 512},\n",
        "                'c':{'n_layer': 6, 'n_head': 6, 'n_embd': 192},\n",
        "                'b':{'n_layer': 4, 'n_head': 4, 'n_embd': 128},\n",
        "                'a':{'n_layer': 3, 'n_head': 3, 'n_embd': 48}}\n",
        "\n",
        "        type_ = config.model_type is not None\n",
        "        #assert type_ in \"abcdefgh\"\n",
        "        p = all([config.n_layer is not None, config.n_head is not None, config.n_embd is not None])\n",
        "        #assert type_ == True and p == True\n",
        "        if type_:\n",
        "            config.merge_from_dict(self.model_states[config.model_type])\n",
        "        self.closs = nn.BCELoss()\n",
        "        self.ny = NY()\n",
        "        #self.l = nn.Linear(512,1,64)\n",
        "        self.transformer = nn.ModuleDict(dict(\n",
        "            wte = nn.Embedding(config.vocab_size, config.n_embd),\n",
        "            wpe = nn.Embedding(config.max_length, config.n_embd),\n",
        "            drop = nn.Dropout(config.embd_pdrop),\n",
        "            h = nn.ModuleList([Block(config) for _ in range(config.n_layer)]),\n",
        "            ln_f = nn.LayerNorm(config.n_embd),))\n",
        "        self.classifier_head = nn.Sequential(#nn.Tanh(),\n",
        "                                             nn.Linear(config.n_embd, 2)\n",
        "                                             )\n",
        "        self.apply(self._init_weights)\n",
        "        for pn, p in self.named_parameters():\n",
        "            if pn.endswith('c_proj.weight'):\n",
        "                torch.nn.init.normal_(p, mean=0.0, std=0.02/math.sqrt(2 * config.n_layer))\n",
        "\n",
        "        n_params = sum(p.numel() for p in self.transformer.parameters())\n",
        "        #print(\"[ Number of trainable parameters: %.2fM ]\" % (n_params/1e6,))\n",
        "\n",
        "    def _init_weights(self, module):\n",
        "        if isinstance(module, nn.Linear):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "            if module.bias is not None:\n",
        "                torch.nn.init.zeros_(module.bias)\n",
        "        elif isinstance(module, nn.Embedding):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "        elif isinstance(module, nn.LayerNorm):\n",
        "            torch.nn.init.zeros_(module.bias)\n",
        "            torch.nn.init.ones_(module.weight)\n",
        "\n",
        "    @classmethod\n",
        "    def from_pretrained(cls, model_type):\n",
        "        from transformers import GPT2LMHeadModel\n",
        "        config = cls.get_default_config()\n",
        "        config.model_type = model_type\n",
        "        config.vocab_size = 25\n",
        "        config.max_length = 512\n",
        "        model = ClassifierI(config)\n",
        "        sd = model.state_dict()\n",
        "        model_hf = GPT2LMHeadModel.from_pretrained(model_type)\n",
        "        sd_hf = model_hf.state_dict()\n",
        "        keys = [k for k in sd_hf if not k.endswith('attn.masked_bias')]\n",
        "        transposed = ['attn.c_attn.weight', 'attn.c_proj.weight', 'mlp.c_fc.weight', 'mlp.c_proj.weight']\n",
        "        assert len(keys) == len(sd)\n",
        "        for k in keys:\n",
        "            if any(k.endswith(w) for w in transposed):\n",
        "                assert sd_hf[k].shape[::-1] == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k].t())\n",
        "            else:\n",
        "                assert sd_hf[k].shape == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k])\n",
        "        return model\n",
        "\n",
        "    def configure_optimizers(self, train_config):\n",
        "        decay = set()\n",
        "        no_decay = set()\n",
        "        whitelist_weight_modules = (torch.nn.Linear, )\n",
        "        blacklist_weight_modules = (torch.nn.LayerNorm, torch.nn.Embedding)\n",
        "        for mn, m in self.named_modules():\n",
        "            for pn, p in m.named_parameters():\n",
        "                fpn = '%s.%s' % (mn, pn) if mn else pn\n",
        "                if pn.endswith('bias'):\n",
        "                    no_decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, whitelist_weight_modules):\n",
        "                    decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, blacklist_weight_modules):\n",
        "                    no_decay.add(fpn)\n",
        "        param_dict = {pn: p for pn, p in self.named_parameters()}\n",
        "        inter_params = decay & no_decay\n",
        "        union_params = decay | no_decay\n",
        "        assert len(inter_params) == 0, \"parameters %s made it into both decay/no_decay sets!\" % (str(inter_params), )\n",
        "        assert len(param_dict.keys() - union_params) == 0, \"parameters %s were not separated into either decay/no_decay set!\" \\\n",
        "                                                    % (str(param_dict.keys() - union_params), )\n",
        "        optim_groups = [\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(decay))], \"weight_decay\": train_config.weight_decay},\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(no_decay))], \"weight_decay\": 0.0},]\n",
        "        optimizer = torch.optim.AdamW(optim_groups, lr=train_config.learning_rate, betas=train_config.betas)\n",
        "        return optimizer\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict_proba(self,idx,j =\"None\"):\n",
        "      sigmoid = nn.Sigmoid()\n",
        "      soft = nn.Softmax(dim=1)\n",
        "      ny = NY()\n",
        "      si = SiLU()\n",
        "      self.eval()\n",
        "      x,_ = self.forward(idx)\n",
        "      if j == \"None\":\n",
        "        return x[:,0:1]\n",
        "      elif j == \"soft\":\n",
        "        return self.soft(x)[:,0:1]\n",
        "      else:\n",
        "        return sigmoid(x)[:,0:1]\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict(self,idx):\n",
        "      g = torch.zeros((idx.shape[0],1))\n",
        "      e = -1\n",
        "      for i in self.predict_proba(idx):\n",
        "        e+=1\n",
        "        if i[0].item()>=0.5:\n",
        "          g[e] = 1\n",
        "      return g\n",
        "\n",
        "    def forward(self, idx, targets=None):\n",
        "        device = idx.device\n",
        "        b, t = idx.size()\n",
        "        pos = torch.arange(0, t, dtype=torch.long, device=device).unsqueeze(0)\n",
        "        tok_emb = self.transformer.wte(idx)\n",
        "        pos_emb = self.transformer.wpe(pos)\n",
        "        x = self.transformer.drop(tok_emb + pos_emb)\n",
        "        for block in self.transformer.h:\n",
        "            x = block(x)\n",
        "        x = self.transformer.ln_f(x)\n",
        "        logits = self.ny(self.classifier_head(x)).mean(1).to(device)\n",
        "        #logits = self.l(logits)\n",
        "        # = F.sigmoid(logits.view(b,2).mean(1).view(b,1))\n",
        "        loss = None\n",
        "        if targets is not None:\n",
        "            loss = F.cross_entropy(logits,targets,ignore_index = -1)\n",
        "        return logits, loss\n",
        "\n",
        "\n",
        "class Trainer:\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.device = 'cuda'\n",
        "        C.num_workers = 4\n",
        "        C.max_iters = None\n",
        "        C.batch_size = 64\n",
        "        C.max_length = 512\n",
        "        C.learning_rate = 8e-4\n",
        "        C.betas = (0.9, 0.95)\n",
        "        C.weight_decay = 0.1\n",
        "        C.grad_norm_clip = 1.0\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config, model, train_dataset,val_dataset):\n",
        "        self.config = config\n",
        "        self.model = model\n",
        "        self.optimizer = None\n",
        "        self.train_dataset = train_dataset\n",
        "        self.val_dataset = val_dataset\n",
        "        self.callbacks = defaultdict(list)\n",
        "        self.device ='cuda'\n",
        "        self.model = self.model.to(self.device)\n",
        "        self.iter_num = 0\n",
        "        self.iter_time = 0.0\n",
        "        self.iter_dt = 0.0\n",
        "\n",
        "    def add_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent].append(callback)\n",
        "\n",
        "    def set_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent] = [callback]\n",
        "\n",
        "    def trigger_callbacks(self, onevent: str):\n",
        "        for callback in self.callbacks.get(onevent, []):\n",
        "            callback(self)\n",
        "\n",
        "    def run(self):\n",
        "        model, config = self.model, self.config\n",
        "        self.optimizer = model.configure_optimizers(config)\n",
        "        batch_size = self.config.batch_size\n",
        "        def get_batch(mode):\n",
        "            batch_size = 64\n",
        "            if mode == \"train\":\n",
        "              data = train_dataset\n",
        "              pos_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]),513))\n",
        "              neg_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]),513))\n",
        "              N = np.random.randint(batch_size)\n",
        "              ix_pos = np.random.randint(pos_train.shape[0]-N)\n",
        "              ix_neg = np.random.randint(neg_train.shape[0]-batch_size-N)\n",
        "              pos_data = pos_train[ix_pos:ix_pos+N,:]\n",
        "              neg_data = neg_train[ix_neg:ix_neg+batch_size-N,:]\n",
        "              data = torch.concat((pos_data,neg_data))\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,2), device=self.device)\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o][0] = 1\n",
        "                else:\n",
        "                  targets[o][1] = 1\n",
        "            else:\n",
        "              data = val_dataset\n",
        "              ix = np.random.randint(data.shape[0]-batch_size)\n",
        "              data = data[ix:ix+batch_size,:]\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,2), device=self.device)\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o][0] = 1\n",
        "                else:\n",
        "                  targets[o][1] = 1\n",
        "            targets = targets.view(batch_size,2).to(self.device)\n",
        "            seq = seq.view(batch_size,512).to(self.device)\n",
        "            return seq, targets\n",
        "\n",
        "        @torch.no_grad\n",
        "        def cross_val():\n",
        "          model.eval()\n",
        "          out = []\n",
        "          for i in [\"train\",\"val\"]:\n",
        "            losses = torch.zeros(200+1)\n",
        "            for k in range(200):\n",
        "              X,Y = get_batch(i)\n",
        "              logits,loss = model(X,Y)\n",
        "              losses[k]=loss.item()\n",
        "              out1 = losses.mean()\n",
        "            out.append(out1)\n",
        "          model.train()\n",
        "          return out\n",
        "        losses = cross_val()\n",
        "        LOSS = [losses]\n",
        "        print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        model.train()\n",
        "        for epoch in range(20):\n",
        "          print(\"[epoch {o}] \\n\".format(o = epoch))\n",
        "          iters = 200\n",
        "          for i in range(iters):\n",
        "            if i == 0:\n",
        "              Y = \"| =\"\n",
        "            elif i == iters -1 :\n",
        "              Y = \"=> 100% |\"\n",
        "            else:\n",
        "              if i%(int(iters/50)) == 0 :\n",
        "                Y = \"=\"\n",
        "              else:\n",
        "                Y = \"\"\n",
        "            print(\"{y}\".format(y = Y),end=\"\")\n",
        "            x, y = get_batch(\"train\")\n",
        "            logits, self.loss = model(x, y)\n",
        "            model.zero_grad(set_to_none=True)\n",
        "            self.loss.backward()\n",
        "            torch.nn.utils.clip_grad_norm_(model.parameters(), config.grad_norm_clip)\n",
        "            self.optimizer.step()\n",
        "          losses = cross_val()\n",
        "          LOSS.append(losses)\n",
        "          print(LOSS)\n",
        "          print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        PATH = \"model {h}\".format(h = model_config.model_type)\n",
        "        torch.save(model.state_dict(), PATH)\n",
        "class Transformer():\n",
        "  def __init__(self,mode):\n",
        "    self.mode = mode\n",
        "    self.device = device\n",
        "    self.vocab_dict = tokenizer.get_vocab()\n",
        "\n",
        "  def classifier(self):\n",
        "    device = self.device\n",
        "    model_config = ClassifierI.get_default_config()\n",
        "    model_config.vocab_size = 25\n",
        "    model_config.block_size = 512\n",
        "    if self.mode == \"b\":\n",
        "      model_config.model_type = 'b'\n",
        "      model2 = ClassifierI(model_config)\n",
        "      model2.load_state_dict(torch.load(\"classi/model_b\",map_location=self.device))\n",
        "    else:\n",
        "      model_config.model_type = 'c'\n",
        "      model2 = ClassifierI(model_config)\n",
        "      model2.load_state_dict(torch.load(\"classi/model_c\",map_location = self.device))\n",
        "    model2.to(device)\n",
        "    return model2\n",
        "\n",
        "  def Encode(self, i):\n",
        "        def encode_char(x):\n",
        "            return [self.vocab_dict[char] for char in x]\n",
        "\n",
        "        def pad_sequences(x, PAD=0, max_len=512):\n",
        "            return np.array([seq + [PAD] * (max_len - len(seq)) for seq in x])\n",
        "\n",
        "        encoded = list(map(encode_char, i))\n",
        "        padded = pad_sequences(encoded)\n",
        "        return torch.tensor(padded).to(self.device)\n",
        "\n",
        "  def Decode(self,i):\n",
        "    def decode(k):\n",
        "      l = [j for j in k if j != 0 ]\n",
        "      seq = [chars[j] for j in l]\n",
        "      return \"\".join(seq)\n",
        "    H = list(map(decode,i))\n",
        "    G = []\n",
        "    for i in H:\n",
        "      if len(i)>512:\n",
        "        u = H.index(i)\n",
        "        for j in range(len(i)-512):\n",
        "          G.append(i[j:j+512])\n",
        "      else:\n",
        "        G.append(i)\n",
        "    return G,u\n",
        "  def Mean(self,i):\n",
        "    I = list(i.ravel())\n",
        "    sig = lambda x:np.e**(0*x)\n",
        "    Sum = [sig(np.abs(x-int(len(I)/2))) for x in range(len(I))]\n",
        "    wei = np.array([Sum[i]/sum(Sum) for i in range(len(I))]).reshape(len(I),1)\n",
        "    return np.sum(wei*i)\n",
        "\n",
        "  def enhancer(self,x):\n",
        "    wei = np.array([np.exp((i[0]-1)/(i[0])) for i in x])\n",
        "    return np.sum(np.array([i[0] for i in x])*wei)/np.sum(wei)\n",
        "\n",
        "  def predict_proba(self,i):\n",
        "    if len(i[0])>512:\n",
        "       l = [self.Encode([i[0][j:j+512]]) for j in range(len(i[0])-512)]\n",
        "       if len(l)>700:\n",
        "        L = [torch.concat(tuple(l[i*700:(i+1)*700])) for i in range(len(l)//700)] + [torch.concat(tuple(l[(len(l)//700)*700:(len(l)//700)*700+len(l)%700]))]\n",
        "        T = [self.classifier().predict_proba(i,\"sig\").tolist() for i in L ]\n",
        "        return self.enhancer(np.concatenate(tuple([i for i in T])))\n",
        "       else:\n",
        "         t = torch.concat(tuple(l))\n",
        "         return self.enhancer(self.classifier().predict_proba(t,\"sig\").tolist())\n",
        "    else:\n",
        "       seq = self.Encode(i)\n",
        "       U = np.array(self.classifier().predict_proba(seq,\"sig\").tolist())\n",
        "       out = np.array(U)\n",
        "       return out.reshape(len(i),1)\n",
        "\n",
        "model2 = Transformer(\"b\")\n",
        "model3 = Transformer(\"c\")\n",
        "\n",
        "\n",
        "import functools as FUNC\n",
        "from numba import njit\n",
        "\n",
        "class PPA():\n",
        "    def __init__(self, seq, model):\n",
        "        self.model = model\n",
        "        self.seq = seq\n",
        "        self.len = len(self.seq)\n",
        "        self.idx = None\n",
        "\n",
        "    def attention_mask(self):\n",
        "        idx2 = self.splicer(self.seq)[1]\n",
        "        k = self.T()\n",
        "        mask = [[True if len(k[0]) > idx2[j] - i > -1 else False for i in range(len(k[j]))] for j in range(len(k))]\n",
        "        return mask\n",
        "\n",
        "    def mean(self, mask, spliced):\n",
        "        u = [np.mean(self.model.predict_proba([spliced[i][j] for j in range(len(spliced[i])) if mask[i][j]])) for i in trange(len(spliced))]\n",
        "        return u\n",
        "\n",
        "    def splice_scorer(self, i):\n",
        "        lmin = len(i) // 2\n",
        "        return [i[j:j + lmin] for j in range(len(i) - lmin + 1)]\n",
        "\n",
        "    def splicer(self, i):\n",
        "        idx = self.idx\n",
        "        o, Before = [], []\n",
        "        for j in range(len(i)):\n",
        "            n_b, n_a = j, len(i) - j - 1\n",
        "            if n_b <= idx // 2:\n",
        "                before = i[max(0, j - idx // 2):j]\n",
        "                after = i[j + 1:j + idx - len(before)]\n",
        "            elif n_a <= idx // 2:\n",
        "                before = i[j - idx + n_a + 1:j]\n",
        "                after = i[j + 1:]\n",
        "            else:\n",
        "                before = i[j - idx // 2:j]\n",
        "                after = i[j + 1:j + idx // 2 + 1]\n",
        "            o.append(before + i[j] + after)\n",
        "            Before.append(len(before))\n",
        "        return o, Before\n",
        "\n",
        "    def T(self):\n",
        "        return list(map(self.splice_scorer, self.splicer(self.seq)[0]))\n",
        "\n",
        "    def Out(self):\n",
        "        s = self.s\n",
        "        def d(x):\n",
        "            if s > 0.7:\n",
        "                return x * np.exp(-1.2 * (x - s))\n",
        "            else:\n",
        "                return x if x > 0.7 else x\n",
        "\n",
        "        return list(map(d, self.mean(self.attention_mask(), self.T())))\n",
        "def show(i):\n",
        "  values = np.array(i).reshape(1, len(i))\n",
        "  y = range(len(i))\n",
        "  Y = [(j,1) for j in range(len(i)) if np.mean([i[j],i[min((j+1),len(i)-1)],i[max(0,(j-1))]])>0.6]\n",
        "  Y2 = [(j,1) for j in range(len(i)) if np.mean([i[j],i[min((j+1),len(i)-1)],i[min((j+2),len(i)-1)],i[min((j+3),len(i)-1)],i[min((j+4),len(i)-1)],i[min((j+5),len(i)-1)],i[max(0,(j-1))],i[max(0,(j-2))],i[max(0,(j-3))],i[max(0,(j-4))],i[max(0,(j-5))]])>0.85]\n",
        "  threshold = 0.5\n",
        "  above_threshold = np.maximum(values - threshold, 0)\n",
        "  below_threshold = np.minimum(values, threshold)\n",
        "  fig, (ax3 , ax4) = plt.subplots(2,1,sharex = True,sharey = True)\n",
        "  ax3.bar(y, below_threshold.ravel(), 0.9, color=\"lightblue\")\n",
        "  ax3.bar(y, above_threshold.ravel(), 0.9, color=\"lightgreen\",\n",
        "               bottom=below_threshold.ravel())\n",
        "  ax4.broken_barh(Y, (0.7, 0.03),\n",
        "               facecolors=( 'red'))\n",
        "  ax4.plot([0., y[-1]],[0.715,0.715],\"-b\")\n",
        "  for k in Y:\n",
        "    ax4.plot([k[0],k[0]+1],[0.715,0.715],\"-r\")\n",
        "\n",
        "  ax4.text(0,0.8 , \"LLPS-susceptible regions\",fontsize = 20)\n",
        "  ax3.plot([0., y[-1]], [threshold, threshold], \"k--\")\n",
        "  ax4.set_ylim([0,1])\n",
        "  ax3.set_ylim([0,1])\n",
        "\n",
        "  plt.subplots_adjust(\n",
        "                    wspace=0.4,\n",
        "                    hspace=0)\n",
        "  fig.set_figwidth(20)\n",
        "  fig.set_figheight(15)\n",
        "  plt.show()\n"
      ],
      "metadata": {
        "cellView": "form",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ycPUcFXh7mTu",
        "outputId": "3e98712a-b1d3-4ce5-8b32-d91896d67802"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Cuda is not available üòî\n",
            "\n",
            "The allocated device is : cpu\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Loading sequence , then hit `Runtime` -> `Run all`\n",
        "#@markdown - Load the sequence of your protein :\n",
        "Sequence = \"MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGQQQQQQWQWQWWQQQQQQQWWWWWWWWWWWWWWPPPPPPPPPPPPPQQQQPPPPPPPPPPPPPQQPW\" #@param {type:\"string\"}\n",
        "ID = None #@param {type:\"string\"}\n",
        "Directory = None #@param {type:\"string\"}\n",
        "#@markdown  - Use `End_sequence` to focus the scoring on sequences up to the specified endpoint\n",
        "\n",
        "End_sequence = 644 #@param {type:\"slider\" , min : 0 , max: 1000 , step : 1}\n",
        "Plot = True #@param {type: \"boolean\"}\n",
        "\n",
        "#@markdown - Specify the path of `FASTA format file` of your sequences.\n",
        "Fasta_file = \"your_sequences.fasta\" #@param {type:\"string\"}\n",
        "import pandas as pd\n",
        "#assert [Sequence == None , Fasta_file == None].count(True) == 1\n",
        "\n",
        "def enh(x):\n",
        "  wei = np.array([np.exp((i-1)/(i)) for i in x])\n",
        "  return np.sum(np.array(x)*wei)/np.sum(wei)\n",
        "\n",
        "def edit(i):\n",
        "  for j in i:\n",
        "    if j in \"\\t\\nBJOUXZ, _@#$%^& *n({}[])\\/\\\\\":\n",
        "      i = i.replace(j,\"\")\n",
        "  return i\n",
        "\n",
        "\n",
        "def SW(s):\n",
        "  if len(s)> 700:\n",
        "    L= [i for i in  [s[i*700:(i+1)*700] for i in range(len(s)//700+1)] if len(i)>0]\n",
        "    H = []\n",
        "    for i in range(len(L)):\n",
        "      H.append(model3.predict_proba(L[i]))\n",
        "    return H\n",
        "  else:\n",
        "    return model3.predict_proba(s)\n",
        "\n",
        "\n",
        "def Score1(i,Sc):\n",
        "    i = i-1\n",
        "    if i<= L:\n",
        "      return sum([Sc[j] for j in range(i+1)])/(i+1)\n",
        "\n",
        "    elif L+1<=i<n-L:\n",
        "       return sum([Sc[j] for j in range(i-L,i+1)])/(L+1)\n",
        "\n",
        "    else:\n",
        "       u = (n-(i)+1)\n",
        "       return sum([Sc[j] for j in range(i-L,n-L+1)])/(u)\n",
        "Sequence = edit(Sequence)\n",
        "\n",
        "u = model3.predict_proba([Sequence])\n",
        "\n",
        "def d(x):\n",
        "    if u>0.7:\n",
        "        return x*np.exp(-1.2*(x-u))\n",
        "    else:\n",
        "        if x > 0.7 :\n",
        "            return x\n",
        "        else:\n",
        "            return x\n",
        "\n",
        "\n",
        "if Sequence != None and Sequence !=\"\":\n",
        "  Sequence = edit(Sequence)\n",
        "  k = max(5,min(50,int(np.ceil(0.1*len(Sequence)))))\n",
        "  L = k//2\n",
        "  if Plot == True:\n",
        "    n = len(Sequence)\n",
        "    S = [Sequence[i:i+L] for i in range(len(Sequence)-L+1)]\n",
        "    Sc = SW(S)\n",
        "    Sc1 = Sc[0]\n",
        "    for i in Sc[1:]:\n",
        "      Sc1 = np.concatenate((Sc1,i))\n",
        "\n",
        "    p = list(map(d,[Score1(i,Sc1) for i in range(1,n+1)]))\n",
        "    if type(p[0]) != np.float64:\n",
        "      p = [i[0] for i in p]\n",
        "    if type(p[0]) != np.float64:\n",
        "      p = [i[0] for i in p]\n",
        "\n",
        "    u = model3.predict_proba([Sequence])\n",
        "    i = p\n",
        "    values = np.array(i).reshape(1, len(i))\n",
        "    below_threshold = [k if k<= 0.5 else 0.5 for k in i]\n",
        "    above_threshold = [0 if k<= 0.5 else k-0.5 for k in i]\n",
        "    threshold = 0.5\n",
        "    y = list(range(len(i)))\n",
        "    Y = [(j, 1) for j in range(len(i)) if np.mean([i[j], i[min((j + 1), len(i) - 1)], i[max(0, (j - 1))]]) > 0.6]\n",
        "    Y2 = [(j, 1) for j in range(len(i)) if np.mean([i[j], i[min((j + 1), len(i) - 1)], i[min((j + 2), len(i) - 1)],\n",
        "                                               i[min((j + 3), len(i) - 1)], i[min((j + 4), len(i) - 1)],\n",
        "                                               i[min((j + 5), len(i) - 1)], i[max(0, (j - 1))],\n",
        "                                               i[max(0, (j - 2))], i[max(0, (j - 3))], i[max(0, (j - 4))],\n",
        "                                               i[max(0, (j - 5))]]) > 0.85]\n",
        "    #if n>= 1500:\n",
        "      #c1 = \"blue\"\n",
        "      #c2 = \"green\"\n",
        "    #else:\n",
        "    c1 = \"lightblue\"\n",
        "    c2 = \"lightgreen\"\n",
        "\n",
        "    fig1 = make_subplots(rows=1, cols=1, shared_xaxes=True, shared_yaxes=True, vertical_spacing=0.02)\n",
        "    fig1.add_trace(go.Bar(x=y, y=below_threshold, marker_color= c1, name='Below Threshold', width= 0.6), row=1, col=1)\n",
        "    fig1.add_trace(go.Bar(x=y, y=above_threshold, marker_color=c2, name='Above Threshold', base=below_threshold, width= 0.6), row=1, col=1)\n",
        "\n",
        "    fig1.add_trace(go.Scatter(x=[0, len(i) - 1], y=[threshold, threshold], mode='lines', line=dict(color='black', dash='dash'), showlegend=False), row=1, col=1)\n",
        "    for start, duration in Y:\n",
        "        fig1.add_trace(go.Scatter(x=[start, start + duration], y=[0.01, 0.01], mode='lines', line=dict(color='red', width=7), showlegend=False), row=1, col=1)\n",
        "    fig1.update_layout(height=800, width=1700, title_text=\"LLPS Prediction Plot\")\n",
        "    u2 = RF([Sequence])\n",
        "    u2 = u2.predict_proba()[0][1]\n",
        "    score = str(((1*u+1*enh(p)+4*u2)/6))\n",
        "    print(\"Score : \"+score)\n",
        "    output.clear()\n",
        "    print(\"Score : \"+score)\n",
        "  else:\n",
        "    score = u\n",
        "    print(\"Score : \"+str(score))\n",
        "\n",
        "\n",
        "else:\n",
        "  if os.path.exists(Fasta_file):\n",
        "    with open(Fasta_file) as f:\n",
        "      U = f.read().split(\">\")\n",
        "      w = len(U)\n",
        "    End_sequence = min(End_sequence,len(U))\n",
        "    j = U[1:End_sequence]\n",
        "    D = {\"id\":[i[:i.index(\"\\n\")] if \"\\n\" in i else i for i in j],\"seq\":[edit(i[i.index(\"\\n\"):]) if \"\\n\" in i else None for i in j],\"score\":[]}\n",
        "    for sequence in tqdm(D[\"seq\"]):\n",
        "      output.clear()\n",
        "      if sequence == None :\n",
        "        D[\"score\"].append(\"_\")\n",
        "      else:\n",
        "        sequence = edit(sequence)\n",
        "        k = max(5,min(50,int(np.ceil(0.1*len(sequence)))))\n",
        "        L = k//2\n",
        "        n = len(sequence)\n",
        "        S = [sequence[i:i+L] for i in range(len(sequence)-L+1)]\n",
        "        if len(S)%700 == 0:\n",
        "          D[\"score\"].append(\"_\")\n",
        "        else:\n",
        "          Sc = SW(S)\n",
        "          Sc1 = Sc[0]\n",
        "          for i in Sc[1:]:\n",
        "            Sc1 = np.concatenate((Sc1,i))\n",
        "          u = model3.predict_proba([sequence])\n",
        "          p = list(map(d,[Score1(i,Sc1) for i in range(1,n+1)]))\n",
        "          u2 = RF([sequence])\n",
        "          u2 = u2.predict_proba()[0][1]\n",
        "          score = str(((1*u+1*enh(p)+4*u2)/6))\n",
        "          D[\"score\"].append((u,enh(p),u2))\n",
        "    D[\"score\"] = [i[0][0] if type(i) != np.float64 else i for i in D[score]]\n",
        "    D = pd.DataFrame(D)\n",
        "    D.to_csv(\"Your_sequences.csv\")\n",
        "    files.download(\"Your_sequences.csv\")\n",
        "\n",
        "\n",
        "\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "cellView": "form",
        "id": "b3jzYV5n7q-b",
        "outputId": "531e72b8-c4c2-46c8-e339-a91761b419f2"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Score : [[0.87043094]]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title plot\n",
        "import matplotlib.pyplot as plt\n",
        "import streamlit as st\n",
        "from stmol import showmol\n",
        "import py3Dmol\n",
        "import requests\n",
        "import biotite.structure.io as bsio\n",
        "import certifi\n",
        "import matplotlib.colors as mcolors\n",
        "import urllib3\n",
        "\n",
        "if Plot == True and Sequence != \"\" and Sequence != None:\n",
        "  fig1.show()\n",
        "\n",
        "\n",
        "  urllib3.disable_warnings()\n",
        "\n",
        "  st.sidebar.title('üéà ESMFold')\n",
        "  st.sidebar.write('[*ESMFold*](https://esmatlas.com/about) is an end-to-end single sequence protein structure predictor based on the ESM-2 language model. For more information, read the [research article](https://www.biorxiv.org/content/10.1101/2022.07.20.500902v2) and the [news article](https://www.nature.com/articles/d41586-022-03539-1) published in *Nature*.')\n",
        "\n",
        "  def render_mol(pdb):\n",
        "    pdbview = py3Dmol.view()\n",
        "    pdbview.addModel(pdb, 'pdb')\n",
        "    pdbview.setStyle({'cartoon': {'color': 'spectrum'}})\n",
        "    pdbview.setBackgroundColor('white')  # ('0xeeeeee')\n",
        "    pdbview.zoomTo()\n",
        "    pdbview.zoom(2, 800)\n",
        "    pdbview.spin(True)\n",
        "    showmol(pdbview, height=500, width=800)\n",
        "\n",
        "\n",
        "  DEFAULT_SEQ = Sequence\n",
        "  show_mainchains = False #@param {type: \"boolean\"}\n",
        "  show_sidechains = False #@param {type: \"boolean\"}\n",
        "  Color_Style =  \"red_blue_spectrum\" #@param [\"red_blue_spectrum\",\"green_magenta_spectrum\",\"yellow_blue_spectrum\",\"cyan_red_spectrum\"]\n",
        "\n",
        "  import streamlit as st\n",
        "  import requests\n",
        "  import biotite.structure.io as bsio\n",
        "  import py3Dmol\n",
        "\n",
        "  DEFAULT_SEQ = Sequence\n",
        "\n",
        "  txt = st.sidebar.text_area('Input sequence', DEFAULT_SEQ, height=275)\n",
        "\n",
        "  def update(sequence,name):\n",
        "    headers = {\n",
        "        'Content-Type': 'application/x-www-form-urlencoded',\n",
        "    }\n",
        "\n",
        "    def get_pdb(sequence):\n",
        "        response = requests.post('https://api.esmatlas.com/foldSequence/v1/pdb/', headers=headers, data=sequence, verify=False)\n",
        "        if response.status_code != 200:\n",
        "            st.error(f\"Failed to get prediction: {response.status_code}\")\n",
        "            return None\n",
        "        pdb_string = response.content.decode('utf-8')\n",
        "        if not pdb_string or \"ATOM\" not in pdb_string:\n",
        "            st.error(\"Received an invalid PDB file.\")\n",
        "            return None\n",
        "        return pdb_string\n",
        "\n",
        "    def save_and_render_pdb(pdb_string, file_name):\n",
        "        with open(file_name, 'w') as f:\n",
        "            f.write(pdb_string)\n",
        "        try:\n",
        "            struct = bsio.load_structure(file_name, extra_fields=[\"b_factor\"])\n",
        "            b_value = round(struct.b_factor.mean(), 4)\n",
        "        except Exception as e:\n",
        "            st.error(f\"Error loading structure: {e}\")\n",
        "            return\n",
        "        st.subheader('Visualization of predicted protein structure')\n",
        "        render_mol(pdb_string)\n",
        "        st.subheader('plDDT')\n",
        "        st.write('plDDT is a per-residue estimate of the confidence in prediction on a scale from 0-100.')\n",
        "        st.info(f'plDDT: {b_value}')\n",
        "        st.download_button(label=\"Download PDB\", data=pdb_string, file_name=file_name, mime='text/plain')\n",
        "\n",
        "    def render_mol(pdb_string,neme):\n",
        "        view = py3Dmol.view(width=1000, height=800)\n",
        "        view.addModel(pdb_string, 'pdb')\n",
        "        F1 = Get_color(p)\n",
        "        F1.i = F1.J.index(Color_Style)\n",
        "\n",
        "        custom_colors=F1.spectrum()\n",
        "\n",
        "        for i, color in enumerate(custom_colors[1:-1]):\n",
        "          view.setStyle({ 'resi': i+1}, {'cartoon': {'color': color}})\n",
        "        #view.setStyle({'cartoon': {'color': 'spectrum'}})\n",
        "        view.zoomTo()\n",
        "        view.show()\n",
        "        html_content = view._make_html()\n",
        "        with open(name+'.html', 'w') as file:\n",
        "          file.write(html_content)\n",
        "        files.download(name+'.html')\n",
        "\n",
        "\n",
        "    chunk_size = 400\n",
        "    chunks = [sequence[i:i + chunk_size] for i in range(0, len(sequence), chunk_size)]\n",
        "\n",
        "    combined_pdb_string = \"\"\n",
        "    for i, chunk in enumerate(chunks):\n",
        "        st.write(f\"Processing chunk {i + 1} of {len(chunks)}\")\n",
        "        pdb_string = get_pdb(chunk)\n",
        "        if pdb_string:\n",
        "            combined_pdb_string += pdb_string\n",
        "\n",
        "    if combined_pdb_string:\n",
        "        render_mol(combined_pdb_string,name)\n",
        "        with open('combined_predicted.pdb', 'w') as f:\n",
        "            f.write(combined_pdb_string)\n",
        "        st.download_button(label=\"Download Combined PDB\", data=combined_pdb_string, file_name='combined_predicted.pdb', mime='text/plain')\n",
        "  predict = st.sidebar.button('Predict', on_click=update)\n",
        "\n",
        "  if not predict:\n",
        "    st.warning('üëà Enter protein sequence data!')\n",
        "\n",
        "  class Get_color():\n",
        "    def __init__(self,values):\n",
        "      self.J = [\"red_blue_spectrum\",\"green_magenta_spectrum\",\"yellow_blue_spectrum\",\"cyan_red_spectrum\"]\n",
        "      self.i = None\n",
        "      self.values = values\n",
        "\n",
        "    def get_red_blue_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=min(values), vmax=max(values))\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"RedBlue\", [\"blue\", \"red\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "    def get_green_magenta_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=min(values), vmax=max(values))\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"GreenMagenta\", [\"green\", \"magenta\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "    def get_yellow_blue_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=min(values), vmax=max(values))\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"YellowBlue\", [\"yellow\", \"blue\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "\n",
        "    def get_cyan_red_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=min(values), vmax=max(values))\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"CyanRed\", [\"cyan\", \"red\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "    def spectrum(self):\n",
        "      if self.i == 0 : return self.get_red_blue_spectrum(self.values)\n",
        "      if self.i == 1 : return self.get_green_magenta_spectrum(self.values)\n",
        "      if self.i == 2 : return self.get_yellow_blue_spectrum(self.values)\n",
        "      if self.i == 3 : return self.get_cyan_red_spectrum(self.values)\n",
        "  update(txt,\"A\")\n",
        "\n",
        "\n",
        "# Create a color bar legend using matplotlib\n",
        "  fig, ax = plt.subplots(figsize=(6, 1))\n",
        "  fig.subplots_adjust(bottom=0.5)\n",
        "\n",
        "  cmap = plt.cm.Reds\n",
        "  norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "  fig.colorbar(plt.cm.ScalarMappable(norm=norm, cmap=cmap), cax=ax, orientation='horizontal', label='Predicted LLPS Score')\n",
        "  plt.show()\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "cellView": "form",
        "id": "SH8mY8HM7vfj",
        "outputId": "3261ab9d-e4bf-47af-d21f-178686175c8c"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<html>\n",
              "<head><meta charset=\"utf-8\" /></head>\n",
              "<body>\n",
              "    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax && window.MathJax.Hub && window.MathJax.Hub.Config) {window.MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n",
              "        <script charset=\"utf-8\" src=\"https://cdn.plot.ly/plotly-2.24.1.min.js\"></script>                <div id=\"376248ba-66c4-4b05-8fca-ce9ede94be5a\" class=\"plotly-graph-div\" style=\"height:800px; width:1700px;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"376248ba-66c4-4b05-8fca-ce9ede94be5a\")) {                    Plotly.newPlot(                        \"376248ba-66c4-4b05-8fca-ce9ede94be5a\",                        [{\"marker\":{\"color\":\"lightblue\"},\"name\":\"Below Threshold\",\"width\":0.6,\"x\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131,132,133,134,135,136,137,138,139,140,141,142,143,144,145,146,147,148,149,150,151,152,153,154,155,156,157,158,159,160,161,162,163,164,165,166,167,168,169,170,171,172,173,174,175,176,177,178,179,180,181,182,183,184,185,186,187,188,189,190,191,192,193,194,195,196,197,198,199,200,201,202,203,204,205,206,207,208,209,210,211,212,213,214,215,216,217,218,219,220,221,222,223,224,225,226,227,228,229,230,231,232,233,234,235,236,237,238,239,240,241,242,243,244,245,246,247,248,249,250,251,252,253,254,255,256,257,258,259,260,261,262,263,264,265,266,267,268,269,270,271,272,273,274,275,276,277,278,279,280,281,282,283,284,285,286,287,288,289,290,291],\"y\":[0.15031300326229827,0.38056562125025384,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.44399128961619316,0.33258649524937867,0.20234154674547855,0.20743522293913672,0.2132099412221344,0.19174019851268387,0.19514574160883114,0.2017936322009285,0.20508015718502892,0.20442204042244624,0.1991970368093503,0.19471859350256038,0.19193384444360964,0.18948196553045862,0.18695634088258428,0.18541579209771197,0.3231629834491623,0.4427178585419512,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5],\"type\":\"bar\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"base\":[0.15031300326229827,0.38056562125025384,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.44399128961619316,0.33258649524937867,0.20234154674547855,0.20743522293913672,0.2132099412221344,0.19174019851268387,0.19514574160883114,0.2017936322009285,0.20508015718502892,0.20442204042244624,0.1991970368093503,0.19471859350256038,0.19193384444360964,0.18948196553045862,0.18695634088258428,0.18541579209771197,0.3231629834491623,0.4427178585419512,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5],\"marker\":{\"color\":\"lightgreen\"},\"name\":\"Above Threshold\",\"width\":0.6,\"x\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131,132,133,134,135,136,137,138,139,140,141,142,143,144,145,146,147,148,149,150,151,152,153,154,155,156,157,158,159,160,161,162,163,164,165,166,167,168,169,170,171,172,173,174,175,176,177,178,179,180,181,182,183,184,185,186,187,188,189,190,191,192,193,194,195,196,197,198,199,200,201,202,203,204,205,206,207,208,209,210,211,212,213,214,215,216,217,218,219,220,221,222,223,224,225,226,227,228,229,230,231,232,233,234,235,236,237,238,239,240,241,242,243,244,245,246,247,248,249,250,251,252,253,254,255,256,257,258,259,260,261,262,263,264,265,266,267,268,269,270,271,272,273,274,275,276,277,278,279,280,281,282,283,284,285,286,287,288,289,290,291],\"y\":[0,0,0.27493731497725105,0.1744967121761707,0.09893075418257713,0.04204762486335267,0.0001973143721301085,0.14746537119123515,0.23639714985584515,0.2897046952670119,0.2619772003210349,0.3040709599021294,0.33509064113900044,0.3580817557649827,0.37554164195692186,0.38896085260120605,0.41314263966997855,0.4094215637981976,0.38820932584195444,0.41256395054929595,0.43000382485934463,0.44125758187743225,0.44707986729163984,0.4470866926263004,0.44708547968173495,0.44725788072460226,0.4482562975654907,0.44824399415799143,0.44824130913736693,0.4482403121872547,0.4482403548445061,0.4482390199709526,0.4482440834684499,0.44539256998625787,0.439926951142786,0.4399658449940732,0.4400335087149747,0.44605358537970063,0.446070916051273,0.4460746246688764,0.4483629720375484,0.44836359594243136,0.4483639478773548,0.44637805654048623,0.44031918257660174,0.4285701920913372,0.4113009126544841,0.4113056266719234,0.4113813252194769,0.4113129927721748,0.41138260647619396,0.41143296073619506,0.4115276490279197,0.4292099305296043,0.42920459410553824,0.4291874402475634,0.42925898139098584,0.4292575730662457,0.42928088012986076,0.44081948646183966,0.4432939404838271,0.4478201770453859,0.4477438215806483,0.443442115229,0.4434496701505545,0.44346262454300034,0.4356577266905596,0.42614438160511414,0.42620305923579416,0.4072446352249516,0.38294678911976077,0.38304280833311943,0.40867276068481384,0.40873294805720106,0.408726460884297,0.40877515295547395,0.3994909979906236,0.39835160923588897,0.4199087405067551,0.43407526220263426,0.43405770285568757,0.4340653793462256,0.44249268250297147,0.4465697103361592,0.4465629184833947,0.44835021097347116,0.4461547799089872,0.44615853820624074,0.44616782661155074,0.4461710650538494,0.4461803082695006,0.4461810714900676,0.4402889882056319,0.43990709552136564,0.4399143978641403,0.4399095949339369,0.4399258500070702,0.43991962739985324,0.4399348164280231,0.43996308155920927,0.439956252977309,0.43996503204991144,0.4400086289060946,0.4400304760829552,0.4435260361421902,0.443524118378402,0.4435263302294501,0.44772512392529795,0.4479739159545192,0.44797468279259034,0.44797431764262285,0.44797342377594385,0.44797691129884887,0.4479748842759125,0.4479756708583591,0.4479825792890173,0.44798199022163676,0.447986550871455,0.4479952972882446,0.4479973710942362,0.44835057238765563,0.44835176357716366,0.44835021617855286,0.445910052816167,0.4399163611731447,0.43990510906081004,0.4398959488169777,0.43988645621611255,0.4398821977545079,0.4398888809294339,0.439869562855141,0.4398534587276035,0.4398489398050133,0.4398453476940536,0.43983673357798925,0.4398167619491674,0.43980440377644503,0.4397766932847029,0.4397774203868495,0.4397785187779001,0.4397794254871149,0.4397925170712872,0.43979932639063635,0.43979066904478825,0.43977935849804606,0.43977006291269827,0.43978428857643737,0.43980898569369364,0.43982470546334573,0.4398351730730654,0.4398320086952048,0.43983234271192384,0.439834904346734,0.4398332882839179,0.4398480648664729,0.439844237116647,0.43984907898429004,0.4398429789700167,0.43984107062994515,0.43985125139343983,0.43985376753725636,0.43984219433273175,0.4398300562626566,0.4398056525568912,0.4397791102729336,0.4397632618447104,0.4397593358907069,0.43975174835003816,0.43974257616534573,0.43974172962488756,0.4397352562827981,0.43974035541290457,0.43972979759758035,0.43973646171090164,0.43973196699404926,0.43976549449153457,0.4397806977154627,0.43978833708911713,0.4397984579799511,0.43981130941741065,0.4398226066883486,0.4398312541582483,0.4398364772995077,0.43984791326153505,0.43985276505134374,0.4398532244013341,0.43984226058304754,0.43983220237383347,0.43983584461333824,0.4398230180686159,0.43981744709815795,0.43978669733487075,0.43976853122080783,0.43977647462928116,0.4397729669707856,0.43976669801693824,0.43976009900168844,0.43975339578790507,0.4397519750891705,0.43974318021959924,0.4397387219445653,0.4397388837885138,0.43974634690039205,0.4397498443634168,0.439749427239721,0.43975613028404215,0.4397665296526605,0.43977165993955536,0.4397654331699894,0.4397516573416035,0.43975344362804736,0.4407583051578222,0.44075657540656277,0.4407552201954862,0.44075248780457843,0.4407553672283938,0.4407549571066357,0.44076029577366826,0.44075397405218886,0.44075996197237677,0.44076103021144863,0.440765621439159,0.4407734257535797,0.4407832948393283,0.44080449388283005,0.44084205932177,0.44089874728833245,0.4403713667498357,0.4404196244464559,0.44625632860752507,0.4483702704728235,0.4468839559512102,0.44066225467655495,0.42898856960862,0.4110791225310775,0.38625785321817663,0.35469502378065054,0.3146563391556746,0.2646655388251974,0.20388374989548141,0.13112207500360373,0.04505459110837884,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.04160475334858671,0.1262598538573183,0.19827378581940658,0.2594509365937949,0.310134930063255,0.3504259747773326,0.3832689956653298,0.40898973283882156,0.4275846880680667,0.4398573306951882,0.44655453927843136,0.4483468494332947,0.4458429158867909,0.4396270606693897,0.4396272239757587,0.4396271470335059,0.43962704130283825,0.4396330047649336,0.4396333996477363,0.43963390594710283,0.43963452038851025,0.4396353223192181,0.43963631262748826,0.4396374415020743,0.4396387002531895,0.4396396291517142,0.4396403762469945,0.43964099295130277,0.43964370729319824,0.43964836996418366,0.4396559657714648,0.4396724518950117],\"type\":\"bar\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"black\",\"dash\":\"dash\"},\"mode\":\"lines\",\"showlegend\":false,\"x\":[0,291],\"y\":[0.5,0.5],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[2,3],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[3,4],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[4,5],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[7,8],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[8,9],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[9,10],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[10,11],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[11,12],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[12,13],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[13,14],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[14,15],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[15,16],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[16,17],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[17,18],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[18,19],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[19,20],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[20,21],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[21,22],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[22,23],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[23,24],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[24,25],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[25,26],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[26,27],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[27,28],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[28,29],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[29,30],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[30,31],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[31,32],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[32,33],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[33,34],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[34,35],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[35,36],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[36,37],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[37,38],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[38,39],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[39,40],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[40,41],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[41,42],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[42,43],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[43,44],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[44,45],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[45,46],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[46,47],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[47,48],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[48,49],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[49,50],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[50,51],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[51,52],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[52,53],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[53,54],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[54,55],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[55,56],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[56,57],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[57,58],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[58,59],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[59,60],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[60,61],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[61,62],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[62,63],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[63,64],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[64,65],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[65,66],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[66,67],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[67,68],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[68,69],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[69,70],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[70,71],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[71,72],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[72,73],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[73,74],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[74,75],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[75,76],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[76,77],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[77,78],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[78,79],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[79,80],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[80,81],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[81,82],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[82,83],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[83,84],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[84,85],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[85,86],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[86,87],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[87,88],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[88,89],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[89,90],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[90,91],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[91,92],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[92,93],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[93,94],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[94,95],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[95,96],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[96,97],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[97,98],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[98,99],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[99,100],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[100,101],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[101,102],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[102,103],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[103,104],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[104,105],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[105,106],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[106,107],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[107,108],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[108,109],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[109,110],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[110,111],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[111,112],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[112,113],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[113,114],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[114,115],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[115,116],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[116,117],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[117,118],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[118,119],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[119,120],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[120,121],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[121,122],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[122,123],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[123,124],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[124,125],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[125,126],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[126,127],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[127,128],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[128,129],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[129,130],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[130,131],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[131,132],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[132,133],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[133,134],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[134,135],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[135,136],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[136,137],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[137,138],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[138,139],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[139,140],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[140,141],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[141,142],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[142,143],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[143,144],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[144,145],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[145,146],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[146,147],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[147,148],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[148,149],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[149,150],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[150,151],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[151,152],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[152,153],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[153,154],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[154,155],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[155,156],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[156,157],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[157,158],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[158,159],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[159,160],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[160,161],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[161,162],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[162,163],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[163,164],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[164,165],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[165,166],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[166,167],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[167,168],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[168,169],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[169,170],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[170,171],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[171,172],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[172,173],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[173,174],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[174,175],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[175,176],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[176,177],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[177,178],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[178,179],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[179,180],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[180,181],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[181,182],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[182,183],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[183,184],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[184,185],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[185,186],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[186,187],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[187,188],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[188,189],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[189,190],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[190,191],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[191,192],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[192,193],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[193,194],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[194,195],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[195,196],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[196,197],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[197,198],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[198,199],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[199,200],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[200,201],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[201,202],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[202,203],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[203,204],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[204,205],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[205,206],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[206,207],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[207,208],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[208,209],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[209,210],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[210,211],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[211,212],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[212,213],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[213,214],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[214,215],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[215,216],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[216,217],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[217,218],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[218,219],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[219,220],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[220,221],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[221,222],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[222,223],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[223,224],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[224,225],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[225,226],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[226,227],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[227,228],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[228,229],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[229,230],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[230,231],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[231,232],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[232,233],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[233,234],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[234,235],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[235,236],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[236,237],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[237,238],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[238,239],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[239,240],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[240,241],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[261,262],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[262,263],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[263,264],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[264,265],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[265,266],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[266,267],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[267,268],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[268,269],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[269,270],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[270,271],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[271,272],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[272,273],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[273,274],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[274,275],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[275,276],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[276,277],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[277,278],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[278,279],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[279,280],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[280,281],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[281,282],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[282,283],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[283,284],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[284,285],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[285,286],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[286,287],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[287,288],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[288,289],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[289,290],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[290,291],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[291,292],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"}],                        {\"template\":{\"data\":{\"histogram2dcontour\":[{\"type\":\"histogram2dcontour\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"choropleth\":[{\"type\":\"choropleth\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}],\"histogram2d\":[{\"type\":\"histogram2d\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"heatmap\":[{\"type\":\"heatmap\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"heatmapgl\":[{\"type\":\"heatmapgl\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"contourcarpet\":[{\"type\":\"contourcarpet\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}],\"contour\":[{\"type\":\"contour\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"surface\":[{\"type\":\"surface\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"mesh3d\":[{\"type\":\"mesh3d\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}],\"scatter\":[{\"fillpattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2},\"type\":\"scatter\"}],\"parcoords\":[{\"type\":\"parcoords\",\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scatterpolargl\":[{\"type\":\"scatterpolargl\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"bar\":[{\"error_x\":{\"color\":\"#2a3f5f\"},\"error_y\":{\"color\":\"#2a3f5f\"},\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"bar\"}],\"scattergeo\":[{\"type\":\"scattergeo\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scatterpolar\":[{\"type\":\"scatterpolar\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"histogram\":[{\"marker\":{\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"histogram\"}],\"scattergl\":[{\"type\":\"scattergl\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scatter3d\":[{\"type\":\"scatter3d\",\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scattermapbox\":[{\"type\":\"scattermapbox\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scatterternary\":[{\"type\":\"scatterternary\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scattercarpet\":[{\"type\":\"scattercarpet\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"carpet\":[{\"aaxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"baxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"type\":\"carpet\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#EBF0F8\"},\"line\":{\"color\":\"white\"}},\"header\":{\"fill\":{\"color\":\"#C8D4E3\"},\"line\":{\"color\":\"white\"}},\"type\":\"table\"}],\"barpolar\":[{\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"barpolar\"}],\"pie\":[{\"automargin\":true,\"type\":\"pie\"}]},\"layout\":{\"autotypenumbers\":\"strict\",\"colorway\":[\"#636efa\",\"#EF553B\",\"#00cc96\",\"#ab63fa\",\"#FFA15A\",\"#19d3f3\",\"#FF6692\",\"#B6E880\",\"#FF97FF\",\"#FECB52\"],\"font\":{\"color\":\"#2a3f5f\"},\"hovermode\":\"closest\",\"hoverlabel\":{\"align\":\"left\"},\"paper_bgcolor\":\"white\",\"plot_bgcolor\":\"#E5ECF6\",\"polar\":{\"bgcolor\":\"#E5ECF6\",\"angularaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"radialaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"ternary\":{\"bgcolor\":\"#E5ECF6\",\"aaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"baxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"caxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"coloraxis\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"colorscale\":{\"sequential\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"sequentialminus\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"diverging\":[[0,\"#8e0152\"],[0.1,\"#c51b7d\"],[0.2,\"#de77ae\"],[0.3,\"#f1b6da\"],[0.4,\"#fde0ef\"],[0.5,\"#f7f7f7\"],[0.6,\"#e6f5d0\"],[0.7,\"#b8e186\"],[0.8,\"#7fbc41\"],[0.9,\"#4d9221\"],[1,\"#276419\"]]},\"xaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"automargin\":true,\"zerolinewidth\":2},\"yaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"automargin\":true,\"zerolinewidth\":2},\"scene\":{\"xaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\",\"gridwidth\":2},\"yaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\",\"gridwidth\":2},\"zaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\",\"gridwidth\":2}},\"shapedefaults\":{\"line\":{\"color\":\"#2a3f5f\"}},\"annotationdefaults\":{\"arrowcolor\":\"#2a3f5f\",\"arrowhead\":0,\"arrowwidth\":1},\"geo\":{\"bgcolor\":\"white\",\"landcolor\":\"#E5ECF6\",\"subunitcolor\":\"white\",\"showland\":true,\"showlakes\":true,\"lakecolor\":\"white\"},\"title\":{\"x\":0.05},\"mapbox\":{\"style\":\"light\"}}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0]},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0]},\"title\":{\"text\":\"LLPS Prediction Plot\"},\"height\":800,\"width\":1700},                        {\"responsive\": true}                    ).then(function(){\n",
              "                            \n",
              "var gd = document.getElementById('376248ba-66c4-4b05-8fca-ce9ede94be5a');\n",
              "var x = new MutationObserver(function (mutations, observer) {{\n",
              "        var display = window.getComputedStyle(gd).display;\n",
              "        if (!display || display === 'none') {{\n",
              "            console.log([gd, 'removed!']);\n",
              "            Plotly.purge(gd);\n",
              "            observer.disconnect();\n",
              "        }}\n",
              "}});\n",
              "\n",
              "// Listen for the removal of the full notebook cells\n",
              "var notebookContainer = gd.closest('#notebook-container');\n",
              "if (notebookContainer) {{\n",
              "    x.observe(notebookContainer, {childList: true});\n",
              "}}\n",
              "\n",
              "// Listen for the clearing of the current output cell\n",
              "var outputEl = gd.closest('.output');\n",
              "if (outputEl) {{\n",
              "    x.observe(outputEl, {childList: true});\n",
              "}}\n",
              "\n",
              "                        })                };                            </script>        </div>\n",
              "</body>\n",
              "</html>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "2024-08-29 11:58:04.276 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.281 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.289 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.291 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.295 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.297 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.300 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.305 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.307 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.309 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.315 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.317 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.319 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.320 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.324 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.326 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.328 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.331 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.333 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.337 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:04.339 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17249326855875523\"  style=\"position: relative; width: 1000px; height: 800px;\">\n        <p id=\"3dmolwarning_17249326855875523\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.3.0/3Dmol-min.js');\n}\n\nvar viewer_17249326855875523 = null;\nvar warn = document.getElementById(\"3dmolwarning_17249326855875523\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17249326855875523 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17249326855875523\"),{backgroundColor:\"white\"});\nviewer_17249326855875523.zoomTo();\n\tviewer_17249326855875523.addModel(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   MET A   1      23.752   3.572 -41.536  1.00  0.41           N  \\nATOM      2  CA  MET A   1      22.656   4.343 -42.116  1.00  0.51           C  \\nATOM      3  C   MET A   1      21.409   3.480 -42.276  1.00  0.42           C  \\nATOM      4  CB  MET A   1      23.065   4.928 -43.468  1.00  0.40           C  \\nATOM      5  O   MET A   1      21.316   2.683 -43.212  1.00  0.42           O  \\nATOM      6  CG  MET A   1      24.094   6.042 -43.369  1.00  0.42           C  \\nATOM      7  SD  MET A   1      24.535   6.741 -45.007  1.00  0.41           S  \\nATOM      8  CE  MET A   1      25.852   7.890 -44.521  1.00  0.39           C  \\nATOM      9  N   ALA A   2      20.842   2.859 -41.170  1.00  0.52           N  \\nATOM     10  CA  ALA A   2      19.464   2.619 -40.750  1.00  0.54           C  \\nATOM     11  C   ALA A   2      19.271   2.967 -39.277  1.00  0.53           C  \\nATOM     12  CB  ALA A   2      19.076   1.164 -41.005  1.00  0.47           C  \\nATOM     13  O   ALA A   2      19.908   2.374 -38.404  1.00  0.50           O  \\nATOM     14  N   SER A   3      19.334   4.290 -38.985  1.00  0.49           N  \\nATOM     15  CA  SER A   3      18.579   4.884 -37.886  1.00  0.51           C  \\nATOM     16  C   SER A   3      17.235   4.188 -37.700  1.00  0.49           C  \\nATOM     17  CB  SER A   3      18.360   6.377 -38.131  1.00  0.45           C  \\nATOM     18  O   SER A   3      16.390   4.208 -38.598  1.00  0.47           O  \\nATOM     19  OG  SER A   3      18.869   7.143 -37.052  1.00  0.49           O  \\nATOM     20  N   ASN A   4      17.145   2.994 -37.422  1.00  0.52           N  \\nATOM     21  CA  ASN A   4      15.959   2.278 -36.964  1.00  0.56           C  \\nATOM     22  C   ASN A   4      15.155   3.106 -35.965  1.00  0.52           C  \\nATOM     23  CB  ASN A   4      16.348   0.933 -36.347  1.00  0.47           C  \\nATOM     24  O   ASN A   4      15.616   3.362 -34.852  1.00  0.49           O  \\nATOM     25  CG  ASN A   4      16.044  -0.238 -37.260  1.00  0.45           C  \\nATOM     26  ND2 ASN A   4      16.374  -1.443 -36.810  1.00  0.42           N  \\nATOM     27  OD1 ASN A   4      15.517  -0.062 -38.362  1.00  0.46           O  \\nATOM     28  N   ASP A   5      14.617   4.301 -36.512  1.00  0.46           N  \\nATOM     29  CA  ASP A   5      13.463   5.136 -36.191  1.00  0.48           C  \\nATOM     30  C   ASP A   5      12.356   4.314 -35.534  1.00  0.46           C  \\nATOM     31  CB  ASP A   5      12.930   5.823 -37.450  1.00  0.42           C  \\nATOM     32  O   ASP A   5      11.548   3.688 -36.223  1.00  0.44           O  \\nATOM     33  CG  ASP A   5      13.313   7.290 -37.531  1.00  0.41           C  \\nATOM     34  OD1 ASP A   5      13.719   7.873 -36.503  1.00  0.45           O  \\nATOM     35  OD2 ASP A   5      13.205   7.870 -38.633  1.00  0.50           O  \\nATOM     36  N   TYR A   6      12.669   3.370 -34.615  1.00  0.51           N  \\nATOM     37  CA  TYR A   6      11.709   2.589 -33.843  1.00  0.54           C  \\nATOM     38  C   TYR A   6      11.235   3.364 -32.619  1.00  0.50           C  \\nATOM     39  CB  TYR A   6      12.325   1.256 -33.410  1.00  0.47           C  \\nATOM     40  O   TYR A   6      10.373   2.893 -31.874  1.00  0.48           O  \\nATOM     41  CG  TYR A   6      12.326   0.207 -34.495  1.00  0.46           C  \\nATOM     42  CD1 TYR A   6      11.196  -0.569 -34.743  1.00  0.43           C  \\nATOM     43  CD2 TYR A   6      13.457  -0.011 -35.274  1.00  0.45           C  \\nATOM     44  CE1 TYR A   6      11.194  -1.538 -35.740  1.00  0.43           C  \\nATOM     45  CE2 TYR A   6      13.466  -0.977 -36.275  1.00  0.46           C  \\nATOM     46  OH  TYR A   6      12.334  -2.692 -37.490  1.00  0.33           O  \\nATOM     47  CZ  TYR A   6      12.331  -1.734 -36.500  1.00  0.37           C  \\nATOM     48  N   THR A   7      11.370   4.658 -32.630  1.00  0.45           N  \\nATOM     49  CA  THR A   7      10.948   5.584 -31.584  1.00  0.46           C  \\nATOM     50  C   THR A   7       9.886   6.545 -32.110  1.00  0.44           C  \\nATOM     51  CB  THR A   7      12.142   6.385 -31.033  1.00  0.41           C  \\nATOM     52  O   THR A   7       9.059   7.044 -31.343  1.00  0.42           O  \\nATOM     53  CG2 THR A   7      12.247   6.243 -29.518  1.00  0.39           C  \\nATOM     54  OG1 THR A   7      13.351   5.903 -31.632  1.00  0.38           O  \\nATOM     55  N   GLN A   8       9.220   6.333 -33.334  1.00  0.42           N  \\nATOM     56  CA  GLN A   8       8.232   7.287 -33.824  1.00  0.43           C  \\nATOM     57  C   GLN A   8       7.009   6.571 -34.392  1.00  0.42           C  \\nATOM     58  CB  GLN A   8       8.846   8.199 -34.887  1.00  0.38           C  \\nATOM     59  O   GLN A   8       5.927   7.154 -34.480  1.00  0.39           O  \\nATOM     60  CG  GLN A   8       9.431   9.489 -34.328  1.00  0.38           C  \\nATOM     61  CD  GLN A   8       9.987  10.398 -35.408  1.00  0.45           C  \\nATOM     62  NE2 GLN A   8      10.571  11.518 -34.997  1.00  0.35           N  \\nATOM     63  OE1 GLN A   8       9.895  10.094 -36.602  1.00  0.47           O  \\nATOM     64  N   GLN A   9       6.645   5.288 -33.989  1.00  0.38           N  \\nATOM     65  CA  GLN A   9       5.350   4.796 -34.448  1.00  0.41           C  \\nATOM     66  C   GLN A   9       4.701   3.896 -33.400  1.00  0.38           C  \\nATOM     67  CB  GLN A   9       5.499   4.040 -35.769  1.00  0.35           C  \\nATOM     68  O   GLN A   9       3.544   3.495 -33.549  1.00  0.36           O  \\nATOM     69  CG  GLN A   9       5.371   4.925 -37.001  1.00  0.35           C  \\nATOM     70  CD  GLN A   9       5.424   4.138 -38.297  1.00  0.44           C  \\nATOM     71  NE2 GLN A   9       5.320   4.839 -39.421  1.00  0.34           N  \\nATOM     72  OE1 GLN A   9       5.557   2.910 -38.288  1.00  0.45           O  \\nATOM     73  N   ALA A  10       4.979   4.125 -32.081  1.00  0.39           N  \\nATOM     74  CA  ALA A  10       4.090   3.523 -31.091  1.00  0.40           C  \\nATOM     75  C   ALA A  10       3.500   4.584 -30.166  1.00  0.39           C  \\nATOM     76  CB  ALA A  10       4.837   2.468 -30.278  1.00  0.36           C  \\nATOM     77  O   ALA A  10       2.874   4.256 -29.155  1.00  0.37           O  \\nATOM     78  N   THR A  11       3.317   5.785 -30.721  1.00  0.39           N  \\nATOM     79  CA  THR A  11       2.437   6.881 -30.331  1.00  0.41           C  \\nATOM     80  C   THR A  11       1.463   7.220 -31.456  1.00  0.39           C  \\nATOM     81  CB  THR A  11       3.243   8.136 -29.950  1.00  0.36           C  \\nATOM     82  O   THR A  11       1.804   7.969 -32.374  1.00  0.37           O  \\nATOM     83  CG2 THR A  11       2.746   8.732 -28.637  1.00  0.36           C  \\nATOM     84  OG1 THR A  11       4.625   7.785 -29.810  1.00  0.33           O  \\nATOM     85  N   GLN A  12       0.909   6.243 -32.205  1.00  0.36           N  \\nATOM     86  CA  GLN A  12      -0.147   6.670 -33.117  1.00  0.39           C  \\nATOM     87  C   GLN A  12      -1.514   6.190 -32.639  1.00  0.36           C  \\nATOM     88  CB  GLN A  12       0.125   6.156 -34.532  1.00  0.33           C  \\nATOM     89  O   GLN A  12      -1.705   5.000 -32.381  1.00  0.34           O  \\nATOM     90  CG  GLN A  12       1.091   7.023 -35.327  1.00  0.33           C  \\nATOM     91  CD  GLN A  12       1.263   6.550 -36.758  1.00  0.41           C  \\nATOM     92  NE2 GLN A  12       2.064   7.277 -37.530  1.00  0.31           N  \\nATOM     93  OE1 GLN A  12       0.682   5.540 -37.166  1.00  0.42           O  \\nATOM     94  N   SER A  13      -2.103   6.912 -31.568  1.00  0.33           N  \\nATOM     95  CA  SER A  13      -3.487   7.354 -31.438  1.00  0.37           C  \\nATOM     96  C   SER A  13      -4.327   6.328 -30.684  1.00  0.34           C  \\nATOM     97  CB  SER A  13      -4.100   7.613 -32.815  1.00  0.31           C  \\nATOM     98  O   SER A  13      -4.540   5.215 -31.170  1.00  0.32           O  \\nATOM     99  OG  SER A  13      -3.796   6.555 -33.708  1.00  0.32           O  \\nATOM    100  N   TYR A  14      -4.104   6.052 -29.391  1.00  0.39           N  \\nATOM    101  CA  TYR A  14      -5.405   5.993 -28.735  1.00  0.41           C  \\nATOM    102  C   TYR A  14      -6.428   6.846 -29.474  1.00  0.39           C  \\nATOM    103  CB  TYR A  14      -5.294   6.454 -27.279  1.00  0.37           C  \\nATOM    104  O   TYR A  14      -6.413   8.075 -29.368  1.00  0.38           O  \\nATOM    105  CG  TYR A  14      -4.689   5.422 -26.359  1.00  0.37           C  \\nATOM    106  CD1 TYR A  14      -5.479   4.437 -25.770  1.00  0.35           C  \\nATOM    107  CD2 TYR A  14      -3.327   5.429 -26.076  1.00  0.37           C  \\nATOM    108  CE1 TYR A  14      -4.926   3.485 -24.920  1.00  0.36           C  \\nATOM    109  CE2 TYR A  14      -2.764   4.482 -25.228  1.00  0.40           C  \\nATOM    110  OH  TYR A  14      -3.017   2.575 -23.815  1.00  0.26           O  \\nATOM    111  CZ  TYR A  14      -3.569   3.515 -24.656  1.00  0.32           C  \\nATOM    112  N   GLY A  15      -6.563   6.790 -30.734  1.00  0.33           N  \\nATOM    113  CA  GLY A  15      -7.605   7.343 -31.585  1.00  0.36           C  \\nATOM    114  C   GLY A  15      -8.995   6.856 -31.219  1.00  0.34           C  \\nATOM    115  O   GLY A  15      -9.212   5.655 -31.047  1.00  0.32           O  \\nATOM    116  N   ALA A  16      -9.523   7.556 -30.118  1.00  0.34           N  \\nATOM    117  CA  ALA A  16     -10.950   7.860 -30.064  1.00  0.36           C  \\nATOM    118  C   ALA A  16     -11.785   6.583 -30.096  1.00  0.35           C  \\nATOM    119  CB  ALA A  16     -11.343   8.778 -31.219  1.00  0.32           C  \\nATOM    120  O   ALA A  16     -11.810   5.876 -31.106  1.00  0.33           O  \\nATOM    121  N   TYR A  17     -11.623   5.630 -29.177  1.00  0.42           N  \\nATOM    122  CA  TYR A  17     -12.884   4.911 -29.034  1.00  0.44           C  \\nATOM    123  C   TYR A  17     -14.052   5.753 -29.533  1.00  0.42           C  \\nATOM    124  CB  TYR A  17     -13.114   4.514 -27.573  1.00  0.40           C  \\nATOM    125  O   TYR A  17     -14.176   6.927 -29.177  1.00  0.41           O  \\nATOM    126  CG  TYR A  17     -12.215   3.398 -27.098  1.00  0.40           C  \\nATOM    127  CD1 TYR A  17     -12.535   2.065 -27.348  1.00  0.38           C  \\nATOM    128  CD2 TYR A  17     -11.044   3.674 -26.399  1.00  0.39           C  \\nATOM    129  CE1 TYR A  17     -11.710   1.034 -26.911  1.00  0.39           C  \\nATOM    130  CE2 TYR A  17     -10.212   2.650 -25.957  1.00  0.42           C  \\nATOM    131  OH  TYR A  17      -9.733   0.318 -25.783  1.00  0.31           O  \\nATOM    132  CZ  TYR A  17     -10.553   1.335 -26.218  1.00  0.35           C  \\nATOM    133  N   PRO A  18     -14.488   5.596 -30.788  1.00  0.38           N  \\nATOM    134  CA  PRO A  18     -15.769   6.200 -31.162  1.00  0.40           C  \\nATOM    135  C   PRO A  18     -16.716   6.359 -29.974  1.00  0.39           C  \\nATOM    136  CB  PRO A  18     -16.339   5.214 -32.185  1.00  0.36           C  \\nATOM    137  O   PRO A  18     -17.064   5.372 -29.321  1.00  0.37           O  \\nATOM    138  CG  PRO A  18     -15.712   3.901 -31.841  1.00  0.36           C  \\nATOM    139  CD  PRO A  18     -14.423   4.164 -31.116  1.00  0.41           C  \\nATOM    140  N   THR A  19     -16.547   7.364 -28.978  1.00  0.37           N  \\nATOM    141  CA  THR A  19     -17.746   7.875 -28.324  1.00  0.38           C  \\nATOM    142  C   THR A  19     -18.988   7.570 -29.156  1.00  0.38           C  \\nATOM    143  CB  THR A  19     -17.647   9.393 -28.084  1.00  0.36           C  \\nATOM    144  O   THR A  19     -19.086   7.987 -30.312  1.00  0.36           O  \\nATOM    145  CG2 THR A  19     -17.876   9.734 -26.615  1.00  0.35           C  \\nATOM    146  OG1 THR A  19     -16.344   9.848 -28.471  1.00  0.33           O  \\nATOM    147  N   GLN A  20     -19.432   6.414 -29.142  1.00  0.44           N  \\nATOM    148  CA  GLN A  20     -20.834   6.301 -29.528  1.00  0.47           C  \\nATOM    149  C   GLN A  20     -21.584   7.604 -29.266  1.00  0.45           C  \\nATOM    150  CB  GLN A  20     -21.506   5.149 -28.778  1.00  0.42           C  \\nATOM    151  O   GLN A  20     -21.412   8.225 -28.215  1.00  0.43           O  \\nATOM    152  CG  GLN A  20     -21.338   3.795 -29.453  1.00  0.42           C  \\nATOM    153  CD  GLN A  20     -22.219   2.721 -28.842  1.00  0.47           C  \\nATOM    154  NE2 GLN A  20     -22.409   1.625 -29.568  1.00  0.40           N  \\nATOM    155  OE1 GLN A  20     -22.725   2.876 -27.726  1.00  0.51           O  \\nATOM    156  N   PRO A  21     -21.808   8.485 -30.228  1.00  0.41           N  \\nATOM    157  CA  PRO A  21     -22.781   9.572 -30.096  1.00  0.43           C  \\nATOM    158  C   PRO A  21     -23.903   9.243 -29.114  1.00  0.41           C  \\nATOM    159  CB  PRO A  21     -23.328   9.731 -31.517  1.00  0.39           C  \\nATOM    160  O   PRO A  21     -24.713   8.350 -29.374  1.00  0.40           O  \\nATOM    161  CG  PRO A  21     -22.751   8.582 -32.278  1.00  0.41           C  \\nATOM    162  CD  PRO A  21     -21.798   7.845 -31.382  1.00  0.47           C  \\nATOM    163  N   GLY A  22     -23.555   9.317 -27.787  1.00  0.39           N  \\nATOM    164  CA  GLY A  22     -24.570   9.422 -26.751  1.00  0.42           C  \\nATOM    165  C   GLY A  22     -24.145   8.800 -25.434  1.00  0.40           C  \\nATOM    166  O   GLY A  22     -24.929   8.751 -24.484  1.00  0.37           O  \\nATOM    167  N   GLN A  23     -22.798   8.690 -25.122  1.00  0.40           N  \\nATOM    168  CA  GLN A  23     -22.318   8.457 -23.764  1.00  0.41           C  \\nATOM    169  C   GLN A  23     -21.023   9.221 -23.499  1.00  0.40           C  \\nATOM    170  CB  GLN A  23     -22.105   6.962 -23.518  1.00  0.37           C  \\nATOM    171  O   GLN A  23     -19.997   8.948 -24.125  1.00  0.39           O  \\nATOM    172  CG  GLN A  23     -23.378   6.212 -23.152  1.00  0.37           C  \\nATOM    173  CD  GLN A  23     -23.113   4.783 -22.716  1.00  0.43           C  \\nATOM    174  NE2 GLN A  23     -24.167   4.080 -22.314  1.00  0.36           N  \\nATOM    175  OE1 GLN A  23     -21.971   4.314 -22.741  1.00  0.48           O  \\nATOM    176  N   GLY A  24     -21.081  10.477 -23.333  1.00  0.40           N  \\nATOM    177  CA  GLY A  24     -20.282  11.480 -22.647  1.00  0.41           C  \\nATOM    178  C   GLY A  24     -19.443  10.906 -21.520  1.00  0.40           C  \\nATOM    179  O   GLY A  24     -19.952  10.168 -20.674  1.00  0.38           O  \\nATOM    180  N   TYR A  25     -18.326  10.342 -21.711  1.00  0.42           N  \\nATOM    181  CA  TYR A  25     -17.347  10.220 -20.637  1.00  0.44           C  \\nATOM    182  C   TYR A  25     -17.659  11.188 -19.502  1.00  0.42           C  \\nATOM    183  CB  TYR A  25     -15.933  10.475 -21.167  1.00  0.40           C  \\nATOM    184  O   TYR A  25     -17.474  12.399 -19.642  1.00  0.41           O  \\nATOM    185  CG  TYR A  25     -15.364   9.322 -21.957  1.00  0.40           C  \\nATOM    186  CD1 TYR A  25     -14.742   8.252 -21.316  1.00  0.38           C  \\nATOM    187  CD2 TYR A  25     -15.445   9.300 -23.345  1.00  0.39           C  \\nATOM    188  CE1 TYR A  25     -14.214   7.188 -22.040  1.00  0.37           C  \\nATOM    189  CE2 TYR A  25     -14.920   8.241 -24.079  1.00  0.42           C  \\nATOM    190  OH  TYR A  25     -13.788   6.141 -24.141  1.00  0.27           O  \\nATOM    191  CZ  TYR A  25     -14.308   7.191 -23.419  1.00  0.34           C  \\nATOM    192  N   SER A  26     -18.642  11.047 -18.725  1.00  0.39           N  \\nATOM    193  CA  SER A  26     -18.791  11.609 -17.387  1.00  0.41           C  \\nATOM    194  C   SER A  26     -17.579  11.293 -16.517  1.00  0.39           C  \\nATOM    195  CB  SER A  26     -20.060  11.078 -16.721  1.00  0.37           C  \\nATOM    196  O   SER A  26     -17.203  10.129 -16.366  1.00  0.38           O  \\nATOM    197  OG  SER A  26     -20.668  10.078 -17.520  1.00  0.35           O  \\nATOM    198  N   GLN A  27     -16.404  11.914 -16.658  1.00  0.42           N  \\nATOM    199  CA  GLN A  27     -15.580  12.078 -15.465  1.00  0.43           C  \\nATOM    200  C   GLN A  27     -16.203  11.370 -14.265  1.00  0.42           C  \\nATOM    201  CB  GLN A  27     -15.376  13.561 -15.152  1.00  0.39           C  \\nATOM    202  O   GLN A  27     -17.242  11.797 -13.756  1.00  0.39           O  \\nATOM    203  CG  GLN A  27     -14.204  14.190 -15.893  1.00  0.38           C  \\nATOM    204  CD  GLN A  27     -14.016  15.658 -15.557  1.00  0.43           C  \\nATOM    205  NE2 GLN A  27     -12.992  16.273 -16.138  1.00  0.36           N  \\nATOM    206  OE1 GLN A  27     -14.786  16.234 -14.781  1.00  0.48           O  \\nATOM    207  N   GLN A  28     -16.396  10.119 -14.200  1.00  0.36           N  \\nATOM    208  CA  GLN A  28     -16.601   9.405 -12.944  1.00  0.39           C  \\nATOM    209  C   GLN A  28     -15.650   9.909 -11.863  1.00  0.37           C  \\nATOM    210  CB  GLN A  28     -16.417   7.900 -13.145  1.00  0.35           C  \\nATOM    211  O   GLN A  28     -14.430   9.807 -12.005  1.00  0.35           O  \\nATOM    212  CG  GLN A  28     -17.722   7.115 -13.146  1.00  0.34           C  \\nATOM    213  CD  GLN A  28     -17.520   5.641 -13.440  1.00  0.42           C  \\nATOM    214  NE2 GLN A  28     -18.619   4.899 -13.537  1.00  0.34           N  \\nATOM    215  OE1 GLN A  28     -16.386   5.171 -13.579  1.00  0.44           O  \\nATOM    216  N   SER A  29     -15.814  11.041 -11.268  1.00  0.34           N  \\nATOM    217  CA  SER A  29     -15.737  11.546  -9.901  1.00  0.36           C  \\nATOM    218  C   SER A  29     -15.379  10.433  -8.921  1.00  0.34           C  \\nATOM    219  CB  SER A  29     -17.061  12.191  -9.491  1.00  0.32           C  \\nATOM    220  O   SER A  29     -15.979   9.357  -8.949  1.00  0.32           O  \\nATOM    221  OG  SER A  29     -17.942  11.225  -8.942  1.00  0.32           O  \\nATOM    222  N   SER A  30     -14.009  10.021  -8.765  1.00  0.36           N  \\nATOM    223  CA  SER A  30     -13.524   9.561  -7.468  1.00  0.38           C  \\nATOM    224  C   SER A  30     -14.675   9.098  -6.581  1.00  0.36           C  \\nATOM    225  CB  SER A  30     -12.741  10.669  -6.763  1.00  0.34           C  \\nATOM    226  O   SER A  30     -15.326   9.913  -5.924  1.00  0.35           O  \\nATOM    227  OG  SER A  30     -13.522  11.847  -6.656  1.00  0.36           O  \\nATOM    228  N   GLN A  31     -15.541   8.380  -7.018  1.00  0.37           N  \\nATOM    229  CA  GLN A  31     -16.294   7.593  -6.046  1.00  0.41           C  \\nATOM    230  C   GLN A  31     -15.370   6.685  -5.240  1.00  0.38           C  \\nATOM    231  CB  GLN A  31     -17.369   6.760  -6.746  1.00  0.36           C  \\nATOM    232  O   GLN A  31     -14.481   6.040  -5.800  1.00  0.37           O  \\nATOM    233  CG  GLN A  31     -18.569   7.573  -7.213  1.00  0.37           C  \\nATOM    234  CD  GLN A  31     -19.880   6.823  -7.064  1.00  0.45           C  \\nATOM    235  NE2 GLN A  31     -20.954   7.555  -6.788  1.00  0.36           N  \\nATOM    236  OE1 GLN A  31     -19.927   5.596  -7.197  1.00  0.48           O  \\nATOM    237  N   PRO A  32     -14.884   6.989  -4.080  1.00  0.42           N  \\nATOM    238  CA  PRO A  32     -14.368   5.960  -3.174  1.00  0.41           C  \\nATOM    239  C   PRO A  32     -14.927   4.572  -3.479  1.00  0.42           C  \\nATOM    240  CB  PRO A  32     -14.826   6.442  -1.795  1.00  0.40           C  \\nATOM    241  O   PRO A  32     -16.109   4.436  -3.805  1.00  0.40           O  \\nATOM    242  CG  PRO A  32     -15.993   7.334  -2.071  1.00  0.39           C  \\nATOM    243  CD  PRO A  32     -15.911   7.801  -3.495  1.00  0.43           C  \\nATOM    244  N   TYR A  33     -14.640   3.788  -4.556  1.00  0.40           N  \\nATOM    245  CA  TYR A  33     -15.226   2.521  -4.980  1.00  0.42           C  \\nATOM    246  C   TYR A  33     -16.747   2.609  -5.017  1.00  0.40           C  \\nATOM    247  CB  TYR A  33     -14.792   1.388  -4.045  1.00  0.38           C  \\nATOM    248  O   TYR A  33     -17.390   2.793  -3.980  1.00  0.38           O  \\nATOM    249  CG  TYR A  33     -13.366   0.940  -4.256  1.00  0.37           C  \\nATOM    250  CD1 TYR A  33     -13.045   0.015  -5.247  1.00  0.37           C  \\nATOM    251  CD2 TYR A  33     -12.337   1.441  -3.465  1.00  0.38           C  \\nATOM    252  CE1 TYR A  33     -11.732  -0.402  -5.443  1.00  0.38           C  \\nATOM    253  CE2 TYR A  33     -11.021   1.031  -3.653  1.00  0.40           C  \\nATOM    254  OH  TYR A  33      -9.428  -0.298  -4.833  1.00  0.27           O  \\nATOM    255  CZ  TYR A  33     -10.729   0.111  -4.643  1.00  0.35           C  \\nATOM    256  N   GLY A  34     -17.383   3.354  -5.722  1.00  0.33           N  \\nATOM    257  CA  GLY A  34     -18.770   3.154  -6.110  1.00  0.34           C  \\nATOM    258  C   GLY A  34     -19.201   1.701  -6.055  1.00  0.33           C  \\nATOM    259  O   GLY A  34     -18.511   0.823  -6.577  1.00  0.32           O  \\nATOM    260  N   GLN A  35     -19.332   1.030  -4.828  1.00  0.41           N  \\nATOM    261  CA  GLN A  35     -20.401   0.055  -5.015  1.00  0.42           C  \\nATOM    262  C   GLN A  35     -20.787  -0.063  -6.486  1.00  0.41           C  \\nATOM    263  CB  GLN A  35     -21.626   0.434  -4.180  1.00  0.38           C  \\nATOM    264  O   GLN A  35     -21.485   0.800  -7.022  1.00  0.39           O  \\nATOM    265  CG  GLN A  35     -21.605  -0.133  -2.766  1.00  0.37           C  \\nATOM    266  CD  GLN A  35     -22.792   0.318  -1.936  1.00  0.43           C  \\nATOM    267  NE2 GLN A  35     -23.007  -0.340  -0.802  1.00  0.36           N  \\nATOM    268  OE1 GLN A  35     -23.510   1.251  -2.311  1.00  0.47           O  \\nATOM    269  N   GLN A  36     -19.905  -0.360  -7.366  1.00  0.38           N  \\nATOM    270  CA  GLN A  36     -20.645  -1.153  -8.341  1.00  0.41           C  \\nATOM    271  C   GLN A  36     -21.245  -2.399  -7.695  1.00  0.38           C  \\nATOM    272  CB  GLN A  36     -19.741  -1.551  -9.509  1.00  0.36           C  \\nATOM    273  O   GLN A  36     -20.518  -3.233  -7.151  1.00  0.36           O  \\nATOM    274  CG  GLN A  36     -19.568  -0.457 -10.553  1.00  0.35           C  \\nATOM    275  CD  GLN A  36     -18.797  -0.927 -11.773  1.00  0.41           C  \\nATOM    276  NE2 GLN A  36     -18.636  -0.040 -12.750  1.00  0.35           N  \\nATOM    277  OE1 GLN A  36     -18.350  -2.076 -11.837  1.00  0.45           O  \\nATOM    278  N   SER A  37     -22.234  -2.317  -6.825  1.00  0.35           N  \\nATOM    279  CA  SER A  37     -23.228  -3.385  -6.841  1.00  0.38           C  \\nATOM    280  C   SER A  37     -23.125  -4.217  -8.115  1.00  0.36           C  \\nATOM    281  CB  SER A  37     -24.638  -2.807  -6.712  1.00  0.34           C  \\nATOM    282  O   SER A  37     -23.435  -3.733  -9.206  1.00  0.34           O  \\nATOM    283  OG  SER A  37     -24.601  -1.391  -6.682  1.00  0.34           O  \\nATOM    284  N   TYR A  38     -22.070  -4.921  -8.414  1.00  0.39           N  \\nATOM    285  CA  TYR A  38     -22.294  -6.124  -9.207  1.00  0.43           C  \\nATOM    286  C   TYR A  38     -23.656  -6.734  -8.900  1.00  0.40           C  \\nATOM    287  CB  TYR A  38     -21.190  -7.154  -8.945  1.00  0.38           C  \\nATOM    288  O   TYR A  38     -23.817  -7.439  -7.901  1.00  0.38           O  \\nATOM    289  CG  TYR A  38     -19.948  -6.937  -9.775  1.00  0.39           C  \\nATOM    290  CD1 TYR A  38     -19.827  -7.501 -11.043  1.00  0.37           C  \\nATOM    291  CD2 TYR A  38     -18.894  -6.170  -9.293  1.00  0.39           C  \\nATOM    292  CE1 TYR A  38     -18.684  -7.305 -11.811  1.00  0.38           C  \\nATOM    293  CE2 TYR A  38     -17.745  -5.968 -10.052  1.00  0.42           C  \\nATOM    294  OH  TYR A  38     -16.516  -6.341 -12.063  1.00  0.28           O  \\nATOM    295  CZ  TYR A  38     -17.650  -6.538 -11.308  1.00  0.34           C  \\nATOM    296  N   SER A  39     -24.645  -6.042  -8.858  1.00  0.35           N  \\nATOM    297  CA  SER A  39     -25.997  -6.556  -9.049  1.00  0.39           C  \\nATOM    298  C   SER A  39     -26.253  -6.909 -10.511  1.00  0.35           C  \\nATOM    299  CB  SER A  39     -27.032  -5.535  -8.574  1.00  0.34           C  \\nATOM    300  O   SER A  39     -26.337  -6.022 -11.363  1.00  0.34           O  \\nATOM    301  OG  SER A  39     -27.108  -4.441  -9.471  1.00  0.35           O  \\nATOM    302  N   GLY A  40     -25.351  -7.737 -11.150  1.00  0.33           N  \\nATOM    303  CA  GLY A  40     -25.725  -8.343 -12.418  1.00  0.38           C  \\nATOM    304  C   GLY A  40     -24.652  -9.256 -12.982  1.00  0.34           C  \\nATOM    305  O   GLY A  40     -23.653  -8.785 -13.529  1.00  0.33           O  \\nATOM    306  N   TYR A  41     -24.147 -10.178 -12.215  1.00  0.36           N  \\nATOM    307  CA  TYR A  41     -23.750 -11.492 -12.707  1.00  0.40           C  \\nATOM    308  C   TYR A  41     -24.253 -12.595 -11.785  1.00  0.37           C  \\nATOM    309  CB  TYR A  41     -22.226 -11.578 -12.841  1.00  0.35           C  \\nATOM    310  O   TYR A  41     -23.798 -12.716 -10.645  1.00  0.35           O  \\nATOM    311  CG  TYR A  41     -21.705 -11.091 -14.171  1.00  0.36           C  \\nATOM    312  CD1 TYR A  41     -21.624 -11.948 -15.266  1.00  0.34           C  \\nATOM    313  CD2 TYR A  41     -21.292  -9.773 -14.336  1.00  0.35           C  \\nATOM    314  CE1 TYR A  41     -21.143 -11.504 -16.493  1.00  0.34           C  \\nATOM    315  CE2 TYR A  41     -20.809  -9.318 -15.558  1.00  0.39           C  \\nATOM    316  OH  TYR A  41     -20.261  -9.745 -17.842  1.00  0.22           O  \\nATOM    317  CZ  TYR A  41     -20.739 -10.190 -16.630  1.00  0.31           C  \\nATOM    318  N   SER A  42     -25.553 -12.900 -11.615  1.00  0.38           N  \\nATOM    319  CA  SER A  42     -26.034 -14.272 -11.491  1.00  0.41           C  \\nATOM    320  C   SER A  42     -25.071 -15.257 -12.144  1.00  0.38           C  \\nATOM    321  CB  SER A  42     -27.422 -14.412 -12.118  1.00  0.37           C  \\nATOM    322  O   SER A  42     -24.973 -15.317 -13.371  1.00  0.37           O  \\nATOM    323  OG  SER A  42     -28.358 -14.889 -11.167  1.00  0.38           O  \\nATOM    324  N   GLN A  43     -23.810 -15.382 -11.824  1.00  0.42           N  \\nATOM    325  CA  GLN A  43     -23.112 -16.638 -12.081  1.00  0.44           C  \\nATOM    326  C   GLN A  43     -23.939 -17.832 -11.614  1.00  0.42           C  \\nATOM    327  CB  GLN A  43     -21.747 -16.644 -11.392  1.00  0.39           C  \\nATOM    328  O   GLN A  43     -24.375 -17.881 -10.462  1.00  0.41           O  \\nATOM    329  CG  GLN A  43     -20.687 -15.827 -12.119  1.00  0.40           C  \\nATOM    330  CD  GLN A  43     -19.341 -15.855 -11.420  1.00  0.45           C  \\nATOM    331  NE2 GLN A  43     -18.434 -14.982 -11.844  1.00  0.38           N  \\nATOM    332  OE1 GLN A  43     -19.119 -16.656 -10.506  1.00  0.49           O  \\nATOM    333  N   SER A  44     -24.765 -18.322 -12.451  1.00  0.39           N  \\nATOM    334  CA  SER A  44     -25.224 -19.707 -12.405  1.00  0.43           C  \\nATOM    335  C   SER A  44     -24.279 -20.576 -11.582  1.00  0.40           C  \\nATOM    336  CB  SER A  44     -25.352 -20.276 -13.819  1.00  0.38           C  \\nATOM    337  O   SER A  44     -23.086 -20.660 -11.880  1.00  0.38           O  \\nATOM    338  OG  SER A  44     -24.087 -20.326 -14.457  1.00  0.39           O  \\nATOM    339  N   THR A  45     -24.286 -20.564 -10.283  1.00  0.41           N  \\nATOM    340  CA  THR A  45     -24.073 -21.756  -9.470  1.00  0.43           C  \\nATOM    341  C   THR A  45     -24.416 -23.017 -10.258  1.00  0.41           C  \\nATOM    342  CB  THR A  45     -24.913 -21.710  -8.180  1.00  0.39           C  \\nATOM    343  O   THR A  45     -25.582 -23.407 -10.338  1.00  0.40           O  \\nATOM    344  CG2 THR A  45     -24.215 -20.891  -7.098  1.00  0.39           C  \\nATOM    345  OG1 THR A  45     -26.186 -21.117  -8.466  1.00  0.35           O  \\nATOM    346  N   ASP A  46     -23.749 -23.224 -11.394  1.00  0.39           N  \\nATOM    347  CA  ASP A  46     -23.664 -24.651 -11.689  1.00  0.43           C  \\nATOM    348  C   ASP A  46     -22.583 -25.322 -10.845  1.00  0.39           C  \\nATOM    349  CB  ASP A  46     -23.384 -24.875 -13.176  1.00  0.37           C  \\nATOM    350  O   ASP A  46     -21.391 -25.088 -11.051  1.00  0.38           O  \\nATOM    351  CG  ASP A  46     -24.627 -25.252 -13.963  1.00  0.39           C  \\nATOM    352  OD1 ASP A  46     -25.620 -25.707 -13.355  1.00  0.42           O  \\nATOM    353  OD2 ASP A  46     -24.613 -25.095 -15.203  1.00  0.43           O  \\nATOM    354  N   THR A  47     -22.759 -25.537  -9.558  1.00  0.38           N  \\nATOM    355  CA  THR A  47     -22.334 -26.618  -8.676  1.00  0.41           C  \\nATOM    356  C   THR A  47     -22.459 -27.969  -9.376  1.00  0.38           C  \\nATOM    357  CB  THR A  47     -23.157 -26.634  -7.374  1.00  0.37           C  \\nATOM    358  O   THR A  47     -23.012 -28.916  -8.813  1.00  0.37           O  \\nATOM    359  CG2 THR A  47     -22.318 -26.173  -6.187  1.00  0.36           C  \\nATOM    360  OG1 THR A  47     -24.283 -25.759  -7.517  1.00  0.33           O  \\nATOM    361  N   SER A  48     -22.106 -27.997 -10.622  1.00  0.35           N  \\nATOM    362  CA  SER A  48     -21.935 -29.329 -11.195  1.00  0.40           C  \\nATOM    363  C   SER A  48     -20.466 -29.623 -11.481  1.00  0.36           C  \\nATOM    364  CB  SER A  48     -22.751 -29.468 -12.481  1.00  0.33           C  \\nATOM    365  O   SER A  48     -19.801 -28.861 -12.185  1.00  0.34           O  \\nATOM    366  OG  SER A  48     -22.429 -28.434 -13.395  1.00  0.36           O  \\nATOM    367  N   GLY A  49     -19.566 -29.850 -10.356  1.00  0.36           N  \\nATOM    368  CA  GLY A  49     -18.435 -30.760 -10.271  1.00  0.38           C  \\nATOM    369  C   GLY A  49     -17.096 -30.063 -10.422  1.00  0.37           C  \\nATOM    370  O   GLY A  49     -16.773 -29.554 -11.497  1.00  0.35           O  \\nATOM    371  N   TYR A  50     -16.724 -29.148  -9.497  1.00  0.39           N  \\nATOM    372  CA  TYR A  50     -15.306 -28.859  -9.318  1.00  0.42           C  \\nATOM    373  C   TYR A  50     -14.472 -30.129  -9.436  1.00  0.39           C  \\nATOM    374  CB  TYR A  50     -15.059 -28.197  -7.959  1.00  0.38           C  \\nATOM    375  O   TYR A  50     -14.575 -31.028  -8.598  1.00  0.39           O  \\nATOM    376  CG  TYR A  50     -15.528 -26.765  -7.887  1.00  0.39           C  \\nATOM    377  CD1 TYR A  50     -14.738 -25.727  -8.376  1.00  0.37           C  \\nATOM    378  CD2 TYR A  50     -16.762 -26.446  -7.331  1.00  0.38           C  \\nATOM    379  CE1 TYR A  50     -15.165 -24.405  -8.310  1.00  0.37           C  \\nATOM    380  CE2 TYR A  50     -17.200 -25.127  -7.260  1.00  0.41           C  \\nATOM    381  OH  TYR A  50     -16.824 -22.808  -7.685  1.00  0.25           O  \\nATOM    382  CZ  TYR A  50     -16.396 -24.115  -7.752  1.00  0.34           C  \\nATOM    383  N   GLY A  51     -14.282 -30.647 -10.507  1.00  0.38           N  \\nATOM    384  CA  GLY A  51     -13.161 -31.513 -10.835  1.00  0.39           C  \\nATOM    385  C   GLY A  51     -11.831 -30.982 -10.336  1.00  0.38           C  \\nATOM    386  O   GLY A  51     -11.481 -29.828 -10.593  1.00  0.37           O  \\nATOM    387  N   GLN A  52     -11.518 -31.149  -8.996  1.00  0.40           N  \\nATOM    388  CA  GLN A  52     -10.130 -31.356  -8.597  1.00  0.43           C  \\nATOM    389  C   GLN A  52      -9.194 -31.281  -9.800  1.00  0.41           C  \\nATOM    390  CB  GLN A  52      -9.970 -32.704  -7.891  1.00  0.38           C  \\nATOM    391  O   GLN A  52      -9.141 -32.208 -10.610  1.00  0.39           O  \\nATOM    392  CG  GLN A  52      -9.815 -32.592  -6.380  1.00  0.39           C  \\nATOM    393  CD  GLN A  52      -9.757 -33.944  -5.694  1.00  0.45           C  \\nATOM    394  NE2 GLN A  52      -9.645 -33.933  -4.370  1.00  0.37           N  \\nATOM    395  OE1 GLN A  52      -9.812 -34.990  -6.349  1.00  0.49           O  \\nATOM    396  N   SER A  53      -8.947 -30.150 -10.338  1.00  0.38           N  \\nATOM    397  CA  SER A  53      -7.666 -30.104 -11.036  1.00  0.41           C  \\nATOM    398  C   SER A  53      -6.505 -29.985 -10.053  1.00  0.38           C  \\nATOM    399  CB  SER A  53      -7.633 -28.934 -12.020  1.00  0.37           C  \\nATOM    400  O   SER A  53      -6.447 -29.039  -9.265  1.00  0.37           O  \\nATOM    401  OG  SER A  53      -7.621 -27.696 -11.330  1.00  0.37           O  \\nATOM    402  N   SER A  54      -6.117 -31.056  -9.430  1.00  0.35           N  \\nATOM    403  CA  SER A  54      -4.749 -31.363  -9.025  1.00  0.38           C  \\nATOM    404  C   SER A  54      -3.737 -30.600  -9.873  1.00  0.36           C  \\nATOM    405  CB  SER A  54      -4.484 -32.865  -9.128  1.00  0.34           C  \\nATOM    406  O   SER A  54      -3.613 -30.849 -11.074  1.00  0.34           O  \\nATOM    407  OG  SER A  54      -4.796 -33.343 -10.426  1.00  0.35           O  \\nATOM    408  N   TYR A  55      -3.497 -29.346  -9.660  1.00  0.40           N  \\nATOM    409  CA  TYR A  55      -2.211 -28.769 -10.035  1.00  0.43           C  \\nATOM    410  C   TYR A  55      -1.060 -29.557  -9.421  1.00  0.40           C  \\nATOM    411  CB  TYR A  55      -2.131 -27.303  -9.598  1.00  0.39           C  \\nATOM    412  O   TYR A  55      -0.824 -29.485  -8.213  1.00  0.39           O  \\nATOM    413  CG  TYR A  55      -2.857 -26.355 -10.522  1.00  0.40           C  \\nATOM    414  CD1 TYR A  55      -2.226 -25.828 -11.647  1.00  0.39           C  \\nATOM    415  CD2 TYR A  55      -4.173 -25.982 -10.270  1.00  0.40           C  \\nATOM    416  CE1 TYR A  55      -2.890 -24.952 -12.499  1.00  0.38           C  \\nATOM    417  CE2 TYR A  55      -4.846 -25.107 -11.116  1.00  0.42           C  \\nATOM    418  OH  TYR A  55      -4.860 -23.732 -13.067  1.00  0.27           O  \\nATOM    419  CZ  TYR A  55      -4.198 -24.598 -12.226  1.00  0.36           C  \\nATOM    420  N   SER A  56      -0.783 -30.681  -9.821  1.00  0.34           N  \\nATOM    421  CA  SER A  56       0.402 -31.517  -9.663  1.00  0.38           C  \\nATOM    422  C   SER A  56       1.310 -31.425 -10.885  1.00  0.34           C  \\nATOM    423  CB  SER A  56       0.003 -32.974  -9.424  1.00  0.33           C  \\nATOM    424  O   SER A  56       0.915 -31.805 -11.989  1.00  0.34           O  \\nATOM    425  OG  SER A  56      -0.850 -33.438 -10.456  1.00  0.34           O  \\nATOM    426  N   SER A  57       1.967 -30.296 -11.115  1.00  0.30           N  \\nATOM    427  CA  SER A  57       3.127 -30.608 -11.943  1.00  0.35           C  \\nATOM    428  C   SER A  57       3.864 -29.340 -12.362  1.00  0.31           C  \\nATOM    429  CB  SER A  57       2.705 -31.395 -13.184  1.00  0.29           C  \\nATOM    430  O   SER A  57       3.251 -28.397 -12.867  1.00  0.30           O  \\nATOM    431  OG  SER A  57       1.802 -30.641 -13.975  1.00  0.30           O  \\nATOM    432  N   TYR A  58       4.966 -28.838 -11.252  1.00  0.32           N  \\nATOM    433  CA  TYR A  58       6.383 -28.735 -11.581  1.00  0.36           C  \\nATOM    434  C   TYR A  58       6.794 -27.280 -11.774  1.00  0.32           C  \\nATOM    435  CB  TYR A  58       6.701 -29.539 -12.845  1.00  0.31           C  \\nATOM    436  O   TYR A  58       6.397 -26.640 -12.750  1.00  0.32           O  \\nATOM    437  CG  TYR A  58       6.918 -31.010 -12.590  1.00  0.33           C  \\nATOM    438  CD1 TYR A  58       8.164 -31.492 -12.193  1.00  0.33           C  \\nATOM    439  CD2 TYR A  58       5.880 -31.922 -12.746  1.00  0.34           C  \\nATOM    440  CE1 TYR A  58       8.369 -32.848 -11.959  1.00  0.30           C  \\nATOM    441  CE2 TYR A  58       6.074 -33.280 -12.514  1.00  0.34           C  \\nATOM    442  OH  TYR A  58       7.518 -35.075 -11.890  1.00  0.18           O  \\nATOM    443  CZ  TYR A  58       7.320 -33.732 -12.121  1.00  0.30           C  \\nATOM    444  N   GLY A  59       7.197 -26.600 -10.508  1.00  0.33           N  \\nATOM    445  CA  GLY A  59       8.459 -25.880 -10.438  1.00  0.36           C  \\nATOM    446  C   GLY A  59       8.816 -25.437  -9.032  1.00  0.33           C  \\nATOM    447  O   GLY A  59       7.988 -24.851  -8.331  1.00  0.32           O  \\nATOM    448  N   GLN A  60       9.434 -26.227  -8.259  1.00  0.33           N  \\nATOM    449  CA  GLN A  60      10.568 -26.074  -7.354  1.00  0.39           C  \\nATOM    450  C   GLN A  60      10.430 -24.812  -6.507  1.00  0.34           C  \\nATOM    451  CB  GLN A  60      11.881 -26.038  -8.138  1.00  0.32           C  \\nATOM    452  O   GLN A  60      10.428 -23.699  -7.037  1.00  0.34           O  \\nATOM    453  CG  GLN A  60      12.533 -27.404  -8.309  1.00  0.35           C  \\nATOM    454  CD  GLN A  60      13.836 -27.339  -9.083  1.00  0.45           C  \\nATOM    455  NE2 GLN A  60      14.581 -28.440  -9.083  1.00  0.34           N  \\nATOM    456  OE1 GLN A  60      14.171 -26.309  -9.676  1.00  0.48           O  \\nATOM    457  N   SER A  61       9.651 -24.976  -5.316  1.00  0.34           N  \\nATOM    458  CA  SER A  61      10.193 -24.859  -3.966  1.00  0.39           C  \\nATOM    459  C   SER A  61       9.123 -25.136  -2.915  1.00  0.34           C  \\nATOM    460  CB  SER A  61      10.787 -23.467  -3.745  1.00  0.32           C  \\nATOM    461  O   SER A  61       8.065 -24.503  -2.916  1.00  0.34           O  \\nATOM    462  OG  SER A  61      12.037 -23.553  -3.084  1.00  0.30           O  \\nATOM    463  N   GLN A  62       8.849 -26.350  -2.755  1.00  0.32           N  \\nATOM    464  CA  GLN A  62       8.832 -26.805  -1.369  1.00  0.37           C  \\nATOM    465  C   GLN A  62       8.004 -25.870  -0.493  1.00  0.32           C  \\nATOM    466  CB  GLN A  62      10.256 -26.914  -0.822  1.00  0.31           C  \\nATOM    467  O   GLN A  62       8.334 -24.690  -0.350  1.00  0.32           O  \\nATOM    468  CG  GLN A  62      11.081 -28.018  -1.470  1.00  0.34           C  \\nATOM    469  CD  GLN A  62      12.432 -28.208  -0.806  1.00  0.43           C  \\nATOM    470  NE2 GLN A  62      13.277 -29.039  -1.407  1.00  0.34           N  \\nATOM    471  OE1 GLN A  62      12.715 -27.612   0.238  1.00  0.47           O  \\nATOM    472  N   ASN A  63       6.594 -26.118  -0.673  1.00  0.35           N  \\nATOM    473  CA  ASN A  63       5.706 -26.293   0.471  1.00  0.42           C  \\nATOM    474  C   ASN A  63       4.243 -26.110   0.078  1.00  0.36           C  \\nATOM    475  CB  ASN A  63       6.079 -25.324   1.595  1.00  0.34           C  \\nATOM    476  O   ASN A  63       3.774 -24.982  -0.078  1.00  0.35           O  \\nATOM    477  CG  ASN A  63       5.846 -25.909   2.973  1.00  0.35           C  \\nATOM    478  ND2 ASN A  63       6.086 -25.109   4.006  1.00  0.27           N  \\nATOM    479  OD1 ASN A  63       5.454 -27.071   3.110  1.00  0.37           O  \\nATOM    480  N   THR A  64       3.829 -27.014  -0.819  1.00  0.34           N  \\nATOM    481  CA  THR A  64       2.519 -27.654  -0.791  1.00  0.36           C  \\nATOM    482  C   THR A  64       2.075 -27.917   0.645  1.00  0.34           C  \\nATOM    483  CB  THR A  64       2.526 -28.977  -1.579  1.00  0.33           C  \\nATOM    484  O   THR A  64       2.516 -28.884   1.269  1.00  0.32           O  \\nATOM    485  CG2 THR A  64       2.037 -28.768  -3.009  1.00  0.32           C  \\nATOM    486  OG1 THR A  64       3.860 -29.499  -1.613  1.00  0.30           O  \\nATOM    487  N   GLY A  65       1.991 -26.856   1.444  1.00  0.37           N  \\nATOM    488  CA  GLY A  65       0.944 -27.005   2.442  1.00  0.39           C  \\nATOM    489  C   GLY A  65      -0.445 -26.739   1.894  1.00  0.38           C  \\nATOM    490  O   GLY A  65      -0.820 -25.587   1.668  1.00  0.36           O  \\nATOM    491  N   TYR A  66      -0.841 -27.369   0.836  1.00  0.39           N  \\nATOM    492  CA  TYR A  66      -2.259 -27.706   0.888  1.00  0.42           C  \\nATOM    493  C   TYR A  66      -2.645 -28.227   2.267  1.00  0.39           C  \\nATOM    494  CB  TYR A  66      -2.603 -28.750  -0.179  1.00  0.38           C  \\nATOM    495  O   TYR A  66      -2.166 -29.279   2.697  1.00  0.38           O  \\nATOM    496  CG  TYR A  66      -2.474 -28.238  -1.593  1.00  0.39           C  \\nATOM    497  CD1 TYR A  66      -3.485 -27.475  -2.173  1.00  0.38           C  \\nATOM    498  CD2 TYR A  66      -1.342 -28.516  -2.352  1.00  0.39           C  \\nATOM    499  CE1 TYR A  66      -3.372 -27.003  -3.477  1.00  0.38           C  \\nATOM    500  CE2 TYR A  66      -1.218 -28.048  -3.656  1.00  0.42           C  \\nATOM    501  OH  TYR A  66      -2.119 -26.828  -5.500  1.00  0.28           O  \\nATOM    502  CZ  TYR A  66      -2.236 -27.294  -4.209  1.00  0.35           C  \\nATOM    503  N   GLY A  67      -2.669 -27.455   3.281  1.00  0.38           N  \\nATOM    504  CA  GLY A  67      -3.566 -27.725   4.393  1.00  0.39           C  \\nATOM    505  C   GLY A  67      -4.935 -28.209   3.953  1.00  0.38           C  \\nATOM    506  O   GLY A  67      -5.567 -27.597   3.090  1.00  0.36           O  \\nATOM    507  N   THR A  68      -4.967 -29.519   3.582  1.00  0.39           N  \\nATOM    508  CA  THR A  68      -6.258 -30.184   3.721  1.00  0.41           C  \\nATOM    509  C   THR A  68      -7.398 -29.173   3.635  1.00  0.39           C  \\nATOM    510  CB  THR A  68      -6.350 -30.953   5.052  1.00  0.38           C  \\nATOM    511  O   THR A  68      -7.583 -28.359   4.542  1.00  0.38           O  \\nATOM    512  CG2 THR A  68      -5.982 -32.421   4.865  1.00  0.37           C  \\nATOM    513  OG1 THR A  68      -5.451 -30.364   6.001  1.00  0.33           O  \\nATOM    514  N   GLN A  69      -7.617 -28.555   2.526  1.00  0.40           N  \\nATOM    515  CA  GLN A  69      -9.038 -28.240   2.428  1.00  0.43           C  \\nATOM    516  C   GLN A  69      -9.896 -29.441   2.814  1.00  0.40           C  \\nATOM    517  CB  GLN A  69      -9.389 -27.777   1.013  1.00  0.37           C  \\nATOM    518  O   GLN A  69      -9.835 -30.488   2.166  1.00  0.38           O  \\nATOM    519  CG  GLN A  69      -9.177 -26.287   0.783  1.00  0.37           C  \\nATOM    520  CD  GLN A  69      -9.615 -25.838  -0.598  1.00  0.44           C  \\nATOM    521  NE2 GLN A  69      -9.430 -24.555  -0.891  1.00  0.36           N  \\nATOM    522  OE1 GLN A  69     -10.115 -26.637  -1.396  1.00  0.47           O  \\nATOM    523  N   SER A  70     -10.093 -29.657   4.051  1.00  0.36           N  \\nATOM    524  CA  SER A  70     -11.357 -30.263   4.456  1.00  0.39           C  \\nATOM    525  C   SER A  70     -12.491 -29.855   3.521  1.00  0.37           C  \\nATOM    526  CB  SER A  70     -11.705 -29.871   5.892  1.00  0.35           C  \\nATOM    527  O   SER A  70     -12.823 -28.673   3.418  1.00  0.35           O  \\nATOM    528  OG  SER A  70     -11.599 -28.469   6.071  1.00  0.37           O  \\nATOM    529  N   THR A  71     -12.500 -30.144   2.190  1.00  0.41           N  \\nATOM    530  CA  THR A  71     -13.865 -30.391   1.742  1.00  0.42           C  \\nATOM    531  C   THR A  71     -14.808 -30.539   2.933  1.00  0.41           C  \\nATOM    532  CB  THR A  71     -13.945 -31.653   0.862  1.00  0.39           C  \\nATOM    533  O   THR A  71     -14.531 -31.303   3.859  1.00  0.39           O  \\nATOM    534  CG2 THR A  71     -14.079 -31.287  -0.613  1.00  0.38           C  \\nATOM    535  OG1 THR A  71     -12.755 -32.430   1.043  1.00  0.35           O  \\nATOM    536  N   PRO A  72     -15.501 -29.541   3.412  1.00  0.42           N  \\nATOM    537  CA  PRO A  72     -16.560 -30.163   4.210  1.00  0.43           C  \\nATOM    538  C   PRO A  72     -16.996 -31.518   3.659  1.00  0.42           C  \\nATOM    539  CB  PRO A  72     -17.703 -29.147   4.130  1.00  0.40           C  \\nATOM    540  O   PRO A  72     -17.439 -31.609   2.511  1.00  0.40           O  \\nATOM    541  CG  PRO A  72     -17.244 -28.133   3.134  1.00  0.39           C  \\nATOM    542  CD  PRO A  72     -15.838 -28.467   2.726  1.00  0.44           C  \\nATOM    543  N   GLN A  73     -16.202 -32.487   3.708  1.00  0.34           N  \\nATOM    544  CA  GLN A  73     -16.822 -33.775   4.003  1.00  0.36           C  \\nATOM    545  C   GLN A  73     -18.178 -33.590   4.678  1.00  0.34           C  \\nATOM    546  CB  GLN A  73     -15.906 -34.621   4.888  1.00  0.33           C  \\nATOM    547  O   GLN A  73     -18.284 -32.895   5.691  1.00  0.33           O  \\nATOM    548  CG  GLN A  73     -15.241 -35.779   4.156  1.00  0.34           C  \\nATOM    549  CD  GLN A  73     -14.324 -36.591   5.051  1.00  0.42           C  \\nATOM    550  NE2 GLN A  73     -13.942 -37.777   4.589  1.00  0.36           N  \\nATOM    551  OE1 GLN A  73     -13.960 -36.155   6.148  1.00  0.47           O  \\nATOM    552  N   GLY A  74     -19.261 -33.669   4.009  1.00  0.38           N  \\nATOM    553  CA  GLY A  74     -20.538 -34.340   4.195  1.00  0.39           C  \\nATOM    554  C   GLY A  74     -20.595 -35.175   5.460  1.00  0.38           C  \\nATOM    555  O   GLY A  74     -19.676 -35.947   5.742  1.00  0.36           O  \\nATOM    556  N   TYR A  75     -20.751 -34.577   6.757  1.00  0.35           N  \\nATOM    557  CA  TYR A  75     -21.558 -35.357   7.689  1.00  0.42           C  \\nATOM    558  C   TYR A  75     -21.510 -36.840   7.342  1.00  0.36           C  \\nATOM    559  CB  TYR A  75     -23.009 -34.867   7.685  1.00  0.34           C  \\nATOM    560  O   TYR A  75     -21.920 -37.243   6.250  1.00  0.35           O  \\nATOM    561  CG  TYR A  75     -23.224 -33.602   8.480  1.00  0.38           C  \\nATOM    562  CD1 TYR A  75     -23.452 -33.651   9.854  1.00  0.39           C  \\nATOM    563  CD2 TYR A  75     -23.202 -32.358   7.861  1.00  0.38           C  \\nATOM    564  CE1 TYR A  75     -23.653 -32.488  10.591  1.00  0.34           C  \\nATOM    565  CE2 TYR A  75     -23.402 -31.189   8.588  1.00  0.39           C  \\nATOM    566  OH  TYR A  75     -23.824 -30.110  10.675  1.00  0.20           O  \\nATOM    567  CZ  TYR A  75     -23.626 -31.264   9.950  1.00  0.32           C  \\nATOM    568  N   GLY A  76     -20.320 -37.411   7.785  1.00  0.37           N  \\nATOM    569  CA  GLY A  76     -20.588 -38.710   8.382  1.00  0.41           C  \\nATOM    570  C   GLY A  76     -19.330 -39.511   8.657  1.00  0.37           C  \\nATOM    571  O   GLY A  76     -18.459 -39.627   7.792  1.00  0.35           O  \\nATOM    572  N   SER A  77     -18.714 -39.337   9.838  1.00  0.33           N  \\nATOM    573  CA  SER A  77     -18.441 -40.455  10.735  1.00  0.40           C  \\nATOM    574  C   SER A  77     -17.237 -41.263  10.260  1.00  0.34           C  \\nATOM    575  CB  SER A  77     -19.664 -41.366  10.846  1.00  0.31           C  \\nATOM    576  O   SER A  77     -17.163 -41.646   9.090  1.00  0.33           O  \\nATOM    577  OG  SER A  77     -19.317 -42.712  10.569  1.00  0.39           O  \\nATOM    578  N   THR A  78     -15.954 -40.901  10.811  1.00  0.35           N  \\nATOM    579  CA  THR A  78     -15.255 -41.990  11.485  1.00  0.42           C  \\nATOM    580  C   THR A  78     -14.111 -42.514  10.621  1.00  0.35           C  \\nATOM    581  CB  THR A  78     -16.215 -43.145  11.825  1.00  0.33           C  \\nATOM    582  O   THR A  78     -14.339 -43.025   9.523  1.00  0.35           O  \\nATOM    583  CG2 THR A  78     -16.852 -42.946  13.196  1.00  0.34           C  \\nATOM    584  OG1 THR A  78     -17.249 -43.206  10.834  1.00  0.32           O  \\nATOM    585  N   GLY A  79     -12.802 -41.871  10.832  1.00  0.38           N  \\nATOM    586  CA  GLY A  79     -11.700 -42.787  11.074  1.00  0.43           C  \\nATOM    587  C   GLY A  79     -10.404 -42.349  10.419  1.00  0.39           C  \\nATOM    588  O   GLY A  79     -10.373 -42.071   9.218  1.00  0.37           O  \\nATOM    589  N   GLY A  80      -9.463 -41.666  11.163  1.00  0.41           N  \\nATOM    590  CA  GLY A  80      -8.075 -41.985  11.457  1.00  0.45           C  \\nATOM    591  C   GLY A  80      -7.161 -41.836  10.256  1.00  0.41           C  \\nATOM    592  O   GLY A  80      -7.230 -42.628   9.314  1.00  0.40           O  \\nATOM    593  N   TYR A  81      -6.770 -40.539   9.933  1.00  0.46           N  \\nATOM    594  CA  TYR A  81      -5.443 -40.366   9.353  1.00  0.52           C  \\nATOM    595  C   TYR A  81      -4.656 -39.297  10.102  1.00  0.47           C  \\nATOM    596  CB  TYR A  81      -5.549 -39.994   7.871  1.00  0.44           C  \\nATOM    597  O   TYR A  81      -5.141 -38.180  10.293  1.00  0.46           O  \\nATOM    598  CG  TYR A  81      -5.846 -41.168   6.970  1.00  0.45           C  \\nATOM    599  CD1 TYR A  81      -4.824 -41.998   6.514  1.00  0.41           C  \\nATOM    600  CD2 TYR A  81      -7.149 -41.450   6.573  1.00  0.42           C  \\nATOM    601  CE1 TYR A  81      -5.094 -43.080   5.683  1.00  0.39           C  \\nATOM    602  CE2 TYR A  81      -7.430 -42.529   5.742  1.00  0.44           C  \\nATOM    603  OH  TYR A  81      -6.670 -44.408   4.480  1.00  0.28           O  \\nATOM    604  CZ  TYR A  81      -6.397 -43.338   5.303  1.00  0.35           C  \\nATOM    605  N   GLY A  82      -3.523 -39.644  10.756  1.00  0.44           N  \\nATOM    606  CA  GLY A  82      -2.174 -39.803  11.275  1.00  0.46           C  \\nATOM    607  C   GLY A  82      -1.215 -38.737  10.781  1.00  0.44           C  \\nATOM    608  O   GLY A  82      -1.091 -38.517   9.574  1.00  0.42           O  \\nATOM    609  N   SER A  83      -0.969 -37.573  11.483  1.00  0.42           N  \\nATOM    610  CA  SER A  83       0.165 -36.797  11.976  1.00  0.47           C  \\nATOM    611  C   SER A  83       1.366 -36.923  11.045  1.00  0.42           C  \\nATOM    612  CB  SER A  83       0.552 -37.248  13.385  1.00  0.40           C  \\nATOM    613  O   SER A  83       1.969 -37.993  10.942  1.00  0.41           O  \\nATOM    614  OG  SER A  83       0.987 -38.597  13.380  1.00  0.46           O  \\nATOM    615  N   SER A  84       1.521 -35.979   9.983  1.00  0.43           N  \\nATOM    616  CA  SER A  84       2.909 -35.735   9.607  1.00  0.46           C  \\nATOM    617  C   SER A  84       3.182 -34.245   9.435  1.00  0.43           C  \\nATOM    618  CB  SER A  84       3.253 -36.477   8.314  1.00  0.40           C  \\nATOM    619  O   SER A  84       2.487 -33.564   8.678  1.00  0.41           O  \\nATOM    620  OG  SER A  84       2.163 -36.441   7.409  1.00  0.42           O  \\nATOM    621  N   GLN A  85       3.921 -33.609  10.447  1.00  0.47           N  \\nATOM    622  CA  GLN A  85       5.017 -32.675  10.685  1.00  0.51           C  \\nATOM    623  C   GLN A  85       5.727 -32.317   9.383  1.00  0.47           C  \\nATOM    624  CB  GLN A  85       6.016 -33.262  11.684  1.00  0.44           C  \\nATOM    625  O   GLN A  85       6.442 -33.143   8.812  1.00  0.46           O  \\nATOM    626  CG  GLN A  85       5.643 -33.022  13.140  1.00  0.45           C  \\nATOM    627  CD  GLN A  85       6.658 -33.599  14.109  1.00  0.51           C  \\nATOM    628  NE2 GLN A  85       6.439 -33.376  15.400  1.00  0.40           N  \\nATOM    629  OE1 GLN A  85       7.631 -34.240  13.700  1.00  0.55           O  \\nATOM    630  N   SER A  86       5.176 -31.369   8.613  1.00  0.43           N  \\nATOM    631  CA  SER A  86       5.976 -30.654   7.624  1.00  0.47           C  \\nATOM    632  C   SER A  86       6.494 -29.333   8.182  1.00  0.44           C  \\nATOM    633  CB  SER A  86       5.158 -30.397   6.358  1.00  0.41           C  \\nATOM    634  O   SER A  86       5.710 -28.486   8.615  1.00  0.43           O  \\nATOM    635  OG  SER A  86       3.999 -29.636   6.654  1.00  0.42           O  \\nATOM    636  N   SER A  87       7.728 -29.256   8.691  1.00  0.43           N  \\nATOM    637  CA  SER A  87       8.843 -28.325   8.824  1.00  0.46           C  \\nATOM    638  C   SER A  87       8.878 -27.334   7.665  1.00  0.43           C  \\nATOM    639  CB  SER A  87      10.169 -29.083   8.897  1.00  0.42           C  \\nATOM    640  O   SER A  87       9.004 -27.731   6.505  1.00  0.42           O  \\nATOM    641  OG  SER A  87      10.210 -30.119   7.931  1.00  0.44           O  \\nATOM    642  N   GLN A  88       8.230 -26.247   7.717  1.00  0.44           N  \\nATOM    643  CA  GLN A  88       8.286 -24.839   7.337  1.00  0.45           C  \\nATOM    644  C   GLN A  88       9.723 -24.326   7.338  1.00  0.44           C  \\nATOM    645  CB  GLN A  88       7.426 -23.994   8.278  1.00  0.41           C  \\nATOM    646  O   GLN A  88      10.382 -24.311   8.379  1.00  0.43           O  \\nATOM    647  CG  GLN A  88       6.019 -23.731   7.756  1.00  0.41           C  \\nATOM    648  CD  GLN A  88       5.166 -22.951   8.738  1.00  0.47           C  \\nATOM    649  NE2 GLN A  88       3.987 -22.528   8.293  1.00  0.39           N  \\nATOM    650  OE1 GLN A  88       5.561 -22.731   9.887  1.00  0.52           O  \\nATOM    651  N   SER A  89      10.364 -24.228   6.349  1.00  0.40           N  \\nATOM    652  CA  SER A  89      11.636 -23.559   6.094  1.00  0.44           C  \\nATOM    653  C   SER A  89      11.434 -22.072   5.824  1.00  0.40           C  \\nATOM    654  CB  SER A  89      12.355 -24.207   4.910  1.00  0.38           C  \\nATOM    655  O   SER A  89      10.616 -21.694   4.983  1.00  0.39           O  \\nATOM    656  OG  SER A  89      11.618 -24.025   3.713  1.00  0.40           O  \\nATOM    657  N   SER A  90      11.375 -21.079   6.851  1.00  0.36           N  \\nATOM    658  CA  SER A  90      12.051 -19.830   7.187  1.00  0.39           C  \\nATOM    659  C   SER A  90      12.440 -19.057   5.931  1.00  0.36           C  \\nATOM    660  CB  SER A  90      13.296 -20.105   8.032  1.00  0.34           C  \\nATOM    661  O   SER A  90      13.349 -19.462   5.203  1.00  0.35           O  \\nATOM    662  OG  SER A  90      14.040 -21.185   7.495  1.00  0.38           O  \\nATOM    663  N   TYR A  91      11.542 -18.524   5.167  1.00  0.43           N  \\nATOM    664  CA  TYR A  91      11.757 -17.134   4.779  1.00  0.48           C  \\nATOM    665  C   TYR A  91      11.379 -16.187   5.911  1.00  0.43           C  \\nATOM    666  CB  TYR A  91      10.948 -16.796   3.523  1.00  0.41           C  \\nATOM    667  O   TYR A  91      10.204 -16.075   6.269  1.00  0.42           O  \\nATOM    668  CG  TYR A  91      11.549 -17.340   2.249  1.00  0.43           C  \\nATOM    669  CD1 TYR A  91      12.567 -16.656   1.588  1.00  0.41           C  \\nATOM    670  CD2 TYR A  91      11.100 -18.538   1.703  1.00  0.42           C  \\nATOM    671  CE1 TYR A  91      13.122 -17.152   0.414  1.00  0.40           C  \\nATOM    672  CE2 TYR A  91      11.648 -19.043   0.529  1.00  0.43           C  \\nATOM    673  OH  TYR A  91      13.203 -18.840  -1.270  1.00  0.28           O  \\nATOM    674  CZ  TYR A  91      12.657 -18.344  -0.107  1.00  0.37           C  \\nATOM    675  N   GLY A  92      12.025 -16.012   6.970  1.00  0.33           N  \\nATOM    676  CA  GLY A  92      12.587 -15.270   8.087  1.00  0.36           C  \\nATOM    677  C   GLY A  92      12.672 -13.777   7.830  1.00  0.34           C  \\nATOM    678  O   GLY A  92      13.346 -13.341   6.895  1.00  0.32           O  \\nATOM    679  N   GLN A  93      11.498 -13.058   7.703  1.00  0.38           N  \\nATOM    680  CA  GLN A  93      11.312 -11.736   8.293  1.00  0.42           C  \\nATOM    681  C   GLN A  93       9.901 -11.215   8.040  1.00  0.39           C  \\nATOM    682  CB  GLN A  93      12.343 -10.751   7.740  1.00  0.36           C  \\nATOM    683  O   GLN A  93       9.555 -10.869   6.909  1.00  0.37           O  \\nATOM    684  CG  GLN A  93      13.500 -10.473   8.690  1.00  0.37           C  \\nATOM    685  CD  GLN A  93      14.569  -9.592   8.072  1.00  0.45           C  \\nATOM    686  NE2 GLN A  93      15.541  -9.182   8.880  1.00  0.37           N  \\nATOM    687  OE1 GLN A  93      14.523  -9.284   6.877  1.00  0.49           O  \\nATOM    688  N   GLN A  94       8.841 -11.688   8.732  1.00  0.40           N  \\nATOM    689  CA  GLN A  94       7.772 -10.926   9.370  1.00  0.43           C  \\nATOM    690  C   GLN A  94       8.338  -9.804  10.236  1.00  0.41           C  \\nATOM    691  CB  GLN A  94       6.887 -11.845  10.214  1.00  0.38           C  \\nATOM    692  O   GLN A  94       8.993 -10.063  11.248  1.00  0.39           O  \\nATOM    693  CG  GLN A  94       5.592 -12.254   9.525  1.00  0.39           C  \\nATOM    694  CD  GLN A  94       4.775 -13.234  10.345  1.00  0.46           C  \\nATOM    695  NE2 GLN A  94       3.626 -13.638   9.814  1.00  0.37           N  \\nATOM    696  OE1 GLN A  94       5.174 -13.626  11.447  1.00  0.49           O  \\nATOM    697  N   SER A  95       8.839  -8.825   9.688  1.00  0.38           N  \\nATOM    698  CA  SER A  95       8.845  -7.543  10.386  1.00  0.43           C  \\nATOM    699  C   SER A  95       7.573  -7.354  11.206  1.00  0.38           C  \\nATOM    700  CB  SER A  95       8.997  -6.392   9.392  1.00  0.37           C  \\nATOM    701  O   SER A  95       6.467  -7.413  10.666  1.00  0.37           O  \\nATOM    702  OG  SER A  95       7.738  -6.018   8.859  1.00  0.39           O  \\nATOM    703  N   SER A  96       7.395  -7.977  12.297  1.00  0.38           N  \\nATOM    704  CA  SER A  96       6.774  -7.367  13.469  1.00  0.40           C  \\nATOM    705  C   SER A  96       6.713  -5.849  13.335  1.00  0.38           C  \\nATOM    706  CB  SER A  96       7.539  -7.744  14.738  1.00  0.36           C  \\nATOM    707  O   SER A  96       7.736  -5.169  13.446  1.00  0.36           O  \\nATOM    708  OG  SER A  96       8.909  -7.399  14.622  1.00  0.38           O  \\nATOM    709  N   TYR A  97       6.071  -5.307  12.412  1.00  0.39           N  \\nATOM    710  CA  TYR A  97       5.535  -4.020  12.839  1.00  0.43           C  \\nATOM    711  C   TYR A  97       4.524  -4.197  13.966  1.00  0.40           C  \\nATOM    712  CB  TYR A  97       4.881  -3.292  11.661  1.00  0.38           C  \\nATOM    713  O   TYR A  97       3.484  -4.833  13.781  1.00  0.39           O  \\nATOM    714  CG  TYR A  97       5.852  -2.492  10.827  1.00  0.40           C  \\nATOM    715  CD1 TYR A  97       6.147  -1.168  11.144  1.00  0.38           C  \\nATOM    716  CD2 TYR A  97       6.475  -3.058   9.720  1.00  0.39           C  \\nATOM    717  CE1 TYR A  97       7.040  -0.427  10.377  1.00  0.39           C  \\nATOM    718  CE2 TYR A  97       7.370  -2.327   8.946  1.00  0.42           C  \\nATOM    719  OH  TYR A  97       8.531  -0.285   8.519  1.00  0.29           O  \\nATOM    720  CZ  TYR A  97       7.646  -1.014   9.282  1.00  0.34           C  \\nATOM    721  N   PRO A  98       4.807  -4.195  15.217  1.00  0.38           N  \\nATOM    722  CA  PRO A  98       3.994  -3.554  16.253  1.00  0.39           C  \\nATOM    723  C   PRO A  98       3.706  -2.085  15.951  1.00  0.38           C  \\nATOM    724  CB  PRO A  98       4.852  -3.695  17.512  1.00  0.36           C  \\nATOM    725  O   PRO A  98       4.630  -1.270  15.890  1.00  0.37           O  \\nATOM    726  CG  PRO A  98       6.238  -3.932  17.006  1.00  0.37           C  \\nATOM    727  CD  PRO A  98       6.153  -4.382  15.576  1.00  0.44           C  \\nATOM    728  N   GLY A  99       2.917  -1.835  14.892  1.00  0.36           N  \\nATOM    729  CA  GLY A  99       2.547  -0.428  14.879  1.00  0.40           C  \\nATOM    730  C   GLY A  99       2.058   0.048  13.524  1.00  0.37           C  \\nATOM    731  O   GLY A  99       2.857   0.247  12.606  1.00  0.35           O  \\nATOM    732  N   TYR A 100       0.887  -0.478  12.937  1.00  0.41           N  \\nATOM    733  CA  TYR A 100      -0.258   0.213  12.355  1.00  0.45           C  \\nATOM    734  C   TYR A 100      -1.537  -0.113  13.116  1.00  0.41           C  \\nATOM    735  CB  TYR A 100      -0.419  -0.163  10.879  1.00  0.39           C  \\nATOM    736  O   TYR A 100      -1.881  -1.284  13.292  1.00  0.40           O  \\nATOM    737  CG  TYR A 100       0.607   0.476   9.974  1.00  0.39           C  \\nATOM    738  CD1 TYR A 100       0.418   1.762   9.473  1.00  0.38           C  \\nATOM    739  CD2 TYR A 100       1.766  -0.205   9.618  1.00  0.39           C  \\nATOM    740  CE1 TYR A 100       1.359   2.354   8.637  1.00  0.37           C  \\nATOM    741  CE2 TYR A 100       2.714   0.377   8.783  1.00  0.41           C  \\nATOM    742  OH  TYR A 100       3.438   2.236   7.472  1.00  0.27           O  \\nATOM    743  CZ  TYR A 100       2.502   1.655   8.299  1.00  0.34           C  \\nATOM    744  N   GLY A 101      -1.710   0.605  14.202  1.00  0.39           N  \\nATOM    745  CA  GLY A 101      -2.753   1.186  15.033  1.00  0.40           C  \\nATOM    746  C   GLY A 101      -4.037   0.378  15.031  1.00  0.39           C  \\nATOM    747  O   GLY A 101      -4.597   0.095  13.970  1.00  0.37           O  \\nATOM    748  N   GLN A 102      -4.060  -0.850  15.642  1.00  0.41           N  \\nATOM    749  CA  GLN A 102      -5.346  -1.114  16.281  1.00  0.43           C  \\nATOM    750  C   GLN A 102      -6.432  -0.193  15.733  1.00  0.41           C  \\nATOM    751  CB  GLN A 102      -5.235  -0.952  17.797  1.00  0.37           C  \\nATOM    752  O   GLN A 102      -6.456   1.000  16.044  1.00  0.39           O  \\nATOM    753  CG  GLN A 102      -4.766  -2.209  18.518  1.00  0.38           C  \\nATOM    754  CD  GLN A 102      -4.569  -1.994  20.007  1.00  0.45           C  \\nATOM    755  NE2 GLN A 102      -3.725  -2.817  20.618  1.00  0.36           N  \\nATOM    756  OE1 GLN A 102      -5.170  -1.094  20.602  1.00  0.50           O  \\nATOM    757  N   GLN A 103      -6.804  -0.233  14.471  1.00  0.47           N  \\nATOM    758  CA  GLN A 103      -8.205   0.166  14.392  1.00  0.50           C  \\nATOM    759  C   GLN A 103      -9.104  -0.836  15.111  1.00  0.47           C  \\nATOM    760  CB  GLN A 103      -8.642   0.310  12.933  1.00  0.44           C  \\nATOM    761  O   GLN A 103      -9.005  -2.043  14.882  1.00  0.46           O  \\nATOM    762  CG  GLN A 103     -10.010   0.956  12.762  1.00  0.43           C  \\nATOM    763  CD  GLN A 103     -10.384   1.159  11.306  1.00  0.49           C  \\nATOM    764  NE2 GLN A 103     -11.681   1.275  11.037  1.00  0.38           N  \\nATOM    765  OE1 GLN A 103      -9.516   1.211  10.429  1.00  0.51           O  \\nATOM    766  N   PRO A 104      -9.466  -0.396  16.305  1.00  0.43           N  \\nATOM    767  CA  PRO A 104     -10.609  -1.071  16.926  1.00  0.42           C  \\nATOM    768  C   PRO A 104     -11.669  -1.492  15.910  1.00  0.43           C  \\nATOM    769  CB  PRO A 104     -11.162  -0.015  17.886  1.00  0.40           C  \\nATOM    770  O   PRO A 104     -11.966  -0.742  14.976  1.00  0.41           O  \\nATOM    771  CG  PRO A 104     -10.525   1.266  17.454  1.00  0.39           C  \\nATOM    772  CD  PRO A 104      -9.377   0.944  16.542  1.00  0.44           C  \\nATOM    773  N   ALA A 105     -11.809  -2.724  15.521  1.00  0.46           N  \\nATOM    774  CA  ALA A 105     -13.035  -3.334  15.013  1.00  0.48           C  \\nATOM    775  C   ALA A 105     -14.228  -2.401  15.196  1.00  0.46           C  \\nATOM    776  CB  ALA A 105     -13.295  -4.667  15.711  1.00  0.44           C  \\nATOM    777  O   ALA A 105     -14.358  -1.744  16.232  1.00  0.46           O  \\nATOM    778  N   PRO A 106     -14.794  -1.762  14.101  1.00  0.47           N  \\nATOM    779  CA  PRO A 106     -16.104  -1.134  14.286  1.00  0.48           C  \\nATOM    780  C   PRO A 106     -17.130  -2.079  14.907  1.00  0.47           C  \\nATOM    781  CB  PRO A 106     -16.513  -0.744  12.863  1.00  0.43           C  \\nATOM    782  O   PRO A 106     -17.299  -3.207  14.438  1.00  0.45           O  \\nATOM    783  CG  PRO A 106     -15.770  -1.694  11.979  1.00  0.42           C  \\nATOM    784  CD  PRO A 106     -14.522  -2.131  12.691  1.00  0.47           C  \\nATOM    785  N   SER A 107     -17.277  -2.103  16.137  1.00  0.43           N  \\nATOM    786  CA  SER A 107     -18.517  -2.422  16.837  1.00  0.44           C  \\nATOM    787  C   SER A 107     -19.683  -2.558  15.863  1.00  0.42           C  \\nATOM    788  CB  SER A 107     -18.836  -1.350  17.880  1.00  0.41           C  \\nATOM    789  O   SER A 107     -19.983  -1.628  15.113  1.00  0.41           O  \\nATOM    790  OG  SER A 107     -18.912  -0.069  17.278  1.00  0.44           O  \\nATOM    791  N   SER A 108     -19.961  -3.751  15.226  1.00  0.47           N  \\nATOM    792  CA  SER A 108     -21.163  -4.405  14.719  1.00  0.50           C  \\nATOM    793  C   SER A 108     -22.388  -4.027  15.546  1.00  0.47           C  \\nATOM    794  CB  SER A 108     -20.987  -5.924  14.718  1.00  0.45           C  \\nATOM    795  O   SER A 108     -22.492  -4.399  16.716  1.00  0.46           O  \\nATOM    796  OG  SER A 108     -20.650  -6.392  16.013  1.00  0.47           O  \\nATOM    797  N   THR A 109     -22.942  -2.959  15.340  1.00  0.43           N  \\nATOM    798  CA  THR A 109     -24.310  -2.678  15.760  1.00  0.47           C  \\nATOM    799  C   THR A 109     -25.293  -3.613  15.060  1.00  0.43           C  \\nATOM    800  CB  THR A 109     -24.694  -1.215  15.471  1.00  0.41           C  \\nATOM    801  O   THR A 109     -25.337  -3.667  13.829  1.00  0.42           O  \\nATOM    802  CG2 THR A 109     -25.089  -0.487  16.751  1.00  0.38           C  \\nATOM    803  OG1 THR A 109     -23.577  -0.542  14.878  1.00  0.36           O  \\nATOM    804  N   SER A 110     -25.702  -4.749  15.645  1.00  0.42           N  \\nATOM    805  CA  SER A 110     -26.935  -5.480  15.919  1.00  0.47           C  \\nATOM    806  C   SER A 110     -28.129  -4.830  15.227  1.00  0.42           C  \\nATOM    807  CB  SER A 110     -27.189  -5.558  17.425  1.00  0.40           C  \\nATOM    808  O   SER A 110     -28.597  -3.770  15.650  1.00  0.42           O  \\nATOM    809  OG  SER A 110     -26.599  -4.456  18.092  1.00  0.46           O  \\nATOM    810  N   GLY A 111     -28.321  -5.103  13.928  1.00  0.43           N  \\nATOM    811  CA  GLY A 111     -29.716  -4.943  13.548  1.00  0.46           C  \\nATOM    812  C   GLY A 111     -30.418  -6.261  13.284  1.00  0.43           C  \\nATOM    813  O   GLY A 111     -29.921  -7.093  12.522  1.00  0.42           O  \\nATOM    814  N   SER A 112     -31.223  -6.820  14.204  1.00  0.44           N  \\nATOM    815  CA  SER A 112     -32.492  -7.530  14.321  1.00  0.49           C  \\nATOM    816  C   SER A 112     -33.255  -7.521  13.000  1.00  0.45           C  \\nATOM    817  CB  SER A 112     -33.355  -6.909  15.421  1.00  0.42           C  \\nATOM    818  O   SER A 112     -33.781  -6.486  12.588  1.00  0.44           O  \\nATOM    819  OG  SER A 112     -34.474  -7.731  15.707  1.00  0.46           O  \\nATOM    820  N   TYR A 113     -32.895  -8.363  12.056  1.00  0.52           N  \\nATOM    821  CA  TYR A 113     -33.869  -8.825  11.074  1.00  0.56           C  \\nATOM    822  C   TYR A 113     -34.327 -10.245  11.387  1.00  0.52           C  \\nATOM    823  CB  TYR A 113     -33.279  -8.766   9.662  1.00  0.49           C  \\nATOM    824  O   TYR A 113     -33.504 -11.153  11.522  1.00  0.51           O  \\nATOM    825  CG  TYR A 113     -33.193  -7.368   9.099  1.00  0.49           C  \\nATOM    826  CD1 TYR A 113     -34.292  -6.777   8.480  1.00  0.45           C  \\nATOM    827  CD2 TYR A 113     -32.013  -6.637   9.182  1.00  0.46           C  \\nATOM    828  CE1 TYR A 113     -34.217  -5.490   7.957  1.00  0.44           C  \\nATOM    829  CE2 TYR A 113     -31.927  -5.349   8.663  1.00  0.47           C  \\nATOM    830  OH  TYR A 113     -32.952  -3.511   7.538  1.00  0.36           O  \\nATOM    831  CZ  TYR A 113     -33.032  -4.785   8.054  1.00  0.39           C  \\nATOM    832  N   GLY A 114     -35.472 -10.396  11.756  1.00  0.45           N  \\nATOM    833  CA  GLY A 114     -36.777 -11.010  11.941  1.00  0.47           C  \\nATOM    834  C   GLY A 114     -37.087 -12.074  10.905  1.00  0.45           C  \\nATOM    835  O   GLY A 114     -36.920 -11.847   9.705  1.00  0.43           O  \\nATOM    836  N   SER A 115     -36.841 -13.432  11.134  1.00  0.39           N  \\nATOM    837  CA  SER A 115     -37.605 -14.675  11.161  1.00  0.48           C  \\nATOM    838  C   SER A 115     -38.501 -14.801   9.933  1.00  0.40           C  \\nATOM    839  CB  SER A 115     -38.453 -14.756  12.431  1.00  0.38           C  \\nATOM    840  O   SER A 115     -39.317 -13.918   9.661  1.00  0.39           O  \\nATOM    841  OG  SER A 115     -39.830 -14.614  12.127  1.00  0.45           O  \\nATOM    842  N   SER A 116     -38.097 -15.613   8.790  1.00  0.39           N  \\nATOM    843  CA  SER A 116     -39.093 -16.533   8.251  1.00  0.45           C  \\nATOM    844  C   SER A 116     -38.831 -16.829   6.778  1.00  0.39           C  \\nATOM    845  CB  SER A 116     -40.500 -15.959   8.421  1.00  0.37           C  \\nATOM    846  O   SER A 116     -38.590 -15.914   5.988  1.00  0.39           O  \\nATOM    847  OG  SER A 116     -40.616 -14.708   7.766  1.00  0.43           O  \\nATOM    848  N   SER A 117     -38.278 -18.100   6.484  1.00  0.40           N  \\nATOM    849  CA  SER A 117     -38.731 -19.236   5.689  1.00  0.47           C  \\nATOM    850  C   SER A 117     -38.279 -19.110   4.238  1.00  0.40           C  \\nATOM    851  CB  SER A 117     -40.255 -19.358   5.748  1.00  0.38           C  \\nATOM    852  O   SER A 117     -38.456 -18.061   3.615  1.00  0.39           O  \\nATOM    853  OG  SER A 117     -40.871 -18.127   5.411  1.00  0.42           O  \\nATOM    854  N   GLN A 118     -37.295 -19.858   3.879  1.00  0.39           N  \\nATOM    855  CA  GLN A 118     -37.010 -20.822   2.821  1.00  0.48           C  \\nATOM    856  C   GLN A 118     -37.158 -20.185   1.442  1.00  0.39           C  \\nATOM    857  CB  GLN A 118     -37.931 -22.037   2.939  1.00  0.38           C  \\nATOM    858  O   GLN A 118     -38.268 -19.850   1.023  1.00  0.39           O  \\nATOM    859  CG  GLN A 118     -37.354 -23.166   3.782  1.00  0.42           C  \\nATOM    860  CD  GLN A 118     -38.311 -24.334   3.933  1.00  0.51           C  \\nATOM    861  NE2 GLN A 118     -37.976 -25.264   4.820  1.00  0.40           N  \\nATOM    862  OE1 GLN A 118     -39.344 -24.399   3.259  1.00  0.54           O  \\nATOM    863  N   SER A 119     -36.064 -19.382   1.000  1.00  0.40           N  \\nATOM    864  CA  SER A 119     -35.509 -19.147  -0.329  1.00  0.46           C  \\nATOM    865  C   SER A 119     -34.091 -18.594  -0.247  1.00  0.40           C  \\nATOM    866  CB  SER A 119     -36.396 -18.183  -1.118  1.00  0.39           C  \\nATOM    867  O   SER A 119     -33.843 -17.607   0.449  1.00  0.39           O  \\nATOM    868  OG  SER A 119     -36.500 -16.936  -0.452  1.00  0.40           O  \\nATOM    869  N   SER A 120     -33.215 -19.377  -0.071  1.00  0.42           N  \\nATOM    870  CA  SER A 120     -31.819 -19.203  -0.459  1.00  0.47           C  \\nATOM    871  C   SER A 120     -31.632 -17.943  -1.298  1.00  0.43           C  \\nATOM    872  CB  SER A 120     -31.323 -20.422  -1.238  1.00  0.41           C  \\nATOM    873  O   SER A 120     -32.039 -17.898  -2.460  1.00  0.42           O  \\nATOM    874  OG  SER A 120     -32.115 -20.637  -2.393  1.00  0.43           O  \\nATOM    875  N   SER A 121     -31.941 -16.803  -0.799  1.00  0.39           N  \\nATOM    876  CA  SER A 121     -31.509 -15.450  -1.133  1.00  0.43           C  \\nATOM    877  C   SER A 121     -30.004 -15.288  -0.943  1.00  0.40           C  \\nATOM    878  CB  SER A 121     -32.252 -14.423  -0.278  1.00  0.37           C  \\nATOM    879  O   SER A 121     -29.486 -15.496   0.155  1.00  0.39           O  \\nATOM    880  OG  SER A 121     -31.372 -13.805   0.645  1.00  0.37           O  \\nATOM    881  N   TYR A 122     -29.159 -15.961  -1.669  1.00  0.46           N  \\nATOM    882  CA  TYR A 122     -27.865 -15.307  -1.821  1.00  0.51           C  \\nATOM    883  C   TYR A 122     -27.937 -13.847  -1.390  1.00  0.46           C  \\nATOM    884  CB  TYR A 122     -27.383 -15.399  -3.272  1.00  0.43           C  \\nATOM    885  O   TYR A 122     -28.618 -13.039  -2.025  1.00  0.45           O  \\nATOM    886  CG  TYR A 122     -26.963 -16.789  -3.685  1.00  0.44           C  \\nATOM    887  CD1 TYR A 122     -25.686 -17.266  -3.398  1.00  0.43           C  \\nATOM    888  CD2 TYR A 122     -27.841 -17.627  -4.364  1.00  0.43           C  \\nATOM    889  CE1 TYR A 122     -25.294 -18.546  -3.778  1.00  0.40           C  \\nATOM    890  CE2 TYR A 122     -27.459 -18.908  -4.749  1.00  0.44           C  \\nATOM    891  OH  TYR A 122     -25.803 -20.625  -4.830  1.00  0.27           O  \\nATOM    892  CZ  TYR A 122     -26.186 -19.358  -4.452  1.00  0.37           C  \\nATOM    893  N   GLY A 123     -28.189 -13.612  -0.156  1.00  0.38           N  \\nATOM    894  CA  GLY A 123     -28.118 -12.407   0.655  1.00  0.40           C  \\nATOM    895  C   GLY A 123     -27.253 -11.323   0.039  1.00  0.38           C  \\nATOM    896  O   GLY A 123     -26.077 -11.552  -0.254  1.00  0.36           O  \\nATOM    897  N   GLN A 124     -27.676 -10.593  -1.056  1.00  0.45           N  \\nATOM    898  CA  GLN A 124     -27.149  -9.243  -1.227  1.00  0.48           C  \\nATOM    899  C   GLN A 124     -26.190  -8.880  -0.098  1.00  0.45           C  \\nATOM    900  CB  GLN A 124     -28.290  -8.226  -1.295  1.00  0.43           C  \\nATOM    901  O   GLN A 124     -26.481  -9.126   1.075  1.00  0.44           O  \\nATOM    902  CG  GLN A 124     -28.867  -8.040  -2.692  1.00  0.42           C  \\nATOM    903  CD  GLN A 124     -29.932  -6.961  -2.749  1.00  0.49           C  \\nATOM    904  NE2 GLN A 124     -29.782  -6.029  -3.683  1.00  0.38           N  \\nATOM    905  OE1 GLN A 124     -30.882  -6.966  -1.959  1.00  0.51           O  \\nATOM    906  N   PRO A 125     -24.849  -8.875  -0.329  1.00  0.46           N  \\nATOM    907  CA  PRO A 125     -24.021  -8.105   0.602  1.00  0.47           C  \\nATOM    908  C   PRO A 125     -24.782  -6.954   1.256  1.00  0.46           C  \\nATOM    909  CB  PRO A 125     -22.890  -7.578  -0.284  1.00  0.42           C  \\nATOM    910  O   PRO A 125     -25.429  -6.165   0.563  1.00  0.44           O  \\nATOM    911  CG  PRO A 125     -23.436  -7.636  -1.674  1.00  0.41           C  \\nATOM    912  CD  PRO A 125     -24.590  -8.596  -1.694  1.00  0.45           C  \\nATOM    913  N   GLN A 126     -25.349  -7.071   2.399  1.00  0.39           N  \\nATOM    914  CA  GLN A 126     -25.623  -6.015   3.368  1.00  0.42           C  \\nATOM    915  C   GLN A 126     -24.597  -4.891   3.262  1.00  0.39           C  \\nATOM    916  CB  GLN A 126     -25.638  -6.580   4.789  1.00  0.36           C  \\nATOM    917  O   GLN A 126     -23.393  -5.129   3.380  1.00  0.37           O  \\nATOM    918  CG  GLN A 126     -26.931  -7.295   5.154  1.00  0.37           C  \\nATOM    919  CD  GLN A 126     -26.988  -7.698   6.616  1.00  0.45           C  \\nATOM    920  NE2 GLN A 126     -28.140  -8.197   7.049  1.00  0.36           N  \\nATOM    921  OE1 GLN A 126     -26.003  -7.562   7.349  1.00  0.48           O  \\nATOM    922  N   SER A 127     -24.663  -3.752   2.461  1.00  0.39           N  \\nATOM    923  CA  SER A 127     -24.270  -2.380   2.766  1.00  0.43           C  \\nATOM    924  C   SER A 127     -23.971  -2.208   4.252  1.00  0.40           C  \\nATOM    925  CB  SER A 127     -25.366  -1.401   2.343  1.00  0.38           C  \\nATOM    926  O   SER A 127     -24.889  -2.079   5.065  1.00  0.38           O  \\nATOM    927  OG  SER A 127     -26.642  -1.867   2.749  1.00  0.40           O  \\nATOM    928  N   GLY A 128     -23.130  -2.998   4.839  1.00  0.37           N  \\nATOM    929  CA  GLY A 128     -22.581  -2.658   6.142  1.00  0.40           C  \\nATOM    930  C   GLY A 128     -21.415  -1.691   6.064  1.00  0.37           C  \\nATOM    931  O   GLY A 128     -20.522  -1.854   5.229  1.00  0.36           O  \\nATOM    932  N   SER A 129     -21.639  -0.286   6.174  1.00  0.40           N  \\nATOM    933  CA  SER A 129     -20.905   0.886   6.639  1.00  0.44           C  \\nATOM    934  C   SER A 129     -19.406   0.728   6.408  1.00  0.41           C  \\nATOM    935  CB  SER A 129     -21.177   1.135   8.123  1.00  0.38           C  \\nATOM    936  O   SER A 129     -18.776  -0.167   6.975  1.00  0.39           O  \\nATOM    937  OG  SER A 129     -21.154   2.522   8.413  1.00  0.40           O  \\nATOM    938  N   TYR A 130     -19.001   0.945   5.152  1.00  0.42           N  \\nATOM    939  CA  TYR A 130     -17.656   1.298   4.710  1.00  0.45           C  \\nATOM    940  C   TYR A 130     -17.026   2.324   5.644  1.00  0.42           C  \\nATOM    941  CB  TYR A 130     -17.687   1.845   3.279  1.00  0.40           C  \\nATOM    942  O   TYR A 130     -17.541   3.434   5.795  1.00  0.41           O  \\nATOM    943  CG  TYR A 130     -17.938   0.789   2.231  1.00  0.40           C  \\nATOM    944  CD1 TYR A 130     -16.887   0.044   1.700  1.00  0.37           C  \\nATOM    945  CD2 TYR A 130     -19.224   0.534   1.768  1.00  0.39           C  \\nATOM    946  CE1 TYR A 130     -17.112  -0.930   0.732  1.00  0.38           C  \\nATOM    947  CE2 TYR A 130     -19.461  -0.437   0.800  1.00  0.43           C  \\nATOM    948  OH  TYR A 130     -18.630  -2.125  -0.668  1.00  0.29           O  \\nATOM    949  CZ  TYR A 130     -18.401  -1.163   0.290  1.00  0.33           C  \\nATOM    950  N   SER A 131     -16.427   1.943   6.688  1.00  0.41           N  \\nATOM    951  CA  SER A 131     -15.359   2.647   7.390  1.00  0.45           C  \\nATOM    952  C   SER A 131     -14.282   3.125   6.421  1.00  0.41           C  \\nATOM    953  CB  SER A 131     -14.733   1.746   8.456  1.00  0.39           C  \\nATOM    954  O   SER A 131     -13.748   2.336   5.639  1.00  0.40           O  \\nATOM    955  OG  SER A 131     -14.171   0.586   7.867  1.00  0.39           O  \\nATOM    956  N   GLN A 132     -14.453   4.328   5.716  1.00  0.45           N  \\nATOM    957  CA  GLN A 132     -13.439   5.259   5.232  1.00  0.46           C  \\nATOM    958  C   GLN A 132     -12.041   4.820   5.658  1.00  0.45           C  \\nATOM    959  CB  GLN A 132     -13.722   6.674   5.738  1.00  0.42           C  \\nATOM    960  O   GLN A 132     -11.745   4.749   6.853  1.00  0.43           O  \\nATOM    961  CG  GLN A 132     -14.854   7.376   5.000  1.00  0.42           C  \\nATOM    962  CD  GLN A 132     -15.121   8.773   5.528  1.00  0.47           C  \\nATOM    963  NE2 GLN A 132     -16.128   9.436   4.970  1.00  0.40           N  \\nATOM    964  OE1 GLN A 132     -14.428   9.253   6.431  1.00  0.52           O  \\nATOM    965  N   GLN A 133     -11.391   3.971   5.029  1.00  0.43           N  \\nATOM    966  CA  GLN A 133      -9.934   4.028   4.969  1.00  0.47           C  \\nATOM    967  C   GLN A 133      -9.444   5.466   4.829  1.00  0.43           C  \\nATOM    968  CB  GLN A 133      -9.411   3.180   3.808  1.00  0.41           C  \\nATOM    969  O   GLN A 133     -10.008   6.247   4.060  1.00  0.42           O  \\nATOM    970  CG  GLN A 133      -8.116   2.441   4.120  1.00  0.41           C  \\nATOM    971  CD  GLN A 133      -7.661   1.548   2.982  1.00  0.48           C  \\nATOM    972  NE2 GLN A 133      -6.408   1.704   2.570  1.00  0.39           N  \\nATOM    973  OE1 GLN A 133      -8.430   0.723   2.477  1.00  0.51           O  \\nATOM    974  N   PRO A 134      -8.789   6.052   5.836  1.00  0.42           N  \\nATOM    975  CA  PRO A 134      -8.054   7.299   5.608  1.00  0.42           C  \\nATOM    976  C   PRO A 134      -7.497   7.405   4.190  1.00  0.42           C  \\nATOM    977  CB  PRO A 134      -6.922   7.234   6.636  1.00  0.39           C  \\nATOM    978  O   PRO A 134      -6.877   6.460   3.695  1.00  0.40           O  \\nATOM    979  CG  PRO A 134      -6.801   5.785   6.980  1.00  0.38           C  \\nATOM    980  CD  PRO A 134      -8.082   5.097   6.605  1.00  0.42           C  \\nATOM    981  N   SER A 135      -8.112   7.983   3.141  1.00  0.36           N  \\nATOM    982  CA  SER A 135      -7.418   8.761   2.120  1.00  0.37           C  \\nATOM    983  C   SER A 135      -6.062   9.247   2.621  1.00  0.36           C  \\nATOM    984  CB  SER A 135      -8.268   9.955   1.685  1.00  0.35           C  \\nATOM    985  O   SER A 135      -5.989  10.012   3.585  1.00  0.35           O  \\nATOM    986  OG  SER A 135      -7.781  11.155   2.260  1.00  0.38           O  \\nATOM    987  N   TYR A 136      -5.187   8.388   2.961  1.00  0.39           N  \\nATOM    988  CA  TYR A 136      -3.844   8.956   2.929  1.00  0.41           C  \\nATOM    989  C   TYR A 136      -3.495   9.451   1.531  1.00  0.39           C  \\nATOM    990  CB  TYR A 136      -2.812   7.923   3.390  1.00  0.37           C  \\nATOM    991  O   TYR A 136      -3.660   8.723   0.549  1.00  0.37           O  \\nATOM    992  CG  TYR A 136      -2.782   7.721   4.886  1.00  0.38           C  \\nATOM    993  CD1 TYR A 136      -2.021   8.552   5.705  1.00  0.36           C  \\nATOM    994  CD2 TYR A 136      -3.512   6.699   5.482  1.00  0.38           C  \\nATOM    995  CE1 TYR A 136      -1.988   8.368   7.084  1.00  0.38           C  \\nATOM    996  CE2 TYR A 136      -3.487   6.506   6.860  1.00  0.41           C  \\nATOM    997  OH  TYR A 136      -2.695   7.159   9.015  1.00  0.28           O  \\nATOM    998  CZ  TYR A 136      -2.724   7.345   7.651  1.00  0.33           C  \\nATOM    999  N   GLY A 137      -3.855  10.742   1.121  1.00  0.36           N  \\nATOM   1000  CA  GLY A 137      -3.296  11.566   0.062  1.00  0.37           C  \\nATOM   1001  C   GLY A 137      -2.895  10.770  -1.166  1.00  0.36           C  \\nATOM   1002  O   GLY A 137      -1.988   9.938  -1.104  1.00  0.34           O  \\nATOM   1003  N   GLY A 138      -3.640  10.006  -1.809  1.00  0.36           N  \\nATOM   1004  CA  GLY A 138      -3.319   9.718  -3.198  1.00  0.36           C  \\nATOM   1005  C   GLY A 138      -2.179  10.562  -3.735  1.00  0.36           C  \\nATOM   1006  O   GLY A 138      -2.301  11.785  -3.839  1.00  0.34           O  \\nATOM   1007  N   GLN A 139      -1.036  10.624  -3.185  1.00  0.36           N  \\nATOM   1008  CA  GLN A 139      -0.090  10.893  -4.263  1.00  0.38           C  \\nATOM   1009  C   GLN A 139      -0.489  10.160  -5.540  1.00  0.36           C  \\nATOM   1010  CB  GLN A 139       1.326  10.492  -3.848  1.00  0.34           C  \\nATOM   1011  O   GLN A 139      -0.505   8.927  -5.576  1.00  0.34           O  \\nATOM   1012  CG  GLN A 139       2.053  11.556  -3.035  1.00  0.34           C  \\nATOM   1013  CD  GLN A 139       3.483  11.169  -2.708  1.00  0.41           C  \\nATOM   1014  NE2 GLN A 139       4.227  12.096  -2.116  1.00  0.35           N  \\nATOM   1015  OE1 GLN A 139       3.915  10.046  -2.986  1.00  0.46           O  \\nATOM   1016  N   GLN A 140      -1.561  10.493  -6.230  1.00  0.38           N  \\nATOM   1017  CA  GLN A 140      -1.350  10.493  -7.674  1.00  0.39           C  \\nATOM   1018  C   GLN A 140       0.113  10.757  -8.015  1.00  0.38           C  \\nATOM   1019  CB  GLN A 140      -2.244  11.535  -8.348  1.00  0.36           C  \\nATOM   1020  O   GLN A 140       0.604  11.875  -7.844  1.00  0.37           O  \\nATOM   1021  CG  GLN A 140      -3.379  10.936  -9.166  1.00  0.36           C  \\nATOM   1022  CD  GLN A 140      -4.302  11.989  -9.749  1.00  0.43           C  \\nATOM   1023  NE2 GLN A 140      -5.190  11.569 -10.645  1.00  0.35           N  \\nATOM   1024  OE1 GLN A 140      -4.219  13.170  -9.398  1.00  0.48           O  \\nATOM   1025  N   GLN A 141       1.006   9.957  -7.662  1.00  0.34           N  \\nATOM   1026  CA  GLN A 141       2.117  10.013  -8.606  1.00  0.35           C  \\nATOM   1027  C   GLN A 141       1.633   9.810 -10.039  1.00  0.34           C  \\nATOM   1028  CB  GLN A 141       3.174   8.965  -8.255  1.00  0.32           C  \\nATOM   1029  O   GLN A 141       0.961   8.821 -10.338  1.00  0.32           O  \\nATOM   1030  CG  GLN A 141       4.041   9.340  -7.061  1.00  0.33           C  \\nATOM   1031  CD  GLN A 141       5.174   8.360  -6.826  1.00  0.40           C  \\nATOM   1032  NE2 GLN A 141       5.997   8.634  -5.820  1.00  0.33           N  \\nATOM   1033  OE1 GLN A 141       5.309   7.363  -7.543  1.00  0.45           O  \\nATOM   1034  N   SER A 142       1.100  10.799 -10.713  1.00  0.35           N  \\nATOM   1035  CA  SER A 142       1.366  10.990 -12.135  1.00  0.36           C  \\nATOM   1036  C   SER A 142       2.522  10.113 -12.604  1.00  0.35           C  \\nATOM   1037  CB  SER A 142       1.676  12.458 -12.430  1.00  0.33           C  \\nATOM   1038  O   SER A 142       3.656  10.276 -12.148  1.00  0.33           O  \\nATOM   1039  OG  SER A 142       0.496  13.162 -12.776  1.00  0.35           O  \\nATOM   1040  N   TYR A 143       2.496   8.847 -12.535  1.00  0.34           N  \\nATOM   1041  CA  TYR A 143       3.304   8.109 -13.499  1.00  0.37           C  \\nATOM   1042  C   TYR A 143       3.199   8.730 -14.887  1.00  0.34           C  \\nATOM   1043  CB  TYR A 143       2.871   6.641 -13.551  1.00  0.33           C  \\nATOM   1044  O   TYR A 143       2.149   8.658 -15.529  1.00  0.33           O  \\nATOM   1045  CG  TYR A 143       3.289   5.842 -12.340  1.00  0.34           C  \\nATOM   1046  CD1 TYR A 143       4.567   5.296 -12.248  1.00  0.33           C  \\nATOM   1047  CD2 TYR A 143       2.406   5.631 -11.286  1.00  0.35           C  \\nATOM   1048  CE1 TYR A 143       4.955   4.557 -11.135  1.00  0.34           C  \\nATOM   1049  CE2 TYR A 143       2.783   4.894 -10.168  1.00  0.38           C  \\nATOM   1050  OH  TYR A 143       4.437   3.633  -8.998  1.00  0.21           O  \\nATOM   1051  CZ  TYR A 143       4.058   4.363 -10.102  1.00  0.31           C  \\nATOM   1052  N   GLY A 144       3.534   9.847 -15.150  1.00  0.34           N  \\nATOM   1053  CA  GLY A 144       4.233  10.527 -16.229  1.00  0.36           C  \\nATOM   1054  C   GLY A 144       5.702  10.160 -16.313  1.00  0.34           C  \\nATOM   1055  O   GLY A 144       6.496  10.553 -15.456  1.00  0.33           O  \\nATOM   1056  N   GLN A 145       5.971   8.815 -16.182  1.00  0.33           N  \\nATOM   1057  CA  GLN A 145       7.095   8.218 -16.894  1.00  0.36           C  \\nATOM   1058  C   GLN A 145       6.718   6.859 -17.477  1.00  0.34           C  \\nATOM   1059  CB  GLN A 145       8.304   8.074 -15.968  1.00  0.31           C  \\nATOM   1060  O   GLN A 145       6.318   5.952 -16.743  1.00  0.32           O  \\nATOM   1061  CG  GLN A 145       9.111   9.355 -15.807  1.00  0.32           C  \\nATOM   1062  CD  GLN A 145      10.335   9.172 -14.930  1.00  0.39           C  \\nATOM   1063  NE2 GLN A 145      11.179  10.196 -14.868  1.00  0.33           N  \\nATOM   1064  OE1 GLN A 145      10.520   8.119 -14.311  1.00  0.45           O  \\nATOM   1065  N   GLN A 146       5.928   6.688 -18.643  1.00  0.36           N  \\nATOM   1066  CA  GLN A 146       6.265   5.702 -19.664  1.00  0.40           C  \\nATOM   1067  C   GLN A 146       7.709   5.230 -19.519  1.00  0.37           C  \\nATOM   1068  CB  GLN A 146       6.039   6.278 -21.063  1.00  0.35           C  \\nATOM   1069  O   GLN A 146       8.643   5.960 -19.857  1.00  0.36           O  \\nATOM   1070  CG  GLN A 146       4.700   5.892 -21.678  1.00  0.36           C  \\nATOM   1071  CD  GLN A 146       4.484   6.509 -23.047  1.00  0.44           C  \\nATOM   1072  NE2 GLN A 146       3.453   6.049 -23.749  1.00  0.36           N  \\nATOM   1073  OE1 GLN A 146       5.236   7.392 -23.471  1.00  0.48           O  \\nATOM   1074  N   GLN A 147       8.090   4.673 -18.461  1.00  0.33           N  \\nATOM   1075  CA  GLN A 147       9.135   3.710 -18.791  1.00  0.36           C  \\nATOM   1076  C   GLN A 147       8.595   2.598 -19.686  1.00  0.34           C  \\nATOM   1077  CB  GLN A 147       9.737   3.113 -17.518  1.00  0.32           C  \\nATOM   1078  O   GLN A 147       7.601   1.951 -19.351  1.00  0.32           O  \\nATOM   1079  CG  GLN A 147      10.728   4.032 -16.817  1.00  0.33           C  \\nATOM   1080  CD  GLN A 147      11.456   3.349 -15.674  1.00  0.40           C  \\nATOM   1081  NE2 GLN A 147      12.405   4.055 -15.070  1.00  0.33           N  \\nATOM   1082  OE1 GLN A 147      11.166   2.197 -15.337  1.00  0.45           O  \\nATOM   1083  N   SER A 148       8.387   2.780 -20.991  1.00  0.37           N  \\nATOM   1084  CA  SER A 148       8.548   1.575 -21.798  1.00  0.39           C  \\nATOM   1085  C   SER A 148       9.109   0.425 -20.968  1.00  0.37           C  \\nATOM   1086  CB  SER A 148       9.463   1.847 -22.992  1.00  0.35           C  \\nATOM   1087  O   SER A 148      10.240   0.498 -20.481  1.00  0.36           O  \\nATOM   1088  OG  SER A 148      10.279   2.981 -22.753  1.00  0.37           O  \\nATOM   1089  N   TYR A 149       8.588  -0.015 -19.954  1.00  0.32           N  \\nATOM   1090  CA  TYR A 149       9.000  -1.347 -19.523  1.00  0.36           C  \\nATOM   1091  C   TYR A 149       9.073  -2.304 -20.707  1.00  0.32           C  \\nATOM   1092  CB  TYR A 149       8.034  -1.894 -18.468  1.00  0.31           C  \\nATOM   1093  O   TYR A 149       8.049  -2.636 -21.309  1.00  0.31           O  \\nATOM   1094  CG  TYR A 149       8.661  -2.903 -17.537  1.00  0.33           C  \\nATOM   1095  CD1 TYR A 149       8.533  -4.270 -17.772  1.00  0.30           C  \\nATOM   1096  CD2 TYR A 149       9.380  -2.492 -16.420  1.00  0.31           C  \\nATOM   1097  CE1 TYR A 149       9.106  -5.204 -16.915  1.00  0.29           C  \\nATOM   1098  CE2 TYR A 149       9.958  -3.417 -15.557  1.00  0.33           C  \\nATOM   1099  OH  TYR A 149      10.387  -5.688 -14.961  1.00  0.12           O  \\nATOM   1100  CZ  TYR A 149       9.817  -4.768 -15.812  1.00  0.28           C  \\nATOM   1101  N   ASN A 150      10.079  -2.148 -21.535  1.00  0.36           N  \\nATOM   1102  CA  ASN A 150      10.721  -3.394 -21.937  1.00  0.39           C  \\nATOM   1103  C   ASN A 150      10.775  -4.394 -20.786  1.00  0.37           C  \\nATOM   1104  CB  ASN A 150      12.129  -3.125 -22.472  1.00  0.36           C  \\nATOM   1105  O   ASN A 150      11.441  -4.152 -19.778  1.00  0.35           O  \\nATOM   1106  CG  ASN A 150      12.303  -3.572 -23.910  1.00  0.36           C  \\nATOM   1107  ND2 ASN A 150      13.432  -3.214 -24.510  1.00  0.31           N  \\nATOM   1108  OD1 ASN A 150      11.429  -4.233 -24.478  1.00  0.37           O  \\nATOM   1109  N   PRO A 151       9.730  -5.032 -20.422  1.00  0.40           N  \\nATOM   1110  CA  PRO A 151      10.030  -6.134 -19.505  1.00  0.41           C  \\nATOM   1111  C   PRO A 151      11.466  -6.636 -19.636  1.00  0.40           C  \\nATOM   1112  CB  PRO A 151       9.033  -7.219 -19.919  1.00  0.39           C  \\nATOM   1113  O   PRO A 151      12.041  -6.597 -20.727  1.00  0.40           O  \\nATOM   1114  CG  PRO A 151       8.658  -6.877 -21.325  1.00  0.39           C  \\nATOM   1115  CD  PRO A 151       8.961  -5.425 -21.560  1.00  0.44           C  \\nATOM   1116  N   PRO A 152      12.244  -6.616 -18.677  1.00  0.35           N  \\nATOM   1117  CA  PRO A 152      13.533  -7.288 -18.857  1.00  0.35           C  \\nATOM   1118  C   PRO A 152      13.465  -8.434 -19.864  1.00  0.35           C  \\nATOM   1119  CB  PRO A 152      13.857  -7.811 -17.455  1.00  0.32           C  \\nATOM   1120  O   PRO A 152      12.554  -9.263 -19.800  1.00  0.32           O  \\nATOM   1121  CG  PRO A 152      12.598  -7.615 -16.674  1.00  0.32           C  \\nATOM   1122  CD  PRO A 152      11.652  -6.787 -17.494  1.00  0.37           C  \\nATOM   1123  N   GLN A 153      13.606  -8.321 -21.232  1.00  0.31           N  \\nATOM   1124  CA  GLN A 153      14.243  -9.422 -21.947  1.00  0.34           C  \\nATOM   1125  C   GLN A 153      15.051 -10.301 -20.997  1.00  0.32           C  \\nATOM   1126  CB  GLN A 153      15.143  -8.889 -23.063  1.00  0.31           C  \\nATOM   1127  O   GLN A 153      16.022  -9.842 -20.392  1.00  0.31           O  \\nATOM   1128  CG  GLN A 153      14.380  -8.358 -24.268  1.00  0.33           C  \\nATOM   1129  CD  GLN A 153      15.286  -8.040 -25.443  1.00  0.42           C  \\nATOM   1130  NE2 GLN A 153      14.686  -7.728 -26.587  1.00  0.33           N  \\nATOM   1131  OE1 GLN A 153      16.515  -8.072 -25.323  1.00  0.47           O  \\nATOM   1132  N   GLY A 154      14.327 -11.029 -19.978  1.00  0.32           N  \\nATOM   1133  CA  GLY A 154      15.010 -12.035 -19.182  1.00  0.35           C  \\nATOM   1134  C   GLY A 154      14.146 -12.606 -18.073  1.00  0.33           C  \\nATOM   1135  O   GLY A 154      13.850 -11.918 -17.093  1.00  0.31           O  \\nATOM   1136  N   TYR A 155      12.988 -13.277 -18.475  1.00  0.30           N  \\nATOM   1137  CA  TYR A 155      12.438 -14.575 -18.103  1.00  0.35           C  \\nATOM   1138  C   TYR A 155      11.657 -15.188 -19.259  1.00  0.31           C  \\nATOM   1139  CB  TYR A 155      11.534 -14.444 -16.873  1.00  0.29           C  \\nATOM   1140  O   TYR A 155      10.588 -14.694 -19.624  1.00  0.30           O  \\nATOM   1141  CG  TYR A 155      12.288 -14.187 -15.591  1.00  0.31           C  \\nATOM   1142  CD1 TYR A 155      12.846 -15.237 -14.865  1.00  0.31           C  \\nATOM   1143  CD2 TYR A 155      12.441 -12.895 -15.101  1.00  0.31           C  \\nATOM   1144  CE1 TYR A 155      13.540 -15.005 -13.682  1.00  0.28           C  \\nATOM   1145  CE2 TYR A 155      13.132 -12.651 -13.919  1.00  0.31           C  \\nATOM   1146  OH  TYR A 155      14.363 -13.475 -12.047  1.00  0.11           O  \\nATOM   1147  CZ  TYR A 155      13.678 -13.711 -13.218  1.00  0.29           C  \\nATOM   1148  N   GLY A 156      12.182 -15.584 -20.408  1.00  0.35           N  \\nATOM   1149  CA  GLY A 156      11.993 -16.937 -20.905  1.00  0.35           C  \\nATOM   1150  C   GLY A 156      11.741 -17.948 -19.802  1.00  0.35           C  \\nATOM   1151  O   GLY A 156      12.669 -18.350 -19.097  1.00  0.34           O  \\nATOM   1152  N   GLN A 157      11.178 -17.767 -18.769  1.00  0.33           N  \\nATOM   1153  CA  GLN A 157      10.736 -19.063 -18.265  1.00  0.36           C  \\nATOM   1154  C   GLN A 157       9.856 -19.779 -19.285  1.00  0.34           C  \\nATOM   1155  CB  GLN A 157       9.982 -18.897 -16.944  1.00  0.32           C  \\nATOM   1156  O   GLN A 157       8.784 -19.285 -19.643  1.00  0.33           O  \\nATOM   1157  CG  GLN A 157      10.889 -18.792 -15.726  1.00  0.33           C  \\nATOM   1158  CD  GLN A 157      10.117 -18.739 -14.422  1.00  0.40           C  \\nATOM   1159  NE2 GLN A 157      10.833 -18.567 -13.315  1.00  0.33           N  \\nATOM   1160  OE1 GLN A 157       8.887 -18.850 -14.408  1.00  0.44           O  \\nATOM   1161  N   GLN A 158      10.513 -20.183 -20.428  1.00  0.34           N  \\nATOM   1162  CA  GLN A 158      10.269 -21.502 -21.001  1.00  0.36           C  \\nATOM   1163  C   GLN A 158       9.440 -22.368 -20.056  1.00  0.35           C  \\nATOM   1164  CB  GLN A 158      11.591 -22.199 -21.327  1.00  0.33           C  \\nATOM   1165  O   GLN A 158       9.923 -22.782 -19.001  1.00  0.33           O  \\nATOM   1166  CG  GLN A 158      12.178 -21.804 -22.675  1.00  0.33           C  \\nATOM   1167  CD  GLN A 158      13.439 -22.576 -23.015  1.00  0.41           C  \\nATOM   1168  NE2 GLN A 158      13.929 -22.403 -24.238  1.00  0.33           N  \\nATOM   1169  OE1 GLN A 158      13.969 -23.321 -22.185  1.00  0.44           O  \\nATOM   1170  N   ASN A 159       8.393 -22.041 -19.561  1.00  0.38           N  \\nATOM   1171  CA  ASN A 159       7.594 -23.219 -19.238  1.00  0.42           C  \\nATOM   1172  C   ASN A 159       7.590 -24.226 -20.384  1.00  0.38           C  \\nATOM   1173  CB  ASN A 159       6.162 -22.815 -18.881  1.00  0.37           C  \\nATOM   1174  O   ASN A 159       7.064 -23.943 -21.462  1.00  0.37           O  \\nATOM   1175  CG  ASN A 159       5.999 -22.478 -17.412  1.00  0.38           C  \\nATOM   1176  ND2 ASN A 159       4.881 -21.849 -17.068  1.00  0.32           N  \\nATOM   1177  OD1 ASN A 159       6.872 -22.779 -16.593  1.00  0.39           O  \\nATOM   1178  N   GLN A 160       8.712 -24.726 -20.735  1.00  0.35           N  \\nATOM   1179  CA  GLN A 160       8.791 -26.103 -21.210  1.00  0.36           C  \\nATOM   1180  C   GLN A 160       7.522 -26.877 -20.866  1.00  0.35           C  \\nATOM   1181  CB  GLN A 160      10.012 -26.807 -20.617  1.00  0.33           C  \\nATOM   1182  O   GLN A 160       7.232 -27.115 -19.691  1.00  0.33           O  \\nATOM   1183  CG  GLN A 160      11.297 -26.574 -21.400  1.00  0.33           C  \\nATOM   1184  CD  GLN A 160      12.480 -27.330 -20.825  1.00  0.40           C  \\nATOM   1185  NE2 GLN A 160      13.655 -27.126 -21.411  1.00  0.33           N  \\nATOM   1186  OE1 GLN A 160      12.340 -28.090 -19.862  1.00  0.44           O  \\nATOM   1187  N   TYR A 161       6.266 -26.583 -21.389  1.00  0.42           N  \\nATOM   1188  CA  TYR A 161       5.534 -27.813 -21.669  1.00  0.45           C  \\nATOM   1189  C   TYR A 161       6.478 -29.008 -21.731  1.00  0.42           C  \\nATOM   1190  CB  TYR A 161       4.760 -27.690 -22.985  1.00  0.40           C  \\nATOM   1191  O   TYR A 161       7.337 -29.082 -22.612  1.00  0.40           O  \\nATOM   1192  CG  TYR A 161       3.400 -27.052 -22.832  1.00  0.41           C  \\nATOM   1193  CD1 TYR A 161       2.283 -27.817 -22.503  1.00  0.39           C  \\nATOM   1194  CD2 TYR A 161       3.229 -25.685 -23.018  1.00  0.40           C  \\nATOM   1195  CE1 TYR A 161       1.028 -27.235 -22.363  1.00  0.39           C  \\nATOM   1196  CE2 TYR A 161       1.978 -25.091 -22.881  1.00  0.42           C  \\nATOM   1197  OH  TYR A 161      -0.354 -25.291 -22.416  1.00  0.26           O  \\nATOM   1198  CZ  TYR A 161       0.886 -25.873 -22.553  1.00  0.36           C  \\nATOM   1199  N   ASN A 162       7.134 -29.247 -20.724  1.00  0.33           N  \\nATOM   1200  CA  ASN A 162       7.399 -30.682 -20.726  1.00  0.39           C  \\nATOM   1201  C   ASN A 162       6.236 -31.465 -21.329  1.00  0.34           C  \\nATOM   1202  CB  ASN A 162       7.692 -31.175 -19.307  1.00  0.32           C  \\nATOM   1203  O   ASN A 162       5.135 -31.474 -20.777  1.00  0.32           O  \\nATOM   1204  CG  ASN A 162       9.122 -31.650 -19.138  1.00  0.34           C  \\nATOM   1205  ND2 ASN A 162       9.448 -32.137 -17.946  1.00  0.18           N  \\nATOM   1206  OD1 ASN A 162       9.927 -31.579 -20.070  1.00  0.34           O  \\nATOM   1207  N   SER A 163       6.067 -31.467 -22.678  1.00  0.36           N  \\nATOM   1208  CA  SER A 163       5.915 -32.818 -23.209  1.00  0.38           C  \\nATOM   1209  C   SER A 163       6.333 -33.866 -22.183  1.00  0.37           C  \\nATOM   1210  CB  SER A 163       6.736 -32.988 -24.488  1.00  0.35           C  \\nATOM   1211  O   SER A 163       7.523 -34.030 -21.906  1.00  0.35           O  \\nATOM   1212  OG  SER A 163       8.111 -32.750 -24.237  1.00  0.36           O  \\nATOM   1213  N   SER A 164       5.810 -33.834 -21.034  1.00  0.35           N  \\nATOM   1214  CA  SER A 164       5.840 -35.210 -20.548  1.00  0.39           C  \\nATOM   1215  C   SER A 164       5.205 -36.166 -21.552  1.00  0.36           C  \\nATOM   1216  CB  SER A 164       5.121 -35.318 -19.202  1.00  0.33           C  \\nATOM   1217  O   SER A 164       3.998 -36.105 -21.797  1.00  0.34           O  \\nATOM   1218  OG  SER A 164       3.752 -34.974 -19.335  1.00  0.35           O  \\nATOM   1219  N   SER A 165       5.813 -36.352 -22.679  1.00  0.33           N  \\nATOM   1220  CA  SER A 165       6.040 -37.739 -23.070  1.00  0.35           C  \\nATOM   1221  C   SER A 165       5.885 -38.682 -21.881  1.00  0.33           C  \\nATOM   1222  CB  SER A 165       7.431 -37.903 -23.683  1.00  0.32           C  \\nATOM   1223  O   SER A 165       6.803 -38.823 -21.071  1.00  0.32           O  \\nATOM   1224  OG  SER A 165       8.433 -37.829 -22.683  1.00  0.35           O  \\nATOM   1225  N   GLY A 166       4.866 -38.682 -21.237  1.00  0.32           N  \\nATOM   1226  CA  GLY A 166       4.498 -39.824 -20.415  1.00  0.37           C  \\nATOM   1227  C   GLY A 166       3.216 -40.497 -20.868  1.00  0.33           C  \\nATOM   1228  O   GLY A 166       2.119 -40.021 -20.568  1.00  0.32           O  \\nATOM   1229  N   GLY A 167       3.302 -41.182 -22.045  1.00  0.32           N  \\nATOM   1230  CA  GLY A 167       2.648 -42.478 -22.137  1.00  0.36           C  \\nATOM   1231  C   GLY A 167       2.749 -43.100 -23.517  1.00  0.32           C  \\nATOM   1232  O   GLY A 167       2.384 -42.474 -24.514  1.00  0.31           O  \\nATOM   1233  N   GLY A 168       3.793 -43.763 -23.649  1.00  0.31           N  \\nATOM   1234  CA  GLY A 168       3.575 -45.148 -24.033  1.00  0.36           C  \\nATOM   1235  C   GLY A 168       3.069 -45.301 -25.454  1.00  0.32           C  \\nATOM   1236  O   GLY A 168       2.021 -44.756 -25.808  1.00  0.31           O  \\nATOM   1237  N   GLY A 169       4.018 -45.174 -26.439  1.00  0.31           N  \\nATOM   1238  CA  GLY A 169       3.898 -46.310 -27.339  1.00  0.36           C  \\nATOM   1239  C   GLY A 169       3.334 -45.938 -28.698  1.00  0.31           C  \\nATOM   1240  O   GLY A 169       2.189 -45.492 -28.799  1.00  0.31           O  \\nATOM   1241  N   GLY A 170       4.210 -45.298 -29.630  1.00  0.32           N  \\nATOM   1242  CA  GLY A 170       4.171 -45.958 -30.925  1.00  0.37           C  \\nATOM   1243  C   GLY A 170       4.125 -44.985 -32.089  1.00  0.32           C  \\nATOM   1244  O   GLY A 170       3.210 -44.164 -32.182  1.00  0.32           O  \\nATOM   1245  N   GLY A 171       5.298 -44.444 -32.539  1.00  0.32           N  \\nATOM   1246  CA  GLY A 171       5.670 -44.626 -33.933  1.00  0.38           C  \\nATOM   1247  C   GLY A 171       5.543 -43.356 -34.755  1.00  0.33           C  \\nATOM   1248  O   GLY A 171       4.457 -42.783 -34.857  1.00  0.33           O  \\nATOM   1249  N   GLY A 172       6.642 -42.435 -34.787  1.00  0.32           N  \\nATOM   1250  CA  GLY A 172       7.122 -42.122 -36.123  1.00  0.39           C  \\nATOM   1251  C   GLY A 172       6.674 -40.759 -36.615  1.00  0.32           C  \\nATOM   1252  O   GLY A 172       5.497 -40.561 -36.926  1.00  0.33           O  \\nATOM   1253  N   GLY A 173       7.413 -39.638 -36.117  1.00  0.35           N  \\nATOM   1254  CA  GLY A 173       7.934 -38.561 -36.943  1.00  0.38           C  \\nATOM   1255  C   GLY A 173       8.295 -37.321 -36.149  1.00  0.35           C  \\nATOM   1256  O   GLY A 173       7.443 -36.741 -35.472  1.00  0.35           O  \\nATOM   1257  N   GLY A 174       9.544 -37.288 -35.551  1.00  0.35           N  \\nATOM   1258  CA  GLY A 174      10.527 -36.217 -35.536  1.00  0.38           C  \\nATOM   1259  C   GLY A 174      10.035 -34.946 -36.203  1.00  0.36           C  \\nATOM   1260  O   GLY A 174      10.097 -34.818 -37.427  1.00  0.35           O  \\nATOM   1261  N   GLY A 175       9.086 -34.293 -35.879  1.00  0.35           N  \\nATOM   1262  CA  GLY A 175       9.001 -32.905 -36.302  1.00  0.38           C  \\nATOM   1263  C   GLY A 175       9.478 -31.928 -35.244  1.00  0.36           C  \\nATOM   1264  O   GLY A 175       9.029 -31.978 -34.097  1.00  0.36           O  \\nATOM   1265  N   ASN A 176      10.859 -31.753 -35.239  1.00  0.36           N  \\nATOM   1266  CA  ASN A 176      11.498 -30.475 -34.945  1.00  0.40           C  \\nATOM   1267  C   ASN A 176      10.476 -29.416 -34.541  1.00  0.36           C  \\nATOM   1268  CB  ASN A 176      12.314 -29.994 -36.147  1.00  0.35           C  \\nATOM   1269  O   ASN A 176       9.735 -28.909 -35.384  1.00  0.36           O  \\nATOM   1270  CG  ASN A 176      13.387 -28.994 -35.763  1.00  0.36           C  \\nATOM   1271  ND2 ASN A 176      13.885 -28.252 -36.744  1.00  0.29           N  \\nATOM   1272  OD1 ASN A 176      13.766 -28.891 -34.593  1.00  0.37           O  \\nATOM   1273  N   TYR A 177       9.926 -29.515 -33.473  1.00  0.39           N  \\nATOM   1274  CA  TYR A 177       9.386 -28.289 -32.895  1.00  0.43           C  \\nATOM   1275  C   TYR A 177      10.501 -27.299 -32.578  1.00  0.39           C  \\nATOM   1276  CB  TYR A 177       8.587 -28.599 -31.626  1.00  0.38           C  \\nATOM   1277  O   TYR A 177      11.290 -27.517 -31.656  1.00  0.39           O  \\nATOM   1278  CG  TYR A 177       7.258 -29.263 -31.892  1.00  0.39           C  \\nATOM   1279  CD1 TYR A 177       6.134 -28.509 -32.220  1.00  0.39           C  \\nATOM   1280  CD2 TYR A 177       7.124 -30.645 -31.816  1.00  0.40           C  \\nATOM   1281  CE1 TYR A 177       4.907 -29.117 -32.464  1.00  0.38           C  \\nATOM   1282  CE2 TYR A 177       5.902 -31.264 -32.058  1.00  0.42           C  \\nATOM   1283  OH  TYR A 177       3.589 -31.099 -32.623  1.00  0.27           O  \\nATOM   1284  CZ  TYR A 177       4.801 -30.493 -32.382  1.00  0.36           C  \\nATOM   1285  N   GLY A 178      11.214 -26.809 -33.554  1.00  0.33           N  \\nATOM   1286  CA  GLY A 178      11.924 -25.540 -33.563  1.00  0.35           C  \\nATOM   1287  C   GLY A 178      11.320 -24.512 -32.625  1.00  0.34           C  \\nATOM   1288  O   GLY A 178      10.142 -24.170 -32.746  1.00  0.33           O  \\nATOM   1289  N   GLN A 179      11.529 -24.673 -31.429  1.00  0.40           N  \\nATOM   1290  CA  GLN A 179      11.717 -23.533 -30.537  1.00  0.41           C  \\nATOM   1291  C   GLN A 179      12.545 -22.440 -31.208  1.00  0.40           C  \\nATOM   1292  CB  GLN A 179      12.387 -23.974 -29.235  1.00  0.37           C  \\nATOM   1293  O   GLN A 179      13.758 -22.588 -31.375  1.00  0.39           O  \\nATOM   1294  CG  GLN A 179      11.418 -24.146 -28.073  1.00  0.37           C  \\nATOM   1295  CD  GLN A 179      12.101 -24.623 -26.805  1.00  0.44           C  \\nATOM   1296  NE2 GLN A 179      11.347 -24.685 -25.713  1.00  0.36           N  \\nATOM   1297  OE1 GLN A 179      13.296 -24.935 -26.807  1.00  0.48           O  \\nATOM   1298  N   ASP A 180      11.879 -21.802 -32.245  1.00  0.34           N  \\nATOM   1299  CA  ASP A 180      12.266 -20.536 -32.860  1.00  0.37           C  \\nATOM   1300  C   ASP A 180      11.632 -19.354 -32.129  1.00  0.35           C  \\nATOM   1301  CB  ASP A 180      11.872 -20.514 -34.338  1.00  0.33           C  \\nATOM   1302  O   ASP A 180      10.420 -19.143 -32.214  1.00  0.33           O  \\nATOM   1303  CG  ASP A 180      12.979 -21.004 -35.254  1.00  0.34           C  \\nATOM   1304  OD1 ASP A 180      14.167 -20.922 -34.874  1.00  0.37           O  \\nATOM   1305  OD2 ASP A 180      12.661 -21.473 -36.368  1.00  0.38           O  \\nATOM   1306  N   GLN A 181      11.870 -19.064 -30.872  1.00  0.39           N  \\nATOM   1307  CA  GLN A 181      12.140 -17.712 -30.396  1.00  0.42           C  \\nATOM   1308  C   GLN A 181      13.259 -17.058 -31.201  1.00  0.39           C  \\nATOM   1309  CB  GLN A 181      12.503 -17.728 -28.910  1.00  0.38           C  \\nATOM   1310  O   GLN A 181      14.439 -17.231 -30.888  1.00  0.38           O  \\nATOM   1311  CG  GLN A 181      11.296 -17.737 -27.982  1.00  0.40           C  \\nATOM   1312  CD  GLN A 181      11.679 -17.624 -26.518  1.00  0.48           C  \\nATOM   1313  NE2 GLN A 181      10.711 -17.833 -25.633  1.00  0.40           N  \\nATOM   1314  OE1 GLN A 181      12.837 -17.352 -26.186  1.00  0.51           O  \\nATOM   1315  N   SER A 182      13.071 -17.146 -32.636  1.00  0.32           N  \\nATOM   1316  CA  SER A 182      13.715 -16.497 -33.774  1.00  0.35           C  \\nATOM   1317  C   SER A 182      13.200 -17.060 -35.095  1.00  0.32           C  \\nATOM   1318  CB  SER A 182      15.233 -16.662 -33.698  1.00  0.30           C  \\nATOM   1319  O   SER A 182      13.425 -18.231 -35.406  1.00  0.31           O  \\nATOM   1320  OG  SER A 182      15.580 -18.012 -33.441  1.00  0.29           O  \\nATOM   1321  N   SER A 183      11.825 -16.959 -35.358  1.00  0.30           N  \\nATOM   1322  CA  SER A 183      11.504 -16.850 -36.778  1.00  0.33           C  \\nATOM   1323  C   SER A 183      10.013 -16.610 -36.990  1.00  0.31           C  \\nATOM   1324  CB  SER A 183      11.937 -18.113 -37.524  1.00  0.28           C  \\nATOM   1325  O   SER A 183       9.185 -17.438 -36.606  1.00  0.29           O  \\nATOM   1326  OG  SER A 183      11.393 -19.270 -36.912  1.00  0.21           O  \\nATOM   1327  N   MET A 184       9.528 -15.370 -36.906  1.00  0.34           N  \\nATOM   1328  CA  MET A 184       9.050 -15.047 -38.247  1.00  0.37           C  \\nATOM   1329  C   MET A 184       9.549 -16.068 -39.264  1.00  0.35           C  \\nATOM   1330  CB  MET A 184       9.497 -13.642 -38.654  1.00  0.33           C  \\nATOM   1331  O   MET A 184      10.696 -15.996 -39.711  1.00  0.33           O  \\nATOM   1332  CG  MET A 184       8.661 -12.531 -38.040  1.00  0.35           C  \\nATOM   1333  SD  MET A 184       9.186 -10.864 -38.601  1.00  0.28           S  \\nATOM   1334  CE  MET A 184       7.819  -9.863 -37.952  1.00  0.36           C  \\nATOM   1335  N   SER A 185       9.339 -17.335 -38.958  1.00  0.30           N  \\nATOM   1336  CA  SER A 185       9.265 -18.073 -40.214  1.00  0.33           C  \\nATOM   1337  C   SER A 185       7.820 -18.242 -40.672  1.00  0.31           C  \\nATOM   1338  CB  SER A 185       9.926 -19.445 -40.071  1.00  0.29           C  \\nATOM   1339  O   SER A 185       6.979 -18.740 -39.920  1.00  0.29           O  \\nATOM   1340  OG  SER A 185       8.955 -20.477 -40.111  1.00  0.19           O  \\nATOM   1341  N   SER A 186       7.326 -17.244 -41.318  1.00  0.32           N  \\nATOM   1342  CA  SER A 186       6.815 -17.631 -42.629  1.00  0.34           C  \\nATOM   1343  C   SER A 186       6.980 -19.128 -42.868  1.00  0.32           C  \\nATOM   1344  CB  SER A 186       7.527 -16.849 -43.734  1.00  0.31           C  \\nATOM   1345  O   SER A 186       8.066 -19.590 -43.224  1.00  0.31           O  \\nATOM   1346  OG  SER A 186       8.890 -17.229 -43.821  1.00  0.35           O  \\nATOM   1347  N   GLY A 187       6.579 -19.976 -42.011  1.00  0.29           N  \\nATOM   1348  CA  GLY A 187       6.230 -21.283 -42.543  1.00  0.33           C  \\nATOM   1349  C   GLY A 187       4.738 -21.467 -42.742  1.00  0.30           C  \\nATOM   1350  O   GLY A 187       3.965 -21.404 -41.784  1.00  0.28           O  \\nATOM   1351  N   GLY A 188       4.337 -20.923 -43.803  1.00  0.31           N  \\nATOM   1352  CA  GLY A 188       3.511 -21.649 -44.755  1.00  0.33           C  \\nATOM   1353  C   GLY A 188       2.971 -22.955 -44.202  1.00  0.32           C  \\nATOM   1354  O   GLY A 188       3.688 -23.957 -44.150  1.00  0.31           O  \\nATOM   1355  N   GLY A 189       2.148 -22.920 -43.139  1.00  0.31           N  \\nATOM   1356  CA  GLY A 189       1.140 -23.968 -43.099  1.00  0.34           C  \\nATOM   1357  C   GLY A 189      -0.266 -23.435 -42.894  1.00  0.32           C  \\nATOM   1358  O   GLY A 189      -0.518 -22.679 -41.953  1.00  0.31           O  \\nATOM   1359  N   SER A 190      -1.040 -22.983 -43.849  1.00  0.32           N  \\nATOM   1360  CA  SER A 190      -2.410 -23.178 -44.312  1.00  0.35           C  \\nATOM   1361  C   SER A 190      -3.340 -23.528 -43.156  1.00  0.32           C  \\nATOM   1362  CB  SER A 190      -2.466 -24.278 -45.373  1.00  0.31           C  \\nATOM   1363  O   SER A 190      -3.166 -24.558 -42.502  1.00  0.31           O  \\nATOM   1364  OG  SER A 190      -3.747 -24.332 -45.976  1.00  0.34           O  \\nATOM   1365  N   GLY A 191      -3.800 -22.439 -42.410  1.00  0.33           N  \\nATOM   1366  CA  GLY A 191      -5.089 -22.676 -41.779  1.00  0.35           C  \\nATOM   1367  C   GLY A 191      -5.490 -21.578 -40.812  1.00  0.34           C  \\nATOM   1368  O   GLY A 191      -4.811 -21.348 -39.809  1.00  0.32           O  \\nATOM   1369  N   GLY A 192      -6.149 -20.415 -41.272  1.00  0.32           N  \\nATOM   1370  CA  GLY A 192      -7.291 -19.530 -41.436  1.00  0.36           C  \\nATOM   1371  C   GLY A 192      -8.149 -19.425 -40.190  1.00  0.33           C  \\nATOM   1372  O   GLY A 192      -8.709 -20.422 -39.729  1.00  0.32           O  \\nATOM   1373  N   GLY A 193      -7.904 -18.388 -39.337  1.00  0.34           N  \\nATOM   1374  CA  GLY A 193      -9.067 -17.591 -38.978  1.00  0.39           C  \\nATOM   1375  C   GLY A 193      -9.151 -17.296 -37.492  1.00  0.34           C  \\nATOM   1376  O   GLY A 193      -9.295 -18.212 -36.679  1.00  0.34           O  \\nATOM   1377  N   TYR A 194      -8.487 -16.161 -37.050  1.00  0.45           N  \\nATOM   1378  CA  TYR A 194      -9.020 -14.984 -36.373  1.00  0.48           C  \\nATOM   1379  C   TYR A 194      -8.060 -13.806 -36.491  1.00  0.44           C  \\nATOM   1380  CB  TYR A 194      -9.292 -15.289 -34.897  1.00  0.41           C  \\nATOM   1381  O   TYR A 194      -6.895 -13.904 -36.098  1.00  0.43           O  \\nATOM   1382  CG  TYR A 194     -10.486 -16.185 -34.673  1.00  0.41           C  \\nATOM   1383  CD1 TYR A 194     -11.779 -15.667 -34.672  1.00  0.41           C  \\nATOM   1384  CD2 TYR A 194     -10.323 -17.550 -34.460  1.00  0.42           C  \\nATOM   1385  CE1 TYR A 194     -12.882 -16.488 -34.463  1.00  0.41           C  \\nATOM   1386  CE2 TYR A 194     -11.420 -18.380 -34.251  1.00  0.44           C  \\nATOM   1387  OH  TYR A 194     -13.781 -18.659 -34.048  1.00  0.30           O  \\nATOM   1388  CZ  TYR A 194     -12.693 -17.841 -34.254  1.00  0.37           C  \\nATOM   1389  N   GLY A 195      -8.347 -12.916 -37.525  1.00  0.38           N  \\nATOM   1390  CA  GLY A 195      -8.142 -11.520 -37.876  1.00  0.40           C  \\nATOM   1391  C   GLY A 195      -7.304 -10.766 -36.861  1.00  0.38           C  \\nATOM   1392  O   GLY A 195      -7.704 -10.619 -35.705  1.00  0.37           O  \\nATOM   1393  N   ASN A 196      -6.009 -10.931 -36.917  1.00  0.42           N  \\nATOM   1394  CA  ASN A 196      -4.937  -9.947 -36.804  1.00  0.46           C  \\nATOM   1395  C   ASN A 196      -5.380  -8.576 -37.305  1.00  0.42           C  \\nATOM   1396  CB  ASN A 196      -3.696 -10.416 -37.568  1.00  0.40           C  \\nATOM   1397  O   ASN A 196      -5.525  -8.367 -38.510  1.00  0.42           O  \\nATOM   1398  CG  ASN A 196      -2.437 -10.360 -36.726  1.00  0.41           C  \\nATOM   1399  ND2 ASN A 196      -1.286 -10.520 -37.369  1.00  0.36           N  \\nATOM   1400  OD1 ASN A 196      -2.498 -10.177 -35.508  1.00  0.42           O  \\nATOM   1401  N   GLN A 197      -6.045  -7.813 -36.477  1.00  0.43           N  \\nATOM   1402  CA  GLN A 197      -6.219  -6.381 -36.693  1.00  0.47           C  \\nATOM   1403  C   GLN A 197      -5.226  -5.574 -35.861  1.00  0.43           C  \\nATOM   1404  CB  GLN A 197      -7.650  -5.956 -36.359  1.00  0.40           C  \\nATOM   1405  O   GLN A 197      -5.179  -5.711 -34.637  1.00  0.42           O  \\nATOM   1406  CG  GLN A 197      -8.545  -5.800 -37.581  1.00  0.41           C  \\nATOM   1407  CD  GLN A 197      -9.959  -5.382 -37.222  1.00  0.48           C  \\nATOM   1408  NE2 GLN A 197     -10.840  -5.354 -38.217  1.00  0.38           N  \\nATOM   1409  OE1 GLN A 197     -10.258  -5.089 -36.060  1.00  0.51           O  \\nATOM   1410  N   ASP A 198      -3.886  -5.315 -36.441  1.00  0.44           N  \\nATOM   1411  CA  ASP A 198      -3.156  -4.063 -36.611  1.00  0.46           C  \\nATOM   1412  C   ASP A 198      -1.663  -4.258 -36.355  1.00  0.44           C  \\nATOM   1413  CB  ASP A 198      -3.713  -2.985 -35.678  1.00  0.40           C  \\nATOM   1414  O   ASP A 198      -1.240  -4.416 -35.207  1.00  0.42           O  \\nATOM   1415  CG  ASP A 198      -4.895  -2.240 -36.273  1.00  0.40           C  \\nATOM   1416  OD1 ASP A 198      -4.981  -2.124 -37.514  1.00  0.44           O  \\nATOM   1417  OD2 ASP A 198      -5.746  -1.762 -35.492  1.00  0.46           O  \\nATOM   1418  N   GLN A 199      -0.985  -4.802 -37.387  1.00  0.43           N  \\nATOM   1419  CA  GLN A 199       0.419  -4.701 -37.773  1.00  0.45           C  \\nATOM   1420  C   GLN A 199       0.676  -3.436 -38.586  1.00  0.43           C  \\nATOM   1421  CB  GLN A 199       0.847  -5.934 -38.570  1.00  0.40           C  \\nATOM   1422  O   GLN A 199       0.271  -3.345 -39.747  1.00  0.42           O  \\nATOM   1423  CG  GLN A 199       1.119  -7.160 -37.709  1.00  0.40           C  \\nATOM   1424  CD  GLN A 199       1.585  -8.355 -38.518  1.00  0.46           C  \\nATOM   1425  NE2 GLN A 199       1.311  -9.554 -38.017  1.00  0.39           N  \\nATOM   1426  OE1 GLN A 199       2.187  -8.201 -39.586  1.00  0.51           O  \\nATOM   1427  N   SER A 200       0.806  -2.276 -37.982  1.00  0.38           N  \\nATOM   1428  CA  SER A 200       1.519  -1.054 -38.339  1.00  0.42           C  \\nATOM   1429  C   SER A 200       3.024  -1.290 -38.403  1.00  0.38           C  \\nATOM   1430  CB  SER A 200       1.212   0.058 -37.335  1.00  0.36           C  \\nATOM   1431  O   SER A 200       3.652  -1.610 -37.391  1.00  0.37           O  \\nATOM   1432  OG  SER A 200       1.999  -0.088 -36.166  1.00  0.39           O  \\nATOM   1433  N   GLY A 201       3.488  -1.930 -39.475  1.00  0.34           N  \\nATOM   1434  CA  GLY A 201       4.789  -1.509 -39.970  1.00  0.38           C  \\nATOM   1435  C   GLY A 201       5.465  -2.556 -40.835  1.00  0.35           C  \\nATOM   1436  O   GLY A 201       5.713  -3.676 -40.383  1.00  0.33           O  \\nATOM   1437  N   GLY A 202       5.219  -2.516 -42.090  1.00  0.34           N  \\nATOM   1438  CA  GLY A 202       6.178  -2.502 -43.183  1.00  0.37           C  \\nATOM   1439  C   GLY A 202       6.022  -3.678 -44.129  1.00  0.34           C  \\nATOM   1440  O   GLY A 202       6.063  -4.834 -43.703  1.00  0.33           O  \\nATOM   1441  N   GLY A 203       5.217  -3.544 -45.132  1.00  0.34           N  \\nATOM   1442  CA  GLY A 203       5.649  -3.847 -46.488  1.00  0.38           C  \\nATOM   1443  C   GLY A 203       5.958  -5.317 -46.700  1.00  0.35           C  \\nATOM   1444  O   GLY A 203       7.029  -5.793 -46.319  1.00  0.34           O  \\nATOM   1445  N   GLY A 204       4.855  -6.265 -46.711  1.00  0.36           N  \\nATOM   1446  CA  GLY A 204       4.929  -7.269 -47.760  1.00  0.39           C  \\nATOM   1447  C   GLY A 204       3.568  -7.676 -48.294  1.00  0.37           C  \\nATOM   1448  O   GLY A 204       2.703  -8.113 -47.532  1.00  0.36           O  \\nATOM   1449  N   SER A 205       2.973  -7.032 -49.297  1.00  0.33           N  \\nATOM   1450  CA  SER A 205       2.330  -7.459 -50.535  1.00  0.36           C  \\nATOM   1451  C   SER A 205       2.481  -8.961 -50.746  1.00  0.33           C  \\nATOM   1452  CB  SER A 205       2.916  -6.706 -51.731  1.00  0.32           C  \\nATOM   1453  O   SER A 205       3.587  -9.453 -50.979  1.00  0.33           O  \\nATOM   1454  OG  SER A 205       2.315  -7.137 -52.940  1.00  0.38           O  \\nATOM   1455  N   GLY A 206       1.709  -9.817 -49.981  1.00  0.36           N  \\nATOM   1456  CA  GLY A 206       1.404 -11.084 -50.626  1.00  0.39           C  \\nATOM   1457  C   GLY A 206       0.062 -11.657 -50.210  1.00  0.37           C  \\nATOM   1458  O   GLY A 206      -0.178 -11.890 -49.023  1.00  0.35           O  \\nATOM   1459  N   GLY A 207      -1.060 -11.422 -51.000  1.00  0.37           N  \\nATOM   1460  CA  GLY A 207      -2.160 -12.105 -51.661  1.00  0.39           C  \\nATOM   1461  C   GLY A 207      -2.276 -13.567 -51.270  1.00  0.37           C  \\nATOM   1462  O   GLY A 207      -1.425 -14.381 -51.634  1.00  0.36           O  \\nATOM   1463  N   TYR A 208      -2.792 -13.976 -50.122  1.00  0.40           N  \\nATOM   1464  CA  TYR A 208      -3.193 -15.363 -49.915  1.00  0.44           C  \\nATOM   1465  C   TYR A 208      -4.645 -15.448 -49.460  1.00  0.40           C  \\nATOM   1466  CB  TYR A 208      -2.282 -16.038 -48.886  1.00  0.37           C  \\nATOM   1467  O   TYR A 208      -5.029 -14.821 -48.470  1.00  0.39           O  \\nATOM   1468  CG  TYR A 208      -0.909 -16.373 -49.417  1.00  0.38           C  \\nATOM   1469  CD1 TYR A 208      -0.696 -17.520 -50.178  1.00  0.36           C  \\nATOM   1470  CD2 TYR A 208       0.177 -15.543 -49.159  1.00  0.38           C  \\nATOM   1471  CE1 TYR A 208       0.567 -17.834 -50.668  1.00  0.38           C  \\nATOM   1472  CE2 TYR A 208       1.445 -15.847 -49.644  1.00  0.40           C  \\nATOM   1473  OH  TYR A 208       2.883 -17.297 -50.879  1.00  0.29           O  \\nATOM   1474  CZ  TYR A 208       1.630 -16.992 -50.397  1.00  0.34           C  \\nATOM   1475  N   GLY A 209      -5.545 -15.865 -50.411  1.00  0.40           N  \\nATOM   1476  CA  GLY A 209      -6.708 -16.600 -50.882  1.00  0.41           C  \\nATOM   1477  C   GLY A 209      -7.395 -17.395 -49.788  1.00  0.40           C  \\nATOM   1478  O   GLY A 209      -6.771 -18.238 -49.140  1.00  0.39           O  \\nATOM   1479  N   GLN A 210      -8.407 -16.954 -49.096  1.00  0.44           N  \\nATOM   1480  CA  GLN A 210      -9.685 -17.438 -48.585  1.00  0.46           C  \\nATOM   1481  C   GLN A 210     -10.099 -18.733 -49.277  1.00  0.44           C  \\nATOM   1482  CB  GLN A 210     -10.772 -16.377 -48.762  1.00  0.42           C  \\nATOM   1483  O   GLN A 210     -10.428 -18.730 -50.466  1.00  0.43           O  \\nATOM   1484  CG  GLN A 210     -11.734 -16.280 -47.586  1.00  0.43           C  \\nATOM   1485  CD  GLN A 210     -12.697 -15.115 -47.710  1.00  0.49           C  \\nATOM   1486  NE2 GLN A 210     -12.413 -14.207 -48.638  1.00  0.40           N  \\nATOM   1487  OE1 GLN A 210     -13.689 -15.031 -46.978  1.00  0.54           O  \\nATOM   1488  N   GLN A 211      -9.788 -19.954 -48.635  1.00  0.46           N  \\nATOM   1489  CA  GLN A 211     -10.548 -21.167 -48.915  1.00  0.48           C  \\nATOM   1490  C   GLN A 211     -11.267 -21.667 -47.665  1.00  0.46           C  \\nATOM   1491  CB  GLN A 211      -9.631 -22.260 -49.466  1.00  0.43           C  \\nATOM   1492  O   GLN A 211     -10.625 -22.064 -46.691  1.00  0.44           O  \\nATOM   1493  CG  GLN A 211      -9.774 -22.485 -50.966  1.00  0.42           C  \\nATOM   1494  CD  GLN A 211      -8.900 -23.615 -51.476  1.00  0.48           C  \\nATOM   1495  NE2 GLN A 211      -7.593 -23.493 -51.271  1.00  0.40           N  \\nATOM   1496  OE1 GLN A 211      -9.395 -24.591 -52.048  1.00  0.52           O  \\nATOM   1497  N   ASP A 212     -12.565 -21.434 -47.498  1.00  0.46           N  \\nATOM   1498  CA  ASP A 212     -13.808 -22.088 -47.100  1.00  0.47           C  \\nATOM   1499  C   ASP A 212     -13.849 -23.534 -47.587  1.00  0.46           C  \\nATOM   1500  CB  ASP A 212     -15.016 -21.317 -47.637  1.00  0.43           C  \\nATOM   1501  O   ASP A 212     -14.017 -23.787 -48.782  1.00  0.44           O  \\nATOM   1502  CG  ASP A 212     -15.726 -20.506 -46.568  1.00  0.43           C  \\nATOM   1503  OD1 ASP A 212     -15.590 -20.828 -45.367  1.00  0.46           O  \\nATOM   1504  OD2 ASP A 212     -16.431 -19.539 -46.929  1.00  0.54           O  \\nATOM   1505  N   ARG A 213     -13.129 -24.415 -46.920  1.00  0.44           N  \\nATOM   1506  CA  ARG A 213     -13.201 -25.849 -47.183  1.00  0.44           C  \\nATOM   1507  C   ARG A 213     -14.294 -26.505 -46.346  1.00  0.44           C  \\nATOM   1508  CB  ARG A 213     -11.853 -26.516 -46.900  1.00  0.41           C  \\nATOM   1509  O   ARG A 213     -14.191 -26.567 -45.119  1.00  0.42           O  \\nATOM   1510  CG  ARG A 213     -10.834 -26.343 -48.015  1.00  0.41           C  \\nATOM   1511  CD  ARG A 213      -9.546 -27.100 -47.724  1.00  0.46           C  \\nATOM   1512  NE  ARG A 213      -8.588 -26.974 -48.819  1.00  0.39           N  \\nATOM   1513  NH1 ARG A 213      -6.882 -28.089 -47.729  1.00  0.29           N  \\nATOM   1514  NH2 ARG A 213      -6.557 -27.276 -49.848  1.00  0.24           N  \\nATOM   1515  CZ  ARG A 213      -7.344 -27.447 -48.796  1.00  0.36           C  \\nATOM   1516  N   GLY A 214     -15.573 -26.578 -46.665  1.00  0.40           N  \\nATOM   1517  CA  GLY A 214     -16.517 -27.664 -46.876  1.00  0.41           C  \\nATOM   1518  C   GLY A 214     -16.435 -28.741 -45.810  1.00  0.41           C  \\nATOM   1519  O   GLY A 214     -15.532 -29.580 -45.836  1.00  0.39           O  \\nATOM   1520  N   GLY A 215     -16.869 -28.469 -44.549  1.00  0.42           N  \\nATOM   1521  CA  GLY A 215     -17.317 -29.701 -43.919  1.00  0.44           C  \\nATOM   1522  C   GLY A 215     -18.805 -29.714 -43.624  1.00  0.42           C  \\nATOM   1523  O   GLY A 215     -19.299 -28.880 -42.863  1.00  0.41           O  \\nATOM   1524  N   ARG A 216     -19.671 -29.991 -44.518  1.00  0.42           N  \\nATOM   1525  CA  ARG A 216     -20.872 -30.819 -44.490  1.00  0.42           C  \\nATOM   1526  C   ARG A 216     -20.730 -31.955 -43.482  1.00  0.42           C  \\nATOM   1527  CB  ARG A 216     -21.167 -31.387 -45.880  1.00  0.39           C  \\nATOM   1528  O   ARG A 216     -20.042 -32.943 -43.744  1.00  0.40           O  \\nATOM   1529  CG  ARG A 216     -22.032 -30.482 -46.743  1.00  0.40           C  \\nATOM   1530  CD  ARG A 216     -22.357 -31.126 -48.084  1.00  0.47           C  \\nATOM   1531  NE  ARG A 216     -23.150 -30.238 -48.930  1.00  0.38           N  \\nATOM   1532  NH1 ARG A 216     -23.311 -31.722 -50.693  1.00  0.28           N  \\nATOM   1533  NH2 ARG A 216     -24.299 -29.658 -50.832  1.00  0.23           N  \\nATOM   1534  CZ  ARG A 216     -23.585 -30.541 -50.150  1.00  0.35           C  \\nATOM   1535  N   GLY A 217     -20.926 -31.775 -42.108  1.00  0.41           N  \\nATOM   1536  CA  GLY A 217     -21.472 -32.887 -41.347  1.00  0.43           C  \\nATOM   1537  C   GLY A 217     -22.854 -32.605 -40.787  1.00  0.41           C  \\nATOM   1538  O   GLY A 217     -23.061 -31.595 -40.111  1.00  0.40           O  \\nATOM   1539  N   ARG A 218     -23.910 -32.825 -41.519  1.00  0.34           N  \\nATOM   1540  CA  ARG A 218     -25.147 -33.565 -41.294  1.00  0.36           C  \\nATOM   1541  C   ARG A 218     -25.158 -34.207 -39.911  1.00  0.34           C  \\nATOM   1542  CB  ARG A 218     -25.337 -34.637 -42.370  1.00  0.32           C  \\nATOM   1543  O   ARG A 218     -24.260 -34.981 -39.573  1.00  0.33           O  \\nATOM   1544  CG  ARG A 218     -26.547 -34.409 -43.261  1.00  0.35           C  \\nATOM   1545  CD  ARG A 218     -26.646 -35.459 -44.358  1.00  0.42           C  \\nATOM   1546  NE  ARG A 218     -27.809 -35.238 -45.213  1.00  0.37           N  \\nATOM   1547  NH1 ARG A 218     -27.547 -37.186 -46.426  1.00  0.28           N  \\nATOM   1548  NH2 ARG A 218     -29.283 -35.764 -46.894  1.00  0.23           N  \\nATOM   1549  CZ  ARG A 218     -28.211 -36.063 -46.175  1.00  0.32           C  \\nATOM   1550  N   GLY A 219     -25.695 -33.471 -38.719  1.00  0.34           N  \\nATOM   1551  CA  GLY A 219     -26.578 -34.233 -37.851  1.00  0.39           C  \\nATOM   1552  C   GLY A 219     -26.028 -34.413 -36.449  1.00  0.35           C  \\nATOM   1553  O   GLY A 219     -25.059 -35.149 -36.246  1.00  0.34           O  \\nATOM   1554  N   GLY A 220     -26.065 -33.413 -35.526  1.00  0.36           N  \\nATOM   1555  CA  GLY A 220     -26.675 -33.448 -34.207  1.00  0.38           C  \\nATOM   1556  C   GLY A 220     -26.971 -32.068 -33.649  1.00  0.37           C  \\nATOM   1557  O   GLY A 220     -26.072 -31.232 -33.538  1.00  0.36           O  \\nATOM   1558  N   SER A 221     -27.922 -31.295 -34.172  1.00  0.32           N  \\nATOM   1559  CA  SER A 221     -28.970 -30.471 -33.579  1.00  0.36           C  \\nATOM   1560  C   SER A 221     -28.971 -30.590 -32.058  1.00  0.32           C  \\nATOM   1561  CB  SER A 221     -30.340 -30.868 -34.130  1.00  0.31           C  \\nATOM   1562  O   SER A 221     -29.190 -31.674 -31.515  1.00  0.32           O  \\nATOM   1563  OG  SER A 221     -31.335 -29.947 -33.716  1.00  0.33           O  \\nATOM   1564  N   GLY A 222     -27.934 -30.186 -31.304  1.00  0.35           N  \\nATOM   1565  CA  GLY A 222     -28.374 -29.725 -29.997  1.00  0.37           C  \\nATOM   1566  C   GLY A 222     -27.262 -29.700 -28.967  1.00  0.35           C  \\nATOM   1567  O   GLY A 222     -27.499 -29.395 -27.796  1.00  0.34           O  \\nATOM   1568  N   GLY A 223     -25.872 -29.765 -29.399  1.00  0.35           N  \\nATOM   1569  CA  GLY A 223     -24.897 -29.739 -28.321  1.00  0.37           C  \\nATOM   1570  C   GLY A 223     -23.644 -28.958 -28.668  1.00  0.36           C  \\nATOM   1571  O   GLY A 223     -22.814 -28.688 -27.798  1.00  0.33           O  \\nATOM   1572  N   GLY A 224     -23.473 -28.590 -30.009  1.00  0.35           N  \\nATOM   1573  CA  GLY A 224     -22.159 -28.058 -30.335  1.00  0.37           C  \\nATOM   1574  C   GLY A 224     -21.977 -26.615 -29.905  1.00  0.36           C  \\nATOM   1575  O   GLY A 224     -20.869 -26.201 -29.555  1.00  0.34           O  \\nATOM   1576  N   GLY A 225     -23.201 -25.847 -29.824  1.00  0.38           N  \\nATOM   1577  CA  GLY A 225     -23.105 -24.453 -29.421  1.00  0.40           C  \\nATOM   1578  C   GLY A 225     -22.743 -24.279 -27.958  1.00  0.39           C  \\nATOM   1579  O   GLY A 225     -21.887 -23.458 -27.618  1.00  0.37           O  \\nATOM   1580  N   GLN A 226     -23.310 -25.227 -27.134  1.00  0.40           N  \\nATOM   1581  CA  GLN A 226     -23.022 -25.166 -25.705  1.00  0.42           C  \\nATOM   1582  C   GLN A 226     -21.571 -25.544 -25.420  1.00  0.40           C  \\nATOM   1583  CB  GLN A 226     -23.966 -26.083 -24.927  1.00  0.39           C  \\nATOM   1584  O   GLN A 226     -20.898 -24.887 -24.623  1.00  0.40           O  \\nATOM   1585  CG  GLN A 226     -25.207 -25.379 -24.395  1.00  0.38           C  \\nATOM   1586  CD  GLN A 226     -26.084 -26.290 -23.556  1.00  0.44           C  \\nATOM   1587  NE2 GLN A 226     -27.025 -25.700 -22.827  1.00  0.36           N  \\nATOM   1588  OE1 GLN A 226     -25.919 -27.514 -23.565  1.00  0.46           O  \\nATOM   1589  N   GLN A 227     -21.061 -26.636 -26.113  1.00  0.45           N  \\nATOM   1590  CA  GLN A 227     -19.679 -27.038 -25.876  1.00  0.45           C  \\nATOM   1591  C   GLN A 227     -18.707 -25.939 -26.296  1.00  0.45           C  \\nATOM   1592  CB  GLN A 227     -19.361 -28.334 -26.623  1.00  0.43           C  \\nATOM   1593  O   GLN A 227     -17.763 -25.626 -25.568  1.00  0.44           O  \\nATOM   1594  CG  GLN A 227     -19.487 -29.586 -25.765  1.00  0.42           C  \\nATOM   1595  CD  GLN A 227     -19.148 -30.855 -26.524  1.00  0.47           C  \\nATOM   1596  NE2 GLN A 227     -19.081 -31.973 -25.810  1.00  0.38           N  \\nATOM   1597  OE1 GLN A 227     -18.951 -30.831 -27.743  1.00  0.48           O  \\nATOM   1598  N   GLN A 228     -18.919 -25.324 -27.465  1.00  0.44           N  \\nATOM   1599  CA  GLN A 228     -18.038 -24.238 -27.882  1.00  0.45           C  \\nATOM   1600  C   GLN A 228     -18.118 -23.062 -26.912  1.00  0.44           C  \\nATOM   1601  CB  GLN A 228     -18.388 -23.776 -29.297  1.00  0.42           C  \\nATOM   1602  O   GLN A 228     -17.092 -22.497 -26.528  1.00  0.44           O  \\nATOM   1603  CG  GLN A 228     -17.684 -24.564 -30.393  1.00  0.41           C  \\nATOM   1604  CD  GLN A 228     -18.018 -24.061 -31.785  1.00  0.46           C  \\nATOM   1605  NE2 GLN A 228     -17.517 -24.753 -32.802  1.00  0.39           N  \\nATOM   1606  OE1 GLN A 228     -18.723 -23.059 -31.945  1.00  0.48           O  \\nATOM   1607  N   GLN A 229     -19.357 -22.737 -26.507  1.00  0.43           N  \\nATOM   1608  CA  GLN A 229     -19.512 -21.677 -25.517  1.00  0.44           C  \\nATOM   1609  C   GLN A 229     -18.780 -22.022 -24.223  1.00  0.43           C  \\nATOM   1610  CB  GLN A 229     -20.992 -21.421 -25.230  1.00  0.41           C  \\nATOM   1611  O   GLN A 229     -18.084 -21.179 -23.654  1.00  0.43           O  \\nATOM   1612  CG  GLN A 229     -21.418 -19.974 -25.439  1.00  0.40           C  \\nATOM   1613  CD  GLN A 229     -22.898 -19.755 -25.188  1.00  0.45           C  \\nATOM   1614  NE2 GLN A 229     -23.329 -18.500 -25.241  1.00  0.39           N  \\nATOM   1615  OE1 GLN A 229     -23.648 -20.707 -24.951  1.00  0.48           O  \\nATOM   1616  N   GLN A 230     -18.952 -23.286 -23.747  1.00  0.46           N  \\nATOM   1617  CA  GLN A 230     -18.244 -23.725 -22.549  1.00  0.46           C  \\nATOM   1618  C   GLN A 230     -16.733 -23.628 -22.736  1.00  0.46           C  \\nATOM   1619  CB  GLN A 230     -18.637 -25.158 -22.188  1.00  0.44           C  \\nATOM   1620  O   GLN A 230     -16.018 -23.180 -21.836  1.00  0.46           O  \\nATOM   1621  CG  GLN A 230     -20.016 -25.276 -21.553  1.00  0.42           C  \\nATOM   1622  CD  GLN A 230     -20.388 -26.709 -21.219  1.00  0.45           C  \\nATOM   1623  NE2 GLN A 230     -21.537 -26.889 -20.577  1.00  0.40           N  \\nATOM   1624  OE1 GLN A 230     -19.649 -27.646 -21.538  1.00  0.50           O  \\nATOM   1625  N   GLN A 231     -16.264 -24.102 -23.931  1.00  0.46           N  \\nATOM   1626  CA  GLN A 231     -14.832 -24.011 -24.199  1.00  0.47           C  \\nATOM   1627  C   GLN A 231     -14.362 -22.560 -24.189  1.00  0.46           C  \\nATOM   1628  CB  GLN A 231     -14.495 -24.663 -25.541  1.00  0.44           C  \\nATOM   1629  O   GLN A 231     -13.332 -22.240 -23.591  1.00  0.46           O  \\nATOM   1630  CG  GLN A 231     -13.299 -25.604 -25.484  1.00  0.43           C  \\nATOM   1631  CD  GLN A 231     -13.015 -26.274 -26.815  1.00  0.47           C  \\nATOM   1632  NE2 GLN A 231     -11.887 -26.970 -26.900  1.00  0.41           N  \\nATOM   1633  OE1 GLN A 231     -13.805 -26.168 -27.759  1.00  0.50           O  \\nATOM   1634  N   TRP A 232     -15.112 -21.654 -24.821  1.00  0.34           N  \\nATOM   1635  CA  TRP A 232     -14.786 -20.232 -24.807  1.00  0.43           C  \\nATOM   1636  C   TRP A 232     -14.811 -19.681 -23.385  1.00  0.34           C  \\nATOM   1637  CB  TRP A 232     -15.762 -19.448 -25.689  1.00  0.35           C  \\nATOM   1638  O   TRP A 232     -13.900 -18.957 -22.978  1.00  0.35           O  \\nATOM   1639  CG  TRP A 232     -15.336 -19.341 -27.123  1.00  0.35           C  \\nATOM   1640  CD1 TRP A 232     -15.591 -20.233 -28.126  1.00  0.33           C  \\nATOM   1641  CD2 TRP A 232     -14.575 -18.282 -27.711  1.00  0.36           C  \\nATOM   1642  CE2 TRP A 232     -14.406 -18.598 -29.077  1.00  0.34           C  \\nATOM   1643  CE3 TRP A 232     -14.019 -17.095 -27.215  1.00  0.46           C  \\nATOM   1644  NE1 TRP A 232     -15.035 -19.792 -29.305  1.00  0.44           N  \\nATOM   1645  CH2 TRP A 232     -13.170 -16.613 -29.443  1.00  0.37           C  \\nATOM   1646  CZ2 TRP A 232     -13.703 -17.768 -29.954  1.00  0.41           C  \\nATOM   1647  CZ3 TRP A 232     -13.321 -16.270 -28.090  1.00  0.35           C  \\nATOM   1648  N   GLN A 233     -15.869 -19.990 -22.637  1.00  0.49           N  \\nATOM   1649  CA  GLN A 233     -15.949 -19.562 -21.244  1.00  0.51           C  \\nATOM   1650  C   GLN A 233     -14.741 -20.049 -20.450  1.00  0.49           C  \\nATOM   1651  CB  GLN A 233     -17.240 -20.068 -20.600  1.00  0.47           C  \\nATOM   1652  O   GLN A 233     -14.154 -19.291 -19.675  1.00  0.49           O  \\nATOM   1653  CG  GLN A 233     -18.464 -19.225 -20.932  1.00  0.45           C  \\nATOM   1654  CD  GLN A 233     -19.741 -19.782 -20.331  1.00  0.49           C  \\nATOM   1655  NE2 GLN A 233     -20.838 -19.049 -20.486  1.00  0.42           N  \\nATOM   1656  OE1 GLN A 233     -19.741 -20.862 -19.732  1.00  0.53           O  \\nATOM   1657  N   TRP A 234     -14.376 -21.306 -20.744  1.00  0.33           N  \\nATOM   1658  CA  TRP A 234     -13.223 -21.892 -20.068  1.00  0.44           C  \\nATOM   1659  C   TRP A 234     -11.938 -21.169 -20.457  1.00  0.34           C  \\nATOM   1660  CB  TRP A 234     -13.106 -23.382 -20.398  1.00  0.35           C  \\nATOM   1661  O   TRP A 234     -11.142 -20.797 -19.593  1.00  0.35           O  \\nATOM   1662  CG  TRP A 234     -12.566 -24.215 -19.274  1.00  0.35           C  \\nATOM   1663  CD1 TRP A 234     -13.250 -24.660 -18.177  1.00  0.32           C  \\nATOM   1664  CD2 TRP A 234     -11.227 -24.697 -19.135  1.00  0.36           C  \\nATOM   1665  CE2 TRP A 234     -11.169 -25.430 -17.929  1.00  0.34           C  \\nATOM   1666  CE3 TRP A 234     -10.068 -24.582 -19.915  1.00  0.45           C  \\nATOM   1667  NE1 TRP A 234     -12.415 -25.392 -17.364  1.00  0.43           N  \\nATOM   1668  CH2 TRP A 234      -8.877 -25.917 -18.266  1.00  0.36           C  \\nATOM   1669  CZ2 TRP A 234      -9.996 -26.045 -17.484  1.00  0.39           C  \\nATOM   1670  CZ3 TRP A 234      -8.902 -25.195 -19.471  1.00  0.35           C  \\nATOM   1671  N   GLN A 235     -11.716 -20.992 -21.705  1.00  0.51           N  \\nATOM   1672  CA  GLN A 235     -10.543 -20.271 -22.189  1.00  0.52           C  \\nATOM   1673  C   GLN A 235     -10.499 -18.853 -21.628  1.00  0.51           C  \\nATOM   1674  CB  GLN A 235     -10.529 -20.232 -23.717  1.00  0.49           C  \\nATOM   1675  O   GLN A 235      -9.445 -18.384 -21.193  1.00  0.52           O  \\nATOM   1676  CG  GLN A 235     -10.139 -21.555 -24.363  1.00  0.47           C  \\nATOM   1677  CD  GLN A 235     -10.228 -21.518 -25.877  1.00  0.50           C  \\nATOM   1678  NE2 GLN A 235      -9.678 -22.537 -26.528  1.00  0.44           N  \\nATOM   1679  OE1 GLN A 235     -10.788 -20.582 -26.456  1.00  0.55           O  \\nATOM   1680  N   TRP A 236     -11.683 -18.222 -21.639  1.00  0.40           N  \\nATOM   1681  CA  TRP A 236     -11.783 -16.880 -21.075  1.00  0.47           C  \\nATOM   1682  C   TRP A 236     -11.447 -16.887 -19.588  1.00  0.40           C  \\nATOM   1683  CB  TRP A 236     -13.187 -16.309 -21.291  1.00  0.39           C  \\nATOM   1684  O   TRP A 236     -10.664 -16.058 -19.118  1.00  0.41           O  \\nATOM   1685  CG  TRP A 236     -13.330 -14.873 -20.885  1.00  0.38           C  \\nATOM   1686  CD1 TRP A 236     -12.920 -13.773 -21.586  1.00  0.33           C  \\nATOM   1687  CD2 TRP A 236     -13.921 -14.381 -19.678  1.00  0.37           C  \\nATOM   1688  CE2 TRP A 236     -13.836 -12.972 -19.715  1.00  0.36           C  \\nATOM   1689  CE3 TRP A 236     -14.517 -14.997 -18.569  1.00  0.53           C  \\nATOM   1690  NE1 TRP A 236     -13.221 -12.627 -20.888  1.00  0.51           N  \\nATOM   1691  CH2 TRP A 236     -14.902 -12.795 -17.609  1.00  0.45           C  \\nATOM   1692  CZ2 TRP A 236     -14.324 -12.167 -18.682  1.00  0.57           C  \\nATOM   1693  CZ3 TRP A 236     -15.002 -14.194 -17.543  1.00  0.39           C  \\nATOM   1694  N   TRP A 237     -11.927 -17.832 -18.792  1.00  0.41           N  \\nATOM   1695  CA  TRP A 237     -11.650 -17.974 -17.366  1.00  0.49           C  \\nATOM   1696  C   TRP A 237     -10.164 -18.211 -17.121  1.00  0.41           C  \\nATOM   1697  CB  TRP A 237     -12.468 -19.123 -16.771  1.00  0.41           C  \\nATOM   1698  O   TRP A 237      -9.564 -17.577 -16.250  1.00  0.42           O  \\nATOM   1699  CG  TRP A 237     -12.582 -19.080 -15.277  1.00  0.40           C  \\nATOM   1700  CD1 TRP A 237     -13.374 -18.249 -14.534  1.00  0.35           C  \\nATOM   1701  CD2 TRP A 237     -11.874 -19.901 -14.342  1.00  0.38           C  \\nATOM   1702  CE2 TRP A 237     -12.288 -19.513 -13.048  1.00  0.37           C  \\nATOM   1703  CE3 TRP A 237     -10.930 -20.928 -14.473  1.00  0.50           C  \\nATOM   1704  NE1 TRP A 237     -13.202 -18.505 -13.193  1.00  0.48           N  \\nATOM   1705  CH2 TRP A 237     -10.866 -21.120 -12.049  1.00  0.42           C  \\nATOM   1706  CZ2 TRP A 237     -11.789 -20.118 -11.892  1.00  0.53           C  \\nATOM   1707  CZ3 TRP A 237     -10.434 -21.528 -13.322  1.00  0.38           C  \\nATOM   1708  N   GLN A 238      -9.520 -19.134 -17.892  1.00  0.53           N  \\nATOM   1709  CA  GLN A 238      -8.084 -19.376 -17.815  1.00  0.54           C  \\nATOM   1710  C   GLN A 238      -7.295 -18.094 -18.069  1.00  0.53           C  \\nATOM   1711  CB  GLN A 238      -7.666 -20.456 -18.814  1.00  0.51           C  \\nATOM   1712  O   GLN A 238      -6.335 -17.798 -17.354  1.00  0.53           O  \\nATOM   1713  CG  GLN A 238      -7.949 -21.875 -18.340  1.00  0.49           C  \\nATOM   1714  CD  GLN A 238      -7.194 -22.922 -19.138  1.00  0.52           C  \\nATOM   1715  NE2 GLN A 238      -7.005 -24.096 -18.545  1.00  0.44           N  \\nATOM   1716  OE1 GLN A 238      -6.784 -22.677 -20.277  1.00  0.54           O  \\nATOM   1717  N   GLN A 239      -7.768 -17.324 -19.094  1.00  0.55           N  \\nATOM   1718  CA  GLN A 239      -7.123 -16.047 -19.380  1.00  0.56           C  \\nATOM   1719  C   GLN A 239      -7.246 -15.090 -18.198  1.00  0.55           C  \\nATOM   1720  CB  GLN A 239      -7.725 -15.411 -20.635  1.00  0.53           C  \\nATOM   1721  O   GLN A 239      -6.270 -14.445 -17.810  1.00  0.54           O  \\nATOM   1722  CG  GLN A 239      -6.990 -15.770 -21.919  1.00  0.50           C  \\nATOM   1723  CD  GLN A 239      -7.650 -15.188 -23.155  1.00  0.52           C  \\nATOM   1724  NE2 GLN A 239      -7.070 -15.458 -24.320  1.00  0.44           N  \\nATOM   1725  OE1 GLN A 239      -8.673 -14.501 -23.064  1.00  0.55           O  \\nATOM   1726  N   GLN A 240      -8.465 -15.015 -17.586  1.00  0.56           N  \\nATOM   1727  CA  GLN A 240      -8.664 -14.165 -16.416  1.00  0.57           C  \\nATOM   1728  C   GLN A 240      -7.761 -14.595 -15.263  1.00  0.55           C  \\nATOM   1729  CB  GLN A 240     -10.128 -14.194 -15.974  1.00  0.53           C  \\nATOM   1730  O   GLN A 240      -7.128 -13.757 -14.619  1.00  0.55           O  \\nATOM   1731  CG  GLN A 240     -11.079 -13.518 -16.953  1.00  0.50           C  \\nATOM   1732  CD  GLN A 240     -10.792 -12.038 -17.122  1.00  0.53           C  \\nATOM   1733  NE2 GLN A 240     -11.111 -11.502 -18.296  1.00  0.44           N  \\nATOM   1734  OE1 GLN A 240     -10.288 -11.381 -16.206  1.00  0.55           O  \\nATOM   1735  N   GLN A 241      -7.715 -15.950 -14.960  1.00  0.56           N  \\nATOM   1736  CA  GLN A 241      -6.850 -16.466 -13.904  1.00  0.57           C  \\nATOM   1737  C   GLN A 241      -5.388 -16.123 -14.173  1.00  0.55           C  \\nATOM   1738  CB  GLN A 241      -7.018 -17.980 -13.764  1.00  0.53           C  \\nATOM   1739  O   GLN A 241      -4.670 -15.688 -13.271  1.00  0.55           O  \\nATOM   1740  CG  GLN A 241      -8.377 -18.400 -13.222  1.00  0.50           C  \\nATOM   1741  CD  GLN A 241      -8.579 -18.004 -11.771  1.00  0.53           C  \\nATOM   1742  NE2 GLN A 241      -9.817 -18.099 -11.297  1.00  0.44           N  \\nATOM   1743  OE1 GLN A 241      -7.631 -17.615 -11.082  1.00  0.54           O  \\nATOM   1744  N   GLN A 242      -4.963 -16.365 -15.487  1.00  0.54           N  \\nATOM   1745  CA  GLN A 242      -3.597 -16.016 -15.865  1.00  0.54           C  \\nATOM   1746  C   GLN A 242      -3.328 -14.531 -15.646  1.00  0.53           C  \\nATOM   1747  CB  GLN A 242      -3.333 -16.388 -17.325  1.00  0.52           C  \\nATOM   1748  O   GLN A 242      -2.279 -14.156 -15.117  1.00  0.53           O  \\nATOM   1749  CG  GLN A 242      -3.001 -17.860 -17.534  1.00  0.49           C  \\nATOM   1750  CD  GLN A 242      -2.771 -18.208 -18.993  1.00  0.52           C  \\nATOM   1751  NE2 GLN A 242      -2.569 -19.491 -19.272  1.00  0.44           N  \\nATOM   1752  OE1 GLN A 242      -2.776 -17.331 -19.863  1.00  0.54           O  \\nATOM   1753  N   GLN A 243      -4.290 -13.682 -16.080  1.00  0.55           N  \\nATOM   1754  CA  GLN A 243      -4.154 -12.247 -15.853  1.00  0.55           C  \\nATOM   1755  C   GLN A 243      -4.064 -11.932 -14.363  1.00  0.54           C  \\nATOM   1756  CB  GLN A 243      -5.326 -11.490 -16.480  1.00  0.52           C  \\nATOM   1757  O   GLN A 243      -3.241 -11.116 -13.944  1.00  0.54           O  \\nATOM   1758  CG  GLN A 243      -5.209 -11.315 -17.988  1.00  0.49           C  \\nATOM   1759  CD  GLN A 243      -6.380 -10.556 -18.583  1.00  0.52           C  \\nATOM   1760  NE2 GLN A 243      -6.377 -10.401 -19.903  1.00  0.44           N  \\nATOM   1761  OE1 GLN A 243      -7.281 -10.114 -17.863  1.00  0.54           O  \\nATOM   1762  N   GLN A 244      -4.961 -12.622 -13.491  1.00  0.57           N  \\nATOM   1763  CA  GLN A 244      -4.908 -12.435 -12.045  1.00  0.57           C  \\nATOM   1764  C   GLN A 244      -3.554 -12.860 -11.484  1.00  0.56           C  \\nATOM   1765  CB  GLN A 244      -6.029 -13.219 -11.361  1.00  0.53           C  \\nATOM   1766  O   GLN A 244      -2.968 -12.155 -10.660  1.00  0.56           O  \\nATOM   1767  CG  GLN A 244      -7.386 -12.529 -11.417  1.00  0.50           C  \\nATOM   1768  CD  GLN A 244      -8.458 -13.280 -10.651  1.00  0.53           C  \\nATOM   1769  NE2 GLN A 244      -9.642 -12.684 -10.550  1.00  0.44           N  \\nATOM   1770  OE1 GLN A 244      -8.226 -14.387 -10.154  1.00  0.54           O  \\nATOM   1771  N   TRP A 245      -2.986 -14.069 -11.943  1.00  0.48           N  \\nATOM   1772  CA  TRP A 245      -1.678 -14.554 -11.513  1.00  0.55           C  \\nATOM   1773  C   TRP A 245      -0.579 -13.568 -11.895  1.00  0.47           C  \\nATOM   1774  CB  TRP A 245      -1.386 -15.927 -12.125  1.00  0.46           C  \\nATOM   1775  O   TRP A 245       0.306 -13.272 -11.089  1.00  0.47           O  \\nATOM   1776  CG  TRP A 245      -1.917 -17.077 -11.323  1.00  0.44           C  \\nATOM   1777  CD1 TRP A 245      -3.227 -17.430 -11.157  1.00  0.38           C  \\nATOM   1778  CD2 TRP A 245      -1.148 -18.022 -10.573  1.00  0.41           C  \\nATOM   1779  CE2 TRP A 245      -2.058 -18.922  -9.977  1.00  0.38           C  \\nATOM   1780  CE3 TRP A 245       0.224 -18.195 -10.347  1.00  0.49           C  \\nATOM   1781  NE1 TRP A 245      -3.319 -18.540 -10.349  1.00  0.44           N  \\nATOM   1782  CH2 TRP A 245      -0.292 -20.129  -8.964  1.00  0.37           C  \\nATOM   1783  CZ2 TRP A 245      -1.640 -19.982  -9.169  1.00  0.43           C  \\nATOM   1784  CZ3 TRP A 245       0.639 -19.250  -9.542  1.00  0.36           C  \\nATOM   1785  N   TRP A 246      -0.622 -13.034 -13.214  1.00  0.50           N  \\nATOM   1786  CA  TRP A 246       0.333 -12.027 -13.664  1.00  0.57           C  \\nATOM   1787  C   TRP A 246       0.266 -10.782 -12.786  1.00  0.50           C  \\nATOM   1788  CB  TRP A 246       0.071 -11.651 -15.125  1.00  0.50           C  \\nATOM   1789  O   TRP A 246       1.299 -10.238 -12.389  1.00  0.50           O  \\nATOM   1790  CG  TRP A 246       0.815 -12.496 -16.115  1.00  0.49           C  \\nATOM   1791  CD1 TRP A 246       0.657 -13.836 -16.338  1.00  0.42           C  \\nATOM   1792  CD2 TRP A 246       1.840 -12.058 -17.012  1.00  0.44           C  \\nATOM   1793  CE2 TRP A 246       2.258 -13.187 -17.752  1.00  0.41           C  \\nATOM   1794  CE3 TRP A 246       2.445 -10.819 -17.264  1.00  0.47           C  \\nATOM   1795  NE1 TRP A 246       1.522 -14.257 -17.321  1.00  0.43           N  \\nATOM   1796  CH2 TRP A 246       3.830 -11.889 -18.954  1.00  0.36           C  \\nATOM   1797  CZ2 TRP A 246       3.255 -13.112 -18.727  1.00  0.39           C  \\nATOM   1798  CZ3 TRP A 246       3.437 -10.748 -18.236  1.00  0.35           C  \\nATOM   1799  N   TRP A 247      -1.026 -10.334 -12.456  1.00  0.54           N  \\nATOM   1800  CA  TRP A 247      -1.198  -9.191 -11.565  1.00  0.60           C  \\nATOM   1801  C   TRP A 247      -0.580  -9.469 -10.199  1.00  0.53           C  \\nATOM   1802  CB  TRP A 247      -2.682  -8.849 -11.408  1.00  0.53           C  \\nATOM   1803  O   TRP A 247       0.129  -8.624  -9.648  1.00  0.53           O  \\nATOM   1804  CG  TRP A 247      -3.201  -7.901 -12.447  1.00  0.50           C  \\nATOM   1805  CD1 TRP A 247      -3.675  -8.219 -13.690  1.00  0.44           C  \\nATOM   1806  CD2 TRP A 247      -3.294  -6.477 -12.334  1.00  0.45           C  \\nATOM   1807  CE2 TRP A 247      -3.835  -5.996 -13.547  1.00  0.43           C  \\nATOM   1808  CE3 TRP A 247      -2.973  -5.561 -11.324  1.00  0.49           C  \\nATOM   1809  NE1 TRP A 247      -4.058  -7.078 -14.356  1.00  0.45           N  \\nATOM   1810  CH2 TRP A 247      -3.738  -3.762 -12.772  1.00  0.38           C  \\nATOM   1811  CZ2 TRP A 247      -4.061  -4.637 -13.777  1.00  0.43           C  \\nATOM   1812  CZ3 TRP A 247      -3.199  -4.209 -11.554  1.00  0.37           C  \\nATOM   1813  N   TRP A 248      -0.836 -10.733  -9.643  1.00  0.56           N  \\nATOM   1814  CA  TRP A 248      -0.267 -11.125  -8.358  1.00  0.62           C  \\nATOM   1815  C   TRP A 248       1.257 -11.118  -8.412  1.00  0.55           C  \\nATOM   1816  CB  TRP A 248      -0.770 -12.512  -7.947  1.00  0.54           C  \\nATOM   1817  O   TRP A 248       1.914 -10.573  -7.522  1.00  0.54           O  \\nATOM   1818  CG  TRP A 248      -1.956 -12.483  -7.031  1.00  0.50           C  \\nATOM   1819  CD1 TRP A 248      -3.275 -12.496  -7.390  1.00  0.43           C  \\nATOM   1820  CD2 TRP A 248      -1.930 -12.431  -5.601  1.00  0.45           C  \\nATOM   1821  CE2 TRP A 248      -3.271 -12.416  -5.159  1.00  0.43           C  \\nATOM   1822  CE3 TRP A 248      -0.901 -12.397  -4.650  1.00  0.53           C  \\nATOM   1823  NE1 TRP A 248      -4.071 -12.456  -6.269  1.00  0.47           N  \\nATOM   1824  CH2 TRP A 248      -2.584 -12.335  -2.895  1.00  0.41           C  \\nATOM   1825  CZ2 TRP A 248      -3.610 -12.368  -3.805  1.00  0.48           C  \\nATOM   1826  CZ3 TRP A 248      -1.240 -12.349  -3.303  1.00  0.39           C  \\nATOM   1827  N   TRP A 249       1.861 -11.785  -9.517  1.00  0.52           N  \\nATOM   1828  CA  TRP A 249       3.309 -11.833  -9.690  1.00  0.59           C  \\nATOM   1829  C   TRP A 249       3.888 -10.430  -9.830  1.00  0.52           C  \\nATOM   1830  CB  TRP A 249       3.677 -12.676 -10.914  1.00  0.51           C  \\nATOM   1831  O   TRP A 249       4.898 -10.102  -9.202  1.00  0.52           O  \\nATOM   1832  CG  TRP A 249       4.483 -13.898 -10.593  1.00  0.50           C  \\nATOM   1833  CD1 TRP A 249       4.012 -15.165 -10.385  1.00  0.42           C  \\nATOM   1834  CD2 TRP A 249       5.904 -13.968 -10.437  1.00  0.44           C  \\nATOM   1835  CE2 TRP A 249       6.226 -15.310 -10.136  1.00  0.42           C  \\nATOM   1836  CE3 TRP A 249       6.937 -13.025 -10.525  1.00  0.50           C  \\nATOM   1837  NE1 TRP A 249       5.055 -16.019 -10.110  1.00  0.45           N  \\nATOM   1838  CH2 TRP A 249       8.531 -14.790 -10.013  1.00  0.37           C  \\nATOM   1839  CZ2 TRP A 249       7.540 -15.732  -9.922  1.00  0.42           C  \\nATOM   1840  CZ3 TRP A 249       8.244 -13.448 -10.311  1.00  0.37           C  \\nATOM   1841  N   TRP A 250       3.191  -9.594 -10.718  1.00  0.54           N  \\nATOM   1842  CA  TRP A 250       3.614  -8.204 -10.857  1.00  0.60           C  \\nATOM   1843  C   TRP A 250       3.567  -7.483  -9.514  1.00  0.53           C  \\nATOM   1844  CB  TRP A 250       2.733  -7.473 -11.874  1.00  0.53           C  \\nATOM   1845  O   TRP A 250       4.513  -6.783  -9.145  1.00  0.54           O  \\nATOM   1846  CG  TRP A 250       3.464  -6.446 -12.686  1.00  0.51           C  \\nATOM   1847  CD1 TRP A 250       4.146  -6.650 -13.853  1.00  0.44           C  \\nATOM   1848  CD2 TRP A 250       3.589  -5.052 -12.386  1.00  0.46           C  \\nATOM   1849  CE2 TRP A 250       4.361  -4.471 -13.417  1.00  0.44           C  \\nATOM   1850  CE3 TRP A 250       3.123  -4.237 -11.346  1.00  0.51           C  \\nATOM   1851  NE1 TRP A 250       4.688  -5.466 -14.298  1.00  0.45           N  \\nATOM   1852  CH2 TRP A 250       4.208  -2.336 -12.407  1.00  0.37           C  \\nATOM   1853  CZ2 TRP A 250       4.677  -3.110 -13.436  1.00  0.42           C  \\nATOM   1854  CZ3 TRP A 250       3.438  -2.884 -11.367  1.00  0.37           C  \\nATOM   1855  N   TRP A 251       2.437  -7.694  -8.757  1.00  0.56           N  \\nATOM   1856  CA  TRP A 251       2.322  -7.093  -7.432  1.00  0.62           C  \\nATOM   1857  C   TRP A 251       3.449  -7.565  -6.520  1.00  0.56           C  \\nATOM   1858  CB  TRP A 251       0.966  -7.430  -6.805  1.00  0.55           C  \\nATOM   1859  O   TRP A 251       4.076  -6.759  -5.829  1.00  0.56           O  \\nATOM   1860  CG  TRP A 251       0.161  -6.228  -6.412  1.00  0.53           C  \\nATOM   1861  CD1 TRP A 251      -0.825  -5.623  -7.140  1.00  0.45           C  \\nATOM   1862  CD2 TRP A 251       0.279  -5.482  -5.196  1.00  0.47           C  \\nATOM   1863  CE2 TRP A 251      -0.670  -4.437  -5.255  1.00  0.45           C  \\nATOM   1864  CE3 TRP A 251       1.093  -5.596  -4.062  1.00  0.53           C  \\nATOM   1865  NE1 TRP A 251      -1.329  -4.545  -6.450  1.00  0.47           N  \\nATOM   1866  CH2 TRP A 251      -0.015  -3.645  -3.123  1.00  0.40           C  \\nATOM   1867  CZ2 TRP A 251      -0.825  -3.510  -4.221  1.00  0.46           C  \\nATOM   1868  CZ3 TRP A 251       0.938  -4.674  -3.034  1.00  0.39           C  \\nATOM   1869  N   TRP A 252       3.753  -8.919  -6.558  1.00  0.53           N  \\nATOM   1870  CA  TRP A 252       4.816  -9.493  -5.740  1.00  0.60           C  \\nATOM   1871  C   TRP A 252       6.175  -8.929  -6.139  1.00  0.52           C  \\nATOM   1872  CB  TRP A 252       4.825 -11.019  -5.863  1.00  0.52           C  \\nATOM   1873  O   TRP A 252       6.971  -8.543  -5.279  1.00  0.52           O  \\nATOM   1874  CG  TRP A 252       4.885 -11.736  -4.548  1.00  0.50           C  \\nATOM   1875  CD1 TRP A 252       3.830 -12.090  -3.754  1.00  0.43           C  \\nATOM   1876  CD2 TRP A 252       6.064 -12.183  -3.872  1.00  0.45           C  \\nATOM   1877  CE2 TRP A 252       5.649 -12.801  -2.671  1.00  0.43           C  \\nATOM   1878  CE3 TRP A 252       7.433 -12.121  -4.165  1.00  0.52           C  \\nATOM   1879  NE1 TRP A 252       4.283 -12.731  -2.624  1.00  0.46           N  \\nATOM   1880  CH2 TRP A 252       7.889 -13.280  -2.075  1.00  0.38           C  \\nATOM   1881  CZ2 TRP A 252       6.556 -13.354  -1.764  1.00  0.44           C  \\nATOM   1882  CZ3 TRP A 252       8.334 -12.672  -3.260  1.00  0.38           C  \\nATOM   1883  N   TRP A 253       6.514  -8.835  -7.487  1.00  0.49           N  \\nATOM   1884  CA  TRP A 253       7.777  -8.289  -7.972  1.00  0.58           C  \\nATOM   1885  C   TRP A 253       7.889  -6.803  -7.645  1.00  0.49           C  \\nATOM   1886  CB  TRP A 253       7.910  -8.503  -9.483  1.00  0.50           C  \\nATOM   1887  O   TRP A 253       8.943  -6.335  -7.209  1.00  0.50           O  \\nATOM   1888  CG  TRP A 253       9.106  -9.316  -9.879  1.00  0.49           C  \\nATOM   1889  CD1 TRP A 253       9.209 -10.680  -9.885  1.00  0.42           C  \\nATOM   1890  CD2 TRP A 253      10.371  -8.816 -10.323  1.00  0.44           C  \\nATOM   1891  CE2 TRP A 253      11.195  -9.933 -10.582  1.00  0.42           C  \\nATOM   1892  CE3 TRP A 253      10.887  -7.529 -10.527  1.00  0.49           C  \\nATOM   1893  NE1 TRP A 253      10.463 -11.057 -10.307  1.00  0.44           N  \\nATOM   1894  CH2 TRP A 253      12.990  -8.532 -11.227  1.00  0.35           C  \\nATOM   1895  CZ2 TRP A 253      12.510  -9.802 -11.035  1.00  0.39           C  \\nATOM   1896  CZ3 TRP A 253      12.196  -7.401 -10.979  1.00  0.36           C  \\nATOM   1897  N   TRP A 254       6.751  -6.031  -7.827  1.00  0.51           N  \\nATOM   1898  CA  TRP A 254       6.735  -4.624  -7.443  1.00  0.59           C  \\nATOM   1899  C   TRP A 254       7.019  -4.463  -5.953  1.00  0.51           C  \\nATOM   1900  CB  TRP A 254       5.388  -3.985  -7.791  1.00  0.52           C  \\nATOM   1901  O   TRP A 254       7.781  -3.581  -5.552  1.00  0.52           O  \\nATOM   1902  CG  TRP A 254       5.477  -2.893  -8.814  1.00  0.51           C  \\nATOM   1903  CD1 TRP A 254       5.642  -3.039 -10.163  1.00  0.45           C  \\nATOM   1904  CD2 TRP A 254       5.411  -1.485  -8.568  1.00  0.46           C  \\nATOM   1905  CE2 TRP A 254       5.543  -0.836  -9.816  1.00  0.44           C  \\nATOM   1906  CE3 TRP A 254       5.252  -0.708  -7.413  1.00  0.49           C  \\nATOM   1907  NE1 TRP A 254       5.682  -1.806 -10.772  1.00  0.44           N  \\nATOM   1908  CH2 TRP A 254       5.364   1.291  -8.794  1.00  0.35           C  \\nATOM   1909  CZ2 TRP A 254       5.520   0.555  -9.940  1.00  0.40           C  \\nATOM   1910  CZ3 TRP A 254       5.230   0.676  -7.538  1.00  0.36           C  \\nATOM   1911  N   TRP A 255       6.550  -5.348  -5.052  1.00  0.52           N  \\nATOM   1912  CA  TRP A 255       6.842  -5.274  -3.624  1.00  0.60           C  \\nATOM   1913  C   TRP A 255       8.327  -5.493  -3.360  1.00  0.52           C  \\nATOM   1914  CB  TRP A 255       6.015  -6.306  -2.851  1.00  0.52           C  \\nATOM   1915  O   TRP A 255       8.908  -4.858  -2.477  1.00  0.52           O  \\nATOM   1916  CG  TRP A 255       4.681  -5.798  -2.392  1.00  0.51           C  \\nATOM   1917  CD1 TRP A 255       3.500  -5.847  -3.078  1.00  0.45           C  \\nATOM   1918  CD2 TRP A 255       4.395  -5.158  -1.145  1.00  0.46           C  \\nATOM   1919  CE2 TRP A 255       3.017  -4.846  -1.142  1.00  0.44           C  \\nATOM   1920  CE3 TRP A 255       5.170  -4.818  -0.028  1.00  0.51           C  \\nATOM   1921  NE1 TRP A 255       2.495  -5.277  -2.332  1.00  0.47           N  \\nATOM   1922  CH2 TRP A 255       3.178  -3.888   1.016  1.00  0.40           C  \\nATOM   1923  CZ2 TRP A 255       2.397  -4.210  -0.064  1.00  0.47           C  \\nATOM   1924  CZ3 TRP A 255       4.551  -4.185   1.044  1.00  0.39           C  \\nATOM   1925  N   TRP A 256       9.035  -6.370  -4.199  1.00  0.50           N  \\nATOM   1926  CA  TRP A 256      10.446  -6.651  -3.955  1.00  0.59           C  \\nATOM   1927  C   TRP A 256      11.320  -5.486  -4.409  1.00  0.51           C  \\nATOM   1928  CB  TRP A 256      10.871  -7.935  -4.673  1.00  0.50           C  \\nATOM   1929  O   TRP A 256      12.399  -5.259  -3.857  1.00  0.51           O  \\nATOM   1930  CG  TRP A 256      10.395  -9.191  -4.008  1.00  0.49           C  \\nATOM   1931  CD1 TRP A 256       9.183  -9.802  -4.172  1.00  0.42           C  \\nATOM   1932  CD2 TRP A 256      11.121  -9.987  -3.067  1.00  0.44           C  \\nATOM   1933  CE2 TRP A 256      10.288 -11.068  -2.702  1.00  0.43           C  \\nATOM   1934  CE3 TRP A 256      12.398  -9.892  -2.496  1.00  0.52           C  \\nATOM   1935  NE1 TRP A 256       9.112 -10.931  -3.390  1.00  0.47           N  \\nATOM   1936  CH2 TRP A 256      11.944 -11.931  -1.249  1.00  0.39           C  \\nATOM   1937  CZ2 TRP A 256      10.691 -12.048  -1.792  1.00  0.46           C  \\nATOM   1938  CZ3 TRP A 256      12.797 -10.868  -1.590  1.00  0.39           C  \\nATOM   1939  N   TRP A 257      10.813  -4.564  -5.378  1.00  0.43           N  \\nATOM   1940  CA  TRP A 257      11.606  -3.437  -5.857  1.00  0.53           C  \\nATOM   1941  C   TRP A 257      11.175  -2.141  -5.178  1.00  0.44           C  \\nATOM   1942  CB  TRP A 257      11.481  -3.297  -7.376  1.00  0.43           C  \\nATOM   1943  O   TRP A 257      11.733  -1.075  -5.450  1.00  0.44           O  \\nATOM   1944  CG  TRP A 257      12.252  -4.326  -8.148  1.00  0.44           C  \\nATOM   1945  CD1 TRP A 257      11.891  -5.623  -8.381  1.00  0.39           C  \\nATOM   1946  CD2 TRP A 257      13.520  -4.142  -8.784  1.00  0.41           C  \\nATOM   1947  CE2 TRP A 257      13.868  -5.372  -9.386  1.00  0.39           C  \\nATOM   1948  CE3 TRP A 257      14.397  -3.055  -8.905  1.00  0.46           C  \\nATOM   1949  NE1 TRP A 257      12.858  -6.258  -9.125  1.00  0.41           N  \\nATOM   1950  CH2 TRP A 257      15.897  -4.465 -10.201  1.00  0.31           C  \\nATOM   1951  CZ2 TRP A 257      15.058  -5.544 -10.098  1.00  0.34           C  \\nATOM   1952  CZ3 TRP A 257      15.580  -3.229  -9.615  1.00  0.34           C  \\nATOM   1953  N   TRP A 258      10.100  -2.120  -4.356  1.00  0.41           N  \\nATOM   1954  CA  TRP A 258       9.932  -0.964  -3.482  1.00  0.54           C  \\nATOM   1955  C   TRP A 258      11.028  -0.918  -2.422  1.00  0.43           C  \\nATOM   1956  CB  TRP A 258       8.556  -0.993  -2.810  1.00  0.43           C  \\nATOM   1957  O   TRP A 258      11.213  -1.878  -1.670  1.00  0.43           O  \\nATOM   1958  CG  TRP A 258       7.971   0.365  -2.562  1.00  0.45           C  \\nATOM   1959  CD1 TRP A 258       8.308   1.238  -1.565  1.00  0.43           C  \\nATOM   1960  CD2 TRP A 258       6.940   1.003  -3.323  1.00  0.43           C  \\nATOM   1961  CE2 TRP A 258       6.703   2.263  -2.730  1.00  0.41           C  \\nATOM   1962  CE3 TRP A 258       6.195   0.632  -4.449  1.00  0.48           C  \\nATOM   1963  NE1 TRP A 258       7.550   2.382  -1.661  1.00  0.44           N  \\nATOM   1964  CH2 TRP A 258       5.034   2.766  -4.331  1.00  0.36           C  \\nATOM   1965  CZ2 TRP A 258       5.750   3.155  -3.228  1.00  0.39           C  \\nATOM   1966  CZ3 TRP A 258       5.246   1.520  -4.943  1.00  0.36           C  \\nATOM   1967  N   PRO A 259      12.017  -0.071  -2.552  1.00  0.52           N  \\nATOM   1968  CA  PRO A 259      12.678   0.238  -1.281  1.00  0.52           C  \\nATOM   1969  C   PRO A 259      11.688   0.484  -0.145  1.00  0.53           C  \\nATOM   1970  CB  PRO A 259      13.474   1.507  -1.597  1.00  0.50           C  \\nATOM   1971  O   PRO A 259      10.575   0.960  -0.384  1.00  0.52           O  \\nATOM   1972  CG  PRO A 259      12.765   2.123  -2.759  1.00  0.50           C  \\nATOM   1973  CD  PRO A 259      12.030   1.042  -3.498  1.00  0.51           C  \\nATOM   1974  N   PRO A 260      11.751  -0.308   0.908  1.00  0.54           N  \\nATOM   1975  CA  PRO A 260      10.976   0.125   2.073  1.00  0.55           C  \\nATOM   1976  C   PRO A 260      10.783   1.638   2.124  1.00  0.54           C  \\nATOM   1977  CB  PRO A 260      11.819  -0.359   3.256  1.00  0.52           C  \\nATOM   1978  O   PRO A 260      11.638   2.390   1.649  1.00  0.54           O  \\nATOM   1979  CG  PRO A 260      13.198  -0.519   2.702  1.00  0.51           C  \\nATOM   1980  CD  PRO A 260      13.095  -0.729   1.219  1.00  0.52           C  \\nATOM   1981  N   PRO A 261       9.430   2.140   2.195  1.00  0.53           N  \\nATOM   1982  CA  PRO A 261       9.361   3.568   2.511  1.00  0.55           C  \\nATOM   1983  C   PRO A 261      10.484   4.022   3.441  1.00  0.55           C  \\nATOM   1984  CB  PRO A 261       7.997   3.713   3.191  1.00  0.52           C  \\nATOM   1985  O   PRO A 261      10.922   3.258   4.305  1.00  0.55           O  \\nATOM   1986  CG  PRO A 261       7.666   2.339   3.680  1.00  0.52           C  \\nATOM   1987  CD  PRO A 261       8.419   1.342   2.846  1.00  0.55           C  \\nATOM   1988  N   PRO A 262      11.228   5.075   3.039  1.00  0.53           N  \\nATOM   1989  CA  PRO A 262      12.120   5.650   4.049  1.00  0.55           C  \\nATOM   1990  C   PRO A 262      11.493   5.679   5.441  1.00  0.54           C  \\nATOM   1991  CB  PRO A 262      12.370   7.069   3.532  1.00  0.51           C  \\nATOM   1992  O   PRO A 262      10.269   5.771   5.569  1.00  0.54           O  \\nATOM   1993  CG  PRO A 262      11.226   7.347   2.612  1.00  0.51           C  \\nATOM   1994  CD  PRO A 262      10.677   6.038   2.123  1.00  0.54           C  \\nATOM   1995  N   PRO A 263      12.079   5.103   6.418  1.00  0.60           N  \\nATOM   1996  CA  PRO A 263      11.622   5.254   7.801  1.00  0.63           C  \\nATOM   1997  C   PRO A 263      10.970   6.609   8.063  1.00  0.62           C  \\nATOM   1998  CB  PRO A 263      12.909   5.100   8.616  1.00  0.58           C  \\nATOM   1999  O   PRO A 263      11.315   7.600   7.414  1.00  0.60           O  \\nATOM   2000  CG  PRO A 263      14.007   5.455   7.666  1.00  0.57           C  \\nATOM   2001  CD  PRO A 263      13.538   5.172   6.268  1.00  0.58           C  \\nATOM   2002  N   PRO A 264       9.738   6.618   8.615  1.00  0.63           N  \\nATOM   2003  CA  PRO A 264       9.215   7.914   9.054  1.00  0.68           C  \\nATOM   2004  C   PRO A 264      10.297   8.821   9.635  1.00  0.66           C  \\nATOM   2005  CB  PRO A 264       8.187   7.536  10.123  1.00  0.61           C  \\nATOM   2006  O   PRO A 264      11.249   8.335  10.251  1.00  0.65           O  \\nATOM   2007  CG  PRO A 264       8.616   6.188  10.606  1.00  0.60           C  \\nATOM   2008  CD  PRO A 264       9.364   5.501   9.499  1.00  0.61           C  \\nATOM   2009  N   PRO A 265      10.325  10.079   9.172  1.00  0.62           N  \\nATOM   2010  CA  PRO A 265      11.201  11.020   9.874  1.00  0.66           C  \\nATOM   2011  C   PRO A 265      11.098  10.900  11.393  1.00  0.64           C  \\nATOM   2012  CB  PRO A 265      10.702  12.387   9.400  1.00  0.60           C  \\nATOM   2013  O   PRO A 265      10.040  10.541  11.918  1.00  0.62           O  \\nATOM   2014  CG  PRO A 265       9.290  12.151   8.971  1.00  0.58           C  \\nATOM   2015  CD  PRO A 265       9.153  10.716   8.549  1.00  0.60           C  \\nATOM   2016  N   PRO A 266      12.131  10.739  12.066  1.00  0.69           N  \\nATOM   2017  CA  PRO A 266      12.131  10.787  13.530  1.00  0.72           C  \\nATOM   2018  C   PRO A 266      11.161  11.827  14.086  1.00  0.71           C  \\nATOM   2019  CB  PRO A 266      13.578  11.153  13.871  1.00  0.66           C  \\nATOM   2020  O   PRO A 266      10.883  12.832  13.427  1.00  0.67           O  \\nATOM   2021  CG  PRO A 266      14.089  11.862  12.658  1.00  0.64           C  \\nATOM   2022  CD  PRO A 266      13.344  11.352  11.458  1.00  0.64           C  \\nATOM   2023  N   PRO A 267      10.302  11.468  15.010  1.00  0.72           N  \\nATOM   2024  CA  PRO A 267       9.492  12.472  15.702  1.00  0.75           C  \\nATOM   2025  C   PRO A 267      10.262  13.760  15.983  1.00  0.74           C  \\nATOM   2026  CB  PRO A 267       9.102  11.773  17.007  1.00  0.69           C  \\nATOM   2027  O   PRO A 267      11.479  13.725  16.183  1.00  0.70           O  \\nATOM   2028  CG  PRO A 267      10.170  10.750  17.225  1.00  0.66           C  \\nATOM   2029  CD  PRO A 267      10.720  10.342  15.888  1.00  0.66           C  \\nATOM   2030  N   PRO A 268       9.693  14.901  15.649  1.00  0.71           N  \\nATOM   2031  CA  PRO A 268      10.311  16.154  16.088  1.00  0.73           C  \\nATOM   2032  C   PRO A 268      10.806  16.094  17.531  1.00  0.72           C  \\nATOM   2033  CB  PRO A 268       9.181  17.176  15.940  1.00  0.67           C  \\nATOM   2034  O   PRO A 268      10.235  15.373  18.354  1.00  0.68           O  \\nATOM   2035  CG  PRO A 268       7.925  16.371  16.025  1.00  0.65           C  \\nATOM   2036  CD  PRO A 268       8.214  14.982  15.535  1.00  0.65           C  \\nATOM   2037  N   PRO A 269      12.009  16.440  17.797  1.00  0.72           N  \\nATOM   2038  CA  PRO A 269      12.512  16.577  19.166  1.00  0.73           C  \\nATOM   2039  C   PRO A 269      11.491  17.206  20.111  1.00  0.73           C  \\nATOM   2040  CB  PRO A 269      13.735  17.483  19.003  1.00  0.69           C  \\nATOM   2041  O   PRO A 269      10.659  18.009  19.680  1.00  0.68           O  \\nATOM   2042  CG  PRO A 269      13.477  18.251  17.747  1.00  0.67           C  \\nATOM   2043  CD  PRO A 269      12.612  17.416  16.847  1.00  0.67           C  \\nATOM   2044  N   PRO A 270      11.168  16.605  21.200  1.00  0.74           N  \\nATOM   2045  CA  PRO A 270      10.350  17.235  22.238  1.00  0.75           C  \\nATOM   2046  C   PRO A 270      10.634  18.728  22.389  1.00  0.74           C  \\nATOM   2047  CB  PRO A 270      10.742  16.473  23.507  1.00  0.71           C  \\nATOM   2048  O   PRO A 270      11.768  19.168  22.181  1.00  0.70           O  \\nATOM   2049  CG  PRO A 270      12.126  15.974  23.240  1.00  0.69           C  \\nATOM   2050  CD  PRO A 270      12.278  15.758  21.762  1.00  0.68           C  \\nATOM   2051  N   PRO A 271       9.598  19.567  22.219  1.00  0.74           N  \\nATOM   2052  CA  PRO A 271       9.804  20.982  22.534  1.00  0.75           C  \\nATOM   2053  C   PRO A 271      10.812  21.195  23.661  1.00  0.73           C  \\nATOM   2054  CB  PRO A 271       8.410  21.457  22.951  1.00  0.69           C  \\nATOM   2055  O   PRO A 271      10.708  20.562  24.715  1.00  0.68           O  \\nATOM   2056  CG  PRO A 271       7.629  20.202  23.171  1.00  0.65           C  \\nATOM   2057  CD  PRO A 271       8.380  19.058  22.553  1.00  0.65           C  \\nATOM   2058  N   GLN A 272      12.069  21.577  23.282  1.00  0.74           N  \\nATOM   2059  CA  GLN A 272      13.046  22.053  24.254  1.00  0.76           C  \\nATOM   2060  C   GLN A 272      12.363  22.766  25.418  1.00  0.73           C  \\nATOM   2061  CB  GLN A 272      14.057  22.988  23.587  1.00  0.69           C  \\nATOM   2062  O   GLN A 272      11.619  23.728  25.213  1.00  0.67           O  \\nATOM   2063  CG  GLN A 272      15.283  22.273  23.034  1.00  0.64           C  \\nATOM   2064  CD  GLN A 272      16.211  23.203  22.275  1.00  0.63           C  \\nATOM   2065  NE2 GLN A 272      17.243  22.637  21.659  1.00  0.55           N  \\nATOM   2066  OE1 GLN A 272      16.002  24.420  22.243  1.00  0.67           O  \\nATOM   2067  N   GLN A 273      12.026  21.993  26.571  1.00  0.78           N  \\nATOM   2068  CA  GLN A 273      11.672  22.635  27.833  1.00  0.79           C  \\nATOM   2069  C   GLN A 273      12.427  23.950  28.011  1.00  0.78           C  \\nATOM   2070  CB  GLN A 273      11.959  21.701  29.010  1.00  0.72           C  \\nATOM   2071  O   GLN A 273      13.658  23.964  28.054  1.00  0.71           O  \\nATOM   2072  CG  GLN A 273      10.733  20.954  29.515  1.00  0.66           C  \\nATOM   2073  CD  GLN A 273      11.069  19.924  30.577  1.00  0.65           C  \\nATOM   2074  NE2 GLN A 273      10.076  19.136  30.975  1.00  0.57           N  \\nATOM   2075  OE1 GLN A 273      12.213  19.837  31.036  1.00  0.69           O  \\nATOM   2076  N   GLN A 274      11.826  25.101  27.497  1.00  0.78           N  \\nATOM   2077  CA  GLN A 274      12.301  26.423  27.891  1.00  0.80           C  \\nATOM   2078  C   GLN A 274      12.762  26.431  29.346  1.00  0.79           C  \\nATOM   2079  CB  GLN A 274      11.208  27.472  27.681  1.00  0.73           C  \\nATOM   2080  O   GLN A 274      11.986  26.115  30.250  1.00  0.73           O  \\nATOM   2081  CG  GLN A 274      11.058  27.923  26.235  1.00  0.67           C  \\nATOM   2082  CD  GLN A 274       9.932  28.922  26.045  1.00  0.66           C  \\nATOM   2083  NE2 GLN A 274       9.721  29.349  24.805  1.00  0.60           N  \\nATOM   2084  OE1 GLN A 274       9.257  29.306  27.005  1.00  0.72           O  \\nATOM   2085  N   GLN A 275      14.116  26.195  29.546  1.00  0.79           N  \\nATOM   2086  CA  GLN A 275      14.702  26.500  30.847  1.00  0.81           C  \\nATOM   2087  C   GLN A 275      14.109  27.780  31.430  1.00  0.81           C  \\nATOM   2088  CB  GLN A 275      16.221  26.630  30.735  1.00  0.75           C  \\nATOM   2089  O   GLN A 275      14.043  28.806  30.750  1.00  0.75           O  \\nATOM   2090  CG  GLN A 275      16.941  26.606  32.077  1.00  0.69           C  \\nATOM   2091  CD  GLN A 275      18.452  26.622  31.933  1.00  0.68           C  \\nATOM   2092  NE2 GLN A 275      19.151  26.824  33.044  1.00  0.60           N  \\nATOM   2093  OE1 GLN A 275      18.985  26.453  30.832  1.00  0.73           O  \\nATOM   2094  N   PRO A 276      13.313  27.591  32.542  1.00  0.81           N  \\nATOM   2095  CA  PRO A 276      12.874  28.797  33.247  1.00  0.83           C  \\nATOM   2096  C   PRO A 276      13.971  29.855  33.346  1.00  0.83           C  \\nATOM   2097  CB  PRO A 276      12.492  28.276  34.635  1.00  0.79           C  \\nATOM   2098  O   PRO A 276      15.155  29.516  33.429  1.00  0.79           O  \\nATOM   2099  CG  PRO A 276      13.223  26.979  34.770  1.00  0.76           C  \\nATOM   2100  CD  PRO A 276      13.496  26.439  33.396  1.00  0.75           C  \\nATOM   2101  N   PRO A 277      13.647  31.056  32.919  1.00  0.82           N  \\nATOM   2102  CA  PRO A 277      14.600  32.153  33.107  1.00  0.84           C  \\nATOM   2103  C   PRO A 277      15.309  32.093  34.458  1.00  0.85           C  \\nATOM   2104  CB  PRO A 277      13.723  33.403  33.005  1.00  0.80           C  \\nATOM   2105  O   PRO A 277      14.730  31.628  35.443  1.00  0.81           O  \\nATOM   2106  CG  PRO A 277      12.354  32.943  33.390  1.00  0.77           C  \\nATOM   2107  CD  PRO A 277      12.223  31.489  33.037  1.00  0.76           C  \\nATOM   2108  N   PRO A 278      16.605  32.188  34.450  1.00  0.84           N  \\nATOM   2109  CA  PRO A 278      17.367  32.296  35.696  1.00  0.86           C  \\nATOM   2110  C   PRO A 278      16.725  33.251  36.700  1.00  0.87           C  \\nATOM   2111  CB  PRO A 278      18.727  32.826  35.233  1.00  0.83           C  \\nATOM   2112  O   PRO A 278      16.072  34.220  36.305  1.00  0.84           O  \\nATOM   2113  CG  PRO A 278      18.439  33.561  33.964  1.00  0.80           C  \\nATOM   2114  CD  PRO A 278      17.253  32.922  33.301  1.00  0.78           C  \\nATOM   2115  N   PRO A 279      16.637  32.862  37.930  1.00  0.88           N  \\nATOM   2116  CA  PRO A 279      16.186  33.774  38.984  1.00  0.90           C  \\nATOM   2117  C   PRO A 279      16.865  35.140  38.912  1.00  0.90           C  \\nATOM   2118  CB  PRO A 279      16.569  33.041  40.272  1.00  0.87           C  \\nATOM   2119  O   PRO A 279      18.014  35.240  38.475  1.00  0.88           O  \\nATOM   2120  CG  PRO A 279      17.714  32.161  39.886  1.00  0.82           C  \\nATOM   2121  CD  PRO A 279      17.565  31.794  38.437  1.00  0.80           C  \\nATOM   2122  N   PRO A 280      16.135  36.206  39.053  1.00  0.90           N  \\nATOM   2123  CA  PRO A 280      16.718  37.545  39.161  1.00  0.91           C  \\nATOM   2124  C   PRO A 280      17.887  37.602  40.142  1.00  0.92           C  \\nATOM   2125  CB  PRO A 280      15.548  38.400  39.656  1.00  0.89           C  \\nATOM   2126  O   PRO A 280      17.919  36.843  41.114  1.00  0.89           O  \\nATOM   2127  CG  PRO A 280      14.652  37.443  40.374  1.00  0.84           C  \\nATOM   2128  CD  PRO A 280      14.809  36.085  39.752  1.00  0.80           C  \\nATOM   2129  N   PRO A 281      18.933  38.292  39.794  1.00  0.92           N  \\nATOM   2130  CA  PRO A 281      20.037  38.535  40.724  1.00  0.93           C  \\nATOM   2131  C   PRO A 281      19.562  39.040  42.085  1.00  0.93           C  \\nATOM   2132  CB  PRO A 281      20.874  39.599  40.009  1.00  0.91           C  \\nATOM   2133  O   PRO A 281      18.531  39.711  42.173  1.00  0.91           O  \\nATOM   2134  CG  PRO A 281      19.906  40.312  39.120  1.00  0.86           C  \\nATOM   2135  CD  PRO A 281      18.818  39.352  38.733  1.00  0.82           C  \\nATOM   2136  N   PRO A 282      20.208  38.590  43.100  1.00  0.93           N  \\nATOM   2137  CA  PRO A 282      19.933  39.153  44.424  1.00  0.94           C  \\nATOM   2138  C   PRO A 282      20.041  40.676  44.450  1.00  0.95           C  \\nATOM   2139  CB  PRO A 282      21.006  38.514  45.309  1.00  0.92           C  \\nATOM   2140  O   PRO A 282      20.813  41.258  43.683  1.00  0.93           O  \\nATOM   2141  CG  PRO A 282      22.127  38.189  44.376  1.00  0.88           C  \\nATOM   2142  CD  PRO A 282      21.555  37.940  43.010  1.00  0.84           C  \\nATOM   2143  N   PRO A 283      19.185  41.277  45.149  1.00  0.94           N  \\nATOM   2144  CA  PRO A 283      19.328  42.718  45.369  1.00  0.94           C  \\nATOM   2145  C   PRO A 283      20.715  43.103  45.878  1.00  0.95           C  \\nATOM   2146  CB  PRO A 283      18.257  43.022  46.420  1.00  0.93           C  \\nATOM   2147  O   PRO A 283      21.373  42.305  46.551  1.00  0.93           O  \\nATOM   2148  CG  PRO A 283      18.046  41.727  47.135  1.00  0.88           C  \\nATOM   2149  CD  PRO A 283      18.371  40.603  46.194  1.00  0.84           C  \\nATOM   2150  N   PRO A 284      21.218  44.164  45.423  1.00  0.94           N  \\nATOM   2151  CA  PRO A 284      22.464  44.696  45.980  1.00  0.94           C  \\nATOM   2152  C   PRO A 284      22.423  44.823  47.501  1.00  0.95           C  \\nATOM   2153  CB  PRO A 284      22.586  46.071  45.320  1.00  0.92           C  \\nATOM   2154  O   PRO A 284      21.352  45.022  48.079  1.00  0.93           O  \\nATOM   2155  CG  PRO A 284      21.183  46.459  44.982  1.00  0.88           C  \\nATOM   2156  CD  PRO A 284      20.382  45.208  44.762  1.00  0.84           C  \\nATOM   2157  N   PRO A 285      23.494  44.527  48.092  1.00  0.92           N  \\nATOM   2158  CA  PRO A 285      23.580  44.788  49.531  1.00  0.93           C  \\nATOM   2159  C   PRO A 285      23.203  46.223  49.894  1.00  0.93           C  \\nATOM   2160  CB  PRO A 285      25.051  44.513  49.852  1.00  0.91           C  \\nATOM   2161  O   PRO A 285      23.380  47.135  49.083  1.00  0.91           O  \\nATOM   2162  CG  PRO A 285      25.775  44.762  48.568  1.00  0.86           C  \\nATOM   2163  CD  PRO A 285      24.843  44.464  47.429  1.00  0.82           C  \\nATOM   2164  N   PRO A 286      22.567  46.400  50.908  1.00  0.89           N  \\nATOM   2165  CA  PRO A 286      22.328  47.756  51.411  1.00  0.90           C  \\nATOM   2166  C   PRO A 286      23.611  48.574  51.533  1.00  0.91           C  \\nATOM   2167  CB  PRO A 286      21.700  47.516  52.786  1.00  0.87           C  \\nATOM   2168  O   PRO A 286      24.686  48.014  51.764  1.00  0.88           O  \\nATOM   2169  CG  PRO A 286      22.197  46.170  53.203  1.00  0.83           C  \\nATOM   2170  CD  PRO A 286      22.462  45.355  51.970  1.00  0.80           C  \\nATOM   2171  N   PRO A 287      23.603  49.736  51.203  1.00  0.87           N  \\nATOM   2172  CA  PRO A 287      24.730  50.638  51.450  1.00  0.88           C  \\nATOM   2173  C   PRO A 287      25.218  50.592  52.896  1.00  0.88           C  \\nATOM   2174  CB  PRO A 287      24.159  52.017  51.110  1.00  0.85           C  \\nATOM   2175  O   PRO A 287      24.428  50.341  53.810  1.00  0.85           O  \\nATOM   2176  CG  PRO A 287      22.688  51.889  51.343  1.00  0.82           C  \\nATOM   2177  CD  PRO A 287      22.289  50.462  51.097  1.00  0.80           C  \\nATOM   2178  N   PRO A 288      26.461  50.600  53.020  1.00  0.89           N  \\nATOM   2179  CA  PRO A 288      26.997  50.740  54.376  1.00  0.90           C  \\nATOM   2180  C   PRO A 288      26.326  51.863  55.163  1.00  0.89           C  \\nATOM   2181  CB  PRO A 288      28.478  51.047  54.140  1.00  0.87           C  \\nATOM   2182  O   PRO A 288      26.190  52.980  54.657  1.00  0.87           O  \\nATOM   2183  CG  PRO A 288      28.548  51.527  52.726  1.00  0.83           C  \\nATOM   2184  CD  PRO A 288      27.324  51.049  51.999  1.00  0.81           C  \\nATOM   2185  N   GLN A 289      25.729  51.499  56.183  1.00  0.88           N  \\nATOM   2186  CA  GLN A 289      25.332  52.514  57.153  1.00  0.89           C  \\nATOM   2187  C   GLN A 289      26.508  53.416  57.517  1.00  0.88           C  \\nATOM   2188  CB  GLN A 289      24.763  51.860  58.413  1.00  0.85           C  \\nATOM   2189  O   GLN A 289      27.556  52.934  57.950  1.00  0.85           O  \\nATOM   2190  CG  GLN A 289      23.431  51.156  58.192  1.00  0.77           C  \\nATOM   2191  CD  GLN A 289      22.868  50.551  59.465  1.00  0.75           C  \\nATOM   2192  NE2 GLN A 289      21.726  49.882  59.349  1.00  0.77           N  \\nATOM   2193  OE1 GLN A 289      23.457  50.682  60.543  1.00  0.81           O  \\nATOM   2194  N   GLN A 290      26.475  54.681  57.082  1.00  0.90           N  \\nATOM   2195  CA  GLN A 290      27.328  55.721  57.646  1.00  0.91           C  \\nATOM   2196  C   GLN A 290      27.196  55.776  59.165  1.00  0.92           C  \\nATOM   2197  CB  GLN A 290      26.988  57.083  57.040  1.00  0.87           C  \\nATOM   2198  O   GLN A 290      26.084  55.771  59.698  1.00  0.89           O  \\nATOM   2199  CG  GLN A 290      27.557  57.294  55.643  1.00  0.80           C  \\nATOM   2200  CD  GLN A 290      27.270  58.679  55.095  1.00  0.77           C  \\nATOM   2201  NE2 GLN A 290      28.266  59.282  54.454  1.00  0.76           N  \\nATOM   2202  OE1 GLN A 290      26.162  59.204  55.246  1.00  0.81           O  \\nATOM   2203  N   PRO A 291      28.282  55.614  59.717  1.00  0.89           N  \\nATOM   2204  CA  PRO A 291      28.269  56.028  61.122  1.00  0.90           C  \\nATOM   2205  C   PRO A 291      27.947  57.510  61.297  1.00  0.89           C  \\nATOM   2206  CB  PRO A 291      29.691  55.717  61.594  1.00  0.87           C  \\nATOM   2207  O   PRO A 291      28.375  58.338  60.488  1.00  0.85           O  \\nATOM   2208  CG  PRO A 291      30.536  55.838  60.367  1.00  0.85           C  \\nATOM   2209  CD  PRO A 291      29.688  55.517  59.171  1.00  0.84           C  \\nATOM   2210  N   TRP A 292      27.182  57.815  62.156  1.00  0.88           N  \\nATOM   2211  CA  TRP A 292      26.880  59.166  62.617  1.00  0.90           C  \\nATOM   2212  C   TRP A 292      28.122  59.832  63.201  1.00  0.84           C  \\nATOM   2213  CB  TRP A 292      25.760  59.141  63.661  1.00  0.85           C  \\nATOM   2214  O   TRP A 292      28.932  59.178  63.862  1.00  0.76           O  \\nATOM   2215  CG  TRP A 292      24.473  58.558  63.159  1.00  0.70           C  \\nATOM   2216  CD1 TRP A 292      23.982  57.307  63.411  1.00  0.68           C  \\nATOM   2217  CD2 TRP A 292      23.515  59.203  62.315  1.00  0.73           C  \\nATOM   2218  CE2 TRP A 292      22.466  58.283  62.095  1.00  0.76           C  \\nATOM   2219  CE3 TRP A 292      23.441  60.471  61.723  1.00  0.73           C  \\nATOM   2220  NE1 TRP A 292      22.775  57.135  62.774  1.00  0.81           N  \\nATOM   2221  CH2 TRP A 292      21.307  59.839  60.740  1.00  0.77           C  \\nATOM   2222  CZ2 TRP A 292      21.355  58.592  61.307  1.00  0.79           C  \\nATOM   2223  CZ3 TRP A 292      22.334  60.777  60.939  1.00  0.77           C  \",\"pdb\");\n\tviewer_17249326855875523.setStyle({\"resi\": 1},{\"cartoon\": {\"color\": \"#4900b6\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 2},{\"cartoon\": {\"color\": \"#c80037\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 3},{\"cartoon\": {\"color\": \"#a80057\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 4},{\"cartoon\": {\"color\": \"#8f0070\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 5},{\"cartoon\": {\"color\": \"#7d0082\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 6},{\"cartoon\": {\"color\": \"#70008f\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 7},{\"cartoon\": {\"color\": \"#9f0060\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 8},{\"cartoon\": {\"color\": \"#bc0043\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 9},{\"cartoon\": {\"color\": \"#cd0032\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 10},{\"cartoon\": {\"color\": \"#c4003b\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 11},{\"cartoon\": {\"color\": \"#d1002e\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 12},{\"cartoon\": {\"color\": \"#db0024\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 13},{\"cartoon\": {\"color\": \"#e3001c\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 14},{\"cartoon\": {\"color\": \"#e80017\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 15},{\"cartoon\": {\"color\": \"#ec0013\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 16},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 17},{\"cartoon\": {\"color\": \"#f3000c\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 18},{\"cartoon\": {\"color\": \"#ec0013\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 19},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 20},{\"cartoon\": {\"color\": \"#fa0005\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 21},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 22},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 23},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 24},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 25},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 26},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 27},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 28},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 29},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 30},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 31},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 32},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 33},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 34},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 35},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 36},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 37},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 38},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 39},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 40},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 41},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 42},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 43},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 44},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 45},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 46},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 47},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 48},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 49},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 50},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 51},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 52},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 53},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 54},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 55},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 56},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 57},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 58},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 59},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 60},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 61},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 62},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 63},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 64},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 65},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 66},{\"cartoon\": {\"color\": \"#fb0004\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 67},{\"cartoon\": {\"color\": \"#f80007\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 68},{\"cartoon\": {\"color\": \"#f80007\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 69},{\"cartoon\": {\"color\": \"#f2000d\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 70},{\"cartoon\": {\"color\": \"#eb0014\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 71},{\"cartoon\": {\"color\": \"#eb0014\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 72},{\"cartoon\": {\"color\": \"#f3000c\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 73},{\"cartoon\": {\"color\": \"#f3000c\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 74},{\"cartoon\": {\"color\": \"#f3000c\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 75},{\"cartoon\": {\"color\": \"#f3000c\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 76},{\"cartoon\": {\"color\": \"#f0000f\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 77},{\"cartoon\": {\"color\": \"#ef0010\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 78},{\"cartoon\": {\"color\": \"#f60009\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 79},{\"cartoon\": {\"color\": \"#fb0004\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 80},{\"cartoon\": {\"color\": \"#fb0004\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 81},{\"cartoon\": {\"color\": \"#fb0004\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 82},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 83},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 84},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 85},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 86},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 87},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 88},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 89},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 90},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 91},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 92},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 93},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 94},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 95},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 96},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 97},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 98},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 99},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 100},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 101},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 102},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 103},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 104},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 105},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 106},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 107},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 108},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 109},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 110},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 111},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 112},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 113},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 114},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 115},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 116},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 117},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 118},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 119},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 120},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 121},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 122},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 123},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 124},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 125},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 126},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 127},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 128},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 129},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 130},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 131},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 132},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 133},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 134},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 135},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 136},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 137},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 138},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 139},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 140},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 141},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 142},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 143},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 144},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 145},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 146},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 147},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 148},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 149},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 150},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 151},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 152},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 153},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 154},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 155},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 156},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 157},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 158},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 159},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 160},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 161},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 162},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 163},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 164},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 165},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 166},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 167},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 168},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 169},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 170},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 171},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 172},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 173},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 174},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 175},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 176},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 177},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 178},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 179},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 180},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 181},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 182},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 183},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 184},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 185},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 186},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 187},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 188},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 189},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 190},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 191},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 192},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 193},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 194},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 195},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 196},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 197},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 198},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 199},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 200},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 201},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 202},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 203},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 204},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 205},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 206},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 207},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 208},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 209},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 210},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 211},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 212},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 213},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 214},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 215},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 216},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 217},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 218},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 219},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 220},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 221},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 222},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 223},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 224},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 225},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 226},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 227},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 228},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 229},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 230},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 231},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 232},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 233},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 234},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 235},{\"cartoon\": {\"color\": \"#ec0013\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 236},{\"cartoon\": {\"color\": \"#e1001e\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 237},{\"cartoon\": {\"color\": \"#d5002a\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 238},{\"cartoon\": {\"color\": \"#c5003a\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 239},{\"cartoon\": {\"color\": \"#b1004e\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 240},{\"cartoon\": {\"color\": \"#9a0065\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 241},{\"cartoon\": {\"color\": \"#7e0081\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 242},{\"cartoon\": {\"color\": \"#5e00a1\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 243},{\"cartoon\": {\"color\": \"#3a00c5\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 244},{\"cartoon\": {\"color\": \"#1000ef\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 245},{\"cartoon\": {\"color\": \"#1200ed\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 246},{\"cartoon\": {\"color\": \"#1400eb\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 247},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 248},{\"cartoon\": {\"color\": \"#0e00f1\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 249},{\"cartoon\": {\"color\": \"#1000ef\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 250},{\"cartoon\": {\"color\": \"#1100ee\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 251},{\"cartoon\": {\"color\": \"#1100ee\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 252},{\"cartoon\": {\"color\": \"#0f00f0\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 253},{\"cartoon\": {\"color\": \"#0e00f1\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 254},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 255},{\"cartoon\": {\"color\": \"#0c00f3\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 256},{\"cartoon\": {\"color\": \"#0b00f4\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 257},{\"cartoon\": {\"color\": \"#0b00f4\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 258},{\"cartoon\": {\"color\": \"#3700c8\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 259},{\"cartoon\": {\"color\": \"#5d00a2\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 260},{\"cartoon\": {\"color\": \"#7d0082\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 261},{\"cartoon\": {\"color\": \"#980067\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 262},{\"cartoon\": {\"color\": \"#af0050\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 263},{\"cartoon\": {\"color\": \"#c3003c\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 264},{\"cartoon\": {\"color\": \"#d3002c\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 265},{\"cartoon\": {\"color\": \"#e0001f\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 266},{\"cartoon\": {\"color\": \"#eb0014\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 267},{\"cartoon\": {\"color\": \"#f3000c\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 268},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 269},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 270},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 271},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 272},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 273},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 274},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 275},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 276},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 277},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 278},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 279},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 280},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 281},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 282},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 283},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 284},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 285},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 286},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 287},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 288},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 289},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.setStyle({\"resi\": 290},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17249326855875523.zoomTo();\nviewer_17249326855875523.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_17249326855875523\"  style=\"position: relative; width: 1000px; height: 800px;\">\n",
              "        <p id=\"3dmolwarning_17249326855875523\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.3.0/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_17249326855875523 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_17249326855875523\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_17249326855875523 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17249326855875523\"),{backgroundColor:\"white\"});\n",
              "viewer_17249326855875523.zoomTo();\n",
              "\tviewer_17249326855875523.addModel(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   MET A   1      23.752   3.572 -41.536  1.00  0.41           N  \\nATOM      2  CA  MET A   1      22.656   4.343 -42.116  1.00  0.51           C  \\nATOM      3  C   MET A   1      21.409   3.480 -42.276  1.00  0.42           C  \\nATOM      4  CB  MET A   1      23.065   4.928 -43.468  1.00  0.40           C  \\nATOM      5  O   MET A   1      21.316   2.683 -43.212  1.00  0.42           O  \\nATOM      6  CG  MET A   1      24.094   6.042 -43.369  1.00  0.42           C  \\nATOM      7  SD  MET A   1      24.535   6.741 -45.007  1.00  0.41           S  \\nATOM      8  CE  MET A   1      25.852   7.890 -44.521  1.00  0.39           C  \\nATOM      9  N   ALA A   2      20.842   2.859 -41.170  1.00  0.52           N  \\nATOM     10  CA  ALA A   2      19.464   2.619 -40.750  1.00  0.54           C  \\nATOM     11  C   ALA A   2      19.271   2.967 -39.277  1.00  0.53           C  \\nATOM     12  CB  ALA A   2      19.076   1.164 -41.005  1.00  0.47           C  \\nATOM     13  O   ALA A   2      19.908   2.374 -38.404  1.00  0.50           O  \\nATOM     14  N   SER A   3      19.334   4.290 -38.985  1.00  0.49           N  \\nATOM     15  CA  SER A   3      18.579   4.884 -37.886  1.00  0.51           C  \\nATOM     16  C   SER A   3      17.235   4.188 -37.700  1.00  0.49           C  \\nATOM     17  CB  SER A   3      18.360   6.377 -38.131  1.00  0.45           C  \\nATOM     18  O   SER A   3      16.390   4.208 -38.598  1.00  0.47           O  \\nATOM     19  OG  SER A   3      18.869   7.143 -37.052  1.00  0.49           O  \\nATOM     20  N   ASN A   4      17.145   2.994 -37.422  1.00  0.52           N  \\nATOM     21  CA  ASN A   4      15.959   2.278 -36.964  1.00  0.56           C  \\nATOM     22  C   ASN A   4      15.155   3.106 -35.965  1.00  0.52           C  \\nATOM     23  CB  ASN A   4      16.348   0.933 -36.347  1.00  0.47           C  \\nATOM     24  O   ASN A   4      15.616   3.362 -34.852  1.00  0.49           O  \\nATOM     25  CG  ASN A   4      16.044  -0.238 -37.260  1.00  0.45           C  \\nATOM     26  ND2 ASN A   4      16.374  -1.443 -36.810  1.00  0.42           N  \\nATOM     27  OD1 ASN A   4      15.517  -0.062 -38.362  1.00  0.46           O  \\nATOM     28  N   ASP A   5      14.617   4.301 -36.512  1.00  0.46           N  \\nATOM     29  CA  ASP A   5      13.463   5.136 -36.191  1.00  0.48           C  \\nATOM     30  C   ASP A   5      12.356   4.314 -35.534  1.00  0.46           C  \\nATOM     31  CB  ASP A   5      12.930   5.823 -37.450  1.00  0.42           C  \\nATOM     32  O   ASP A   5      11.548   3.688 -36.223  1.00  0.44           O  \\nATOM     33  CG  ASP A   5      13.313   7.290 -37.531  1.00  0.41           C  \\nATOM     34  OD1 ASP A   5      13.719   7.873 -36.503  1.00  0.45           O  \\nATOM     35  OD2 ASP A   5      13.205   7.870 -38.633  1.00  0.50           O  \\nATOM     36  N   TYR A   6      12.669   3.370 -34.615  1.00  0.51           N  \\nATOM     37  CA  TYR A   6      11.709   2.589 -33.843  1.00  0.54           C  \\nATOM     38  C   TYR A   6      11.235   3.364 -32.619  1.00  0.50           C  \\nATOM     39  CB  TYR A   6      12.325   1.256 -33.410  1.00  0.47           C  \\nATOM     40  O   TYR A   6      10.373   2.893 -31.874  1.00  0.48           O  \\nATOM     41  CG  TYR A   6      12.326   0.207 -34.495  1.00  0.46           C  \\nATOM     42  CD1 TYR A   6      11.196  -0.569 -34.743  1.00  0.43           C  \\nATOM     43  CD2 TYR A   6      13.457  -0.011 -35.274  1.00  0.45           C  \\nATOM     44  CE1 TYR A   6      11.194  -1.538 -35.740  1.00  0.43           C  \\nATOM     45  CE2 TYR A   6      13.466  -0.977 -36.275  1.00  0.46           C  \\nATOM     46  OH  TYR A   6      12.334  -2.692 -37.490  1.00  0.33           O  \\nATOM     47  CZ  TYR A   6      12.331  -1.734 -36.500  1.00  0.37           C  \\nATOM     48  N   THR A   7      11.370   4.658 -32.630  1.00  0.45           N  \\nATOM     49  CA  THR A   7      10.948   5.584 -31.584  1.00  0.46           C  \\nATOM     50  C   THR A   7       9.886   6.545 -32.110  1.00  0.44           C  \\nATOM     51  CB  THR A   7      12.142   6.385 -31.033  1.00  0.41           C  \\nATOM     52  O   THR A   7       9.059   7.044 -31.343  1.00  0.42           O  \\nATOM     53  CG2 THR A   7      12.247   6.243 -29.518  1.00  0.39           C  \\nATOM     54  OG1 THR A   7      13.351   5.903 -31.632  1.00  0.38           O  \\nATOM     55  N   GLN A   8       9.220   6.333 -33.334  1.00  0.42           N  \\nATOM     56  CA  GLN A   8       8.232   7.287 -33.824  1.00  0.43           C  \\nATOM     57  C   GLN A   8       7.009   6.571 -34.392  1.00  0.42           C  \\nATOM     58  CB  GLN A   8       8.846   8.199 -34.887  1.00  0.38           C  \\nATOM     59  O   GLN A   8       5.927   7.154 -34.480  1.00  0.39           O  \\nATOM     60  CG  GLN A   8       9.431   9.489 -34.328  1.00  0.38           C  \\nATOM     61  CD  GLN A   8       9.987  10.398 -35.408  1.00  0.45           C  \\nATOM     62  NE2 GLN A   8      10.571  11.518 -34.997  1.00  0.35           N  \\nATOM     63  OE1 GLN A   8       9.895  10.094 -36.602  1.00  0.47           O  \\nATOM     64  N   GLN A   9       6.645   5.288 -33.989  1.00  0.38           N  \\nATOM     65  CA  GLN A   9       5.350   4.796 -34.448  1.00  0.41           C  \\nATOM     66  C   GLN A   9       4.701   3.896 -33.400  1.00  0.38           C  \\nATOM     67  CB  GLN A   9       5.499   4.040 -35.769  1.00  0.35           C  \\nATOM     68  O   GLN A   9       3.544   3.495 -33.549  1.00  0.36           O  \\nATOM     69  CG  GLN A   9       5.371   4.925 -37.001  1.00  0.35           C  \\nATOM     70  CD  GLN A   9       5.424   4.138 -38.297  1.00  0.44           C  \\nATOM     71  NE2 GLN A   9       5.320   4.839 -39.421  1.00  0.34           N  \\nATOM     72  OE1 GLN A   9       5.557   2.910 -38.288  1.00  0.45           O  \\nATOM     73  N   ALA A  10       4.979   4.125 -32.081  1.00  0.39           N  \\nATOM     74  CA  ALA A  10       4.090   3.523 -31.091  1.00  0.40           C  \\nATOM     75  C   ALA A  10       3.500   4.584 -30.166  1.00  0.39           C  \\nATOM     76  CB  ALA A  10       4.837   2.468 -30.278  1.00  0.36           C  \\nATOM     77  O   ALA A  10       2.874   4.256 -29.155  1.00  0.37           O  \\nATOM     78  N   THR A  11       3.317   5.785 -30.721  1.00  0.39           N  \\nATOM     79  CA  THR A  11       2.437   6.881 -30.331  1.00  0.41           C  \\nATOM     80  C   THR A  11       1.463   7.220 -31.456  1.00  0.39           C  \\nATOM     81  CB  THR A  11       3.243   8.136 -29.950  1.00  0.36           C  \\nATOM     82  O   THR A  11       1.804   7.969 -32.374  1.00  0.37           O  \\nATOM     83  CG2 THR A  11       2.746   8.732 -28.637  1.00  0.36           C  \\nATOM     84  OG1 THR A  11       4.625   7.785 -29.810  1.00  0.33           O  \\nATOM     85  N   GLN A  12       0.909   6.243 -32.205  1.00  0.36           N  \\nATOM     86  CA  GLN A  12      -0.147   6.670 -33.117  1.00  0.39           C  \\nATOM     87  C   GLN A  12      -1.514   6.190 -32.639  1.00  0.36           C  \\nATOM     88  CB  GLN A  12       0.125   6.156 -34.532  1.00  0.33           C  \\nATOM     89  O   GLN A  12      -1.705   5.000 -32.381  1.00  0.34           O  \\nATOM     90  CG  GLN A  12       1.091   7.023 -35.327  1.00  0.33           C  \\nATOM     91  CD  GLN A  12       1.263   6.550 -36.758  1.00  0.41           C  \\nATOM     92  NE2 GLN A  12       2.064   7.277 -37.530  1.00  0.31           N  \\nATOM     93  OE1 GLN A  12       0.682   5.540 -37.166  1.00  0.42           O  \\nATOM     94  N   SER A  13      -2.103   6.912 -31.568  1.00  0.33           N  \\nATOM     95  CA  SER A  13      -3.487   7.354 -31.438  1.00  0.37           C  \\nATOM     96  C   SER A  13      -4.327   6.328 -30.684  1.00  0.34           C  \\nATOM     97  CB  SER A  13      -4.100   7.613 -32.815  1.00  0.31           C  \\nATOM     98  O   SER A  13      -4.540   5.215 -31.170  1.00  0.32           O  \\nATOM     99  OG  SER A  13      -3.796   6.555 -33.708  1.00  0.32           O  \\nATOM    100  N   TYR A  14      -4.104   6.052 -29.391  1.00  0.39           N  \\nATOM    101  CA  TYR A  14      -5.405   5.993 -28.735  1.00  0.41           C  \\nATOM    102  C   TYR A  14      -6.428   6.846 -29.474  1.00  0.39           C  \\nATOM    103  CB  TYR A  14      -5.294   6.454 -27.279  1.00  0.37           C  \\nATOM    104  O   TYR A  14      -6.413   8.075 -29.368  1.00  0.38           O  \\nATOM    105  CG  TYR A  14      -4.689   5.422 -26.359  1.00  0.37           C  \\nATOM    106  CD1 TYR A  14      -5.479   4.437 -25.770  1.00  0.35           C  \\nATOM    107  CD2 TYR A  14      -3.327   5.429 -26.076  1.00  0.37           C  \\nATOM    108  CE1 TYR A  14      -4.926   3.485 -24.920  1.00  0.36           C  \\nATOM    109  CE2 TYR A  14      -2.764   4.482 -25.228  1.00  0.40           C  \\nATOM    110  OH  TYR A  14      -3.017   2.575 -23.815  1.00  0.26           O  \\nATOM    111  CZ  TYR A  14      -3.569   3.515 -24.656  1.00  0.32           C  \\nATOM    112  N   GLY A  15      -6.563   6.790 -30.734  1.00  0.33           N  \\nATOM    113  CA  GLY A  15      -7.605   7.343 -31.585  1.00  0.36           C  \\nATOM    114  C   GLY A  15      -8.995   6.856 -31.219  1.00  0.34           C  \\nATOM    115  O   GLY A  15      -9.212   5.655 -31.047  1.00  0.32           O  \\nATOM    116  N   ALA A  16      -9.523   7.556 -30.118  1.00  0.34           N  \\nATOM    117  CA  ALA A  16     -10.950   7.860 -30.064  1.00  0.36           C  \\nATOM    118  C   ALA A  16     -11.785   6.583 -30.096  1.00  0.35           C  \\nATOM    119  CB  ALA A  16     -11.343   8.778 -31.219  1.00  0.32           C  \\nATOM    120  O   ALA A  16     -11.810   5.876 -31.106  1.00  0.33           O  \\nATOM    121  N   TYR A  17     -11.623   5.630 -29.177  1.00  0.42           N  \\nATOM    122  CA  TYR A  17     -12.884   4.911 -29.034  1.00  0.44           C  \\nATOM    123  C   TYR A  17     -14.052   5.753 -29.533  1.00  0.42           C  \\nATOM    124  CB  TYR A  17     -13.114   4.514 -27.573  1.00  0.40           C  \\nATOM    125  O   TYR A  17     -14.176   6.927 -29.177  1.00  0.41           O  \\nATOM    126  CG  TYR A  17     -12.215   3.398 -27.098  1.00  0.40           C  \\nATOM    127  CD1 TYR A  17     -12.535   2.065 -27.348  1.00  0.38           C  \\nATOM    128  CD2 TYR A  17     -11.044   3.674 -26.399  1.00  0.39           C  \\nATOM    129  CE1 TYR A  17     -11.710   1.034 -26.911  1.00  0.39           C  \\nATOM    130  CE2 TYR A  17     -10.212   2.650 -25.957  1.00  0.42           C  \\nATOM    131  OH  TYR A  17      -9.733   0.318 -25.783  1.00  0.31           O  \\nATOM    132  CZ  TYR A  17     -10.553   1.335 -26.218  1.00  0.35           C  \\nATOM    133  N   PRO A  18     -14.488   5.596 -30.788  1.00  0.38           N  \\nATOM    134  CA  PRO A  18     -15.769   6.200 -31.162  1.00  0.40           C  \\nATOM    135  C   PRO A  18     -16.716   6.359 -29.974  1.00  0.39           C  \\nATOM    136  CB  PRO A  18     -16.339   5.214 -32.185  1.00  0.36           C  \\nATOM    137  O   PRO A  18     -17.064   5.372 -29.321  1.00  0.37           O  \\nATOM    138  CG  PRO A  18     -15.712   3.901 -31.841  1.00  0.36           C  \\nATOM    139  CD  PRO A  18     -14.423   4.164 -31.116  1.00  0.41           C  \\nATOM    140  N   THR A  19     -16.547   7.364 -28.978  1.00  0.37           N  \\nATOM    141  CA  THR A  19     -17.746   7.875 -28.324  1.00  0.38           C  \\nATOM    142  C   THR A  19     -18.988   7.570 -29.156  1.00  0.38           C  \\nATOM    143  CB  THR A  19     -17.647   9.393 -28.084  1.00  0.36           C  \\nATOM    144  O   THR A  19     -19.086   7.987 -30.312  1.00  0.36           O  \\nATOM    145  CG2 THR A  19     -17.876   9.734 -26.615  1.00  0.35           C  \\nATOM    146  OG1 THR A  19     -16.344   9.848 -28.471  1.00  0.33           O  \\nATOM    147  N   GLN A  20     -19.432   6.414 -29.142  1.00  0.44           N  \\nATOM    148  CA  GLN A  20     -20.834   6.301 -29.528  1.00  0.47           C  \\nATOM    149  C   GLN A  20     -21.584   7.604 -29.266  1.00  0.45           C  \\nATOM    150  CB  GLN A  20     -21.506   5.149 -28.778  1.00  0.42           C  \\nATOM    151  O   GLN A  20     -21.412   8.225 -28.215  1.00  0.43           O  \\nATOM    152  CG  GLN A  20     -21.338   3.795 -29.453  1.00  0.42           C  \\nATOM    153  CD  GLN A  20     -22.219   2.721 -28.842  1.00  0.47           C  \\nATOM    154  NE2 GLN A  20     -22.409   1.625 -29.568  1.00  0.40           N  \\nATOM    155  OE1 GLN A  20     -22.725   2.876 -27.726  1.00  0.51           O  \\nATOM    156  N   PRO A  21     -21.808   8.485 -30.228  1.00  0.41           N  \\nATOM    157  CA  PRO A  21     -22.781   9.572 -30.096  1.00  0.43           C  \\nATOM    158  C   PRO A  21     -23.903   9.243 -29.114  1.00  0.41           C  \\nATOM    159  CB  PRO A  21     -23.328   9.731 -31.517  1.00  0.39           C  \\nATOM    160  O   PRO A  21     -24.713   8.350 -29.374  1.00  0.40           O  \\nATOM    161  CG  PRO A  21     -22.751   8.582 -32.278  1.00  0.41           C  \\nATOM    162  CD  PRO A  21     -21.798   7.845 -31.382  1.00  0.47           C  \\nATOM    163  N   GLY A  22     -23.555   9.317 -27.787  1.00  0.39           N  \\nATOM    164  CA  GLY A  22     -24.570   9.422 -26.751  1.00  0.42           C  \\nATOM    165  C   GLY A  22     -24.145   8.800 -25.434  1.00  0.40           C  \\nATOM    166  O   GLY A  22     -24.929   8.751 -24.484  1.00  0.37           O  \\nATOM    167  N   GLN A  23     -22.798   8.690 -25.122  1.00  0.40           N  \\nATOM    168  CA  GLN A  23     -22.318   8.457 -23.764  1.00  0.41           C  \\nATOM    169  C   GLN A  23     -21.023   9.221 -23.499  1.00  0.40           C  \\nATOM    170  CB  GLN A  23     -22.105   6.962 -23.518  1.00  0.37           C  \\nATOM    171  O   GLN A  23     -19.997   8.948 -24.125  1.00  0.39           O  \\nATOM    172  CG  GLN A  23     -23.378   6.212 -23.152  1.00  0.37           C  \\nATOM    173  CD  GLN A  23     -23.113   4.783 -22.716  1.00  0.43           C  \\nATOM    174  NE2 GLN A  23     -24.167   4.080 -22.314  1.00  0.36           N  \\nATOM    175  OE1 GLN A  23     -21.971   4.314 -22.741  1.00  0.48           O  \\nATOM    176  N   GLY A  24     -21.081  10.477 -23.333  1.00  0.40           N  \\nATOM    177  CA  GLY A  24     -20.282  11.480 -22.647  1.00  0.41           C  \\nATOM    178  C   GLY A  24     -19.443  10.906 -21.520  1.00  0.40           C  \\nATOM    179  O   GLY A  24     -19.952  10.168 -20.674  1.00  0.38           O  \\nATOM    180  N   TYR A  25     -18.326  10.342 -21.711  1.00  0.42           N  \\nATOM    181  CA  TYR A  25     -17.347  10.220 -20.637  1.00  0.44           C  \\nATOM    182  C   TYR A  25     -17.659  11.188 -19.502  1.00  0.42           C  \\nATOM    183  CB  TYR A  25     -15.933  10.475 -21.167  1.00  0.40           C  \\nATOM    184  O   TYR A  25     -17.474  12.399 -19.642  1.00  0.41           O  \\nATOM    185  CG  TYR A  25     -15.364   9.322 -21.957  1.00  0.40           C  \\nATOM    186  CD1 TYR A  25     -14.742   8.252 -21.316  1.00  0.38           C  \\nATOM    187  CD2 TYR A  25     -15.445   9.300 -23.345  1.00  0.39           C  \\nATOM    188  CE1 TYR A  25     -14.214   7.188 -22.040  1.00  0.37           C  \\nATOM    189  CE2 TYR A  25     -14.920   8.241 -24.079  1.00  0.42           C  \\nATOM    190  OH  TYR A  25     -13.788   6.141 -24.141  1.00  0.27           O  \\nATOM    191  CZ  TYR A  25     -14.308   7.191 -23.419  1.00  0.34           C  \\nATOM    192  N   SER A  26     -18.642  11.047 -18.725  1.00  0.39           N  \\nATOM    193  CA  SER A  26     -18.791  11.609 -17.387  1.00  0.41           C  \\nATOM    194  C   SER A  26     -17.579  11.293 -16.517  1.00  0.39           C  \\nATOM    195  CB  SER A  26     -20.060  11.078 -16.721  1.00  0.37           C  \\nATOM    196  O   SER A  26     -17.203  10.129 -16.366  1.00  0.38           O  \\nATOM    197  OG  SER A  26     -20.668  10.078 -17.520  1.00  0.35           O  \\nATOM    198  N   GLN A  27     -16.404  11.914 -16.658  1.00  0.42           N  \\nATOM    199  CA  GLN A  27     -15.580  12.078 -15.465  1.00  0.43           C  \\nATOM    200  C   GLN A  27     -16.203  11.370 -14.265  1.00  0.42           C  \\nATOM    201  CB  GLN A  27     -15.376  13.561 -15.152  1.00  0.39           C  \\nATOM    202  O   GLN A  27     -17.242  11.797 -13.756  1.00  0.39           O  \\nATOM    203  CG  GLN A  27     -14.204  14.190 -15.893  1.00  0.38           C  \\nATOM    204  CD  GLN A  27     -14.016  15.658 -15.557  1.00  0.43           C  \\nATOM    205  NE2 GLN A  27     -12.992  16.273 -16.138  1.00  0.36           N  \\nATOM    206  OE1 GLN A  27     -14.786  16.234 -14.781  1.00  0.48           O  \\nATOM    207  N   GLN A  28     -16.396  10.119 -14.200  1.00  0.36           N  \\nATOM    208  CA  GLN A  28     -16.601   9.405 -12.944  1.00  0.39           C  \\nATOM    209  C   GLN A  28     -15.650   9.909 -11.863  1.00  0.37           C  \\nATOM    210  CB  GLN A  28     -16.417   7.900 -13.145  1.00  0.35           C  \\nATOM    211  O   GLN A  28     -14.430   9.807 -12.005  1.00  0.35           O  \\nATOM    212  CG  GLN A  28     -17.722   7.115 -13.146  1.00  0.34           C  \\nATOM    213  CD  GLN A  28     -17.520   5.641 -13.440  1.00  0.42           C  \\nATOM    214  NE2 GLN A  28     -18.619   4.899 -13.537  1.00  0.34           N  \\nATOM    215  OE1 GLN A  28     -16.386   5.171 -13.579  1.00  0.44           O  \\nATOM    216  N   SER A  29     -15.814  11.041 -11.268  1.00  0.34           N  \\nATOM    217  CA  SER A  29     -15.737  11.546  -9.901  1.00  0.36           C  \\nATOM    218  C   SER A  29     -15.379  10.433  -8.921  1.00  0.34           C  \\nATOM    219  CB  SER A  29     -17.061  12.191  -9.491  1.00  0.32           C  \\nATOM    220  O   SER A  29     -15.979   9.357  -8.949  1.00  0.32           O  \\nATOM    221  OG  SER A  29     -17.942  11.225  -8.942  1.00  0.32           O  \\nATOM    222  N   SER A  30     -14.009  10.021  -8.765  1.00  0.36           N  \\nATOM    223  CA  SER A  30     -13.524   9.561  -7.468  1.00  0.38           C  \\nATOM    224  C   SER A  30     -14.675   9.098  -6.581  1.00  0.36           C  \\nATOM    225  CB  SER A  30     -12.741  10.669  -6.763  1.00  0.34           C  \\nATOM    226  O   SER A  30     -15.326   9.913  -5.924  1.00  0.35           O  \\nATOM    227  OG  SER A  30     -13.522  11.847  -6.656  1.00  0.36           O  \\nATOM    228  N   GLN A  31     -15.541   8.380  -7.018  1.00  0.37           N  \\nATOM    229  CA  GLN A  31     -16.294   7.593  -6.046  1.00  0.41           C  \\nATOM    230  C   GLN A  31     -15.370   6.685  -5.240  1.00  0.38           C  \\nATOM    231  CB  GLN A  31     -17.369   6.760  -6.746  1.00  0.36           C  \\nATOM    232  O   GLN A  31     -14.481   6.040  -5.800  1.00  0.37           O  \\nATOM    233  CG  GLN A  31     -18.569   7.573  -7.213  1.00  0.37           C  \\nATOM    234  CD  GLN A  31     -19.880   6.823  -7.064  1.00  0.45           C  \\nATOM    235  NE2 GLN A  31     -20.954   7.555  -6.788  1.00  0.36           N  \\nATOM    236  OE1 GLN A  31     -19.927   5.596  -7.197  1.00  0.48           O  \\nATOM    237  N   PRO A  32     -14.884   6.989  -4.080  1.00  0.42           N  \\nATOM    238  CA  PRO A  32     -14.368   5.960  -3.174  1.00  0.41           C  \\nATOM    239  C   PRO A  32     -14.927   4.572  -3.479  1.00  0.42           C  \\nATOM    240  CB  PRO A  32     -14.826   6.442  -1.795  1.00  0.40           C  \\nATOM    241  O   PRO A  32     -16.109   4.436  -3.805  1.00  0.40           O  \\nATOM    242  CG  PRO A  32     -15.993   7.334  -2.071  1.00  0.39           C  \\nATOM    243  CD  PRO A  32     -15.911   7.801  -3.495  1.00  0.43           C  \\nATOM    244  N   TYR A  33     -14.640   3.788  -4.556  1.00  0.40           N  \\nATOM    245  CA  TYR A  33     -15.226   2.521  -4.980  1.00  0.42           C  \\nATOM    246  C   TYR A  33     -16.747   2.609  -5.017  1.00  0.40           C  \\nATOM    247  CB  TYR A  33     -14.792   1.388  -4.045  1.00  0.38           C  \\nATOM    248  O   TYR A  33     -17.390   2.793  -3.980  1.00  0.38           O  \\nATOM    249  CG  TYR A  33     -13.366   0.940  -4.256  1.00  0.37           C  \\nATOM    250  CD1 TYR A  33     -13.045   0.015  -5.247  1.00  0.37           C  \\nATOM    251  CD2 TYR A  33     -12.337   1.441  -3.465  1.00  0.38           C  \\nATOM    252  CE1 TYR A  33     -11.732  -0.402  -5.443  1.00  0.38           C  \\nATOM    253  CE2 TYR A  33     -11.021   1.031  -3.653  1.00  0.40           C  \\nATOM    254  OH  TYR A  33      -9.428  -0.298  -4.833  1.00  0.27           O  \\nATOM    255  CZ  TYR A  33     -10.729   0.111  -4.643  1.00  0.35           C  \\nATOM    256  N   GLY A  34     -17.383   3.354  -5.722  1.00  0.33           N  \\nATOM    257  CA  GLY A  34     -18.770   3.154  -6.110  1.00  0.34           C  \\nATOM    258  C   GLY A  34     -19.201   1.701  -6.055  1.00  0.33           C  \\nATOM    259  O   GLY A  34     -18.511   0.823  -6.577  1.00  0.32           O  \\nATOM    260  N   GLN A  35     -19.332   1.030  -4.828  1.00  0.41           N  \\nATOM    261  CA  GLN A  35     -20.401   0.055  -5.015  1.00  0.42           C  \\nATOM    262  C   GLN A  35     -20.787  -0.063  -6.486  1.00  0.41           C  \\nATOM    263  CB  GLN A  35     -21.626   0.434  -4.180  1.00  0.38           C  \\nATOM    264  O   GLN A  35     -21.485   0.800  -7.022  1.00  0.39           O  \\nATOM    265  CG  GLN A  35     -21.605  -0.133  -2.766  1.00  0.37           C  \\nATOM    266  CD  GLN A  35     -22.792   0.318  -1.936  1.00  0.43           C  \\nATOM    267  NE2 GLN A  35     -23.007  -0.340  -0.802  1.00  0.36           N  \\nATOM    268  OE1 GLN A  35     -23.510   1.251  -2.311  1.00  0.47           O  \\nATOM    269  N   GLN A  36     -19.905  -0.360  -7.366  1.00  0.38           N  \\nATOM    270  CA  GLN A  36     -20.645  -1.153  -8.341  1.00  0.41           C  \\nATOM    271  C   GLN A  36     -21.245  -2.399  -7.695  1.00  0.38           C  \\nATOM    272  CB  GLN A  36     -19.741  -1.551  -9.509  1.00  0.36           C  \\nATOM    273  O   GLN A  36     -20.518  -3.233  -7.151  1.00  0.36           O  \\nATOM    274  CG  GLN A  36     -19.568  -0.457 -10.553  1.00  0.35           C  \\nATOM    275  CD  GLN A  36     -18.797  -0.927 -11.773  1.00  0.41           C  \\nATOM    276  NE2 GLN A  36     -18.636  -0.040 -12.750  1.00  0.35           N  \\nATOM    277  OE1 GLN A  36     -18.350  -2.076 -11.837  1.00  0.45           O  \\nATOM    278  N   SER A  37     -22.234  -2.317  -6.825  1.00  0.35           N  \\nATOM    279  CA  SER A  37     -23.228  -3.385  -6.841  1.00  0.38           C  \\nATOM    280  C   SER A  37     -23.125  -4.217  -8.115  1.00  0.36           C  \\nATOM    281  CB  SER A  37     -24.638  -2.807  -6.712  1.00  0.34           C  \\nATOM    282  O   SER A  37     -23.435  -3.733  -9.206  1.00  0.34           O  \\nATOM    283  OG  SER A  37     -24.601  -1.391  -6.682  1.00  0.34           O  \\nATOM    284  N   TYR A  38     -22.070  -4.921  -8.414  1.00  0.39           N  \\nATOM    285  CA  TYR A  38     -22.294  -6.124  -9.207  1.00  0.43           C  \\nATOM    286  C   TYR A  38     -23.656  -6.734  -8.900  1.00  0.40           C  \\nATOM    287  CB  TYR A  38     -21.190  -7.154  -8.945  1.00  0.38           C  \\nATOM    288  O   TYR A  38     -23.817  -7.439  -7.901  1.00  0.38           O  \\nATOM    289  CG  TYR A  38     -19.948  -6.937  -9.775  1.00  0.39           C  \\nATOM    290  CD1 TYR A  38     -19.827  -7.501 -11.043  1.00  0.37           C  \\nATOM    291  CD2 TYR A  38     -18.894  -6.170  -9.293  1.00  0.39           C  \\nATOM    292  CE1 TYR A  38     -18.684  -7.305 -11.811  1.00  0.38           C  \\nATOM    293  CE2 TYR A  38     -17.745  -5.968 -10.052  1.00  0.42           C  \\nATOM    294  OH  TYR A  38     -16.516  -6.341 -12.063  1.00  0.28           O  \\nATOM    295  CZ  TYR A  38     -17.650  -6.538 -11.308  1.00  0.34           C  \\nATOM    296  N   SER A  39     -24.645  -6.042  -8.858  1.00  0.35           N  \\nATOM    297  CA  SER A  39     -25.997  -6.556  -9.049  1.00  0.39           C  \\nATOM    298  C   SER A  39     -26.253  -6.909 -10.511  1.00  0.35           C  \\nATOM    299  CB  SER A  39     -27.032  -5.535  -8.574  1.00  0.34           C  \\nATOM    300  O   SER A  39     -26.337  -6.022 -11.363  1.00  0.34           O  \\nATOM    301  OG  SER A  39     -27.108  -4.441  -9.471  1.00  0.35           O  \\nATOM    302  N   GLY A  40     -25.351  -7.737 -11.150  1.00  0.33           N  \\nATOM    303  CA  GLY A  40     -25.725  -8.343 -12.418  1.00  0.38           C  \\nATOM    304  C   GLY A  40     -24.652  -9.256 -12.982  1.00  0.34           C  \\nATOM    305  O   GLY A  40     -23.653  -8.785 -13.529  1.00  0.33           O  \\nATOM    306  N   TYR A  41     -24.147 -10.178 -12.215  1.00  0.36           N  \\nATOM    307  CA  TYR A  41     -23.750 -11.492 -12.707  1.00  0.40           C  \\nATOM    308  C   TYR A  41     -24.253 -12.595 -11.785  1.00  0.37           C  \\nATOM    309  CB  TYR A  41     -22.226 -11.578 -12.841  1.00  0.35           C  \\nATOM    310  O   TYR A  41     -23.798 -12.716 -10.645  1.00  0.35           O  \\nATOM    311  CG  TYR A  41     -21.705 -11.091 -14.171  1.00  0.36           C  \\nATOM    312  CD1 TYR A  41     -21.624 -11.948 -15.266  1.00  0.34           C  \\nATOM    313  CD2 TYR A  41     -21.292  -9.773 -14.336  1.00  0.35           C  \\nATOM    314  CE1 TYR A  41     -21.143 -11.504 -16.493  1.00  0.34           C  \\nATOM    315  CE2 TYR A  41     -20.809  -9.318 -15.558  1.00  0.39           C  \\nATOM    316  OH  TYR A  41     -20.261  -9.745 -17.842  1.00  0.22           O  \\nATOM    317  CZ  TYR A  41     -20.739 -10.190 -16.630  1.00  0.31           C  \\nATOM    318  N   SER A  42     -25.553 -12.900 -11.615  1.00  0.38           N  \\nATOM    319  CA  SER A  42     -26.034 -14.272 -11.491  1.00  0.41           C  \\nATOM    320  C   SER A  42     -25.071 -15.257 -12.144  1.00  0.38           C  \\nATOM    321  CB  SER A  42     -27.422 -14.412 -12.118  1.00  0.37           C  \\nATOM    322  O   SER A  42     -24.973 -15.317 -13.371  1.00  0.37           O  \\nATOM    323  OG  SER A  42     -28.358 -14.889 -11.167  1.00  0.38           O  \\nATOM    324  N   GLN A  43     -23.810 -15.382 -11.824  1.00  0.42           N  \\nATOM    325  CA  GLN A  43     -23.112 -16.638 -12.081  1.00  0.44           C  \\nATOM    326  C   GLN A  43     -23.939 -17.832 -11.614  1.00  0.42           C  \\nATOM    327  CB  GLN A  43     -21.747 -16.644 -11.392  1.00  0.39           C  \\nATOM    328  O   GLN A  43     -24.375 -17.881 -10.462  1.00  0.41           O  \\nATOM    329  CG  GLN A  43     -20.687 -15.827 -12.119  1.00  0.40           C  \\nATOM    330  CD  GLN A  43     -19.341 -15.855 -11.420  1.00  0.45           C  \\nATOM    331  NE2 GLN A  43     -18.434 -14.982 -11.844  1.00  0.38           N  \\nATOM    332  OE1 GLN A  43     -19.119 -16.656 -10.506  1.00  0.49           O  \\nATOM    333  N   SER A  44     -24.765 -18.322 -12.451  1.00  0.39           N  \\nATOM    334  CA  SER A  44     -25.224 -19.707 -12.405  1.00  0.43           C  \\nATOM    335  C   SER A  44     -24.279 -20.576 -11.582  1.00  0.40           C  \\nATOM    336  CB  SER A  44     -25.352 -20.276 -13.819  1.00  0.38           C  \\nATOM    337  O   SER A  44     -23.086 -20.660 -11.880  1.00  0.38           O  \\nATOM    338  OG  SER A  44     -24.087 -20.326 -14.457  1.00  0.39           O  \\nATOM    339  N   THR A  45     -24.286 -20.564 -10.283  1.00  0.41           N  \\nATOM    340  CA  THR A  45     -24.073 -21.756  -9.470  1.00  0.43           C  \\nATOM    341  C   THR A  45     -24.416 -23.017 -10.258  1.00  0.41           C  \\nATOM    342  CB  THR A  45     -24.913 -21.710  -8.180  1.00  0.39           C  \\nATOM    343  O   THR A  45     -25.582 -23.407 -10.338  1.00  0.40           O  \\nATOM    344  CG2 THR A  45     -24.215 -20.891  -7.098  1.00  0.39           C  \\nATOM    345  OG1 THR A  45     -26.186 -21.117  -8.466  1.00  0.35           O  \\nATOM    346  N   ASP A  46     -23.749 -23.224 -11.394  1.00  0.39           N  \\nATOM    347  CA  ASP A  46     -23.664 -24.651 -11.689  1.00  0.43           C  \\nATOM    348  C   ASP A  46     -22.583 -25.322 -10.845  1.00  0.39           C  \\nATOM    349  CB  ASP A  46     -23.384 -24.875 -13.176  1.00  0.37           C  \\nATOM    350  O   ASP A  46     -21.391 -25.088 -11.051  1.00  0.38           O  \\nATOM    351  CG  ASP A  46     -24.627 -25.252 -13.963  1.00  0.39           C  \\nATOM    352  OD1 ASP A  46     -25.620 -25.707 -13.355  1.00  0.42           O  \\nATOM    353  OD2 ASP A  46     -24.613 -25.095 -15.203  1.00  0.43           O  \\nATOM    354  N   THR A  47     -22.759 -25.537  -9.558  1.00  0.38           N  \\nATOM    355  CA  THR A  47     -22.334 -26.618  -8.676  1.00  0.41           C  \\nATOM    356  C   THR A  47     -22.459 -27.969  -9.376  1.00  0.38           C  \\nATOM    357  CB  THR A  47     -23.157 -26.634  -7.374  1.00  0.37           C  \\nATOM    358  O   THR A  47     -23.012 -28.916  -8.813  1.00  0.37           O  \\nATOM    359  CG2 THR A  47     -22.318 -26.173  -6.187  1.00  0.36           C  \\nATOM    360  OG1 THR A  47     -24.283 -25.759  -7.517  1.00  0.33           O  \\nATOM    361  N   SER A  48     -22.106 -27.997 -10.622  1.00  0.35           N  \\nATOM    362  CA  SER A  48     -21.935 -29.329 -11.195  1.00  0.40           C  \\nATOM    363  C   SER A  48     -20.466 -29.623 -11.481  1.00  0.36           C  \\nATOM    364  CB  SER A  48     -22.751 -29.468 -12.481  1.00  0.33           C  \\nATOM    365  O   SER A  48     -19.801 -28.861 -12.185  1.00  0.34           O  \\nATOM    366  OG  SER A  48     -22.429 -28.434 -13.395  1.00  0.36           O  \\nATOM    367  N   GLY A  49     -19.566 -29.850 -10.356  1.00  0.36           N  \\nATOM    368  CA  GLY A  49     -18.435 -30.760 -10.271  1.00  0.38           C  \\nATOM    369  C   GLY A  49     -17.096 -30.063 -10.422  1.00  0.37           C  \\nATOM    370  O   GLY A  49     -16.773 -29.554 -11.497  1.00  0.35           O  \\nATOM    371  N   TYR A  50     -16.724 -29.148  -9.497  1.00  0.39           N  \\nATOM    372  CA  TYR A  50     -15.306 -28.859  -9.318  1.00  0.42           C  \\nATOM    373  C   TYR A  50     -14.472 -30.129  -9.436  1.00  0.39           C  \\nATOM    374  CB  TYR A  50     -15.059 -28.197  -7.959  1.00  0.38           C  \\nATOM    375  O   TYR A  50     -14.575 -31.028  -8.598  1.00  0.39           O  \\nATOM    376  CG  TYR A  50     -15.528 -26.765  -7.887  1.00  0.39           C  \\nATOM    377  CD1 TYR A  50     -14.738 -25.727  -8.376  1.00  0.37           C  \\nATOM    378  CD2 TYR A  50     -16.762 -26.446  -7.331  1.00  0.38           C  \\nATOM    379  CE1 TYR A  50     -15.165 -24.405  -8.310  1.00  0.37           C  \\nATOM    380  CE2 TYR A  50     -17.200 -25.127  -7.260  1.00  0.41           C  \\nATOM    381  OH  TYR A  50     -16.824 -22.808  -7.685  1.00  0.25           O  \\nATOM    382  CZ  TYR A  50     -16.396 -24.115  -7.752  1.00  0.34           C  \\nATOM    383  N   GLY A  51     -14.282 -30.647 -10.507  1.00  0.38           N  \\nATOM    384  CA  GLY A  51     -13.161 -31.513 -10.835  1.00  0.39           C  \\nATOM    385  C   GLY A  51     -11.831 -30.982 -10.336  1.00  0.38           C  \\nATOM    386  O   GLY A  51     -11.481 -29.828 -10.593  1.00  0.37           O  \\nATOM    387  N   GLN A  52     -11.518 -31.149  -8.996  1.00  0.40           N  \\nATOM    388  CA  GLN A  52     -10.130 -31.356  -8.597  1.00  0.43           C  \\nATOM    389  C   GLN A  52      -9.194 -31.281  -9.800  1.00  0.41           C  \\nATOM    390  CB  GLN A  52      -9.970 -32.704  -7.891  1.00  0.38           C  \\nATOM    391  O   GLN A  52      -9.141 -32.208 -10.610  1.00  0.39           O  \\nATOM    392  CG  GLN A  52      -9.815 -32.592  -6.380  1.00  0.39           C  \\nATOM    393  CD  GLN A  52      -9.757 -33.944  -5.694  1.00  0.45           C  \\nATOM    394  NE2 GLN A  52      -9.645 -33.933  -4.370  1.00  0.37           N  \\nATOM    395  OE1 GLN A  52      -9.812 -34.990  -6.349  1.00  0.49           O  \\nATOM    396  N   SER A  53      -8.947 -30.150 -10.338  1.00  0.38           N  \\nATOM    397  CA  SER A  53      -7.666 -30.104 -11.036  1.00  0.41           C  \\nATOM    398  C   SER A  53      -6.505 -29.985 -10.053  1.00  0.38           C  \\nATOM    399  CB  SER A  53      -7.633 -28.934 -12.020  1.00  0.37           C  \\nATOM    400  O   SER A  53      -6.447 -29.039  -9.265  1.00  0.37           O  \\nATOM    401  OG  SER A  53      -7.621 -27.696 -11.330  1.00  0.37           O  \\nATOM    402  N   SER A  54      -6.117 -31.056  -9.430  1.00  0.35           N  \\nATOM    403  CA  SER A  54      -4.749 -31.363  -9.025  1.00  0.38           C  \\nATOM    404  C   SER A  54      -3.737 -30.600  -9.873  1.00  0.36           C  \\nATOM    405  CB  SER A  54      -4.484 -32.865  -9.128  1.00  0.34           C  \\nATOM    406  O   SER A  54      -3.613 -30.849 -11.074  1.00  0.34           O  \\nATOM    407  OG  SER A  54      -4.796 -33.343 -10.426  1.00  0.35           O  \\nATOM    408  N   TYR A  55      -3.497 -29.346  -9.660  1.00  0.40           N  \\nATOM    409  CA  TYR A  55      -2.211 -28.769 -10.035  1.00  0.43           C  \\nATOM    410  C   TYR A  55      -1.060 -29.557  -9.421  1.00  0.40           C  \\nATOM    411  CB  TYR A  55      -2.131 -27.303  -9.598  1.00  0.39           C  \\nATOM    412  O   TYR A  55      -0.824 -29.485  -8.213  1.00  0.39           O  \\nATOM    413  CG  TYR A  55      -2.857 -26.355 -10.522  1.00  0.40           C  \\nATOM    414  CD1 TYR A  55      -2.226 -25.828 -11.647  1.00  0.39           C  \\nATOM    415  CD2 TYR A  55      -4.173 -25.982 -10.270  1.00  0.40           C  \\nATOM    416  CE1 TYR A  55      -2.890 -24.952 -12.499  1.00  0.38           C  \\nATOM    417  CE2 TYR A  55      -4.846 -25.107 -11.116  1.00  0.42           C  \\nATOM    418  OH  TYR A  55      -4.860 -23.732 -13.067  1.00  0.27           O  \\nATOM    419  CZ  TYR A  55      -4.198 -24.598 -12.226  1.00  0.36           C  \\nATOM    420  N   SER A  56      -0.783 -30.681  -9.821  1.00  0.34           N  \\nATOM    421  CA  SER A  56       0.402 -31.517  -9.663  1.00  0.38           C  \\nATOM    422  C   SER A  56       1.310 -31.425 -10.885  1.00  0.34           C  \\nATOM    423  CB  SER A  56       0.003 -32.974  -9.424  1.00  0.33           C  \\nATOM    424  O   SER A  56       0.915 -31.805 -11.989  1.00  0.34           O  \\nATOM    425  OG  SER A  56      -0.850 -33.438 -10.456  1.00  0.34           O  \\nATOM    426  N   SER A  57       1.967 -30.296 -11.115  1.00  0.30           N  \\nATOM    427  CA  SER A  57       3.127 -30.608 -11.943  1.00  0.35           C  \\nATOM    428  C   SER A  57       3.864 -29.340 -12.362  1.00  0.31           C  \\nATOM    429  CB  SER A  57       2.705 -31.395 -13.184  1.00  0.29           C  \\nATOM    430  O   SER A  57       3.251 -28.397 -12.867  1.00  0.30           O  \\nATOM    431  OG  SER A  57       1.802 -30.641 -13.975  1.00  0.30           O  \\nATOM    432  N   TYR A  58       4.966 -28.838 -11.252  1.00  0.32           N  \\nATOM    433  CA  TYR A  58       6.383 -28.735 -11.581  1.00  0.36           C  \\nATOM    434  C   TYR A  58       6.794 -27.280 -11.774  1.00  0.32           C  \\nATOM    435  CB  TYR A  58       6.701 -29.539 -12.845  1.00  0.31           C  \\nATOM    436  O   TYR A  58       6.397 -26.640 -12.750  1.00  0.32           O  \\nATOM    437  CG  TYR A  58       6.918 -31.010 -12.590  1.00  0.33           C  \\nATOM    438  CD1 TYR A  58       8.164 -31.492 -12.193  1.00  0.33           C  \\nATOM    439  CD2 TYR A  58       5.880 -31.922 -12.746  1.00  0.34           C  \\nATOM    440  CE1 TYR A  58       8.369 -32.848 -11.959  1.00  0.30           C  \\nATOM    441  CE2 TYR A  58       6.074 -33.280 -12.514  1.00  0.34           C  \\nATOM    442  OH  TYR A  58       7.518 -35.075 -11.890  1.00  0.18           O  \\nATOM    443  CZ  TYR A  58       7.320 -33.732 -12.121  1.00  0.30           C  \\nATOM    444  N   GLY A  59       7.197 -26.600 -10.508  1.00  0.33           N  \\nATOM    445  CA  GLY A  59       8.459 -25.880 -10.438  1.00  0.36           C  \\nATOM    446  C   GLY A  59       8.816 -25.437  -9.032  1.00  0.33           C  \\nATOM    447  O   GLY A  59       7.988 -24.851  -8.331  1.00  0.32           O  \\nATOM    448  N   GLN A  60       9.434 -26.227  -8.259  1.00  0.33           N  \\nATOM    449  CA  GLN A  60      10.568 -26.074  -7.354  1.00  0.39           C  \\nATOM    450  C   GLN A  60      10.430 -24.812  -6.507  1.00  0.34           C  \\nATOM    451  CB  GLN A  60      11.881 -26.038  -8.138  1.00  0.32           C  \\nATOM    452  O   GLN A  60      10.428 -23.699  -7.037  1.00  0.34           O  \\nATOM    453  CG  GLN A  60      12.533 -27.404  -8.309  1.00  0.35           C  \\nATOM    454  CD  GLN A  60      13.836 -27.339  -9.083  1.00  0.45           C  \\nATOM    455  NE2 GLN A  60      14.581 -28.440  -9.083  1.00  0.34           N  \\nATOM    456  OE1 GLN A  60      14.171 -26.309  -9.676  1.00  0.48           O  \\nATOM    457  N   SER A  61       9.651 -24.976  -5.316  1.00  0.34           N  \\nATOM    458  CA  SER A  61      10.193 -24.859  -3.966  1.00  0.39           C  \\nATOM    459  C   SER A  61       9.123 -25.136  -2.915  1.00  0.34           C  \\nATOM    460  CB  SER A  61      10.787 -23.467  -3.745  1.00  0.32           C  \\nATOM    461  O   SER A  61       8.065 -24.503  -2.916  1.00  0.34           O  \\nATOM    462  OG  SER A  61      12.037 -23.553  -3.084  1.00  0.30           O  \\nATOM    463  N   GLN A  62       8.849 -26.350  -2.755  1.00  0.32           N  \\nATOM    464  CA  GLN A  62       8.832 -26.805  -1.369  1.00  0.37           C  \\nATOM    465  C   GLN A  62       8.004 -25.870  -0.493  1.00  0.32           C  \\nATOM    466  CB  GLN A  62      10.256 -26.914  -0.822  1.00  0.31           C  \\nATOM    467  O   GLN A  62       8.334 -24.690  -0.350  1.00  0.32           O  \\nATOM    468  CG  GLN A  62      11.081 -28.018  -1.470  1.00  0.34           C  \\nATOM    469  CD  GLN A  62      12.432 -28.208  -0.806  1.00  0.43           C  \\nATOM    470  NE2 GLN A  62      13.277 -29.039  -1.407  1.00  0.34           N  \\nATOM    471  OE1 GLN A  62      12.715 -27.612   0.238  1.00  0.47           O  \\nATOM    472  N   ASN A  63       6.594 -26.118  -0.673  1.00  0.35           N  \\nATOM    473  CA  ASN A  63       5.706 -26.293   0.471  1.00  0.42           C  \\nATOM    474  C   ASN A  63       4.243 -26.110   0.078  1.00  0.36           C  \\nATOM    475  CB  ASN A  63       6.079 -25.324   1.595  1.00  0.34           C  \\nATOM    476  O   ASN A  63       3.774 -24.982  -0.078  1.00  0.35           O  \\nATOM    477  CG  ASN A  63       5.846 -25.909   2.973  1.00  0.35           C  \\nATOM    478  ND2 ASN A  63       6.086 -25.109   4.006  1.00  0.27           N  \\nATOM    479  OD1 ASN A  63       5.454 -27.071   3.110  1.00  0.37           O  \\nATOM    480  N   THR A  64       3.829 -27.014  -0.819  1.00  0.34           N  \\nATOM    481  CA  THR A  64       2.519 -27.654  -0.791  1.00  0.36           C  \\nATOM    482  C   THR A  64       2.075 -27.917   0.645  1.00  0.34           C  \\nATOM    483  CB  THR A  64       2.526 -28.977  -1.579  1.00  0.33           C  \\nATOM    484  O   THR A  64       2.516 -28.884   1.269  1.00  0.32           O  \\nATOM    485  CG2 THR A  64       2.037 -28.768  -3.009  1.00  0.32           C  \\nATOM    486  OG1 THR A  64       3.860 -29.499  -1.613  1.00  0.30           O  \\nATOM    487  N   GLY A  65       1.991 -26.856   1.444  1.00  0.37           N  \\nATOM    488  CA  GLY A  65       0.944 -27.005   2.442  1.00  0.39           C  \\nATOM    489  C   GLY A  65      -0.445 -26.739   1.894  1.00  0.38           C  \\nATOM    490  O   GLY A  65      -0.820 -25.587   1.668  1.00  0.36           O  \\nATOM    491  N   TYR A  66      -0.841 -27.369   0.836  1.00  0.39           N  \\nATOM    492  CA  TYR A  66      -2.259 -27.706   0.888  1.00  0.42           C  \\nATOM    493  C   TYR A  66      -2.645 -28.227   2.267  1.00  0.39           C  \\nATOM    494  CB  TYR A  66      -2.603 -28.750  -0.179  1.00  0.38           C  \\nATOM    495  O   TYR A  66      -2.166 -29.279   2.697  1.00  0.38           O  \\nATOM    496  CG  TYR A  66      -2.474 -28.238  -1.593  1.00  0.39           C  \\nATOM    497  CD1 TYR A  66      -3.485 -27.475  -2.173  1.00  0.38           C  \\nATOM    498  CD2 TYR A  66      -1.342 -28.516  -2.352  1.00  0.39           C  \\nATOM    499  CE1 TYR A  66      -3.372 -27.003  -3.477  1.00  0.38           C  \\nATOM    500  CE2 TYR A  66      -1.218 -28.048  -3.656  1.00  0.42           C  \\nATOM    501  OH  TYR A  66      -2.119 -26.828  -5.500  1.00  0.28           O  \\nATOM    502  CZ  TYR A  66      -2.236 -27.294  -4.209  1.00  0.35           C  \\nATOM    503  N   GLY A  67      -2.669 -27.455   3.281  1.00  0.38           N  \\nATOM    504  CA  GLY A  67      -3.566 -27.725   4.393  1.00  0.39           C  \\nATOM    505  C   GLY A  67      -4.935 -28.209   3.953  1.00  0.38           C  \\nATOM    506  O   GLY A  67      -5.567 -27.597   3.090  1.00  0.36           O  \\nATOM    507  N   THR A  68      -4.967 -29.519   3.582  1.00  0.39           N  \\nATOM    508  CA  THR A  68      -6.258 -30.184   3.721  1.00  0.41           C  \\nATOM    509  C   THR A  68      -7.398 -29.173   3.635  1.00  0.39           C  \\nATOM    510  CB  THR A  68      -6.350 -30.953   5.052  1.00  0.38           C  \\nATOM    511  O   THR A  68      -7.583 -28.359   4.542  1.00  0.38           O  \\nATOM    512  CG2 THR A  68      -5.982 -32.421   4.865  1.00  0.37           C  \\nATOM    513  OG1 THR A  68      -5.451 -30.364   6.001  1.00  0.33           O  \\nATOM    514  N   GLN A  69      -7.617 -28.555   2.526  1.00  0.40           N  \\nATOM    515  CA  GLN A  69      -9.038 -28.240   2.428  1.00  0.43           C  \\nATOM    516  C   GLN A  69      -9.896 -29.441   2.814  1.00  0.40           C  \\nATOM    517  CB  GLN A  69      -9.389 -27.777   1.013  1.00  0.37           C  \\nATOM    518  O   GLN A  69      -9.835 -30.488   2.166  1.00  0.38           O  \\nATOM    519  CG  GLN A  69      -9.177 -26.287   0.783  1.00  0.37           C  \\nATOM    520  CD  GLN A  69      -9.615 -25.838  -0.598  1.00  0.44           C  \\nATOM    521  NE2 GLN A  69      -9.430 -24.555  -0.891  1.00  0.36           N  \\nATOM    522  OE1 GLN A  69     -10.115 -26.637  -1.396  1.00  0.47           O  \\nATOM    523  N   SER A  70     -10.093 -29.657   4.051  1.00  0.36           N  \\nATOM    524  CA  SER A  70     -11.357 -30.263   4.456  1.00  0.39           C  \\nATOM    525  C   SER A  70     -12.491 -29.855   3.521  1.00  0.37           C  \\nATOM    526  CB  SER A  70     -11.705 -29.871   5.892  1.00  0.35           C  \\nATOM    527  O   SER A  70     -12.823 -28.673   3.418  1.00  0.35           O  \\nATOM    528  OG  SER A  70     -11.599 -28.469   6.071  1.00  0.37           O  \\nATOM    529  N   THR A  71     -12.500 -30.144   2.190  1.00  0.41           N  \\nATOM    530  CA  THR A  71     -13.865 -30.391   1.742  1.00  0.42           C  \\nATOM    531  C   THR A  71     -14.808 -30.539   2.933  1.00  0.41           C  \\nATOM    532  CB  THR A  71     -13.945 -31.653   0.862  1.00  0.39           C  \\nATOM    533  O   THR A  71     -14.531 -31.303   3.859  1.00  0.39           O  \\nATOM    534  CG2 THR A  71     -14.079 -31.287  -0.613  1.00  0.38           C  \\nATOM    535  OG1 THR A  71     -12.755 -32.430   1.043  1.00  0.35           O  \\nATOM    536  N   PRO A  72     -15.501 -29.541   3.412  1.00  0.42           N  \\nATOM    537  CA  PRO A  72     -16.560 -30.163   4.210  1.00  0.43           C  \\nATOM    538  C   PRO A  72     -16.996 -31.518   3.659  1.00  0.42           C  \\nATOM    539  CB  PRO A  72     -17.703 -29.147   4.130  1.00  0.40           C  \\nATOM    540  O   PRO A  72     -17.439 -31.609   2.511  1.00  0.40           O  \\nATOM    541  CG  PRO A  72     -17.244 -28.133   3.134  1.00  0.39           C  \\nATOM    542  CD  PRO A  72     -15.838 -28.467   2.726  1.00  0.44           C  \\nATOM    543  N   GLN A  73     -16.202 -32.487   3.708  1.00  0.34           N  \\nATOM    544  CA  GLN A  73     -16.822 -33.775   4.003  1.00  0.36           C  \\nATOM    545  C   GLN A  73     -18.178 -33.590   4.678  1.00  0.34           C  \\nATOM    546  CB  GLN A  73     -15.906 -34.621   4.888  1.00  0.33           C  \\nATOM    547  O   GLN A  73     -18.284 -32.895   5.691  1.00  0.33           O  \\nATOM    548  CG  GLN A  73     -15.241 -35.779   4.156  1.00  0.34           C  \\nATOM    549  CD  GLN A  73     -14.324 -36.591   5.051  1.00  0.42           C  \\nATOM    550  NE2 GLN A  73     -13.942 -37.777   4.589  1.00  0.36           N  \\nATOM    551  OE1 GLN A  73     -13.960 -36.155   6.148  1.00  0.47           O  \\nATOM    552  N   GLY A  74     -19.261 -33.669   4.009  1.00  0.38           N  \\nATOM    553  CA  GLY A  74     -20.538 -34.340   4.195  1.00  0.39           C  \\nATOM    554  C   GLY A  74     -20.595 -35.175   5.460  1.00  0.38           C  \\nATOM    555  O   GLY A  74     -19.676 -35.947   5.742  1.00  0.36           O  \\nATOM    556  N   TYR A  75     -20.751 -34.577   6.757  1.00  0.35           N  \\nATOM    557  CA  TYR A  75     -21.558 -35.357   7.689  1.00  0.42           C  \\nATOM    558  C   TYR A  75     -21.510 -36.840   7.342  1.00  0.36           C  \\nATOM    559  CB  TYR A  75     -23.009 -34.867   7.685  1.00  0.34           C  \\nATOM    560  O   TYR A  75     -21.920 -37.243   6.250  1.00  0.35           O  \\nATOM    561  CG  TYR A  75     -23.224 -33.602   8.480  1.00  0.38           C  \\nATOM    562  CD1 TYR A  75     -23.452 -33.651   9.854  1.00  0.39           C  \\nATOM    563  CD2 TYR A  75     -23.202 -32.358   7.861  1.00  0.38           C  \\nATOM    564  CE1 TYR A  75     -23.653 -32.488  10.591  1.00  0.34           C  \\nATOM    565  CE2 TYR A  75     -23.402 -31.189   8.588  1.00  0.39           C  \\nATOM    566  OH  TYR A  75     -23.824 -30.110  10.675  1.00  0.20           O  \\nATOM    567  CZ  TYR A  75     -23.626 -31.264   9.950  1.00  0.32           C  \\nATOM    568  N   GLY A  76     -20.320 -37.411   7.785  1.00  0.37           N  \\nATOM    569  CA  GLY A  76     -20.588 -38.710   8.382  1.00  0.41           C  \\nATOM    570  C   GLY A  76     -19.330 -39.511   8.657  1.00  0.37           C  \\nATOM    571  O   GLY A  76     -18.459 -39.627   7.792  1.00  0.35           O  \\nATOM    572  N   SER A  77     -18.714 -39.337   9.838  1.00  0.33           N  \\nATOM    573  CA  SER A  77     -18.441 -40.455  10.735  1.00  0.40           C  \\nATOM    574  C   SER A  77     -17.237 -41.263  10.260  1.00  0.34           C  \\nATOM    575  CB  SER A  77     -19.664 -41.366  10.846  1.00  0.31           C  \\nATOM    576  O   SER A  77     -17.163 -41.646   9.090  1.00  0.33           O  \\nATOM    577  OG  SER A  77     -19.317 -42.712  10.569  1.00  0.39           O  \\nATOM    578  N   THR A  78     -15.954 -40.901  10.811  1.00  0.35           N  \\nATOM    579  CA  THR A  78     -15.255 -41.990  11.485  1.00  0.42           C  \\nATOM    580  C   THR A  78     -14.111 -42.514  10.621  1.00  0.35           C  \\nATOM    581  CB  THR A  78     -16.215 -43.145  11.825  1.00  0.33           C  \\nATOM    582  O   THR A  78     -14.339 -43.025   9.523  1.00  0.35           O  \\nATOM    583  CG2 THR A  78     -16.852 -42.946  13.196  1.00  0.34           C  \\nATOM    584  OG1 THR A  78     -17.249 -43.206  10.834  1.00  0.32           O  \\nATOM    585  N   GLY A  79     -12.802 -41.871  10.832  1.00  0.38           N  \\nATOM    586  CA  GLY A  79     -11.700 -42.787  11.074  1.00  0.43           C  \\nATOM    587  C   GLY A  79     -10.404 -42.349  10.419  1.00  0.39           C  \\nATOM    588  O   GLY A  79     -10.373 -42.071   9.218  1.00  0.37           O  \\nATOM    589  N   GLY A  80      -9.463 -41.666  11.163  1.00  0.41           N  \\nATOM    590  CA  GLY A  80      -8.075 -41.985  11.457  1.00  0.45           C  \\nATOM    591  C   GLY A  80      -7.161 -41.836  10.256  1.00  0.41           C  \\nATOM    592  O   GLY A  80      -7.230 -42.628   9.314  1.00  0.40           O  \\nATOM    593  N   TYR A  81      -6.770 -40.539   9.933  1.00  0.46           N  \\nATOM    594  CA  TYR A  81      -5.443 -40.366   9.353  1.00  0.52           C  \\nATOM    595  C   TYR A  81      -4.656 -39.297  10.102  1.00  0.47           C  \\nATOM    596  CB  TYR A  81      -5.549 -39.994   7.871  1.00  0.44           C  \\nATOM    597  O   TYR A  81      -5.141 -38.180  10.293  1.00  0.46           O  \\nATOM    598  CG  TYR A  81      -5.846 -41.168   6.970  1.00  0.45           C  \\nATOM    599  CD1 TYR A  81      -4.824 -41.998   6.514  1.00  0.41           C  \\nATOM    600  CD2 TYR A  81      -7.149 -41.450   6.573  1.00  0.42           C  \\nATOM    601  CE1 TYR A  81      -5.094 -43.080   5.683  1.00  0.39           C  \\nATOM    602  CE2 TYR A  81      -7.430 -42.529   5.742  1.00  0.44           C  \\nATOM    603  OH  TYR A  81      -6.670 -44.408   4.480  1.00  0.28           O  \\nATOM    604  CZ  TYR A  81      -6.397 -43.338   5.303  1.00  0.35           C  \\nATOM    605  N   GLY A  82      -3.523 -39.644  10.756  1.00  0.44           N  \\nATOM    606  CA  GLY A  82      -2.174 -39.803  11.275  1.00  0.46           C  \\nATOM    607  C   GLY A  82      -1.215 -38.737  10.781  1.00  0.44           C  \\nATOM    608  O   GLY A  82      -1.091 -38.517   9.574  1.00  0.42           O  \\nATOM    609  N   SER A  83      -0.969 -37.573  11.483  1.00  0.42           N  \\nATOM    610  CA  SER A  83       0.165 -36.797  11.976  1.00  0.47           C  \\nATOM    611  C   SER A  83       1.366 -36.923  11.045  1.00  0.42           C  \\nATOM    612  CB  SER A  83       0.552 -37.248  13.385  1.00  0.40           C  \\nATOM    613  O   SER A  83       1.969 -37.993  10.942  1.00  0.41           O  \\nATOM    614  OG  SER A  83       0.987 -38.597  13.380  1.00  0.46           O  \\nATOM    615  N   SER A  84       1.521 -35.979   9.983  1.00  0.43           N  \\nATOM    616  CA  SER A  84       2.909 -35.735   9.607  1.00  0.46           C  \\nATOM    617  C   SER A  84       3.182 -34.245   9.435  1.00  0.43           C  \\nATOM    618  CB  SER A  84       3.253 -36.477   8.314  1.00  0.40           C  \\nATOM    619  O   SER A  84       2.487 -33.564   8.678  1.00  0.41           O  \\nATOM    620  OG  SER A  84       2.163 -36.441   7.409  1.00  0.42           O  \\nATOM    621  N   GLN A  85       3.921 -33.609  10.447  1.00  0.47           N  \\nATOM    622  CA  GLN A  85       5.017 -32.675  10.685  1.00  0.51           C  \\nATOM    623  C   GLN A  85       5.727 -32.317   9.383  1.00  0.47           C  \\nATOM    624  CB  GLN A  85       6.016 -33.262  11.684  1.00  0.44           C  \\nATOM    625  O   GLN A  85       6.442 -33.143   8.812  1.00  0.46           O  \\nATOM    626  CG  GLN A  85       5.643 -33.022  13.140  1.00  0.45           C  \\nATOM    627  CD  GLN A  85       6.658 -33.599  14.109  1.00  0.51           C  \\nATOM    628  NE2 GLN A  85       6.439 -33.376  15.400  1.00  0.40           N  \\nATOM    629  OE1 GLN A  85       7.631 -34.240  13.700  1.00  0.55           O  \\nATOM    630  N   SER A  86       5.176 -31.369   8.613  1.00  0.43           N  \\nATOM    631  CA  SER A  86       5.976 -30.654   7.624  1.00  0.47           C  \\nATOM    632  C   SER A  86       6.494 -29.333   8.182  1.00  0.44           C  \\nATOM    633  CB  SER A  86       5.158 -30.397   6.358  1.00  0.41           C  \\nATOM    634  O   SER A  86       5.710 -28.486   8.615  1.00  0.43           O  \\nATOM    635  OG  SER A  86       3.999 -29.636   6.654  1.00  0.42           O  \\nATOM    636  N   SER A  87       7.728 -29.256   8.691  1.00  0.43           N  \\nATOM    637  CA  SER A  87       8.843 -28.325   8.824  1.00  0.46           C  \\nATOM    638  C   SER A  87       8.878 -27.334   7.665  1.00  0.43           C  \\nATOM    639  CB  SER A  87      10.169 -29.083   8.897  1.00  0.42           C  \\nATOM    640  O   SER A  87       9.004 -27.731   6.505  1.00  0.42           O  \\nATOM    641  OG  SER A  87      10.210 -30.119   7.931  1.00  0.44           O  \\nATOM    642  N   GLN A  88       8.230 -26.247   7.717  1.00  0.44           N  \\nATOM    643  CA  GLN A  88       8.286 -24.839   7.337  1.00  0.45           C  \\nATOM    644  C   GLN A  88       9.723 -24.326   7.338  1.00  0.44           C  \\nATOM    645  CB  GLN A  88       7.426 -23.994   8.278  1.00  0.41           C  \\nATOM    646  O   GLN A  88      10.382 -24.311   8.379  1.00  0.43           O  \\nATOM    647  CG  GLN A  88       6.019 -23.731   7.756  1.00  0.41           C  \\nATOM    648  CD  GLN A  88       5.166 -22.951   8.738  1.00  0.47           C  \\nATOM    649  NE2 GLN A  88       3.987 -22.528   8.293  1.00  0.39           N  \\nATOM    650  OE1 GLN A  88       5.561 -22.731   9.887  1.00  0.52           O  \\nATOM    651  N   SER A  89      10.364 -24.228   6.349  1.00  0.40           N  \\nATOM    652  CA  SER A  89      11.636 -23.559   6.094  1.00  0.44           C  \\nATOM    653  C   SER A  89      11.434 -22.072   5.824  1.00  0.40           C  \\nATOM    654  CB  SER A  89      12.355 -24.207   4.910  1.00  0.38           C  \\nATOM    655  O   SER A  89      10.616 -21.694   4.983  1.00  0.39           O  \\nATOM    656  OG  SER A  89      11.618 -24.025   3.713  1.00  0.40           O  \\nATOM    657  N   SER A  90      11.375 -21.079   6.851  1.00  0.36           N  \\nATOM    658  CA  SER A  90      12.051 -19.830   7.187  1.00  0.39           C  \\nATOM    659  C   SER A  90      12.440 -19.057   5.931  1.00  0.36           C  \\nATOM    660  CB  SER A  90      13.296 -20.105   8.032  1.00  0.34           C  \\nATOM    661  O   SER A  90      13.349 -19.462   5.203  1.00  0.35           O  \\nATOM    662  OG  SER A  90      14.040 -21.185   7.495  1.00  0.38           O  \\nATOM    663  N   TYR A  91      11.542 -18.524   5.167  1.00  0.43           N  \\nATOM    664  CA  TYR A  91      11.757 -17.134   4.779  1.00  0.48           C  \\nATOM    665  C   TYR A  91      11.379 -16.187   5.911  1.00  0.43           C  \\nATOM    666  CB  TYR A  91      10.948 -16.796   3.523  1.00  0.41           C  \\nATOM    667  O   TYR A  91      10.204 -16.075   6.269  1.00  0.42           O  \\nATOM    668  CG  TYR A  91      11.549 -17.340   2.249  1.00  0.43           C  \\nATOM    669  CD1 TYR A  91      12.567 -16.656   1.588  1.00  0.41           C  \\nATOM    670  CD2 TYR A  91      11.100 -18.538   1.703  1.00  0.42           C  \\nATOM    671  CE1 TYR A  91      13.122 -17.152   0.414  1.00  0.40           C  \\nATOM    672  CE2 TYR A  91      11.648 -19.043   0.529  1.00  0.43           C  \\nATOM    673  OH  TYR A  91      13.203 -18.840  -1.270  1.00  0.28           O  \\nATOM    674  CZ  TYR A  91      12.657 -18.344  -0.107  1.00  0.37           C  \\nATOM    675  N   GLY A  92      12.025 -16.012   6.970  1.00  0.33           N  \\nATOM    676  CA  GLY A  92      12.587 -15.270   8.087  1.00  0.36           C  \\nATOM    677  C   GLY A  92      12.672 -13.777   7.830  1.00  0.34           C  \\nATOM    678  O   GLY A  92      13.346 -13.341   6.895  1.00  0.32           O  \\nATOM    679  N   GLN A  93      11.498 -13.058   7.703  1.00  0.38           N  \\nATOM    680  CA  GLN A  93      11.312 -11.736   8.293  1.00  0.42           C  \\nATOM    681  C   GLN A  93       9.901 -11.215   8.040  1.00  0.39           C  \\nATOM    682  CB  GLN A  93      12.343 -10.751   7.740  1.00  0.36           C  \\nATOM    683  O   GLN A  93       9.555 -10.869   6.909  1.00  0.37           O  \\nATOM    684  CG  GLN A  93      13.500 -10.473   8.690  1.00  0.37           C  \\nATOM    685  CD  GLN A  93      14.569  -9.592   8.072  1.00  0.45           C  \\nATOM    686  NE2 GLN A  93      15.541  -9.182   8.880  1.00  0.37           N  \\nATOM    687  OE1 GLN A  93      14.523  -9.284   6.877  1.00  0.49           O  \\nATOM    688  N   GLN A  94       8.841 -11.688   8.732  1.00  0.40           N  \\nATOM    689  CA  GLN A  94       7.772 -10.926   9.370  1.00  0.43           C  \\nATOM    690  C   GLN A  94       8.338  -9.804  10.236  1.00  0.41           C  \\nATOM    691  CB  GLN A  94       6.887 -11.845  10.214  1.00  0.38           C  \\nATOM    692  O   GLN A  94       8.993 -10.063  11.248  1.00  0.39           O  \\nATOM    693  CG  GLN A  94       5.592 -12.254   9.525  1.00  0.39           C  \\nATOM    694  CD  GLN A  94       4.775 -13.234  10.345  1.00  0.46           C  \\nATOM    695  NE2 GLN A  94       3.626 -13.638   9.814  1.00  0.37           N  \\nATOM    696  OE1 GLN A  94       5.174 -13.626  11.447  1.00  0.49           O  \\nATOM    697  N   SER A  95       8.839  -8.825   9.688  1.00  0.38           N  \\nATOM    698  CA  SER A  95       8.845  -7.543  10.386  1.00  0.43           C  \\nATOM    699  C   SER A  95       7.573  -7.354  11.206  1.00  0.38           C  \\nATOM    700  CB  SER A  95       8.997  -6.392   9.392  1.00  0.37           C  \\nATOM    701  O   SER A  95       6.467  -7.413  10.666  1.00  0.37           O  \\nATOM    702  OG  SER A  95       7.738  -6.018   8.859  1.00  0.39           O  \\nATOM    703  N   SER A  96       7.395  -7.977  12.297  1.00  0.38           N  \\nATOM    704  CA  SER A  96       6.774  -7.367  13.469  1.00  0.40           C  \\nATOM    705  C   SER A  96       6.713  -5.849  13.335  1.00  0.38           C  \\nATOM    706  CB  SER A  96       7.539  -7.744  14.738  1.00  0.36           C  \\nATOM    707  O   SER A  96       7.736  -5.169  13.446  1.00  0.36           O  \\nATOM    708  OG  SER A  96       8.909  -7.399  14.622  1.00  0.38           O  \\nATOM    709  N   TYR A  97       6.071  -5.307  12.412  1.00  0.39           N  \\nATOM    710  CA  TYR A  97       5.535  -4.020  12.839  1.00  0.43           C  \\nATOM    711  C   TYR A  97       4.524  -4.197  13.966  1.00  0.40           C  \\nATOM    712  CB  TYR A  97       4.881  -3.292  11.661  1.00  0.38           C  \\nATOM    713  O   TYR A  97       3.484  -4.833  13.781  1.00  0.39           O  \\nATOM    714  CG  TYR A  97       5.852  -2.492  10.827  1.00  0.40           C  \\nATOM    715  CD1 TYR A  97       6.147  -1.168  11.144  1.00  0.38           C  \\nATOM    716  CD2 TYR A  97       6.475  -3.058   9.720  1.00  0.39           C  \\nATOM    717  CE1 TYR A  97       7.040  -0.427  10.377  1.00  0.39           C  \\nATOM    718  CE2 TYR A  97       7.370  -2.327   8.946  1.00  0.42           C  \\nATOM    719  OH  TYR A  97       8.531  -0.285   8.519  1.00  0.29           O  \\nATOM    720  CZ  TYR A  97       7.646  -1.014   9.282  1.00  0.34           C  \\nATOM    721  N   PRO A  98       4.807  -4.195  15.217  1.00  0.38           N  \\nATOM    722  CA  PRO A  98       3.994  -3.554  16.253  1.00  0.39           C  \\nATOM    723  C   PRO A  98       3.706  -2.085  15.951  1.00  0.38           C  \\nATOM    724  CB  PRO A  98       4.852  -3.695  17.512  1.00  0.36           C  \\nATOM    725  O   PRO A  98       4.630  -1.270  15.890  1.00  0.37           O  \\nATOM    726  CG  PRO A  98       6.238  -3.932  17.006  1.00  0.37           C  \\nATOM    727  CD  PRO A  98       6.153  -4.382  15.576  1.00  0.44           C  \\nATOM    728  N   GLY A  99       2.917  -1.835  14.892  1.00  0.36           N  \\nATOM    729  CA  GLY A  99       2.547  -0.428  14.879  1.00  0.40           C  \\nATOM    730  C   GLY A  99       2.058   0.048  13.524  1.00  0.37           C  \\nATOM    731  O   GLY A  99       2.857   0.247  12.606  1.00  0.35           O  \\nATOM    732  N   TYR A 100       0.887  -0.478  12.937  1.00  0.41           N  \\nATOM    733  CA  TYR A 100      -0.258   0.213  12.355  1.00  0.45           C  \\nATOM    734  C   TYR A 100      -1.537  -0.113  13.116  1.00  0.41           C  \\nATOM    735  CB  TYR A 100      -0.419  -0.163  10.879  1.00  0.39           C  \\nATOM    736  O   TYR A 100      -1.881  -1.284  13.292  1.00  0.40           O  \\nATOM    737  CG  TYR A 100       0.607   0.476   9.974  1.00  0.39           C  \\nATOM    738  CD1 TYR A 100       0.418   1.762   9.473  1.00  0.38           C  \\nATOM    739  CD2 TYR A 100       1.766  -0.205   9.618  1.00  0.39           C  \\nATOM    740  CE1 TYR A 100       1.359   2.354   8.637  1.00  0.37           C  \\nATOM    741  CE2 TYR A 100       2.714   0.377   8.783  1.00  0.41           C  \\nATOM    742  OH  TYR A 100       3.438   2.236   7.472  1.00  0.27           O  \\nATOM    743  CZ  TYR A 100       2.502   1.655   8.299  1.00  0.34           C  \\nATOM    744  N   GLY A 101      -1.710   0.605  14.202  1.00  0.39           N  \\nATOM    745  CA  GLY A 101      -2.753   1.186  15.033  1.00  0.40           C  \\nATOM    746  C   GLY A 101      -4.037   0.378  15.031  1.00  0.39           C  \\nATOM    747  O   GLY A 101      -4.597   0.095  13.970  1.00  0.37           O  \\nATOM    748  N   GLN A 102      -4.060  -0.850  15.642  1.00  0.41           N  \\nATOM    749  CA  GLN A 102      -5.346  -1.114  16.281  1.00  0.43           C  \\nATOM    750  C   GLN A 102      -6.432  -0.193  15.733  1.00  0.41           C  \\nATOM    751  CB  GLN A 102      -5.235  -0.952  17.797  1.00  0.37           C  \\nATOM    752  O   GLN A 102      -6.456   1.000  16.044  1.00  0.39           O  \\nATOM    753  CG  GLN A 102      -4.766  -2.209  18.518  1.00  0.38           C  \\nATOM    754  CD  GLN A 102      -4.569  -1.994  20.007  1.00  0.45           C  \\nATOM    755  NE2 GLN A 102      -3.725  -2.817  20.618  1.00  0.36           N  \\nATOM    756  OE1 GLN A 102      -5.170  -1.094  20.602  1.00  0.50           O  \\nATOM    757  N   GLN A 103      -6.804  -0.233  14.471  1.00  0.47           N  \\nATOM    758  CA  GLN A 103      -8.205   0.166  14.392  1.00  0.50           C  \\nATOM    759  C   GLN A 103      -9.104  -0.836  15.111  1.00  0.47           C  \\nATOM    760  CB  GLN A 103      -8.642   0.310  12.933  1.00  0.44           C  \\nATOM    761  O   GLN A 103      -9.005  -2.043  14.882  1.00  0.46           O  \\nATOM    762  CG  GLN A 103     -10.010   0.956  12.762  1.00  0.43           C  \\nATOM    763  CD  GLN A 103     -10.384   1.159  11.306  1.00  0.49           C  \\nATOM    764  NE2 GLN A 103     -11.681   1.275  11.037  1.00  0.38           N  \\nATOM    765  OE1 GLN A 103      -9.516   1.211  10.429  1.00  0.51           O  \\nATOM    766  N   PRO A 104      -9.466  -0.396  16.305  1.00  0.43           N  \\nATOM    767  CA  PRO A 104     -10.609  -1.071  16.926  1.00  0.42           C  \\nATOM    768  C   PRO A 104     -11.669  -1.492  15.910  1.00  0.43           C  \\nATOM    769  CB  PRO A 104     -11.162  -0.015  17.886  1.00  0.40           C  \\nATOM    770  O   PRO A 104     -11.966  -0.742  14.976  1.00  0.41           O  \\nATOM    771  CG  PRO A 104     -10.525   1.266  17.454  1.00  0.39           C  \\nATOM    772  CD  PRO A 104      -9.377   0.944  16.542  1.00  0.44           C  \\nATOM    773  N   ALA A 105     -11.809  -2.724  15.521  1.00  0.46           N  \\nATOM    774  CA  ALA A 105     -13.035  -3.334  15.013  1.00  0.48           C  \\nATOM    775  C   ALA A 105     -14.228  -2.401  15.196  1.00  0.46           C  \\nATOM    776  CB  ALA A 105     -13.295  -4.667  15.711  1.00  0.44           C  \\nATOM    777  O   ALA A 105     -14.358  -1.744  16.232  1.00  0.46           O  \\nATOM    778  N   PRO A 106     -14.794  -1.762  14.101  1.00  0.47           N  \\nATOM    779  CA  PRO A 106     -16.104  -1.134  14.286  1.00  0.48           C  \\nATOM    780  C   PRO A 106     -17.130  -2.079  14.907  1.00  0.47           C  \\nATOM    781  CB  PRO A 106     -16.513  -0.744  12.863  1.00  0.43           C  \\nATOM    782  O   PRO A 106     -17.299  -3.207  14.438  1.00  0.45           O  \\nATOM    783  CG  PRO A 106     -15.770  -1.694  11.979  1.00  0.42           C  \\nATOM    784  CD  PRO A 106     -14.522  -2.131  12.691  1.00  0.47           C  \\nATOM    785  N   SER A 107     -17.277  -2.103  16.137  1.00  0.43           N  \\nATOM    786  CA  SER A 107     -18.517  -2.422  16.837  1.00  0.44           C  \\nATOM    787  C   SER A 107     -19.683  -2.558  15.863  1.00  0.42           C  \\nATOM    788  CB  SER A 107     -18.836  -1.350  17.880  1.00  0.41           C  \\nATOM    789  O   SER A 107     -19.983  -1.628  15.113  1.00  0.41           O  \\nATOM    790  OG  SER A 107     -18.912  -0.069  17.278  1.00  0.44           O  \\nATOM    791  N   SER A 108     -19.961  -3.751  15.226  1.00  0.47           N  \\nATOM    792  CA  SER A 108     -21.163  -4.405  14.719  1.00  0.50           C  \\nATOM    793  C   SER A 108     -22.388  -4.027  15.546  1.00  0.47           C  \\nATOM    794  CB  SER A 108     -20.987  -5.924  14.718  1.00  0.45           C  \\nATOM    795  O   SER A 108     -22.492  -4.399  16.716  1.00  0.46           O  \\nATOM    796  OG  SER A 108     -20.650  -6.392  16.013  1.00  0.47           O  \\nATOM    797  N   THR A 109     -22.942  -2.959  15.340  1.00  0.43           N  \\nATOM    798  CA  THR A 109     -24.310  -2.678  15.760  1.00  0.47           C  \\nATOM    799  C   THR A 109     -25.293  -3.613  15.060  1.00  0.43           C  \\nATOM    800  CB  THR A 109     -24.694  -1.215  15.471  1.00  0.41           C  \\nATOM    801  O   THR A 109     -25.337  -3.667  13.829  1.00  0.42           O  \\nATOM    802  CG2 THR A 109     -25.089  -0.487  16.751  1.00  0.38           C  \\nATOM    803  OG1 THR A 109     -23.577  -0.542  14.878  1.00  0.36           O  \\nATOM    804  N   SER A 110     -25.702  -4.749  15.645  1.00  0.42           N  \\nATOM    805  CA  SER A 110     -26.935  -5.480  15.919  1.00  0.47           C  \\nATOM    806  C   SER A 110     -28.129  -4.830  15.227  1.00  0.42           C  \\nATOM    807  CB  SER A 110     -27.189  -5.558  17.425  1.00  0.40           C  \\nATOM    808  O   SER A 110     -28.597  -3.770  15.650  1.00  0.42           O  \\nATOM    809  OG  SER A 110     -26.599  -4.456  18.092  1.00  0.46           O  \\nATOM    810  N   GLY A 111     -28.321  -5.103  13.928  1.00  0.43           N  \\nATOM    811  CA  GLY A 111     -29.716  -4.943  13.548  1.00  0.46           C  \\nATOM    812  C   GLY A 111     -30.418  -6.261  13.284  1.00  0.43           C  \\nATOM    813  O   GLY A 111     -29.921  -7.093  12.522  1.00  0.42           O  \\nATOM    814  N   SER A 112     -31.223  -6.820  14.204  1.00  0.44           N  \\nATOM    815  CA  SER A 112     -32.492  -7.530  14.321  1.00  0.49           C  \\nATOM    816  C   SER A 112     -33.255  -7.521  13.000  1.00  0.45           C  \\nATOM    817  CB  SER A 112     -33.355  -6.909  15.421  1.00  0.42           C  \\nATOM    818  O   SER A 112     -33.781  -6.486  12.588  1.00  0.44           O  \\nATOM    819  OG  SER A 112     -34.474  -7.731  15.707  1.00  0.46           O  \\nATOM    820  N   TYR A 113     -32.895  -8.363  12.056  1.00  0.52           N  \\nATOM    821  CA  TYR A 113     -33.869  -8.825  11.074  1.00  0.56           C  \\nATOM    822  C   TYR A 113     -34.327 -10.245  11.387  1.00  0.52           C  \\nATOM    823  CB  TYR A 113     -33.279  -8.766   9.662  1.00  0.49           C  \\nATOM    824  O   TYR A 113     -33.504 -11.153  11.522  1.00  0.51           O  \\nATOM    825  CG  TYR A 113     -33.193  -7.368   9.099  1.00  0.49           C  \\nATOM    826  CD1 TYR A 113     -34.292  -6.777   8.480  1.00  0.45           C  \\nATOM    827  CD2 TYR A 113     -32.013  -6.637   9.182  1.00  0.46           C  \\nATOM    828  CE1 TYR A 113     -34.217  -5.490   7.957  1.00  0.44           C  \\nATOM    829  CE2 TYR A 113     -31.927  -5.349   8.663  1.00  0.47           C  \\nATOM    830  OH  TYR A 113     -32.952  -3.511   7.538  1.00  0.36           O  \\nATOM    831  CZ  TYR A 113     -33.032  -4.785   8.054  1.00  0.39           C  \\nATOM    832  N   GLY A 114     -35.472 -10.396  11.756  1.00  0.45           N  \\nATOM    833  CA  GLY A 114     -36.777 -11.010  11.941  1.00  0.47           C  \\nATOM    834  C   GLY A 114     -37.087 -12.074  10.905  1.00  0.45           C  \\nATOM    835  O   GLY A 114     -36.920 -11.847   9.705  1.00  0.43           O  \\nATOM    836  N   SER A 115     -36.841 -13.432  11.134  1.00  0.39           N  \\nATOM    837  CA  SER A 115     -37.605 -14.675  11.161  1.00  0.48           C  \\nATOM    838  C   SER A 115     -38.501 -14.801   9.933  1.00  0.40           C  \\nATOM    839  CB  SER A 115     -38.453 -14.756  12.431  1.00  0.38           C  \\nATOM    840  O   SER A 115     -39.317 -13.918   9.661  1.00  0.39           O  \\nATOM    841  OG  SER A 115     -39.830 -14.614  12.127  1.00  0.45           O  \\nATOM    842  N   SER A 116     -38.097 -15.613   8.790  1.00  0.39           N  \\nATOM    843  CA  SER A 116     -39.093 -16.533   8.251  1.00  0.45           C  \\nATOM    844  C   SER A 116     -38.831 -16.829   6.778  1.00  0.39           C  \\nATOM    845  CB  SER A 116     -40.500 -15.959   8.421  1.00  0.37           C  \\nATOM    846  O   SER A 116     -38.590 -15.914   5.988  1.00  0.39           O  \\nATOM    847  OG  SER A 116     -40.616 -14.708   7.766  1.00  0.43           O  \\nATOM    848  N   SER A 117     -38.278 -18.100   6.484  1.00  0.40           N  \\nATOM    849  CA  SER A 117     -38.731 -19.236   5.689  1.00  0.47           C  \\nATOM    850  C   SER A 117     -38.279 -19.110   4.238  1.00  0.40           C  \\nATOM    851  CB  SER A 117     -40.255 -19.358   5.748  1.00  0.38           C  \\nATOM    852  O   SER A 117     -38.456 -18.061   3.615  1.00  0.39           O  \\nATOM    853  OG  SER A 117     -40.871 -18.127   5.411  1.00  0.42           O  \\nATOM    854  N   GLN A 118     -37.295 -19.858   3.879  1.00  0.39           N  \\nATOM    855  CA  GLN A 118     -37.010 -20.822   2.821  1.00  0.48           C  \\nATOM    856  C   GLN A 118     -37.158 -20.185   1.442  1.00  0.39           C  \\nATOM    857  CB  GLN A 118     -37.931 -22.037   2.939  1.00  0.38           C  \\nATOM    858  O   GLN A 118     -38.268 -19.850   1.023  1.00  0.39           O  \\nATOM    859  CG  GLN A 118     -37.354 -23.166   3.782  1.00  0.42           C  \\nATOM    860  CD  GLN A 118     -38.311 -24.334   3.933  1.00  0.51           C  \\nATOM    861  NE2 GLN A 118     -37.976 -25.264   4.820  1.00  0.40           N  \\nATOM    862  OE1 GLN A 118     -39.344 -24.399   3.259  1.00  0.54           O  \\nATOM    863  N   SER A 119     -36.064 -19.382   1.000  1.00  0.40           N  \\nATOM    864  CA  SER A 119     -35.509 -19.147  -0.329  1.00  0.46           C  \\nATOM    865  C   SER A 119     -34.091 -18.594  -0.247  1.00  0.40           C  \\nATOM    866  CB  SER A 119     -36.396 -18.183  -1.118  1.00  0.39           C  \\nATOM    867  O   SER A 119     -33.843 -17.607   0.449  1.00  0.39           O  \\nATOM    868  OG  SER A 119     -36.500 -16.936  -0.452  1.00  0.40           O  \\nATOM    869  N   SER A 120     -33.215 -19.377  -0.071  1.00  0.42           N  \\nATOM    870  CA  SER A 120     -31.819 -19.203  -0.459  1.00  0.47           C  \\nATOM    871  C   SER A 120     -31.632 -17.943  -1.298  1.00  0.43           C  \\nATOM    872  CB  SER A 120     -31.323 -20.422  -1.238  1.00  0.41           C  \\nATOM    873  O   SER A 120     -32.039 -17.898  -2.460  1.00  0.42           O  \\nATOM    874  OG  SER A 120     -32.115 -20.637  -2.393  1.00  0.43           O  \\nATOM    875  N   SER A 121     -31.941 -16.803  -0.799  1.00  0.39           N  \\nATOM    876  CA  SER A 121     -31.509 -15.450  -1.133  1.00  0.43           C  \\nATOM    877  C   SER A 121     -30.004 -15.288  -0.943  1.00  0.40           C  \\nATOM    878  CB  SER A 121     -32.252 -14.423  -0.278  1.00  0.37           C  \\nATOM    879  O   SER A 121     -29.486 -15.496   0.155  1.00  0.39           O  \\nATOM    880  OG  SER A 121     -31.372 -13.805   0.645  1.00  0.37           O  \\nATOM    881  N   TYR A 122     -29.159 -15.961  -1.669  1.00  0.46           N  \\nATOM    882  CA  TYR A 122     -27.865 -15.307  -1.821  1.00  0.51           C  \\nATOM    883  C   TYR A 122     -27.937 -13.847  -1.390  1.00  0.46           C  \\nATOM    884  CB  TYR A 122     -27.383 -15.399  -3.272  1.00  0.43           C  \\nATOM    885  O   TYR A 122     -28.618 -13.039  -2.025  1.00  0.45           O  \\nATOM    886  CG  TYR A 122     -26.963 -16.789  -3.685  1.00  0.44           C  \\nATOM    887  CD1 TYR A 122     -25.686 -17.266  -3.398  1.00  0.43           C  \\nATOM    888  CD2 TYR A 122     -27.841 -17.627  -4.364  1.00  0.43           C  \\nATOM    889  CE1 TYR A 122     -25.294 -18.546  -3.778  1.00  0.40           C  \\nATOM    890  CE2 TYR A 122     -27.459 -18.908  -4.749  1.00  0.44           C  \\nATOM    891  OH  TYR A 122     -25.803 -20.625  -4.830  1.00  0.27           O  \\nATOM    892  CZ  TYR A 122     -26.186 -19.358  -4.452  1.00  0.37           C  \\nATOM    893  N   GLY A 123     -28.189 -13.612  -0.156  1.00  0.38           N  \\nATOM    894  CA  GLY A 123     -28.118 -12.407   0.655  1.00  0.40           C  \\nATOM    895  C   GLY A 123     -27.253 -11.323   0.039  1.00  0.38           C  \\nATOM    896  O   GLY A 123     -26.077 -11.552  -0.254  1.00  0.36           O  \\nATOM    897  N   GLN A 124     -27.676 -10.593  -1.056  1.00  0.45           N  \\nATOM    898  CA  GLN A 124     -27.149  -9.243  -1.227  1.00  0.48           C  \\nATOM    899  C   GLN A 124     -26.190  -8.880  -0.098  1.00  0.45           C  \\nATOM    900  CB  GLN A 124     -28.290  -8.226  -1.295  1.00  0.43           C  \\nATOM    901  O   GLN A 124     -26.481  -9.126   1.075  1.00  0.44           O  \\nATOM    902  CG  GLN A 124     -28.867  -8.040  -2.692  1.00  0.42           C  \\nATOM    903  CD  GLN A 124     -29.932  -6.961  -2.749  1.00  0.49           C  \\nATOM    904  NE2 GLN A 124     -29.782  -6.029  -3.683  1.00  0.38           N  \\nATOM    905  OE1 GLN A 124     -30.882  -6.966  -1.959  1.00  0.51           O  \\nATOM    906  N   PRO A 125     -24.849  -8.875  -0.329  1.00  0.46           N  \\nATOM    907  CA  PRO A 125     -24.021  -8.105   0.602  1.00  0.47           C  \\nATOM    908  C   PRO A 125     -24.782  -6.954   1.256  1.00  0.46           C  \\nATOM    909  CB  PRO A 125     -22.890  -7.578  -0.284  1.00  0.42           C  \\nATOM    910  O   PRO A 125     -25.429  -6.165   0.563  1.00  0.44           O  \\nATOM    911  CG  PRO A 125     -23.436  -7.636  -1.674  1.00  0.41           C  \\nATOM    912  CD  PRO A 125     -24.590  -8.596  -1.694  1.00  0.45           C  \\nATOM    913  N   GLN A 126     -25.349  -7.071   2.399  1.00  0.39           N  \\nATOM    914  CA  GLN A 126     -25.623  -6.015   3.368  1.00  0.42           C  \\nATOM    915  C   GLN A 126     -24.597  -4.891   3.262  1.00  0.39           C  \\nATOM    916  CB  GLN A 126     -25.638  -6.580   4.789  1.00  0.36           C  \\nATOM    917  O   GLN A 126     -23.393  -5.129   3.380  1.00  0.37           O  \\nATOM    918  CG  GLN A 126     -26.931  -7.295   5.154  1.00  0.37           C  \\nATOM    919  CD  GLN A 126     -26.988  -7.698   6.616  1.00  0.45           C  \\nATOM    920  NE2 GLN A 126     -28.140  -8.197   7.049  1.00  0.36           N  \\nATOM    921  OE1 GLN A 126     -26.003  -7.562   7.349  1.00  0.48           O  \\nATOM    922  N   SER A 127     -24.663  -3.752   2.461  1.00  0.39           N  \\nATOM    923  CA  SER A 127     -24.270  -2.380   2.766  1.00  0.43           C  \\nATOM    924  C   SER A 127     -23.971  -2.208   4.252  1.00  0.40           C  \\nATOM    925  CB  SER A 127     -25.366  -1.401   2.343  1.00  0.38           C  \\nATOM    926  O   SER A 127     -24.889  -2.079   5.065  1.00  0.38           O  \\nATOM    927  OG  SER A 127     -26.642  -1.867   2.749  1.00  0.40           O  \\nATOM    928  N   GLY A 128     -23.130  -2.998   4.839  1.00  0.37           N  \\nATOM    929  CA  GLY A 128     -22.581  -2.658   6.142  1.00  0.40           C  \\nATOM    930  C   GLY A 128     -21.415  -1.691   6.064  1.00  0.37           C  \\nATOM    931  O   GLY A 128     -20.522  -1.854   5.229  1.00  0.36           O  \\nATOM    932  N   SER A 129     -21.639  -0.286   6.174  1.00  0.40           N  \\nATOM    933  CA  SER A 129     -20.905   0.886   6.639  1.00  0.44           C  \\nATOM    934  C   SER A 129     -19.406   0.728   6.408  1.00  0.41           C  \\nATOM    935  CB  SER A 129     -21.177   1.135   8.123  1.00  0.38           C  \\nATOM    936  O   SER A 129     -18.776  -0.167   6.975  1.00  0.39           O  \\nATOM    937  OG  SER A 129     -21.154   2.522   8.413  1.00  0.40           O  \\nATOM    938  N   TYR A 130     -19.001   0.945   5.152  1.00  0.42           N  \\nATOM    939  CA  TYR A 130     -17.656   1.298   4.710  1.00  0.45           C  \\nATOM    940  C   TYR A 130     -17.026   2.324   5.644  1.00  0.42           C  \\nATOM    941  CB  TYR A 130     -17.687   1.845   3.279  1.00  0.40           C  \\nATOM    942  O   TYR A 130     -17.541   3.434   5.795  1.00  0.41           O  \\nATOM    943  CG  TYR A 130     -17.938   0.789   2.231  1.00  0.40           C  \\nATOM    944  CD1 TYR A 130     -16.887   0.044   1.700  1.00  0.37           C  \\nATOM    945  CD2 TYR A 130     -19.224   0.534   1.768  1.00  0.39           C  \\nATOM    946  CE1 TYR A 130     -17.112  -0.930   0.732  1.00  0.38           C  \\nATOM    947  CE2 TYR A 130     -19.461  -0.437   0.800  1.00  0.43           C  \\nATOM    948  OH  TYR A 130     -18.630  -2.125  -0.668  1.00  0.29           O  \\nATOM    949  CZ  TYR A 130     -18.401  -1.163   0.290  1.00  0.33           C  \\nATOM    950  N   SER A 131     -16.427   1.943   6.688  1.00  0.41           N  \\nATOM    951  CA  SER A 131     -15.359   2.647   7.390  1.00  0.45           C  \\nATOM    952  C   SER A 131     -14.282   3.125   6.421  1.00  0.41           C  \\nATOM    953  CB  SER A 131     -14.733   1.746   8.456  1.00  0.39           C  \\nATOM    954  O   SER A 131     -13.748   2.336   5.639  1.00  0.40           O  \\nATOM    955  OG  SER A 131     -14.171   0.586   7.867  1.00  0.39           O  \\nATOM    956  N   GLN A 132     -14.453   4.328   5.716  1.00  0.45           N  \\nATOM    957  CA  GLN A 132     -13.439   5.259   5.232  1.00  0.46           C  \\nATOM    958  C   GLN A 132     -12.041   4.820   5.658  1.00  0.45           C  \\nATOM    959  CB  GLN A 132     -13.722   6.674   5.738  1.00  0.42           C  \\nATOM    960  O   GLN A 132     -11.745   4.749   6.853  1.00  0.43           O  \\nATOM    961  CG  GLN A 132     -14.854   7.376   5.000  1.00  0.42           C  \\nATOM    962  CD  GLN A 132     -15.121   8.773   5.528  1.00  0.47           C  \\nATOM    963  NE2 GLN A 132     -16.128   9.436   4.970  1.00  0.40           N  \\nATOM    964  OE1 GLN A 132     -14.428   9.253   6.431  1.00  0.52           O  \\nATOM    965  N   GLN A 133     -11.391   3.971   5.029  1.00  0.43           N  \\nATOM    966  CA  GLN A 133      -9.934   4.028   4.969  1.00  0.47           C  \\nATOM    967  C   GLN A 133      -9.444   5.466   4.829  1.00  0.43           C  \\nATOM    968  CB  GLN A 133      -9.411   3.180   3.808  1.00  0.41           C  \\nATOM    969  O   GLN A 133     -10.008   6.247   4.060  1.00  0.42           O  \\nATOM    970  CG  GLN A 133      -8.116   2.441   4.120  1.00  0.41           C  \\nATOM    971  CD  GLN A 133      -7.661   1.548   2.982  1.00  0.48           C  \\nATOM    972  NE2 GLN A 133      -6.408   1.704   2.570  1.00  0.39           N  \\nATOM    973  OE1 GLN A 133      -8.430   0.723   2.477  1.00  0.51           O  \\nATOM    974  N   PRO A 134      -8.789   6.052   5.836  1.00  0.42           N  \\nATOM    975  CA  PRO A 134      -8.054   7.299   5.608  1.00  0.42           C  \\nATOM    976  C   PRO A 134      -7.497   7.405   4.190  1.00  0.42           C  \\nATOM    977  CB  PRO A 134      -6.922   7.234   6.636  1.00  0.39           C  \\nATOM    978  O   PRO A 134      -6.877   6.460   3.695  1.00  0.40           O  \\nATOM    979  CG  PRO A 134      -6.801   5.785   6.980  1.00  0.38           C  \\nATOM    980  CD  PRO A 134      -8.082   5.097   6.605  1.00  0.42           C  \\nATOM    981  N   SER A 135      -8.112   7.983   3.141  1.00  0.36           N  \\nATOM    982  CA  SER A 135      -7.418   8.761   2.120  1.00  0.37           C  \\nATOM    983  C   SER A 135      -6.062   9.247   2.621  1.00  0.36           C  \\nATOM    984  CB  SER A 135      -8.268   9.955   1.685  1.00  0.35           C  \\nATOM    985  O   SER A 135      -5.989  10.012   3.585  1.00  0.35           O  \\nATOM    986  OG  SER A 135      -7.781  11.155   2.260  1.00  0.38           O  \\nATOM    987  N   TYR A 136      -5.187   8.388   2.961  1.00  0.39           N  \\nATOM    988  CA  TYR A 136      -3.844   8.956   2.929  1.00  0.41           C  \\nATOM    989  C   TYR A 136      -3.495   9.451   1.531  1.00  0.39           C  \\nATOM    990  CB  TYR A 136      -2.812   7.923   3.390  1.00  0.37           C  \\nATOM    991  O   TYR A 136      -3.660   8.723   0.549  1.00  0.37           O  \\nATOM    992  CG  TYR A 136      -2.782   7.721   4.886  1.00  0.38           C  \\nATOM    993  CD1 TYR A 136      -2.021   8.552   5.705  1.00  0.36           C  \\nATOM    994  CD2 TYR A 136      -3.512   6.699   5.482  1.00  0.38           C  \\nATOM    995  CE1 TYR A 136      -1.988   8.368   7.084  1.00  0.38           C  \\nATOM    996  CE2 TYR A 136      -3.487   6.506   6.860  1.00  0.41           C  \\nATOM    997  OH  TYR A 136      -2.695   7.159   9.015  1.00  0.28           O  \\nATOM    998  CZ  TYR A 136      -2.724   7.345   7.651  1.00  0.33           C  \\nATOM    999  N   GLY A 137      -3.855  10.742   1.121  1.00  0.36           N  \\nATOM   1000  CA  GLY A 137      -3.296  11.566   0.062  1.00  0.37           C  \\nATOM   1001  C   GLY A 137      -2.895  10.770  -1.166  1.00  0.36           C  \\nATOM   1002  O   GLY A 137      -1.988   9.938  -1.104  1.00  0.34           O  \\nATOM   1003  N   GLY A 138      -3.640  10.006  -1.809  1.00  0.36           N  \\nATOM   1004  CA  GLY A 138      -3.319   9.718  -3.198  1.00  0.36           C  \\nATOM   1005  C   GLY A 138      -2.179  10.562  -3.735  1.00  0.36           C  \\nATOM   1006  O   GLY A 138      -2.301  11.785  -3.839  1.00  0.34           O  \\nATOM   1007  N   GLN A 139      -1.036  10.624  -3.185  1.00  0.36           N  \\nATOM   1008  CA  GLN A 139      -0.090  10.893  -4.263  1.00  0.38           C  \\nATOM   1009  C   GLN A 139      -0.489  10.160  -5.540  1.00  0.36           C  \\nATOM   1010  CB  GLN A 139       1.326  10.492  -3.848  1.00  0.34           C  \\nATOM   1011  O   GLN A 139      -0.505   8.927  -5.576  1.00  0.34           O  \\nATOM   1012  CG  GLN A 139       2.053  11.556  -3.035  1.00  0.34           C  \\nATOM   1013  CD  GLN A 139       3.483  11.169  -2.708  1.00  0.41           C  \\nATOM   1014  NE2 GLN A 139       4.227  12.096  -2.116  1.00  0.35           N  \\nATOM   1015  OE1 GLN A 139       3.915  10.046  -2.986  1.00  0.46           O  \\nATOM   1016  N   GLN A 140      -1.561  10.493  -6.230  1.00  0.38           N  \\nATOM   1017  CA  GLN A 140      -1.350  10.493  -7.674  1.00  0.39           C  \\nATOM   1018  C   GLN A 140       0.113  10.757  -8.015  1.00  0.38           C  \\nATOM   1019  CB  GLN A 140      -2.244  11.535  -8.348  1.00  0.36           C  \\nATOM   1020  O   GLN A 140       0.604  11.875  -7.844  1.00  0.37           O  \\nATOM   1021  CG  GLN A 140      -3.379  10.936  -9.166  1.00  0.36           C  \\nATOM   1022  CD  GLN A 140      -4.302  11.989  -9.749  1.00  0.43           C  \\nATOM   1023  NE2 GLN A 140      -5.190  11.569 -10.645  1.00  0.35           N  \\nATOM   1024  OE1 GLN A 140      -4.219  13.170  -9.398  1.00  0.48           O  \\nATOM   1025  N   GLN A 141       1.006   9.957  -7.662  1.00  0.34           N  \\nATOM   1026  CA  GLN A 141       2.117  10.013  -8.606  1.00  0.35           C  \\nATOM   1027  C   GLN A 141       1.633   9.810 -10.039  1.00  0.34           C  \\nATOM   1028  CB  GLN A 141       3.174   8.965  -8.255  1.00  0.32           C  \\nATOM   1029  O   GLN A 141       0.961   8.821 -10.338  1.00  0.32           O  \\nATOM   1030  CG  GLN A 141       4.041   9.340  -7.061  1.00  0.33           C  \\nATOM   1031  CD  GLN A 141       5.174   8.360  -6.826  1.00  0.40           C  \\nATOM   1032  NE2 GLN A 141       5.997   8.634  -5.820  1.00  0.33           N  \\nATOM   1033  OE1 GLN A 141       5.309   7.363  -7.543  1.00  0.45           O  \\nATOM   1034  N   SER A 142       1.100  10.799 -10.713  1.00  0.35           N  \\nATOM   1035  CA  SER A 142       1.366  10.990 -12.135  1.00  0.36           C  \\nATOM   1036  C   SER A 142       2.522  10.113 -12.604  1.00  0.35           C  \\nATOM   1037  CB  SER A 142       1.676  12.458 -12.430  1.00  0.33           C  \\nATOM   1038  O   SER A 142       3.656  10.276 -12.148  1.00  0.33           O  \\nATOM   1039  OG  SER A 142       0.496  13.162 -12.776  1.00  0.35           O  \\nATOM   1040  N   TYR A 143       2.496   8.847 -12.535  1.00  0.34           N  \\nATOM   1041  CA  TYR A 143       3.304   8.109 -13.499  1.00  0.37           C  \\nATOM   1042  C   TYR A 143       3.199   8.730 -14.887  1.00  0.34           C  \\nATOM   1043  CB  TYR A 143       2.871   6.641 -13.551  1.00  0.33           C  \\nATOM   1044  O   TYR A 143       2.149   8.658 -15.529  1.00  0.33           O  \\nATOM   1045  CG  TYR A 143       3.289   5.842 -12.340  1.00  0.34           C  \\nATOM   1046  CD1 TYR A 143       4.567   5.296 -12.248  1.00  0.33           C  \\nATOM   1047  CD2 TYR A 143       2.406   5.631 -11.286  1.00  0.35           C  \\nATOM   1048  CE1 TYR A 143       4.955   4.557 -11.135  1.00  0.34           C  \\nATOM   1049  CE2 TYR A 143       2.783   4.894 -10.168  1.00  0.38           C  \\nATOM   1050  OH  TYR A 143       4.437   3.633  -8.998  1.00  0.21           O  \\nATOM   1051  CZ  TYR A 143       4.058   4.363 -10.102  1.00  0.31           C  \\nATOM   1052  N   GLY A 144       3.534   9.847 -15.150  1.00  0.34           N  \\nATOM   1053  CA  GLY A 144       4.233  10.527 -16.229  1.00  0.36           C  \\nATOM   1054  C   GLY A 144       5.702  10.160 -16.313  1.00  0.34           C  \\nATOM   1055  O   GLY A 144       6.496  10.553 -15.456  1.00  0.33           O  \\nATOM   1056  N   GLN A 145       5.971   8.815 -16.182  1.00  0.33           N  \\nATOM   1057  CA  GLN A 145       7.095   8.218 -16.894  1.00  0.36           C  \\nATOM   1058  C   GLN A 145       6.718   6.859 -17.477  1.00  0.34           C  \\nATOM   1059  CB  GLN A 145       8.304   8.074 -15.968  1.00  0.31           C  \\nATOM   1060  O   GLN A 145       6.318   5.952 -16.743  1.00  0.32           O  \\nATOM   1061  CG  GLN A 145       9.111   9.355 -15.807  1.00  0.32           C  \\nATOM   1062  CD  GLN A 145      10.335   9.172 -14.930  1.00  0.39           C  \\nATOM   1063  NE2 GLN A 145      11.179  10.196 -14.868  1.00  0.33           N  \\nATOM   1064  OE1 GLN A 145      10.520   8.119 -14.311  1.00  0.45           O  \\nATOM   1065  N   GLN A 146       5.928   6.688 -18.643  1.00  0.36           N  \\nATOM   1066  CA  GLN A 146       6.265   5.702 -19.664  1.00  0.40           C  \\nATOM   1067  C   GLN A 146       7.709   5.230 -19.519  1.00  0.37           C  \\nATOM   1068  CB  GLN A 146       6.039   6.278 -21.063  1.00  0.35           C  \\nATOM   1069  O   GLN A 146       8.643   5.960 -19.857  1.00  0.36           O  \\nATOM   1070  CG  GLN A 146       4.700   5.892 -21.678  1.00  0.36           C  \\nATOM   1071  CD  GLN A 146       4.484   6.509 -23.047  1.00  0.44           C  \\nATOM   1072  NE2 GLN A 146       3.453   6.049 -23.749  1.00  0.36           N  \\nATOM   1073  OE1 GLN A 146       5.236   7.392 -23.471  1.00  0.48           O  \\nATOM   1074  N   GLN A 147       8.090   4.673 -18.461  1.00  0.33           N  \\nATOM   1075  CA  GLN A 147       9.135   3.710 -18.791  1.00  0.36           C  \\nATOM   1076  C   GLN A 147       8.595   2.598 -19.686  1.00  0.34           C  \\nATOM   1077  CB  GLN A 147       9.737   3.113 -17.518  1.00  0.32           C  \\nATOM   1078  O   GLN A 147       7.601   1.951 -19.351  1.00  0.32           O  \\nATOM   1079  CG  GLN A 147      10.728   4.032 -16.817  1.00  0.33           C  \\nATOM   1080  CD  GLN A 147      11.456   3.349 -15.674  1.00  0.40           C  \\nATOM   1081  NE2 GLN A 147      12.405   4.055 -15.070  1.00  0.33           N  \\nATOM   1082  OE1 GLN A 147      11.166   2.197 -15.337  1.00  0.45           O  \\nATOM   1083  N   SER A 148       8.387   2.780 -20.991  1.00  0.37           N  \\nATOM   1084  CA  SER A 148       8.548   1.575 -21.798  1.00  0.39           C  \\nATOM   1085  C   SER A 148       9.109   0.425 -20.968  1.00  0.37           C  \\nATOM   1086  CB  SER A 148       9.463   1.847 -22.992  1.00  0.35           C  \\nATOM   1087  O   SER A 148      10.240   0.498 -20.481  1.00  0.36           O  \\nATOM   1088  OG  SER A 148      10.279   2.981 -22.753  1.00  0.37           O  \\nATOM   1089  N   TYR A 149       8.588  -0.015 -19.954  1.00  0.32           N  \\nATOM   1090  CA  TYR A 149       9.000  -1.347 -19.523  1.00  0.36           C  \\nATOM   1091  C   TYR A 149       9.073  -2.304 -20.707  1.00  0.32           C  \\nATOM   1092  CB  TYR A 149       8.034  -1.894 -18.468  1.00  0.31           C  \\nATOM   1093  O   TYR A 149       8.049  -2.636 -21.309  1.00  0.31           O  \\nATOM   1094  CG  TYR A 149       8.661  -2.903 -17.537  1.00  0.33           C  \\nATOM   1095  CD1 TYR A 149       8.533  -4.270 -17.772  1.00  0.30           C  \\nATOM   1096  CD2 TYR A 149       9.380  -2.492 -16.420  1.00  0.31           C  \\nATOM   1097  CE1 TYR A 149       9.106  -5.204 -16.915  1.00  0.29           C  \\nATOM   1098  CE2 TYR A 149       9.958  -3.417 -15.557  1.00  0.33           C  \\nATOM   1099  OH  TYR A 149      10.387  -5.688 -14.961  1.00  0.12           O  \\nATOM   1100  CZ  TYR A 149       9.817  -4.768 -15.812  1.00  0.28           C  \\nATOM   1101  N   ASN A 150      10.079  -2.148 -21.535  1.00  0.36           N  \\nATOM   1102  CA  ASN A 150      10.721  -3.394 -21.937  1.00  0.39           C  \\nATOM   1103  C   ASN A 150      10.775  -4.394 -20.786  1.00  0.37           C  \\nATOM   1104  CB  ASN A 150      12.129  -3.125 -22.472  1.00  0.36           C  \\nATOM   1105  O   ASN A 150      11.441  -4.152 -19.778  1.00  0.35           O  \\nATOM   1106  CG  ASN A 150      12.303  -3.572 -23.910  1.00  0.36           C  \\nATOM   1107  ND2 ASN A 150      13.432  -3.214 -24.510  1.00  0.31           N  \\nATOM   1108  OD1 ASN A 150      11.429  -4.233 -24.478  1.00  0.37           O  \\nATOM   1109  N   PRO A 151       9.730  -5.032 -20.422  1.00  0.40           N  \\nATOM   1110  CA  PRO A 151      10.030  -6.134 -19.505  1.00  0.41           C  \\nATOM   1111  C   PRO A 151      11.466  -6.636 -19.636  1.00  0.40           C  \\nATOM   1112  CB  PRO A 151       9.033  -7.219 -19.919  1.00  0.39           C  \\nATOM   1113  O   PRO A 151      12.041  -6.597 -20.727  1.00  0.40           O  \\nATOM   1114  CG  PRO A 151       8.658  -6.877 -21.325  1.00  0.39           C  \\nATOM   1115  CD  PRO A 151       8.961  -5.425 -21.560  1.00  0.44           C  \\nATOM   1116  N   PRO A 152      12.244  -6.616 -18.677  1.00  0.35           N  \\nATOM   1117  CA  PRO A 152      13.533  -7.288 -18.857  1.00  0.35           C  \\nATOM   1118  C   PRO A 152      13.465  -8.434 -19.864  1.00  0.35           C  \\nATOM   1119  CB  PRO A 152      13.857  -7.811 -17.455  1.00  0.32           C  \\nATOM   1120  O   PRO A 152      12.554  -9.263 -19.800  1.00  0.32           O  \\nATOM   1121  CG  PRO A 152      12.598  -7.615 -16.674  1.00  0.32           C  \\nATOM   1122  CD  PRO A 152      11.652  -6.787 -17.494  1.00  0.37           C  \\nATOM   1123  N   GLN A 153      13.606  -8.321 -21.232  1.00  0.31           N  \\nATOM   1124  CA  GLN A 153      14.243  -9.422 -21.947  1.00  0.34           C  \\nATOM   1125  C   GLN A 153      15.051 -10.301 -20.997  1.00  0.32           C  \\nATOM   1126  CB  GLN A 153      15.143  -8.889 -23.063  1.00  0.31           C  \\nATOM   1127  O   GLN A 153      16.022  -9.842 -20.392  1.00  0.31           O  \\nATOM   1128  CG  GLN A 153      14.380  -8.358 -24.268  1.00  0.33           C  \\nATOM   1129  CD  GLN A 153      15.286  -8.040 -25.443  1.00  0.42           C  \\nATOM   1130  NE2 GLN A 153      14.686  -7.728 -26.587  1.00  0.33           N  \\nATOM   1131  OE1 GLN A 153      16.515  -8.072 -25.323  1.00  0.47           O  \\nATOM   1132  N   GLY A 154      14.327 -11.029 -19.978  1.00  0.32           N  \\nATOM   1133  CA  GLY A 154      15.010 -12.035 -19.182  1.00  0.35           C  \\nATOM   1134  C   GLY A 154      14.146 -12.606 -18.073  1.00  0.33           C  \\nATOM   1135  O   GLY A 154      13.850 -11.918 -17.093  1.00  0.31           O  \\nATOM   1136  N   TYR A 155      12.988 -13.277 -18.475  1.00  0.30           N  \\nATOM   1137  CA  TYR A 155      12.438 -14.575 -18.103  1.00  0.35           C  \\nATOM   1138  C   TYR A 155      11.657 -15.188 -19.259  1.00  0.31           C  \\nATOM   1139  CB  TYR A 155      11.534 -14.444 -16.873  1.00  0.29           C  \\nATOM   1140  O   TYR A 155      10.588 -14.694 -19.624  1.00  0.30           O  \\nATOM   1141  CG  TYR A 155      12.288 -14.187 -15.591  1.00  0.31           C  \\nATOM   1142  CD1 TYR A 155      12.846 -15.237 -14.865  1.00  0.31           C  \\nATOM   1143  CD2 TYR A 155      12.441 -12.895 -15.101  1.00  0.31           C  \\nATOM   1144  CE1 TYR A 155      13.540 -15.005 -13.682  1.00  0.28           C  \\nATOM   1145  CE2 TYR A 155      13.132 -12.651 -13.919  1.00  0.31           C  \\nATOM   1146  OH  TYR A 155      14.363 -13.475 -12.047  1.00  0.11           O  \\nATOM   1147  CZ  TYR A 155      13.678 -13.711 -13.218  1.00  0.29           C  \\nATOM   1148  N   GLY A 156      12.182 -15.584 -20.408  1.00  0.35           N  \\nATOM   1149  CA  GLY A 156      11.993 -16.937 -20.905  1.00  0.35           C  \\nATOM   1150  C   GLY A 156      11.741 -17.948 -19.802  1.00  0.35           C  \\nATOM   1151  O   GLY A 156      12.669 -18.350 -19.097  1.00  0.34           O  \\nATOM   1152  N   GLN A 157      11.178 -17.767 -18.769  1.00  0.33           N  \\nATOM   1153  CA  GLN A 157      10.736 -19.063 -18.265  1.00  0.36           C  \\nATOM   1154  C   GLN A 157       9.856 -19.779 -19.285  1.00  0.34           C  \\nATOM   1155  CB  GLN A 157       9.982 -18.897 -16.944  1.00  0.32           C  \\nATOM   1156  O   GLN A 157       8.784 -19.285 -19.643  1.00  0.33           O  \\nATOM   1157  CG  GLN A 157      10.889 -18.792 -15.726  1.00  0.33           C  \\nATOM   1158  CD  GLN A 157      10.117 -18.739 -14.422  1.00  0.40           C  \\nATOM   1159  NE2 GLN A 157      10.833 -18.567 -13.315  1.00  0.33           N  \\nATOM   1160  OE1 GLN A 157       8.887 -18.850 -14.408  1.00  0.44           O  \\nATOM   1161  N   GLN A 158      10.513 -20.183 -20.428  1.00  0.34           N  \\nATOM   1162  CA  GLN A 158      10.269 -21.502 -21.001  1.00  0.36           C  \\nATOM   1163  C   GLN A 158       9.440 -22.368 -20.056  1.00  0.35           C  \\nATOM   1164  CB  GLN A 158      11.591 -22.199 -21.327  1.00  0.33           C  \\nATOM   1165  O   GLN A 158       9.923 -22.782 -19.001  1.00  0.33           O  \\nATOM   1166  CG  GLN A 158      12.178 -21.804 -22.675  1.00  0.33           C  \\nATOM   1167  CD  GLN A 158      13.439 -22.576 -23.015  1.00  0.41           C  \\nATOM   1168  NE2 GLN A 158      13.929 -22.403 -24.238  1.00  0.33           N  \\nATOM   1169  OE1 GLN A 158      13.969 -23.321 -22.185  1.00  0.44           O  \\nATOM   1170  N   ASN A 159       8.393 -22.041 -19.561  1.00  0.38           N  \\nATOM   1171  CA  ASN A 159       7.594 -23.219 -19.238  1.00  0.42           C  \\nATOM   1172  C   ASN A 159       7.590 -24.226 -20.384  1.00  0.38           C  \\nATOM   1173  CB  ASN A 159       6.162 -22.815 -18.881  1.00  0.37           C  \\nATOM   1174  O   ASN A 159       7.064 -23.943 -21.462  1.00  0.37           O  \\nATOM   1175  CG  ASN A 159       5.999 -22.478 -17.412  1.00  0.38           C  \\nATOM   1176  ND2 ASN A 159       4.881 -21.849 -17.068  1.00  0.32           N  \\nATOM   1177  OD1 ASN A 159       6.872 -22.779 -16.593  1.00  0.39           O  \\nATOM   1178  N   GLN A 160       8.712 -24.726 -20.735  1.00  0.35           N  \\nATOM   1179  CA  GLN A 160       8.791 -26.103 -21.210  1.00  0.36           C  \\nATOM   1180  C   GLN A 160       7.522 -26.877 -20.866  1.00  0.35           C  \\nATOM   1181  CB  GLN A 160      10.012 -26.807 -20.617  1.00  0.33           C  \\nATOM   1182  O   GLN A 160       7.232 -27.115 -19.691  1.00  0.33           O  \\nATOM   1183  CG  GLN A 160      11.297 -26.574 -21.400  1.00  0.33           C  \\nATOM   1184  CD  GLN A 160      12.480 -27.330 -20.825  1.00  0.40           C  \\nATOM   1185  NE2 GLN A 160      13.655 -27.126 -21.411  1.00  0.33           N  \\nATOM   1186  OE1 GLN A 160      12.340 -28.090 -19.862  1.00  0.44           O  \\nATOM   1187  N   TYR A 161       6.266 -26.583 -21.389  1.00  0.42           N  \\nATOM   1188  CA  TYR A 161       5.534 -27.813 -21.669  1.00  0.45           C  \\nATOM   1189  C   TYR A 161       6.478 -29.008 -21.731  1.00  0.42           C  \\nATOM   1190  CB  TYR A 161       4.760 -27.690 -22.985  1.00  0.40           C  \\nATOM   1191  O   TYR A 161       7.337 -29.082 -22.612  1.00  0.40           O  \\nATOM   1192  CG  TYR A 161       3.400 -27.052 -22.832  1.00  0.41           C  \\nATOM   1193  CD1 TYR A 161       2.283 -27.817 -22.503  1.00  0.39           C  \\nATOM   1194  CD2 TYR A 161       3.229 -25.685 -23.018  1.00  0.40           C  \\nATOM   1195  CE1 TYR A 161       1.028 -27.235 -22.363  1.00  0.39           C  \\nATOM   1196  CE2 TYR A 161       1.978 -25.091 -22.881  1.00  0.42           C  \\nATOM   1197  OH  TYR A 161      -0.354 -25.291 -22.416  1.00  0.26           O  \\nATOM   1198  CZ  TYR A 161       0.886 -25.873 -22.553  1.00  0.36           C  \\nATOM   1199  N   ASN A 162       7.134 -29.247 -20.724  1.00  0.33           N  \\nATOM   1200  CA  ASN A 162       7.399 -30.682 -20.726  1.00  0.39           C  \\nATOM   1201  C   ASN A 162       6.236 -31.465 -21.329  1.00  0.34           C  \\nATOM   1202  CB  ASN A 162       7.692 -31.175 -19.307  1.00  0.32           C  \\nATOM   1203  O   ASN A 162       5.135 -31.474 -20.777  1.00  0.32           O  \\nATOM   1204  CG  ASN A 162       9.122 -31.650 -19.138  1.00  0.34           C  \\nATOM   1205  ND2 ASN A 162       9.448 -32.137 -17.946  1.00  0.18           N  \\nATOM   1206  OD1 ASN A 162       9.927 -31.579 -20.070  1.00  0.34           O  \\nATOM   1207  N   SER A 163       6.067 -31.467 -22.678  1.00  0.36           N  \\nATOM   1208  CA  SER A 163       5.915 -32.818 -23.209  1.00  0.38           C  \\nATOM   1209  C   SER A 163       6.333 -33.866 -22.183  1.00  0.37           C  \\nATOM   1210  CB  SER A 163       6.736 -32.988 -24.488  1.00  0.35           C  \\nATOM   1211  O   SER A 163       7.523 -34.030 -21.906  1.00  0.35           O  \\nATOM   1212  OG  SER A 163       8.111 -32.750 -24.237  1.00  0.36           O  \\nATOM   1213  N   SER A 164       5.810 -33.834 -21.034  1.00  0.35           N  \\nATOM   1214  CA  SER A 164       5.840 -35.210 -20.548  1.00  0.39           C  \\nATOM   1215  C   SER A 164       5.205 -36.166 -21.552  1.00  0.36           C  \\nATOM   1216  CB  SER A 164       5.121 -35.318 -19.202  1.00  0.33           C  \\nATOM   1217  O   SER A 164       3.998 -36.105 -21.797  1.00  0.34           O  \\nATOM   1218  OG  SER A 164       3.752 -34.974 -19.335  1.00  0.35           O  \\nATOM   1219  N   SER A 165       5.813 -36.352 -22.679  1.00  0.33           N  \\nATOM   1220  CA  SER A 165       6.040 -37.739 -23.070  1.00  0.35           C  \\nATOM   1221  C   SER A 165       5.885 -38.682 -21.881  1.00  0.33           C  \\nATOM   1222  CB  SER A 165       7.431 -37.903 -23.683  1.00  0.32           C  \\nATOM   1223  O   SER A 165       6.803 -38.823 -21.071  1.00  0.32           O  \\nATOM   1224  OG  SER A 165       8.433 -37.829 -22.683  1.00  0.35           O  \\nATOM   1225  N   GLY A 166       4.866 -38.682 -21.237  1.00  0.32           N  \\nATOM   1226  CA  GLY A 166       4.498 -39.824 -20.415  1.00  0.37           C  \\nATOM   1227  C   GLY A 166       3.216 -40.497 -20.868  1.00  0.33           C  \\nATOM   1228  O   GLY A 166       2.119 -40.021 -20.568  1.00  0.32           O  \\nATOM   1229  N   GLY A 167       3.302 -41.182 -22.045  1.00  0.32           N  \\nATOM   1230  CA  GLY A 167       2.648 -42.478 -22.137  1.00  0.36           C  \\nATOM   1231  C   GLY A 167       2.749 -43.100 -23.517  1.00  0.32           C  \\nATOM   1232  O   GLY A 167       2.384 -42.474 -24.514  1.00  0.31           O  \\nATOM   1233  N   GLY A 168       3.793 -43.763 -23.649  1.00  0.31           N  \\nATOM   1234  CA  GLY A 168       3.575 -45.148 -24.033  1.00  0.36           C  \\nATOM   1235  C   GLY A 168       3.069 -45.301 -25.454  1.00  0.32           C  \\nATOM   1236  O   GLY A 168       2.021 -44.756 -25.808  1.00  0.31           O  \\nATOM   1237  N   GLY A 169       4.018 -45.174 -26.439  1.00  0.31           N  \\nATOM   1238  CA  GLY A 169       3.898 -46.310 -27.339  1.00  0.36           C  \\nATOM   1239  C   GLY A 169       3.334 -45.938 -28.698  1.00  0.31           C  \\nATOM   1240  O   GLY A 169       2.189 -45.492 -28.799  1.00  0.31           O  \\nATOM   1241  N   GLY A 170       4.210 -45.298 -29.630  1.00  0.32           N  \\nATOM   1242  CA  GLY A 170       4.171 -45.958 -30.925  1.00  0.37           C  \\nATOM   1243  C   GLY A 170       4.125 -44.985 -32.089  1.00  0.32           C  \\nATOM   1244  O   GLY A 170       3.210 -44.164 -32.182  1.00  0.32           O  \\nATOM   1245  N   GLY A 171       5.298 -44.444 -32.539  1.00  0.32           N  \\nATOM   1246  CA  GLY A 171       5.670 -44.626 -33.933  1.00  0.38           C  \\nATOM   1247  C   GLY A 171       5.543 -43.356 -34.755  1.00  0.33           C  \\nATOM   1248  O   GLY A 171       4.457 -42.783 -34.857  1.00  0.33           O  \\nATOM   1249  N   GLY A 172       6.642 -42.435 -34.787  1.00  0.32           N  \\nATOM   1250  CA  GLY A 172       7.122 -42.122 -36.123  1.00  0.39           C  \\nATOM   1251  C   GLY A 172       6.674 -40.759 -36.615  1.00  0.32           C  \\nATOM   1252  O   GLY A 172       5.497 -40.561 -36.926  1.00  0.33           O  \\nATOM   1253  N   GLY A 173       7.413 -39.638 -36.117  1.00  0.35           N  \\nATOM   1254  CA  GLY A 173       7.934 -38.561 -36.943  1.00  0.38           C  \\nATOM   1255  C   GLY A 173       8.295 -37.321 -36.149  1.00  0.35           C  \\nATOM   1256  O   GLY A 173       7.443 -36.741 -35.472  1.00  0.35           O  \\nATOM   1257  N   GLY A 174       9.544 -37.288 -35.551  1.00  0.35           N  \\nATOM   1258  CA  GLY A 174      10.527 -36.217 -35.536  1.00  0.38           C  \\nATOM   1259  C   GLY A 174      10.035 -34.946 -36.203  1.00  0.36           C  \\nATOM   1260  O   GLY A 174      10.097 -34.818 -37.427  1.00  0.35           O  \\nATOM   1261  N   GLY A 175       9.086 -34.293 -35.879  1.00  0.35           N  \\nATOM   1262  CA  GLY A 175       9.001 -32.905 -36.302  1.00  0.38           C  \\nATOM   1263  C   GLY A 175       9.478 -31.928 -35.244  1.00  0.36           C  \\nATOM   1264  O   GLY A 175       9.029 -31.978 -34.097  1.00  0.36           O  \\nATOM   1265  N   ASN A 176      10.859 -31.753 -35.239  1.00  0.36           N  \\nATOM   1266  CA  ASN A 176      11.498 -30.475 -34.945  1.00  0.40           C  \\nATOM   1267  C   ASN A 176      10.476 -29.416 -34.541  1.00  0.36           C  \\nATOM   1268  CB  ASN A 176      12.314 -29.994 -36.147  1.00  0.35           C  \\nATOM   1269  O   ASN A 176       9.735 -28.909 -35.384  1.00  0.36           O  \\nATOM   1270  CG  ASN A 176      13.387 -28.994 -35.763  1.00  0.36           C  \\nATOM   1271  ND2 ASN A 176      13.885 -28.252 -36.744  1.00  0.29           N  \\nATOM   1272  OD1 ASN A 176      13.766 -28.891 -34.593  1.00  0.37           O  \\nATOM   1273  N   TYR A 177       9.926 -29.515 -33.473  1.00  0.39           N  \\nATOM   1274  CA  TYR A 177       9.386 -28.289 -32.895  1.00  0.43           C  \\nATOM   1275  C   TYR A 177      10.501 -27.299 -32.578  1.00  0.39           C  \\nATOM   1276  CB  TYR A 177       8.587 -28.599 -31.626  1.00  0.38           C  \\nATOM   1277  O   TYR A 177      11.290 -27.517 -31.656  1.00  0.39           O  \\nATOM   1278  CG  TYR A 177       7.258 -29.263 -31.892  1.00  0.39           C  \\nATOM   1279  CD1 TYR A 177       6.134 -28.509 -32.220  1.00  0.39           C  \\nATOM   1280  CD2 TYR A 177       7.124 -30.645 -31.816  1.00  0.40           C  \\nATOM   1281  CE1 TYR A 177       4.907 -29.117 -32.464  1.00  0.38           C  \\nATOM   1282  CE2 TYR A 177       5.902 -31.264 -32.058  1.00  0.42           C  \\nATOM   1283  OH  TYR A 177       3.589 -31.099 -32.623  1.00  0.27           O  \\nATOM   1284  CZ  TYR A 177       4.801 -30.493 -32.382  1.00  0.36           C  \\nATOM   1285  N   GLY A 178      11.214 -26.809 -33.554  1.00  0.33           N  \\nATOM   1286  CA  GLY A 178      11.924 -25.540 -33.563  1.00  0.35           C  \\nATOM   1287  C   GLY A 178      11.320 -24.512 -32.625  1.00  0.34           C  \\nATOM   1288  O   GLY A 178      10.142 -24.170 -32.746  1.00  0.33           O  \\nATOM   1289  N   GLN A 179      11.529 -24.673 -31.429  1.00  0.40           N  \\nATOM   1290  CA  GLN A 179      11.717 -23.533 -30.537  1.00  0.41           C  \\nATOM   1291  C   GLN A 179      12.545 -22.440 -31.208  1.00  0.40           C  \\nATOM   1292  CB  GLN A 179      12.387 -23.974 -29.235  1.00  0.37           C  \\nATOM   1293  O   GLN A 179      13.758 -22.588 -31.375  1.00  0.39           O  \\nATOM   1294  CG  GLN A 179      11.418 -24.146 -28.073  1.00  0.37           C  \\nATOM   1295  CD  GLN A 179      12.101 -24.623 -26.805  1.00  0.44           C  \\nATOM   1296  NE2 GLN A 179      11.347 -24.685 -25.713  1.00  0.36           N  \\nATOM   1297  OE1 GLN A 179      13.296 -24.935 -26.807  1.00  0.48           O  \\nATOM   1298  N   ASP A 180      11.879 -21.802 -32.245  1.00  0.34           N  \\nATOM   1299  CA  ASP A 180      12.266 -20.536 -32.860  1.00  0.37           C  \\nATOM   1300  C   ASP A 180      11.632 -19.354 -32.129  1.00  0.35           C  \\nATOM   1301  CB  ASP A 180      11.872 -20.514 -34.338  1.00  0.33           C  \\nATOM   1302  O   ASP A 180      10.420 -19.143 -32.214  1.00  0.33           O  \\nATOM   1303  CG  ASP A 180      12.979 -21.004 -35.254  1.00  0.34           C  \\nATOM   1304  OD1 ASP A 180      14.167 -20.922 -34.874  1.00  0.37           O  \\nATOM   1305  OD2 ASP A 180      12.661 -21.473 -36.368  1.00  0.38           O  \\nATOM   1306  N   GLN A 181      11.870 -19.064 -30.872  1.00  0.39           N  \\nATOM   1307  CA  GLN A 181      12.140 -17.712 -30.396  1.00  0.42           C  \\nATOM   1308  C   GLN A 181      13.259 -17.058 -31.201  1.00  0.39           C  \\nATOM   1309  CB  GLN A 181      12.503 -17.728 -28.910  1.00  0.38           C  \\nATOM   1310  O   GLN A 181      14.439 -17.231 -30.888  1.00  0.38           O  \\nATOM   1311  CG  GLN A 181      11.296 -17.737 -27.982  1.00  0.40           C  \\nATOM   1312  CD  GLN A 181      11.679 -17.624 -26.518  1.00  0.48           C  \\nATOM   1313  NE2 GLN A 181      10.711 -17.833 -25.633  1.00  0.40           N  \\nATOM   1314  OE1 GLN A 181      12.837 -17.352 -26.186  1.00  0.51           O  \\nATOM   1315  N   SER A 182      13.071 -17.146 -32.636  1.00  0.32           N  \\nATOM   1316  CA  SER A 182      13.715 -16.497 -33.774  1.00  0.35           C  \\nATOM   1317  C   SER A 182      13.200 -17.060 -35.095  1.00  0.32           C  \\nATOM   1318  CB  SER A 182      15.233 -16.662 -33.698  1.00  0.30           C  \\nATOM   1319  O   SER A 182      13.425 -18.231 -35.406  1.00  0.31           O  \\nATOM   1320  OG  SER A 182      15.580 -18.012 -33.441  1.00  0.29           O  \\nATOM   1321  N   SER A 183      11.825 -16.959 -35.358  1.00  0.30           N  \\nATOM   1322  CA  SER A 183      11.504 -16.850 -36.778  1.00  0.33           C  \\nATOM   1323  C   SER A 183      10.013 -16.610 -36.990  1.00  0.31           C  \\nATOM   1324  CB  SER A 183      11.937 -18.113 -37.524  1.00  0.28           C  \\nATOM   1325  O   SER A 183       9.185 -17.438 -36.606  1.00  0.29           O  \\nATOM   1326  OG  SER A 183      11.393 -19.270 -36.912  1.00  0.21           O  \\nATOM   1327  N   MET A 184       9.528 -15.370 -36.906  1.00  0.34           N  \\nATOM   1328  CA  MET A 184       9.050 -15.047 -38.247  1.00  0.37           C  \\nATOM   1329  C   MET A 184       9.549 -16.068 -39.264  1.00  0.35           C  \\nATOM   1330  CB  MET A 184       9.497 -13.642 -38.654  1.00  0.33           C  \\nATOM   1331  O   MET A 184      10.696 -15.996 -39.711  1.00  0.33           O  \\nATOM   1332  CG  MET A 184       8.661 -12.531 -38.040  1.00  0.35           C  \\nATOM   1333  SD  MET A 184       9.186 -10.864 -38.601  1.00  0.28           S  \\nATOM   1334  CE  MET A 184       7.819  -9.863 -37.952  1.00  0.36           C  \\nATOM   1335  N   SER A 185       9.339 -17.335 -38.958  1.00  0.30           N  \\nATOM   1336  CA  SER A 185       9.265 -18.073 -40.214  1.00  0.33           C  \\nATOM   1337  C   SER A 185       7.820 -18.242 -40.672  1.00  0.31           C  \\nATOM   1338  CB  SER A 185       9.926 -19.445 -40.071  1.00  0.29           C  \\nATOM   1339  O   SER A 185       6.979 -18.740 -39.920  1.00  0.29           O  \\nATOM   1340  OG  SER A 185       8.955 -20.477 -40.111  1.00  0.19           O  \\nATOM   1341  N   SER A 186       7.326 -17.244 -41.318  1.00  0.32           N  \\nATOM   1342  CA  SER A 186       6.815 -17.631 -42.629  1.00  0.34           C  \\nATOM   1343  C   SER A 186       6.980 -19.128 -42.868  1.00  0.32           C  \\nATOM   1344  CB  SER A 186       7.527 -16.849 -43.734  1.00  0.31           C  \\nATOM   1345  O   SER A 186       8.066 -19.590 -43.224  1.00  0.31           O  \\nATOM   1346  OG  SER A 186       8.890 -17.229 -43.821  1.00  0.35           O  \\nATOM   1347  N   GLY A 187       6.579 -19.976 -42.011  1.00  0.29           N  \\nATOM   1348  CA  GLY A 187       6.230 -21.283 -42.543  1.00  0.33           C  \\nATOM   1349  C   GLY A 187       4.738 -21.467 -42.742  1.00  0.30           C  \\nATOM   1350  O   GLY A 187       3.965 -21.404 -41.784  1.00  0.28           O  \\nATOM   1351  N   GLY A 188       4.337 -20.923 -43.803  1.00  0.31           N  \\nATOM   1352  CA  GLY A 188       3.511 -21.649 -44.755  1.00  0.33           C  \\nATOM   1353  C   GLY A 188       2.971 -22.955 -44.202  1.00  0.32           C  \\nATOM   1354  O   GLY A 188       3.688 -23.957 -44.150  1.00  0.31           O  \\nATOM   1355  N   GLY A 189       2.148 -22.920 -43.139  1.00  0.31           N  \\nATOM   1356  CA  GLY A 189       1.140 -23.968 -43.099  1.00  0.34           C  \\nATOM   1357  C   GLY A 189      -0.266 -23.435 -42.894  1.00  0.32           C  \\nATOM   1358  O   GLY A 189      -0.518 -22.679 -41.953  1.00  0.31           O  \\nATOM   1359  N   SER A 190      -1.040 -22.983 -43.849  1.00  0.32           N  \\nATOM   1360  CA  SER A 190      -2.410 -23.178 -44.312  1.00  0.35           C  \\nATOM   1361  C   SER A 190      -3.340 -23.528 -43.156  1.00  0.32           C  \\nATOM   1362  CB  SER A 190      -2.466 -24.278 -45.373  1.00  0.31           C  \\nATOM   1363  O   SER A 190      -3.166 -24.558 -42.502  1.00  0.31           O  \\nATOM   1364  OG  SER A 190      -3.747 -24.332 -45.976  1.00  0.34           O  \\nATOM   1365  N   GLY A 191      -3.800 -22.439 -42.410  1.00  0.33           N  \\nATOM   1366  CA  GLY A 191      -5.089 -22.676 -41.779  1.00  0.35           C  \\nATOM   1367  C   GLY A 191      -5.490 -21.578 -40.812  1.00  0.34           C  \\nATOM   1368  O   GLY A 191      -4.811 -21.348 -39.809  1.00  0.32           O  \\nATOM   1369  N   GLY A 192      -6.149 -20.415 -41.272  1.00  0.32           N  \\nATOM   1370  CA  GLY A 192      -7.291 -19.530 -41.436  1.00  0.36           C  \\nATOM   1371  C   GLY A 192      -8.149 -19.425 -40.190  1.00  0.33           C  \\nATOM   1372  O   GLY A 192      -8.709 -20.422 -39.729  1.00  0.32           O  \\nATOM   1373  N   GLY A 193      -7.904 -18.388 -39.337  1.00  0.34           N  \\nATOM   1374  CA  GLY A 193      -9.067 -17.591 -38.978  1.00  0.39           C  \\nATOM   1375  C   GLY A 193      -9.151 -17.296 -37.492  1.00  0.34           C  \\nATOM   1376  O   GLY A 193      -9.295 -18.212 -36.679  1.00  0.34           O  \\nATOM   1377  N   TYR A 194      -8.487 -16.161 -37.050  1.00  0.45           N  \\nATOM   1378  CA  TYR A 194      -9.020 -14.984 -36.373  1.00  0.48           C  \\nATOM   1379  C   TYR A 194      -8.060 -13.806 -36.491  1.00  0.44           C  \\nATOM   1380  CB  TYR A 194      -9.292 -15.289 -34.897  1.00  0.41           C  \\nATOM   1381  O   TYR A 194      -6.895 -13.904 -36.098  1.00  0.43           O  \\nATOM   1382  CG  TYR A 194     -10.486 -16.185 -34.673  1.00  0.41           C  \\nATOM   1383  CD1 TYR A 194     -11.779 -15.667 -34.672  1.00  0.41           C  \\nATOM   1384  CD2 TYR A 194     -10.323 -17.550 -34.460  1.00  0.42           C  \\nATOM   1385  CE1 TYR A 194     -12.882 -16.488 -34.463  1.00  0.41           C  \\nATOM   1386  CE2 TYR A 194     -11.420 -18.380 -34.251  1.00  0.44           C  \\nATOM   1387  OH  TYR A 194     -13.781 -18.659 -34.048  1.00  0.30           O  \\nATOM   1388  CZ  TYR A 194     -12.693 -17.841 -34.254  1.00  0.37           C  \\nATOM   1389  N   GLY A 195      -8.347 -12.916 -37.525  1.00  0.38           N  \\nATOM   1390  CA  GLY A 195      -8.142 -11.520 -37.876  1.00  0.40           C  \\nATOM   1391  C   GLY A 195      -7.304 -10.766 -36.861  1.00  0.38           C  \\nATOM   1392  O   GLY A 195      -7.704 -10.619 -35.705  1.00  0.37           O  \\nATOM   1393  N   ASN A 196      -6.009 -10.931 -36.917  1.00  0.42           N  \\nATOM   1394  CA  ASN A 196      -4.937  -9.947 -36.804  1.00  0.46           C  \\nATOM   1395  C   ASN A 196      -5.380  -8.576 -37.305  1.00  0.42           C  \\nATOM   1396  CB  ASN A 196      -3.696 -10.416 -37.568  1.00  0.40           C  \\nATOM   1397  O   ASN A 196      -5.525  -8.367 -38.510  1.00  0.42           O  \\nATOM   1398  CG  ASN A 196      -2.437 -10.360 -36.726  1.00  0.41           C  \\nATOM   1399  ND2 ASN A 196      -1.286 -10.520 -37.369  1.00  0.36           N  \\nATOM   1400  OD1 ASN A 196      -2.498 -10.177 -35.508  1.00  0.42           O  \\nATOM   1401  N   GLN A 197      -6.045  -7.813 -36.477  1.00  0.43           N  \\nATOM   1402  CA  GLN A 197      -6.219  -6.381 -36.693  1.00  0.47           C  \\nATOM   1403  C   GLN A 197      -5.226  -5.574 -35.861  1.00  0.43           C  \\nATOM   1404  CB  GLN A 197      -7.650  -5.956 -36.359  1.00  0.40           C  \\nATOM   1405  O   GLN A 197      -5.179  -5.711 -34.637  1.00  0.42           O  \\nATOM   1406  CG  GLN A 197      -8.545  -5.800 -37.581  1.00  0.41           C  \\nATOM   1407  CD  GLN A 197      -9.959  -5.382 -37.222  1.00  0.48           C  \\nATOM   1408  NE2 GLN A 197     -10.840  -5.354 -38.217  1.00  0.38           N  \\nATOM   1409  OE1 GLN A 197     -10.258  -5.089 -36.060  1.00  0.51           O  \\nATOM   1410  N   ASP A 198      -3.886  -5.315 -36.441  1.00  0.44           N  \\nATOM   1411  CA  ASP A 198      -3.156  -4.063 -36.611  1.00  0.46           C  \\nATOM   1412  C   ASP A 198      -1.663  -4.258 -36.355  1.00  0.44           C  \\nATOM   1413  CB  ASP A 198      -3.713  -2.985 -35.678  1.00  0.40           C  \\nATOM   1414  O   ASP A 198      -1.240  -4.416 -35.207  1.00  0.42           O  \\nATOM   1415  CG  ASP A 198      -4.895  -2.240 -36.273  1.00  0.40           C  \\nATOM   1416  OD1 ASP A 198      -4.981  -2.124 -37.514  1.00  0.44           O  \\nATOM   1417  OD2 ASP A 198      -5.746  -1.762 -35.492  1.00  0.46           O  \\nATOM   1418  N   GLN A 199      -0.985  -4.802 -37.387  1.00  0.43           N  \\nATOM   1419  CA  GLN A 199       0.419  -4.701 -37.773  1.00  0.45           C  \\nATOM   1420  C   GLN A 199       0.676  -3.436 -38.586  1.00  0.43           C  \\nATOM   1421  CB  GLN A 199       0.847  -5.934 -38.570  1.00  0.40           C  \\nATOM   1422  O   GLN A 199       0.271  -3.345 -39.747  1.00  0.42           O  \\nATOM   1423  CG  GLN A 199       1.119  -7.160 -37.709  1.00  0.40           C  \\nATOM   1424  CD  GLN A 199       1.585  -8.355 -38.518  1.00  0.46           C  \\nATOM   1425  NE2 GLN A 199       1.311  -9.554 -38.017  1.00  0.39           N  \\nATOM   1426  OE1 GLN A 199       2.187  -8.201 -39.586  1.00  0.51           O  \\nATOM   1427  N   SER A 200       0.806  -2.276 -37.982  1.00  0.38           N  \\nATOM   1428  CA  SER A 200       1.519  -1.054 -38.339  1.00  0.42           C  \\nATOM   1429  C   SER A 200       3.024  -1.290 -38.403  1.00  0.38           C  \\nATOM   1430  CB  SER A 200       1.212   0.058 -37.335  1.00  0.36           C  \\nATOM   1431  O   SER A 200       3.652  -1.610 -37.391  1.00  0.37           O  \\nATOM   1432  OG  SER A 200       1.999  -0.088 -36.166  1.00  0.39           O  \\nATOM   1433  N   GLY A 201       3.488  -1.930 -39.475  1.00  0.34           N  \\nATOM   1434  CA  GLY A 201       4.789  -1.509 -39.970  1.00  0.38           C  \\nATOM   1435  C   GLY A 201       5.465  -2.556 -40.835  1.00  0.35           C  \\nATOM   1436  O   GLY A 201       5.713  -3.676 -40.383  1.00  0.33           O  \\nATOM   1437  N   GLY A 202       5.219  -2.516 -42.090  1.00  0.34           N  \\nATOM   1438  CA  GLY A 202       6.178  -2.502 -43.183  1.00  0.37           C  \\nATOM   1439  C   GLY A 202       6.022  -3.678 -44.129  1.00  0.34           C  \\nATOM   1440  O   GLY A 202       6.063  -4.834 -43.703  1.00  0.33           O  \\nATOM   1441  N   GLY A 203       5.217  -3.544 -45.132  1.00  0.34           N  \\nATOM   1442  CA  GLY A 203       5.649  -3.847 -46.488  1.00  0.38           C  \\nATOM   1443  C   GLY A 203       5.958  -5.317 -46.700  1.00  0.35           C  \\nATOM   1444  O   GLY A 203       7.029  -5.793 -46.319  1.00  0.34           O  \\nATOM   1445  N   GLY A 204       4.855  -6.265 -46.711  1.00  0.36           N  \\nATOM   1446  CA  GLY A 204       4.929  -7.269 -47.760  1.00  0.39           C  \\nATOM   1447  C   GLY A 204       3.568  -7.676 -48.294  1.00  0.37           C  \\nATOM   1448  O   GLY A 204       2.703  -8.113 -47.532  1.00  0.36           O  \\nATOM   1449  N   SER A 205       2.973  -7.032 -49.297  1.00  0.33           N  \\nATOM   1450  CA  SER A 205       2.330  -7.459 -50.535  1.00  0.36           C  \\nATOM   1451  C   SER A 205       2.481  -8.961 -50.746  1.00  0.33           C  \\nATOM   1452  CB  SER A 205       2.916  -6.706 -51.731  1.00  0.32           C  \\nATOM   1453  O   SER A 205       3.587  -9.453 -50.979  1.00  0.33           O  \\nATOM   1454  OG  SER A 205       2.315  -7.137 -52.940  1.00  0.38           O  \\nATOM   1455  N   GLY A 206       1.709  -9.817 -49.981  1.00  0.36           N  \\nATOM   1456  CA  GLY A 206       1.404 -11.084 -50.626  1.00  0.39           C  \\nATOM   1457  C   GLY A 206       0.062 -11.657 -50.210  1.00  0.37           C  \\nATOM   1458  O   GLY A 206      -0.178 -11.890 -49.023  1.00  0.35           O  \\nATOM   1459  N   GLY A 207      -1.060 -11.422 -51.000  1.00  0.37           N  \\nATOM   1460  CA  GLY A 207      -2.160 -12.105 -51.661  1.00  0.39           C  \\nATOM   1461  C   GLY A 207      -2.276 -13.567 -51.270  1.00  0.37           C  \\nATOM   1462  O   GLY A 207      -1.425 -14.381 -51.634  1.00  0.36           O  \\nATOM   1463  N   TYR A 208      -2.792 -13.976 -50.122  1.00  0.40           N  \\nATOM   1464  CA  TYR A 208      -3.193 -15.363 -49.915  1.00  0.44           C  \\nATOM   1465  C   TYR A 208      -4.645 -15.448 -49.460  1.00  0.40           C  \\nATOM   1466  CB  TYR A 208      -2.282 -16.038 -48.886  1.00  0.37           C  \\nATOM   1467  O   TYR A 208      -5.029 -14.821 -48.470  1.00  0.39           O  \\nATOM   1468  CG  TYR A 208      -0.909 -16.373 -49.417  1.00  0.38           C  \\nATOM   1469  CD1 TYR A 208      -0.696 -17.520 -50.178  1.00  0.36           C  \\nATOM   1470  CD2 TYR A 208       0.177 -15.543 -49.159  1.00  0.38           C  \\nATOM   1471  CE1 TYR A 208       0.567 -17.834 -50.668  1.00  0.38           C  \\nATOM   1472  CE2 TYR A 208       1.445 -15.847 -49.644  1.00  0.40           C  \\nATOM   1473  OH  TYR A 208       2.883 -17.297 -50.879  1.00  0.29           O  \\nATOM   1474  CZ  TYR A 208       1.630 -16.992 -50.397  1.00  0.34           C  \\nATOM   1475  N   GLY A 209      -5.545 -15.865 -50.411  1.00  0.40           N  \\nATOM   1476  CA  GLY A 209      -6.708 -16.600 -50.882  1.00  0.41           C  \\nATOM   1477  C   GLY A 209      -7.395 -17.395 -49.788  1.00  0.40           C  \\nATOM   1478  O   GLY A 209      -6.771 -18.238 -49.140  1.00  0.39           O  \\nATOM   1479  N   GLN A 210      -8.407 -16.954 -49.096  1.00  0.44           N  \\nATOM   1480  CA  GLN A 210      -9.685 -17.438 -48.585  1.00  0.46           C  \\nATOM   1481  C   GLN A 210     -10.099 -18.733 -49.277  1.00  0.44           C  \\nATOM   1482  CB  GLN A 210     -10.772 -16.377 -48.762  1.00  0.42           C  \\nATOM   1483  O   GLN A 210     -10.428 -18.730 -50.466  1.00  0.43           O  \\nATOM   1484  CG  GLN A 210     -11.734 -16.280 -47.586  1.00  0.43           C  \\nATOM   1485  CD  GLN A 210     -12.697 -15.115 -47.710  1.00  0.49           C  \\nATOM   1486  NE2 GLN A 210     -12.413 -14.207 -48.638  1.00  0.40           N  \\nATOM   1487  OE1 GLN A 210     -13.689 -15.031 -46.978  1.00  0.54           O  \\nATOM   1488  N   GLN A 211      -9.788 -19.954 -48.635  1.00  0.46           N  \\nATOM   1489  CA  GLN A 211     -10.548 -21.167 -48.915  1.00  0.48           C  \\nATOM   1490  C   GLN A 211     -11.267 -21.667 -47.665  1.00  0.46           C  \\nATOM   1491  CB  GLN A 211      -9.631 -22.260 -49.466  1.00  0.43           C  \\nATOM   1492  O   GLN A 211     -10.625 -22.064 -46.691  1.00  0.44           O  \\nATOM   1493  CG  GLN A 211      -9.774 -22.485 -50.966  1.00  0.42           C  \\nATOM   1494  CD  GLN A 211      -8.900 -23.615 -51.476  1.00  0.48           C  \\nATOM   1495  NE2 GLN A 211      -7.593 -23.493 -51.271  1.00  0.40           N  \\nATOM   1496  OE1 GLN A 211      -9.395 -24.591 -52.048  1.00  0.52           O  \\nATOM   1497  N   ASP A 212     -12.565 -21.434 -47.498  1.00  0.46           N  \\nATOM   1498  CA  ASP A 212     -13.808 -22.088 -47.100  1.00  0.47           C  \\nATOM   1499  C   ASP A 212     -13.849 -23.534 -47.587  1.00  0.46           C  \\nATOM   1500  CB  ASP A 212     -15.016 -21.317 -47.637  1.00  0.43           C  \\nATOM   1501  O   ASP A 212     -14.017 -23.787 -48.782  1.00  0.44           O  \\nATOM   1502  CG  ASP A 212     -15.726 -20.506 -46.568  1.00  0.43           C  \\nATOM   1503  OD1 ASP A 212     -15.590 -20.828 -45.367  1.00  0.46           O  \\nATOM   1504  OD2 ASP A 212     -16.431 -19.539 -46.929  1.00  0.54           O  \\nATOM   1505  N   ARG A 213     -13.129 -24.415 -46.920  1.00  0.44           N  \\nATOM   1506  CA  ARG A 213     -13.201 -25.849 -47.183  1.00  0.44           C  \\nATOM   1507  C   ARG A 213     -14.294 -26.505 -46.346  1.00  0.44           C  \\nATOM   1508  CB  ARG A 213     -11.853 -26.516 -46.900  1.00  0.41           C  \\nATOM   1509  O   ARG A 213     -14.191 -26.567 -45.119  1.00  0.42           O  \\nATOM   1510  CG  ARG A 213     -10.834 -26.343 -48.015  1.00  0.41           C  \\nATOM   1511  CD  ARG A 213      -9.546 -27.100 -47.724  1.00  0.46           C  \\nATOM   1512  NE  ARG A 213      -8.588 -26.974 -48.819  1.00  0.39           N  \\nATOM   1513  NH1 ARG A 213      -6.882 -28.089 -47.729  1.00  0.29           N  \\nATOM   1514  NH2 ARG A 213      -6.557 -27.276 -49.848  1.00  0.24           N  \\nATOM   1515  CZ  ARG A 213      -7.344 -27.447 -48.796  1.00  0.36           C  \\nATOM   1516  N   GLY A 214     -15.573 -26.578 -46.665  1.00  0.40           N  \\nATOM   1517  CA  GLY A 214     -16.517 -27.664 -46.876  1.00  0.41           C  \\nATOM   1518  C   GLY A 214     -16.435 -28.741 -45.810  1.00  0.41           C  \\nATOM   1519  O   GLY A 214     -15.532 -29.580 -45.836  1.00  0.39           O  \\nATOM   1520  N   GLY A 215     -16.869 -28.469 -44.549  1.00  0.42           N  \\nATOM   1521  CA  GLY A 215     -17.317 -29.701 -43.919  1.00  0.44           C  \\nATOM   1522  C   GLY A 215     -18.805 -29.714 -43.624  1.00  0.42           C  \\nATOM   1523  O   GLY A 215     -19.299 -28.880 -42.863  1.00  0.41           O  \\nATOM   1524  N   ARG A 216     -19.671 -29.991 -44.518  1.00  0.42           N  \\nATOM   1525  CA  ARG A 216     -20.872 -30.819 -44.490  1.00  0.42           C  \\nATOM   1526  C   ARG A 216     -20.730 -31.955 -43.482  1.00  0.42           C  \\nATOM   1527  CB  ARG A 216     -21.167 -31.387 -45.880  1.00  0.39           C  \\nATOM   1528  O   ARG A 216     -20.042 -32.943 -43.744  1.00  0.40           O  \\nATOM   1529  CG  ARG A 216     -22.032 -30.482 -46.743  1.00  0.40           C  \\nATOM   1530  CD  ARG A 216     -22.357 -31.126 -48.084  1.00  0.47           C  \\nATOM   1531  NE  ARG A 216     -23.150 -30.238 -48.930  1.00  0.38           N  \\nATOM   1532  NH1 ARG A 216     -23.311 -31.722 -50.693  1.00  0.28           N  \\nATOM   1533  NH2 ARG A 216     -24.299 -29.658 -50.832  1.00  0.23           N  \\nATOM   1534  CZ  ARG A 216     -23.585 -30.541 -50.150  1.00  0.35           C  \\nATOM   1535  N   GLY A 217     -20.926 -31.775 -42.108  1.00  0.41           N  \\nATOM   1536  CA  GLY A 217     -21.472 -32.887 -41.347  1.00  0.43           C  \\nATOM   1537  C   GLY A 217     -22.854 -32.605 -40.787  1.00  0.41           C  \\nATOM   1538  O   GLY A 217     -23.061 -31.595 -40.111  1.00  0.40           O  \\nATOM   1539  N   ARG A 218     -23.910 -32.825 -41.519  1.00  0.34           N  \\nATOM   1540  CA  ARG A 218     -25.147 -33.565 -41.294  1.00  0.36           C  \\nATOM   1541  C   ARG A 218     -25.158 -34.207 -39.911  1.00  0.34           C  \\nATOM   1542  CB  ARG A 218     -25.337 -34.637 -42.370  1.00  0.32           C  \\nATOM   1543  O   ARG A 218     -24.260 -34.981 -39.573  1.00  0.33           O  \\nATOM   1544  CG  ARG A 218     -26.547 -34.409 -43.261  1.00  0.35           C  \\nATOM   1545  CD  ARG A 218     -26.646 -35.459 -44.358  1.00  0.42           C  \\nATOM   1546  NE  ARG A 218     -27.809 -35.238 -45.213  1.00  0.37           N  \\nATOM   1547  NH1 ARG A 218     -27.547 -37.186 -46.426  1.00  0.28           N  \\nATOM   1548  NH2 ARG A 218     -29.283 -35.764 -46.894  1.00  0.23           N  \\nATOM   1549  CZ  ARG A 218     -28.211 -36.063 -46.175  1.00  0.32           C  \\nATOM   1550  N   GLY A 219     -25.695 -33.471 -38.719  1.00  0.34           N  \\nATOM   1551  CA  GLY A 219     -26.578 -34.233 -37.851  1.00  0.39           C  \\nATOM   1552  C   GLY A 219     -26.028 -34.413 -36.449  1.00  0.35           C  \\nATOM   1553  O   GLY A 219     -25.059 -35.149 -36.246  1.00  0.34           O  \\nATOM   1554  N   GLY A 220     -26.065 -33.413 -35.526  1.00  0.36           N  \\nATOM   1555  CA  GLY A 220     -26.675 -33.448 -34.207  1.00  0.38           C  \\nATOM   1556  C   GLY A 220     -26.971 -32.068 -33.649  1.00  0.37           C  \\nATOM   1557  O   GLY A 220     -26.072 -31.232 -33.538  1.00  0.36           O  \\nATOM   1558  N   SER A 221     -27.922 -31.295 -34.172  1.00  0.32           N  \\nATOM   1559  CA  SER A 221     -28.970 -30.471 -33.579  1.00  0.36           C  \\nATOM   1560  C   SER A 221     -28.971 -30.590 -32.058  1.00  0.32           C  \\nATOM   1561  CB  SER A 221     -30.340 -30.868 -34.130  1.00  0.31           C  \\nATOM   1562  O   SER A 221     -29.190 -31.674 -31.515  1.00  0.32           O  \\nATOM   1563  OG  SER A 221     -31.335 -29.947 -33.716  1.00  0.33           O  \\nATOM   1564  N   GLY A 222     -27.934 -30.186 -31.304  1.00  0.35           N  \\nATOM   1565  CA  GLY A 222     -28.374 -29.725 -29.997  1.00  0.37           C  \\nATOM   1566  C   GLY A 222     -27.262 -29.700 -28.967  1.00  0.35           C  \\nATOM   1567  O   GLY A 222     -27.499 -29.395 -27.796  1.00  0.34           O  \\nATOM   1568  N   GLY A 223     -25.872 -29.765 -29.399  1.00  0.35           N  \\nATOM   1569  CA  GLY A 223     -24.897 -29.739 -28.321  1.00  0.37           C  \\nATOM   1570  C   GLY A 223     -23.644 -28.958 -28.668  1.00  0.36           C  \\nATOM   1571  O   GLY A 223     -22.814 -28.688 -27.798  1.00  0.33           O  \\nATOM   1572  N   GLY A 224     -23.473 -28.590 -30.009  1.00  0.35           N  \\nATOM   1573  CA  GLY A 224     -22.159 -28.058 -30.335  1.00  0.37           C  \\nATOM   1574  C   GLY A 224     -21.977 -26.615 -29.905  1.00  0.36           C  \\nATOM   1575  O   GLY A 224     -20.869 -26.201 -29.555  1.00  0.34           O  \\nATOM   1576  N   GLY A 225     -23.201 -25.847 -29.824  1.00  0.38           N  \\nATOM   1577  CA  GLY A 225     -23.105 -24.453 -29.421  1.00  0.40           C  \\nATOM   1578  C   GLY A 225     -22.743 -24.279 -27.958  1.00  0.39           C  \\nATOM   1579  O   GLY A 225     -21.887 -23.458 -27.618  1.00  0.37           O  \\nATOM   1580  N   GLN A 226     -23.310 -25.227 -27.134  1.00  0.40           N  \\nATOM   1581  CA  GLN A 226     -23.022 -25.166 -25.705  1.00  0.42           C  \\nATOM   1582  C   GLN A 226     -21.571 -25.544 -25.420  1.00  0.40           C  \\nATOM   1583  CB  GLN A 226     -23.966 -26.083 -24.927  1.00  0.39           C  \\nATOM   1584  O   GLN A 226     -20.898 -24.887 -24.623  1.00  0.40           O  \\nATOM   1585  CG  GLN A 226     -25.207 -25.379 -24.395  1.00  0.38           C  \\nATOM   1586  CD  GLN A 226     -26.084 -26.290 -23.556  1.00  0.44           C  \\nATOM   1587  NE2 GLN A 226     -27.025 -25.700 -22.827  1.00  0.36           N  \\nATOM   1588  OE1 GLN A 226     -25.919 -27.514 -23.565  1.00  0.46           O  \\nATOM   1589  N   GLN A 227     -21.061 -26.636 -26.113  1.00  0.45           N  \\nATOM   1590  CA  GLN A 227     -19.679 -27.038 -25.876  1.00  0.45           C  \\nATOM   1591  C   GLN A 227     -18.707 -25.939 -26.296  1.00  0.45           C  \\nATOM   1592  CB  GLN A 227     -19.361 -28.334 -26.623  1.00  0.43           C  \\nATOM   1593  O   GLN A 227     -17.763 -25.626 -25.568  1.00  0.44           O  \\nATOM   1594  CG  GLN A 227     -19.487 -29.586 -25.765  1.00  0.42           C  \\nATOM   1595  CD  GLN A 227     -19.148 -30.855 -26.524  1.00  0.47           C  \\nATOM   1596  NE2 GLN A 227     -19.081 -31.973 -25.810  1.00  0.38           N  \\nATOM   1597  OE1 GLN A 227     -18.951 -30.831 -27.743  1.00  0.48           O  \\nATOM   1598  N   GLN A 228     -18.919 -25.324 -27.465  1.00  0.44           N  \\nATOM   1599  CA  GLN A 228     -18.038 -24.238 -27.882  1.00  0.45           C  \\nATOM   1600  C   GLN A 228     -18.118 -23.062 -26.912  1.00  0.44           C  \\nATOM   1601  CB  GLN A 228     -18.388 -23.776 -29.297  1.00  0.42           C  \\nATOM   1602  O   GLN A 228     -17.092 -22.497 -26.528  1.00  0.44           O  \\nATOM   1603  CG  GLN A 228     -17.684 -24.564 -30.393  1.00  0.41           C  \\nATOM   1604  CD  GLN A 228     -18.018 -24.061 -31.785  1.00  0.46           C  \\nATOM   1605  NE2 GLN A 228     -17.517 -24.753 -32.802  1.00  0.39           N  \\nATOM   1606  OE1 GLN A 228     -18.723 -23.059 -31.945  1.00  0.48           O  \\nATOM   1607  N   GLN A 229     -19.357 -22.737 -26.507  1.00  0.43           N  \\nATOM   1608  CA  GLN A 229     -19.512 -21.677 -25.517  1.00  0.44           C  \\nATOM   1609  C   GLN A 229     -18.780 -22.022 -24.223  1.00  0.43           C  \\nATOM   1610  CB  GLN A 229     -20.992 -21.421 -25.230  1.00  0.41           C  \\nATOM   1611  O   GLN A 229     -18.084 -21.179 -23.654  1.00  0.43           O  \\nATOM   1612  CG  GLN A 229     -21.418 -19.974 -25.439  1.00  0.40           C  \\nATOM   1613  CD  GLN A 229     -22.898 -19.755 -25.188  1.00  0.45           C  \\nATOM   1614  NE2 GLN A 229     -23.329 -18.500 -25.241  1.00  0.39           N  \\nATOM   1615  OE1 GLN A 229     -23.648 -20.707 -24.951  1.00  0.48           O  \\nATOM   1616  N   GLN A 230     -18.952 -23.286 -23.747  1.00  0.46           N  \\nATOM   1617  CA  GLN A 230     -18.244 -23.725 -22.549  1.00  0.46           C  \\nATOM   1618  C   GLN A 230     -16.733 -23.628 -22.736  1.00  0.46           C  \\nATOM   1619  CB  GLN A 230     -18.637 -25.158 -22.188  1.00  0.44           C  \\nATOM   1620  O   GLN A 230     -16.018 -23.180 -21.836  1.00  0.46           O  \\nATOM   1621  CG  GLN A 230     -20.016 -25.276 -21.553  1.00  0.42           C  \\nATOM   1622  CD  GLN A 230     -20.388 -26.709 -21.219  1.00  0.45           C  \\nATOM   1623  NE2 GLN A 230     -21.537 -26.889 -20.577  1.00  0.40           N  \\nATOM   1624  OE1 GLN A 230     -19.649 -27.646 -21.538  1.00  0.50           O  \\nATOM   1625  N   GLN A 231     -16.264 -24.102 -23.931  1.00  0.46           N  \\nATOM   1626  CA  GLN A 231     -14.832 -24.011 -24.199  1.00  0.47           C  \\nATOM   1627  C   GLN A 231     -14.362 -22.560 -24.189  1.00  0.46           C  \\nATOM   1628  CB  GLN A 231     -14.495 -24.663 -25.541  1.00  0.44           C  \\nATOM   1629  O   GLN A 231     -13.332 -22.240 -23.591  1.00  0.46           O  \\nATOM   1630  CG  GLN A 231     -13.299 -25.604 -25.484  1.00  0.43           C  \\nATOM   1631  CD  GLN A 231     -13.015 -26.274 -26.815  1.00  0.47           C  \\nATOM   1632  NE2 GLN A 231     -11.887 -26.970 -26.900  1.00  0.41           N  \\nATOM   1633  OE1 GLN A 231     -13.805 -26.168 -27.759  1.00  0.50           O  \\nATOM   1634  N   TRP A 232     -15.112 -21.654 -24.821  1.00  0.34           N  \\nATOM   1635  CA  TRP A 232     -14.786 -20.232 -24.807  1.00  0.43           C  \\nATOM   1636  C   TRP A 232     -14.811 -19.681 -23.385  1.00  0.34           C  \\nATOM   1637  CB  TRP A 232     -15.762 -19.448 -25.689  1.00  0.35           C  \\nATOM   1638  O   TRP A 232     -13.900 -18.957 -22.978  1.00  0.35           O  \\nATOM   1639  CG  TRP A 232     -15.336 -19.341 -27.123  1.00  0.35           C  \\nATOM   1640  CD1 TRP A 232     -15.591 -20.233 -28.126  1.00  0.33           C  \\nATOM   1641  CD2 TRP A 232     -14.575 -18.282 -27.711  1.00  0.36           C  \\nATOM   1642  CE2 TRP A 232     -14.406 -18.598 -29.077  1.00  0.34           C  \\nATOM   1643  CE3 TRP A 232     -14.019 -17.095 -27.215  1.00  0.46           C  \\nATOM   1644  NE1 TRP A 232     -15.035 -19.792 -29.305  1.00  0.44           N  \\nATOM   1645  CH2 TRP A 232     -13.170 -16.613 -29.443  1.00  0.37           C  \\nATOM   1646  CZ2 TRP A 232     -13.703 -17.768 -29.954  1.00  0.41           C  \\nATOM   1647  CZ3 TRP A 232     -13.321 -16.270 -28.090  1.00  0.35           C  \\nATOM   1648  N   GLN A 233     -15.869 -19.990 -22.637  1.00  0.49           N  \\nATOM   1649  CA  GLN A 233     -15.949 -19.562 -21.244  1.00  0.51           C  \\nATOM   1650  C   GLN A 233     -14.741 -20.049 -20.450  1.00  0.49           C  \\nATOM   1651  CB  GLN A 233     -17.240 -20.068 -20.600  1.00  0.47           C  \\nATOM   1652  O   GLN A 233     -14.154 -19.291 -19.675  1.00  0.49           O  \\nATOM   1653  CG  GLN A 233     -18.464 -19.225 -20.932  1.00  0.45           C  \\nATOM   1654  CD  GLN A 233     -19.741 -19.782 -20.331  1.00  0.49           C  \\nATOM   1655  NE2 GLN A 233     -20.838 -19.049 -20.486  1.00  0.42           N  \\nATOM   1656  OE1 GLN A 233     -19.741 -20.862 -19.732  1.00  0.53           O  \\nATOM   1657  N   TRP A 234     -14.376 -21.306 -20.744  1.00  0.33           N  \\nATOM   1658  CA  TRP A 234     -13.223 -21.892 -20.068  1.00  0.44           C  \\nATOM   1659  C   TRP A 234     -11.938 -21.169 -20.457  1.00  0.34           C  \\nATOM   1660  CB  TRP A 234     -13.106 -23.382 -20.398  1.00  0.35           C  \\nATOM   1661  O   TRP A 234     -11.142 -20.797 -19.593  1.00  0.35           O  \\nATOM   1662  CG  TRP A 234     -12.566 -24.215 -19.274  1.00  0.35           C  \\nATOM   1663  CD1 TRP A 234     -13.250 -24.660 -18.177  1.00  0.32           C  \\nATOM   1664  CD2 TRP A 234     -11.227 -24.697 -19.135  1.00  0.36           C  \\nATOM   1665  CE2 TRP A 234     -11.169 -25.430 -17.929  1.00  0.34           C  \\nATOM   1666  CE3 TRP A 234     -10.068 -24.582 -19.915  1.00  0.45           C  \\nATOM   1667  NE1 TRP A 234     -12.415 -25.392 -17.364  1.00  0.43           N  \\nATOM   1668  CH2 TRP A 234      -8.877 -25.917 -18.266  1.00  0.36           C  \\nATOM   1669  CZ2 TRP A 234      -9.996 -26.045 -17.484  1.00  0.39           C  \\nATOM   1670  CZ3 TRP A 234      -8.902 -25.195 -19.471  1.00  0.35           C  \\nATOM   1671  N   GLN A 235     -11.716 -20.992 -21.705  1.00  0.51           N  \\nATOM   1672  CA  GLN A 235     -10.543 -20.271 -22.189  1.00  0.52           C  \\nATOM   1673  C   GLN A 235     -10.499 -18.853 -21.628  1.00  0.51           C  \\nATOM   1674  CB  GLN A 235     -10.529 -20.232 -23.717  1.00  0.49           C  \\nATOM   1675  O   GLN A 235      -9.445 -18.384 -21.193  1.00  0.52           O  \\nATOM   1676  CG  GLN A 235     -10.139 -21.555 -24.363  1.00  0.47           C  \\nATOM   1677  CD  GLN A 235     -10.228 -21.518 -25.877  1.00  0.50           C  \\nATOM   1678  NE2 GLN A 235      -9.678 -22.537 -26.528  1.00  0.44           N  \\nATOM   1679  OE1 GLN A 235     -10.788 -20.582 -26.456  1.00  0.55           O  \\nATOM   1680  N   TRP A 236     -11.683 -18.222 -21.639  1.00  0.40           N  \\nATOM   1681  CA  TRP A 236     -11.783 -16.880 -21.075  1.00  0.47           C  \\nATOM   1682  C   TRP A 236     -11.447 -16.887 -19.588  1.00  0.40           C  \\nATOM   1683  CB  TRP A 236     -13.187 -16.309 -21.291  1.00  0.39           C  \\nATOM   1684  O   TRP A 236     -10.664 -16.058 -19.118  1.00  0.41           O  \\nATOM   1685  CG  TRP A 236     -13.330 -14.873 -20.885  1.00  0.38           C  \\nATOM   1686  CD1 TRP A 236     -12.920 -13.773 -21.586  1.00  0.33           C  \\nATOM   1687  CD2 TRP A 236     -13.921 -14.381 -19.678  1.00  0.37           C  \\nATOM   1688  CE2 TRP A 236     -13.836 -12.972 -19.715  1.00  0.36           C  \\nATOM   1689  CE3 TRP A 236     -14.517 -14.997 -18.569  1.00  0.53           C  \\nATOM   1690  NE1 TRP A 236     -13.221 -12.627 -20.888  1.00  0.51           N  \\nATOM   1691  CH2 TRP A 236     -14.902 -12.795 -17.609  1.00  0.45           C  \\nATOM   1692  CZ2 TRP A 236     -14.324 -12.167 -18.682  1.00  0.57           C  \\nATOM   1693  CZ3 TRP A 236     -15.002 -14.194 -17.543  1.00  0.39           C  \\nATOM   1694  N   TRP A 237     -11.927 -17.832 -18.792  1.00  0.41           N  \\nATOM   1695  CA  TRP A 237     -11.650 -17.974 -17.366  1.00  0.49           C  \\nATOM   1696  C   TRP A 237     -10.164 -18.211 -17.121  1.00  0.41           C  \\nATOM   1697  CB  TRP A 237     -12.468 -19.123 -16.771  1.00  0.41           C  \\nATOM   1698  O   TRP A 237      -9.564 -17.577 -16.250  1.00  0.42           O  \\nATOM   1699  CG  TRP A 237     -12.582 -19.080 -15.277  1.00  0.40           C  \\nATOM   1700  CD1 TRP A 237     -13.374 -18.249 -14.534  1.00  0.35           C  \\nATOM   1701  CD2 TRP A 237     -11.874 -19.901 -14.342  1.00  0.38           C  \\nATOM   1702  CE2 TRP A 237     -12.288 -19.513 -13.048  1.00  0.37           C  \\nATOM   1703  CE3 TRP A 237     -10.930 -20.928 -14.473  1.00  0.50           C  \\nATOM   1704  NE1 TRP A 237     -13.202 -18.505 -13.193  1.00  0.48           N  \\nATOM   1705  CH2 TRP A 237     -10.866 -21.120 -12.049  1.00  0.42           C  \\nATOM   1706  CZ2 TRP A 237     -11.789 -20.118 -11.892  1.00  0.53           C  \\nATOM   1707  CZ3 TRP A 237     -10.434 -21.528 -13.322  1.00  0.38           C  \\nATOM   1708  N   GLN A 238      -9.520 -19.134 -17.892  1.00  0.53           N  \\nATOM   1709  CA  GLN A 238      -8.084 -19.376 -17.815  1.00  0.54           C  \\nATOM   1710  C   GLN A 238      -7.295 -18.094 -18.069  1.00  0.53           C  \\nATOM   1711  CB  GLN A 238      -7.666 -20.456 -18.814  1.00  0.51           C  \\nATOM   1712  O   GLN A 238      -6.335 -17.798 -17.354  1.00  0.53           O  \\nATOM   1713  CG  GLN A 238      -7.949 -21.875 -18.340  1.00  0.49           C  \\nATOM   1714  CD  GLN A 238      -7.194 -22.922 -19.138  1.00  0.52           C  \\nATOM   1715  NE2 GLN A 238      -7.005 -24.096 -18.545  1.00  0.44           N  \\nATOM   1716  OE1 GLN A 238      -6.784 -22.677 -20.277  1.00  0.54           O  \\nATOM   1717  N   GLN A 239      -7.768 -17.324 -19.094  1.00  0.55           N  \\nATOM   1718  CA  GLN A 239      -7.123 -16.047 -19.380  1.00  0.56           C  \\nATOM   1719  C   GLN A 239      -7.246 -15.090 -18.198  1.00  0.55           C  \\nATOM   1720  CB  GLN A 239      -7.725 -15.411 -20.635  1.00  0.53           C  \\nATOM   1721  O   GLN A 239      -6.270 -14.445 -17.810  1.00  0.54           O  \\nATOM   1722  CG  GLN A 239      -6.990 -15.770 -21.919  1.00  0.50           C  \\nATOM   1723  CD  GLN A 239      -7.650 -15.188 -23.155  1.00  0.52           C  \\nATOM   1724  NE2 GLN A 239      -7.070 -15.458 -24.320  1.00  0.44           N  \\nATOM   1725  OE1 GLN A 239      -8.673 -14.501 -23.064  1.00  0.55           O  \\nATOM   1726  N   GLN A 240      -8.465 -15.015 -17.586  1.00  0.56           N  \\nATOM   1727  CA  GLN A 240      -8.664 -14.165 -16.416  1.00  0.57           C  \\nATOM   1728  C   GLN A 240      -7.761 -14.595 -15.263  1.00  0.55           C  \\nATOM   1729  CB  GLN A 240     -10.128 -14.194 -15.974  1.00  0.53           C  \\nATOM   1730  O   GLN A 240      -7.128 -13.757 -14.619  1.00  0.55           O  \\nATOM   1731  CG  GLN A 240     -11.079 -13.518 -16.953  1.00  0.50           C  \\nATOM   1732  CD  GLN A 240     -10.792 -12.038 -17.122  1.00  0.53           C  \\nATOM   1733  NE2 GLN A 240     -11.111 -11.502 -18.296  1.00  0.44           N  \\nATOM   1734  OE1 GLN A 240     -10.288 -11.381 -16.206  1.00  0.55           O  \\nATOM   1735  N   GLN A 241      -7.715 -15.950 -14.960  1.00  0.56           N  \\nATOM   1736  CA  GLN A 241      -6.850 -16.466 -13.904  1.00  0.57           C  \\nATOM   1737  C   GLN A 241      -5.388 -16.123 -14.173  1.00  0.55           C  \\nATOM   1738  CB  GLN A 241      -7.018 -17.980 -13.764  1.00  0.53           C  \\nATOM   1739  O   GLN A 241      -4.670 -15.688 -13.271  1.00  0.55           O  \\nATOM   1740  CG  GLN A 241      -8.377 -18.400 -13.222  1.00  0.50           C  \\nATOM   1741  CD  GLN A 241      -8.579 -18.004 -11.771  1.00  0.53           C  \\nATOM   1742  NE2 GLN A 241      -9.817 -18.099 -11.297  1.00  0.44           N  \\nATOM   1743  OE1 GLN A 241      -7.631 -17.615 -11.082  1.00  0.54           O  \\nATOM   1744  N   GLN A 242      -4.963 -16.365 -15.487  1.00  0.54           N  \\nATOM   1745  CA  GLN A 242      -3.597 -16.016 -15.865  1.00  0.54           C  \\nATOM   1746  C   GLN A 242      -3.328 -14.531 -15.646  1.00  0.53           C  \\nATOM   1747  CB  GLN A 242      -3.333 -16.388 -17.325  1.00  0.52           C  \\nATOM   1748  O   GLN A 242      -2.279 -14.156 -15.117  1.00  0.53           O  \\nATOM   1749  CG  GLN A 242      -3.001 -17.860 -17.534  1.00  0.49           C  \\nATOM   1750  CD  GLN A 242      -2.771 -18.208 -18.993  1.00  0.52           C  \\nATOM   1751  NE2 GLN A 242      -2.569 -19.491 -19.272  1.00  0.44           N  \\nATOM   1752  OE1 GLN A 242      -2.776 -17.331 -19.863  1.00  0.54           O  \\nATOM   1753  N   GLN A 243      -4.290 -13.682 -16.080  1.00  0.55           N  \\nATOM   1754  CA  GLN A 243      -4.154 -12.247 -15.853  1.00  0.55           C  \\nATOM   1755  C   GLN A 243      -4.064 -11.932 -14.363  1.00  0.54           C  \\nATOM   1756  CB  GLN A 243      -5.326 -11.490 -16.480  1.00  0.52           C  \\nATOM   1757  O   GLN A 243      -3.241 -11.116 -13.944  1.00  0.54           O  \\nATOM   1758  CG  GLN A 243      -5.209 -11.315 -17.988  1.00  0.49           C  \\nATOM   1759  CD  GLN A 243      -6.380 -10.556 -18.583  1.00  0.52           C  \\nATOM   1760  NE2 GLN A 243      -6.377 -10.401 -19.903  1.00  0.44           N  \\nATOM   1761  OE1 GLN A 243      -7.281 -10.114 -17.863  1.00  0.54           O  \\nATOM   1762  N   GLN A 244      -4.961 -12.622 -13.491  1.00  0.57           N  \\nATOM   1763  CA  GLN A 244      -4.908 -12.435 -12.045  1.00  0.57           C  \\nATOM   1764  C   GLN A 244      -3.554 -12.860 -11.484  1.00  0.56           C  \\nATOM   1765  CB  GLN A 244      -6.029 -13.219 -11.361  1.00  0.53           C  \\nATOM   1766  O   GLN A 244      -2.968 -12.155 -10.660  1.00  0.56           O  \\nATOM   1767  CG  GLN A 244      -7.386 -12.529 -11.417  1.00  0.50           C  \\nATOM   1768  CD  GLN A 244      -8.458 -13.280 -10.651  1.00  0.53           C  \\nATOM   1769  NE2 GLN A 244      -9.642 -12.684 -10.550  1.00  0.44           N  \\nATOM   1770  OE1 GLN A 244      -8.226 -14.387 -10.154  1.00  0.54           O  \\nATOM   1771  N   TRP A 245      -2.986 -14.069 -11.943  1.00  0.48           N  \\nATOM   1772  CA  TRP A 245      -1.678 -14.554 -11.513  1.00  0.55           C  \\nATOM   1773  C   TRP A 245      -0.579 -13.568 -11.895  1.00  0.47           C  \\nATOM   1774  CB  TRP A 245      -1.386 -15.927 -12.125  1.00  0.46           C  \\nATOM   1775  O   TRP A 245       0.306 -13.272 -11.089  1.00  0.47           O  \\nATOM   1776  CG  TRP A 245      -1.917 -17.077 -11.323  1.00  0.44           C  \\nATOM   1777  CD1 TRP A 245      -3.227 -17.430 -11.157  1.00  0.38           C  \\nATOM   1778  CD2 TRP A 245      -1.148 -18.022 -10.573  1.00  0.41           C  \\nATOM   1779  CE2 TRP A 245      -2.058 -18.922  -9.977  1.00  0.38           C  \\nATOM   1780  CE3 TRP A 245       0.224 -18.195 -10.347  1.00  0.49           C  \\nATOM   1781  NE1 TRP A 245      -3.319 -18.540 -10.349  1.00  0.44           N  \\nATOM   1782  CH2 TRP A 245      -0.292 -20.129  -8.964  1.00  0.37           C  \\nATOM   1783  CZ2 TRP A 245      -1.640 -19.982  -9.169  1.00  0.43           C  \\nATOM   1784  CZ3 TRP A 245       0.639 -19.250  -9.542  1.00  0.36           C  \\nATOM   1785  N   TRP A 246      -0.622 -13.034 -13.214  1.00  0.50           N  \\nATOM   1786  CA  TRP A 246       0.333 -12.027 -13.664  1.00  0.57           C  \\nATOM   1787  C   TRP A 246       0.266 -10.782 -12.786  1.00  0.50           C  \\nATOM   1788  CB  TRP A 246       0.071 -11.651 -15.125  1.00  0.50           C  \\nATOM   1789  O   TRP A 246       1.299 -10.238 -12.389  1.00  0.50           O  \\nATOM   1790  CG  TRP A 246       0.815 -12.496 -16.115  1.00  0.49           C  \\nATOM   1791  CD1 TRP A 246       0.657 -13.836 -16.338  1.00  0.42           C  \\nATOM   1792  CD2 TRP A 246       1.840 -12.058 -17.012  1.00  0.44           C  \\nATOM   1793  CE2 TRP A 246       2.258 -13.187 -17.752  1.00  0.41           C  \\nATOM   1794  CE3 TRP A 246       2.445 -10.819 -17.264  1.00  0.47           C  \\nATOM   1795  NE1 TRP A 246       1.522 -14.257 -17.321  1.00  0.43           N  \\nATOM   1796  CH2 TRP A 246       3.830 -11.889 -18.954  1.00  0.36           C  \\nATOM   1797  CZ2 TRP A 246       3.255 -13.112 -18.727  1.00  0.39           C  \\nATOM   1798  CZ3 TRP A 246       3.437 -10.748 -18.236  1.00  0.35           C  \\nATOM   1799  N   TRP A 247      -1.026 -10.334 -12.456  1.00  0.54           N  \\nATOM   1800  CA  TRP A 247      -1.198  -9.191 -11.565  1.00  0.60           C  \\nATOM   1801  C   TRP A 247      -0.580  -9.469 -10.199  1.00  0.53           C  \\nATOM   1802  CB  TRP A 247      -2.682  -8.849 -11.408  1.00  0.53           C  \\nATOM   1803  O   TRP A 247       0.129  -8.624  -9.648  1.00  0.53           O  \\nATOM   1804  CG  TRP A 247      -3.201  -7.901 -12.447  1.00  0.50           C  \\nATOM   1805  CD1 TRP A 247      -3.675  -8.219 -13.690  1.00  0.44           C  \\nATOM   1806  CD2 TRP A 247      -3.294  -6.477 -12.334  1.00  0.45           C  \\nATOM   1807  CE2 TRP A 247      -3.835  -5.996 -13.547  1.00  0.43           C  \\nATOM   1808  CE3 TRP A 247      -2.973  -5.561 -11.324  1.00  0.49           C  \\nATOM   1809  NE1 TRP A 247      -4.058  -7.078 -14.356  1.00  0.45           N  \\nATOM   1810  CH2 TRP A 247      -3.738  -3.762 -12.772  1.00  0.38           C  \\nATOM   1811  CZ2 TRP A 247      -4.061  -4.637 -13.777  1.00  0.43           C  \\nATOM   1812  CZ3 TRP A 247      -3.199  -4.209 -11.554  1.00  0.37           C  \\nATOM   1813  N   TRP A 248      -0.836 -10.733  -9.643  1.00  0.56           N  \\nATOM   1814  CA  TRP A 248      -0.267 -11.125  -8.358  1.00  0.62           C  \\nATOM   1815  C   TRP A 248       1.257 -11.118  -8.412  1.00  0.55           C  \\nATOM   1816  CB  TRP A 248      -0.770 -12.512  -7.947  1.00  0.54           C  \\nATOM   1817  O   TRP A 248       1.914 -10.573  -7.522  1.00  0.54           O  \\nATOM   1818  CG  TRP A 248      -1.956 -12.483  -7.031  1.00  0.50           C  \\nATOM   1819  CD1 TRP A 248      -3.275 -12.496  -7.390  1.00  0.43           C  \\nATOM   1820  CD2 TRP A 248      -1.930 -12.431  -5.601  1.00  0.45           C  \\nATOM   1821  CE2 TRP A 248      -3.271 -12.416  -5.159  1.00  0.43           C  \\nATOM   1822  CE3 TRP A 248      -0.901 -12.397  -4.650  1.00  0.53           C  \\nATOM   1823  NE1 TRP A 248      -4.071 -12.456  -6.269  1.00  0.47           N  \\nATOM   1824  CH2 TRP A 248      -2.584 -12.335  -2.895  1.00  0.41           C  \\nATOM   1825  CZ2 TRP A 248      -3.610 -12.368  -3.805  1.00  0.48           C  \\nATOM   1826  CZ3 TRP A 248      -1.240 -12.349  -3.303  1.00  0.39           C  \\nATOM   1827  N   TRP A 249       1.861 -11.785  -9.517  1.00  0.52           N  \\nATOM   1828  CA  TRP A 249       3.309 -11.833  -9.690  1.00  0.59           C  \\nATOM   1829  C   TRP A 249       3.888 -10.430  -9.830  1.00  0.52           C  \\nATOM   1830  CB  TRP A 249       3.677 -12.676 -10.914  1.00  0.51           C  \\nATOM   1831  O   TRP A 249       4.898 -10.102  -9.202  1.00  0.52           O  \\nATOM   1832  CG  TRP A 249       4.483 -13.898 -10.593  1.00  0.50           C  \\nATOM   1833  CD1 TRP A 249       4.012 -15.165 -10.385  1.00  0.42           C  \\nATOM   1834  CD2 TRP A 249       5.904 -13.968 -10.437  1.00  0.44           C  \\nATOM   1835  CE2 TRP A 249       6.226 -15.310 -10.136  1.00  0.42           C  \\nATOM   1836  CE3 TRP A 249       6.937 -13.025 -10.525  1.00  0.50           C  \\nATOM   1837  NE1 TRP A 249       5.055 -16.019 -10.110  1.00  0.45           N  \\nATOM   1838  CH2 TRP A 249       8.531 -14.790 -10.013  1.00  0.37           C  \\nATOM   1839  CZ2 TRP A 249       7.540 -15.732  -9.922  1.00  0.42           C  \\nATOM   1840  CZ3 TRP A 249       8.244 -13.448 -10.311  1.00  0.37           C  \\nATOM   1841  N   TRP A 250       3.191  -9.594 -10.718  1.00  0.54           N  \\nATOM   1842  CA  TRP A 250       3.614  -8.204 -10.857  1.00  0.60           C  \\nATOM   1843  C   TRP A 250       3.567  -7.483  -9.514  1.00  0.53           C  \\nATOM   1844  CB  TRP A 250       2.733  -7.473 -11.874  1.00  0.53           C  \\nATOM   1845  O   TRP A 250       4.513  -6.783  -9.145  1.00  0.54           O  \\nATOM   1846  CG  TRP A 250       3.464  -6.446 -12.686  1.00  0.51           C  \\nATOM   1847  CD1 TRP A 250       4.146  -6.650 -13.853  1.00  0.44           C  \\nATOM   1848  CD2 TRP A 250       3.589  -5.052 -12.386  1.00  0.46           C  \\nATOM   1849  CE2 TRP A 250       4.361  -4.471 -13.417  1.00  0.44           C  \\nATOM   1850  CE3 TRP A 250       3.123  -4.237 -11.346  1.00  0.51           C  \\nATOM   1851  NE1 TRP A 250       4.688  -5.466 -14.298  1.00  0.45           N  \\nATOM   1852  CH2 TRP A 250       4.208  -2.336 -12.407  1.00  0.37           C  \\nATOM   1853  CZ2 TRP A 250       4.677  -3.110 -13.436  1.00  0.42           C  \\nATOM   1854  CZ3 TRP A 250       3.438  -2.884 -11.367  1.00  0.37           C  \\nATOM   1855  N   TRP A 251       2.437  -7.694  -8.757  1.00  0.56           N  \\nATOM   1856  CA  TRP A 251       2.322  -7.093  -7.432  1.00  0.62           C  \\nATOM   1857  C   TRP A 251       3.449  -7.565  -6.520  1.00  0.56           C  \\nATOM   1858  CB  TRP A 251       0.966  -7.430  -6.805  1.00  0.55           C  \\nATOM   1859  O   TRP A 251       4.076  -6.759  -5.829  1.00  0.56           O  \\nATOM   1860  CG  TRP A 251       0.161  -6.228  -6.412  1.00  0.53           C  \\nATOM   1861  CD1 TRP A 251      -0.825  -5.623  -7.140  1.00  0.45           C  \\nATOM   1862  CD2 TRP A 251       0.279  -5.482  -5.196  1.00  0.47           C  \\nATOM   1863  CE2 TRP A 251      -0.670  -4.437  -5.255  1.00  0.45           C  \\nATOM   1864  CE3 TRP A 251       1.093  -5.596  -4.062  1.00  0.53           C  \\nATOM   1865  NE1 TRP A 251      -1.329  -4.545  -6.450  1.00  0.47           N  \\nATOM   1866  CH2 TRP A 251      -0.015  -3.645  -3.123  1.00  0.40           C  \\nATOM   1867  CZ2 TRP A 251      -0.825  -3.510  -4.221  1.00  0.46           C  \\nATOM   1868  CZ3 TRP A 251       0.938  -4.674  -3.034  1.00  0.39           C  \\nATOM   1869  N   TRP A 252       3.753  -8.919  -6.558  1.00  0.53           N  \\nATOM   1870  CA  TRP A 252       4.816  -9.493  -5.740  1.00  0.60           C  \\nATOM   1871  C   TRP A 252       6.175  -8.929  -6.139  1.00  0.52           C  \\nATOM   1872  CB  TRP A 252       4.825 -11.019  -5.863  1.00  0.52           C  \\nATOM   1873  O   TRP A 252       6.971  -8.543  -5.279  1.00  0.52           O  \\nATOM   1874  CG  TRP A 252       4.885 -11.736  -4.548  1.00  0.50           C  \\nATOM   1875  CD1 TRP A 252       3.830 -12.090  -3.754  1.00  0.43           C  \\nATOM   1876  CD2 TRP A 252       6.064 -12.183  -3.872  1.00  0.45           C  \\nATOM   1877  CE2 TRP A 252       5.649 -12.801  -2.671  1.00  0.43           C  \\nATOM   1878  CE3 TRP A 252       7.433 -12.121  -4.165  1.00  0.52           C  \\nATOM   1879  NE1 TRP A 252       4.283 -12.731  -2.624  1.00  0.46           N  \\nATOM   1880  CH2 TRP A 252       7.889 -13.280  -2.075  1.00  0.38           C  \\nATOM   1881  CZ2 TRP A 252       6.556 -13.354  -1.764  1.00  0.44           C  \\nATOM   1882  CZ3 TRP A 252       8.334 -12.672  -3.260  1.00  0.38           C  \\nATOM   1883  N   TRP A 253       6.514  -8.835  -7.487  1.00  0.49           N  \\nATOM   1884  CA  TRP A 253       7.777  -8.289  -7.972  1.00  0.58           C  \\nATOM   1885  C   TRP A 253       7.889  -6.803  -7.645  1.00  0.49           C  \\nATOM   1886  CB  TRP A 253       7.910  -8.503  -9.483  1.00  0.50           C  \\nATOM   1887  O   TRP A 253       8.943  -6.335  -7.209  1.00  0.50           O  \\nATOM   1888  CG  TRP A 253       9.106  -9.316  -9.879  1.00  0.49           C  \\nATOM   1889  CD1 TRP A 253       9.209 -10.680  -9.885  1.00  0.42           C  \\nATOM   1890  CD2 TRP A 253      10.371  -8.816 -10.323  1.00  0.44           C  \\nATOM   1891  CE2 TRP A 253      11.195  -9.933 -10.582  1.00  0.42           C  \\nATOM   1892  CE3 TRP A 253      10.887  -7.529 -10.527  1.00  0.49           C  \\nATOM   1893  NE1 TRP A 253      10.463 -11.057 -10.307  1.00  0.44           N  \\nATOM   1894  CH2 TRP A 253      12.990  -8.532 -11.227  1.00  0.35           C  \\nATOM   1895  CZ2 TRP A 253      12.510  -9.802 -11.035  1.00  0.39           C  \\nATOM   1896  CZ3 TRP A 253      12.196  -7.401 -10.979  1.00  0.36           C  \\nATOM   1897  N   TRP A 254       6.751  -6.031  -7.827  1.00  0.51           N  \\nATOM   1898  CA  TRP A 254       6.735  -4.624  -7.443  1.00  0.59           C  \\nATOM   1899  C   TRP A 254       7.019  -4.463  -5.953  1.00  0.51           C  \\nATOM   1900  CB  TRP A 254       5.388  -3.985  -7.791  1.00  0.52           C  \\nATOM   1901  O   TRP A 254       7.781  -3.581  -5.552  1.00  0.52           O  \\nATOM   1902  CG  TRP A 254       5.477  -2.893  -8.814  1.00  0.51           C  \\nATOM   1903  CD1 TRP A 254       5.642  -3.039 -10.163  1.00  0.45           C  \\nATOM   1904  CD2 TRP A 254       5.411  -1.485  -8.568  1.00  0.46           C  \\nATOM   1905  CE2 TRP A 254       5.543  -0.836  -9.816  1.00  0.44           C  \\nATOM   1906  CE3 TRP A 254       5.252  -0.708  -7.413  1.00  0.49           C  \\nATOM   1907  NE1 TRP A 254       5.682  -1.806 -10.772  1.00  0.44           N  \\nATOM   1908  CH2 TRP A 254       5.364   1.291  -8.794  1.00  0.35           C  \\nATOM   1909  CZ2 TRP A 254       5.520   0.555  -9.940  1.00  0.40           C  \\nATOM   1910  CZ3 TRP A 254       5.230   0.676  -7.538  1.00  0.36           C  \\nATOM   1911  N   TRP A 255       6.550  -5.348  -5.052  1.00  0.52           N  \\nATOM   1912  CA  TRP A 255       6.842  -5.274  -3.624  1.00  0.60           C  \\nATOM   1913  C   TRP A 255       8.327  -5.493  -3.360  1.00  0.52           C  \\nATOM   1914  CB  TRP A 255       6.015  -6.306  -2.851  1.00  0.52           C  \\nATOM   1915  O   TRP A 255       8.908  -4.858  -2.477  1.00  0.52           O  \\nATOM   1916  CG  TRP A 255       4.681  -5.798  -2.392  1.00  0.51           C  \\nATOM   1917  CD1 TRP A 255       3.500  -5.847  -3.078  1.00  0.45           C  \\nATOM   1918  CD2 TRP A 255       4.395  -5.158  -1.145  1.00  0.46           C  \\nATOM   1919  CE2 TRP A 255       3.017  -4.846  -1.142  1.00  0.44           C  \\nATOM   1920  CE3 TRP A 255       5.170  -4.818  -0.028  1.00  0.51           C  \\nATOM   1921  NE1 TRP A 255       2.495  -5.277  -2.332  1.00  0.47           N  \\nATOM   1922  CH2 TRP A 255       3.178  -3.888   1.016  1.00  0.40           C  \\nATOM   1923  CZ2 TRP A 255       2.397  -4.210  -0.064  1.00  0.47           C  \\nATOM   1924  CZ3 TRP A 255       4.551  -4.185   1.044  1.00  0.39           C  \\nATOM   1925  N   TRP A 256       9.035  -6.370  -4.199  1.00  0.50           N  \\nATOM   1926  CA  TRP A 256      10.446  -6.651  -3.955  1.00  0.59           C  \\nATOM   1927  C   TRP A 256      11.320  -5.486  -4.409  1.00  0.51           C  \\nATOM   1928  CB  TRP A 256      10.871  -7.935  -4.673  1.00  0.50           C  \\nATOM   1929  O   TRP A 256      12.399  -5.259  -3.857  1.00  0.51           O  \\nATOM   1930  CG  TRP A 256      10.395  -9.191  -4.008  1.00  0.49           C  \\nATOM   1931  CD1 TRP A 256       9.183  -9.802  -4.172  1.00  0.42           C  \\nATOM   1932  CD2 TRP A 256      11.121  -9.987  -3.067  1.00  0.44           C  \\nATOM   1933  CE2 TRP A 256      10.288 -11.068  -2.702  1.00  0.43           C  \\nATOM   1934  CE3 TRP A 256      12.398  -9.892  -2.496  1.00  0.52           C  \\nATOM   1935  NE1 TRP A 256       9.112 -10.931  -3.390  1.00  0.47           N  \\nATOM   1936  CH2 TRP A 256      11.944 -11.931  -1.249  1.00  0.39           C  \\nATOM   1937  CZ2 TRP A 256      10.691 -12.048  -1.792  1.00  0.46           C  \\nATOM   1938  CZ3 TRP A 256      12.797 -10.868  -1.590  1.00  0.39           C  \\nATOM   1939  N   TRP A 257      10.813  -4.564  -5.378  1.00  0.43           N  \\nATOM   1940  CA  TRP A 257      11.606  -3.437  -5.857  1.00  0.53           C  \\nATOM   1941  C   TRP A 257      11.175  -2.141  -5.178  1.00  0.44           C  \\nATOM   1942  CB  TRP A 257      11.481  -3.297  -7.376  1.00  0.43           C  \\nATOM   1943  O   TRP A 257      11.733  -1.075  -5.450  1.00  0.44           O  \\nATOM   1944  CG  TRP A 257      12.252  -4.326  -8.148  1.00  0.44           C  \\nATOM   1945  CD1 TRP A 257      11.891  -5.623  -8.381  1.00  0.39           C  \\nATOM   1946  CD2 TRP A 257      13.520  -4.142  -8.784  1.00  0.41           C  \\nATOM   1947  CE2 TRP A 257      13.868  -5.372  -9.386  1.00  0.39           C  \\nATOM   1948  CE3 TRP A 257      14.397  -3.055  -8.905  1.00  0.46           C  \\nATOM   1949  NE1 TRP A 257      12.858  -6.258  -9.125  1.00  0.41           N  \\nATOM   1950  CH2 TRP A 257      15.897  -4.465 -10.201  1.00  0.31           C  \\nATOM   1951  CZ2 TRP A 257      15.058  -5.544 -10.098  1.00  0.34           C  \\nATOM   1952  CZ3 TRP A 257      15.580  -3.229  -9.615  1.00  0.34           C  \\nATOM   1953  N   TRP A 258      10.100  -2.120  -4.356  1.00  0.41           N  \\nATOM   1954  CA  TRP A 258       9.932  -0.964  -3.482  1.00  0.54           C  \\nATOM   1955  C   TRP A 258      11.028  -0.918  -2.422  1.00  0.43           C  \\nATOM   1956  CB  TRP A 258       8.556  -0.993  -2.810  1.00  0.43           C  \\nATOM   1957  O   TRP A 258      11.213  -1.878  -1.670  1.00  0.43           O  \\nATOM   1958  CG  TRP A 258       7.971   0.365  -2.562  1.00  0.45           C  \\nATOM   1959  CD1 TRP A 258       8.308   1.238  -1.565  1.00  0.43           C  \\nATOM   1960  CD2 TRP A 258       6.940   1.003  -3.323  1.00  0.43           C  \\nATOM   1961  CE2 TRP A 258       6.703   2.263  -2.730  1.00  0.41           C  \\nATOM   1962  CE3 TRP A 258       6.195   0.632  -4.449  1.00  0.48           C  \\nATOM   1963  NE1 TRP A 258       7.550   2.382  -1.661  1.00  0.44           N  \\nATOM   1964  CH2 TRP A 258       5.034   2.766  -4.331  1.00  0.36           C  \\nATOM   1965  CZ2 TRP A 258       5.750   3.155  -3.228  1.00  0.39           C  \\nATOM   1966  CZ3 TRP A 258       5.246   1.520  -4.943  1.00  0.36           C  \\nATOM   1967  N   PRO A 259      12.017  -0.071  -2.552  1.00  0.52           N  \\nATOM   1968  CA  PRO A 259      12.678   0.238  -1.281  1.00  0.52           C  \\nATOM   1969  C   PRO A 259      11.688   0.484  -0.145  1.00  0.53           C  \\nATOM   1970  CB  PRO A 259      13.474   1.507  -1.597  1.00  0.50           C  \\nATOM   1971  O   PRO A 259      10.575   0.960  -0.384  1.00  0.52           O  \\nATOM   1972  CG  PRO A 259      12.765   2.123  -2.759  1.00  0.50           C  \\nATOM   1973  CD  PRO A 259      12.030   1.042  -3.498  1.00  0.51           C  \\nATOM   1974  N   PRO A 260      11.751  -0.308   0.908  1.00  0.54           N  \\nATOM   1975  CA  PRO A 260      10.976   0.125   2.073  1.00  0.55           C  \\nATOM   1976  C   PRO A 260      10.783   1.638   2.124  1.00  0.54           C  \\nATOM   1977  CB  PRO A 260      11.819  -0.359   3.256  1.00  0.52           C  \\nATOM   1978  O   PRO A 260      11.638   2.390   1.649  1.00  0.54           O  \\nATOM   1979  CG  PRO A 260      13.198  -0.519   2.702  1.00  0.51           C  \\nATOM   1980  CD  PRO A 260      13.095  -0.729   1.219  1.00  0.52           C  \\nATOM   1981  N   PRO A 261       9.430   2.140   2.195  1.00  0.53           N  \\nATOM   1982  CA  PRO A 261       9.361   3.568   2.511  1.00  0.55           C  \\nATOM   1983  C   PRO A 261      10.484   4.022   3.441  1.00  0.55           C  \\nATOM   1984  CB  PRO A 261       7.997   3.713   3.191  1.00  0.52           C  \\nATOM   1985  O   PRO A 261      10.922   3.258   4.305  1.00  0.55           O  \\nATOM   1986  CG  PRO A 261       7.666   2.339   3.680  1.00  0.52           C  \\nATOM   1987  CD  PRO A 261       8.419   1.342   2.846  1.00  0.55           C  \\nATOM   1988  N   PRO A 262      11.228   5.075   3.039  1.00  0.53           N  \\nATOM   1989  CA  PRO A 262      12.120   5.650   4.049  1.00  0.55           C  \\nATOM   1990  C   PRO A 262      11.493   5.679   5.441  1.00  0.54           C  \\nATOM   1991  CB  PRO A 262      12.370   7.069   3.532  1.00  0.51           C  \\nATOM   1992  O   PRO A 262      10.269   5.771   5.569  1.00  0.54           O  \\nATOM   1993  CG  PRO A 262      11.226   7.347   2.612  1.00  0.51           C  \\nATOM   1994  CD  PRO A 262      10.677   6.038   2.123  1.00  0.54           C  \\nATOM   1995  N   PRO A 263      12.079   5.103   6.418  1.00  0.60           N  \\nATOM   1996  CA  PRO A 263      11.622   5.254   7.801  1.00  0.63           C  \\nATOM   1997  C   PRO A 263      10.970   6.609   8.063  1.00  0.62           C  \\nATOM   1998  CB  PRO A 263      12.909   5.100   8.616  1.00  0.58           C  \\nATOM   1999  O   PRO A 263      11.315   7.600   7.414  1.00  0.60           O  \\nATOM   2000  CG  PRO A 263      14.007   5.455   7.666  1.00  0.57           C  \\nATOM   2001  CD  PRO A 263      13.538   5.172   6.268  1.00  0.58           C  \\nATOM   2002  N   PRO A 264       9.738   6.618   8.615  1.00  0.63           N  \\nATOM   2003  CA  PRO A 264       9.215   7.914   9.054  1.00  0.68           C  \\nATOM   2004  C   PRO A 264      10.297   8.821   9.635  1.00  0.66           C  \\nATOM   2005  CB  PRO A 264       8.187   7.536  10.123  1.00  0.61           C  \\nATOM   2006  O   PRO A 264      11.249   8.335  10.251  1.00  0.65           O  \\nATOM   2007  CG  PRO A 264       8.616   6.188  10.606  1.00  0.60           C  \\nATOM   2008  CD  PRO A 264       9.364   5.501   9.499  1.00  0.61           C  \\nATOM   2009  N   PRO A 265      10.325  10.079   9.172  1.00  0.62           N  \\nATOM   2010  CA  PRO A 265      11.201  11.020   9.874  1.00  0.66           C  \\nATOM   2011  C   PRO A 265      11.098  10.900  11.393  1.00  0.64           C  \\nATOM   2012  CB  PRO A 265      10.702  12.387   9.400  1.00  0.60           C  \\nATOM   2013  O   PRO A 265      10.040  10.541  11.918  1.00  0.62           O  \\nATOM   2014  CG  PRO A 265       9.290  12.151   8.971  1.00  0.58           C  \\nATOM   2015  CD  PRO A 265       9.153  10.716   8.549  1.00  0.60           C  \\nATOM   2016  N   PRO A 266      12.131  10.739  12.066  1.00  0.69           N  \\nATOM   2017  CA  PRO A 266      12.131  10.787  13.530  1.00  0.72           C  \\nATOM   2018  C   PRO A 266      11.161  11.827  14.086  1.00  0.71           C  \\nATOM   2019  CB  PRO A 266      13.578  11.153  13.871  1.00  0.66           C  \\nATOM   2020  O   PRO A 266      10.883  12.832  13.427  1.00  0.67           O  \\nATOM   2021  CG  PRO A 266      14.089  11.862  12.658  1.00  0.64           C  \\nATOM   2022  CD  PRO A 266      13.344  11.352  11.458  1.00  0.64           C  \\nATOM   2023  N   PRO A 267      10.302  11.468  15.010  1.00  0.72           N  \\nATOM   2024  CA  PRO A 267       9.492  12.472  15.702  1.00  0.75           C  \\nATOM   2025  C   PRO A 267      10.262  13.760  15.983  1.00  0.74           C  \\nATOM   2026  CB  PRO A 267       9.102  11.773  17.007  1.00  0.69           C  \\nATOM   2027  O   PRO A 267      11.479  13.725  16.183  1.00  0.70           O  \\nATOM   2028  CG  PRO A 267      10.170  10.750  17.225  1.00  0.66           C  \\nATOM   2029  CD  PRO A 267      10.720  10.342  15.888  1.00  0.66           C  \\nATOM   2030  N   PRO A 268       9.693  14.901  15.649  1.00  0.71           N  \\nATOM   2031  CA  PRO A 268      10.311  16.154  16.088  1.00  0.73           C  \\nATOM   2032  C   PRO A 268      10.806  16.094  17.531  1.00  0.72           C  \\nATOM   2033  CB  PRO A 268       9.181  17.176  15.940  1.00  0.67           C  \\nATOM   2034  O   PRO A 268      10.235  15.373  18.354  1.00  0.68           O  \\nATOM   2035  CG  PRO A 268       7.925  16.371  16.025  1.00  0.65           C  \\nATOM   2036  CD  PRO A 268       8.214  14.982  15.535  1.00  0.65           C  \\nATOM   2037  N   PRO A 269      12.009  16.440  17.797  1.00  0.72           N  \\nATOM   2038  CA  PRO A 269      12.512  16.577  19.166  1.00  0.73           C  \\nATOM   2039  C   PRO A 269      11.491  17.206  20.111  1.00  0.73           C  \\nATOM   2040  CB  PRO A 269      13.735  17.483  19.003  1.00  0.69           C  \\nATOM   2041  O   PRO A 269      10.659  18.009  19.680  1.00  0.68           O  \\nATOM   2042  CG  PRO A 269      13.477  18.251  17.747  1.00  0.67           C  \\nATOM   2043  CD  PRO A 269      12.612  17.416  16.847  1.00  0.67           C  \\nATOM   2044  N   PRO A 270      11.168  16.605  21.200  1.00  0.74           N  \\nATOM   2045  CA  PRO A 270      10.350  17.235  22.238  1.00  0.75           C  \\nATOM   2046  C   PRO A 270      10.634  18.728  22.389  1.00  0.74           C  \\nATOM   2047  CB  PRO A 270      10.742  16.473  23.507  1.00  0.71           C  \\nATOM   2048  O   PRO A 270      11.768  19.168  22.181  1.00  0.70           O  \\nATOM   2049  CG  PRO A 270      12.126  15.974  23.240  1.00  0.69           C  \\nATOM   2050  CD  PRO A 270      12.278  15.758  21.762  1.00  0.68           C  \\nATOM   2051  N   PRO A 271       9.598  19.567  22.219  1.00  0.74           N  \\nATOM   2052  CA  PRO A 271       9.804  20.982  22.534  1.00  0.75           C  \\nATOM   2053  C   PRO A 271      10.812  21.195  23.661  1.00  0.73           C  \\nATOM   2054  CB  PRO A 271       8.410  21.457  22.951  1.00  0.69           C  \\nATOM   2055  O   PRO A 271      10.708  20.562  24.715  1.00  0.68           O  \\nATOM   2056  CG  PRO A 271       7.629  20.202  23.171  1.00  0.65           C  \\nATOM   2057  CD  PRO A 271       8.380  19.058  22.553  1.00  0.65           C  \\nATOM   2058  N   GLN A 272      12.069  21.577  23.282  1.00  0.74           N  \\nATOM   2059  CA  GLN A 272      13.046  22.053  24.254  1.00  0.76           C  \\nATOM   2060  C   GLN A 272      12.363  22.766  25.418  1.00  0.73           C  \\nATOM   2061  CB  GLN A 272      14.057  22.988  23.587  1.00  0.69           C  \\nATOM   2062  O   GLN A 272      11.619  23.728  25.213  1.00  0.67           O  \\nATOM   2063  CG  GLN A 272      15.283  22.273  23.034  1.00  0.64           C  \\nATOM   2064  CD  GLN A 272      16.211  23.203  22.275  1.00  0.63           C  \\nATOM   2065  NE2 GLN A 272      17.243  22.637  21.659  1.00  0.55           N  \\nATOM   2066  OE1 GLN A 272      16.002  24.420  22.243  1.00  0.67           O  \\nATOM   2067  N   GLN A 273      12.026  21.993  26.571  1.00  0.78           N  \\nATOM   2068  CA  GLN A 273      11.672  22.635  27.833  1.00  0.79           C  \\nATOM   2069  C   GLN A 273      12.427  23.950  28.011  1.00  0.78           C  \\nATOM   2070  CB  GLN A 273      11.959  21.701  29.010  1.00  0.72           C  \\nATOM   2071  O   GLN A 273      13.658  23.964  28.054  1.00  0.71           O  \\nATOM   2072  CG  GLN A 273      10.733  20.954  29.515  1.00  0.66           C  \\nATOM   2073  CD  GLN A 273      11.069  19.924  30.577  1.00  0.65           C  \\nATOM   2074  NE2 GLN A 273      10.076  19.136  30.975  1.00  0.57           N  \\nATOM   2075  OE1 GLN A 273      12.213  19.837  31.036  1.00  0.69           O  \\nATOM   2076  N   GLN A 274      11.826  25.101  27.497  1.00  0.78           N  \\nATOM   2077  CA  GLN A 274      12.301  26.423  27.891  1.00  0.80           C  \\nATOM   2078  C   GLN A 274      12.762  26.431  29.346  1.00  0.79           C  \\nATOM   2079  CB  GLN A 274      11.208  27.472  27.681  1.00  0.73           C  \\nATOM   2080  O   GLN A 274      11.986  26.115  30.250  1.00  0.73           O  \\nATOM   2081  CG  GLN A 274      11.058  27.923  26.235  1.00  0.67           C  \\nATOM   2082  CD  GLN A 274       9.932  28.922  26.045  1.00  0.66           C  \\nATOM   2083  NE2 GLN A 274       9.721  29.349  24.805  1.00  0.60           N  \\nATOM   2084  OE1 GLN A 274       9.257  29.306  27.005  1.00  0.72           O  \\nATOM   2085  N   GLN A 275      14.116  26.195  29.546  1.00  0.79           N  \\nATOM   2086  CA  GLN A 275      14.702  26.500  30.847  1.00  0.81           C  \\nATOM   2087  C   GLN A 275      14.109  27.780  31.430  1.00  0.81           C  \\nATOM   2088  CB  GLN A 275      16.221  26.630  30.735  1.00  0.75           C  \\nATOM   2089  O   GLN A 275      14.043  28.806  30.750  1.00  0.75           O  \\nATOM   2090  CG  GLN A 275      16.941  26.606  32.077  1.00  0.69           C  \\nATOM   2091  CD  GLN A 275      18.452  26.622  31.933  1.00  0.68           C  \\nATOM   2092  NE2 GLN A 275      19.151  26.824  33.044  1.00  0.60           N  \\nATOM   2093  OE1 GLN A 275      18.985  26.453  30.832  1.00  0.73           O  \\nATOM   2094  N   PRO A 276      13.313  27.591  32.542  1.00  0.81           N  \\nATOM   2095  CA  PRO A 276      12.874  28.797  33.247  1.00  0.83           C  \\nATOM   2096  C   PRO A 276      13.971  29.855  33.346  1.00  0.83           C  \\nATOM   2097  CB  PRO A 276      12.492  28.276  34.635  1.00  0.79           C  \\nATOM   2098  O   PRO A 276      15.155  29.516  33.429  1.00  0.79           O  \\nATOM   2099  CG  PRO A 276      13.223  26.979  34.770  1.00  0.76           C  \\nATOM   2100  CD  PRO A 276      13.496  26.439  33.396  1.00  0.75           C  \\nATOM   2101  N   PRO A 277      13.647  31.056  32.919  1.00  0.82           N  \\nATOM   2102  CA  PRO A 277      14.600  32.153  33.107  1.00  0.84           C  \\nATOM   2103  C   PRO A 277      15.309  32.093  34.458  1.00  0.85           C  \\nATOM   2104  CB  PRO A 277      13.723  33.403  33.005  1.00  0.80           C  \\nATOM   2105  O   PRO A 277      14.730  31.628  35.443  1.00  0.81           O  \\nATOM   2106  CG  PRO A 277      12.354  32.943  33.390  1.00  0.77           C  \\nATOM   2107  CD  PRO A 277      12.223  31.489  33.037  1.00  0.76           C  \\nATOM   2108  N   PRO A 278      16.605  32.188  34.450  1.00  0.84           N  \\nATOM   2109  CA  PRO A 278      17.367  32.296  35.696  1.00  0.86           C  \\nATOM   2110  C   PRO A 278      16.725  33.251  36.700  1.00  0.87           C  \\nATOM   2111  CB  PRO A 278      18.727  32.826  35.233  1.00  0.83           C  \\nATOM   2112  O   PRO A 278      16.072  34.220  36.305  1.00  0.84           O  \\nATOM   2113  CG  PRO A 278      18.439  33.561  33.964  1.00  0.80           C  \\nATOM   2114  CD  PRO A 278      17.253  32.922  33.301  1.00  0.78           C  \\nATOM   2115  N   PRO A 279      16.637  32.862  37.930  1.00  0.88           N  \\nATOM   2116  CA  PRO A 279      16.186  33.774  38.984  1.00  0.90           C  \\nATOM   2117  C   PRO A 279      16.865  35.140  38.912  1.00  0.90           C  \\nATOM   2118  CB  PRO A 279      16.569  33.041  40.272  1.00  0.87           C  \\nATOM   2119  O   PRO A 279      18.014  35.240  38.475  1.00  0.88           O  \\nATOM   2120  CG  PRO A 279      17.714  32.161  39.886  1.00  0.82           C  \\nATOM   2121  CD  PRO A 279      17.565  31.794  38.437  1.00  0.80           C  \\nATOM   2122  N   PRO A 280      16.135  36.206  39.053  1.00  0.90           N  \\nATOM   2123  CA  PRO A 280      16.718  37.545  39.161  1.00  0.91           C  \\nATOM   2124  C   PRO A 280      17.887  37.602  40.142  1.00  0.92           C  \\nATOM   2125  CB  PRO A 280      15.548  38.400  39.656  1.00  0.89           C  \\nATOM   2126  O   PRO A 280      17.919  36.843  41.114  1.00  0.89           O  \\nATOM   2127  CG  PRO A 280      14.652  37.443  40.374  1.00  0.84           C  \\nATOM   2128  CD  PRO A 280      14.809  36.085  39.752  1.00  0.80           C  \\nATOM   2129  N   PRO A 281      18.933  38.292  39.794  1.00  0.92           N  \\nATOM   2130  CA  PRO A 281      20.037  38.535  40.724  1.00  0.93           C  \\nATOM   2131  C   PRO A 281      19.562  39.040  42.085  1.00  0.93           C  \\nATOM   2132  CB  PRO A 281      20.874  39.599  40.009  1.00  0.91           C  \\nATOM   2133  O   PRO A 281      18.531  39.711  42.173  1.00  0.91           O  \\nATOM   2134  CG  PRO A 281      19.906  40.312  39.120  1.00  0.86           C  \\nATOM   2135  CD  PRO A 281      18.818  39.352  38.733  1.00  0.82           C  \\nATOM   2136  N   PRO A 282      20.208  38.590  43.100  1.00  0.93           N  \\nATOM   2137  CA  PRO A 282      19.933  39.153  44.424  1.00  0.94           C  \\nATOM   2138  C   PRO A 282      20.041  40.676  44.450  1.00  0.95           C  \\nATOM   2139  CB  PRO A 282      21.006  38.514  45.309  1.00  0.92           C  \\nATOM   2140  O   PRO A 282      20.813  41.258  43.683  1.00  0.93           O  \\nATOM   2141  CG  PRO A 282      22.127  38.189  44.376  1.00  0.88           C  \\nATOM   2142  CD  PRO A 282      21.555  37.940  43.010  1.00  0.84           C  \\nATOM   2143  N   PRO A 283      19.185  41.277  45.149  1.00  0.94           N  \\nATOM   2144  CA  PRO A 283      19.328  42.718  45.369  1.00  0.94           C  \\nATOM   2145  C   PRO A 283      20.715  43.103  45.878  1.00  0.95           C  \\nATOM   2146  CB  PRO A 283      18.257  43.022  46.420  1.00  0.93           C  \\nATOM   2147  O   PRO A 283      21.373  42.305  46.551  1.00  0.93           O  \\nATOM   2148  CG  PRO A 283      18.046  41.727  47.135  1.00  0.88           C  \\nATOM   2149  CD  PRO A 283      18.371  40.603  46.194  1.00  0.84           C  \\nATOM   2150  N   PRO A 284      21.218  44.164  45.423  1.00  0.94           N  \\nATOM   2151  CA  PRO A 284      22.464  44.696  45.980  1.00  0.94           C  \\nATOM   2152  C   PRO A 284      22.423  44.823  47.501  1.00  0.95           C  \\nATOM   2153  CB  PRO A 284      22.586  46.071  45.320  1.00  0.92           C  \\nATOM   2154  O   PRO A 284      21.352  45.022  48.079  1.00  0.93           O  \\nATOM   2155  CG  PRO A 284      21.183  46.459  44.982  1.00  0.88           C  \\nATOM   2156  CD  PRO A 284      20.382  45.208  44.762  1.00  0.84           C  \\nATOM   2157  N   PRO A 285      23.494  44.527  48.092  1.00  0.92           N  \\nATOM   2158  CA  PRO A 285      23.580  44.788  49.531  1.00  0.93           C  \\nATOM   2159  C   PRO A 285      23.203  46.223  49.894  1.00  0.93           C  \\nATOM   2160  CB  PRO A 285      25.051  44.513  49.852  1.00  0.91           C  \\nATOM   2161  O   PRO A 285      23.380  47.135  49.083  1.00  0.91           O  \\nATOM   2162  CG  PRO A 285      25.775  44.762  48.568  1.00  0.86           C  \\nATOM   2163  CD  PRO A 285      24.843  44.464  47.429  1.00  0.82           C  \\nATOM   2164  N   PRO A 286      22.567  46.400  50.908  1.00  0.89           N  \\nATOM   2165  CA  PRO A 286      22.328  47.756  51.411  1.00  0.90           C  \\nATOM   2166  C   PRO A 286      23.611  48.574  51.533  1.00  0.91           C  \\nATOM   2167  CB  PRO A 286      21.700  47.516  52.786  1.00  0.87           C  \\nATOM   2168  O   PRO A 286      24.686  48.014  51.764  1.00  0.88           O  \\nATOM   2169  CG  PRO A 286      22.197  46.170  53.203  1.00  0.83           C  \\nATOM   2170  CD  PRO A 286      22.462  45.355  51.970  1.00  0.80           C  \\nATOM   2171  N   PRO A 287      23.603  49.736  51.203  1.00  0.87           N  \\nATOM   2172  CA  PRO A 287      24.730  50.638  51.450  1.00  0.88           C  \\nATOM   2173  C   PRO A 287      25.218  50.592  52.896  1.00  0.88           C  \\nATOM   2174  CB  PRO A 287      24.159  52.017  51.110  1.00  0.85           C  \\nATOM   2175  O   PRO A 287      24.428  50.341  53.810  1.00  0.85           O  \\nATOM   2176  CG  PRO A 287      22.688  51.889  51.343  1.00  0.82           C  \\nATOM   2177  CD  PRO A 287      22.289  50.462  51.097  1.00  0.80           C  \\nATOM   2178  N   PRO A 288      26.461  50.600  53.020  1.00  0.89           N  \\nATOM   2179  CA  PRO A 288      26.997  50.740  54.376  1.00  0.90           C  \\nATOM   2180  C   PRO A 288      26.326  51.863  55.163  1.00  0.89           C  \\nATOM   2181  CB  PRO A 288      28.478  51.047  54.140  1.00  0.87           C  \\nATOM   2182  O   PRO A 288      26.190  52.980  54.657  1.00  0.87           O  \\nATOM   2183  CG  PRO A 288      28.548  51.527  52.726  1.00  0.83           C  \\nATOM   2184  CD  PRO A 288      27.324  51.049  51.999  1.00  0.81           C  \\nATOM   2185  N   GLN A 289      25.729  51.499  56.183  1.00  0.88           N  \\nATOM   2186  CA  GLN A 289      25.332  52.514  57.153  1.00  0.89           C  \\nATOM   2187  C   GLN A 289      26.508  53.416  57.517  1.00  0.88           C  \\nATOM   2188  CB  GLN A 289      24.763  51.860  58.413  1.00  0.85           C  \\nATOM   2189  O   GLN A 289      27.556  52.934  57.950  1.00  0.85           O  \\nATOM   2190  CG  GLN A 289      23.431  51.156  58.192  1.00  0.77           C  \\nATOM   2191  CD  GLN A 289      22.868  50.551  59.465  1.00  0.75           C  \\nATOM   2192  NE2 GLN A 289      21.726  49.882  59.349  1.00  0.77           N  \\nATOM   2193  OE1 GLN A 289      23.457  50.682  60.543  1.00  0.81           O  \\nATOM   2194  N   GLN A 290      26.475  54.681  57.082  1.00  0.90           N  \\nATOM   2195  CA  GLN A 290      27.328  55.721  57.646  1.00  0.91           C  \\nATOM   2196  C   GLN A 290      27.196  55.776  59.165  1.00  0.92           C  \\nATOM   2197  CB  GLN A 290      26.988  57.083  57.040  1.00  0.87           C  \\nATOM   2198  O   GLN A 290      26.084  55.771  59.698  1.00  0.89           O  \\nATOM   2199  CG  GLN A 290      27.557  57.294  55.643  1.00  0.80           C  \\nATOM   2200  CD  GLN A 290      27.270  58.679  55.095  1.00  0.77           C  \\nATOM   2201  NE2 GLN A 290      28.266  59.282  54.454  1.00  0.76           N  \\nATOM   2202  OE1 GLN A 290      26.162  59.204  55.246  1.00  0.81           O  \\nATOM   2203  N   PRO A 291      28.282  55.614  59.717  1.00  0.89           N  \\nATOM   2204  CA  PRO A 291      28.269  56.028  61.122  1.00  0.90           C  \\nATOM   2205  C   PRO A 291      27.947  57.510  61.297  1.00  0.89           C  \\nATOM   2206  CB  PRO A 291      29.691  55.717  61.594  1.00  0.87           C  \\nATOM   2207  O   PRO A 291      28.375  58.338  60.488  1.00  0.85           O  \\nATOM   2208  CG  PRO A 291      30.536  55.838  60.367  1.00  0.85           C  \\nATOM   2209  CD  PRO A 291      29.688  55.517  59.171  1.00  0.84           C  \\nATOM   2210  N   TRP A 292      27.182  57.815  62.156  1.00  0.88           N  \\nATOM   2211  CA  TRP A 292      26.880  59.166  62.617  1.00  0.90           C  \\nATOM   2212  C   TRP A 292      28.122  59.832  63.201  1.00  0.84           C  \\nATOM   2213  CB  TRP A 292      25.760  59.141  63.661  1.00  0.85           C  \\nATOM   2214  O   TRP A 292      28.932  59.178  63.862  1.00  0.76           O  \\nATOM   2215  CG  TRP A 292      24.473  58.558  63.159  1.00  0.70           C  \\nATOM   2216  CD1 TRP A 292      23.982  57.307  63.411  1.00  0.68           C  \\nATOM   2217  CD2 TRP A 292      23.515  59.203  62.315  1.00  0.73           C  \\nATOM   2218  CE2 TRP A 292      22.466  58.283  62.095  1.00  0.76           C  \\nATOM   2219  CE3 TRP A 292      23.441  60.471  61.723  1.00  0.73           C  \\nATOM   2220  NE1 TRP A 292      22.775  57.135  62.774  1.00  0.81           N  \\nATOM   2221  CH2 TRP A 292      21.307  59.839  60.740  1.00  0.77           C  \\nATOM   2222  CZ2 TRP A 292      21.355  58.592  61.307  1.00  0.79           C  \\nATOM   2223  CZ3 TRP A 292      22.334  60.777  60.939  1.00  0.77           C  \",\"pdb\");\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 1},{\"cartoon\": {\"color\": \"#4900b6\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 2},{\"cartoon\": {\"color\": \"#c80037\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 3},{\"cartoon\": {\"color\": \"#a80057\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 4},{\"cartoon\": {\"color\": \"#8f0070\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 5},{\"cartoon\": {\"color\": \"#7d0082\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 6},{\"cartoon\": {\"color\": \"#70008f\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 7},{\"cartoon\": {\"color\": \"#9f0060\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 8},{\"cartoon\": {\"color\": \"#bc0043\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 9},{\"cartoon\": {\"color\": \"#cd0032\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 10},{\"cartoon\": {\"color\": \"#c4003b\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 11},{\"cartoon\": {\"color\": \"#d1002e\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 12},{\"cartoon\": {\"color\": \"#db0024\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 13},{\"cartoon\": {\"color\": \"#e3001c\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 14},{\"cartoon\": {\"color\": \"#e80017\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 15},{\"cartoon\": {\"color\": \"#ec0013\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 16},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 17},{\"cartoon\": {\"color\": \"#f3000c\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 18},{\"cartoon\": {\"color\": \"#ec0013\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 19},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 20},{\"cartoon\": {\"color\": \"#fa0005\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 21},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 22},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 23},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 24},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 25},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 26},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 27},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 28},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 29},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 30},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 31},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 32},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 33},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 34},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 35},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 36},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 37},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 38},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 39},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 40},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 41},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 42},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 43},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 44},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 45},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 46},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 47},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 48},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 49},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 50},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 51},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 52},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 53},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 54},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 55},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 56},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 57},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 58},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 59},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 60},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 61},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 62},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 63},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 64},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 65},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 66},{\"cartoon\": {\"color\": \"#fb0004\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 67},{\"cartoon\": {\"color\": \"#f80007\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 68},{\"cartoon\": {\"color\": \"#f80007\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 69},{\"cartoon\": {\"color\": \"#f2000d\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 70},{\"cartoon\": {\"color\": \"#eb0014\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 71},{\"cartoon\": {\"color\": \"#eb0014\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 72},{\"cartoon\": {\"color\": \"#f3000c\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 73},{\"cartoon\": {\"color\": \"#f3000c\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 74},{\"cartoon\": {\"color\": \"#f3000c\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 75},{\"cartoon\": {\"color\": \"#f3000c\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 76},{\"cartoon\": {\"color\": \"#f0000f\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 77},{\"cartoon\": {\"color\": \"#ef0010\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 78},{\"cartoon\": {\"color\": \"#f60009\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 79},{\"cartoon\": {\"color\": \"#fb0004\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 80},{\"cartoon\": {\"color\": \"#fb0004\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 81},{\"cartoon\": {\"color\": \"#fb0004\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 82},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 83},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 84},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 85},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 86},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 87},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 88},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 89},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 90},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 91},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 92},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 93},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 94},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 95},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 96},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 97},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 98},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 99},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 100},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 101},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 102},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 103},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 104},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 105},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 106},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 107},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 108},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 109},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 110},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 111},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 112},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 113},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 114},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 115},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 116},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 117},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 118},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 119},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 120},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 121},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 122},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 123},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 124},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 125},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 126},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 127},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 128},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 129},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 130},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 131},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 132},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 133},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 134},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 135},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 136},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 137},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 138},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 139},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 140},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 141},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 142},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 143},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 144},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 145},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 146},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 147},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 148},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 149},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 150},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 151},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 152},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 153},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 154},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 155},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 156},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 157},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 158},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 159},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 160},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 161},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 162},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 163},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 164},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 165},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 166},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 167},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 168},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 169},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 170},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 171},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 172},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 173},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 174},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 175},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 176},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 177},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 178},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 179},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 180},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 181},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 182},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 183},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 184},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 185},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 186},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 187},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 188},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 189},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 190},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 191},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 192},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 193},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 194},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 195},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 196},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 197},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 198},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 199},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 200},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 201},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 202},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 203},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 204},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 205},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 206},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 207},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 208},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 209},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 210},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 211},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 212},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 213},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 214},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 215},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 216},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 217},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 218},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 219},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 220},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 221},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 222},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 223},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 224},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 225},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 226},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 227},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 228},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 229},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 230},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 231},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 232},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 233},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 234},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 235},{\"cartoon\": {\"color\": \"#ec0013\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 236},{\"cartoon\": {\"color\": \"#e1001e\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 237},{\"cartoon\": {\"color\": \"#d5002a\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 238},{\"cartoon\": {\"color\": \"#c5003a\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 239},{\"cartoon\": {\"color\": \"#b1004e\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 240},{\"cartoon\": {\"color\": \"#9a0065\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 241},{\"cartoon\": {\"color\": \"#7e0081\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 242},{\"cartoon\": {\"color\": \"#5e00a1\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 243},{\"cartoon\": {\"color\": \"#3a00c5\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 244},{\"cartoon\": {\"color\": \"#1000ef\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 245},{\"cartoon\": {\"color\": \"#1200ed\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 246},{\"cartoon\": {\"color\": \"#1400eb\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 247},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 248},{\"cartoon\": {\"color\": \"#0e00f1\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 249},{\"cartoon\": {\"color\": \"#1000ef\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 250},{\"cartoon\": {\"color\": \"#1100ee\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 251},{\"cartoon\": {\"color\": \"#1100ee\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 252},{\"cartoon\": {\"color\": \"#0f00f0\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 253},{\"cartoon\": {\"color\": \"#0e00f1\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 254},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 255},{\"cartoon\": {\"color\": \"#0c00f3\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 256},{\"cartoon\": {\"color\": \"#0b00f4\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 257},{\"cartoon\": {\"color\": \"#0b00f4\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 258},{\"cartoon\": {\"color\": \"#3700c8\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 259},{\"cartoon\": {\"color\": \"#5d00a2\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 260},{\"cartoon\": {\"color\": \"#7d0082\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 261},{\"cartoon\": {\"color\": \"#980067\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 262},{\"cartoon\": {\"color\": \"#af0050\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 263},{\"cartoon\": {\"color\": \"#c3003c\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 264},{\"cartoon\": {\"color\": \"#d3002c\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 265},{\"cartoon\": {\"color\": \"#e0001f\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 266},{\"cartoon\": {\"color\": \"#eb0014\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 267},{\"cartoon\": {\"color\": \"#f3000c\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 268},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 269},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 270},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 271},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 272},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 273},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 274},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 275},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 276},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 277},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 278},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 279},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 280},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 281},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 282},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 283},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 284},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 285},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 286},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 287},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 288},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 289},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.setStyle({\"resi\": 290},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17249326855875523.zoomTo();\n",
              "viewer_17249326855875523.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_52472be9-033f-4d3f-b3bc-2d9f37fcd1b6\", \"A.html\", 210008)"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "2024-08-29 11:58:05.603 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:05.606 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:05.608 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:05.610 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-29 11:58:05.612 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 600x100 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAfoAAABlCAYAAAClUiFrAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAAAWWElEQVR4nO3dfVTV9R0H8Pfl4V4gLgIaXECGCAFGpinpIUNGY4PcfNjOSVuGsNTWtJOD5UO5hmmgh7QsjkXDBO2wbGu4Y0ia2dRJuZ0hOI8yTEDTBrbOCh95vJ/9ofdyH34XuQRX+fl+ncO5/H7fp8/3w7184PrDn0ZEBERERKRKbjc7ACIiIho8LPREREQqxkJPRESkYiz0REREKsZCT0REpGIs9ERERCrGQk9ERKRiLPREREQq5jEYk7a1taGjo2MwpiYiIlI1rVYLLy+vAZtvwAt9W1sbIkeNQsv58wM9NRERkeoZDAY0NTUNWLEf8ELf0dGBlvPncfbkcfj5+l4/K4Dpf9o1/4e7pmPLR5s+5v+dVxSOrc+JUt9ex9uu04e1Hc0nRutjo9Gir02bZbxGo/J4y2PT50YHa1k+OpzH2LOmoz5K89uuYVQa21ubzbx28yms6XCfCsd2cUFhvn7E5TDHSvtU2P8N89bzPBK7NtiPVfpaO5rXtk3pdWa0ec2Yu14/b+ytr8LrzUEfpfnEZi3FWGxf032Jy8F817681rk1t6HnuGe4aTys1rROsfU5uzFW3yp64ri2pv1YsY1L4dg2HqOjvViuef2cbcrFHIVFH4s203nbp6QRNmuZPrEYZ/Oss3i0yLFtm8IeeuJS2N/1eE3xGG3jNK8jFn1t92nbV2GcQh9TTErjLfdidDDOuq9Fm8WaHRCUtbSgo6Pj1i30Jn56Pfz0+utHCt/8vkuht+qr8I3HdP6Ghd7JtftT6B0UrIEr9H3p299Cr7Av27G9/hBgM29f1rzRPpXmc7R2fwu9o8KsOJ/CGCfmsy/0CmO/S6FX3IPNa6aXwnzDQu/kfDcu9KIcx43i6kOh762gOizaGtu+9uPtC71C8dYor+10obctbkpF0sWFXpTigu2j2Mdj0Wa3BwdrGi0ebQuyfR+x6Ou4zXEfx4+O2qQPfRTnEeu2gcaL8YiIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxTwGa+ILFy8CItePpOdz0ynTJ2L5aNPHcrzdsfU5Uerb63jbdfqwtqP5xGh9bDRa9LVps4zXaFQeb3ls+tzoYC3LR4fzGHvWdNRHaX7bNYxKY3trs5nXbj6FNR3uU+HYLi4ozNePuBzmWGmfCvu/Yd56nkdi1wb7sUpfa0fz2rYpvc6MNq8Zc9fr54299VV4vTnoozSf2KylGIvta7ovcTmY79qX1zq35jb0HPcMN42H1ZrWKbY+ZzfG6ltFTxzX1rQfK7ZxKRzbxmN0tBfLNa+fs025mKOw6GPRZjpv+5Q0wmYt0ycW42yedRaPFjm2bVPYQ09cCvu7Hq8pHqNtnOZ1xKKv7T5t+yqMU+hjiklpvOVejA7GWfe1aLNYs6OnSA6YAS/0IgJfX1+Ex8QP9NRERESq5+vra/HD9Hc34IVeo9Hg0qVLOHv2LPz8/AZ6elJw4cIFhIeHM+cuxJy7HnPuesy565lyrtFoBmzOQXvr3s/Pj08MF2POXY85dz3m3PWY86GNF+MRERGpGAs9ERGRig14odfpdMjNzYVOpxvoqckB5tz1mHPXY85djzl3vcHIuUYG8tI+IiIiuqXwrXsiIiIVY6EnIiJSMRZ6IiIiFWOhJyIiUrF+FfpNmzZh1KhR8PLywuTJk/GPf/yj1/5/+tOfEBcXBy8vL4wdOxaVlZX9CvZ25kzOi4uLkZSUhICAAAQEBCA1NfWGXyOy5+zz3GT79u3QaDSYNWvW4AaoQs7m/Ntvv8XixYsREhICnU6HmJgYfn9xkrM537hxI2JjY+Ht7Y3w8HBkZ2ejra3NRdEObQcPHsT06dMRGhoKjUaDv/zlLzccs3//fkyYMAE6nQ7R0dEoLS11fmFx0vbt20Wr1cqWLVvk+PHjsnDhQvH395fz588r9q+qqhJ3d3cpKCiQEydOyG9/+1vx9PSUY8eOObv0bcvZnD/22GOyadMmqampkbq6OsnKypJhw4bJuXPnXBz50OVszk2ampokLCxMkpKSZObMma4JViWczXl7e7skJCTItGnT5NChQ9LU1CT79++X2tpaF0c+dDmb87KyMtHpdFJWViZNTU2yZ88eCQkJkezsbBdHPjRVVlbKypUrpby8XADIjh07eu3f2NgoPj4+kpOTIydOnJDCwkJxd3eX3bt3O7Wu04V+0qRJsnjxYvNxd3e3hIaGytq1axX7z549W3784x9bnZs8ebL88pe/dHbp25azObfV1dUler1etm7dOlghqk5/ct7V1SUPPPCAbN68WTIzM1noneRszt98800ZPXq0dHR0uCpE1XE254sXL5aHHnrI6lxOTo5MmTJlUONUo74U+mXLlkl8fLzVuTlz5khaWppTazn11n1HRweqq6uRmppqPufm5obU1FR89tlnimM+++wzq/4AkJaW5rA/WetPzm1duXIFnZ2dCAwMHKwwVaW/OV+9ejWCgoIwf/58V4SpKv3J+c6dO5GYmIjFixcjODgY99xzD/Lz89Hd3e2qsIe0/uT8gQceQHV1tfnt/cbGRlRWVmLatGkuifl2M1D106mb2nz99dfo7u5GcHCw1fng4GD8+9//VhzT0tKi2L+lpcWpQG9X/cm5reXLlyM0NNTuCUPK+pPzQ4cO4e2330Ztba0LIlSf/uS8sbERn3zyCebOnYvKykqcOnUKixYtQmdnJ3Jzc10R9pDWn5w/9thj+Prrr/Hggw9CRNDV1YWnnnoKzz//vCtCvu04qp8XLlzA1atX4e3t3ad5eNW9yq1btw7bt2/Hjh074OXldbPDUaWLFy8iIyMDxcXFGDFixM0O57ZhNBoRFBSE3//+95g4cSLmzJmDlStXoqio6GaHplr79+9Hfn4+3njjDRw5cgTl5eXYtWsX1qxZc7NDo1449Rv9iBEj4O7ujvPnz1udP3/+PAwGg+IYg8HgVH+y1p+cm6xfvx7r1q3Dxx9/jHvvvXcww1QVZ3Pe0NCA06dPY/r06eZzRqMRAODh4YH6+npERUUNbtBDXH+e5yEhIfD09IS7u7v53JgxY9DS0oKOjg5otdpBjXmo60/OX3jhBWRkZGDBggUAgLFjx+Ly5ct48sknsXLlSri58XfHgeSofvr5+fX5t3nAyd/otVotJk6ciH379pnPGY1G7Nu3D4mJiYpjEhMTrfoDwN69ex32J2v9yTkAFBQUYM2aNdi9ezcSEhJcEapqOJvzuLg4HDt2DLW1teaPGTNmICUlBbW1tQgPD3dl+ENSf57nU6ZMwalTp8w/VAHAyZMnERISwiLfB/3J+ZUrV+yKuekHLeFtUwbcgNVP564TvPbnGDqdTkpLS+XEiRPy5JNPir+/v7S0tIiISEZGhqxYscLcv6qqSjw8PGT9+vVSV1cnubm5/PM6Jzmb83Xr1olWq5X3339fmpubzR8XL168WVsYcpzNuS1ede88Z3P+xRdfiF6vl6efflrq6+uloqJCgoKC5KWXXrpZWxhynM15bm6u6PV6effdd6WxsVE++ugjiYqKktmzZ9+sLQwpFy9elJqaGqmpqREA8sorr0hNTY2cOXNGRERWrFghGRkZ5v6mP69bunSp1NXVyaZNm1zz53UiIoWFhfK9731PtFqtTJo0SQ4fPmxuS05OlszMTKv+f/zjHyUmJka0Wq3Ex8fLrl27+rPsbc2ZnEdERAgAu4/c3FzXBz6EOfs8t8RC3z/O5vzTTz+VyZMni06nk9GjR0teXp50dXW5OOqhzZmcd3Z2yqpVqyQqKkq8vLwkPDxcFi1aJN98843rAx+C/vrXvyp+bzblODMzU5KTk+3GjB8/XrRarYwePVpKSkqcXpe3qSUiIlIxXjlBRESkYiz0REREKsZCT0REpGIs9ERERCrGQk9ERKRiLPREREQqxkJPRESkYiz0RN9RVlYWZs2aZT7+/ve/j1//+tcuj2P//v3QaDT49ttvXb42Ed26WOhJlbKysqDRaKDRaKDVahEdHY3Vq1ejq6tr0NcuLy/v8928XF2cR40ahY0bNyq2nT59GhqNxuGtdktLS805dXNzw8iRI/GLX/wCX331lbnPgQMH8NBDDyEwMBA+Pj646667kJmZiY6ODocxHT16FDNmzEBQUBC8vLwwatQozJkzx2peIuo/FnpSrfT0dDQ3N+Pzzz/Hb37zG6xatQovv/yyYt/eCpGzAgMDodfrB2y+W4mfnx+am5tx7tw5FBcX48MPP0RGRgYA4MSJE0hPT0dCQgIOHjyIY8eOobCwEFqtFt3d3Yrz/fe//8UPfvADBAYGYs+ePairq0NJSQlCQ0Nx+fLlQdtHZ2fnoM1NdKthoSfV0ul0MBgMiIiIwK9+9SukpqZi586dAHrebs/Ly0NoaChiY2MBAGfPnsXs2bPh7++PwMBAzJw5E6dPnzbP2d3djZycHPj7+2P48OFYtmyZ3V27bN+6b29vx/LlyxEeHg6dTofo6Gi8/fbbOH36NFJSUgAAAQEB0Gg0yMrKAnDtLmJr165FZGQkvL29MW7cOLz//vtW61RWViImJgbe3t5ISUmxinOwaDQaGAwGhIaG4uGHH8YzzzyDjz/+GFevXsVHH30Eg8GAgoIC3HPPPYiKikJ6ejqKi4sd3lKzqqoKra2t2Lx5M+677z5ERkYiJSUFr776KiIjI839jh8/jp/85Cfw8/ODXq9HUlISGhoaAFzL1erVqzFy5EjodDqMHz8eu3fvNo81vVPx3nvvITk5GV5eXigrKwMAbN68GWPGjIGXlxfi4uLwxhtvDGL2iG4OFnq6bXh7e1v95r5v3z7U19dj7969qKioQGdnJ9LS0qDX6/G3v/0NVVVV8PX1RXp6unnchg0bUFpaii1btuDQoUP43//+hx07dvS67rx58/Duu+/i9ddfR11dHd566y34+voiPDwcf/7znwEA9fX1aG5uxmuvvQYAWLt2LbZt24aioiIcP34c2dnZePzxx3HgwAEA134g+dnPfobp06ejtrYWCxYswIoVKwYjbb3y9vaG0WhEV1cXDAYDmpubcfDgwT6PNxgM6Orqwo4dOxze5vTLL7/E1KlTodPp8Mknn6C6uhpPPPGE+Z9hXnvtNWzYsAHr16/Hv/71L6SlpWHGjBn4/PPPreZZsWIFlixZgrq6OqSlpaGsrAy/+93vkJeXh7q6OuTn5+OFF17A1q1b+58QolvRd7sXD9GtyfLucUajUfbu3Ss6nU6effZZc3twcLC0t7ebx7zzzjsSGxsrRqPRfK69vV28vb1lz549IiISEhIiBQUF5vbOzk4ZOXKk1Z3qkpOTZcmSJSIiUl9fLwBk7969inGa7mZlefevtrY28fHxkU8//dSq7/z58+XnP/+5iIg899xzcvfdd1u1L1++3G4uWxEREfLqq68qtjU1NQkAqampUWwvKSmRYcOGmY9PnjwpMTExkpCQICIiXV1dkpWVJQDEYDDIrFmzpLCwUFpbWx3GIyLy/PPPi4eHhwQGBkp6eroUFBSYb5Nq2mtkZKR0dHQojg8NDZW8vDyrc/fff78sWrTIal8bN2606hMVFSV/+MMfrM6tWbNGEhMTe42XaKjhb/SkWhUVFfD19YWXlxcefvhhzJkzB6tWrTK3jx07Flqt1nx89OhRnDp1Cnq9Hr6+vvD19UVgYCDa2trQ0NCA1tZWNDc3Y/LkyeYxHh4eSEhIcBhDbW0t3N3dkZyc3Oe4T506hStXruCHP/yhOQ5fX19s27bN/HZ1XV2dVRwAkJiY2Oc1+qu1tRW+vr7w8fFBbGwsgoODzW+Du7u7o6SkBOfOnUNBQQHCwsKQn5+P+Ph4NDc3O5wzLy8PLS0tKCoqQnx8PIqKihAXF4djx44BuJbDpKQkeHp62o29cOEC/vOf/2DKlClW56dMmYK6ujqrc5Zfp8uXL6OhoQHz58+3yvFLL71kzjGRWnjc7ACIBktKSgrefPNNaLVahIaGwsPD+ul+xx13WB1funQJEydONBcuS3feeWe/YnD0b9O9uXTpEgBg165dCAsLs2rT6XT9imOg6PV6HDlyBG5ubggJCVHcX1hYGDIyMpCRkYE1a9YgJiYGRUVFePHFFx3OO3z4cDzyyCN45JFHkJ+fj/vuuw/r16/H1q1b+5VDJZZfb1OOi4uL7X5gcnd3H5D1iG4VLPSkWnfccQeio6P73H/ChAl47733EBQUBD8/P8U+ISEh+Pvf/46pU6cCALq6ulBdXY0JEyYo9h87diyMRiMOHDiA1NRUu3bTOwqWV6Xffffd0Ol0+OKLLxy+EzBmzBjzhYUmhw8fvvEmvyM3NzenchoQEICQkBCnrqDXarWIiooyj7n33nuxdetWdHZ22v1W7+fnh9DQUFRVVVnlqqqqCpMmTXK4RnBwMEJDQ9HY2Ii5c+f2OTaioYiFnui6uXPn4uWXX8bMmTPNV3GfOXMG5eXlWLZsGUaOHIklS5Zg3bp1uOuuuxAXF4dXXnml17+BHzVqFDIzM/HEE0/g9ddfx7hx43DmzBl89dVXmD17NiIiIqDRaFBRUYFp06bB29sber0ezz77LLKzs2E0GvHggw+itbUVVVVV8PPzQ2ZmJp566ils2LABS5cuxYIFC1BdXY3S0tI+7fPLL7+0+1v5iIgI8+f19fV2Y+Lj428471tvvYXa2lr89Kc/RVRUFNra2rBt2zYcP34chYWFimMqKiqwfft2PProo4iJiYGI4IMPPkBlZSVKSkoAAE8//TQKCwvx6KOP4rnnnsOwYcNw+PBhTJo0CbGxsVi6dClyc3MRFRWF8ePHo6SkBLW1tYrvzFh68cUX8cwzz2DYsGFIT09He3s7/vnPf+Kbb75BTk7ODfdLNGTc7IsEiAaD5cV4zrQ3NzfLvHnzZMSIEaLT6WT06NGycOFC8wVlnZ2dsmTJEvHz8xN/f3/JycmRefPmObwYT0Tk6tWrkp2dLSEhIaLVaiU6Olq2bNlibl+9erUYDAbRaDSSmZkpItcuINy4caPExsaKp6en3HnnnZKWliYHDhwwj/vggw8kOjpadDqdJCUlyZYtW/p0MR4Au4933nnHfNGa0sfZs2ftLsazdeTIEXn88cclMjJSdDqdDB8+XKZOnSo7d+50OKahoUEWLlwoMTEx4u3tLf7+/nL//fdLSUmJVb+jR4/Kj370I/Hx8RG9Xi9JSUnS0NAgIiLd3d2yatUqCQsLE09PTxk3bpx8+OGH5rG9XWRYVlYm48ePF61WKwEBATJ16lQpLy93GC/RUKQRcfA3LURERDTk8ap7IiIiFWOhJyIiUjEWeiIiIhVjoSciIlIxFnoiIiIVY6EnIiJSMRZ6IiIiFWOhJyIiUjEWeiIiIhVjoSciIlIxFnoiIiIVY6EnIiJSsf8DMIiDYEQUuXYAAAAASUVORK5CYII=\n"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Downloading the results`\n",
        "\n",
        "if os.path.exists(\"d\"):\n",
        "    !rm -rf \"d\"\n",
        "if os.path.exists(\"text_dir\"):\n",
        "    !rm -rf \"text_dir\"\n",
        "\n",
        "!mkdir text_dir\n",
        "!mkdir d\n",
        "c = PPA(Sequence , model3)\n",
        "if Sequence != None and Sequence !=\"\" :\n",
        "  if ID == None:\n",
        "    c.name = \"Your peptide\"\n",
        "  with open(\"seq.txt\",\"w\") as w:\n",
        "          w.write(\"                                                 <<<<<<  LLPS prediction results  >>>>>>\\n\\n\\n\")\n",
        "          w.write(\"[Sequence id] : \" +c.name+\"\\n\\n\")\n",
        "          w.write(\"[Sequence length] : \"+str(len(c.seq))+\"\\n\\n\")\n",
        "          w.write(\"[Sequence_of_peptide] : \" +c.seq+\"\\n\\n\")\n",
        "          w.write(\"[Predicted_score] : \" +score +\"\\n\\n\")\n",
        "          w.close()\n",
        "\n",
        "\n",
        "\n",
        "  if c.name != None:\n",
        "    img_name = \"Info_of_sequenceID:\"+c.name+\".png\"\n",
        "    fig1.write_image(img_name)\n",
        "  else:\n",
        "    name = np.random.randint(1000000)\n",
        "    img_name = \"Info_of_sequenceID:\"+str(name)+\".png\"\n",
        "    fig1.write_image(img_name)\n",
        "\n",
        "  shutil.move(img_name, \"d\")\n",
        "  shutil.move(\"seq.txt\", \"text_dir\")\n",
        "  text_files_path = \"text_dir\"\n",
        "  png_files_path = \"d\"\n",
        "\n",
        "  zip_output_path = 'zipped_files.zip'\n",
        "\n",
        "  text_files = [f for f in os.listdir(text_files_path) if f.endswith('.txt')]\n",
        "  png_files = [f for f in os.listdir(png_files_path) if f.endswith('.png')]\n",
        "\n",
        "  with zipfile.ZipFile(zip_output_path, 'w') as zipf:\n",
        "    for text_file in text_files:\n",
        "      zipf.write(os.path.join(text_files_path, text_file), arcname=text_file)\n",
        "\n",
        "    for png_file in png_files:\n",
        "      zipf.write(os.path.join(png_files_path, png_file), arcname=png_file)\n",
        "\n",
        "  print(\"Files zipped successfully!\")\n",
        "\n",
        "\n",
        "else:\n",
        "  h = pd.DataFrame(D)\n",
        "  h.to_json(\"h.json\")\n",
        "  with open(\"results.txt\",\"w\") as w:\n",
        "    w.write(\"                                                 <<<<<<  LLPS prediction results  >>>>>>\\n\\n\\n\")\n",
        "    w.write(\"Number of sequences : \"+str(len(D[\"seq\"]))+\"\\n\\n\")\n",
        "    for i in range(len(D[\"id\"])):\n",
        "      w.write(\" [Sequence {i}]\".format(i = i)+\"\\n\\n\")\n",
        "      w.write(\"   Sequence id : \"+D[\"id\"][i]+\"\\n\\n\")\n",
        "      w.write(\"   Sequence length : \"+str(len(D[\"seq\"][i]))+\"\\n\\n\")\n",
        "      w.write(\"   Sequence : \" + D[\"seq\"][i]+\"\\n\\n\")\n",
        "      w.write(\"   Predicted Score : \" + str(D[\"score\"][i])+\"\\n\\n\")\n",
        "      w.write(\"-\"*150+\"\\n\\n\")\n",
        "    w.close()\n",
        "    shutil.move(\"h.json\", \"d\")\n",
        "    shutil.move(\"results.txt\", \"text_dir\")\n",
        "    text_files_path = \"text_dir\"\n",
        "    png_files_path = \"d\"\n",
        "    zip_output_path = 'zipped_files.zip'\n",
        "\n",
        "    text_files = [f for f in os.listdir(text_files_path) if f.endswith('.txt')]\n",
        "    png_files = [f for f in os.listdir(png_files_path) if f.endswith('.json')]\n",
        "\n",
        "    with zipfile.ZipFile(zip_output_path, 'w') as zipf:\n",
        "      for text_file in text_files:\n",
        "        zipf.write(os.path.join(text_files_path, text_file), arcname=text_file)\n",
        "\n",
        "      for png_file in png_files:\n",
        "        zipf.write(os.path.join(png_files_path, png_file), arcname=png_file)\n",
        "\n",
        "    print(\"Files zipped successfully!\")\n",
        "\n",
        "files.download(\"zipped_files.zip\")\n",
        "\n",
        "\n"
      ],
      "metadata": {
        "cellView": "form",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "id": "aUjOEsaH73ma",
        "outputId": "5796fe75-fbb8-4b53-e2b3-4cc4f2d256b5"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Files zipped successfully!\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_bdeb3bd8-b76e-4d16-887f-e665d17ed619\", \"zipped_files.zip\", 81660)"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "##NCBI BLAST+\n",
        "To determine the BLAST database, you can either provide a FASTA file path to create your own database or use the URL link to NCBI's protein databases available on the NCBI page: https://ftp.ncbi.nlm.nih.gov/blast/db/\n",
        "\n",
        "The DrLLPS database has already been provided. By setting the Blast_Database to DrLLPS, the BLAST search will be conducted against the DrLLPS database.\n",
        "\n",
        "To load your sequences, you can either input a single sequence or upload a FASTA format file containing your sequences into the designated box\n",
        "\n",
        "as example :\n",
        "  **MAVRVAKMTEREKLLLKKYGNRRLYDTKRSAYVTLSQVADIIKQGVDVEVVDAKTQEDVTAFILTQIVLEQARRKNALLPAELLHLIIRYGEDLLQEFFQKYLEQTIHGYIQFKKAMDEQFSKWIDMQLNYSDMAHKTLSGLTRFPNIFSPPPGSPGTKKDKG**\n",
        "\n",
        "or\n",
        "\n",
        "**Your_file.fasta**\n",
        "\n"
      ],
      "metadata": {
        "id": "jmcvJH0578AX"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#@title BLAST+ Tool\n",
        "\n",
        "\n",
        "import os\n",
        "#@markdown - Specify the sequence or the path of your FASTA format file in this box : like  `MAVRVAKMTEREKLL`  or `Your_file.fasta`\n",
        "\n",
        "Sequence = \"MAVRVAKMTEREKLLLKKYGNRRLYDTKRSAYVTLSQVADIIKQGVDVEVVDAKTQEDVTAFILTQIVLEQARRKNALLPAELLHLIIRYGEDLLQEFFQKYLEQTIHGYIQFKKAMDEQFSKWIDMQLNYSDMAHKTLSGLTRFPNIFSPPPGSPGTKKDKG\" #@param {type:\"string\"}\n",
        "#@markdown - Specify the fasta file or URL of your Database here :                    like  `https://ftp.ncbi.nlm.nih.gov/blast/db/swissprot.tar.gz`  or `Your_Database.fasta`\n",
        "\n",
        "Blast_Database = \"DrLLPS\" #@param [\"DrLLPS\",\"https://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/swissprot.gz\"] {allow-input : true}\n",
        "Blast_result_format = \"1\" #@param [1,2,3,4,5,6,7]\n",
        "Blast_result_file_name = \"Blast_result.txt\" #@param {type:\"string\"}\n",
        "E_value_threshold = \"100\" #@param {type:\"string\"}\n",
        "Max_target_Sequences = 6 #@param {type:\"integer\"}\n",
        "\n",
        "if Blast_Database == \"DrLLPS\":\n",
        "  Blast_Database = \"DrLLPS.fasta\"\n",
        "\n",
        "import sys\n",
        "from os import path\n",
        "\n",
        "if path.exists(\"/content/bioinformatics_stockholm/\"):\n",
        "  pass\n",
        "else:\n",
        "  !git clone https://github.com/Intertangler/bioinformatics_stockholm\n",
        "sys.path.insert(0,'/content')\n",
        "\n",
        "import importlib.util\n",
        "spec = importlib.util.spec_from_file_location(\"msa\", \"/content/bioinformatics_stockholm/msa/msa.py\")\n",
        "msa = importlib.util.module_from_spec(spec)\n",
        "spec.loader.exec_module(msa)\n",
        "\n",
        "spec = importlib.util.spec_from_file_location(\"miniblast\", \"/content/bioinformatics_stockholm/Miniblast/miniblast.py\")\n",
        "miniblast = importlib.util.module_from_spec(spec)\n",
        "spec.loader.exec_module(miniblast)\n",
        "\n",
        "def Fasta(fasta_path):\n",
        "  from command_runner import command_runner\n",
        "  command = \"makeblastdb -in {k} -dbtype prot -out blastdb\".format(k =fasta_path )\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  print(\"-- Blast Database has been created -- \\n\")\n",
        "\n",
        "\n",
        "if \".fasta\" in Blast_Database or \".txt\" in Blast_Database:\n",
        "  if os.path.exists(Blast_Database):\n",
        "    Fasta(Blast_Database)\n",
        "  with open(Blast_Database) as w:\n",
        "    j = w.read().split(\">\")[1:]\n",
        "  Dict = {i[:i.index(\"\\n\")].split(\" \")[0] if \"\\n\" in i else i : edit(i[i.index(\"\\n\"):]) if \"\\n\" in i else None for i in j}\n",
        "\n",
        "else:\n",
        "  command = \"wget {k}\".format(k = Blast_Database)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  print(\"-- Blast Database has been downloaded -- \\n\")\n",
        "  exit_code, output = command_runner(\"gunzip {l}\".format(l = Blast_Database.split(\"/\")[-1]), shell=True, live_output=True)\n",
        "  Fasta(Blast_Database.split(\"/\")[-1][:-3])\n",
        "  db = miniblast.read_database_fasta(Blast_Database.split(\"/\")[-1][:-3])\n",
        "  Dict = {i.id :\"\".join(list(i.seq)) for i in db}\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "cellView": "form",
        "id": "hXJaSuNL742n",
        "outputId": "16357f41-7cfe-4f5a-f8a7-29035fc199c8"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Cloning into 'bioinformatics_stockholm'...\n",
            "remote: Enumerating objects: 296, done.\u001b[K\n",
            "remote: Total 296 (delta 0), reused 0 (delta 0), pack-reused 296 (from 1)\u001b[K\n",
            "Receiving objects: 100% (296/296), 42.10 MiB | 21.85 MiB/s, done.\n",
            "Resolving deltas: 100% (127/127), done.\n",
            "\n",
            "\n",
            "Building a new DB, current time: 08/29/2024 11:53:18\n",
            "New DB name:   /content/blastdb\n",
            "New DB title:  DrLLPS.fasta\n",
            "Sequence type: Protein\n",
            "Keep MBits: T\n",
            "Maximum file size: 1000000000B\n",
            "Adding sequences from FASTA; added 1272 sequences in 0.0364358 seconds.\n",
            "\n",
            "\n",
            "-- Blast Database has been created -- \n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Blast Result\n",
        "\n",
        "blosum62 = {\n",
        "    ('W', 'F'): 1, ('L', 'R'): -2, ('S', 'P'): -1, ('V', 'T'): 0,\n",
        "    ('Q', 'Q'): 5, ('N', 'A'): -2, ('Z', 'Y'): -2, ('W', 'R'): -3,\n",
        "    ('Q', 'A'): -1, ('S', 'D'): 0, ('H', 'H'): 8, ('S', 'H'): -1,\n",
        "    ('H', 'D'): -1, ('L', 'N'): -3, ('W', 'A'): -3, ('Y', 'M'): -1,\n",
        "    ('G', 'R'): -2, ('Y', 'I'): -1, ('Y', 'E'): -2, ('B', 'Y'): -3,\n",
        "    ('Y', 'A'): -2, ('V', 'D'): -3, ('B', 'S'): 0, ('Y', 'Y'): 7,\n",
        "    ('G', 'N'): 0, ('E', 'C'): -4, ('Y', 'Q'): -1, ('Z', 'Z'): 4,\n",
        "    ('V', 'A'): 0, ('C', 'C'): 9, ('M', 'R'): -1, ('V', 'E'): -2,\n",
        "    ('T', 'N'): 0, ('P', 'P'): 7, ('V', 'I'): 3, ('V', 'S'): -2,\n",
        "    ('Z', 'P'): -1, ('V', 'M'): 1, ('T', 'F'): -2, ('V', 'Q'): -2,\n",
        "    ('K', 'K'): 5, ('P', 'D'): -1, ('I', 'H'): -3, ('I', 'D'): -3,\n",
        "    ('T', 'R'): -1, ('P', 'L'): -3, ('K', 'G'): -2, ('M', 'N'): -2,\n",
        "    ('P', 'H'): -2, ('F', 'Q'): -3, ('Z', 'G'): -2, ('X', 'L'): -1,\n",
        "    ('T', 'M'): -1, ('Z', 'C'): -3, ('X', 'H'): -1, ('D', 'R'): -2,\n",
        "    ('B', 'W'): -4, ('X', 'D'): -1, ('Z', 'K'): 1, ('F', 'A'): -2,\n",
        "    ('Z', 'W'): -3, ('F', 'E'): -3, ('D', 'N'): 1, ('B', 'K'): 0,\n",
        "    ('X', 'X'): -1, ('F', 'I'): 0, ('B', 'G'): -1, ('X', 'T'): 0,\n",
        "    ('F', 'M'): 0, ('B', 'C'): -3, ('Z', 'I'): -3, ('Z', 'V'): -2,\n",
        "    ('S', 'S'): 4, ('L', 'Q'): -2, ('W', 'E'): -3, ('Q', 'R'): 1,\n",
        "    ('N', 'N'): 6, ('W', 'M'): -1, ('Q', 'C'): -3, ('W', 'I'): -3,\n",
        "    ('S', 'C'): -1, ('L', 'A'): -1, ('S', 'G'): 0, ('L', 'E'): -3,\n",
        "    ('W', 'Q'): -2, ('H', 'G'): -2, ('S', 'K'): 0, ('Q', 'N'): 0,\n",
        "    ('N', 'R'): 0, ('H', 'C'): -3, ('Y', 'N'): -2, ('G', 'Q'): -2,\n",
        "    ('Y', 'F'): 3, ('C', 'A'): 0, ('V', 'L'): 1, ('G', 'E'): -2,\n",
        "    ('G', 'A'): 0, ('K', 'R'): 2, ('E', 'D'): 2, ('Y', 'R'): -2,\n",
        "    ('M', 'Q'): 0, ('T', 'I'): -1, ('C', 'D'): -3, ('V', 'F'): -1,\n",
        "    ('T', 'A'): 0, ('T', 'P'): -1, ('B', 'P'): -2, ('T', 'E'): -1,\n",
        "    ('V', 'N'): -3, ('P', 'G'): -2, ('M', 'A'): -1, ('K', 'H'): -1,\n",
        "    ('V', 'R'): -3, ('P', 'C'): -3, ('M', 'E'): -2, ('K', 'L'): -2,\n",
        "    ('V', 'V'): 4, ('M', 'I'): 1, ('T', 'Q'): -1, ('I', 'G'): -4,\n",
        "    ('P', 'K'): -1, ('M', 'M'): 5, ('K', 'D'): -1, ('I', 'C'): -1,\n",
        "    ('Z', 'D'): 1, ('F', 'R'): -3, ('X', 'K'): -1, ('Q', 'D'): 0,\n",
        "    ('X', 'G'): -1, ('Z', 'L'): -3, ('X', 'C'): -2, ('Z', 'H'): 0,\n",
        "    ('B', 'L'): -4, ('B', 'H'): 0, ('F', 'F'): 6, ('X', 'W'): -2,\n",
        "    ('B', 'D'): 4, ('D', 'A'): -2, ('S', 'L'): -2, ('X', 'S'): 0,\n",
        "    ('F', 'N'): -3, ('S', 'R'): -1, ('W', 'D'): -4, ('V', 'Y'): -1,\n",
        "    ('W', 'L'): -2, ('H', 'R'): 0, ('W', 'H'): -2, ('H', 'N'): 1,\n",
        "    ('W', 'T'): -2, ('T', 'T'): 5, ('S', 'F'): -2, ('W', 'P'): -4,\n",
        "    ('L', 'D'): -4, ('B', 'I'): -3, ('L', 'H'): -3, ('S', 'N'): 1,\n",
        "    ('B', 'T'): -1, ('L', 'L'): 4, ('Y', 'K'): -2, ('E', 'Q'): 2,\n",
        "    ('Y', 'G'): -3, ('Z', 'S'): 0, ('Y', 'C'): -2, ('G', 'D'): -1,\n",
        "    ('B', 'V'): -3, ('E', 'A'): -1, ('Y', 'W'): 2, ('E', 'E'): 5,\n",
        "    ('Y', 'S'): -2, ('C', 'N'): -3, ('V', 'C'): -1, ('T', 'H'): -2,\n",
        "    ('P', 'R'): -2, ('V', 'G'): -3, ('T', 'L'): -1, ('V', 'K'): -2,\n",
        "    ('K', 'Q'): 1, ('R', 'A'): -1, ('I', 'R'): -3, ('T', 'D'): -1,\n",
        "    ('P', 'F'): -4, ('I', 'N'): -3, ('K', 'I'): -3, ('M', 'D'): -3,\n",
        "    ('V', 'W'): -3, ('W', 'W'): 11, ('M', 'H'): -2, ('P', 'N'): -2,\n",
        "    ('K', 'A'): -1, ('M', 'L'): 2, ('K', 'E'): 1, ('Z', 'E'): 4,\n",
        "    ('X', 'N'): -1, ('Z', 'A'): -1, ('Z', 'M'): -1, ('X', 'F'): -1,\n",
        "    ('K', 'C'): -3, ('B', 'Q'): 0, ('X', 'B'): -1, ('B', 'M'): -3,\n",
        "    ('F', 'C'): -2, ('Z', 'Q'): 3, ('X', 'Z'): -1, ('F', 'G'): -3,\n",
        "    ('B', 'E'): 1, ('X', 'V'): -1, ('F', 'K'): -3, ('B', 'A'): -2,\n",
        "    ('X', 'R'): -1, ('D', 'D'): 6, ('W', 'G'): -2, ('Z', 'F'): -3,\n",
        "    ('S', 'Q'): 0, ('W', 'C'): -2, ('W', 'K'): -3, ('H', 'Q'): 0,\n",
        "    ('L', 'C'): -1, ('W', 'N'): -4, ('S', 'A'): 1, ('L', 'G'): -4,\n",
        "    ('W', 'S'): -3, ('S', 'E'): 0, ('H', 'E'): 0, ('S', 'I'): -2,\n",
        "    ('H', 'A'): -2, ('S', 'M'): -1, ('Y', 'L'): -1, ('Y', 'H'): 2,\n",
        "    ('Y', 'D'): -3, ('E', 'R'): 0, ('X', 'P'): -2, ('G', 'G'): 6,\n",
        "    ('G', 'C'): -3, ('E', 'N'): 0, ('Y', 'T'): -2, ('Y', 'P'): -3,\n",
        "    ('T', 'K'): -1, ('A', 'A'): 4, ('P', 'Q'): -1, ('T', 'C'): -1,\n",
        "    ('V', 'H'): -3, ('T', 'G'): -2, ('I', 'Q'): -3, ('Z', 'T'): -1,\n",
        "    ('C', 'R'): -3, ('V', 'P'): -2, ('P', 'E'): -1, ('M', 'C'): -1,\n",
        "    ('K', 'N'): 0, ('I', 'I'): 4, ('P', 'A'): -1, ('M', 'G'): -3,\n",
        "    ('T', 'S'): 1, ('I', 'E'): -3, ('P', 'M'): -2, ('M', 'K'): -1,\n",
        "    ('I', 'A'): -1, ('P', 'I'): -3, ('R', 'R'): 5, ('X', 'M'): -1,\n",
        "    ('L', 'I'): 2, ('X', 'I'): -1, ('Z', 'B'): 1, ('X', 'E'): -1,\n",
        "    ('Z', 'N'): 0, ('X', 'A'): 0, ('B', 'R'): -1, ('B', 'N'): 3,\n",
        "    ('F', 'D'): -3, ('X', 'Y'): -1, ('Z', 'R'): 0, ('F', 'H'): -1,\n",
        "    ('B', 'F'): -3, ('F', 'L'): 0, ('X', 'Q'): -1, ('B', 'B'): 4\n",
        "}\n",
        "\n",
        "Blosum62 = {}\n",
        "for i in blosum62:\n",
        "  Blosum62[i] = blosum62[i]\n",
        "  Blosum62[i[::-1]] = blosum62[i]\n",
        "if \".\" not in Sequence:\n",
        "  with open(\"sequence.txt\",\"w\") as w:\n",
        "    w.write(\">seq1\\n{o}\".format(o = Sequence))\n",
        "    w.close()\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = \"sequence.txt\",d = \"blastdb\"\n",
        "  ,f = Blast_result_format,o = Blast_result_file_name , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  files.download(Blast_result_file_name)\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = \"sequence.txt\",d = \"blastdb\"\n",
        "  ,f = 6 ,o = \"results_dict.txt\" , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "\n",
        "  with open(\"results_dict.txt\") as w:\n",
        "    y = w.read()\n",
        "  idx = [i.split(\"\\t\")[1] for i in y.split(\"\\nseq\")]\n",
        "  for i in idx:\n",
        "    target_sequence = Dict[i]\n",
        "    aln = pairwise2.align.localds(Sequence,target_sequence, match_dict = Blosum62, open = -3, extend = -1)[0]\n",
        "    print(\"Subject_Id : \"+i+\"\\n\")\n",
        "    print(pairwise2.format_alignment(align1=aln[0], align2=aln[1], score=aln[2], begin=aln[3], end=aln[4], full_sequences=True))\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "if \".\" in Sequence:\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = Sequence,d = \"blastdb\"\n",
        "  ,f = Blast_result_format,o = Blast_result_file_name , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  files.download(Blast_result_file_name)"
      ],
      "metadata": {
        "cellView": "form",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 887
        },
        "id": "Og5pElPc--eM",
        "outputId": "77bdbc13-106c-4e40-b5d1-00f60bc9ac93"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_8e0c452d-cd65-4edd-8006-ace85a578dcc\", \"Blast_result.txt\", 3234)"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Subject_Id : seq925\n",
            "\n",
            "--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MAVRVAKMTEREKLLLKKYG--NRRL------YDTKRSAY----VTLSQ--VADIIKQGVD-V-EVVDAKTQEDVTAF-ILTQIVLEQARRKNALLPAEL--L----HLIIRYGE-DLLQEFFQKYLEQTIHGYIQFKKAMDEQFSKWIDMQLNYSDMAHKT-LSGLTRFPNIFSPPPGSP-GTKKDKG---------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |.|.....|.   ...|  .||.      |.....||    .||.|  .. ..|...| . ||      ..| .. .| ..|||  |    ..|..|  .    |..|.  | |  | .. |     .|....||....|....|..|.|...| |... |..||.  ||.|.....| | ...|                                                                                                                \n",
            "MAAESALQVVEKLQARLAANPDPKKLLKYLKKLSILPITVDILVETGVGKTVNSFRKHEQVGNFARDLVAQWKKLVPVERNSEAEDQDFEKNNSRKRPRDALQREEELEGNYQESWKPSGSRSYSPEHRQKKHKKLSEPERPHKVAHSHEKRDERKRCHKVSPPYSSDPESSDYGHVQSPPPSSPHQMYTDLSRSPEEDQEPIISHQKPGKVHSNTFQDRLGVSHLGEQGKGAVSHHKQHRSSHKEKHPADAREDEKISAVSREKSHKASSKEESRRLLSGDSAKEKLPSSVVKKDKDREGSSLKKKFSPALDVASDNHFKKPKHKDSEKAKSDKNKQSVDGVDSGRGTGDPLPKAKEKVPNHLKAQEGKVRTNADGKSAGPLHPKAEETDVDDEFERPTMSFESYLSYDQPRKKKKKVVKTSSTALGEKGLKKKDSKSTSKNLNSAQKLPKVNENKSEKLQPAGAEPTRPRKVPTDVLPALPDIPLPAIHANYRPLPSLELIPSFQPKRKAFSSPQEE---EEAGFTGRRMNSKMQVYSGSKCAYLPKMMTLHQQCIR-VLKNNIDSIFEV------GGV-PYSVL-EPVLE--R----CTPDQLYRIEECNHVLIE--ETD--Q-LW-K-----VHCHRDFKEERPEEYESWREMYLRLQD-AREQRLRLLTN--NIRSAHANKPKG-RQAKMAFVNSVAKPPRDVRRRQEKFGTGGAAVPEKVRIKPAPYTTGSSHVPASNSSSNFHSSPEELAYDGPSTSSAHLAPVASSSVSYDPRKPAVKKIAPMMAKTIKAFKNRFSRR\n",
            "  Score=141\n",
            "\n",
            "Subject_Id : seq245\n",
            "\n",
            "--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MAVRV---AKMTEREKLLLKKY-GNRRLYDTKRSAYVTLSQVADIIKQGVDVEVV-DAKTQE-DVTAFILTQIV-LE-QA-RR-KNA-LLPAELLHLIIRYGEDLLQE----FFQKYLEQTIHGY--I-QFKKAMDE-QFSKWIDMQLNYSDMAHKTLSGLTRFPNIFSPPPGSPGTKK--DKG----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                            ||.   |..|...  |.|.| ..|||.|   ||   |..| ||...|  |.|. |.|..| |.....| |.. .| .| .| ..| ......|.| .|..|..|||    ..|.|.|.. ||.  . |.||..|| . .|..|....| |.|  |...| |.   |.    .|..||  .|                                                                                                                                                                                                                                                                                                                                                                                                     \n",
            "MNDQTQFTERALTILTLAQKLASDHQHPQLQPIHILAAFIETPEDGSVPYLQNLIEKGRYDYDLFKKVVNRNLVRIPQQQPAPAEITPSYALGKVLQDAAKIQKQQKDSFIAQDHILFALFNDSSIQQIFKEAQVDIEAIKQQALELRGNTRIDSRGADTNTPLEYLSKYAIDMTEQARQGKLDPVIGREEEIRSTIRVLARRIKSNPCLIGEPGIGKTAIIEGVAQRIIDDDVPTILQGAKLFSLDLAALTAGAKYKGDFEERFKGVLKEIEESKTLIVLFIDEIHMLMGNGKDDAANILKPALSRGQLKVIGATTNNEYRSIVEKDGAFERRFQKIEVAEPSVRQTVAILRGLQPKYEIHHGVRILDSALVTAAQ--LAKRYLPYRRLPD---SA---LDLV-DISCAG--VAVARDSKPEELDSKERQL-QLIQVEIKALERDEDADSTTKDRLKL-ARQKEASLQEELEPLRQRYNEEK-HGHEELTQAKKKLDELE-NKALDAERRY-DTA--TAADL-RY---FA----IPDIKKQIEKLEDQVAEEERRAGANSMIQNVVDSDTISETAARLTGIPVKKLSESENEKLIHMERDLSSEVVGQMDAIKAVSNAVRLSRSGLANPRQPASFLFLGLSGSGKTELAKKVAGFLFNDEDMMIRVDCSELSEKYAVSKLLGTTAGYVGYDEGGFLTNQLQYKPYSVLLFDEVEKAHPDVLTVMLQMLDDGRITSGQGKTIDCSNCIVIMTSNLGAEFINSQQGSKIQESTKNLVMGAVRQHFRPEFLNRISSIVIFNKLSRKAIHKIVDIRLKEIEERFEQNDKHYKLNLTQEAKDFLAKYGYSDDMGARPLNRLIQNEILNKLALRILKNEIKDKETVNVVLKKGKSRDENVPEEAEECLEVLPNHEATIGADTLGDDDNEDSMEIDDDLD\n",
            "  Score=174\n",
            "\n",
            "Subject_Id : seq108\n",
            "\n",
            "---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MAVRVAK-MT--EREKLL--LKKY-G-----NRRLYDTKR--SAYVTLSQVADIIK-QGVD--V----EVVD-AKTQEDVTAF-ILTQI-VLEQARRKNALLPAELLHLIIRYG--EDLLQEFFQKYLEQTIHG---YI---QFKKAMD---EQFSKWIDM--QLNYSD---MAHKTLSGLT-RF-PNIF---SPP---P--G-SP-------GTKKDKG----------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                       ..||. ..  ||...|  .|.| |     |.|....|.  .|| .|...|   | |||.  |    ..|| .|..|..... ..|.. |.   |..|.  ......... ..  |........|. .|...|   |.   | .|  |   .|....|..  |..|..   .|.....|.. .| |..|   .||   |  | .|       |..|. |                                                                                                          \n",
            "MGSSHHHHHHHHASENLYFQSADITDKTAEQLENLNIQDDQKQAATGSESQSVENSSASLYVGDLEPSVSEAHLYDIFSPIGSVSSIRVCRDAITKTSLGYAYVNFNDHEAGRKAIEQLNYTPIKGRLCRIMWSQRDPSLRKKGSGNIFIKNLHPDIDNKALYDTFSVFGDILSSKIATDENGKSKGFGFVHFEEEGAAKEAIDALNGMLLNGQEIYVAPHLSRKERDSQLEETKLYVGRAQKKNERMHVLKKQYEAY-RLEKMA---KYQGVNLFVKNLDDSVDDEKLEEEFAPYGTITSAKVM---RTENG--KSKGFGFVC-FSTPEEATKAITEKN-QQIVAGKPLYVAIAQ-RK--DVRRSQLAQQIQARNQMRYQQATAAAAAAAAGMPGQFMPPMFYGVMPPRGVPFNGPNPQQMNPMGGMPKN-GMPPQFRNGPVYGVPPQGGFPRNANDNNQFYQQKQRQALGEQLYKKVSAKTSNEEAAGKITGMILDLPPQEVFPLLESDELFEQHYKEASAAYESFKKEQEQQTEQA\n",
            "  Score=155\n",
            "\n",
            "Subject_Id : seq1231\n",
            "\n",
            "---------------------------------------------------------------------------------------------------------------------------MAVRVAK-M-TEREKLLLKK-YGN---RRLY---DTKRSA-YVTLSQV-ADII--KQGVDVEVVDAKTQEDVTAFILTQIVLEQARRKNALLPAELLH-LIIRYGEDLLQEFFQKYLEQTIHGYIQFKKAMDEQFSKWIDMQLNYSDMAH-KTL-SGLT--RFPNIFSP--PPGSPGTKKD-----KG-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                              |... | ..|...|.|| .||   ..|.   |.|  | |.|.| | .||.  |...| |  ..|...  ..|....   |....|.|.   ..|. .... |    ||.   |...  |  .. .|..|.|..   .....|... . |.. |..|  .|...|..  |..|....||     ||                                                                                                                                                                                                                                         \n",
            "MGSSHHHHHHHHASENLYFQSADITDKTAEQLENLNIQDDQKQAATGSESQSVENSSASLYVGDLEPSVSEAHLYDIFSPIGSVSSIRVCRDAITKTSLGYAYVNFNDHEAGRKAIEQLNYTPIKGRLCRIMWSQRDPSLRKKGSGNIFIKNLHPDIDNK--ALYDTFS-VFGDILSSKIATD-E--NGKSKG--FGFVHFE---EEGAAKEAI---DALNGMLLN-G----QEI---YVAP--H--LS-RKERDSQLE---ETKAHYTNL-YVKNINSETTDEQFQELFAKFGPIVSASLEKDADGKLKGFGFVNYEKHEDAVKAVEALNDSELNGEKLYVGRAQKKNERMHVLKKQYEAYRLEKMAKYQGQLAQQIQARNQMRYQQATAAAAAAAAGMPGQFMPPMFYGVMPPRGVPFNGPNPQQMNPMGGMPKNGMPPQFRNGPVYGVPPQGGFPRNANDNNQFYQQKQRQALGEQLYKKVSAKTSNEEAAGKITGMILDLPPQEVFPLLESDELFEQHYKEASAAYESFKKEQEQQTEQA\n",
            "  Score=147\n",
            "\n",
            "Subject_Id : seq884\n",
            "\n",
            "-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MAVRVAKMTER-------EKL-LLK-K--YGNRRLYDTKRSAYVTLSQVADIIKQGVDVEVVDAKTQED--V-----TAFILTQIVLE----QARRKNALLPAELLHLIIRYGEDL-L----QEFFQ-KYLE--QT-IH-G-----YIQFKKAMD-EQFSKWID--MQLNYSDMAH--KTLSG---LTRFPNI---FSPPPGSPGT-K---KD-KG--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |.|.||.       ||. ..| |  ..|  | |....|.||    |...|...||| ||  |.||  |     |.||  ....|    .| .|.|...|| |.   ...||. .    .|... |..|  .| |. |     ... ||  | |...| .|  |. | |..|.  |...|   .... ||   |......|.. |   || ||                                                                                                                                                                                                                                            \n",
            "MVESLTVENQEHNVQPPSVTSAGDSYSTLATDLPLPSTNDIIESRDQLTESDLDEAINATENFAQELSSQRKSSKLKGHKKKNQGQIKANRDRDTIVKLSSSVGETEEASTRDAISHDLERKDDVIEIATDTINDATESPTQIPIDVNVVIKETSTNNVAEGTENVPPIKESTGIEVGNSPITRRKKNKKKKTTNRRGRNSSNPADTTDLSKQSTLDSILVGIEEYLQEDGSKNEDIKVNIVQDEPVNVEKMDIRTRNESSDKTFDIDVPNKDNVDETSSKSENNINEEKAEHTLPREENEILNVNEGNAASFKHQLEPHGLEAGDENGQASTKDVESESLTKNGFNFKENESKHLKAGEKQQTESDRDGISPSVLAKNQKETEIGKEDHVFEQKDKEDEKCRKELSVNHENNMSHNFNAAGSDSIIPPETERETYDDETMGPTKRISDNEKNLQHGTNDISVEVEKEEEEEEEEEENSTFSKVKKENVTGEQEAVRNNEVSGTEEESTSKGEEIMGGDEKQSEAGEKSSIIEIEGSANSAKISKDNLVLEDEAEAPTQENKPTEVVGEIDIPDAPRDDVEIVEAVEKNIIPEDLEVAKEDQEGEQVKLDEPVKAMKDDKIAMRGAESISEDMKKKQEGTAELSNEKAKKEVDETARESAEGVEVEKSKTPESPKVVKRCTSGRPEDLQINERDPEVLKEDVRVPDEDVKPEIATTIENSEEEDPKSQRVQISTEQAETTQKDMGDVGSTTSFKEEEKPKRFEITQEGDKITGKDTNHEHGEATEAASENSKASDVGTAEKYIEPSSESVKKDTEEDAEVENSEKTEFIKVKAELEN--L-DAPKEAEVT----AELNKENEDVE-VD--TEEDAEVENSEKTEFI--KVKAELGNLDA-PKEAEVTAE-LN---KENEDVEVAATSKEDIETKCSEPAETPIEDGTCTEAEVS-KK--DAEAVTK-EDENME-N-SKIAEALKDVTGDQEIDDI-NISDEFQRTVELPELEKQDIKDNKGEDKELEVEETEKETSLPDLVVEENITEEKNEIKQEEEEVSQLDFNETESISKEAPNNDENGFEDQSTRENPKKASADDIFKDILDETNEFLEQLKIVDDSELNALLQSLDAKDSTTQTTEQSKKNNDKPQDVITTSEIRKLNEKEPVYIYTSLAGGGFHMIPRTNRLSTILTANRIPFTYRDLGTDDEARKVWKTFSKGRSLPGVVRGHNDLIGNWEEIEEANEDYKLRELIYDTI\n",
            "  Score=149\n",
            "\n",
            "Subject_Id : seq884\n",
            "\n",
            "-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MAVRVAKMTER-------EKL-LLK-K--YGNRRLYDTKRSAYVTLSQVADIIKQGVDVEVVDAKTQED--V-----TAFILTQIVLE----QARRKNALLPAELLHLIIRYGEDL-L----QEFFQ-KYLE--QT-IH-G-----YIQFKKAMD-EQFSKWID--MQLNYSDMAH--KTLSG---LTRFPNI---FSPPPGSPGT-K---KD-KG--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |.|.||.       ||. ..| |  ..|  | |....|.||    |...|...||| ||  |.||  |     |.||  ....|    .| .|.|...|| |.   ...||. .    .|... |..|  .| |. |     ... ||  | |...| .|  |. | |..|.  |...|   .... ||   |......|.. |   || ||                                                                                                                                                                                                                                            \n",
            "MVESLTVENQEHNVQPPSVTSAGDSYSTLATDLPLPSTNDIIESRDQLTESDLDEAINATENFAQELSSQRKSSKLKGHKKKNQGQIKANRDRDTIVKLSSSVGETEEASTRDAISHDLERKDDVIEIATDTINDATESPTQIPIDVNVVIKETSTNNVAEGTENVPPIKESTGIEVGNSPITRRKKNKKKKTTNRRGRNSSNPADTTDLSKQSTLDSILVGIEEYLQEDGSKNEDIKVNIVQDEPVNVEKMDIRTRNESSDKTFDIDVPNKDNVDETSSKSENNINEEKAEHTLPREENEILNVNEGNAASFKHQLEPHGLEAGDENGQASTKDVESESLTKNGFNFKENESKHLKAGEKQQTESDRDGISPSVLAKNQKETEIGKEDHVFEQKDKEDEKCRKELSVNHENNMSHNFNAAGSDSIIPPETERETYDDETMGPTKRISDNEKNLQHGTNDISVEVEKEEEEEEEEEENSTFSKVKKENVTGEQEAVRNNEVSGTEEESTSKGEEIMGGDEKQSEAGEKSSIIEIEGSANSAKISKDNLVLEDEAEAPTQENKPTEVVGEIDIPDAPRDDVEIVEAVEKNIIPEDLEVAKEDQEGEQVKLDEPVKAMKDDKIAMRGAESISEDMKKKQEGTAELSNEKAKKEVDETARESAEGVEVEKSKTPESPKVVKRCTSGRPEDLQINERDPEVLKEDVRVPDEDVKPEIATTIENSEEEDPKSQRVQISTEQAETTQKDMGDVGSTTSFKEEEKPKRFEITQEGDKITGKDTNHEHGEATEAASENSKASDVGTAEKYIEPSSESVKKDTEEDAEVENSEKTEFIKVKAELEN--L-DAPKEAEVT----AELNKENEDVE-VD--TEEDAEVENSEKTEFI--KVKAELGNLDA-PKEAEVTAE-LN---KENEDVEVAATSKEDIETKCSEPAETPIEDGTCTEAEVS-KK--DAEAVTK-EDENME-N-SKIAEALKDVTGDQEIDDI-NISDEFQRTVELPELEKQDIKDNKGEDKELEVEETEKETSLPDLVVEENITEEKNEIKQEEEEVSQLDFNETESISKEAPNNDENGFEDQSTRENPKKASADDIFKDILDETNEFLEQLKIVDDSELNALLQSLDAKDSTTQTTEQSKKNNDKPQDVITTSEIRKLNEKEPVYIYTSLAGGGFHMIPRTNRLSTILTANRIPFTYRDLGTDDEARKVWKTFSKGRSLPGVVRGHNDLIGNWEEIEEANEDYKLRELIYDTI\n",
            "  Score=149\n",
            "\n",
            "Subject_Id : seq694\n",
            "\n",
            "--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MAVR-VAKM--TE-R-EKLLL----KKYGNRRLYDTKR--SAYVTLSQVADIIK-QGVD--V----EVVD-AKTQEDVTAF-ILTQI-VLEQARRKNALLPAELLHLIIRYG--EDLLQEFFQKYLEQTIHG---YI---QFKKAMD---EQFSKWIDM--QLNYSD---MAHKTLSGLT-RF-PNIF---SPP---P--G-SP-------GTKKDKG----------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                   ||. |...  .| . |||..    ||  |.|....|.  .|| .|...|   | |||.  |    ..|| .|..|..... ..|.. |.   |..|.  ......... ..  |........|. .|...|   |.   | .|  |   .|....|..  |..|..   .|.....|.. .| |..|   .||   |  | .|       |..|. |                                                                                                          \n",
            "MGSSHHHHHHHHASENLYFQSADITDKTAEQLENLNIQDDQKQAATGSESQSVENSSASLYVGDLEPSVSEAHLYDIFSPIGSVSSIRVCRDAITKTSLGYAYVNFNDHEAGRKAIEQLNYTPIKGRLCRIMWSQRDPSLRKKGSGNIFIKNLHPDIDNKALYDTFSVFGDILSSKIATDENGKSKGFGFVHFEEEGAAKEAIDALNGMLLNGQEIYVAPHLSRKERDSQLEETKAHYTNLYVKNINSETTDEQFQELFAKFGPIVSASLEKDADGKLKGFGFVNYEKHEDAVKAVEALNDSELNGEKLYVGRAQKK--NERMHVLKKQYEAY-RLEKMA---KYQGVNLFVKNLDDSVDDEKLEEEFAPYGTITSAKVM---RTENG--KSKGFGFVC-FSTPEEATKAITEKN-QQIVAGKPLYVAIAQ-RK--DVRRSQLAQQIQARNQMRYQQATAAAAAAAAGMPGQFMPPMFYGVMPPRGVPFNGPNPQQMNPMGGMPKN-GMPPQFRNGPVYGVPPQGGFPRNANDNNQFYQQKQRQALGEQLYKKVSAKTSNEEAAGKITGMILDLPPQEVFPLLESDELFEQHYKEASAAYESFKKEQEQQTEQA\n",
            "  Score=157\n",
            "\n"
          ]
        }
      ]
    }
  ]
}