{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "gpuType": "T4",
      "authorship_tag": "ABX9TyMyiLmmvNDdtnSOc/gopgnn",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "accelerator": "GPU"
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/AMIRMOHAMMAD-OSS/Bioinformatics/blob/main/Updated_LLPS.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#Liquid-Liquid Phase Separation (LLPS) Protein Scorer!\n",
        "\n",
        "This tool allows you to analyze protein sequences for their propensity to undergo phase separation:\n",
        "\n",
        "Input Options:\n",
        "Upload a single protein sequence directly into the designated box.\n",
        "Alternatively, load a FASTA or TXT file containing multiple protein sequences.\n",
        "\n",
        "To maintain efficiency, we advise against loading more than 100 sequences at once\n",
        "\n",
        "Specify the sequence ID or directory for your analysis.\n",
        "\n",
        "By setting the 'End_Sequence'  parameter,you can focus the scoring on sequences up to the specified endpoint.\n",
        "\n",
        "         \n",
        "\n",
        "\n",
        "\n",
        "#BLAST+ tool\n",
        "You can utilize the BLAST+ tool to run ncbi BLAST searches on either your protein sequence or your FASTA file against your dataset. This powerful tool allows you to compare sequences and identify homologous regions, aiding in various biological analyses.\n"
      ],
      "metadata": {
        "id": "7DkACTVHMVCm"
      }
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "cellView": "form",
        "id": "-c3Fkzg9MPNn"
      },
      "outputs": [],
      "source": [
        "#@title Importing libraries\n",
        "%%capture\n",
        "from google.colab import output\n",
        "!git clone https://github.com/AMIRMOHAMMAD-OSS/classi\n",
        "!pip install biopython\n",
        "!apt-get update\n",
        "!apt-get install -y ncbi-blast+\n",
        "!pip install streamlit\n",
        "\n",
        "!pip install py3Dmol\n",
        "!pip install requests\n",
        "!pip install biotite\n",
        "!pip install stmol\n",
        "!pip install command_runner\n",
        "from command_runner import command_runner\n",
        "!pip install plotly\n",
        "!pip install --upgrade \"kaleido==0.1.*\"\n",
        "import kaleido\n",
        "from Bio import SeqIO\n",
        "from Bio.Seq import Seq\n",
        "from Bio import pairwise2\n",
        "from google.colab import files\n",
        "import plotly.graph_objects as go\n",
        "from plotly.subplots import make_subplots\n",
        "import zipfile\n",
        "import shutil\n",
        "from transformers import AutoTokenizer\n",
        "from tqdm import tqdm\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "import numpy as np\n",
        "import time\n",
        "from collections import defaultdict\n",
        "import torch\n",
        "from torch.utils.data.dataloader import DataLoader\n",
        "import time\n",
        "import os\n",
        "import sys\n",
        "import json\n",
        "import random\n",
        "from ast import literal_eval\n",
        "import numpy as np\n",
        "import torch\n",
        "from transformers import AutoTokenizer\n",
        "from transformers import AutoTokenizer\n",
        "from tokenizers import Tokenizer, decoders, models, normalizers, pre_tokenizers, trainers\n",
        "from transformers import CanineTokenizer, CanineModel\n",
        "tokenizer = Tokenizer.from_file(\"classi/Trained_BPE2.json\")\n",
        "tokenizer.model_max_length = 256\n",
        "tokenizer\n",
        "\n",
        "from sklearn.preprocessing import StandardScaler as Sc\n",
        "from sklearn.decomposition import PCA\n",
        "import numpy as np\n",
        "import pandas as pd\n",
        "\n",
        "import torch\n",
        "\n",
        "from sklearn.model_selection import train_test_split as tts\n",
        "\n",
        "\n",
        "def edit(x):\n",
        "    for i in x:\n",
        "      if i in \"BJOUX\\Z_n\\n\":\n",
        "        x = x.replace(i,\"\")\n",
        "      else:\n",
        "        pass\n",
        "    return x\n",
        "\n",
        "def encode(x):\n",
        "  l = []\n",
        "  chars = tokenizer.get_vocab()\n",
        "  for i in x:\n",
        "    l.append(chars[i])\n",
        "  return l\n",
        "\n",
        "with open(\"classi/training.txt\",\"r\",encoding=\"utf-8\") as f:\n",
        "  text = f.read()\n",
        "  k = \"#\".join([i for i in edit(text).split(\"<|edoftext|>\")])\n",
        "chars = tokenizer.get_vocab()\n",
        "vocab_size = len(tokenizer.get_vocab())\n",
        "\n",
        "with open(\"classi/validation.txt\",\"r\",encoding=\"utf-8\") as f:\n",
        "  text = f.read()\n",
        "  l = \"#\".join([i for i in edit(text).split(\"<|edoftext|>\")])\n",
        "\n",
        "def Padding(x,PAD = 0,max_len = 512):\n",
        "  l = []\n",
        "  max_len = max(512,len(max(x,key = lambda x: len(x))))\n",
        "  for i in x:\n",
        "    a = i\n",
        "    if len(a) < max_len:\n",
        "      a = a + [PAD for i in range(max_len-len(a))]\n",
        "    l.append(a)\n",
        "  return np.array(l).reshape(len(x),max_len)\n",
        "with open(\"classi/negative_dataset.txt\") as x:\n",
        "  c = x.read()\n",
        "Filtered_train  = [i for i in k.split(\"#\") if len(i)<= 512][1:]\n",
        "Filtered_val = [i for i in l.split(\"#\") if len(i)<= 512][1:]\n",
        "Filtered_neg = [i for i in c.split(\"\\n\") if len(i)<= 512]\n",
        "pos = Filtered_train + Filtered_val\n",
        "pos2 = list(set([i for i in k.split(\"#\")][1:] +  [i for i in l.split(\"#\")][1:]))\n",
        "with open(\"DrLLPS.fasta\",\"w\") as w:\n",
        "  for i in range(len(pos2)):\n",
        "    w.write(\">seq{i}\\n\".format(i=i))\n",
        "    w.write(pos2[i]+\"\\n\")\n",
        "  w.close()\n",
        "!unzip /content/classi/TRAIN.zip\n",
        "\n",
        "T = pd.read_csv(\"/content/TRAIN (3).csv\")\n",
        "\n",
        "data = T.iloc[:,1:-1].values\n",
        "labels = T.iloc[:,-1].values.reshape(6128,1)\n",
        "import xgboost as xgb\n",
        "\n",
        "train_d,test_d,train_l,test_l = tts(data,labels,random_state = 48,test_size = 0.1)\n",
        "clf = xgb.XGBClassifier(tree_method=\"hist\", early_stopping_rounds=3)\n",
        "clf.fit(train_d, train_l, eval_set=[(test_d, test_l)])\n",
        "print(\"Everything looks ok! ‚ú®üç∞‚ú®\")"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Loading models and requierments\n",
        "if torch.cuda.is_available():\n",
        "  print(\"Cuda is available üéâ\\n\")\n",
        "  print(\"The allocated device is : \"+torch.cuda.get_device_name())\n",
        "  device = torch.device(\"cuda\")\n",
        "else:\n",
        "  print(\"Cuda is not available üòî\\n\")\n",
        "  print(\"The allocated device is : cpu\")\n",
        "  device = torch.device(\"cpu\")\n",
        "\n",
        "\n",
        "\n",
        "def set_seed(seed):\n",
        "    random.seed(seed)\n",
        "    np.random.seed(seed)\n",
        "    torch.manual_seed(seed)\n",
        "    torch.cuda.manual_seed_all(seed)\n",
        "\n",
        "def setup_logging(config):\n",
        "    work_dir = config.system.work_dir\n",
        "    os.makedirs(work_dir, exist_ok=True)\n",
        "    with open(os.path.join(work_dir, 'args.txt'), 'w') as f:\n",
        "        f.write(' '.join(sys.argv))\n",
        "    with open(os.path.join(work_dir, 'config.json'), 'w') as f:\n",
        "        f.write(json.dumps(config.to_dict(), indent=4))\n",
        "\n",
        "class CfgNode:\n",
        "    def __init__(self, **kwargs):\n",
        "        self.__dict__.update(kwargs)\n",
        "\n",
        "    def __str__(self):\n",
        "        return self._str_helper(0)\n",
        "\n",
        "    def _str_helper(self, indent):\n",
        "        parts = []\n",
        "        for k, v in self.__dict__.items():\n",
        "            if isinstance(v, CfgNode):\n",
        "                parts.append(\"%s:\\n\" % k)\n",
        "                parts.append(v._str_helper(indent + 1))\n",
        "            else:\n",
        "                parts.append(\"%s: %s\\n\" % (k, v))\n",
        "        parts = [' ' * (indent * 4) + p for p in parts]\n",
        "        return \"\".join(parts)\n",
        "\n",
        "    def to_dict(self):\n",
        "        return { k: v.to_dict() if isinstance(v, CfgNode) else v for k, v in self.__dict__.items() }\n",
        "\n",
        "    def merge_from_dict(self, d):\n",
        "        self.__dict__.update(d)\n",
        "\n",
        "    def merge_from_args(self, args):\n",
        "\n",
        "        for arg in args:\n",
        "            keyval = arg.split('=')\n",
        "            assert len(keyval) == 2, \"expecting each override arg to be of form --arg=value, got %s\" % arg\n",
        "            key, val = keyval\n",
        "            try:\n",
        "                val = literal_eval(val)\n",
        "            except ValueError:\n",
        "                pass\n",
        "\n",
        "            assert key[:2] == '--'\n",
        "            key = key[2:]\n",
        "            keys = key.split('.')\n",
        "            obj = self\n",
        "            for k in keys[:-1]:\n",
        "                obj = getattr(obj, k)\n",
        "            leaf_key = keys[-1]\n",
        "            assert hasattr(obj, leaf_key), f\"{key} is not an attribute that exists in the config\"\n",
        "            print(\"command line overwriting config attribute %s with %s\" % (key, val))\n",
        "            setattr(obj, leaf_key, val)\n",
        "\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "\n",
        "class SiLU(nn.Module):\n",
        "   def forward(self, x):\n",
        "        return x*F.sigmoid(x)\n",
        "\n",
        "class NY(nn.Module):\n",
        "  def forward(self,x):\n",
        "    return 3*torch.tanh(0.3*x)\n",
        "\n",
        "class NewGELU(nn.Module):\n",
        "    def forward(self, x):\n",
        "        return 0.5 * x * (1.0 + torch.tanh(math.sqrt(2.0 / math.pi) * (x + 0.044715 * torch.pow(x, 3.0))))\n",
        "\n",
        "class SelfAttention(nn.Module):\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        assert config.n_embd % config.n_head == 0\n",
        "        self.c_attn = nn.Linear(config.n_embd, 3 * config.n_embd)\n",
        "        self.c_proj = nn.Linear(config.n_embd, config.n_embd)\n",
        "        self.attn_dropout = nn.Dropout(config.attn_pdrop)\n",
        "        self.resid_dropout = nn.Dropout(config.resid_pdrop)\n",
        "        self.register_buffer(\"bias\", torch.tril(torch.ones(config.block_size, config.block_size))\n",
        "                                     .view(1, 1, config.block_size, config.block_size))\n",
        "        self.n_head = config.n_head\n",
        "        self.n_embd = config.n_embd\n",
        "\n",
        "    def forward(self, x):\n",
        "        B, T, C = x.size()\n",
        "        q, k ,v  = self.c_attn(x).split(self.n_embd, dim=2)\n",
        "        k = k.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        q = q.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        v = v.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        att = (q @ k.transpose(-2, -1)) * (1.0 / math.sqrt(k.size(-1)))\n",
        "        att = att.masked_fill(self.bias[:,:,:T,:T] == 0, float('-inf'))\n",
        "        att = F.softmax(att, dim=-1)\n",
        "        att = self.attn_dropout(att)\n",
        "        y = att @ v\n",
        "        y = y.transpose(1, 2).contiguous().view(B, T, C)\n",
        "        y = self.resid_dropout(self.c_proj(y))\n",
        "        return y\n",
        "\n",
        "class Block(nn.Module):\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.ln_1 = nn.LayerNorm(config.n_embd)\n",
        "        self.attn = SelfAttention(config)\n",
        "        self.ln_2 = nn.LayerNorm(config.n_embd)\n",
        "        self.mlp = nn.ModuleDict(dict(\n",
        "            c_fc    = nn.Linear(config.n_embd, 4 * config.n_embd),\n",
        "            c_proj  = nn.Linear(4 * config.n_embd, config.n_embd),\n",
        "            act     = NewGELU(),\n",
        "            dropout = nn.Dropout(config.resid_pdrop),\n",
        "        ))\n",
        "        m = self.mlp\n",
        "        self.mlpf = lambda x: m.dropout(m.c_proj(m.act(m.c_fc(x))))\n",
        "    def forward(self, x):\n",
        "        x = x + self.attn(self.ln_1(x))\n",
        "        x = x + self.mlpf(self.ln_2(x))\n",
        "        return x\n",
        "\n",
        "class ClassifierII(nn.Module):\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.model_type = 'gpt'\n",
        "        C.n_layer = None\n",
        "        C.n_head = None\n",
        "        C.n_embd =  None\n",
        "        C.vocab_size = len(chars)\n",
        "        C.max_length = 512\n",
        "        C.embd_pdrop = 0.1\n",
        "        C.resid_pdrop = 0.1\n",
        "        C.attn_pdrop = 0.1\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.max_length = config.block_size\n",
        "        self.soft = nn.Softmax(1)\n",
        "        self.config = self.get_default_config()\n",
        "        self.device = \"cuda\"\n",
        "        self.model_states = {'h':{'n_layer': 48, 'n_head': 25, 'n_embd': 1600},\n",
        "                                    'g':{'n_layer': 12, '': 12, 'n_embd': 768},\n",
        "                'f':   {'n_layer': 24, 'n_head': 16, 'n_embd': 1024},\n",
        "                'e':   {'n_layer': 36, 'n_head': 20, 'n_embd': 1280},\n",
        "                'd':{'n_layer': 8, 'n_head': 16, 'n_embd': 512},\n",
        "                'c':{'n_layer': 6, 'n_head': 6, 'n_embd': 192},\n",
        "                'b':{'n_layer': 4, 'n_head': 4, 'n_embd': 128},\n",
        "                'a':{'n_layer': 3, 'n_head': 3, 'n_embd': 48}}\n",
        "\n",
        "        type_ = config.model_type is not None\n",
        "        #assert type_ in \"abcdefgh\"\n",
        "        p = all([config.n_layer is not None, config.n_head is not None, config.n_embd is not None])\n",
        "        #assert type_ == True and p == True\n",
        "        if type_:\n",
        "            config.merge_from_dict(self.model_states[config.model_type])\n",
        "        self.closs = nn.BCELoss()\n",
        "        self.ny = NY()\n",
        "        self.l = nn.Linear(512,1,64)\n",
        "        self.transformer = nn.ModuleDict(dict(\n",
        "            wte = nn.Embedding(config.vocab_size, config.n_embd),\n",
        "            wpe = nn.Embedding(config.max_length, config.n_embd),\n",
        "            drop = nn.Dropout(config.embd_pdrop),\n",
        "            h = nn.ModuleList([Block(config) for _ in range(config.n_layer)]),\n",
        "            ln_f = nn.LayerNorm(config.n_embd),))\n",
        "        self.classifier_head = nn.Sequential(#nn.Tanh(),\n",
        "                                             nn.Linear(config.n_embd, 2)\n",
        "                                             )\n",
        "        self.apply(self._init_weights)\n",
        "        for pn, p in self.named_parameters():\n",
        "            if pn.endswith('c_proj.weight'):\n",
        "                torch.nn.init.normal_(p, mean=0.0, std=0.02/math.sqrt(2 * config.n_layer))\n",
        "\n",
        "        n_params = sum(p.numel() for p in self.transformer.parameters())\n",
        "        print(\"[ Number of trainable parameters: %.2fM ]\" % (n_params/1e6,))\n",
        "\n",
        "    def _init_weights(self, module):\n",
        "        if isinstance(module, nn.Linear):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "            if module.bias is not None:\n",
        "                torch.nn.init.zeros_(module.bias)\n",
        "        elif isinstance(module, nn.Embedding):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "        elif isinstance(module, nn.LayerNorm):\n",
        "            torch.nn.init.zeros_(module.bias)\n",
        "            torch.nn.init.ones_(module.weight)\n",
        "\n",
        "    @classmethod\n",
        "    def from_pretrained(cls, model_type):\n",
        "        from transformers import GPT2LMHeadModel\n",
        "        config = cls.get_default_config()\n",
        "        config.model_type = model_type\n",
        "        config.vocab_size = 25\n",
        "        config.max_length = 512\n",
        "        model = ClassifierII(config)\n",
        "        sd = model.state_dict()\n",
        "        model_hf = GPT2LMHeadModel.from_pretrained(model_type)\n",
        "        sd_hf = model_hf.state_dict()\n",
        "        keys = [k for k in sd_hf if not k.endswith('attn.masked_bias')]\n",
        "        transposed = ['attn.c_attn.weight', 'attn.c_proj.weight', 'mlp.c_fc.weight', 'mlp.c_proj.weight']\n",
        "        assert len(keys) == len(sd)\n",
        "        for k in keys:\n",
        "            if any(k.endswith(w) for w in transposed):\n",
        "                assert sd_hf[k].shape[::-1] == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k].t())\n",
        "            else:\n",
        "                assert sd_hf[k].shape == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k])\n",
        "        return model\n",
        "\n",
        "    def configure_optimizers(self, train_config):\n",
        "        decay = set()\n",
        "        no_decay = set()\n",
        "        whitelist_weight_modules = (torch.nn.Linear, )\n",
        "        blacklist_weight_modules = (torch.nn.LayerNorm, torch.nn.Embedding)\n",
        "        for mn, m in self.named_modules():\n",
        "            for pn, p in m.named_parameters():\n",
        "                fpn = '%s.%s' % (mn, pn) if mn else pn\n",
        "                if pn.endswith('bias'):\n",
        "                    no_decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, whitelist_weight_modules):\n",
        "                    decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, blacklist_weight_modules):\n",
        "                    no_decay.add(fpn)\n",
        "        param_dict = {pn: p for pn, p in self.named_parameters()}\n",
        "        inter_params = decay & no_decay\n",
        "        union_params = decay | no_decay\n",
        "        assert len(inter_params) == 0, \"parameters %s made it into both decay/no_decay sets!\" % (str(inter_params), )\n",
        "        assert len(param_dict.keys() - union_params) == 0, \"parameters %s were not separated into either decay/no_decay set!\" \\\n",
        "                                                    % (str(param_dict.keys() - union_params), )\n",
        "        optim_groups = [\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(decay))], \"weight_decay\": train_config.weight_decay},\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(no_decay))], \"weight_decay\": 0.0},]\n",
        "        optimizer = torch.optim.AdamW(optim_groups, lr=train_config.learning_rate, betas=train_config.betas)\n",
        "        return optimizer\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict_proba(self,idx,j =\"None\"):\n",
        "      sigmoid = nn.Sigmoid()\n",
        "      soft = nn.Softmax(dim=1)\n",
        "      ny = NY()\n",
        "      si = SiLU()\n",
        "      self.eval()\n",
        "      x,_ = self.forward(idx)\n",
        "      if j == \"None\":\n",
        "        return x[:,0:1]\n",
        "      elif j == \"soft\":\n",
        "        return self.soft(x)[:,0:1]\n",
        "      else:\n",
        "        return sigmoid(x)[:,0:1]\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict(self,idx):\n",
        "      g = torch.zeros((idx.shape[0],1))\n",
        "      e = -1\n",
        "      for i in self.predict_proba(idx):\n",
        "        e+=1\n",
        "        if i[0].item()>=0.5:\n",
        "          g[e] = 1\n",
        "      return g\n",
        "\n",
        "    def forward(self, idx, targets=None):\n",
        "        device = idx.device\n",
        "        b, t = idx.size()\n",
        "        pos = torch.arange(0, t, dtype=torch.long, device=device).unsqueeze(0)\n",
        "        tok_emb = self.transformer.wte(idx)\n",
        "        pos_emb = self.transformer.wpe(pos)\n",
        "        x = self.transformer.drop(tok_emb + pos_emb)\n",
        "        for block in self.transformer.h:\n",
        "            x = block(x)\n",
        "        x = self.transformer.ln_f(x)\n",
        "        logits = self.classifier_head(x).view(b,2,t).to(device)\n",
        "        logits = self.l(logits)\n",
        "        logits = F.sigmoid(self.ny(logits).view(b,2).mean(1).view(b,1))\n",
        "        loss = None\n",
        "        if targets is not None:\n",
        "            loss = F.binary_cross_entropy(logits,targets)\n",
        "        return logits, loss\n",
        "\n",
        "\n",
        "class Trainer:\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.device = 'cuda'\n",
        "        C.num_workers = 4\n",
        "        C.max_iters = None\n",
        "        C.batch_size = 128\n",
        "        C.max_length = 512\n",
        "        C.learning_rate = 8e-4\n",
        "        C.betas = (0.9, 0.95)\n",
        "        C.weight_decay = 0.1\n",
        "        C.grad_norm_clip = 1.0\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config, model, train_dataset,val_dataset):\n",
        "        self.config = config\n",
        "        self.model = model\n",
        "        self.optimizer = None\n",
        "        self.train_dataset = train_dataset\n",
        "        self.val_dataset = val_dataset\n",
        "        self.callbacks = defaultdict(list)\n",
        "        self.device ='cuda'\n",
        "        self.model = self.model.to(self.device)\n",
        "        self.iter_num = 0\n",
        "        self.iter_time = 0.0\n",
        "        self.iter_dt = 0.0\n",
        "\n",
        "    def add_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent].append(callback)\n",
        "\n",
        "    def set_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent] = [callback]\n",
        "\n",
        "    def trigger_callbacks(self, onevent: str):\n",
        "        for callback in self.callbacks.get(onevent, []):\n",
        "            callback(self)\n",
        "\n",
        "    def run(self):\n",
        "        model, config = self.model, self.config\n",
        "        self.optimizer = model.configure_optimizers(config)\n",
        "        batch_size = self.config.batch_size\n",
        "        def get_batch(mode):\n",
        "            batch_size = 64\n",
        "            if mode == \"train\":\n",
        "              data = train_dataset\n",
        "              pos_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]),513))\n",
        "              neg_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]),513))\n",
        "              N = np.random.randint(batch_size)\n",
        "              ix_pos = np.random.randint(pos_train.shape[0]-N)\n",
        "              ix_neg = np.random.randint(neg_train.shape[0]-batch_size-N)\n",
        "              pos_data = pos_train[ix_pos:ix_pos+N,:]\n",
        "              neg_data = neg_train[ix_neg:ix_neg+batch_size-N,:]\n",
        "              data = torch.concat((pos_data,neg_data))\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,1), device=\"cuda\")\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o] = 1\n",
        "                #else:\n",
        "                  #targets[o][1] = 1\n",
        "            else:\n",
        "              data = val_dataset\n",
        "              ix = np.random.randint(data.shape[0]-batch_size)\n",
        "              data = data[ix:ix+batch_size,:]\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,1), device=\"cuda\")\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o] = 1\n",
        "                #else:\n",
        "                 # targets[o][1] = 1\n",
        "            targets = targets.view(batch_size,1).to(\"cuda\")\n",
        "            seq = seq.view(batch_size,512).to(\"cuda\")\n",
        "            return seq, targets\n",
        "\n",
        "        @torch.no_grad\n",
        "        def cross_val():\n",
        "          model.eval()\n",
        "          out = []\n",
        "          for i in [\"train\",\"val\"]:\n",
        "            losses = torch.zeros(200+1)\n",
        "            for k in range(200):\n",
        "              X,Y = get_batch(i)\n",
        "              logits,loss = model(X,Y)\n",
        "              losses[k]=loss.item()\n",
        "              out1 = losses.mean()\n",
        "            out.append(out1)\n",
        "          model.train()\n",
        "          return out\n",
        "        losses = cross_val()\n",
        "        LOSS = [losses]\n",
        "        print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        model.train()\n",
        "        for epoch in range(20):\n",
        "          print(\"[epoch {o}] \\n\".format(o = epoch))\n",
        "          iters = 200\n",
        "          for i in range(iters):\n",
        "            if i == 0:\n",
        "              Y = \"| =\"\n",
        "            elif i == iters -1 :\n",
        "              Y = \"=> 100% |\"\n",
        "            else:\n",
        "              if i%(int(iters/50)) == 0 :\n",
        "                Y = \"=\"\n",
        "              else:\n",
        "                Y = \"\"\n",
        "            print(\"{y}\".format(y = Y),end=\"\")\n",
        "            x, y = get_batch(\"train\")\n",
        "            logits, self.loss = model(x, y)\n",
        "            model.zero_grad(set_to_none=True)\n",
        "            self.loss.backward()\n",
        "            torch.nn.utils.clip_grad_norm_(model.parameters(), config.grad_norm_clip)\n",
        "            self.optimizer.step()\n",
        "          losses = cross_val()\n",
        "          LOSS.append(losses)\n",
        "          print(LOSS)\n",
        "          print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        PATH = \"model {h}\".format(h = model_config.model_type)\n",
        "        torch.save(model.state_dict(), PATH)\n",
        "\n",
        "def set_seed(seed):\n",
        "    random.seed(seed)\n",
        "    np.random.seed(seed)\n",
        "    torch.manual_seed(seed)\n",
        "    torch.cuda.manual_seed_all(seed)\n",
        "\n",
        "def setup_logging(config):\n",
        "    work_dir = config.system.work_dir\n",
        "    os.makedirs(work_dir, exist_ok=True)\n",
        "    with open(os.path.join(work_dir, 'args.txt'), 'w') as f:\n",
        "        f.write(' '.join(sys.argv))\n",
        "    with open(os.path.join(work_dir, 'config.json'), 'w') as f:\n",
        "        f.write(json.dumps(config.to_dict(), indent=4))\n",
        "\n",
        "class CfgNode:\n",
        "    def __init__(self, **kwargs):\n",
        "        self.__dict__.update(kwargs)\n",
        "\n",
        "    def __str__(self):\n",
        "        return self._str_helper(0)\n",
        "\n",
        "    def _str_helper(self, indent):\n",
        "        parts = []\n",
        "        for k, v in self.__dict__.items():\n",
        "            if isinstance(v, CfgNode):\n",
        "                parts.append(\"%s:\\n\" % k)\n",
        "                parts.append(v._str_helper(indent + 1))\n",
        "            else:\n",
        "                parts.append(\"%s: %s\\n\" % (k, v))\n",
        "        parts = [' ' * (indent * 4) + p for p in parts]\n",
        "        return \"\".join(parts)\n",
        "\n",
        "    def to_dict(self):\n",
        "        return { k: v.to_dict() if isinstance(v, CfgNode) else v for k, v in self.__dict__.items() }\n",
        "\n",
        "    def merge_from_dict(self, d):\n",
        "        self.__dict__.update(d)\n",
        "\n",
        "    def merge_from_args(self, args):\n",
        "\n",
        "        for arg in args:\n",
        "            keyval = arg.split('=')\n",
        "            assert len(keyval) == 2, \"expecting each override arg to be of form --arg=value, got %s\" % arg\n",
        "            key, val = keyval\n",
        "            try:\n",
        "                val = literal_eval(val)\n",
        "            except ValueError:\n",
        "                pass\n",
        "\n",
        "            assert key[:2] == '--'\n",
        "            key = key[2:]\n",
        "            keys = key.split('.')\n",
        "            obj = self\n",
        "            for k in keys[:-1]:\n",
        "                obj = getattr(obj, k)\n",
        "            leaf_key = keys[-1]\n",
        "            assert hasattr(obj, leaf_key), f\"{key} is not an attribute that exists in the config\"\n",
        "            print(\"command line overwriting config attribute %s with %s\" % (key, val))\n",
        "            setattr(obj, leaf_key, val)\n",
        "\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "\n",
        "class SiLU(nn.Module):\n",
        "   def forward(self, x):\n",
        "        return x*F.sigmoid(x)\n",
        "\n",
        "class NY(nn.Module):\n",
        "  def forward(self,x):\n",
        "    return 3*torch.tanh(0.3*x)\n",
        "\n",
        "class NewGELU(nn.Module):\n",
        "    def forward(self, x):\n",
        "        return 0.5 * x * (1.0 + torch.tanh(math.sqrt(2.0 / math.pi) * (x + 0.044715 * torch.pow(x, 3.0))))\n",
        "\n",
        "class SelfAttention(nn.Module):\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        assert config.n_embd % config.n_head == 0\n",
        "        self.c_attn = nn.Linear(config.n_embd, 3 * config.n_embd)\n",
        "        self.c_proj = nn.Linear(config.n_embd, config.n_embd)\n",
        "        self.attn_dropout = nn.Dropout(config.attn_pdrop)\n",
        "        self.resid_dropout = nn.Dropout(config.resid_pdrop)\n",
        "        self.register_buffer(\"bias\", torch.tril(torch.ones(config.block_size, config.block_size))\n",
        "                                     .view(1, 1, config.block_size, config.block_size))\n",
        "        self.n_head = config.n_head\n",
        "        self.n_embd = config.n_embd\n",
        "\n",
        "    def forward(self, x):\n",
        "        B, T, C = x.size()\n",
        "        q, k ,v  = self.c_attn(x).split(self.n_embd, dim=2)\n",
        "        k = k.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        q = q.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        v = v.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        att = (q @ k.transpose(-2, -1)) * (1.0 / math.sqrt(k.size(-1)))\n",
        "        att = att.masked_fill(self.bias[:,:,:T,:T] == 0, float('-inf'))\n",
        "        att = F.softmax(att, dim=-1)\n",
        "        att = self.attn_dropout(att)\n",
        "        y = att @ v\n",
        "        y = y.transpose(1, 2).contiguous().view(B, T, C)\n",
        "        y = self.resid_dropout(self.c_proj(y))\n",
        "        return y\n",
        "\n",
        "class Block(nn.Module):\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.ln_1 = nn.LayerNorm(config.n_embd)\n",
        "        self.attn = SelfAttention(config)\n",
        "        self.ln_2 = nn.LayerNorm(config.n_embd)\n",
        "        self.mlp = nn.ModuleDict(dict(\n",
        "            c_fc    = nn.Linear(config.n_embd, 4 * config.n_embd),\n",
        "            c_proj  = nn.Linear(4 * config.n_embd, config.n_embd),\n",
        "            act     = NewGELU(),\n",
        "            dropout = nn.Dropout(config.resid_pdrop),\n",
        "        ))\n",
        "        m = self.mlp\n",
        "        self.mlpf = lambda x: m.dropout(m.c_proj(m.act(m.c_fc(x))))\n",
        "    def forward(self, x):\n",
        "        x = x + self.attn(self.ln_1(x))\n",
        "        x = x + self.mlpf(self.ln_2(x))\n",
        "        return x\n",
        "\n",
        "class ClassifierI(nn.Module):\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.model_type = 'gpt'\n",
        "        C.n_layer = None\n",
        "        C.n_head = None\n",
        "        C.n_embd =  None\n",
        "        C.vocab_size = len(chars)\n",
        "        C.max_length = 512\n",
        "        C.embd_pdrop = 0.1\n",
        "        C.resid_pdrop = 0.1\n",
        "        C.attn_pdrop = 0.1\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.max_length = config.block_size\n",
        "        self.soft = nn.Softmax(1)\n",
        "        self.config = self.get_default_config()\n",
        "        self.device = \"cpu\"\n",
        "        self.model_states = {'h':{'n_layer': 48, 'n_head': 25, 'n_embd': 1600},\n",
        "                                    'g':{'n_layer': 12, '': 12, 'n_embd': 768},\n",
        "                'f':   {'n_layer': 24, 'n_head': 16, 'n_embd': 1024},\n",
        "                'e':   {'n_layer': 36, 'n_head': 20, 'n_embd': 1280},\n",
        "                'd':{'n_layer': 8, 'n_head': 16, 'n_embd': 512},\n",
        "                'c':{'n_layer': 6, 'n_head': 6, 'n_embd': 192},\n",
        "                'b':{'n_layer': 4, 'n_head': 4, 'n_embd': 128},\n",
        "                'a':{'n_layer': 3, 'n_head': 3, 'n_embd': 48}}\n",
        "\n",
        "        type_ = config.model_type is not None\n",
        "        #assert type_ in \"abcdefgh\"\n",
        "        p = all([config.n_layer is not None, config.n_head is not None, config.n_embd is not None])\n",
        "        #assert type_ == True and p == True\n",
        "        if type_:\n",
        "            config.merge_from_dict(self.model_states[config.model_type])\n",
        "        self.closs = nn.BCELoss()\n",
        "        self.ny = NY()\n",
        "        #self.l = nn.Linear(512,1,64)\n",
        "        self.transformer = nn.ModuleDict(dict(\n",
        "            wte = nn.Embedding(config.vocab_size, config.n_embd),\n",
        "            wpe = nn.Embedding(config.max_length, config.n_embd),\n",
        "            drop = nn.Dropout(config.embd_pdrop),\n",
        "            h = nn.ModuleList([Block(config) for _ in range(config.n_layer)]),\n",
        "            ln_f = nn.LayerNorm(config.n_embd),))\n",
        "        self.classifier_head = nn.Sequential(#nn.Tanh(),\n",
        "                                             nn.Linear(config.n_embd, 2)\n",
        "                                             )\n",
        "        self.apply(self._init_weights)\n",
        "        for pn, p in self.named_parameters():\n",
        "            if pn.endswith('c_proj.weight'):\n",
        "                torch.nn.init.normal_(p, mean=0.0, std=0.02/math.sqrt(2 * config.n_layer))\n",
        "\n",
        "        n_params = sum(p.numel() for p in self.transformer.parameters())\n",
        "        #print(\"[ Number of trainable parameters: %.2fM ]\" % (n_params/1e6,))\n",
        "\n",
        "    def _init_weights(self, module):\n",
        "        if isinstance(module, nn.Linear):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "            if module.bias is not None:\n",
        "                torch.nn.init.zeros_(module.bias)\n",
        "        elif isinstance(module, nn.Embedding):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "        elif isinstance(module, nn.LayerNorm):\n",
        "            torch.nn.init.zeros_(module.bias)\n",
        "            torch.nn.init.ones_(module.weight)\n",
        "\n",
        "    @classmethod\n",
        "    def from_pretrained(cls, model_type):\n",
        "        from transformers import GPT2LMHeadModel\n",
        "        config = cls.get_default_config()\n",
        "        config.model_type = model_type\n",
        "        config.vocab_size = 25\n",
        "        config.max_length = 512\n",
        "        model = ClassifierI(config)\n",
        "        sd = model.state_dict()\n",
        "        model_hf = GPT2LMHeadModel.from_pretrained(model_type)\n",
        "        sd_hf = model_hf.state_dict()\n",
        "        keys = [k for k in sd_hf if not k.endswith('attn.masked_bias')]\n",
        "        transposed = ['attn.c_attn.weight', 'attn.c_proj.weight', 'mlp.c_fc.weight', 'mlp.c_proj.weight']\n",
        "        assert len(keys) == len(sd)\n",
        "        for k in keys:\n",
        "            if any(k.endswith(w) for w in transposed):\n",
        "                assert sd_hf[k].shape[::-1] == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k].t())\n",
        "            else:\n",
        "                assert sd_hf[k].shape == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k])\n",
        "        return model\n",
        "\n",
        "    def configure_optimizers(self, train_config):\n",
        "        decay = set()\n",
        "        no_decay = set()\n",
        "        whitelist_weight_modules = (torch.nn.Linear, )\n",
        "        blacklist_weight_modules = (torch.nn.LayerNorm, torch.nn.Embedding)\n",
        "        for mn, m in self.named_modules():\n",
        "            for pn, p in m.named_parameters():\n",
        "                fpn = '%s.%s' % (mn, pn) if mn else pn\n",
        "                if pn.endswith('bias'):\n",
        "                    no_decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, whitelist_weight_modules):\n",
        "                    decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, blacklist_weight_modules):\n",
        "                    no_decay.add(fpn)\n",
        "        param_dict = {pn: p for pn, p in self.named_parameters()}\n",
        "        inter_params = decay & no_decay\n",
        "        union_params = decay | no_decay\n",
        "        assert len(inter_params) == 0, \"parameters %s made it into both decay/no_decay sets!\" % (str(inter_params), )\n",
        "        assert len(param_dict.keys() - union_params) == 0, \"parameters %s were not separated into either decay/no_decay set!\" \\\n",
        "                                                    % (str(param_dict.keys() - union_params), )\n",
        "        optim_groups = [\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(decay))], \"weight_decay\": train_config.weight_decay},\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(no_decay))], \"weight_decay\": 0.0},]\n",
        "        optimizer = torch.optim.AdamW(optim_groups, lr=train_config.learning_rate, betas=train_config.betas)\n",
        "        return optimizer\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict_proba(self,idx,j =\"None\"):\n",
        "      sigmoid = nn.Sigmoid()\n",
        "      soft = nn.Softmax(dim=1)\n",
        "      ny = NY()\n",
        "      si = SiLU()\n",
        "      self.eval()\n",
        "      x,_ = self.forward(idx)\n",
        "      if j == \"None\":\n",
        "        return x[:,0:1]\n",
        "      elif j == \"soft\":\n",
        "        return self.soft(x)[:,0:1]\n",
        "      else:\n",
        "        return sigmoid(x)[:,0:1]\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict(self,idx):\n",
        "      g = torch.zeros((idx.shape[0],1))\n",
        "      e = -1\n",
        "      for i in self.predict_proba(idx):\n",
        "        e+=1\n",
        "        if i[0].item()>=0.5:\n",
        "          g[e] = 1\n",
        "      return g\n",
        "\n",
        "    def forward(self, idx, targets=None):\n",
        "        device = idx.device\n",
        "        b, t = idx.size()\n",
        "        pos = torch.arange(0, t, dtype=torch.long, device=device).unsqueeze(0)\n",
        "        tok_emb = self.transformer.wte(idx)\n",
        "        pos_emb = self.transformer.wpe(pos)\n",
        "        x = self.transformer.drop(tok_emb + pos_emb)\n",
        "        for block in self.transformer.h:\n",
        "            x = block(x)\n",
        "        x = self.transformer.ln_f(x)\n",
        "        logits = self.ny(self.classifier_head(x)).mean(1).to(device)\n",
        "        #logits = self.l(logits)\n",
        "        # = F.sigmoid(logits.view(b,2).mean(1).view(b,1))\n",
        "        loss = None\n",
        "        if targets is not None:\n",
        "            loss = F.cross_entropy(logits,targets,ignore_index = -1)\n",
        "        return logits, loss\n",
        "\n",
        "\n",
        "class Trainer:\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.device = 'cuda'\n",
        "        C.num_workers = 4\n",
        "        C.max_iters = None\n",
        "        C.batch_size = 64\n",
        "        C.max_length = 512\n",
        "        C.learning_rate = 8e-4\n",
        "        C.betas = (0.9, 0.95)\n",
        "        C.weight_decay = 0.1\n",
        "        C.grad_norm_clip = 1.0\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config, model, train_dataset,val_dataset):\n",
        "        self.config = config\n",
        "        self.model = model\n",
        "        self.optimizer = None\n",
        "        self.train_dataset = train_dataset\n",
        "        self.val_dataset = val_dataset\n",
        "        self.callbacks = defaultdict(list)\n",
        "        self.device ='cuda'\n",
        "        self.model = self.model.to(self.device)\n",
        "        self.iter_num = 0\n",
        "        self.iter_time = 0.0\n",
        "        self.iter_dt = 0.0\n",
        "\n",
        "    def add_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent].append(callback)\n",
        "\n",
        "    def set_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent] = [callback]\n",
        "\n",
        "    def trigger_callbacks(self, onevent: str):\n",
        "        for callback in self.callbacks.get(onevent, []):\n",
        "            callback(self)\n",
        "\n",
        "    def run(self):\n",
        "        model, config = self.model, self.config\n",
        "        self.optimizer = model.configure_optimizers(config)\n",
        "        batch_size = self.config.batch_size\n",
        "        def get_batch(mode):\n",
        "            batch_size = 64\n",
        "            if mode == \"train\":\n",
        "              data = train_dataset\n",
        "              pos_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]),513))\n",
        "              neg_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]),513))\n",
        "              N = np.random.randint(batch_size)\n",
        "              ix_pos = np.random.randint(pos_train.shape[0]-N)\n",
        "              ix_neg = np.random.randint(neg_train.shape[0]-batch_size-N)\n",
        "              pos_data = pos_train[ix_pos:ix_pos+N,:]\n",
        "              neg_data = neg_train[ix_neg:ix_neg+batch_size-N,:]\n",
        "              data = torch.concat((pos_data,neg_data))\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,2), device=self.device)\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o][0] = 1\n",
        "                else:\n",
        "                  targets[o][1] = 1\n",
        "            else:\n",
        "              data = val_dataset\n",
        "              ix = np.random.randint(data.shape[0]-batch_size)\n",
        "              data = data[ix:ix+batch_size,:]\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,2), device=self.device)\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o][0] = 1\n",
        "                else:\n",
        "                  targets[o][1] = 1\n",
        "            targets = targets.view(batch_size,2).to(self.device)\n",
        "            seq = seq.view(batch_size,512).to(self.device)\n",
        "            return seq, targets\n",
        "\n",
        "        @torch.no_grad\n",
        "        def cross_val():\n",
        "          model.eval()\n",
        "          out = []\n",
        "          for i in [\"train\",\"val\"]:\n",
        "            losses = torch.zeros(200+1)\n",
        "            for k in range(200):\n",
        "              X,Y = get_batch(i)\n",
        "              logits,loss = model(X,Y)\n",
        "              losses[k]=loss.item()\n",
        "              out1 = losses.mean()\n",
        "            out.append(out1)\n",
        "          model.train()\n",
        "          return out\n",
        "        losses = cross_val()\n",
        "        LOSS = [losses]\n",
        "        print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        model.train()\n",
        "        for epoch in range(20):\n",
        "          print(\"[epoch {o}] \\n\".format(o = epoch))\n",
        "          iters = 200\n",
        "          for i in range(iters):\n",
        "            if i == 0:\n",
        "              Y = \"| =\"\n",
        "            elif i == iters -1 :\n",
        "              Y = \"=> 100% |\"\n",
        "            else:\n",
        "              if i%(int(iters/50)) == 0 :\n",
        "                Y = \"=\"\n",
        "              else:\n",
        "                Y = \"\"\n",
        "            print(\"{y}\".format(y = Y),end=\"\")\n",
        "            x, y = get_batch(\"train\")\n",
        "            logits, self.loss = model(x, y)\n",
        "            model.zero_grad(set_to_none=True)\n",
        "            self.loss.backward()\n",
        "            torch.nn.utils.clip_grad_norm_(model.parameters(), config.grad_norm_clip)\n",
        "            self.optimizer.step()\n",
        "          losses = cross_val()\n",
        "          LOSS.append(losses)\n",
        "          print(LOSS)\n",
        "          print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        PATH = \"model {h}\".format(h = model_config.model_type)\n",
        "        torch.save(model.state_dict(), PATH)\n",
        "class Transformer():\n",
        "  def __init__(self,mode):\n",
        "    self.mode = mode\n",
        "    self.device = device\n",
        "    self.vocab_dict = tokenizer.get_vocab()\n",
        "\n",
        "  def classifier(self):\n",
        "    device = self.device\n",
        "    model_config = ClassifierI.get_default_config()\n",
        "    model_config.vocab_size = 25\n",
        "    model_config.block_size = 512\n",
        "    if self.mode == \"b\":\n",
        "      model_config.model_type = 'b'\n",
        "      model2 = ClassifierI(model_config)\n",
        "      model2.load_state_dict(torch.load(\"classi/model_b\",map_location=self.device,weights_only = True))\n",
        "    else:\n",
        "      model_config.model_type = 'c'\n",
        "      model2 = ClassifierI(model_config)\n",
        "      model2.load_state_dict(torch.load(\"classi/model_c\",map_location = self.device,weights_only = True))\n",
        "    model2.to(device)\n",
        "    return model2\n",
        "\n",
        "  def Encode(self, i):\n",
        "        def encode_char(x):\n",
        "            return [self.vocab_dict[char] for char in x]\n",
        "\n",
        "        def pad_sequences(x, PAD=0, max_len=512):\n",
        "            return np.array([seq + [PAD] * (max_len - len(seq)) for seq in x])\n",
        "\n",
        "        encoded = list(map(encode_char, i))\n",
        "        padded = pad_sequences(encoded)\n",
        "        return torch.tensor(padded).to(self.device)\n",
        "\n",
        "  def Decode(self,i):\n",
        "    def decode(k):\n",
        "      l = [j for j in k if j != 0 ]\n",
        "      seq = [chars[j] for j in l]\n",
        "      return \"\".join(seq)\n",
        "    H = list(map(decode,i))\n",
        "    G = []\n",
        "    for i in H:\n",
        "      if len(i)>512:\n",
        "        u = H.index(i)\n",
        "        for j in range(len(i)-512):\n",
        "          G.append(i[j:j+512])\n",
        "      else:\n",
        "        G.append(i)\n",
        "    return G,u\n",
        "  def Mean(self,i):\n",
        "    I = list(i.ravel())\n",
        "    sig = lambda x:np.e**(0*x)\n",
        "    Sum = [sig(np.abs(x-int(len(I)/2))) for x in range(len(I))]\n",
        "    wei = np.array([Sum[i]/sum(Sum) for i in range(len(I))]).reshape(len(I),1)\n",
        "    return np.sum(wei*i)\n",
        "\n",
        "  def enhancer(self,x):\n",
        "    wei = np.array([np.exp((i[0]-1)/(i[0])) for i in x])\n",
        "    return np.sum(np.array([i[0] for i in x])*wei)/np.sum(wei)\n",
        "\n",
        "  def predict_proba(self,i):\n",
        "    if len(i[0])>512:\n",
        "       l = [self.Encode([i[0][j:j+512]]) for j in range(len(i[0])-512)]\n",
        "       if len(l)>700:\n",
        "        L = [torch.concat(tuple(l[i*700:(i+1)*700])) for i in range(len(l)//700)] + [torch.concat(tuple(l[(len(l)//700)*700:(len(l)//700)*700+len(l)%700]))]\n",
        "        T = [self.classifier().predict_proba(i,\"sig\").tolist() for i in L ]\n",
        "        return self.enhancer(np.concatenate(tuple([i for i in T])))\n",
        "       else:\n",
        "         t = torch.concat(tuple(l))\n",
        "         return self.enhancer(self.classifier().predict_proba(t,\"sig\").tolist())\n",
        "    else:\n",
        "       seq = self.Encode(i)\n",
        "       U = np.array(self.classifier().predict_proba(seq,\"sig\").tolist())\n",
        "       out = np.array(U)\n",
        "       return out.reshape(len(i),1)\n",
        "\n",
        "model2 = Transformer(\"b\")\n",
        "model3 = Transformer(\"c\")\n",
        "\n",
        "\n",
        "import functools as FUNC\n",
        "from numba import njit\n",
        "\n",
        "class PPA():\n",
        "    def __init__(self, seq, model):\n",
        "        self.model = model\n",
        "        self.seq = seq\n",
        "        self.len = len(self.seq)\n",
        "        self.idx = None\n",
        "\n",
        "    def attention_mask(self):\n",
        "        idx2 = self.splicer(self.seq)[1]\n",
        "        k = self.T()\n",
        "        mask = [[True if len(k[0]) > idx2[j] - i > -1 else False for i in range(len(k[j]))] for j in range(len(k))]\n",
        "        return mask\n",
        "\n",
        "    def mean(self, mask, spliced):\n",
        "        u = [np.mean(self.model.predict_proba([spliced[i][j] for j in range(len(spliced[i])) if mask[i][j]])) for i in trange(len(spliced))]\n",
        "        return u\n",
        "\n",
        "    def splice_scorer(self, i):\n",
        "        lmin = len(i) // 2\n",
        "        return [i[j:j + lmin] for j in range(len(i) - lmin + 1)]\n",
        "\n",
        "    def splicer(self, i):\n",
        "        idx = self.idx\n",
        "        o, Before = [], []\n",
        "        for j in range(len(i)):\n",
        "            n_b, n_a = j, len(i) - j - 1\n",
        "            if n_b <= idx // 2:\n",
        "                before = i[max(0, j - idx // 2):j]\n",
        "                after = i[j + 1:j + idx - len(before)]\n",
        "            elif n_a <= idx // 2:\n",
        "                before = i[j - idx + n_a + 1:j]\n",
        "                after = i[j + 1:]\n",
        "            else:\n",
        "                before = i[j - idx // 2:j]\n",
        "                after = i[j + 1:j + idx // 2 + 1]\n",
        "            o.append(before + i[j] + after)\n",
        "            Before.append(len(before))\n",
        "        return o, Before\n",
        "\n",
        "    def T(self):\n",
        "        return list(map(self.splice_scorer, self.splicer(self.seq)[0]))\n",
        "\n",
        "    def Out(self):\n",
        "        s = self.s\n",
        "        def d(x):\n",
        "            if s > 0.7:\n",
        "                return x * np.exp(-1.2 * (x - s))\n",
        "            else:\n",
        "                return x if x > 0.7 else x\n",
        "\n",
        "        return list(map(d, self.mean(self.attention_mask(), self.T())))\n",
        "def show(i):\n",
        "  values = np.array(i).reshape(1, len(i))\n",
        "  y = range(len(i))\n",
        "  Y = [(j,1) for j in range(len(i)) if np.mean([i[j],i[min((j+1),len(i)-1)],i[max(0,(j-1))]])>0.6]\n",
        "  Y2 = [(j,1) for j in range(len(i)) if np.mean([i[j],i[min((j+1),len(i)-1)],i[min((j+2),len(i)-1)],i[min((j+3),len(i)-1)],i[min((j+4),len(i)-1)],i[min((j+5),len(i)-1)],i[max(0,(j-1))],i[max(0,(j-2))],i[max(0,(j-3))],i[max(0,(j-4))],i[max(0,(j-5))]])>0.85]\n",
        "  threshold = 0.5\n",
        "  above_threshold = np.maximum(values - threshold, 0)\n",
        "  below_threshold = np.minimum(values, threshold)\n",
        "  fig, (ax3 , ax4) = plt.subplots(2,1,sharex = True,sharey = True)\n",
        "  ax3.bar(y, below_threshold.ravel(), 0.9, color=\"lightblue\")\n",
        "  ax3.bar(y, above_threshold.ravel(), 0.9, color=\"lightgreen\",\n",
        "               bottom=below_threshold.ravel())\n",
        "  ax4.broken_barh(Y, (0.7, 0.03),\n",
        "               facecolors=( 'red'))\n",
        "  ax4.plot([0., y[-1]],[0.715,0.715],\"-b\")\n",
        "  for k in Y:\n",
        "    ax4.plot([k[0],k[0]+1],[0.715,0.715],\"-r\")\n",
        "\n",
        "  ax4.text(0,0.8 , \"LLPS-susceptible regions\",fontsize = 20)\n",
        "  ax3.plot([0., y[-1]], [threshold, threshold], \"k--\")\n",
        "  ax4.set_ylim([0,1])\n",
        "  ax3.set_ylim([0,1])\n",
        "\n",
        "  plt.subplots_adjust(\n",
        "                    wspace=0.4,\n",
        "                    hspace=0)\n",
        "  fig.set_figwidth(20)\n",
        "  fig.set_figheight(15)\n",
        "  plt.show()\n"
      ],
      "metadata": {
        "cellView": "form",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "cgxWphcfMe1N",
        "outputId": "f72a4c74-a993-4852-cfcb-60556ffc68c3"
      },
      "execution_count": 2,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Cuda is available üéâ\n",
            "\n",
            "The allocated device is : Tesla T4\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Loading sequence , then hit `Runtime` -> `Run all`\n",
        "#@markdown - Load the sequence of your protein :\n",
        "Sequence = \"MASNDYTQQATQSYGAYPTQPGQGYSQQSSQPYGQQSYSGYSQSTDTSGYGQSSYSSYGQSQNTGYGTQSTPQGYGSTGGYGSSQSSQSSYGQQSSYPGYGQQPAPSSTSGSYGSSSQSSSYGQPQSGSYSQQPSYGGQQQSYGQQQSYNPPQGYGQQNQYNSSSGGGGGGGGGGNYGQDQSSMSSGGGSGGGYGNQDQSGGGGSGGYGQQDRGGRGRGGSGGGGQQQQQQWQWQWWQQQQQQQWWWWWWWWWWWWWWPPPPPPPPPPPPPQQQQPPPPPPPPPPPPPQQPW GVVPI\" #@param {type:\"string\"}\n",
        "ID = None #@param {type:\"string\"}\n",
        "Directory = None #@param {type:\"string\"}\n",
        "#@markdown  - Use `End_sequence` to focus the scoring on sequences up to the specified endpoint\n",
        "\n",
        "End_sequence = 3 #@param {type:\"slider\" , min : 0 , max: 1000 , step : 1}\n",
        "Plot = True #@param {type: \"boolean\"}\n",
        "\n",
        "#@markdown - Specify the path of `FASTA format file` of your sequences.\n",
        "Fasta_file = \"/content/DrLLPS.fasta\" #@param {type:\"string\"}\n",
        "import pandas as pd\n",
        "#assert [Sequence == None , Fasta_file == None].count(True) == 1\n",
        "\n",
        "def enh(x):\n",
        "  wei = np.array([np.exp((i-1)/(i)) for i in x])\n",
        "  return np.sum(np.array(x)*wei)/np.sum(wei)\n",
        "\n",
        "def edit(i):\n",
        "  for j in i:\n",
        "    if j in \"\\t\\nBJOUXZ, _@#$%^& *n({}[])\\/\\\\\":\n",
        "      i = i.replace(j,\"\")\n",
        "  return i\n",
        "import xgboost as xgb\n",
        "from sklearn.model_selection import train_test_split as tts\n",
        "\n",
        "\n",
        "\n",
        "def enh(x):\n",
        "  wei = np.array([np.exp((i-1)/(i)) for i in x])\n",
        "  return np.sum(np.array(x)*wei)/np.sum(wei)\n",
        "\n",
        "\n",
        "def SCORE(i):\n",
        "  ii = clf.predict_proba(i)[:,1][0]\n",
        "  TT = [j for j in i.ravel() if j != 0]\n",
        "  if ii>= 0.294818:\n",
        "    S = 0.5*(max(TT)+i[0][-1])\n",
        "  else:\n",
        "    S = 0.5*(i[0][-1]+min(TT))\n",
        "  return S\n",
        "\n",
        "\n",
        "def SW(s):\n",
        "  if len(s)> 700:\n",
        "    L= [i for i in  [s[i*700:(i+1)*700] for i in range(len(s)//700+1)] if len(i)>0]\n",
        "    H = []\n",
        "    for i in range(len(L)):\n",
        "      H.append(model3.predict_proba(L[i]))\n",
        "    return H\n",
        "  else:\n",
        "    return model3.predict_proba(s)\n",
        "\n",
        "\n",
        "def Score1(i,Sc):\n",
        "    i = i-1\n",
        "    if i<= L:\n",
        "      return sum([Sc[j] for j in range(i+1)])/(i+1)\n",
        "\n",
        "    elif L+1<=i<n-L:\n",
        "       return sum([Sc[j] for j in range(i-L,i+1)])/(L+1)\n",
        "\n",
        "    else:\n",
        "       u = (n-(i)+1)\n",
        "       return sum([Sc[j] for j in range(i-L,n-L+1)])/(u)\n",
        "Sequence = edit(Sequence)\n",
        "\n",
        "\n",
        "def d(x):\n",
        "    if u>0.7:\n",
        "        return x*np.exp(-1.2*(x-u))\n",
        "    else:\n",
        "        if x > 0.7 :\n",
        "            return x\n",
        "        else:\n",
        "            return x\n",
        "def padd(i):\n",
        "  i = i+[float(0)]*(5537-len(i))\n",
        "  return i\n",
        "\n",
        "if Sequence != None and Sequence !=\"\":\n",
        "  Sequence = edit(Sequence)\n",
        "  k = max(5,min(50,int(np.ceil(0.1*len(Sequence)))))\n",
        "  L = k//2\n",
        "  if Plot == True:\n",
        "    n = len(Sequence)\n",
        "    S = [Sequence[i:i+L] for i in range(len(Sequence)-L+1)]\n",
        "    Sc = SW(S)\n",
        "    Sc1 = Sc[0]\n",
        "    for i in Sc[1:]:\n",
        "      Sc1 = np.concatenate((Sc1,i))\n",
        "    u = model3.predict_proba([Sequence])\n",
        "\n",
        "    p = list(map(d,[Score1(i,Sc1) for i in range(1,n+1)]))\n",
        "    if type(p[0]) != np.float64:\n",
        "      p = [i[0] for i in p]\n",
        "    if type(p[0]) != np.float64:\n",
        "      p = [i[0] for i in p]\n",
        "\n",
        "    i = p\n",
        "    values = np.array(i).reshape(1, len(i))\n",
        "    below_threshold = [k if k<= 0.5 else 0.5 for k in i]\n",
        "    above_threshold = [0 if k<= 0.5 else k-0.5 for k in i]\n",
        "    threshold = 0.5\n",
        "    y = list(range(len(i)))\n",
        "    Y = [(j, 1) for j in range(len(i)) if np.mean([i[j], i[min((j + 1), len(i) - 1)], i[max(0, (j - 1))]]) > 0.6]\n",
        "    Y2 = [(j, 1) for j in range(len(i)) if np.mean([i[j], i[min((j + 1), len(i) - 1)], i[min((j + 2), len(i) - 1)],\n",
        "                                               i[min((j + 3), len(i) - 1)], i[min((j + 4), len(i) - 1)],\n",
        "                                               i[min((j + 5), len(i) - 1)], i[max(0, (j - 1))],\n",
        "                                               i[max(0, (j - 2))], i[max(0, (j - 3))], i[max(0, (j - 4))],\n",
        "                                               i[max(0, (j - 5))]]) > 0.85]\n",
        "    #if n>= 1500:\n",
        "      #c1 = \"blue\"\n",
        "      #c2 = \"green\"\n",
        "    #else:\n",
        "    c1 = \"lightblue\"\n",
        "    c2 = \"lightgreen\"\n",
        "\n",
        "    fig1 = make_subplots(rows=1, cols=1, shared_xaxes=True, shared_yaxes=True, vertical_spacing=0.02)\n",
        "    fig1.add_trace(go.Bar(x=y, y=below_threshold, marker_color= c1, name='Below Threshold', width= 0.6), row=1, col=1)\n",
        "    fig1.add_trace(go.Bar(x=y, y=above_threshold, marker_color=c2, name='Above Threshold', base=below_threshold, width= 0.6), row=1, col=1)\n",
        "\n",
        "    fig1.add_trace(go.Scatter(x=[0, len(i) - 1], y=[threshold, threshold], mode='lines', line=dict(color='black', dash='dash'), showlegend=False), row=1, col=1)\n",
        "    for start, duration in Y:\n",
        "        fig1.add_trace(go.Scatter(x=[start, start + duration], y=[0.01, 0.01], mode='lines', line=dict(color='red', width=7), showlegend=False), row=1, col=1)\n",
        "    fig1.update_layout(height=800, width=1700, title_text=\"LLPS Prediction Plot\")\n",
        "    if type(u) != np.float64:\n",
        "      u = u[0][0]\n",
        "    daa = np.array(padd(p)+[u]).reshape(1,5538)\n",
        "    score = str(SCORE(daa))\n",
        "    print(\"Score : \"+score)\n",
        "\n",
        "\n",
        "else:\n",
        "  if os.path.exists(Fasta_file):\n",
        "    with open(Fasta_file) as f:\n",
        "      U = f.read().split(\">\")\n",
        "      w = len(U)\n",
        "    End_sequence = min(End_sequence,len(U))\n",
        "    j = U[1:End_sequence]\n",
        "    D = {\"id\":[i[:i.index(\"\\n\")] if \"\\n\" in i else i for i in j],\"seq\":[edit(i[i.index(\"\\n\"):]) if \"\\n\" in i else None for i in j],\"score\":[]}\n",
        "    #D[\"score\"] = [i[0][0] if type(i) != np.float64 else i for i in  [model3.predict_proba([s]) for s in tqdm(D[\"seq\"])]]\n",
        "    for sequence in tqdm(D[\"seq\"]):\n",
        "      if sequence == None :\n",
        "        D[\"score\"].append(\"_\")\n",
        "      else:\n",
        "        sequence = edit(sequence)\n",
        "        k = max(5,min(50,int(np.ceil(0.1*len(sequence)))))\n",
        "        L = k//2\n",
        "        n = len(sequence)\n",
        "        S = [sequence[i:i+L] for i in range(len(sequence)-L+1)]\n",
        "        if len(S)%700 == 0:\n",
        "          D[\"score\"].append(\"_\")\n",
        "        else:\n",
        "          Sc = SW(S)\n",
        "          Sc1 = Sc[0]\n",
        "          for i in Sc[1:]:\n",
        "            Sc1 = np.concatenate((Sc1,i))\n",
        "          u = model3.predict_proba([sequence])\n",
        "          p = list(map(d,[Score1(i,Sc1) for i in range(1,n+1)]))\n",
        "          if type(p[0]) != np.float64:\n",
        "            p = [i[0] for i in p]\n",
        "          if type(p[0]) != np.float64:\n",
        "            p = [i[0] for i in p]\n",
        "\n",
        "          if type(u) != np.float64:\n",
        "            u = u[0][0]\n",
        "          daa = np.array(padd(p)+[u]).reshape(1,5538)\n",
        "          score = str(SCORE(daa))\n",
        "          D[\"score\"].append(score)\n",
        "      if len(D[\"score\"])%1000 == 0:\n",
        "        print(D[\"score\"])\n",
        "\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "cellView": "form",
        "id": "YV1vl5J0Mn5W",
        "outputId": "006757bb-3940-4334-9eb3-2001cc65c6e5"
      },
      "execution_count": 7,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Score : 0.9447311731987048\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title plot\n",
        "import matplotlib.pyplot as plt\n",
        "import streamlit as st\n",
        "from stmol import showmol\n",
        "import py3Dmol\n",
        "import requests\n",
        "import biotite.structure.io as bsio\n",
        "import certifi\n",
        "import matplotlib.colors as mcolors\n",
        "import urllib3\n",
        "\n",
        "if Plot == True and Sequence != \"\" and Sequence != None:\n",
        "  fig1.show()\n",
        "\n",
        "\n",
        "  urllib3.disable_warnings()\n",
        "\n",
        "  st.sidebar.title('üéà ESMFold')\n",
        "  st.sidebar.write('[*ESMFold*](https://esmatlas.com/about) is an end-to-end single sequence protein structure predictor based on the ESM-2 language model. For more information, read the [research article](https://www.biorxiv.org/content/10.1101/2022.07.20.500902v2) and the [news article](https://www.nature.com/articles/d41586-022-03539-1) published in *Nature*.')\n",
        "\n",
        "  def render_mol(pdb):\n",
        "    pdbview = py3Dmol.view()\n",
        "    pdbview.addModel(pdb, 'pdb')\n",
        "    pdbview.setStyle({'cartoon': {'color': 'spectrum'}})\n",
        "    pdbview.setBackgroundColor('white')  # ('0xeeeeee')\n",
        "    pdbview.zoomTo()\n",
        "    pdbview.zoom(2, 800)\n",
        "    pdbview.spin(True)\n",
        "    showmol(pdbview, height=500, width=800)\n",
        "\n",
        "\n",
        "  DEFAULT_SEQ = Sequence\n",
        "  show_mainchains = True #@param {type: \"boolean\"}\n",
        "  show_sidechains = True #@param {type: \"boolean\"}\n",
        "  Color_Style =  \"red_blue_spectrum\" #@param [\"red_blue_spectrum\",\"green_magenta_spectrum\",\"yellow_blue_spectrum\",\"cyan_red_spectrum\"]\n",
        "\n",
        "  import streamlit as st\n",
        "  import requests\n",
        "  import biotite.structure.io as bsio\n",
        "  import py3Dmol\n",
        "\n",
        "  DEFAULT_SEQ = Sequence\n",
        "\n",
        "  txt = st.sidebar.text_area('Input sequence', DEFAULT_SEQ, height=275)\n",
        "\n",
        "  def update(sequence,name):\n",
        "    headers = {\n",
        "        'Content-Type': 'application/x-www-form-urlencoded',\n",
        "    }\n",
        "\n",
        "    def get_pdb(sequence):\n",
        "        response = requests.post('https://api.esmatlas.com/foldSequence/v1/pdb/', headers=headers, data=sequence, verify=False)\n",
        "        if response.status_code != 200:\n",
        "            st.error(f\"Failed to get prediction: {response.status_code}\")\n",
        "            return None\n",
        "        pdb_string = response.content.decode('utf-8')\n",
        "        if not pdb_string or \"ATOM\" not in pdb_string:\n",
        "            st.error(\"Received an invalid PDB file.\")\n",
        "            return None\n",
        "        return pdb_string\n",
        "\n",
        "    def save_and_render_pdb(pdb_string, file_name):\n",
        "        with open(file_name, 'w') as f:\n",
        "            f.write(pdb_string)\n",
        "        try:\n",
        "            struct = bsio.load_structure(file_name, extra_fields=[\"b_factor\"])\n",
        "            b_value = round(struct.b_factor.mean(), 4)\n",
        "        except Exception as e:\n",
        "            st.error(f\"Error loading structure: {e}\")\n",
        "            return\n",
        "        st.subheader('Visualization of predicted protein structure')\n",
        "        render_mol(pdb_string)\n",
        "        st.subheader('plDDT')\n",
        "        st.write('plDDT is a per-residue estimate of the confidence in prediction on a scale from 0-100.')\n",
        "        st.info(f'plDDT: {b_value}')\n",
        "        st.download_button(label=\"Download PDB\", data=pdb_string, file_name=file_name, mime='text/plain')\n",
        "\n",
        "    def render_mol(pdb_string,neme):\n",
        "        view = py3Dmol.view(width=1000, height=800)\n",
        "        view.addModel(pdb_string, 'pdb')\n",
        "        F1 = Get_color(p)\n",
        "        F1.i = F1.J.index(Color_Style)\n",
        "\n",
        "        custom_colors=F1.spectrum()\n",
        "\n",
        "        for i, color in enumerate(custom_colors[1:-1]):\n",
        "          view.setStyle({ 'resi': i+1}, {'cartoon': {'color': color}})\n",
        "        #view.setStyle({'cartoon': {'color': 'spectrum'}})\n",
        "        view.zoomTo()\n",
        "        view.show()\n",
        "        html_content = view._make_html()\n",
        "        with open(name+'.html', 'w') as file:\n",
        "          file.write(html_content)\n",
        "        files.download(name+'.html')\n",
        "\n",
        "\n",
        "    chunk_size = 400\n",
        "    chunks = [sequence[i:i + chunk_size] for i in range(0, len(sequence), chunk_size)]\n",
        "\n",
        "    combined_pdb_string = \"\"\n",
        "    for i, chunk in enumerate(chunks):\n",
        "        st.write(f\"Processing chunk {i + 1} of {len(chunks)}\")\n",
        "        pdb_string = get_pdb(chunk)\n",
        "        if pdb_string:\n",
        "            combined_pdb_string += pdb_string\n",
        "\n",
        "    if combined_pdb_string:\n",
        "        render_mol(combined_pdb_string,name)\n",
        "        with open('combined_predicted.pdb', 'w') as f:\n",
        "            f.write(combined_pdb_string)\n",
        "        st.download_button(label=\"Download Combined PDB\", data=combined_pdb_string, file_name='combined_predicted.pdb', mime='text/plain')\n",
        "\n",
        "  predict = st.sidebar.button('Predict', on_click=update)\n",
        "\n",
        "  if not predict:\n",
        "    st.warning('üëà Enter protein sequence data!')\n",
        "\n",
        "  class Get_color():\n",
        "    def __init__(self,values):\n",
        "      self.J = [\"red_blue_spectrum\",\"green_magenta_spectrum\",\"yellow_blue_spectrum\",\"cyan_red_spectrum\"]\n",
        "      self.i = None\n",
        "      self.values = values\n",
        "\n",
        "    def get_red_blue_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=min(values), vmax=max(values))\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"RedBlue\", [\"blue\", \"red\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "    def get_green_magenta_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=min(values), vmax=max(values))\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"GreenMagenta\", [\"green\", \"magenta\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "    def get_yellow_blue_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=min(values), vmax=max(values))\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"YellowBlue\", [\"yellow\", \"blue\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "\n",
        "    def get_cyan_red_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=min(values), vmax=max(values))\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"CyanRed\", [\"cyan\", \"red\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "    def spectrum(self):\n",
        "      if self.i == 0 : return self.get_red_blue_spectrum(self.values)\n",
        "      if self.i == 1 : return self.get_green_magenta_spectrum(self.values)\n",
        "      if self.i == 2 : return self.get_yellow_blue_spectrum(self.values)\n",
        "      if self.i == 3 : return self.get_cyan_red_spectrum(self.values)\n",
        "  update(txt,\"A\")\n",
        "\n",
        "\n",
        "# Create a color bar legend using matplotlib\n",
        "  fig, ax = plt.subplots(figsize=(6, 1))\n",
        "  fig.subplots_adjust(bottom=0.5)\n",
        "\n",
        "  cmap = plt.cm.Reds\n",
        "  norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "  fig.colorbar(plt.cm.ScalarMappable(norm=norm, cmap=cmap), cax=ax, orientation='horizontal', label='Predicted LLPS Score')\n",
        "  plt.show()\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n"
      ],
      "metadata": {
        "cellView": "form",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "LSnHOep_Mvyy",
        "outputId": "7edf7fd8-9ed6-4fb0-b880-031c24ba8c31"
      },
      "execution_count": 9,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<html>\n",
              "<head><meta charset=\"utf-8\" /></head>\n",
              "<body>\n",
              "    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax && window.MathJax.Hub && window.MathJax.Hub.Config) {window.MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n",
              "        <script charset=\"utf-8\" src=\"https://cdn.plot.ly/plotly-2.24.1.min.js\"></script>                <div id=\"73ac95a3-c21b-4975-943d-5d607faa32db\" class=\"plotly-graph-div\" style=\"height:800px; width:1700px;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"73ac95a3-c21b-4975-943d-5d607faa32db\")) {                    Plotly.newPlot(                        \"73ac95a3-c21b-4975-943d-5d607faa32db\",                        [{\"marker\":{\"color\":\"lightblue\"},\"name\":\"Below Threshold\",\"width\":0.6,\"x\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131,132,133,134,135,136,137,138,139,140,141,142,143,144,145,146,147,148,149,150,151,152,153,154,155,156,157,158,159,160,161,162,163,164,165,166,167,168,169,170,171,172,173,174,175,176,177,178,179,180,181,182,183,184,185,186,187,188,189,190,191,192,193,194,195,196,197,198,199,200,201,202,203,204,205,206,207,208,209,210,211,212,213,214,215,216,217,218,219,220,221,222,223,224,225,226,227,228,229,230,231,232,233,234,235,236,237,238,239,240,241,242,243,244,245,246,247,248,249,250,251,252,253,254,255,256,257,258,259,260,261,262,263,264,265,266,267,268,269,270,271,272,273,274,275,276,277,278,279,280,281,282,283,284,285,286,287,288,289,290,291,292,293,294,295,296],\"y\":[0.15031292800375054,0.38056531922952824,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.44399105403839156,0.33258631502585584,0.20234143576192676,0.20743511185991784,0.21320982789060156,0.19174010067969768,0.1951456445123958,0.20179353358725105,0.20508005812438773,0.20442194169362135,0.1991969328297372,0.19471851002192267,0.1919337617838195,0.18948188657861664,0.18695626384723127,0.1854157170870893,0.3231628397640395,0.44271765110415257,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5],\"type\":\"bar\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"base\":[0.15031292800375054,0.38056531922952824,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.44399105403839156,0.33258631502585584,0.20234143576192676,0.20743511185991784,0.21320982789060156,0.19174010067969768,0.1951456445123958,0.20179353358725105,0.20508005812438773,0.20442194169362135,0.1991969328297372,0.19471851002192267,0.1919337617838195,0.18948188657861664,0.18695626384723127,0.1854157170870893,0.3231628397640395,0.44271765110415257,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5],\"marker\":{\"color\":\"lightgreen\"},\"name\":\"Above Threshold\",\"width\":0.6,\"x\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131,132,133,134,135,136,137,138,139,140,141,142,143,144,145,146,147,148,149,150,151,152,153,154,155,156,157,158,159,160,161,162,163,164,165,166,167,168,169,170,171,172,173,174,175,176,177,178,179,180,181,182,183,184,185,186,187,188,189,190,191,192,193,194,195,196,197,198,199,200,201,202,203,204,205,206,207,208,209,210,211,212,213,214,215,216,217,218,219,220,221,222,223,224,225,226,227,228,229,230,231,232,233,234,235,236,237,238,239,240,241,242,243,244,245,246,247,248,249,250,251,252,253,254,255,256,257,258,259,260,261,262,263,264,265,266,267,268,269,270,271,272,273,274,275,276,277,278,279,280,281,282,283,284,285,286,287,288,289,290,291,292,293,294,295,296],\"y\":[0,0,0.2749368935093086,0.17449634064116248,0.09893042243782935,0.042047315621276726,0.00019702150939826524,0.1474650187162614,0.23639676119616193,0.2897042849706257,0.2619768017401203,0.30407053552145713,0.3350901989881584,0.3580813067312446,0.37554118768974865,0.3889603943192953,0.41314217275730825,0.4094210999752895,0.3882088659710584,0.41256348245734475,0.43000335156076186,0.4412571063279328,0.4470793914780582,0.44708621681376315,0.4470850038690113,0.4472574049421507,0.44825582323315194,0.44824351971762233,0.4482408345633052,0.4482398376144169,0.448239880271616,0.44823854532987684,0.44824360895944493,0.44539209831866955,0.4399264820869716,0.4399653759155706,0.4400330395969797,0.4460531125052236,0.446070443165287,0.4460741517804261,0.44836249725971467,0.4483631211622888,0.4483634730958702,0.44637758190269383,0.4403187103904501,0.42856972465468013,0.4113004522578044,0.4113051648681667,0.4113808661910733,0.41131253658480005,0.41138214744720336,0.41143250168413203,0.4115271899324734,0.4292094632411254,0.4292041268194118,0.4291869729689991,0.4292585140808852,0.4292571057567658,0.4292804108821717,0.44081901302134874,0.4432934667312798,0.44781970206716104,0.4477433466608072,0.4434416418578534,0.44344919677644845,0.44346215116382115,0.43565725750489637,0.42614391324579803,0.4262025908540217,0.4072441741782158,0.382946337359779,0.38304235653356766,0.40867229851310627,0.40873248586170885,0.40872600161299766,0.408774693662853,0.39949054010313023,0.39835115181619507,0.4199082744941237,0.43407479052839837,0.4340572311875325,0.43406490767541195,0.4424922085255323,0.44656923623869604,0.44656244438899917,0.44834973616835605,0.446154306223834,0.4461580645191777,0.44616735291976806,0.44617059136042103,0.44617983457137567,0.44618059779155483,0.44028851792794665,0.43990662544552583,0.4399139277844437,0.4399091248567769,0.43992537992132474,0.4399191573173946,0.4399343463375417,0.4399626114537971,0.4399557828755042,0.439964561943469,0.44000815877661936,0.4400300069667309,0.44352556034218615,0.4435236425787348,0.4435258544293943,0.4477246481225782,0.44797344251124305,0.4479742093477601,0.44797384419853303,0.4479729503336646,0.4479764378494969,0.447974410830674,0.44797519741152525,0.4479821058281217,0.44798151676194387,0.44798607740243446,0.4479948238012217,0.4479968980875362,0.4483500975137681,0.44835128870433205,0.4483497413043591,0.44590957863756653,0.4399158895457683,0.439904637438449,0.4398954771986997,0.4398859846020664,0.4398817261423602,0.43988840931430684,0.4398690912486266,0.43985298712827015,0.4398484682076951,0.43984487609833733,0.4398362619861148,0.43981629140275347,0.4398039332362187,0.4397762227583506,0.43977694986013316,0.4397780482506337,0.43977895599798233,0.4397920475748869,0.4397988568904555,0.4397901995494139,0.4397788890089507,0.439769593428763,0.4397838190846052,0.43980851618814976,0.43982423594907316,0.4398347035529798,0.43983153917687656,0.43983187319341,0.4398344348267975,0.439832818764879,0.43984759533922757,0.43984376759152755,0.4398486084216957,0.43984250841047634,0.4398406000713604,0.4398507808297577,0.43985329697231446,0.43984172377358444,0.4398295857095864,0.43980518201603946,0.43977863974537124,0.4397627913250828,0.4397588653730451,0.4397512778361753,0.4397421056560751,0.4397412591160407,0.4397347847362033,0.4397398838640336,0.4397293260534222,0.43973599016376863,0.43973149544892276,0.43976502293144304,0.4397802261485868,0.43978786551883253,0.43979798640515066,0.4398108378368769,0.43982213510277535,0.4398307825688179,0.4398360057077477,0.4398474416646746,0.4398522934523197,0.43985275280210534,0.43984179002386703,0.4398317318196888,0.43983537405737017,0.4398225475190697,0.43981697655140106,0.43978622680350954,0.43976806173772276,0.43977600514178683,0.4397724974852384,0.4397662285348708,0.43975962952328396,0.43975292631322116,0.4397515056152752,0.439742710750585,0.4397382524780252,0.439738414321884,0.43974587742962024,0.43974937489070387,0.43974895880745923,0.43975566288755075,0.43976606224926584,0.43977119253275454,0.43976496472797333,0.4397511889079827,0.439752975193338,0.440757836106639,0.44075610733987025,0.4407547521297104,0.4407520197406508,0.44075489916251853,0.44075448904103776,0.440759827704459,0.44075350598725593,0.4407594939033933,0.44076056115858164,0.44076515140056316,0.4407729557106186,0.44078282479084674,0.44080402382248907,0.4408415892404083,0.4408982771752368,0.4403708969313974,0.4404191535982015,0.4462558548031478,0.4483697956428625,0.4468834819433477,0.4406617838900587,0.4289881046051769,0.41107866654923986,0.38625740972533473,0.3546945976814274,0.3146559278538381,0.26466515225180387,0.20388339187200377,0.13112175168269746,0.045054308796595666,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.041604490464460686,0.1262595561259584,0.19827344843807226,0.2594505646417681,0.3101345304614468,0.3504255538416746,0.3832685569767533,0.4089892803643661,0.4275842256680038,0.4398568613657873,0.44655406588782776,0.4483463745582682,0.44584244173432985,0.43962658917111563,0.4396267524774117,0.4396266755351932,0.4396265698045728,0.43963253431052296,0.43963416419594425,0.4396429930227391,0.43965859448971656,0.4396715694043869,0.4397260884240636,0.43973283868173774,0.43974047304683217,0.4397488838314976,0.4397584325143047,0.43976950562482797,0.43978370362274066,0.439801303339207,0.43982321586312245,0.4398517208752136,0.43988953137805054,0.43992314292854096,0.4399901580253849,0.4400640612186302,0.4401584690294408],\"type\":\"bar\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"black\",\"dash\":\"dash\"},\"mode\":\"lines\",\"showlegend\":false,\"x\":[0,296],\"y\":[0.5,0.5],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[2,3],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[3,4],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[4,5],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[7,8],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[8,9],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[9,10],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[10,11],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[11,12],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[12,13],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[13,14],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[14,15],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[15,16],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[16,17],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[17,18],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[18,19],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[19,20],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[20,21],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[21,22],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[22,23],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[23,24],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[24,25],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[25,26],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[26,27],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[27,28],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[28,29],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[29,30],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[30,31],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[31,32],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[32,33],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[33,34],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[34,35],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[35,36],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[36,37],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[37,38],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[38,39],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[39,40],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[40,41],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[41,42],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[42,43],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[43,44],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[44,45],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[45,46],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[46,47],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[47,48],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[48,49],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[49,50],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[50,51],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[51,52],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[52,53],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[53,54],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[54,55],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[55,56],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[56,57],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[57,58],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[58,59],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[59,60],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[60,61],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[61,62],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[62,63],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[63,64],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[64,65],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[65,66],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[66,67],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[67,68],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[68,69],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[69,70],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[70,71],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[71,72],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[72,73],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[73,74],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[74,75],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[75,76],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[76,77],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[77,78],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[78,79],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[79,80],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[80,81],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[81,82],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[82,83],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[83,84],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[84,85],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[85,86],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[86,87],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[87,88],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[88,89],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[89,90],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[90,91],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[91,92],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[92,93],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[93,94],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[94,95],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[95,96],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[96,97],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[97,98],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[98,99],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[99,100],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[100,101],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[101,102],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[102,103],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[103,104],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[104,105],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[105,106],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[106,107],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[107,108],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[108,109],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[109,110],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[110,111],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[111,112],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[112,113],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[113,114],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[114,115],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[115,116],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[116,117],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[117,118],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[118,119],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[119,120],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[120,121],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[121,122],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[122,123],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[123,124],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[124,125],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[125,126],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[126,127],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[127,128],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[128,129],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[129,130],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[130,131],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[131,132],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[132,133],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[133,134],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[134,135],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[135,136],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[136,137],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[137,138],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[138,139],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[139,140],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[140,141],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[141,142],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[142,143],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[143,144],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[144,145],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[145,146],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[146,147],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[147,148],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[148,149],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[149,150],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[150,151],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[151,152],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[152,153],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[153,154],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[154,155],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[155,156],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[156,157],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[157,158],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[158,159],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[159,160],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[160,161],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[161,162],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[162,163],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[163,164],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[164,165],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[165,166],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[166,167],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[167,168],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[168,169],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[169,170],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[170,171],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[171,172],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[172,173],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[173,174],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[174,175],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[175,176],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[176,177],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[177,178],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[178,179],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[179,180],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[180,181],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[181,182],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[182,183],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[183,184],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[184,185],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[185,186],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[186,187],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[187,188],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[188,189],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[189,190],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[190,191],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[191,192],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[192,193],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[193,194],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[194,195],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[195,196],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[196,197],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[197,198],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[198,199],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[199,200],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[200,201],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[201,202],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[202,203],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[203,204],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[204,205],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[205,206],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[206,207],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[207,208],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[208,209],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[209,210],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[210,211],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[211,212],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[212,213],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[213,214],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[214,215],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[215,216],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[216,217],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[217,218],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[218,219],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[219,220],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[220,221],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[221,222],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[222,223],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[223,224],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[224,225],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[225,226],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[226,227],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[227,228],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[228,229],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[229,230],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[230,231],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[231,232],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[232,233],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[233,234],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[234,235],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[235,236],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[236,237],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[237,238],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[238,239],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[239,240],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[240,241],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[261,262],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[262,263],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[263,264],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[264,265],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[265,266],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[266,267],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[267,268],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[268,269],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[269,270],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[270,271],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[271,272],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[272,273],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[273,274],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[274,275],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[275,276],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[276,277],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[277,278],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[278,279],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[279,280],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[280,281],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[281,282],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[282,283],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[283,284],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[284,285],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[285,286],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[286,287],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[287,288],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[288,289],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[289,290],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[290,291],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[291,292],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[292,293],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[293,294],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[294,295],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[295,296],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[296,297],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"}],                        {\"template\":{\"data\":{\"candlestick\":[{\"decreasing\":{\"line\":{\"color\":\"#000033\"}},\"increasing\":{\"line\":{\"color\":\"#000032\"}},\"type\":\"candlestick\"}],\"contourcarpet\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"contourcarpet\"}],\"contour\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"contour\"}],\"heatmap\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"heatmap\"}],\"histogram2d\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"histogram2d\"}],\"icicle\":[{\"textfont\":{\"color\":\"white\"},\"type\":\"icicle\"}],\"sankey\":[{\"textfont\":{\"color\":\"#000036\"},\"type\":\"sankey\"}],\"scatter\":[{\"marker\":{\"line\":{\"width\":0}},\"type\":\"scatter\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#000038\"},\"font\":{\"color\":\"#000037\"},\"line\":{\"color\":\"#000039\"}},\"header\":{\"fill\":{\"color\":\"#000040\"},\"font\":{\"color\":\"#000036\"},\"line\":{\"color\":\"#000039\"}},\"type\":\"table\"}],\"waterfall\":[{\"connector\":{\"line\":{\"color\":\"#000036\",\"width\":2}},\"decreasing\":{\"marker\":{\"color\":\"#000033\"}},\"increasing\":{\"marker\":{\"color\":\"#000032\"}},\"totals\":{\"marker\":{\"color\":\"#000034\"}},\"type\":\"waterfall\"}]},\"layout\":{\"coloraxis\":{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]]},\"colorscale\":{\"diverging\":[[0.0,\"#000021\"],[0.1,\"#000022\"],[0.2,\"#000023\"],[0.3,\"#000024\"],[0.4,\"#000025\"],[0.5,\"#000026\"],[0.6,\"#000027\"],[0.7,\"#000028\"],[0.8,\"#000029\"],[0.9,\"#000030\"],[1.0,\"#000031\"]],\"sequential\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"sequentialminus\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]]},\"colorway\":[\"#000001\",\"#000002\",\"#000003\",\"#000004\",\"#000005\",\"#000006\",\"#000007\",\"#000008\",\"#000009\",\"#000010\"]}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0]},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0]},\"title\":{\"text\":\"LLPS Prediction Plot\"},\"height\":800,\"width\":1700},                        {\"responsive\": true}                    ).then(function(){\n",
              "                            \n",
              "var gd = document.getElementById('73ac95a3-c21b-4975-943d-5d607faa32db');\n",
              "var x = new MutationObserver(function (mutations, observer) {{\n",
              "        var display = window.getComputedStyle(gd).display;\n",
              "        if (!display || display === 'none') {{\n",
              "            console.log([gd, 'removed!']);\n",
              "            Plotly.purge(gd);\n",
              "            observer.disconnect();\n",
              "        }}\n",
              "}});\n",
              "\n",
              "// Listen for the removal of the full notebook cells\n",
              "var notebookContainer = gd.closest('#notebook-container');\n",
              "if (notebookContainer) {{\n",
              "    x.observe(notebookContainer, {childList: true});\n",
              "}}\n",
              "\n",
              "// Listen for the clearing of the current output cell\n",
              "var outputEl = gd.closest('.output');\n",
              "if (outputEl) {{\n",
              "    x.observe(outputEl, {childList: true});\n",
              "}}\n",
              "\n",
              "                        })                };                            </script>        </div>\n",
              "</body>\n",
              "</html>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "2024-08-31 16:23:39.158 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.160 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.162 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.164 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.166 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.171 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.173 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.173 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.175 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.176 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.179 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.180 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.181 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.182 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.184 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.186 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.188 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.189 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.191 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.192 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.193 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17251214197132845\"  style=\"position: relative; width: 1000px; height: 800px;\">\n        <p id=\"3dmolwarning_17251214197132845\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.3.0/3Dmol-min.js');\n}\n\nvar viewer_17251214197132845 = null;\nvar warn = document.getElementById(\"3dmolwarning_17251214197132845\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17251214197132845 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17251214197132845\"),{backgroundColor:\"white\"});\nviewer_17251214197132845.zoomTo();\n\tviewer_17251214197132845.addModel(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   MET A   1      23.464 -21.164 -38.766  1.00  0.39           N  \\nATOM      2  CA  MET A   1      23.372 -20.209 -37.665  1.00  0.43           C  \\nATOM      3  C   MET A   1      22.209 -19.245 -37.878  1.00  0.41           C  \\nATOM      4  CB  MET A   1      24.679 -19.429 -37.519  1.00  0.37           C  \\nATOM      5  O   MET A   1      22.304 -18.321 -38.688  1.00  0.41           O  \\nATOM      6  CG  MET A   1      25.852 -20.278 -37.057  1.00  0.39           C  \\nATOM      7  SD  MET A   1      27.395 -19.304 -36.870  1.00  0.38           S  \\nATOM      8  CE  MET A   1      28.571 -20.635 -36.501  1.00  0.36           C  \\nATOM      9  N   ALA A   2      21.042 -19.728 -38.337  1.00  0.45           N  \\nATOM     10  CA  ALA A   2      19.997 -18.719 -38.490  1.00  0.47           C  \\nATOM     11  C   ALA A   2      19.503 -18.231 -37.131  1.00  0.45           C  \\nATOM     12  CB  ALA A   2      18.834 -19.275 -39.306  1.00  0.40           C  \\nATOM     13  O   ALA A   2      19.086 -19.032 -36.291  1.00  0.42           O  \\nATOM     14  N   SER A   3      20.351 -17.463 -36.437  1.00  0.43           N  \\nATOM     15  CA  SER A   3      20.108 -16.483 -35.383  1.00  0.45           C  \\nATOM     16  C   SER A   3      18.732 -15.843 -35.530  1.00  0.43           C  \\nATOM     17  CB  SER A   3      21.188 -15.400 -35.398  1.00  0.39           C  \\nATOM     18  O   SER A   3      18.432 -15.234 -36.559  1.00  0.41           O  \\nATOM     19  OG  SER A   3      20.797 -14.289 -34.610  1.00  0.43           O  \\nATOM     20  N   ASN A   4      17.695 -16.560 -35.306  1.00  0.47           N  \\nATOM     21  CA  ASN A   4      16.308 -16.142 -35.135  1.00  0.51           C  \\nATOM     22  C   ASN A   4      16.211 -14.793 -34.429  1.00  0.47           C  \\nATOM     23  CB  ASN A   4      15.519 -17.203 -34.364  1.00  0.42           C  \\nATOM     24  O   ASN A   4      16.559 -14.677 -33.252  1.00  0.45           O  \\nATOM     25  CG  ASN A   4      14.674 -18.076 -35.271  1.00  0.41           C  \\nATOM     26  ND2 ASN A   4      14.021 -19.077 -34.692  1.00  0.39           N  \\nATOM     27  OD1 ASN A   4      14.609 -17.854 -36.483  1.00  0.42           O  \\nATOM     28  N   ASP A   5      16.721 -13.756 -35.098  1.00  0.41           N  \\nATOM     29  CA  ASP A   5      16.711 -12.298 -35.047  1.00  0.44           C  \\nATOM     30  C   ASP A   5      15.283 -11.759 -35.004  1.00  0.41           C  \\nATOM     31  CB  ASP A   5      17.457 -11.714 -36.249  1.00  0.37           C  \\nATOM     32  O   ASP A   5      14.610 -11.685 -36.034  1.00  0.39           O  \\nATOM     33  CG  ASP A   5      18.856 -11.235 -35.904  1.00  0.36           C  \\nATOM     34  OD1 ASP A   5      19.156 -11.043 -34.706  1.00  0.40           O  \\nATOM     35  OD2 ASP A   5      19.664 -11.044 -36.838  1.00  0.41           O  \\nATOM     36  N   TYR A   6      14.344 -12.453 -34.202  1.00  0.46           N  \\nATOM     37  CA  TYR A   6      13.027 -11.846 -34.046  1.00  0.49           C  \\nATOM     38  C   TYR A   6      12.848 -11.285 -32.640  1.00  0.45           C  \\nATOM     39  CB  TYR A   6      11.926 -12.867 -34.345  1.00  0.41           C  \\nATOM     40  O   TYR A   6      12.625 -12.037 -31.689  1.00  0.43           O  \\nATOM     41  CG  TYR A   6      11.702 -13.104 -35.819  1.00  0.41           C  \\nATOM     42  CD1 TYR A   6      10.875 -12.265 -36.562  1.00  0.39           C  \\nATOM     43  CD2 TYR A   6      12.316 -14.168 -36.471  1.00  0.41           C  \\nATOM     44  CE1 TYR A   6      10.664 -12.481 -37.920  1.00  0.38           C  \\nATOM     45  CE2 TYR A   6      12.113 -14.394 -37.828  1.00  0.42           C  \\nATOM     46  OH  TYR A   6      11.082 -13.765 -39.887  1.00  0.28           O  \\nATOM     47  CZ  TYR A   6      11.287 -13.546 -38.543  1.00  0.34           C  \\nATOM     48  N   THR A   7      13.825 -10.482 -32.090  1.00  0.37           N  \\nATOM     49  CA  THR A   7      13.997  -9.788 -30.818  1.00  0.39           C  \\nATOM     50  C   THR A   7      14.228  -8.296 -31.044  1.00  0.37           C  \\nATOM     51  CB  THR A   7      15.172 -10.376 -30.016  1.00  0.34           C  \\nATOM     52  O   THR A   7      15.286  -7.893 -31.531  1.00  0.36           O  \\nATOM     53  CG2 THR A   7      14.711 -10.873 -28.650  1.00  0.33           C  \\nATOM     54  OG1 THR A   7      15.740 -11.471 -30.744  1.00  0.32           O  \\nATOM     55  N   GLN A   8      13.395  -7.471 -31.807  1.00  0.36           N  \\nATOM     56  CA  GLN A   8      13.733  -6.052 -31.793  1.00  0.40           C  \\nATOM     57  C   GLN A   8      12.521  -5.194 -32.145  1.00  0.36           C  \\nATOM     58  CB  GLN A   8      14.880  -5.763 -32.763  1.00  0.34           C  \\nATOM     59  O   GLN A   8      12.001  -5.272 -33.261  1.00  0.35           O  \\nATOM     60  CG  GLN A   8      16.260  -5.841 -32.125  1.00  0.35           C  \\nATOM     61  CD  GLN A   8      17.377  -5.517 -33.099  1.00  0.44           C  \\nATOM     62  NE2 GLN A   8      18.617  -5.609 -32.631  1.00  0.33           N  \\nATOM     63  OE1 GLN A   8      17.127  -5.187 -34.263  1.00  0.47           O  \\nATOM     64  N   GLN A   9      11.306  -5.225 -31.302  1.00  0.31           N  \\nATOM     65  CA  GLN A   9      10.516  -4.023 -31.551  1.00  0.38           C  \\nATOM     66  C   GLN A   9       9.299  -3.965 -30.632  1.00  0.32           C  \\nATOM     67  CB  GLN A   9      10.073  -3.965 -33.014  1.00  0.29           C  \\nATOM     68  O   GLN A   9       8.494  -4.898 -30.600  1.00  0.31           O  \\nATOM     69  CG  GLN A   9      11.061  -3.252 -33.927  1.00  0.31           C  \\nATOM     70  CD  GLN A   9      10.555  -3.122 -35.352  1.00  0.41           C  \\nATOM     71  NE2 GLN A   9      11.355  -2.500 -36.211  1.00  0.32           N  \\nATOM     72  OE1 GLN A   9       9.455  -3.578 -35.677  1.00  0.46           O  \\nATOM     73  N   ALA A  10       9.494  -3.398 -29.307  1.00  0.33           N  \\nATOM     74  CA  ALA A  10       8.569  -2.423 -28.733  1.00  0.37           C  \\nATOM     75  C   ALA A  10       8.372  -2.667 -27.240  1.00  0.33           C  \\nATOM     76  CB  ALA A  10       7.227  -2.472 -29.459  1.00  0.31           C  \\nATOM     77  O   ALA A  10       7.714  -3.633 -26.844  1.00  0.33           O  \\nATOM     78  N   THR A  11       9.561  -2.440 -26.426  1.00  0.34           N  \\nATOM     79  CA  THR A  11       9.285  -2.253 -25.006  1.00  0.37           C  \\nATOM     80  C   THR A  11      10.083  -1.077 -24.450  1.00  0.34           C  \\nATOM     81  CB  THR A  11       9.614  -3.525 -24.201  1.00  0.32           C  \\nATOM     82  O   THR A  11      11.312  -1.133 -24.377  1.00  0.33           O  \\nATOM     83  CG2 THR A  11       8.493  -3.861 -23.223  1.00  0.30           C  \\nATOM     84  OG1 THR A  11       9.789  -4.623 -25.105  1.00  0.29           O  \\nATOM     85  N   GLN A  12       9.820   0.205 -24.774  1.00  0.37           N  \\nATOM     86  CA  GLN A  12      10.113   1.430 -24.037  1.00  0.40           C  \\nATOM     87  C   GLN A  12       9.630   1.332 -22.593  1.00  0.38           C  \\nATOM     88  CB  GLN A  12       9.472   2.637 -24.724  1.00  0.35           C  \\nATOM     89  O   GLN A  12       8.430   1.202 -22.342  1.00  0.36           O  \\nATOM     90  CG  GLN A  12      10.242   3.133 -25.941  1.00  0.34           C  \\nATOM     91  CD  GLN A  12       9.622   4.369 -26.564  1.00  0.42           C  \\nATOM     92  NE2 GLN A  12      10.271   4.903 -27.593  1.00  0.33           N  \\nATOM     93  OE1 GLN A  12       8.568   4.840 -26.124  1.00  0.46           O  \\nATOM     94  N   SER A  13      10.281   0.657 -21.663  1.00  0.35           N  \\nATOM     95  CA  SER A  13      10.150   0.310 -20.251  1.00  0.38           C  \\nATOM     96  C   SER A  13      11.030   1.198 -19.379  1.00  0.35           C  \\nATOM     97  CB  SER A  13      10.510  -1.159 -20.025  1.00  0.32           C  \\nATOM     98  O   SER A  13      12.223   1.356 -19.649  1.00  0.34           O  \\nATOM     99  OG  SER A  13      11.558  -1.559 -20.890  1.00  0.30           O  \\nATOM    100  N   TYR A  14      10.477   2.322 -18.631  1.00  0.37           N  \\nATOM    101  CA  TYR A  14      10.507   3.233 -17.492  1.00  0.41           C  \\nATOM    102  C   TYR A  14      11.507   2.761 -16.443  1.00  0.38           C  \\nATOM    103  CB  TYR A  14       9.114   3.357 -16.866  1.00  0.35           C  \\nATOM    104  O   TYR A  14      11.336   1.694 -15.849  1.00  0.37           O  \\nATOM    105  CG  TYR A  14       8.139   4.143 -17.709  1.00  0.34           C  \\nATOM    106  CD1 TYR A  14       8.094   5.534 -17.643  1.00  0.32           C  \\nATOM    107  CD2 TYR A  14       7.260   3.497 -18.571  1.00  0.35           C  \\nATOM    108  CE1 TYR A  14       7.196   6.262 -18.416  1.00  0.34           C  \\nATOM    109  CE2 TYR A  14       6.357   4.215 -19.348  1.00  0.37           C  \\nATOM    110  OH  TYR A  14       5.441   6.310 -20.032  1.00  0.25           O  \\nATOM    111  CZ  TYR A  14       6.333   5.594 -19.264  1.00  0.31           C  \\nATOM    112  N   GLY A  15      12.741   3.096 -16.410  1.00  0.36           N  \\nATOM    113  CA  GLY A  15      13.912   3.183 -15.553  1.00  0.37           C  \\nATOM    114  C   GLY A  15      13.801   4.268 -14.498  1.00  0.36           C  \\nATOM    115  O   GLY A  15      13.613   5.441 -14.825  1.00  0.35           O  \\nATOM    116  N   ALA A  16      13.440   4.188 -13.190  1.00  0.38           N  \\nATOM    117  CA  ALA A  16      13.467   4.948 -11.943  1.00  0.41           C  \\nATOM    118  C   ALA A  16      14.853   4.906 -11.305  1.00  0.38           C  \\nATOM    119  CB  ALA A  16      12.420   4.411 -10.970  1.00  0.35           C  \\nATOM    120  O   ALA A  16      15.307   3.848 -10.862  1.00  0.38           O  \\nATOM    121  N   TYR A  17      15.722   5.703 -11.634  1.00  0.36           N  \\nATOM    122  CA  TYR A  17      16.863   6.147 -10.841  1.00  0.39           C  \\nATOM    123  C   TYR A  17      16.563   7.470 -10.146  1.00  0.36           C  \\nATOM    124  CB  TYR A  17      18.107   6.292 -11.724  1.00  0.34           C  \\nATOM    125  O   TYR A  17      15.899   8.341 -10.715  1.00  0.35           O  \\nATOM    126  CG  TYR A  17      18.678   4.975 -12.189  1.00  0.35           C  \\nATOM    127  CD1 TYR A  17      19.537   4.240 -11.376  1.00  0.32           C  \\nATOM    128  CD2 TYR A  17      18.360   4.463 -13.442  1.00  0.34           C  \\nATOM    129  CE1 TYR A  17      20.067   3.026 -11.801  1.00  0.34           C  \\nATOM    130  CE2 TYR A  17      18.884   3.250 -13.878  1.00  0.38           C  \\nATOM    131  OH  TYR A  17      20.257   1.339 -13.478  1.00  0.25           O  \\nATOM    132  CZ  TYR A  17      19.735   2.540 -13.052  1.00  0.31           C  \\nATOM    133  N   PRO A  18      16.857   7.768  -8.739  1.00  0.38           N  \\nATOM    134  CA  PRO A  18      16.977   8.990  -7.941  1.00  0.40           C  \\nATOM    135  C   PRO A  18      17.524  10.168  -8.745  1.00  0.38           C  \\nATOM    136  CB  PRO A  18      17.949   8.591  -6.828  1.00  0.36           C  \\nATOM    137  O   PRO A  18      18.588  10.059  -9.359  1.00  0.37           O  \\nATOM    138  CG  PRO A  18      18.765   7.483  -7.413  1.00  0.35           C  \\nATOM    139  CD  PRO A  18      17.968   6.829  -8.505  1.00  0.41           C  \\nATOM    140  N   THR A  19      16.652  10.984  -9.367  1.00  0.35           N  \\nATOM    141  CA  THR A  19      16.871  12.360  -9.799  1.00  0.37           C  \\nATOM    142  C   THR A  19      17.816  13.083  -8.843  1.00  0.35           C  \\nATOM    143  CB  THR A  19      15.543  13.134  -9.895  1.00  0.33           C  \\nATOM    144  O   THR A  19      17.571  13.128  -7.636  1.00  0.34           O  \\nATOM    145  CG2 THR A  19      15.242  13.533 -11.336  1.00  0.32           C  \\nATOM    146  OG1 THR A  19      14.479  12.307  -9.408  1.00  0.31           O  \\nATOM    147  N   GLN A  20      18.986  13.116  -9.146  1.00  0.41           N  \\nATOM    148  CA  GLN A  20      19.956  14.126  -8.735  1.00  0.44           C  \\nATOM    149  C   GLN A  20      19.353  15.526  -8.801  1.00  0.42           C  \\nATOM    150  CB  GLN A  20      21.210  14.053  -9.608  1.00  0.39           C  \\nATOM    151  O   GLN A  20      18.637  15.856  -9.748  1.00  0.40           O  \\nATOM    152  CG  GLN A  20      22.192  12.969  -9.184  1.00  0.40           C  \\nATOM    153  CD  GLN A  20      23.540  13.098  -9.869  1.00  0.47           C  \\nATOM    154  NE2 GLN A  20      24.533  12.374  -9.364  1.00  0.36           N  \\nATOM    155  OE1 GLN A  20      23.687  13.840 -10.845  1.00  0.51           O  \\nATOM    156  N   PRO A  21      18.981  16.251  -7.755  1.00  0.42           N  \\nATOM    157  CA  PRO A  21      18.885  17.711  -7.684  1.00  0.43           C  \\nATOM    158  C   PRO A  21      19.704  18.409  -8.767  1.00  0.42           C  \\nATOM    159  CB  PRO A  21      19.432  18.033  -6.291  1.00  0.39           C  \\nATOM    160  O   PRO A  21      20.926  18.245  -8.824  1.00  0.40           O  \\nATOM    161  CG  PRO A  21      19.959  16.733  -5.777  1.00  0.40           C  \\nATOM    162  CD  PRO A  21      19.642  15.660  -6.780  1.00  0.46           C  \\nATOM    163  N   GLY A  22      19.171  18.688  -9.986  1.00  0.38           N  \\nATOM    164  CA  GLY A  22      19.829  19.689 -10.810  1.00  0.43           C  \\nATOM    165  C   GLY A  22      19.621  19.469 -12.297  1.00  0.38           C  \\nATOM    166  O   GLY A  22      20.301  20.084 -13.122  1.00  0.36           O  \\nATOM    167  N   GLN A  23      18.497  18.863 -12.822  1.00  0.36           N  \\nATOM    168  CA  GLN A  23      18.354  18.906 -14.273  1.00  0.41           C  \\nATOM    169  C   GLN A  23      16.922  19.245 -14.675  1.00  0.37           C  \\nATOM    170  CB  GLN A  23      18.773  17.573 -14.894  1.00  0.33           C  \\nATOM    171  O   GLN A  23      15.974  18.605 -14.214  1.00  0.35           O  \\nATOM    172  CG  GLN A  23      20.275  17.433 -15.098  1.00  0.34           C  \\nATOM    173  CD  GLN A  23      20.642  16.245 -15.966  1.00  0.41           C  \\nATOM    174  NE2 GLN A  23      21.926  16.116 -16.284  1.00  0.34           N  \\nATOM    175  OE1 GLN A  23      19.779  15.447 -16.347  1.00  0.47           O  \\nATOM    176  N   GLY A  24      16.620  20.606 -14.994  1.00  0.42           N  \\nATOM    177  CA  GLY A  24      15.883  21.666 -15.664  1.00  0.45           C  \\nATOM    178  C   GLY A  24      14.860  21.146 -16.656  1.00  0.42           C  \\nATOM    179  O   GLY A  24      15.178  20.308 -17.502  1.00  0.40           O  \\nATOM    180  N   TYR A  25      13.531  20.962 -16.207  1.00  0.39           N  \\nATOM    181  CA  TYR A  25      12.142  20.936 -16.651  1.00  0.43           C  \\nATOM    182  C   TYR A  25      12.012  21.483 -18.068  1.00  0.39           C  \\nATOM    183  CB  TYR A  25      11.256  21.742 -15.697  1.00  0.37           C  \\nATOM    184  O   TYR A  25      12.233  22.673 -18.303  1.00  0.38           O  \\nATOM    185  CG  TYR A  25      11.113  21.120 -14.329  1.00  0.38           C  \\nATOM    186  CD1 TYR A  25      10.191  20.101 -14.099  1.00  0.34           C  \\nATOM    187  CD2 TYR A  25      11.899  21.549 -13.265  1.00  0.36           C  \\nATOM    188  CE1 TYR A  25      10.054  19.526 -12.840  1.00  0.34           C  \\nATOM    189  CE2 TYR A  25      11.771  20.980 -12.002  1.00  0.40           C  \\nATOM    190  OH  TYR A  25      10.716  19.405 -10.551  1.00  0.24           O  \\nATOM    191  CZ  TYR A  25      10.847  19.971 -11.799  1.00  0.31           C  \\nATOM    192  N   SER A  26      12.091  20.682 -19.070  1.00  0.42           N  \\nATOM    193  CA  SER A  26      11.567  21.030 -20.386  1.00  0.47           C  \\nATOM    194  C   SER A  26      10.193  20.410 -20.617  1.00  0.42           C  \\nATOM    195  CB  SER A  26      12.531  20.577 -21.484  1.00  0.39           C  \\nATOM    196  O   SER A  26      10.010  19.206 -20.423  1.00  0.41           O  \\nATOM    197  OG  SER A  26      13.418  19.585 -20.997  1.00  0.39           O  \\nATOM    198  N   GLN A  27       9.006  21.106 -20.239  1.00  0.44           N  \\nATOM    199  CA  GLN A  27       7.612  21.172 -20.664  1.00  0.47           C  \\nATOM    200  C   GLN A  27       7.401  20.425 -21.977  1.00  0.44           C  \\nATOM    201  CB  GLN A  27       7.164  22.627 -20.810  1.00  0.41           C  \\nATOM    202  O   GLN A  27       7.843  20.880 -23.035  1.00  0.43           O  \\nATOM    203  CG  GLN A  27       6.695  23.260 -19.507  1.00  0.42           C  \\nATOM    204  CD  GLN A  27       6.251  24.700 -19.680  1.00  0.47           C  \\nATOM    205  NE2 GLN A  27       5.793  25.314 -18.594  1.00  0.38           N  \\nATOM    206  OE1 GLN A  27       6.320  25.257 -20.780  1.00  0.52           O  \\nATOM    207  N   GLN A  28       7.378  19.093 -21.847  1.00  0.41           N  \\nATOM    208  CA  GLN A  28       6.654  18.400 -22.908  1.00  0.45           C  \\nATOM    209  C   GLN A  28       5.244  18.027 -22.460  1.00  0.41           C  \\nATOM    210  CB  GLN A  28       7.414  17.147 -23.347  1.00  0.39           C  \\nATOM    211  O   GLN A  28       5.064  17.404 -21.412  1.00  0.41           O  \\nATOM    212  CG  GLN A  28       8.127  17.297 -24.684  1.00  0.39           C  \\nATOM    213  CD  GLN A  28       8.934  16.069 -25.060  1.00  0.44           C  \\nATOM    214  NE2 GLN A  28       9.621  16.135 -26.195  1.00  0.37           N  \\nATOM    215  OE1 GLN A  28       8.941  15.069 -24.335  1.00  0.49           O  \\nATOM    216  N   SER A  29       4.224  18.899 -22.791  1.00  0.36           N  \\nATOM    217  CA  SER A  29       2.785  18.722 -22.950  1.00  0.40           C  \\nATOM    218  C   SER A  29       2.405  17.245 -22.945  1.00  0.36           C  \\nATOM    219  CB  SER A  29       2.304  19.376 -24.246  1.00  0.35           C  \\nATOM    220  O   SER A  29       2.845  16.483 -23.808  1.00  0.36           O  \\nATOM    221  OG  SER A  29       2.029  18.395 -25.232  1.00  0.37           O  \\nATOM    222  N   SER A  30       2.459  16.548 -21.905  1.00  0.38           N  \\nATOM    223  CA  SER A  30       1.714  15.318 -21.654  1.00  0.40           C  \\nATOM    224  C   SER A  30       0.264  15.448 -22.107  1.00  0.39           C  \\nATOM    225  CB  SER A  30       1.763  14.955 -20.170  1.00  0.37           C  \\nATOM    226  O   SER A  30      -0.488  16.265 -21.572  1.00  0.38           O  \\nATOM    227  OG  SER A  30       1.317  16.039 -19.372  1.00  0.36           O  \\nATOM    228  N   GLN A  31       0.017  15.512 -23.378  1.00  0.40           N  \\nATOM    229  CA  GLN A  31      -1.310  15.085 -23.806  1.00  0.44           C  \\nATOM    230  C   GLN A  31      -1.896  14.058 -22.842  1.00  0.41           C  \\nATOM    231  CB  GLN A  31      -1.258  14.506 -25.221  1.00  0.39           C  \\nATOM    232  O   GLN A  31      -1.203  13.130 -22.418  1.00  0.40           O  \\nATOM    233  CG  GLN A  31      -1.060  15.554 -26.308  1.00  0.40           C  \\nATOM    234  CD  GLN A  31      -1.790  15.211 -27.593  1.00  0.46           C  \\nATOM    235  NE2 GLN A  31      -2.222  16.236 -28.321  1.00  0.37           N  \\nATOM    236  OE1 GLN A  31      -1.962  14.036 -27.930  1.00  0.51           O  \\nATOM    237  N   PRO A  32      -2.691  14.342 -21.929  1.00  0.43           N  \\nATOM    238  CA  PRO A  32      -3.487  13.355 -21.195  1.00  0.43           C  \\nATOM    239  C   PRO A  32      -3.714  12.071 -21.989  1.00  0.43           C  \\nATOM    240  CB  PRO A  32      -4.810  14.084 -20.945  1.00  0.41           C  \\nATOM    241  O   PRO A  32      -3.975  12.125 -23.194  1.00  0.42           O  \\nATOM    242  CG  PRO A  32      -4.874  15.134 -22.008  1.00  0.40           C  \\nATOM    243  CD  PRO A  32      -3.480  15.403 -22.497  1.00  0.45           C  \\nATOM    244  N   TYR A  33      -2.993  10.900 -22.083  1.00  0.40           N  \\nATOM    245  CA  TYR A  33      -3.651   9.857 -22.862  1.00  0.43           C  \\nATOM    246  C   TYR A  33      -4.941  10.374 -23.488  1.00  0.40           C  \\nATOM    247  CB  TYR A  33      -3.948   8.637 -21.985  1.00  0.39           C  \\nATOM    248  O   TYR A  33      -5.856  10.797 -22.778  1.00  0.39           O  \\nATOM    249  CG  TYR A  33      -2.753   7.742 -21.764  1.00  0.39           C  \\nATOM    250  CD1 TYR A  33      -2.424   6.746 -22.681  1.00  0.39           C  \\nATOM    251  CD2 TYR A  33      -1.950   7.890 -20.638  1.00  0.39           C  \\nATOM    252  CE1 TYR A  33      -1.325   5.918 -22.480  1.00  0.38           C  \\nATOM    253  CE2 TYR A  33      -0.848   7.068 -20.427  1.00  0.41           C  \\nATOM    254  OH  TYR A  33       0.546   5.270 -21.149  1.00  0.23           O  \\nATOM    255  CZ  TYR A  33      -0.544   6.087 -21.352  1.00  0.36           C  \\nATOM    256  N   GLY A  34      -4.938  11.374 -24.324  1.00  0.35           N  \\nATOM    257  CA  GLY A  34      -6.107  11.621 -25.153  1.00  0.35           C  \\nATOM    258  C   GLY A  34      -7.363  10.953 -24.627  1.00  0.35           C  \\nATOM    259  O   GLY A  34      -7.389   9.737 -24.427  1.00  0.33           O  \\nATOM    260  N   GLN A  35      -8.036  11.223 -23.499  1.00  0.37           N  \\nATOM    261  CA  GLN A  35      -9.398  11.112 -24.012  1.00  0.39           C  \\nATOM    262  C   GLN A  35      -9.400  10.866 -25.518  1.00  0.38           C  \\nATOM    263  CB  GLN A  35     -10.199  12.373 -23.685  1.00  0.35           C  \\nATOM    264  O   GLN A  35      -9.112  11.774 -26.301  1.00  0.35           O  \\nATOM    265  CG  GLN A  35     -10.985  12.282 -22.384  1.00  0.35           C  \\nATOM    266  CD  GLN A  35     -11.720  13.567 -22.051  1.00  0.41           C  \\nATOM    267  NE2 GLN A  35     -12.636  13.494 -21.090  1.00  0.35           N  \\nATOM    268  OE1 GLN A  35     -11.467  14.616 -22.652  1.00  0.46           O  \\nATOM    269  N   GLN A  36      -8.705   9.918 -26.078  1.00  0.36           N  \\nATOM    270  CA  GLN A  36      -9.606   9.619 -27.186  1.00  0.38           C  \\nATOM    271  C   GLN A  36     -11.058   9.895 -26.804  1.00  0.36           C  \\nATOM    272  CB  GLN A  36      -9.447   8.164 -27.629  1.00  0.34           C  \\nATOM    273  O   GLN A  36     -11.579   9.305 -25.856  1.00  0.33           O  \\nATOM    274  CG  GLN A  36      -8.264   7.930 -28.558  1.00  0.34           C  \\nATOM    275  CD  GLN A  36      -8.219   6.516 -29.106  1.00  0.41           C  \\nATOM    276  NE2 GLN A  36      -7.229   6.238 -29.948  1.00  0.33           N  \\nATOM    277  OE1 GLN A  36      -9.065   5.680 -28.775  1.00  0.45           O  \\nATOM    278  N   SER A  37     -11.516  11.061 -26.547  1.00  0.33           N  \\nATOM    279  CA  SER A  37     -12.853  11.230 -27.106  1.00  0.35           C  \\nATOM    280  C   SER A  37     -13.167  10.140 -28.126  1.00  0.33           C  \\nATOM    281  CB  SER A  37     -12.992  12.606 -27.759  1.00  0.32           C  \\nATOM    282  O   SER A  37     -12.499  10.036 -29.157  1.00  0.31           O  \\nATOM    283  OG  SER A  37     -11.781  13.335 -27.659  1.00  0.32           O  \\nATOM    284  N   TYR A  38     -13.272   8.886 -27.807  1.00  0.36           N  \\nATOM    285  CA  TYR A  38     -14.283   8.171 -28.578  1.00  0.40           C  \\nATOM    286  C   TYR A  38     -15.341   9.129 -29.112  1.00  0.36           C  \\nATOM    287  CB  TYR A  38     -14.945   7.087 -27.722  1.00  0.35           C  \\nATOM    288  O   TYR A  38     -16.297   9.463 -28.407  1.00  0.35           O  \\nATOM    289  CG  TYR A  38     -14.188   5.781 -27.704  1.00  0.36           C  \\nATOM    290  CD1 TYR A  38     -14.429   4.801 -28.665  1.00  0.36           C  \\nATOM    291  CD2 TYR A  38     -13.233   5.524 -26.727  1.00  0.37           C  \\nATOM    292  CE1 TYR A  38     -13.735   3.596 -28.652  1.00  0.37           C  \\nATOM    293  CE2 TYR A  38     -12.534   4.322 -26.704  1.00  0.40           C  \\nATOM    294  OH  TYR A  38     -12.102   2.174 -27.651  1.00  0.25           O  \\nATOM    295  CZ  TYR A  38     -12.791   3.365 -27.669  1.00  0.34           C  \\nATOM    296  N   SER A  39     -15.047  10.215 -29.530  1.00  0.32           N  \\nATOM    297  CA  SER A  39     -15.905  10.903 -30.489  1.00  0.36           C  \\nATOM    298  C   SER A  39     -15.688  10.376 -31.903  1.00  0.33           C  \\nATOM    299  CB  SER A  39     -15.649  12.410 -30.452  1.00  0.31           C  \\nATOM    300  O   SER A  39     -15.155  11.085 -32.760  1.00  0.31           O  \\nATOM    301  OG  SER A  39     -14.353  12.712 -30.939  1.00  0.31           O  \\nATOM    302  N   GLY A  40     -15.639   9.035 -31.983  1.00  0.32           N  \\nATOM    303  CA  GLY A  40     -15.785   8.437 -33.301  1.00  0.36           C  \\nATOM    304  C   GLY A  40     -16.037   6.942 -33.253  1.00  0.33           C  \\nATOM    305  O   GLY A  40     -15.735   6.225 -34.209  1.00  0.31           O  \\nATOM    306  N   TYR A  41     -16.814   6.518 -32.355  1.00  0.34           N  \\nATOM    307  CA  TYR A  41     -17.569   5.302 -32.639  1.00  0.38           C  \\nATOM    308  C   TYR A  41     -19.042   5.482 -32.293  1.00  0.34           C  \\nATOM    309  CB  TYR A  41     -16.990   4.116 -31.862  1.00  0.32           C  \\nATOM    310  O   TYR A  41     -19.411   5.500 -31.116  1.00  0.33           O  \\nATOM    311  CG  TYR A  41     -15.855   3.420 -32.573  1.00  0.34           C  \\nATOM    312  CD1 TYR A  41     -16.102   2.404 -33.493  1.00  0.34           C  \\nATOM    313  CD2 TYR A  41     -14.534   3.776 -32.325  1.00  0.35           C  \\nATOM    314  CE1 TYR A  41     -15.059   1.758 -34.149  1.00  0.33           C  \\nATOM    315  CE2 TYR A  41     -13.483   3.138 -32.975  1.00  0.37           C  \\nATOM    316  OH  TYR A  41     -12.718   1.496 -34.530  1.00  0.16           O  \\nATOM    317  CZ  TYR A  41     -13.755   2.132 -33.884  1.00  0.31           C  \\nATOM    318  N   SER A  42     -19.813   6.378 -32.813  1.00  0.37           N  \\nATOM    319  CA  SER A  42     -21.127   6.042 -33.349  1.00  0.40           C  \\nATOM    320  C   SER A  42     -21.089   4.728 -34.122  1.00  0.37           C  \\nATOM    321  CB  SER A  42     -21.640   7.162 -34.256  1.00  0.36           C  \\nATOM    322  O   SER A  42     -20.632   4.688 -35.266  1.00  0.36           O  \\nATOM    323  OG  SER A  42     -22.856   7.694 -33.759  1.00  0.38           O  \\nATOM    324  N   GLN A  43     -20.645   3.684 -33.614  1.00  0.38           N  \\nATOM    325  CA  GLN A  43     -21.296   2.541 -34.247  1.00  0.42           C  \\nATOM    326  C   GLN A  43     -22.664   2.274 -33.627  1.00  0.38           C  \\nATOM    327  CB  GLN A  43     -20.416   1.295 -34.138  1.00  0.36           C  \\nATOM    328  O   GLN A  43     -22.770   2.050 -32.419  1.00  0.37           O  \\nATOM    329  CG  GLN A  43     -19.272   1.259 -35.142  1.00  0.37           C  \\nATOM    330  CD  GLN A  43     -18.445  -0.009 -35.045  1.00  0.44           C  \\nATOM    331  NE2 GLN A  43     -17.273  -0.002 -35.671  1.00  0.37           N  \\nATOM    332  OE1 GLN A  43     -18.855  -0.986 -34.410  1.00  0.48           O  \\nATOM    333  N   SER A  44     -23.721   2.852 -34.077  1.00  0.39           N  \\nATOM    334  CA  SER A  44     -25.005   2.204 -34.322  1.00  0.41           C  \\nATOM    335  C   SER A  44     -24.856   0.688 -34.384  1.00  0.39           C  \\nATOM    336  CB  SER A  44     -25.625   2.718 -35.622  1.00  0.37           C  \\nATOM    337  O   SER A  44     -24.339   0.150 -35.366  1.00  0.38           O  \\nATOM    338  OG  SER A  44     -24.814   2.380 -36.734  1.00  0.41           O  \\nATOM    339  N   THR A  45     -24.486   0.067 -33.382  1.00  0.41           N  \\nATOM    340  CA  THR A  45     -24.803  -1.356 -33.322  1.00  0.44           C  \\nATOM    341  C   THR A  45     -26.291  -1.568 -33.062  1.00  0.41           C  \\nATOM    342  CB  THR A  45     -23.982  -2.065 -32.228  1.00  0.40           C  \\nATOM    343  O   THR A  45     -26.791  -1.247 -31.982  1.00  0.41           O  \\nATOM    344  CG2 THR A  45     -22.604  -2.463 -32.746  1.00  0.39           C  \\nATOM    345  OG1 THR A  45     -23.824  -1.183 -31.110  1.00  0.35           O  \\nATOM    346  N   ASP A  46     -27.134  -1.407 -34.077  1.00  0.41           N  \\nATOM    347  CA  ASP A  46     -28.361  -2.176 -34.262  1.00  0.44           C  \\nATOM    348  C   ASP A  46     -28.161  -3.635 -33.857  1.00  0.41           C  \\nATOM    349  CB  ASP A  46     -28.833  -2.094 -35.715  1.00  0.39           C  \\nATOM    350  O   ASP A  46     -27.581  -4.420 -34.610  1.00  0.40           O  \\nATOM    351  CG  ASP A  46     -29.980  -1.118 -35.912  1.00  0.41           C  \\nATOM    352  OD1 ASP A  46     -30.658  -0.766 -34.923  1.00  0.43           O  \\nATOM    353  OD2 ASP A  46     -30.209  -0.699 -37.067  1.00  0.47           O  \\nATOM    354  N   THR A  47     -27.969  -3.955 -32.600  1.00  0.40           N  \\nATOM    355  CA  THR A  47     -28.007  -5.283 -31.998  1.00  0.42           C  \\nATOM    356  C   THR A  47     -29.436  -5.669 -31.629  1.00  0.40           C  \\nATOM    357  CB  THR A  47     -27.113  -5.353 -30.746  1.00  0.38           C  \\nATOM    358  O   THR A  47     -29.932  -5.292 -30.565  1.00  0.39           O  \\nATOM    359  CG2 THR A  47     -25.930  -6.289 -30.966  1.00  0.37           C  \\nATOM    360  OG1 THR A  47     -26.621  -4.042 -30.442  1.00  0.35           O  \\nATOM    361  N   SER A  48     -30.281  -5.701 -32.473  1.00  0.35           N  \\nATOM    362  CA  SER A  48     -31.546  -6.427 -32.509  1.00  0.39           C  \\nATOM    363  C   SER A  48     -31.317  -7.934 -32.566  1.00  0.36           C  \\nATOM    364  CB  SER A  48     -32.385  -5.984 -33.708  1.00  0.34           C  \\nATOM    365  O   SER A  48     -30.653  -8.431 -33.478  1.00  0.34           O  \\nATOM    366  OG  SER A  48     -31.628  -6.055 -34.905  1.00  0.37           O  \\nATOM    367  N   GLY A  49     -30.859  -8.641 -31.382  1.00  0.32           N  \\nATOM    368  CA  GLY A  49     -31.309 -10.024 -31.365  1.00  0.38           C  \\nATOM    369  C   GLY A  49     -30.167 -11.022 -31.338  1.00  0.32           C  \\nATOM    370  O   GLY A  49     -29.440 -11.167 -32.323  1.00  0.32           O  \\nATOM    371  N   TYR A  50     -29.429 -11.176 -30.168  1.00  0.38           N  \\nATOM    372  CA  TYR A  50     -28.889 -12.491 -29.840  1.00  0.43           C  \\nATOM    373  C   TYR A  50     -28.483 -12.564 -28.373  1.00  0.39           C  \\nATOM    374  CB  TYR A  50     -27.685 -12.815 -30.731  1.00  0.36           C  \\nATOM    375  O   TYR A  50     -27.722 -11.722 -27.890  1.00  0.38           O  \\nATOM    376  CG  TYR A  50     -28.057 -13.150 -32.155  1.00  0.37           C  \\nATOM    377  CD1 TYR A  50     -28.472 -14.433 -32.504  1.00  0.35           C  \\nATOM    378  CD2 TYR A  50     -27.994 -12.184 -33.154  1.00  0.36           C  \\nATOM    379  CE1 TYR A  50     -28.813 -14.746 -33.816  1.00  0.34           C  \\nATOM    380  CE2 TYR A  50     -28.334 -12.486 -34.469  1.00  0.38           C  \\nATOM    381  OH  TYR A  50     -29.079 -14.073 -36.089  1.00  0.23           O  \\nATOM    382  CZ  TYR A  50     -28.742 -13.768 -34.789  1.00  0.32           C  \\nATOM    383  N   GLY A  51     -29.357 -13.189 -27.555  1.00  0.37           N  \\nATOM    384  CA  GLY A  51     -29.601 -14.169 -26.509  1.00  0.40           C  \\nATOM    385  C   GLY A  51     -28.562 -14.135 -25.405  1.00  0.37           C  \\nATOM    386  O   GLY A  51     -27.359 -14.155 -25.677  1.00  0.37           O  \\nATOM    387  N   GLN A  52     -28.796 -13.404 -24.311  1.00  0.38           N  \\nATOM    388  CA  GLN A  52     -28.670 -13.396 -22.857  1.00  0.42           C  \\nATOM    389  C   GLN A  52     -27.678 -14.456 -22.385  1.00  0.39           C  \\nATOM    390  CB  GLN A  52     -30.032 -13.622 -22.199  1.00  0.36           C  \\nATOM    391  O   GLN A  52     -27.956 -15.654 -22.467  1.00  0.38           O  \\nATOM    392  CG  GLN A  52     -30.643 -12.362 -21.601  1.00  0.38           C  \\nATOM    393  CD  GLN A  52     -32.033 -12.593 -21.039  1.00  0.46           C  \\nATOM    394  NE2 GLN A  52     -32.666 -11.525 -20.565  1.00  0.37           N  \\nATOM    395  OE1 GLN A  52     -32.535 -13.721 -21.033  1.00  0.50           O  \\nATOM    396  N   SER A  53     -26.358 -14.115 -22.163  1.00  0.41           N  \\nATOM    397  CA  SER A  53     -25.495 -14.871 -21.262  1.00  0.44           C  \\nATOM    398  C   SER A  53     -25.186 -14.077 -19.997  1.00  0.41           C  \\nATOM    399  CB  SER A  53     -24.192 -15.255 -21.964  1.00  0.39           C  \\nATOM    400  O   SER A  53     -24.719 -12.939 -20.071  1.00  0.41           O  \\nATOM    401  OG  SER A  53     -23.403 -14.107 -22.223  1.00  0.39           O  \\nATOM    402  N   SER A  54     -25.875 -14.244 -18.897  1.00  0.38           N  \\nATOM    403  CA  SER A  54     -25.772 -14.091 -17.450  1.00  0.41           C  \\nATOM    404  C   SER A  54     -24.332 -14.261 -16.977  1.00  0.38           C  \\nATOM    405  CB  SER A  54     -26.674 -15.101 -16.738  1.00  0.37           C  \\nATOM    406  O   SER A  54     -23.789 -15.367 -17.009  1.00  0.38           O  \\nATOM    407  OG  SER A  54     -26.405 -16.418 -17.187  1.00  0.37           O  \\nATOM    408  N   TYR A  55     -23.457 -13.297 -16.959  1.00  0.42           N  \\nATOM    409  CA  TYR A  55     -22.185 -13.288 -16.245  1.00  0.45           C  \\nATOM    410  C   TYR A  55     -22.298 -12.507 -14.941  1.00  0.42           C  \\nATOM    411  CB  TYR A  55     -21.081 -12.688 -17.120  1.00  0.40           C  \\nATOM    412  O   TYR A  55     -22.583 -11.307 -14.952  1.00  0.42           O  \\nATOM    413  CG  TYR A  55     -20.553 -13.637 -18.168  1.00  0.40           C  \\nATOM    414  CD1 TYR A  55     -19.530 -14.535 -17.870  1.00  0.39           C  \\nATOM    415  CD2 TYR A  55     -21.076 -13.639 -19.456  1.00  0.40           C  \\nATOM    416  CE1 TYR A  55     -19.039 -15.410 -18.833  1.00  0.38           C  \\nATOM    417  CE2 TYR A  55     -20.592 -14.510 -20.427  1.00  0.42           C  \\nATOM    418  OH  TYR A  55     -19.094 -16.255 -21.064  1.00  0.28           O  \\nATOM    419  CZ  TYR A  55     -19.576 -15.391 -20.107  1.00  0.36           C  \\nATOM    420  N   SER A  56     -22.483 -13.153 -13.837  1.00  0.38           N  \\nATOM    421  CA  SER A  56     -22.520 -13.048 -12.382  1.00  0.41           C  \\nATOM    422  C   SER A  56     -21.132 -12.778 -11.811  1.00  0.39           C  \\nATOM    423  CB  SER A  56     -23.094 -14.325 -11.766  1.00  0.37           C  \\nATOM    424  O   SER A  56     -20.167 -13.460 -12.163  1.00  0.38           O  \\nATOM    425  OG  SER A  56     -22.305 -15.450 -12.114  1.00  0.37           O  \\nATOM    426  N   SER A  57     -20.639 -11.408 -11.702  1.00  0.35           N  \\nATOM    427  CA  SER A  57     -20.112 -10.768 -10.501  1.00  0.39           C  \\nATOM    428  C   SER A  57     -18.608 -10.987 -10.371  1.00  0.35           C  \\nATOM    429  CB  SER A  57     -20.822 -11.300  -9.255  1.00  0.34           C  \\nATOM    430  O   SER A  57     -18.134 -12.123 -10.427  1.00  0.35           O  \\nATOM    431  OG  SER A  57     -20.643 -12.700  -9.134  1.00  0.35           O  \\nATOM    432  N   TYR A  58     -17.799 -10.037 -11.060  1.00  0.39           N  \\nATOM    433  CA  TYR A  58     -16.605  -9.478 -10.437  1.00  0.42           C  \\nATOM    434  C   TYR A  58     -15.578  -9.077 -11.488  1.00  0.39           C  \\nATOM    435  CB  TYR A  58     -15.987 -10.484  -9.460  1.00  0.37           C  \\nATOM    436  O   TYR A  58     -15.122  -9.914 -12.271  1.00  0.39           O  \\nATOM    437  CG  TYR A  58     -16.666 -10.517  -8.113  1.00  0.39           C  \\nATOM    438  CD1 TYR A  58     -16.320  -9.610  -7.114  1.00  0.38           C  \\nATOM    439  CD2 TYR A  58     -17.653 -11.456  -7.834  1.00  0.40           C  \\nATOM    440  CE1 TYR A  58     -16.941  -9.638  -5.870  1.00  0.37           C  \\nATOM    441  CE2 TYR A  58     -18.281 -11.493  -6.594  1.00  0.41           C  \\nATOM    442  OH  TYR A  58     -18.538 -10.614  -4.390  1.00  0.26           O  \\nATOM    443  CZ  TYR A  58     -17.919 -10.581  -5.620  1.00  0.36           C  \\nATOM    444  N   GLY A  59     -15.759  -7.779 -11.960  1.00  0.37           N  \\nATOM    445  CA  GLY A  59     -14.685  -6.810 -12.109  1.00  0.40           C  \\nATOM    446  C   GLY A  59     -14.447  -6.400 -13.549  1.00  0.37           C  \\nATOM    447  O   GLY A  59     -14.358  -7.251 -14.436  1.00  0.36           O  \\nATOM    448  N   GLN A  60     -15.173  -5.375 -13.903  1.00  0.42           N  \\nATOM    449  CA  GLN A  60     -15.287  -4.356 -14.941  1.00  0.44           C  \\nATOM    450  C   GLN A  60     -15.349  -4.988 -16.328  1.00  0.42           C  \\nATOM    451  CB  GLN A  60     -14.115  -3.376 -14.862  1.00  0.40           C  \\nATOM    452  O   GLN A  60     -14.337  -5.466 -16.844  1.00  0.41           O  \\nATOM    453  CG  GLN A  60     -14.382  -2.167 -13.975  1.00  0.40           C  \\nATOM    454  CD  GLN A  60     -13.200  -1.218 -13.905  1.00  0.47           C  \\nATOM    455  NE2 GLN A  60     -13.413  -0.049 -13.311  1.00  0.37           N  \\nATOM    456  OE1 GLN A  60     -12.105  -1.534 -14.380  1.00  0.49           O  \\nATOM    457  N   SER A  61     -16.395  -5.753 -16.512  1.00  0.36           N  \\nATOM    458  CA  SER A  61     -17.000  -5.933 -17.827  1.00  0.39           C  \\nATOM    459  C   SER A  61     -17.801  -4.702 -18.240  1.00  0.37           C  \\nATOM    460  CB  SER A  61     -17.904  -7.166 -17.837  1.00  0.35           C  \\nATOM    461  O   SER A  61     -18.745  -4.310 -17.552  1.00  0.35           O  \\nATOM    462  OG  SER A  61     -18.072  -7.656 -19.157  1.00  0.34           O  \\nATOM    463  N   GLN A  62     -17.228  -3.737 -18.684  1.00  0.40           N  \\nATOM    464  CA  GLN A  62     -18.064  -3.009 -19.633  1.00  0.42           C  \\nATOM    465  C   GLN A  62     -18.938  -3.964 -20.441  1.00  0.40           C  \\nATOM    466  CB  GLN A  62     -17.201  -2.165 -20.572  1.00  0.39           C  \\nATOM    467  O   GLN A  62     -18.449  -4.649 -21.342  1.00  0.39           O  \\nATOM    468  CG  GLN A  62     -16.517  -0.988 -19.888  1.00  0.39           C  \\nATOM    469  CD  GLN A  62     -15.763  -0.101 -20.861  1.00  0.45           C  \\nATOM    470  NE2 GLN A  62     -14.988   0.837 -20.329  1.00  0.37           N  \\nATOM    471  OE1 GLN A  62     -15.876  -0.258 -22.081  1.00  0.49           O  \\nATOM    472  N   ASN A  63     -19.702  -4.757 -19.760  1.00  0.33           N  \\nATOM    473  CA  ASN A  63     -20.934  -5.261 -20.357  1.00  0.38           C  \\nATOM    474  C   ASN A  63     -21.987  -4.163 -20.483  1.00  0.34           C  \\nATOM    475  CB  ASN A  63     -21.485  -6.434 -19.543  1.00  0.32           C  \\nATOM    476  O   ASN A  63     -22.503  -3.675 -19.476  1.00  0.32           O  \\nATOM    477  CG  ASN A  63     -22.238  -7.435 -20.398  1.00  0.33           C  \\nATOM    478  ND2 ASN A  63     -22.824  -8.439 -19.756  1.00  0.24           N  \\nATOM    479  OD1 ASN A  63     -22.293  -7.307 -21.623  1.00  0.35           O  \\nATOM    480  N   THR A  64     -21.870  -3.357 -21.514  1.00  0.36           N  \\nATOM    481  CA  THR A  64     -22.938  -2.664 -22.227  1.00  0.38           C  \\nATOM    482  C   THR A  64     -23.978  -3.655 -22.740  1.00  0.36           C  \\nATOM    483  CB  THR A  64     -22.382  -1.843 -23.405  1.00  0.34           C  \\nATOM    484  O   THR A  64     -23.775  -4.297 -23.773  1.00  0.35           O  \\nATOM    485  CG2 THR A  64     -22.124  -0.397 -22.995  1.00  0.34           C  \\nATOM    486  OG1 THR A  64     -21.152  -2.429 -23.849  1.00  0.32           O  \\nATOM    487  N   GLY A  65     -24.482  -4.530 -21.794  1.00  0.35           N  \\nATOM    488  CA  GLY A  65     -25.790  -5.040 -22.173  1.00  0.37           C  \\nATOM    489  C   GLY A  65     -26.877  -4.698 -21.172  1.00  0.35           C  \\nATOM    490  O   GLY A  65     -26.783  -5.061 -19.997  1.00  0.34           O  \\nATOM    491  N   TYR A  66     -27.485  -3.564 -21.281  1.00  0.37           N  \\nATOM    492  CA  TYR A  66     -28.878  -3.200 -21.052  1.00  0.42           C  \\nATOM    493  C   TYR A  66     -29.819  -4.251 -21.628  1.00  0.38           C  \\nATOM    494  CB  TYR A  66     -29.185  -1.831 -21.667  1.00  0.36           C  \\nATOM    495  O   TYR A  66     -29.877  -4.441 -22.845  1.00  0.37           O  \\nATOM    496  CG  TYR A  66     -28.496  -0.684 -20.970  1.00  0.39           C  \\nATOM    497  CD1 TYR A  66     -29.014  -0.145 -19.795  1.00  0.37           C  \\nATOM    498  CD2 TYR A  66     -27.326  -0.136 -21.484  1.00  0.37           C  \\nATOM    499  CE1 TYR A  66     -28.384   0.914 -19.149  1.00  0.33           C  \\nATOM    500  CE2 TYR A  66     -26.687   0.923 -20.847  1.00  0.39           C  \\nATOM    501  OH  TYR A  66     -26.594   2.488 -19.047  1.00  0.21           O  \\nATOM    502  CZ  TYR A  66     -27.222   1.440 -19.682  1.00  0.32           C  \\nATOM    503  N   GLY A  67     -29.968  -5.309 -20.863  1.00  0.36           N  \\nATOM    504  CA  GLY A  67     -31.340  -5.791 -20.868  1.00  0.39           C  \\nATOM    505  C   GLY A  67     -31.754  -6.427 -19.555  1.00  0.37           C  \\nATOM    506  O   GLY A  67     -31.019  -7.243 -18.995  1.00  0.36           O  \\nATOM    507  N   THR A  68     -32.412  -5.583 -18.733  1.00  0.38           N  \\nATOM    508  CA  THR A  68     -33.538  -5.956 -17.883  1.00  0.43           C  \\nATOM    509  C   THR A  68     -33.852  -7.443 -18.024  1.00  0.38           C  \\nATOM    510  CB  THR A  68     -34.792  -5.130 -18.222  1.00  0.37           C  \\nATOM    511  O   THR A  68     -34.207  -7.908 -19.110  1.00  0.38           O  \\nATOM    512  CG2 THR A  68     -35.045  -4.057 -17.168  1.00  0.38           C  \\nATOM    513  OG1 THR A  68     -34.612  -4.499 -19.496  1.00  0.35           O  \\nATOM    514  N   GLN A  69     -33.106  -8.205 -17.252  1.00  0.37           N  \\nATOM    515  CA  GLN A  69     -34.019  -9.198 -16.697  1.00  0.42           C  \\nATOM    516  C   GLN A  69     -33.276 -10.197 -15.814  1.00  0.38           C  \\nATOM    517  CB  GLN A  69     -34.757  -9.935 -17.816  1.00  0.35           C  \\nATOM    518  O   GLN A  69     -32.248 -10.745 -16.218  1.00  0.37           O  \\nATOM    519  CG  GLN A  69     -36.011  -9.221 -18.300  1.00  0.37           C  \\nATOM    520  CD  GLN A  69     -36.754  -9.999 -19.370  1.00  0.45           C  \\nATOM    521  NE2 GLN A  69     -37.803  -9.398 -19.920  1.00  0.35           N  \\nATOM    522  OE1 GLN A  69     -36.388 -11.132 -19.699  1.00  0.49           O  \\nATOM    523  N   SER A  70     -33.538 -10.031 -14.518  1.00  0.39           N  \\nATOM    524  CA  SER A  70     -34.043 -10.866 -13.433  1.00  0.41           C  \\nATOM    525  C   SER A  70     -32.951 -11.777 -12.882  1.00  0.39           C  \\nATOM    526  CB  SER A  70     -35.227 -11.708 -13.911  1.00  0.37           C  \\nATOM    527  O   SER A  70     -32.268 -12.466 -13.643  1.00  0.38           O  \\nATOM    528  OG  SER A  70     -34.899 -12.410 -15.098  1.00  0.41           O  \\nATOM    529  N   THR A  71     -32.210 -11.420 -11.828  1.00  0.41           N  \\nATOM    530  CA  THR A  71     -31.822 -12.093 -10.593  1.00  0.42           C  \\nATOM    531  C   THR A  71     -31.884 -13.609 -10.761  1.00  0.41           C  \\nATOM    532  CB  THR A  71     -32.721 -11.667  -9.418  1.00  0.39           C  \\nATOM    533  O   THR A  71     -32.953 -14.168 -11.013  1.00  0.40           O  \\nATOM    534  CG2 THR A  71     -31.907 -10.993  -8.319  1.00  0.38           C  \\nATOM    535  OG1 THR A  71     -33.715 -10.750  -9.893  1.00  0.39           O  \\nATOM    536  N   PRO A  72     -30.739 -14.298 -11.001  1.00  0.45           N  \\nATOM    537  CA  PRO A  72     -30.875 -15.678 -10.530  1.00  0.46           C  \\nATOM    538  C   PRO A  72     -30.954 -15.777  -9.008  1.00  0.45           C  \\nATOM    539  CB  PRO A  72     -29.607 -16.356 -11.055  1.00  0.42           C  \\nATOM    540  O   PRO A  72     -30.071 -15.278  -8.306  1.00  0.44           O  \\nATOM    541  CG  PRO A  72     -28.778 -15.242 -11.608  1.00  0.42           C  \\nATOM    542  CD  PRO A  72     -29.574 -13.970 -11.526  1.00  0.48           C  \\nATOM    543  N   GLN A  73     -31.981 -15.903  -8.412  1.00  0.38           N  \\nATOM    544  CA  GLN A  73     -32.495 -16.788  -7.371  1.00  0.42           C  \\nATOM    545  C   GLN A  73     -31.833 -18.161  -7.440  1.00  0.38           C  \\nATOM    546  CB  GLN A  73     -34.013 -16.930  -7.488  1.00  0.36           C  \\nATOM    547  O   GLN A  73     -31.785 -18.780  -8.505  1.00  0.37           O  \\nATOM    548  CG  GLN A  73     -34.782 -16.260  -6.358  1.00  0.38           C  \\nATOM    549  CD  GLN A  73     -36.285 -16.403  -6.505  1.00  0.47           C  \\nATOM    550  NE2 GLN A  73     -37.018 -16.086  -5.443  1.00  0.37           N  \\nATOM    551  OE1 GLN A  73     -36.784 -16.795  -7.565  1.00  0.51           O  \\nATOM    552  N   GLY A  74     -30.768 -18.458  -6.498  1.00  0.39           N  \\nATOM    553  CA  GLY A  74     -30.801 -19.704  -5.749  1.00  0.42           C  \\nATOM    554  C   GLY A  74     -29.871 -20.763  -6.311  1.00  0.39           C  \\nATOM    555  O   GLY A  74     -30.138 -21.332  -7.371  1.00  0.38           O  \\nATOM    556  N   TYR A  75     -28.452 -20.712  -5.891  1.00  0.40           N  \\nATOM    557  CA  TYR A  75     -27.901 -22.002  -5.493  1.00  0.45           C  \\nATOM    558  C   TYR A  75     -27.187 -21.900  -4.150  1.00  0.40           C  \\nATOM    559  CB  TYR A  75     -26.935 -22.527  -6.559  1.00  0.37           C  \\nATOM    560  O   TYR A  75     -26.262 -21.100  -3.992  1.00  0.39           O  \\nATOM    561  CG  TYR A  75     -27.624 -23.094  -7.776  1.00  0.39           C  \\nATOM    562  CD1 TYR A  75     -28.066 -24.414  -7.799  1.00  0.36           C  \\nATOM    563  CD2 TYR A  75     -27.834 -22.310  -8.906  1.00  0.38           C  \\nATOM    564  CE1 TYR A  75     -28.700 -24.942  -8.919  1.00  0.36           C  \\nATOM    565  CE2 TYR A  75     -28.467 -22.827 -10.032  1.00  0.41           C  \\nATOM    566  OH  TYR A  75     -29.523 -24.658 -11.141  1.00  0.25           O  \\nATOM    567  CZ  TYR A  75     -28.896 -24.141 -10.029  1.00  0.32           C  \\nATOM    568  N   GLY A  76     -27.597 -22.509  -3.113  1.00  0.37           N  \\nATOM    569  CA  GLY A  76     -27.697 -23.594  -2.150  1.00  0.41           C  \\nATOM    570  C   GLY A  76     -26.707 -24.714  -2.409  1.00  0.37           C  \\nATOM    571  O   GLY A  76     -26.708 -25.314  -3.486  1.00  0.36           O  \\nATOM    572  N   SER A  77     -25.384 -24.633  -1.770  1.00  0.34           N  \\nATOM    573  CA  SER A  77     -25.070 -25.844  -1.019  1.00  0.38           C  \\nATOM    574  C   SER A  77     -23.572 -25.956  -0.753  1.00  0.34           C  \\nATOM    575  CB  SER A  77     -25.556 -27.083  -1.772  1.00  0.31           C  \\nATOM    576  O   SER A  77     -22.762 -25.838  -1.675  1.00  0.33           O  \\nATOM    577  OG  SER A  77     -24.489 -27.991  -1.988  1.00  0.39           O  \\nATOM    578  N   THR A  78     -22.959 -25.823   0.550  1.00  0.32           N  \\nATOM    579  CA  THR A  78     -22.631 -26.752   1.627  1.00  0.37           C  \\nATOM    580  C   THR A  78     -21.375 -27.550   1.289  1.00  0.32           C  \\nATOM    581  CB  THR A  78     -23.798 -27.718   1.905  1.00  0.31           C  \\nATOM    582  O   THR A  78     -21.349 -28.288   0.302  1.00  0.32           O  \\nATOM    583  CG2 THR A  78     -24.751 -27.144   2.947  1.00  0.32           C  \\nATOM    584  OG1 THR A  78     -24.520 -27.949   0.689  1.00  0.33           O  \\nATOM    585  N   GLY A  79     -19.984 -27.115   1.920  1.00  0.37           N  \\nATOM    586  CA  GLY A  79     -19.199 -28.107   2.636  1.00  0.41           C  \\nATOM    587  C   GLY A  79     -17.706 -27.965   2.406  1.00  0.37           C  \\nATOM    588  O   GLY A  79     -17.252 -27.901   1.261  1.00  0.36           O  \\nATOM    589  N   GLY A  80     -16.809 -27.439   3.451  1.00  0.37           N  \\nATOM    590  CA  GLY A  80     -15.840 -27.967   4.398  1.00  0.42           C  \\nATOM    591  C   GLY A  80     -14.561 -28.447   3.738  1.00  0.37           C  \\nATOM    592  O   GLY A  80     -14.593 -29.338   2.887  1.00  0.36           O  \\nATOM    593  N   TYR A  81     -13.363 -27.407   3.707  1.00  0.41           N  \\nATOM    594  CA  TYR A  81     -12.002 -27.844   3.996  1.00  0.47           C  \\nATOM    595  C   TYR A  81     -10.997 -26.743   3.677  1.00  0.41           C  \\nATOM    596  CB  TYR A  81     -11.662 -29.108   3.200  1.00  0.38           C  \\nATOM    597  O   TYR A  81     -11.024 -26.167   2.587  1.00  0.40           O  \\nATOM    598  CG  TYR A  81     -12.268 -30.367   3.772  1.00  0.40           C  \\nATOM    599  CD1 TYR A  81     -11.636 -31.061   4.801  1.00  0.36           C  \\nATOM    600  CD2 TYR A  81     -13.471 -30.864   3.284  1.00  0.38           C  \\nATOM    601  CE1 TYR A  81     -12.190 -32.223   5.330  1.00  0.36           C  \\nATOM    602  CE2 TYR A  81     -14.034 -32.025   3.805  1.00  0.41           C  \\nATOM    603  OH  TYR A  81     -13.939 -33.845   5.346  1.00  0.24           O  \\nATOM    604  CZ  TYR A  81     -13.387 -32.696   4.827  1.00  0.31           C  \\nATOM    605  N   GLY A  82     -10.084 -26.198   4.677  1.00  0.40           N  \\nATOM    606  CA  GLY A  82      -8.871 -26.125   5.477  1.00  0.46           C  \\nATOM    607  C   GLY A  82      -7.609 -26.042   4.640  1.00  0.40           C  \\nATOM    608  O   GLY A  82      -7.309 -26.954   3.866  1.00  0.39           O  \\nATOM    609  N   SER A  83      -7.158 -24.772   4.229  1.00  0.36           N  \\nATOM    610  CA  SER A  83      -5.714 -24.615   4.090  1.00  0.43           C  \\nATOM    611  C   SER A  83      -5.360 -23.251   3.508  1.00  0.37           C  \\nATOM    612  CB  SER A  83      -5.137 -25.721   3.206  1.00  0.34           C  \\nATOM    613  O   SER A  83      -5.955 -22.819   2.518  1.00  0.36           O  \\nATOM    614  OG  SER A  83      -5.635 -25.619   1.883  1.00  0.41           O  \\nATOM    615  N   SER A  84      -4.538 -22.357   4.291  1.00  0.38           N  \\nATOM    616  CA  SER A  84      -3.291 -21.696   4.660  1.00  0.45           C  \\nATOM    617  C   SER A  84      -2.886 -20.658   3.619  1.00  0.38           C  \\nATOM    618  CB  SER A  84      -2.170 -22.722   4.832  1.00  0.36           C  \\nATOM    619  O   SER A  84      -2.900 -20.938   2.418  1.00  0.37           O  \\nATOM    620  OG  SER A  84      -2.208 -23.687   3.795  1.00  0.41           O  \\nATOM    621  N   GLN A  85      -2.859 -19.264   4.056  1.00  0.40           N  \\nATOM    622  CA  GLN A  85      -2.139 -18.013   4.269  1.00  0.47           C  \\nATOM    623  C   GLN A  85      -1.433 -17.560   2.995  1.00  0.40           C  \\nATOM    624  CB  GLN A  85      -1.125 -18.163   5.405  1.00  0.37           C  \\nATOM    625  O   GLN A  85      -0.492 -18.210   2.535  1.00  0.39           O  \\nATOM    626  CG  GLN A  85      -1.694 -17.849   6.782  1.00  0.39           C  \\nATOM    627  CD  GLN A  85      -0.665 -17.989   7.888  1.00  0.48           C  \\nATOM    628  NE2 GLN A  85      -1.075 -17.705   9.120  1.00  0.38           N  \\nATOM    629  OE1 GLN A  85       0.489 -18.350   7.638  1.00  0.53           O  \\nATOM    630  N   SER A  86      -2.162 -16.552   2.220  1.00  0.43           N  \\nATOM    631  CA  SER A  86      -1.415 -15.487   1.559  1.00  0.48           C  \\nATOM    632  C   SER A  86      -2.185 -14.170   1.590  1.00  0.43           C  \\nATOM    633  CB  SER A  86      -1.103 -15.870   0.112  1.00  0.40           C  \\nATOM    634  O   SER A  86      -3.350 -14.117   1.192  1.00  0.42           O  \\nATOM    635  OG  SER A  86      -2.297 -16.095  -0.618  1.00  0.42           O  \\nATOM    636  N   SER A  87      -1.788 -13.187   2.464  1.00  0.43           N  \\nATOM    637  CA  SER A  87      -1.868 -11.743   2.656  1.00  0.47           C  \\nATOM    638  C   SER A  87      -1.691 -10.999   1.337  1.00  0.43           C  \\nATOM    639  CB  SER A  87      -0.814 -11.276   3.660  1.00  0.41           C  \\nATOM    640  O   SER A  87      -0.596 -10.981   0.771  1.00  0.42           O  \\nATOM    641  OG  SER A  87       0.464 -11.787   3.321  1.00  0.43           O  \\nATOM    642  N   GLN A  88      -2.660 -10.802   0.553  1.00  0.44           N  \\nATOM    643  CA  GLN A  88      -2.815  -9.916  -0.595  1.00  0.47           C  \\nATOM    644  C   GLN A  88      -3.487  -8.606  -0.192  1.00  0.45           C  \\nATOM    645  CB  GLN A  88      -3.622 -10.602  -1.699  1.00  0.41           C  \\nATOM    646  O   GLN A  88      -4.584  -8.612   0.369  1.00  0.43           O  \\nATOM    647  CG  GLN A  88      -2.769 -11.142  -2.839  1.00  0.40           C  \\nATOM    648  CD  GLN A  88      -3.584 -11.886  -3.880  1.00  0.46           C  \\nATOM    649  NE2 GLN A  88      -2.937 -12.272  -4.974  1.00  0.38           N  \\nATOM    650  OE1 GLN A  88      -4.784 -12.114  -3.701  1.00  0.50           O  \\nATOM    651  N   SER A  89      -2.814  -7.508  -0.082  1.00  0.44           N  \\nATOM    652  CA  SER A  89      -2.951  -6.058   0.005  1.00  0.48           C  \\nATOM    653  C   SER A  89      -3.941  -5.533  -1.030  1.00  0.44           C  \\nATOM    654  CB  SER A  89      -1.594  -5.380  -0.186  1.00  0.41           C  \\nATOM    655  O   SER A  89      -3.820  -5.833  -2.220  1.00  0.42           O  \\nATOM    656  OG  SER A  89      -1.079  -5.639  -1.481  1.00  0.42           O  \\nATOM    657  N   SER A  90      -5.283  -5.336  -0.752  1.00  0.35           N  \\nATOM    658  CA  SER A  90      -6.591  -4.707  -0.902  1.00  0.39           C  \\nATOM    659  C   SER A  90      -6.486  -3.385  -1.656  1.00  0.36           C  \\nATOM    660  CB  SER A  90      -7.234  -4.474   0.466  1.00  0.34           C  \\nATOM    661  O   SER A  90      -5.948  -2.407  -1.132  1.00  0.35           O  \\nATOM    662  OG  SER A  90      -6.329  -3.824   1.343  1.00  0.36           O  \\nATOM    663  N   TYR A  91      -6.244  -3.399  -3.008  1.00  0.42           N  \\nATOM    664  CA  TYR A  91      -6.712  -2.367  -3.925  1.00  0.47           C  \\nATOM    665  C   TYR A  91      -7.516  -2.976  -5.068  1.00  0.42           C  \\nATOM    666  CB  TYR A  91      -5.531  -1.569  -4.486  1.00  0.40           C  \\nATOM    667  O   TYR A  91      -7.012  -3.828  -5.804  1.00  0.41           O  \\nATOM    668  CG  TYR A  91      -4.976  -0.548  -3.522  1.00  0.41           C  \\nATOM    669  CD1 TYR A  91      -5.559   0.712  -3.401  1.00  0.41           C  \\nATOM    670  CD2 TYR A  91      -3.869  -0.840  -2.733  1.00  0.42           C  \\nATOM    671  CE1 TYR A  91      -5.050   1.657  -2.516  1.00  0.39           C  \\nATOM    672  CE2 TYR A  91      -3.353   0.098  -1.845  1.00  0.43           C  \\nATOM    673  OH  TYR A  91      -3.441   2.273  -0.866  1.00  0.26           O  \\nATOM    674  CZ  TYR A  91      -3.949   1.341  -1.743  1.00  0.36           C  \\nATOM    675  N   GLY A  92      -8.937  -3.115  -4.982  1.00  0.37           N  \\nATOM    676  CA  GLY A  92     -10.210  -3.101  -5.686  1.00  0.41           C  \\nATOM    677  C   GLY A  92     -10.478  -4.379  -6.458  1.00  0.38           C  \\nATOM    678  O   GLY A  92     -10.035  -4.525  -7.599  1.00  0.37           O  \\nATOM    679  N   GLN A  93     -10.371  -5.496  -5.841  1.00  0.40           N  \\nATOM    680  CA  GLN A  93     -11.100  -6.650  -6.355  1.00  0.42           C  \\nATOM    681  C   GLN A  93     -12.290  -6.991  -5.463  1.00  0.40           C  \\nATOM    682  CB  GLN A  93     -10.173  -7.860  -6.478  1.00  0.37           C  \\nATOM    683  O   GLN A  93     -12.126  -7.233  -4.265  1.00  0.38           O  \\nATOM    684  CG  GLN A  93      -9.669  -8.107  -7.894  1.00  0.37           C  \\nATOM    685  CD  GLN A  93      -8.668  -9.245  -7.970  1.00  0.42           C  \\nATOM    686  NE2 GLN A  93      -8.224  -9.564  -9.181  1.00  0.37           N  \\nATOM    687  OE1 GLN A  93      -8.296  -9.831  -6.949  1.00  0.47           O  \\nATOM    688  N   GLN A  94     -13.480  -6.432  -5.857  1.00  0.42           N  \\nATOM    689  CA  GLN A  94     -14.777  -7.038  -5.570  1.00  0.45           C  \\nATOM    690  C   GLN A  94     -14.728  -8.553  -5.750  1.00  0.42           C  \\nATOM    691  CB  GLN A  94     -15.860  -6.437  -6.467  1.00  0.40           C  \\nATOM    692  O   GLN A  94     -14.530  -9.045  -6.863  1.00  0.41           O  \\nATOM    693  CG  GLN A  94     -16.704  -5.371  -5.783  1.00  0.40           C  \\nATOM    694  CD  GLN A  94     -17.721  -4.740  -6.714  1.00  0.46           C  \\nATOM    695  NE2 GLN A  94     -18.441  -3.740  -6.217  1.00  0.38           N  \\nATOM    696  OE1 GLN A  94     -17.859  -5.147  -7.872  1.00  0.50           O  \\nATOM    697  N   SER A  95     -14.168  -9.240  -4.881  1.00  0.36           N  \\nATOM    698  CA  SER A  95     -14.410 -10.608  -4.437  1.00  0.39           C  \\nATOM    699  C   SER A  95     -15.901 -10.928  -4.421  1.00  0.37           C  \\nATOM    700  CB  SER A  95     -13.817 -10.835  -3.046  1.00  0.35           C  \\nATOM    701  O   SER A  95     -16.669 -10.300  -3.689  1.00  0.36           O  \\nATOM    702  OG  SER A  95     -14.786 -10.592  -2.041  1.00  0.35           O  \\nATOM    703  N   SER A  96     -16.554 -11.191  -5.535  1.00  0.40           N  \\nATOM    704  CA  SER A  96     -17.740 -12.040  -5.477  1.00  0.43           C  \\nATOM    705  C   SER A  96     -17.362 -13.504  -5.280  1.00  0.41           C  \\nATOM    706  CB  SER A  96     -18.573 -11.886  -6.750  1.00  0.39           C  \\nATOM    707  O   SER A  96     -17.058 -14.207  -6.247  1.00  0.40           O  \\nATOM    708  OG  SER A  96     -17.790 -12.158  -7.899  1.00  0.39           O  \\nATOM    709  N   TYR A  97     -16.608 -13.830  -4.306  1.00  0.37           N  \\nATOM    710  CA  TYR A  97     -16.806 -15.114  -3.643  1.00  0.40           C  \\nATOM    711  C   TYR A  97     -17.676 -14.958  -2.402  1.00  0.37           C  \\nATOM    712  CB  TYR A  97     -15.459 -15.735  -3.261  1.00  0.35           C  \\nATOM    713  O   TYR A  97     -17.390 -14.129  -1.535  1.00  0.35           O  \\nATOM    714  CG  TYR A  97     -14.851 -16.584  -4.352  1.00  0.36           C  \\nATOM    715  CD1 TYR A  97     -15.131 -17.946  -4.439  1.00  0.35           C  \\nATOM    716  CD2 TYR A  97     -13.996 -16.026  -5.296  1.00  0.37           C  \\nATOM    717  CE1 TYR A  97     -14.572 -18.732  -5.441  1.00  0.38           C  \\nATOM    718  CE2 TYR A  97     -13.431 -16.803  -6.302  1.00  0.40           C  \\nATOM    719  OH  TYR A  97     -13.168 -18.926  -7.360  1.00  0.30           O  \\nATOM    720  CZ  TYR A  97     -13.725 -18.152  -6.366  1.00  0.34           C  \\nATOM    721  N   PRO A  98     -18.974 -15.240  -2.351  1.00  0.41           N  \\nATOM    722  CA  PRO A  98     -19.700 -15.682  -1.157  1.00  0.42           C  \\nATOM    723  C   PRO A  98     -18.932 -16.729  -0.354  1.00  0.41           C  \\nATOM    724  CB  PRO A  98     -20.992 -16.271  -1.729  1.00  0.38           C  \\nATOM    725  O   PRO A  98     -18.699 -17.838  -0.842  1.00  0.39           O  \\nATOM    726  CG  PRO A  98     -20.689 -16.531  -3.169  1.00  0.38           C  \\nATOM    727  CD  PRO A  98     -19.458 -15.757  -3.545  1.00  0.44           C  \\nATOM    728  N   GLY A  99     -17.771 -16.322   0.139  1.00  0.36           N  \\nATOM    729  CA  GLY A  99     -17.399 -17.214   1.226  1.00  0.40           C  \\nATOM    730  C   GLY A  99     -15.967 -17.027   1.689  1.00  0.36           C  \\nATOM    731  O   GLY A  99     -15.027 -17.242   0.920  1.00  0.35           O  \\nATOM    732  N   TYR A 100     -15.772 -15.810   2.261  1.00  0.33           N  \\nATOM    733  CA  TYR A 100     -14.944 -15.878   3.460  1.00  0.38           C  \\nATOM    734  C   TYR A 100     -14.901 -14.531   4.171  1.00  0.33           C  \\nATOM    735  CB  TYR A 100     -13.522 -16.327   3.107  1.00  0.31           C  \\nATOM    736  O   TYR A 100     -14.564 -13.512   3.564  1.00  0.32           O  \\nATOM    737  CG  TYR A 100     -13.382 -17.820   2.938  1.00  0.32           C  \\nATOM    738  CD1 TYR A 100     -13.138 -18.645   4.034  1.00  0.31           C  \\nATOM    739  CD2 TYR A 100     -13.490 -18.409   1.683  1.00  0.32           C  \\nATOM    740  CE1 TYR A 100     -13.006 -20.021   3.882  1.00  0.32           C  \\nATOM    741  CE2 TYR A 100     -13.360 -19.784   1.519  1.00  0.35           C  \\nATOM    742  OH  TYR A 100     -12.988 -21.943   2.468  1.00  0.18           O  \\nATOM    743  CZ  TYR A 100     -13.118 -20.581   2.623  1.00  0.29           C  \\nATOM    744  N   GLY A 101     -15.967 -14.411   5.187  1.00  0.34           N  \\nATOM    745  CA  GLY A 101     -15.922 -14.175   6.621  1.00  0.37           C  \\nATOM    746  C   GLY A 101     -15.800 -12.707   6.981  1.00  0.34           C  \\nATOM    747  O   GLY A 101     -14.910 -12.012   6.486  1.00  0.32           O  \\nATOM    748  N   GLN A 102     -16.951 -11.842   6.835  1.00  0.41           N  \\nATOM    749  CA  GLN A 102     -17.277 -10.970   7.958  1.00  0.44           C  \\nATOM    750  C   GLN A 102     -16.128 -10.015   8.265  1.00  0.41           C  \\nATOM    751  CB  GLN A 102     -17.618 -11.797   9.199  1.00  0.37           C  \\nATOM    752  O   GLN A 102     -15.100 -10.425   8.807  1.00  0.39           O  \\nATOM    753  CG  GLN A 102     -19.076 -12.231   9.268  1.00  0.37           C  \\nATOM    754  CD  GLN A 102     -19.370 -13.114  10.465  1.00  0.43           C  \\nATOM    755  NE2 GLN A 102     -20.442 -13.895  10.378  1.00  0.35           N  \\nATOM    756  OE1 GLN A 102     -18.639 -13.096  11.461  1.00  0.48           O  \\nATOM    757  N   GLN A 103     -15.899  -8.981   7.436  1.00  0.46           N  \\nATOM    758  CA  GLN A 103     -15.599  -7.785   8.215  1.00  0.51           C  \\nATOM    759  C   GLN A 103     -16.878  -7.109   8.700  1.00  0.46           C  \\nATOM    760  CB  GLN A 103     -14.766  -6.802   7.390  1.00  0.43           C  \\nATOM    761  O   GLN A 103     -17.803  -6.884   7.917  1.00  0.44           O  \\nATOM    762  CG  GLN A 103     -14.213  -5.635   8.198  1.00  0.42           C  \\nATOM    763  CD  GLN A 103     -13.323  -4.722   7.377  1.00  0.48           C  \\nATOM    764  NE2 GLN A 103     -13.103  -3.508   7.870  1.00  0.38           N  \\nATOM    765  OE1 GLN A 103     -12.837  -5.104   6.308  1.00  0.50           O  \\nATOM    766  N   PRO A 104     -17.243  -7.348  10.017  1.00  0.46           N  \\nATOM    767  CA  PRO A 104     -18.233  -6.506  10.693  1.00  0.45           C  \\nATOM    768  C   PRO A 104     -18.334  -5.110  10.083  1.00  0.46           C  \\nATOM    769  CB  PRO A 104     -17.714  -6.437  12.131  1.00  0.42           C  \\nATOM    770  O   PRO A 104     -17.312  -4.475   9.811  1.00  0.44           O  \\nATOM    771  CG  PRO A 104     -16.299  -6.911  12.050  1.00  0.41           C  \\nATOM    772  CD  PRO A 104     -16.097  -7.584  10.723  1.00  0.45           C  \\nATOM    773  N   ALA A 105     -19.380  -4.724   9.227  1.00  0.44           N  \\nATOM    774  CA  ALA A 105     -20.069  -3.441   9.112  1.00  0.45           C  \\nATOM    775  C   ALA A 105     -19.602  -2.469  10.192  1.00  0.44           C  \\nATOM    776  CB  ALA A 105     -21.580  -3.639   9.196  1.00  0.42           C  \\nATOM    777  O   ALA A 105     -19.469  -2.846  11.359  1.00  0.43           O  \\nATOM    778  N   PRO A 106     -18.859  -1.426   9.929  1.00  0.51           N  \\nATOM    779  CA  PRO A 106     -18.782  -0.328  10.895  1.00  0.52           C  \\nATOM    780  C   PRO A 106     -20.156   0.213  11.285  1.00  0.51           C  \\nATOM    781  CB  PRO A 106     -17.966   0.735  10.156  1.00  0.47           C  \\nATOM    782  O   PRO A 106     -20.974   0.516  10.413  1.00  0.48           O  \\nATOM    783  CG  PRO A 106     -18.193   0.451   8.706  1.00  0.46           C  \\nATOM    784  CD  PRO A 106     -18.511  -1.008   8.550  1.00  0.50           C  \\nATOM    785  N   SER A 107     -20.939  -0.280  12.243  1.00  0.47           N  \\nATOM    786  CA  SER A 107     -21.821   0.512  13.094  1.00  0.49           C  \\nATOM    787  C   SER A 107     -21.891   1.960  12.622  1.00  0.46           C  \\nATOM    788  CB  SER A 107     -21.350   0.463  14.548  1.00  0.44           C  \\nATOM    789  O   SER A 107     -20.921   2.709  12.755  1.00  0.44           O  \\nATOM    790  OG  SER A 107     -20.013   0.921  14.659  1.00  0.50           O  \\nATOM    791  N   SER A 108     -22.617   2.336  11.611  1.00  0.49           N  \\nATOM    792  CA  SER A 108     -23.102   3.708  11.510  1.00  0.53           C  \\nATOM    793  C   SER A 108     -24.094   4.028  12.624  1.00  0.49           C  \\nATOM    794  CB  SER A 108     -23.757   3.948  10.149  1.00  0.45           C  \\nATOM    795  O   SER A 108     -25.136   3.379  12.739  1.00  0.47           O  \\nATOM    796  OG  SER A 108     -24.805   3.020   9.925  1.00  0.47           O  \\nATOM    797  N   THR A 109     -23.753   4.549  13.708  1.00  0.44           N  \\nATOM    798  CA  THR A 109     -24.119   5.528  14.725  1.00  0.49           C  \\nATOM    799  C   THR A 109     -25.239   6.436  14.224  1.00  0.44           C  \\nATOM    800  CB  THR A 109     -22.908   6.384  15.139  1.00  0.41           C  \\nATOM    801  O   THR A 109     -25.092   7.103  13.198  1.00  0.41           O  \\nATOM    802  CG2 THR A 109     -22.648   6.280  16.638  1.00  0.38           C  \\nATOM    803  OG1 THR A 109     -21.747   5.931  14.432  1.00  0.39           O  \\nATOM    804  N   SER A 110     -26.600   6.199  14.374  1.00  0.43           N  \\nATOM    805  CA  SER A 110     -27.590   6.934  15.155  1.00  0.49           C  \\nATOM    806  C   SER A 110     -27.446   8.439  14.954  1.00  0.43           C  \\nATOM    807  CB  SER A 110     -27.462   6.595  16.640  1.00  0.40           C  \\nATOM    808  O   SER A 110     -26.459   9.036  15.389  1.00  0.41           O  \\nATOM    809  OG  SER A 110     -26.165   6.105  16.934  1.00  0.47           O  \\nATOM    810  N   GLY A 111     -27.975   9.013  13.830  1.00  0.44           N  \\nATOM    811  CA  GLY A 111     -28.514  10.334  14.112  1.00  0.46           C  \\nATOM    812  C   GLY A 111     -29.794  10.630  13.353  1.00  0.44           C  \\nATOM    813  O   GLY A 111     -29.856  10.450  12.135  1.00  0.42           O  \\nATOM    814  N   SER A 112     -31.101  10.383  13.732  1.00  0.39           N  \\nATOM    815  CA  SER A 112     -32.245  11.125  14.250  1.00  0.43           C  \\nATOM    816  C   SER A 112     -32.598  12.300  13.344  1.00  0.39           C  \\nATOM    817  CB  SER A 112     -31.961  11.630  15.666  1.00  0.36           C  \\nATOM    818  O   SER A 112     -31.726  13.088  12.972  1.00  0.38           O  \\nATOM    819  OG  SER A 112     -33.145  12.114  16.274  1.00  0.42           O  \\nATOM    820  N   TYR A 113     -33.721  12.277  12.392  1.00  0.45           N  \\nATOM    821  CA  TYR A 113     -34.730  13.327  12.462  1.00  0.50           C  \\nATOM    822  C   TYR A 113     -34.283  14.565  11.694  1.00  0.44           C  \\nATOM    823  CB  TYR A 113     -35.023  13.695  13.920  1.00  0.42           C  \\nATOM    824  O   TYR A 113     -33.285  15.196  12.048  1.00  0.43           O  \\nATOM    825  CG  TYR A 113     -35.890  12.690  14.638  1.00  0.46           C  \\nATOM    826  CD1 TYR A 113     -37.278  12.729  14.522  1.00  0.44           C  \\nATOM    827  CD2 TYR A 113     -35.324  11.699  15.433  1.00  0.44           C  \\nATOM    828  CE1 TYR A 113     -38.081  11.806  15.184  1.00  0.41           C  \\nATOM    829  CE2 TYR A 113     -36.117  10.770  16.098  1.00  0.45           C  \\nATOM    830  OH  TYR A 113     -38.283   9.915  16.624  1.00  0.31           O  \\nATOM    831  CZ  TYR A 113     -37.493  10.831  15.968  1.00  0.37           C  \\nATOM    832  N   GLY A 114     -34.672  14.597  10.213  1.00  0.43           N  \\nATOM    833  CA  GLY A 114     -35.365  15.787   9.745  1.00  0.46           C  \\nATOM    834  C   GLY A 114     -35.636  15.772   8.252  1.00  0.44           C  \\nATOM    835  O   GLY A 114     -34.719  15.579   7.451  1.00  0.42           O  \\nATOM    836  N   SER A 115     -36.701  15.165   7.598  1.00  0.38           N  \\nATOM    837  CA  SER A 115     -37.686  15.834   6.755  1.00  0.46           C  \\nATOM    838  C   SER A 115     -37.205  17.219   6.335  1.00  0.38           C  \\nATOM    839  CB  SER A 115     -39.025  15.951   7.486  1.00  0.38           C  \\nATOM    840  O   SER A 115     -36.987  18.089   7.180  1.00  0.38           O  \\nATOM    841  OG  SER A 115     -39.315  17.305   7.791  1.00  0.48           O  \\nATOM    842  N   SER A 116     -36.412  17.359   5.327  1.00  0.40           N  \\nATOM    843  CA  SER A 116     -36.647  18.572   4.551  1.00  0.44           C  \\nATOM    844  C   SER A 116     -36.127  18.425   3.124  1.00  0.40           C  \\nATOM    845  CB  SER A 116     -35.983  19.775   5.222  1.00  0.37           C  \\nATOM    846  O   SER A 116     -34.983  18.015   2.914  1.00  0.39           O  \\nATOM    847  OG  SER A 116     -34.586  19.574   5.348  1.00  0.43           O  \\nATOM    848  N   SER A 117     -36.991  17.996   2.182  1.00  0.39           N  \\nATOM    849  CA  SER A 117     -37.388  18.770   1.011  1.00  0.46           C  \\nATOM    850  C   SER A 117     -36.311  19.777   0.622  1.00  0.40           C  \\nATOM    851  CB  SER A 117     -38.708  19.498   1.271  1.00  0.39           C  \\nATOM    852  O   SER A 117     -36.158  20.813   1.273  1.00  0.39           O  \\nATOM    853  OG  SER A 117     -38.653  20.219   2.489  1.00  0.48           O  \\nATOM    854  N   GLN A 118     -35.284  19.393   0.208  1.00  0.39           N  \\nATOM    855  CA  GLN A 118     -34.627  20.214  -0.804  1.00  0.44           C  \\nATOM    856  C   GLN A 118     -34.318  19.401  -2.057  1.00  0.40           C  \\nATOM    857  CB  GLN A 118     -33.342  20.828  -0.245  1.00  0.37           C  \\nATOM    858  O   GLN A 118     -33.629  18.381  -1.987  1.00  0.38           O  \\nATOM    859  CG  GLN A 118     -33.540  22.198   0.389  1.00  0.39           C  \\nATOM    860  CD  GLN A 118     -32.269  22.746   1.011  1.00  0.45           C  \\nATOM    861  NE2 GLN A 118     -32.405  23.803   1.805  1.00  0.37           N  \\nATOM    862  OE1 GLN A 118     -31.174  22.223   0.782  1.00  0.48           O  \\nATOM    863  N   SER A 119     -35.265  19.303  -3.038  1.00  0.38           N  \\nATOM    864  CA  SER A 119     -35.178  19.759  -4.422  1.00  0.44           C  \\nATOM    865  C   SER A 119     -33.792  20.313  -4.735  1.00  0.38           C  \\nATOM    866  CB  SER A 119     -36.238  20.825  -4.703  1.00  0.38           C  \\nATOM    867  O   SER A 119     -33.435  21.401  -4.278  1.00  0.38           O  \\nATOM    868  OG  SER A 119     -36.026  21.970  -3.896  1.00  0.45           O  \\nATOM    869  N   SER A 120     -32.862  19.750  -4.699  1.00  0.37           N  \\nATOM    870  CA  SER A 120     -31.609  20.086  -5.368  1.00  0.42           C  \\nATOM    871  C   SER A 120     -31.364  19.180  -6.571  1.00  0.38           C  \\nATOM    872  CB  SER A 120     -30.436  19.981  -4.393  1.00  0.36           C  \\nATOM    873  O   SER A 120     -31.435  17.955  -6.456  1.00  0.37           O  \\nATOM    874  OG  SER A 120     -30.295  18.654  -3.916  1.00  0.38           O  \\nATOM    875  N   SER A 121     -32.051  19.623  -7.647  1.00  0.37           N  \\nATOM    876  CA  SER A 121     -31.559  19.752  -9.015  1.00  0.42           C  \\nATOM    877  C   SER A 121     -30.591  18.625  -9.362  1.00  0.38           C  \\nATOM    878  CB  SER A 121     -30.871  21.103  -9.212  1.00  0.37           C  \\nATOM    879  O   SER A 121     -29.531  18.498  -8.747  1.00  0.37           O  \\nATOM    880  OG  SER A 121     -29.467  20.944  -9.310  1.00  0.39           O  \\nATOM    881  N   TYR A 122     -30.998  17.526  -9.191  1.00  0.43           N  \\nATOM    882  CA  TYR A 122     -30.358  16.730 -10.233  1.00  0.48           C  \\nATOM    883  C   TYR A 122     -29.977  17.599 -11.426  1.00  0.43           C  \\nATOM    884  CB  TYR A 122     -31.282  15.596 -10.687  1.00  0.42           C  \\nATOM    885  O   TYR A 122     -30.842  18.013 -12.202  1.00  0.42           O  \\nATOM    886  CG  TYR A 122     -31.402  14.473  -9.686  1.00  0.43           C  \\nATOM    887  CD1 TYR A 122     -30.453  13.454  -9.637  1.00  0.42           C  \\nATOM    888  CD2 TYR A 122     -32.463  14.428  -8.788  1.00  0.42           C  \\nATOM    889  CE1 TYR A 122     -30.560  12.416  -8.718  1.00  0.39           C  \\nATOM    890  CE2 TYR A 122     -32.580  13.394  -7.864  1.00  0.44           C  \\nATOM    891  OH  TYR A 122     -31.736  11.369  -6.924  1.00  0.24           O  \\nATOM    892  CZ  TYR A 122     -31.625  12.395  -7.836  1.00  0.35           C  \\nATOM    893  N   GLY A 123     -29.277  18.575 -11.205  1.00  0.34           N  \\nATOM    894  CA  GLY A 123     -28.641  19.597 -12.020  1.00  0.36           C  \\nATOM    895  C   GLY A 123     -27.540  19.051 -12.910  1.00  0.35           C  \\nATOM    896  O   GLY A 123     -26.700  18.269 -12.458  1.00  0.33           O  \\nATOM    897  N   GLN A 124     -27.926  18.387 -14.048  1.00  0.39           N  \\nATOM    898  CA  GLN A 124     -26.911  18.377 -15.097  1.00  0.41           C  \\nATOM    899  C   GLN A 124     -25.519  18.156 -14.512  1.00  0.39           C  \\nATOM    900  CB  GLN A 124     -26.946  19.684 -15.890  1.00  0.37           C  \\nATOM    901  O   GLN A 124     -25.188  18.704 -13.459  1.00  0.37           O  \\nATOM    902  CG  GLN A 124     -27.814  19.623 -17.139  1.00  0.38           C  \\nATOM    903  CD  GLN A 124     -27.807  20.921 -17.924  1.00  0.47           C  \\nATOM    904  NE2 GLN A 124     -27.548  20.828 -19.224  1.00  0.38           N  \\nATOM    905  OE1 GLN A 124     -28.033  22.000 -17.366  1.00  0.51           O  \\nATOM    906  N   PRO A 125     -24.752  17.099 -14.561  1.00  0.45           N  \\nATOM    907  CA  PRO A 125     -23.354  17.417 -14.260  1.00  0.46           C  \\nATOM    908  C   PRO A 125     -23.071  18.917 -14.284  1.00  0.45           C  \\nATOM    909  CB  PRO A 125     -22.583  16.698 -15.371  1.00  0.43           C  \\nATOM    910  O   PRO A 125     -23.317  19.579 -15.296  1.00  0.44           O  \\nATOM    911  CG  PRO A 125     -23.581  16.505 -16.466  1.00  0.42           C  \\nATOM    912  CD  PRO A 125     -24.959  16.617 -15.880  1.00  0.47           C  \\nATOM    913  N   GLN A 126     -23.361  19.594 -13.407  1.00  0.34           N  \\nATOM    914  CA  GLN A 126     -22.553  20.803 -13.282  1.00  0.37           C  \\nATOM    915  C   GLN A 126     -21.071  20.498 -13.484  1.00  0.35           C  \\nATOM    916  CB  GLN A 126     -22.772  21.456 -11.916  1.00  0.32           C  \\nATOM    917  O   GLN A 126     -20.512  19.633 -12.806  1.00  0.33           O  \\nATOM    918  CG  GLN A 126     -24.024  22.321 -11.843  1.00  0.34           C  \\nATOM    919  CD  GLN A 126     -24.137  23.079 -10.534  1.00  0.43           C  \\nATOM    920  NE2 GLN A 126     -25.106  23.985 -10.455  1.00  0.33           N  \\nATOM    921  OE1 GLN A 126     -23.361  22.853  -9.600  1.00  0.46           O  \\nATOM    922  N   SER A 127     -20.440  20.405 -14.615  1.00  0.37           N  \\nATOM    923  CA  SER A 127     -19.087  20.953 -14.606  1.00  0.40           C  \\nATOM    924  C   SER A 127     -18.726  21.511 -13.234  1.00  0.37           C  \\nATOM    925  CB  SER A 127     -18.946  22.048 -15.665  1.00  0.36           C  \\nATOM    926  O   SER A 127     -19.144  22.614 -12.877  1.00  0.36           O  \\nATOM    927  OG  SER A 127     -20.073  22.907 -15.653  1.00  0.41           O  \\nATOM    928  N   GLY A 128     -18.891  20.941 -12.239  1.00  0.33           N  \\nATOM    929  CA  GLY A 128     -18.338  21.516 -11.024  1.00  0.37           C  \\nATOM    930  C   GLY A 128     -16.861  21.220 -10.843  1.00  0.34           C  \\nATOM    931  O   GLY A 128     -16.428  20.075 -10.996  1.00  0.32           O  \\nATOM    932  N   SER A 129     -15.975  21.967 -11.547  1.00  0.35           N  \\nATOM    933  CA  SER A 129     -14.616  22.248 -11.093  1.00  0.39           C  \\nATOM    934  C   SER A 129     -13.939  20.988 -10.564  1.00  0.36           C  \\nATOM    935  CB  SER A 129     -14.625  23.325 -10.008  1.00  0.34           C  \\nATOM    936  O   SER A 129     -14.385  20.405  -9.574  1.00  0.34           O  \\nATOM    937  OG  SER A 129     -13.415  24.063 -10.022  1.00  0.34           O  \\nATOM    938  N   TYR A 130     -13.646  20.055 -11.373  1.00  0.43           N  \\nATOM    939  CA  TYR A 130     -12.431  19.330 -11.019  1.00  0.47           C  \\nATOM    940  C   TYR A 130     -11.608  20.109 -10.000  1.00  0.44           C  \\nATOM    941  CB  TYR A 130     -11.588  19.052 -12.267  1.00  0.41           C  \\nATOM    942  O   TYR A 130     -11.035  21.152 -10.322  1.00  0.42           O  \\nATOM    943  CG  TYR A 130     -12.156  17.972 -13.156  1.00  0.42           C  \\nATOM    944  CD1 TYR A 130     -11.863  16.629 -12.927  1.00  0.40           C  \\nATOM    945  CD2 TYR A 130     -12.986  18.291 -14.225  1.00  0.41           C  \\nATOM    946  CE1 TYR A 130     -12.383  15.631 -13.744  1.00  0.38           C  \\nATOM    947  CE2 TYR A 130     -13.511  17.301 -15.048  1.00  0.44           C  \\nATOM    948  OH  TYR A 130     -13.722  14.991 -15.612  1.00  0.25           O  \\nATOM    949  CZ  TYR A 130     -13.205  15.976 -14.800  1.00  0.34           C  \\nATOM    950  N   SER A 131     -11.954  20.373  -8.911  1.00  0.36           N  \\nATOM    951  CA  SER A 131     -10.972  20.531  -7.843  1.00  0.40           C  \\nATOM    952  C   SER A 131      -9.960  19.390  -7.850  1.00  0.36           C  \\nATOM    953  CB  SER A 131     -11.665  20.599  -6.482  1.00  0.35           C  \\nATOM    954  O   SER A 131     -10.339  18.217  -7.832  1.00  0.35           O  \\nATOM    955  OG  SER A 131     -11.734  19.315  -5.887  1.00  0.36           O  \\nATOM    956  N   GLN A 132      -8.930  19.471  -8.865  1.00  0.39           N  \\nATOM    957  CA  GLN A 132      -7.552  19.020  -8.707  1.00  0.41           C  \\nATOM    958  C   GLN A 132      -7.420  18.055  -7.533  1.00  0.40           C  \\nATOM    959  CB  GLN A 132      -6.615  20.213  -8.513  1.00  0.37           C  \\nATOM    960  O   GLN A 132      -7.634  18.438  -6.381  1.00  0.38           O  \\nATOM    961  CG  GLN A 132      -6.255  20.930  -9.808  1.00  0.38           C  \\nATOM    962  CD  GLN A 132      -5.336  22.116  -9.587  1.00  0.45           C  \\nATOM    963  NE2 GLN A 132      -4.995  22.812 -10.666  1.00  0.37           N  \\nATOM    964  OE1 GLN A 132      -4.935  22.405  -8.455  1.00  0.51           O  \\nATOM    965  N   GLN A 133      -7.888  16.887  -7.636  1.00  0.41           N  \\nATOM    966  CA  GLN A 133      -7.071  15.974  -6.843  1.00  0.45           C  \\nATOM    967  C   GLN A 133      -5.617  16.435  -6.798  1.00  0.41           C  \\nATOM    968  CB  GLN A 133      -7.154  14.553  -7.403  1.00  0.39           C  \\nATOM    969  O   GLN A 133      -5.048  16.812  -7.825  1.00  0.40           O  \\nATOM    970  CG  GLN A 133      -7.228  13.474  -6.331  1.00  0.40           C  \\nATOM    971  CD  GLN A 133      -7.369  12.079  -6.912  1.00  0.47           C  \\nATOM    972  NE2 GLN A 133      -6.526  11.159  -6.457  1.00  0.38           N  \\nATOM    973  OE1 GLN A 133      -8.230  11.831  -7.763  1.00  0.51           O  \\nATOM    974  N   PRO A 134      -5.081  16.977  -5.782  1.00  0.38           N  \\nATOM    975  CA  PRO A 134      -3.617  17.008  -5.756  1.00  0.39           C  \\nATOM    976  C   PRO A 134      -2.988  15.964  -6.675  1.00  0.38           C  \\nATOM    977  CB  PRO A 134      -3.284  16.717  -4.290  1.00  0.36           C  \\nATOM    978  O   PRO A 134      -3.383  14.795  -6.649  1.00  0.36           O  \\nATOM    979  CG  PRO A 134      -4.523  16.093  -3.734  1.00  0.36           C  \\nATOM    980  CD  PRO A 134      -5.673  16.425  -4.640  1.00  0.41           C  \\nATOM    981  N   SER A 135      -2.692  16.223  -8.005  1.00  0.36           N  \\nATOM    982  CA  SER A 135      -1.567  15.622  -8.712  1.00  0.37           C  \\nATOM    983  C   SER A 135      -0.575  14.993  -7.739  1.00  0.36           C  \\nATOM    984  CB  SER A 135      -0.856  16.665  -9.575  1.00  0.34           C  \\nATOM    985  O   SER A 135      -0.008  15.684  -6.890  1.00  0.35           O  \\nATOM    986  OG  SER A 135       0.340  17.102  -8.952  1.00  0.36           O  \\nATOM    987  N   TYR A 136      -0.828  14.101  -7.051  1.00  0.36           N  \\nATOM    988  CA  TYR A 136       0.382  13.373  -6.689  1.00  0.40           C  \\nATOM    989  C   TYR A 136       1.330  13.266  -7.877  1.00  0.36           C  \\nATOM    990  CB  TYR A 136       0.033  11.974  -6.172  1.00  0.35           C  \\nATOM    991  O   TYR A 136       0.969  12.716  -8.920  1.00  0.35           O  \\nATOM    992  CG  TYR A 136      -0.497  11.963  -4.759  1.00  0.36           C  \\nATOM    993  CD1 TYR A 136       0.371  11.927  -3.669  1.00  0.35           C  \\nATOM    994  CD2 TYR A 136      -1.865  11.989  -4.510  1.00  0.36           C  \\nATOM    995  CE1 TYR A 136      -0.113  11.915  -2.365  1.00  0.34           C  \\nATOM    996  CE2 TYR A 136      -2.360  11.978  -3.210  1.00  0.38           C  \\nATOM    997  OH  TYR A 136      -1.962  11.930  -0.857  1.00  0.19           O  \\nATOM    998  CZ  TYR A 136      -1.478  11.941  -2.146  1.00  0.31           C  \\nATOM    999  N   GLY A 137       2.125  14.316  -8.321  1.00  0.33           N  \\nATOM   1000  CA  GLY A 137       3.463  14.255  -8.887  1.00  0.35           C  \\nATOM   1001  C   GLY A 137       4.040  12.852  -8.905  1.00  0.34           C  \\nATOM   1002  O   GLY A 137       4.507  12.354  -7.878  1.00  0.32           O  \\nATOM   1003  N   GLY A 138       3.488  11.888  -9.423  1.00  0.35           N  \\nATOM   1004  CA  GLY A 138       4.268  10.699  -9.728  1.00  0.37           C  \\nATOM   1005  C   GLY A 138       5.363  10.950 -10.748  1.00  0.35           C  \\nATOM   1006  O   GLY A 138       5.118  11.558 -11.793  1.00  0.34           O  \\nATOM   1007  N   GLN A 139       6.542  11.443 -10.402  1.00  0.37           N  \\nATOM   1008  CA  GLN A 139       7.870  11.292 -10.987  1.00  0.40           C  \\nATOM   1009  C   GLN A 139       7.808  10.520 -12.301  1.00  0.37           C  \\nATOM   1010  CB  GLN A 139       8.812  10.588 -10.008  1.00  0.35           C  \\nATOM   1011  O   GLN A 139       7.463   9.336 -12.316  1.00  0.36           O  \\nATOM   1012  CG  GLN A 139       9.422  11.516  -8.967  1.00  0.35           C  \\nATOM   1013  CD  GLN A 139      10.391  10.803  -8.041  1.00  0.42           C  \\nATOM   1014  NE2 GLN A 139      11.057  11.564  -7.180  1.00  0.36           N  \\nATOM   1015  OE1 GLN A 139      10.538   9.578  -8.100  1.00  0.48           O  \\nATOM   1016  N   GLN A 140       7.600  11.084 -13.506  1.00  0.36           N  \\nATOM   1017  CA  GLN A 140       8.034  10.725 -14.852  1.00  0.38           C  \\nATOM   1018  C   GLN A 140       9.549  10.845 -14.991  1.00  0.36           C  \\nATOM   1019  CB  GLN A 140       7.340  11.603 -15.894  1.00  0.34           C  \\nATOM   1020  O   GLN A 140      10.109  11.930 -14.826  1.00  0.35           O  \\nATOM   1021  CG  GLN A 140       6.275  10.875 -16.701  1.00  0.34           C  \\nATOM   1022  CD  GLN A 140       5.538  11.789 -17.662  1.00  0.42           C  \\nATOM   1023  NE2 GLN A 140       4.689  11.207 -18.502  1.00  0.35           N  \\nATOM   1024  OE1 GLN A 140       5.730  13.009 -17.648  1.00  0.48           O  \\nATOM   1025  N   GLN A 141      10.346   9.771 -14.837  1.00  0.36           N  \\nATOM   1026  CA  GLN A 141      11.724   9.413 -15.154  1.00  0.39           C  \\nATOM   1027  C   GLN A 141      11.920   9.257 -16.660  1.00  0.36           C  \\nATOM   1028  CB  GLN A 141      12.123   8.123 -14.436  1.00  0.34           C  \\nATOM   1029  O   GLN A 141      11.228   8.464 -17.302  1.00  0.35           O  \\nATOM   1030  CG  GLN A 141      12.323   8.291 -12.936  1.00  0.34           C  \\nATOM   1031  CD  GLN A 141      12.810   7.021 -12.263  1.00  0.40           C  \\nATOM   1032  NE2 GLN A 141      13.015   7.085 -10.952  1.00  0.36           N  \\nATOM   1033  OE1 GLN A 141      12.998   5.989 -12.917  1.00  0.46           O  \\nATOM   1034  N   SER A 142      12.477  10.178 -17.463  1.00  0.35           N  \\nATOM   1035  CA  SER A 142      13.252  10.327 -18.690  1.00  0.38           C  \\nATOM   1036  C   SER A 142      14.511   9.468 -18.657  1.00  0.36           C  \\nATOM   1037  CB  SER A 142      13.630  11.792 -18.910  1.00  0.34           C  \\nATOM   1038  O   SER A 142      15.347   9.616 -17.763  1.00  0.35           O  \\nATOM   1039  OG  SER A 142      12.954  12.322 -20.038  1.00  0.35           O  \\nATOM   1040  N   TYR A 143      14.616   8.174 -18.988  1.00  0.34           N  \\nATOM   1041  CA  TYR A 143      15.577   7.110 -19.256  1.00  0.37           C  \\nATOM   1042  C   TYR A 143      16.277   7.330 -20.591  1.00  0.34           C  \\nATOM   1043  CB  TYR A 143      14.882   5.745 -19.250  1.00  0.32           C  \\nATOM   1044  O   TYR A 143      15.640   7.298 -21.647  1.00  0.33           O  \\nATOM   1045  CG  TYR A 143      14.389   5.320 -17.888  1.00  0.32           C  \\nATOM   1046  CD1 TYR A 143      15.251   4.730 -16.966  1.00  0.30           C  \\nATOM   1047  CD2 TYR A 143      13.061   5.508 -17.520  1.00  0.33           C  \\nATOM   1048  CE1 TYR A 143      14.800   4.335 -15.711  1.00  0.33           C  \\nATOM   1049  CE2 TYR A 143      12.599   5.118 -16.268  1.00  0.36           C  \\nATOM   1050  OH  TYR A 143      13.023   4.145 -14.129  1.00  0.26           O  \\nATOM   1051  CZ  TYR A 143      13.475   4.534 -15.371  1.00  0.30           C  \\nATOM   1052  N   GLY A 144      17.458   7.820 -20.694  1.00  0.33           N  \\nATOM   1053  CA  GLY A 144      18.668   8.028 -21.473  1.00  0.38           C  \\nATOM   1054  C   GLY A 144      19.852   7.230 -20.962  1.00  0.34           C  \\nATOM   1055  O   GLY A 144      20.203   7.316 -19.783  1.00  0.34           O  \\nATOM   1056  N   GLN A 145      19.922   5.754 -21.297  1.00  0.37           N  \\nATOM   1057  CA  GLN A 145      21.033   4.890 -21.681  1.00  0.40           C  \\nATOM   1058  C   GLN A 145      20.694   3.421 -21.446  1.00  0.37           C  \\nATOM   1059  CB  GLN A 145      22.298   5.266 -20.908  1.00  0.34           C  \\nATOM   1060  O   GLN A 145      20.273   3.042 -20.351  1.00  0.35           O  \\nATOM   1061  CG  GLN A 145      23.053   6.448 -21.501  1.00  0.35           C  \\nATOM   1062  CD  GLN A 145      24.348   6.745 -20.769  1.00  0.43           C  \\nATOM   1063  NE2 GLN A 145      25.148   7.650 -21.323  1.00  0.34           N  \\nATOM   1064  OE1 GLN A 145      24.628   6.166 -19.714  1.00  0.48           O  \\nATOM   1065  N   GLN A 146      20.258   2.444 -22.572  1.00  0.35           N  \\nATOM   1066  CA  GLN A 146      20.458   1.169 -23.253  1.00  0.39           C  \\nATOM   1067  C   GLN A 146      21.500   0.318 -22.533  1.00  0.35           C  \\nATOM   1068  CB  GLN A 146      20.879   1.395 -24.706  1.00  0.33           C  \\nATOM   1069  O   GLN A 146      22.691   0.636 -22.553  1.00  0.34           O  \\nATOM   1070  CG  GLN A 146      19.728   1.313 -25.700  1.00  0.34           C  \\nATOM   1071  CD  GLN A 146      20.164   1.587 -27.127  1.00  0.43           C  \\nATOM   1072  NE2 GLN A 146      19.276   1.321 -28.079  1.00  0.34           N  \\nATOM   1073  OE1 GLN A 146      21.288   2.035 -27.371  1.00  0.49           O  \\nATOM   1074  N   GLN A 147      21.133  -0.478 -21.416  1.00  0.37           N  \\nATOM   1075  CA  GLN A 147      21.851  -1.722 -21.157  1.00  0.42           C  \\nATOM   1076  C   GLN A 147      20.956  -2.934 -21.401  1.00  0.37           C  \\nATOM   1077  CB  GLN A 147      22.388  -1.745 -19.725  1.00  0.35           C  \\nATOM   1078  O   GLN A 147      19.853  -3.017 -20.858  1.00  0.36           O  \\nATOM   1079  CG  GLN A 147      23.650  -0.917 -19.529  1.00  0.36           C  \\nATOM   1080  CD  GLN A 147      24.264  -1.100 -18.153  1.00  0.43           C  \\nATOM   1081  NE2 GLN A 147      25.408  -0.463 -17.925  1.00  0.36           N  \\nATOM   1082  OE1 GLN A 147      23.715  -1.808 -17.303  1.00  0.49           O  \\nATOM   1083  N   SER A 148      20.929  -3.711 -22.625  1.00  0.36           N  \\nATOM   1084  CA  SER A 148      20.681  -5.043 -23.167  1.00  0.41           C  \\nATOM   1085  C   SER A 148      20.622  -6.089 -22.059  1.00  0.37           C  \\nATOM   1086  CB  SER A 148      21.763  -5.421 -24.179  1.00  0.35           C  \\nATOM   1087  O   SER A 148      21.613  -6.323 -21.365  1.00  0.36           O  \\nATOM   1088  OG  SER A 148      23.055  -5.216 -23.633  1.00  0.36           O  \\nATOM   1089  N   TYR A 149      19.595  -6.231 -21.311  1.00  0.37           N  \\nATOM   1090  CA  TYR A 149      19.323  -7.453 -20.563  1.00  0.41           C  \\nATOM   1091  C   TYR A 149      18.575  -8.464 -21.424  1.00  0.38           C  \\nATOM   1092  CB  TYR A 149      18.514  -7.142 -19.301  1.00  0.35           C  \\nATOM   1093  O   TYR A 149      17.497  -8.168 -21.945  1.00  0.37           O  \\nATOM   1094  CG  TYR A 149      18.702  -8.152 -18.195  1.00  0.35           C  \\nATOM   1095  CD1 TYR A 149      17.802  -9.202 -18.024  1.00  0.31           C  \\nATOM   1096  CD2 TYR A 149      19.778  -8.058 -17.319  1.00  0.33           C  \\nATOM   1097  CE1 TYR A 149      17.970 -10.134 -17.006  1.00  0.31           C  \\nATOM   1098  CE2 TYR A 149      19.955  -8.985 -16.297  1.00  0.37           C  \\nATOM   1099  OH  TYR A 149      19.219 -10.938 -15.138  1.00  0.21           O  \\nATOM   1100  CZ  TYR A 149      19.048 -10.018 -16.148  1.00  0.29           C  \\nATOM   1101  N   ASN A 150      19.271  -9.485 -21.970  1.00  0.36           N  \\nATOM   1102  CA  ASN A 150      19.002 -10.871 -22.341  1.00  0.39           C  \\nATOM   1103  C   ASN A 150      18.105 -11.561 -21.316  1.00  0.36           C  \\nATOM   1104  CB  ASN A 150      20.310 -11.646 -22.508  1.00  0.35           C  \\nATOM   1105  O   ASN A 150      18.497 -11.740 -20.162  1.00  0.35           O  \\nATOM   1106  CG  ASN A 150      20.486 -12.200 -23.908  1.00  0.35           C  \\nATOM   1107  ND2 ASN A 150      21.676 -12.712 -24.199  1.00  0.30           N  \\nATOM   1108  OD1 ASN A 150      19.560 -12.167 -24.723  1.00  0.36           O  \\nATOM   1109  N   PRO A 151      16.760 -11.499 -21.374  1.00  0.42           N  \\nATOM   1110  CA  PRO A 151      15.884 -12.315 -20.529  1.00  0.44           C  \\nATOM   1111  C   PRO A 151      16.315 -13.778 -20.472  1.00  0.42           C  \\nATOM   1112  CB  PRO A 151      14.516 -12.174 -21.201  1.00  0.39           C  \\nATOM   1113  O   PRO A 151      16.923 -14.285 -21.419  1.00  0.41           O  \\nATOM   1114  CG  PRO A 151      14.822 -11.810 -22.618  1.00  0.39           C  \\nATOM   1115  CD  PRO A 151      16.189 -11.191 -22.667  1.00  0.44           C  \\nATOM   1116  N   PRO A 152      16.325 -14.478 -19.337  1.00  0.35           N  \\nATOM   1117  CA  PRO A 152      16.603 -15.916 -19.330  1.00  0.36           C  \\nATOM   1118  C   PRO A 152      15.852 -16.668 -20.427  1.00  0.35           C  \\nATOM   1119  CB  PRO A 152      16.131 -16.360 -17.943  1.00  0.33           C  \\nATOM   1120  O   PRO A 152      14.654 -16.448 -20.622  1.00  0.33           O  \\nATOM   1121  CG  PRO A 152      15.379 -15.188 -17.403  1.00  0.32           C  \\nATOM   1122  CD  PRO A 152      15.619 -14.010 -18.303  1.00  0.37           C  \\nATOM   1123  N   GLN A 153      16.357 -16.999 -21.615  1.00  0.32           N  \\nATOM   1124  CA  GLN A 153      16.200 -18.243 -22.361  1.00  0.34           C  \\nATOM   1125  C   GLN A 153      15.714 -19.370 -21.455  1.00  0.32           C  \\nATOM   1126  CB  GLN A 153      17.516 -18.638 -23.032  1.00  0.31           C  \\nATOM   1127  O   GLN A 153      16.364 -19.697 -20.459  1.00  0.31           O  \\nATOM   1128  CG  GLN A 153      17.893 -17.757 -24.215  1.00  0.33           C  \\nATOM   1129  CD  GLN A 153      19.080 -18.293 -24.992  1.00  0.40           C  \\nATOM   1130  NE2 GLN A 153      19.364 -17.683 -26.138  1.00  0.32           N  \\nATOM   1131  OE1 GLN A 153      19.738 -19.248 -24.565  1.00  0.47           O  \\nATOM   1132  N   GLY A 154      14.334 -19.404 -20.986  1.00  0.33           N  \\nATOM   1133  CA  GLY A 154      13.882 -20.659 -20.407  1.00  0.36           C  \\nATOM   1134  C   GLY A 154      12.581 -20.528 -19.638  1.00  0.33           C  \\nATOM   1135  O   GLY A 154      12.543 -19.909 -18.573  1.00  0.32           O  \\nATOM   1136  N   TYR A 155      11.351 -20.274 -20.327  1.00  0.33           N  \\nATOM   1137  CA  TYR A 155      10.068 -20.963 -20.253  1.00  0.38           C  \\nATOM   1138  C   TYR A 155       9.404 -21.028 -21.623  1.00  0.34           C  \\nATOM   1139  CB  TYR A 155       9.139 -20.263 -19.256  1.00  0.33           C  \\nATOM   1140  O   TYR A 155       8.923 -20.015 -22.136  1.00  0.33           O  \\nATOM   1141  CG  TYR A 155       9.557 -20.430 -17.815  1.00  0.34           C  \\nATOM   1142  CD1 TYR A 155       9.222 -21.579 -17.102  1.00  0.33           C  \\nATOM   1143  CD2 TYR A 155      10.286 -19.441 -17.165  1.00  0.34           C  \\nATOM   1144  CE1 TYR A 155       9.604 -21.737 -15.774  1.00  0.33           C  \\nATOM   1145  CE2 TYR A 155      10.673 -19.589 -15.837  1.00  0.37           C  \\nATOM   1146  OH  TYR A 155      10.708 -20.890 -13.836  1.00  0.18           O  \\nATOM   1147  CZ  TYR A 155      10.328 -20.739 -15.151  1.00  0.32           C  \\nATOM   1148  N   GLY A 156       9.821 -21.783 -22.608  1.00  0.35           N  \\nATOM   1149  CA  GLY A 156       9.278 -22.837 -23.450  1.00  0.35           C  \\nATOM   1150  C   GLY A 156       8.645 -23.966 -22.659  1.00  0.35           C  \\nATOM   1151  O   GLY A 156       9.348 -24.824 -22.121  1.00  0.34           O  \\nATOM   1152  N   GLN A 157       7.878 -23.819 -21.648  1.00  0.34           N  \\nATOM   1153  CA  GLN A 157       6.921 -24.893 -21.406  1.00  0.37           C  \\nATOM   1154  C   GLN A 157       5.573 -24.584 -22.051  1.00  0.35           C  \\nATOM   1155  CB  GLN A 157       6.743 -25.127 -19.905  1.00  0.32           C  \\nATOM   1156  O   GLN A 157       4.898 -23.629 -21.664  1.00  0.33           O  \\nATOM   1157  CG  GLN A 157       7.817 -26.014 -19.289  1.00  0.33           C  \\nATOM   1158  CD  GLN A 157       7.570 -26.301 -17.820  1.00  0.40           C  \\nATOM   1159  NE2 GLN A 157       8.508 -26.996 -17.186  1.00  0.32           N  \\nATOM   1160  OE1 GLN A 157       6.545 -25.900 -17.259  1.00  0.45           O  \\nATOM   1161  N   GLN A 158       5.519 -24.564 -23.423  1.00  0.38           N  \\nATOM   1162  CA  GLN A 158       4.400 -25.169 -24.139  1.00  0.39           C  \\nATOM   1163  C   GLN A 158       3.572 -26.058 -23.215  1.00  0.38           C  \\nATOM   1164  CB  GLN A 158       4.903 -25.979 -25.335  1.00  0.36           C  \\nATOM   1165  O   GLN A 158       4.057 -27.084 -22.733  1.00  0.37           O  \\nATOM   1166  CG  GLN A 158       5.089 -25.154 -26.601  1.00  0.36           C  \\nATOM   1167  CD  GLN A 158       5.489 -25.998 -27.798  1.00  0.42           C  \\nATOM   1168  NE2 GLN A 158       5.511 -25.383 -28.975  1.00  0.35           N  \\nATOM   1169  OE1 GLN A 158       5.777 -27.191 -27.664  1.00  0.46           O  \\nATOM   1170  N   ASN A 159       2.912 -25.656 -22.255  1.00  0.37           N  \\nATOM   1171  CA  ASN A 159       1.796 -26.511 -21.862  1.00  0.42           C  \\nATOM   1172  C   ASN A 159       1.105 -27.125 -23.075  1.00  0.38           C  \\nATOM   1173  CB  ASN A 159       0.790 -25.725 -21.018  1.00  0.36           C  \\nATOM   1174  O   ASN A 159       0.495 -26.412 -23.875  1.00  0.36           O  \\nATOM   1175  CG  ASN A 159       1.093 -25.793 -19.534  1.00  0.37           C  \\nATOM   1176  ND2 ASN A 159       0.452 -24.927 -18.758  1.00  0.30           N  \\nATOM   1177  OD1 ASN A 159       1.896 -26.617 -19.089  1.00  0.38           O  \\nATOM   1178  N   GLN A 160       1.691 -28.038 -23.773  1.00  0.37           N  \\nATOM   1179  CA  GLN A 160       1.053 -29.229 -24.323  1.00  0.38           C  \\nATOM   1180  C   GLN A 160      -0.272 -29.516 -23.622  1.00  0.37           C  \\nATOM   1181  CB  GLN A 160       1.981 -30.439 -24.206  1.00  0.35           C  \\nATOM   1182  O   GLN A 160      -0.289 -30.008 -22.492  1.00  0.36           O  \\nATOM   1183  CG  GLN A 160       2.988 -30.554 -25.342  1.00  0.35           C  \\nATOM   1184  CD  GLN A 160       3.866 -31.786 -25.226  1.00  0.42           C  \\nATOM   1185  NE2 GLN A 160       4.806 -31.933 -26.153  1.00  0.34           N  \\nATOM   1186  OE1 GLN A 160       3.702 -32.597 -24.310  1.00  0.45           O  \\nATOM   1187  N   TYR A 161      -1.213 -28.657 -23.586  1.00  0.40           N  \\nATOM   1188  CA  TYR A 161      -2.479 -29.380 -23.594  1.00  0.45           C  \\nATOM   1189  C   TYR A 161      -2.593 -30.268 -24.828  1.00  0.41           C  \\nATOM   1190  CB  TYR A 161      -3.657 -28.402 -23.545  1.00  0.39           C  \\nATOM   1191  O   TYR A 161      -2.679 -29.770 -25.953  1.00  0.39           O  \\nATOM   1192  CG  TYR A 161      -4.097 -28.052 -22.145  1.00  0.40           C  \\nATOM   1193  CD1 TYR A 161      -5.019 -28.844 -21.465  1.00  0.39           C  \\nATOM   1194  CD2 TYR A 161      -3.593 -26.928 -21.499  1.00  0.39           C  \\nATOM   1195  CE1 TYR A 161      -5.429 -28.524 -20.175  1.00  0.38           C  \\nATOM   1196  CE2 TYR A 161      -3.996 -26.598 -20.209  1.00  0.42           C  \\nATOM   1197  OH  TYR A 161      -5.315 -27.080 -18.279  1.00  0.23           O  \\nATOM   1198  CZ  TYR A 161      -4.913 -27.401 -19.556  1.00  0.34           C  \\nATOM   1199  N   ASN A 162      -1.863 -31.325 -24.959  1.00  0.36           N  \\nATOM   1200  CA  ASN A 162      -2.461 -32.541 -25.499  1.00  0.40           C  \\nATOM   1201  C   ASN A 162      -3.973 -32.560 -25.294  1.00  0.37           C  \\nATOM   1202  CB  ASN A 162      -1.824 -33.780 -24.866  1.00  0.36           C  \\nATOM   1203  O   ASN A 162      -4.452 -32.843 -24.195  1.00  0.36           O  \\nATOM   1204  CG  ASN A 162      -0.980 -34.569 -25.848  1.00  0.37           C  \\nATOM   1205  ND2 ASN A 162      -0.410 -35.675 -25.384  1.00  0.31           N  \\nATOM   1206  OD1 ASN A 162      -0.842 -34.187 -27.013  1.00  0.39           O  \\nATOM   1207  N   SER A 163      -4.711 -31.724 -25.790  1.00  0.43           N  \\nATOM   1208  CA  SER A 163      -6.034 -32.242 -26.123  1.00  0.45           C  \\nATOM   1209  C   SER A 163      -5.936 -33.547 -26.906  1.00  0.43           C  \\nATOM   1210  CB  SER A 163      -6.826 -31.212 -26.930  1.00  0.41           C  \\nATOM   1211  O   SER A 163      -5.600 -33.542 -28.092  1.00  0.42           O  \\nATOM   1212  OG  SER A 163      -6.135 -30.866 -28.119  1.00  0.42           O  \\nATOM   1213  N   SER A 164      -5.301 -34.495 -26.385  1.00  0.35           N  \\nATOM   1214  CA  SER A 164      -5.715 -35.855 -26.713  1.00  0.37           C  \\nATOM   1215  C   SER A 164      -7.196 -36.069 -26.418  1.00  0.35           C  \\nATOM   1216  CB  SER A 164      -4.880 -36.873 -25.935  1.00  0.33           C  \\nATOM   1217  O   SER A 164      -7.595 -36.165 -25.256  1.00  0.33           O  \\nATOM   1218  OG  SER A 164      -5.064 -36.712 -24.539  1.00  0.35           O  \\nATOM   1219  N   SER A 165      -8.053 -35.404 -27.001  1.00  0.39           N  \\nATOM   1220  CA  SER A 165      -9.263 -36.155 -27.317  1.00  0.42           C  \\nATOM   1221  C   SER A 165      -8.928 -37.528 -27.891  1.00  0.39           C  \\nATOM   1222  CB  SER A 165     -10.134 -35.378 -28.305  1.00  0.37           C  \\nATOM   1223  O   SER A 165      -8.303 -37.628 -28.949  1.00  0.37           O  \\nATOM   1224  OG  SER A 165      -9.560 -35.393 -29.601  1.00  0.38           O  \\nATOM   1225  N   GLY A 166      -8.371 -38.351 -27.164  1.00  0.32           N  \\nATOM   1226  CA  GLY A 166      -8.633 -39.781 -27.156  1.00  0.34           C  \\nATOM   1227  C   GLY A 166      -9.698 -40.199 -28.152  1.00  0.32           C  \\nATOM   1228  O   GLY A 166     -10.893 -40.139 -27.853  1.00  0.31           O  \\nATOM   1229  N   GLY A 167      -9.353 -40.065 -29.439  1.00  0.34           N  \\nATOM   1230  CA  GLY A 167      -9.923 -41.090 -30.298  1.00  0.36           C  \\nATOM   1231  C   GLY A 167      -8.912 -42.134 -30.733  1.00  0.35           C  \\nATOM   1232  O   GLY A 167      -7.872 -41.799 -31.303  1.00  0.33           O  \\nATOM   1233  N   GLY A 168      -8.567 -43.035 -29.923  1.00  0.30           N  \\nATOM   1234  CA  GLY A 168      -8.580 -44.464 -30.192  1.00  0.35           C  \\nATOM   1235  C   GLY A 168      -8.280 -44.801 -31.640  1.00  0.30           C  \\nATOM   1236  O   GLY A 168      -9.045 -44.444 -32.538  1.00  0.30           O  \\nATOM   1237  N   GLY A 169      -6.923 -44.833 -31.988  1.00  0.31           N  \\nATOM   1238  CA  GLY A 169      -6.664 -45.958 -32.874  1.00  0.35           C  \\nATOM   1239  C   GLY A 169      -5.450 -45.751 -33.760  1.00  0.31           C  \\nATOM   1240  O   GLY A 169      -5.366 -44.758 -34.485  1.00  0.31           O  \\nATOM   1241  N   GLY A 170      -4.264 -46.025 -33.253  1.00  0.30           N  \\nATOM   1242  CA  GLY A 170      -3.505 -46.977 -34.049  1.00  0.35           C  \\nATOM   1243  C   GLY A 170      -2.458 -46.319 -34.928  1.00  0.31           C  \\nATOM   1244  O   GLY A 170      -2.784 -45.478 -35.768  1.00  0.30           O  \\nATOM   1245  N   GLY A 171      -1.270 -46.045 -34.319  1.00  0.32           N  \\nATOM   1246  CA  GLY A 171      -0.157 -46.600 -35.072  1.00  0.36           C  \\nATOM   1247  C   GLY A 171       0.822 -45.546 -35.553  1.00  0.32           C  \\nATOM   1248  O   GLY A 171       0.435 -44.601 -36.243  1.00  0.32           O  \\nATOM   1249  N   GLY A 172       1.863 -45.185 -34.747  1.00  0.31           N  \\nATOM   1250  CA  GLY A 172       3.173 -45.414 -35.337  1.00  0.36           C  \\nATOM   1251  C   GLY A 172       3.678 -44.234 -36.146  1.00  0.32           C  \\nATOM   1252  O   GLY A 172       3.172 -43.960 -37.236  1.00  0.32           O  \\nATOM   1253  N   GLY A 173       4.212 -43.196 -35.419  1.00  0.34           N  \\nATOM   1254  CA  GLY A 173       5.447 -42.689 -35.997  1.00  0.36           C  \\nATOM   1255  C   GLY A 173       5.946 -41.424 -35.324  1.00  0.34           C  \\nATOM   1256  O   GLY A 173       5.213 -40.439 -35.222  1.00  0.34           O  \\nATOM   1257  N   GLY A 174       6.756 -41.507 -34.232  1.00  0.35           N  \\nATOM   1258  CA  GLY A 174       7.976 -40.725 -34.110  1.00  0.37           C  \\nATOM   1259  C   GLY A 174       8.100 -39.644 -35.167  1.00  0.35           C  \\nATOM   1260  O   GLY A 174       8.516 -39.916 -36.295  1.00  0.34           O  \\nATOM   1261  N   GLY A 175       7.266 -38.772 -35.404  1.00  0.34           N  \\nATOM   1262  CA  GLY A 175       7.704 -37.620 -36.177  1.00  0.36           C  \\nATOM   1263  C   GLY A 175       8.292 -36.515 -35.320  1.00  0.35           C  \\nATOM   1264  O   GLY A 175       7.674 -36.082 -34.346  1.00  0.34           O  \\nATOM   1265  N   ASN A 176       9.640 -36.729 -34.946  1.00  0.34           N  \\nATOM   1266  CA  ASN A 176      10.494 -35.548 -35.002  1.00  0.38           C  \\nATOM   1267  C   ASN A 176       9.701 -34.299 -35.377  1.00  0.35           C  \\nATOM   1268  CB  ASN A 176      11.643 -35.763 -35.989  1.00  0.33           C  \\nATOM   1269  O   ASN A 176       9.218 -34.181 -36.505  1.00  0.34           O  \\nATOM   1270  CG  ASN A 176      12.813 -34.832 -35.736  1.00  0.34           C  \\nATOM   1271  ND2 ASN A 176      13.676 -34.680 -36.733  1.00  0.29           N  \\nATOM   1272  OD1 ASN A 176      12.939 -34.254 -34.653  1.00  0.35           O  \\nATOM   1273  N   TYR A 177       8.884 -33.852 -34.528  1.00  0.35           N  \\nATOM   1274  CA  TYR A 177       8.470 -32.465 -34.706  1.00  0.40           C  \\nATOM   1275  C   TYR A 177       9.674 -31.565 -34.959  1.00  0.36           C  \\nATOM   1276  CB  TYR A 177       7.699 -31.972 -33.478  1.00  0.35           C  \\nATOM   1277  O   TYR A 177      10.482 -31.331 -34.058  1.00  0.35           O  \\nATOM   1278  CG  TYR A 177       6.314 -32.559 -33.356  1.00  0.37           C  \\nATOM   1279  CD1 TYR A 177       5.238 -32.006 -34.046  1.00  0.36           C  \\nATOM   1280  CD2 TYR A 177       6.078 -33.667 -32.549  1.00  0.37           C  \\nATOM   1281  CE1 TYR A 177       3.959 -32.542 -33.935  1.00  0.36           C  \\nATOM   1282  CE2 TYR A 177       4.803 -34.211 -32.430  1.00  0.40           C  \\nATOM   1283  OH  TYR A 177       2.488 -34.178 -33.012  1.00  0.21           O  \\nATOM   1284  CZ  TYR A 177       3.752 -33.643 -33.126  1.00  0.32           C  \\nATOM   1285  N   GLY A 178      10.352 -31.855 -35.997  1.00  0.34           N  \\nATOM   1286  CA  GLY A 178      11.144 -30.850 -36.688  1.00  0.34           C  \\nATOM   1287  C   GLY A 178      10.506 -29.473 -36.670  1.00  0.34           C  \\nATOM   1288  O   GLY A 178       9.311 -29.333 -36.939  1.00  0.33           O  \\nATOM   1289  N   GLN A 179      10.590 -28.829 -35.611  1.00  0.38           N  \\nATOM   1290  CA  GLN A 179      10.562 -27.402 -35.915  1.00  0.41           C  \\nATOM   1291  C   GLN A 179      10.429 -27.162 -37.416  1.00  0.38           C  \\nATOM   1292  CB  GLN A 179      11.820 -26.714 -35.383  1.00  0.36           C  \\nATOM   1293  O   GLN A 179      11.386 -27.356 -38.168  1.00  0.37           O  \\nATOM   1294  CG  GLN A 179      11.640 -26.079 -34.011  1.00  0.37           C  \\nATOM   1295  CD  GLN A 179      12.911 -25.433 -33.492  1.00  0.44           C  \\nATOM   1296  NE2 GLN A 179      12.832 -24.837 -32.307  1.00  0.36           N  \\nATOM   1297  OE1 GLN A 179      13.956 -25.471 -34.150  1.00  0.49           O  \\nATOM   1298  N   ASP A 180       9.345 -27.572 -37.977  1.00  0.34           N  \\nATOM   1299  CA  ASP A 180       8.994 -26.987 -39.267  1.00  0.36           C  \\nATOM   1300  C   ASP A 180       8.251 -25.665 -39.087  1.00  0.34           C  \\nATOM   1301  CB  ASP A 180       8.143 -27.962 -40.085  1.00  0.32           C  \\nATOM   1302  O   ASP A 180       7.124 -25.643 -38.588  1.00  0.33           O  \\nATOM   1303  CG  ASP A 180       8.959 -28.775 -41.074  1.00  0.33           C  \\nATOM   1304  OD1 ASP A 180      10.060 -28.333 -41.466  1.00  0.37           O  \\nATOM   1305  OD2 ASP A 180       8.495 -29.867 -41.467  1.00  0.36           O  \\nATOM   1306  N   GLN A 181       8.842 -24.650 -38.521  1.00  0.37           N  \\nATOM   1307  CA  GLN A 181       8.709 -23.275 -38.989  1.00  0.41           C  \\nATOM   1308  C   GLN A 181       8.546 -23.223 -40.505  1.00  0.37           C  \\nATOM   1309  CB  GLN A 181       9.919 -22.442 -38.561  1.00  0.36           C  \\nATOM   1310  O   GLN A 181       9.530 -23.308 -41.243  1.00  0.37           O  \\nATOM   1311  CG  GLN A 181       9.847 -21.947 -37.123  1.00  0.38           C  \\nATOM   1312  CD  GLN A 181      10.996 -21.023 -36.763  1.00  0.47           C  \\nATOM   1313  NE2 GLN A 181      11.150 -20.743 -35.474  1.00  0.38           N  \\nATOM   1314  OE1 GLN A 181      11.739 -20.565 -37.637  1.00  0.51           O  \\nATOM   1315  N   SER A 182       7.731 -24.026 -40.943  1.00  0.32           N  \\nATOM   1316  CA  SER A 182       7.218 -24.014 -42.310  1.00  0.36           C  \\nATOM   1317  C   SER A 182       5.831 -24.642 -42.385  1.00  0.32           C  \\nATOM   1318  CB  SER A 182       8.173 -24.754 -43.247  1.00  0.30           C  \\nATOM   1319  O   SER A 182       5.670 -25.837 -42.128  1.00  0.31           O  \\nATOM   1320  OG  SER A 182       8.508 -26.027 -42.722  1.00  0.29           O  \\nATOM   1321  N   SER A 183       4.752 -23.978 -41.665  1.00  0.34           N  \\nATOM   1322  CA  SER A 183       3.337 -24.006 -42.020  1.00  0.37           C  \\nATOM   1323  C   SER A 183       2.469 -23.502 -40.872  1.00  0.35           C  \\nATOM   1324  CB  SER A 183       2.909 -25.421 -42.410  1.00  0.33           C  \\nATOM   1325  O   SER A 183       2.174 -24.248 -39.936  1.00  0.33           O  \\nATOM   1326  OG  SER A 183       3.192 -26.338 -41.367  1.00  0.34           O  \\nATOM   1327  N   MET A 184       2.687 -22.256 -40.365  1.00  0.39           N  \\nATOM   1328  CA  MET A 184       1.414 -21.585 -40.119  1.00  0.41           C  \\nATOM   1329  C   MET A 184       0.564 -21.548 -41.385  1.00  0.39           C  \\nATOM   1330  CB  MET A 184       1.645 -20.164 -39.603  1.00  0.37           C  \\nATOM   1331  O   MET A 184       0.843 -20.773 -42.302  1.00  0.38           O  \\nATOM   1332  CG  MET A 184       2.017 -20.100 -38.130  1.00  0.38           C  \\nATOM   1333  SD  MET A 184       2.204 -18.377 -37.527  1.00  0.34           S  \\nATOM   1334  CE  MET A 184       2.315 -18.674 -35.740  1.00  0.37           C  \\nATOM   1335  N   SER A 185       0.274 -22.587 -41.937  1.00  0.34           N  \\nATOM   1336  CA  SER A 185      -0.829 -22.718 -42.883  1.00  0.38           C  \\nATOM   1337  C   SER A 185      -2.140 -22.231 -42.274  1.00  0.35           C  \\nATOM   1338  CB  SER A 185      -0.978 -24.171 -43.336  1.00  0.33           C  \\nATOM   1339  O   SER A 185      -2.498 -22.625 -41.162  1.00  0.33           O  \\nATOM   1340  OG  SER A 185      -2.183 -24.731 -42.845  1.00  0.34           O  \\nATOM   1341  N   SER A 186      -2.491 -20.871 -42.316  1.00  0.35           N  \\nATOM   1342  CA  SER A 186      -3.699 -20.157 -42.717  1.00  0.38           C  \\nATOM   1343  C   SER A 186      -4.947 -20.813 -42.136  1.00  0.35           C  \\nATOM   1344  CB  SER A 186      -3.806 -20.098 -44.241  1.00  0.33           C  \\nATOM   1345  O   SER A 186      -5.376 -21.869 -42.607  1.00  0.34           O  \\nATOM   1346  OG  SER A 186      -3.980 -21.395 -44.784  1.00  0.36           O  \\nATOM   1347  N   GLY A 187      -5.169 -20.889 -40.908  1.00  0.34           N  \\nATOM   1348  CA  GLY A 187      -6.542 -20.982 -40.439  1.00  0.36           C  \\nATOM   1349  C   GLY A 187      -7.210 -19.630 -40.277  1.00  0.35           C  \\nATOM   1350  O   GLY A 187      -6.824 -18.840 -39.412  1.00  0.34           O  \\nATOM   1351  N   GLY A 188      -7.624 -18.958 -41.377  1.00  0.36           N  \\nATOM   1352  CA  GLY A 188      -8.792 -18.104 -41.529  1.00  0.38           C  \\nATOM   1353  C   GLY A 188      -9.821 -18.299 -40.432  1.00  0.36           C  \\nATOM   1354  O   GLY A 188     -10.522 -19.312 -40.404  1.00  0.35           O  \\nATOM   1355  N   GLY A 189      -9.627 -17.887 -39.334  1.00  0.38           N  \\nATOM   1356  CA  GLY A 189     -10.744 -17.675 -38.426  1.00  0.40           C  \\nATOM   1357  C   GLY A 189     -11.579 -16.460 -38.781  1.00  0.38           C  \\nATOM   1358  O   GLY A 189     -11.062 -15.344 -38.853  1.00  0.38           O  \\nATOM   1359  N   SER A 190     -12.602 -16.533 -39.626  1.00  0.35           N  \\nATOM   1360  CA  SER A 190     -13.835 -15.757 -39.698  1.00  0.38           C  \\nATOM   1361  C   SER A 190     -14.186 -15.151 -38.343  1.00  0.36           C  \\nATOM   1362  CB  SER A 190     -14.991 -16.629 -40.191  1.00  0.35           C  \\nATOM   1363  O   SER A 190     -14.634 -15.859 -37.438  1.00  0.36           O  \\nATOM   1364  OG  SER A 190     -16.139 -15.841 -40.454  1.00  0.36           O  \\nATOM   1365  N   GLY A 191     -13.503 -14.185 -37.973  1.00  0.39           N  \\nATOM   1366  CA  GLY A 191     -13.953 -13.237 -36.967  1.00  0.40           C  \\nATOM   1367  C   GLY A 191     -14.453 -11.932 -37.558  1.00  0.40           C  \\nATOM   1368  O   GLY A 191     -13.716 -11.241 -38.263  1.00  0.39           O  \\nATOM   1369  N   GLY A 192     -15.662 -11.963 -37.980  1.00  0.38           N  \\nATOM   1370  CA  GLY A 192     -16.556 -10.833 -38.178  1.00  0.40           C  \\nATOM   1371  C   GLY A 192     -15.956  -9.513 -37.734  1.00  0.38           C  \\nATOM   1372  O   GLY A 192     -15.573  -9.359 -36.572  1.00  0.37           O  \\nATOM   1373  N   GLY A 193     -15.233  -8.844 -38.776  1.00  0.37           N  \\nATOM   1374  CA  GLY A 193     -15.020  -7.460 -39.166  1.00  0.37           C  \\nATOM   1375  C   GLY A 193     -15.821  -6.476 -38.335  1.00  0.37           C  \\nATOM   1376  O   GLY A 193     -17.046  -6.406 -38.455  1.00  0.36           O  \\nATOM   1377  N   TYR A 194     -15.333  -6.140 -37.278  1.00  0.40           N  \\nATOM   1378  CA  TYR A 194     -15.768  -4.833 -36.798  1.00  0.46           C  \\nATOM   1379  C   TYR A 194     -14.579  -3.901 -36.601  1.00  0.40           C  \\nATOM   1380  CB  TYR A 194     -16.544  -4.973 -35.485  1.00  0.39           C  \\nATOM   1381  O   TYR A 194     -13.767  -4.103 -35.695  1.00  0.40           O  \\nATOM   1382  CG  TYR A 194     -17.908  -5.599 -35.650  1.00  0.42           C  \\nATOM   1383  CD1 TYR A 194     -18.997  -4.841 -36.074  1.00  0.42           C  \\nATOM   1384  CD2 TYR A 194     -18.111  -6.948 -35.381  1.00  0.43           C  \\nATOM   1385  CE1 TYR A 194     -20.256  -5.412 -36.225  1.00  0.39           C  \\nATOM   1386  CE2 TYR A 194     -19.365  -7.530 -35.529  1.00  0.43           C  \\nATOM   1387  OH  TYR A 194     -21.674  -7.327 -36.099  1.00  0.27           O  \\nATOM   1388  CZ  TYR A 194     -20.430  -6.756 -35.951  1.00  0.37           C  \\nATOM   1389  N   GLY A 195     -13.937  -3.349 -37.900  1.00  0.40           N  \\nATOM   1390  CA  GLY A 195     -13.421  -1.990 -37.948  1.00  0.43           C  \\nATOM   1391  C   GLY A 195     -12.814  -1.629 -39.290  1.00  0.40           C  \\nATOM   1392  O   GLY A 195     -12.080  -2.425 -39.880  1.00  0.39           O  \\nATOM   1393  N   ASN A 196     -13.521  -0.837 -40.126  1.00  0.40           N  \\nATOM   1394  CA  ASN A 196     -13.169   0.089 -41.197  1.00  0.46           C  \\nATOM   1395  C   ASN A 196     -11.851  -0.298 -41.862  1.00  0.41           C  \\nATOM   1396  CB  ASN A 196     -13.094   1.522 -40.665  1.00  0.39           C  \\nATOM   1397  O   ASN A 196     -10.793  -0.236 -41.235  1.00  0.40           O  \\nATOM   1398  CG  ASN A 196     -13.988   2.477 -41.432  1.00  0.40           C  \\nATOM   1399  ND2 ASN A 196     -13.832   3.770 -41.175  1.00  0.34           N  \\nATOM   1400  OD1 ASN A 196     -14.812   2.055 -42.248  1.00  0.42           O  \\nATOM   1401  N   GLN A 197     -11.883  -1.213 -42.735  1.00  0.43           N  \\nATOM   1402  CA  GLN A 197     -11.311  -1.139 -44.075  1.00  0.45           C  \\nATOM   1403  C   GLN A 197     -11.442   0.268 -44.652  1.00  0.43           C  \\nATOM   1404  CB  GLN A 197     -11.984  -2.151 -45.003  1.00  0.40           C  \\nATOM   1405  O   GLN A 197     -12.541   0.700 -45.006  1.00  0.42           O  \\nATOM   1406  CG  GLN A 197     -11.197  -3.443 -45.178  1.00  0.41           C  \\nATOM   1407  CD  GLN A 197     -11.895  -4.438 -46.086  1.00  0.47           C  \\nATOM   1408  NE2 GLN A 197     -11.324  -5.632 -46.208  1.00  0.39           N  \\nATOM   1409  OE1 GLN A 197     -12.940  -4.137 -46.672  1.00  0.51           O  \\nATOM   1410  N   ASP A 198     -10.521   1.154 -44.192  1.00  0.43           N  \\nATOM   1411  CA  ASP A 198      -9.974   2.056 -45.200  1.00  0.46           C  \\nATOM   1412  C   ASP A 198      -8.750   1.444 -45.879  1.00  0.44           C  \\nATOM   1413  CB  ASP A 198      -9.609   3.404 -44.574  1.00  0.41           C  \\nATOM   1414  O   ASP A 198      -7.672   1.380 -45.285  1.00  0.42           O  \\nATOM   1415  CG  ASP A 198     -10.742   4.413 -44.635  1.00  0.41           C  \\nATOM   1416  OD1 ASP A 198     -11.617   4.295 -45.520  1.00  0.44           O  \\nATOM   1417  OD2 ASP A 198     -10.759   5.336 -43.792  1.00  0.49           O  \\nATOM   1418  N   GLN A 199      -8.918   0.489 -46.765  1.00  0.44           N  \\nATOM   1419  CA  GLN A 199      -8.189   0.241 -48.004  1.00  0.46           C  \\nATOM   1420  C   GLN A 199      -8.206   1.471 -48.908  1.00  0.44           C  \\nATOM   1421  CB  GLN A 199      -8.777  -0.962 -48.743  1.00  0.41           C  \\nATOM   1422  O   GLN A 199      -9.190   1.718 -49.609  1.00  0.43           O  \\nATOM   1423  CG  GLN A 199      -8.384  -2.305 -48.142  1.00  0.42           C  \\nATOM   1424  CD  GLN A 199      -8.932  -3.482 -48.926  1.00  0.49           C  \\nATOM   1425  NE2 GLN A 199      -9.158  -4.598 -48.241  1.00  0.39           N  \\nATOM   1426  OE1 GLN A 199      -9.151  -3.389 -50.138  1.00  0.53           O  \\nATOM   1427  N   SER A 200      -7.630   2.521 -48.405  1.00  0.39           N  \\nATOM   1428  CA  SER A 200      -7.096   3.548 -49.294  1.00  0.43           C  \\nATOM   1429  C   SER A 200      -5.585   3.682 -49.139  1.00  0.39           C  \\nATOM   1430  CB  SER A 200      -7.766   4.895 -49.021  1.00  0.36           C  \\nATOM   1431  O   SER A 200      -5.096   4.061 -48.072  1.00  0.38           O  \\nATOM   1432  OG  SER A 200      -7.210   5.510 -47.871  1.00  0.41           O  \\nATOM   1433  N   GLY A 201      -4.790   2.812 -49.740  1.00  0.37           N  \\nATOM   1434  CA  GLY A 201      -3.552   3.307 -50.321  1.00  0.40           C  \\nATOM   1435  C   GLY A 201      -2.530   2.214 -50.568  1.00  0.37           C  \\nATOM   1436  O   GLY A 201      -2.179   1.467 -49.652  1.00  0.35           O  \\nATOM   1437  N   GLY A 202      -2.573   1.712 -51.799  1.00  0.35           N  \\nATOM   1438  CA  GLY A 202      -1.452   1.688 -52.724  1.00  0.38           C  \\nATOM   1439  C   GLY A 202      -0.647   0.404 -52.653  1.00  0.35           C  \\nATOM   1440  O   GLY A 202      -0.170   0.022 -51.582  1.00  0.34           O  \\nATOM   1441  N   GLY A 203      -1.051  -0.697 -53.438  1.00  0.36           N  \\nATOM   1442  CA  GLY A 203      -0.094  -1.287 -54.361  1.00  0.40           C  \\nATOM   1443  C   GLY A 203       0.873  -2.242 -53.687  1.00  0.37           C  \\nATOM   1444  O   GLY A 203       1.777  -1.814 -52.967  1.00  0.36           O  \\nATOM   1445  N   GLY A 204       0.347  -3.680 -53.302  1.00  0.38           N  \\nATOM   1446  CA  GLY A 204       1.208  -4.799 -53.651  1.00  0.41           C  \\nATOM   1447  C   GLY A 204       0.516  -6.144 -53.532  1.00  0.38           C  \\nATOM   1448  O   GLY A 204      -0.075  -6.454 -52.496  1.00  0.37           O  \\nATOM   1449  N   SER A 205      -0.245  -6.803 -54.585  1.00  0.34           N  \\nATOM   1450  CA  SER A 205      -0.352  -8.077 -55.288  1.00  0.39           C  \\nATOM   1451  C   SER A 205       0.200  -9.222 -54.445  1.00  0.35           C  \\nATOM   1452  CB  SER A 205       0.387  -8.016 -56.626  1.00  0.33           C  \\nATOM   1453  O   SER A 205       1.406  -9.291 -54.196  1.00  0.34           O  \\nATOM   1454  OG  SER A 205       0.411  -9.290 -57.246  1.00  0.38           O  \\nATOM   1455  N   GLY A 206      -0.625  -9.702 -53.341  1.00  0.38           N  \\nATOM   1456  CA  GLY A 206      -0.517 -11.087 -52.910  1.00  0.41           C  \\nATOM   1457  C   GLY A 206      -1.752 -11.584 -52.182  1.00  0.38           C  \\nATOM   1458  O   GLY A 206      -2.166 -10.996 -51.180  1.00  0.37           O  \\nATOM   1459  N   GLY A 207      -2.830 -12.198 -53.008  1.00  0.39           N  \\nATOM   1460  CA  GLY A 207      -3.893 -13.189 -53.057  1.00  0.41           C  \\nATOM   1461  C   GLY A 207      -4.133 -13.872 -51.724  1.00  0.40           C  \\nATOM   1462  O   GLY A 207      -3.319 -14.685 -51.281  1.00  0.39           O  \\nATOM   1463  N   TYR A 208      -4.694 -13.299 -50.706  1.00  0.41           N  \\nATOM   1464  CA  TYR A 208      -5.295 -13.985 -49.567  1.00  0.45           C  \\nATOM   1465  C   TYR A 208      -6.707 -14.453 -49.896  1.00  0.41           C  \\nATOM   1466  CB  TYR A 208      -5.321 -13.069 -48.340  1.00  0.39           C  \\nATOM   1467  O   TYR A 208      -7.580 -13.641 -50.212  1.00  0.40           O  \\nATOM   1468  CG  TYR A 208      -3.965 -12.855 -47.713  1.00  0.39           C  \\nATOM   1469  CD1 TYR A 208      -3.419 -13.800 -46.848  1.00  0.38           C  \\nATOM   1470  CD2 TYR A 208      -3.228 -11.707 -47.984  1.00  0.40           C  \\nATOM   1471  CE1 TYR A 208      -2.170 -13.606 -46.266  1.00  0.38           C  \\nATOM   1472  CE2 TYR A 208      -1.978 -11.503 -47.408  1.00  0.41           C  \\nATOM   1473  OH  TYR A 208      -0.222 -12.259 -45.980  1.00  0.29           O  \\nATOM   1474  CZ  TYR A 208      -1.458 -12.456 -46.553  1.00  0.35           C  \\nATOM   1475  N   GLY A 209      -6.849 -15.607 -50.552  1.00  0.39           N  \\nATOM   1476  CA  GLY A 209      -7.943 -16.542 -50.764  1.00  0.41           C  \\nATOM   1477  C   GLY A 209      -8.960 -16.535 -49.639  1.00  0.40           C  \\nATOM   1478  O   GLY A 209      -8.623 -16.828 -48.489  1.00  0.39           O  \\nATOM   1479  N   GLN A 210      -9.925 -15.575 -49.450  1.00  0.42           N  \\nATOM   1480  CA  GLN A 210     -11.282 -15.841 -48.985  1.00  0.44           C  \\nATOM   1481  C   GLN A 210     -11.702 -17.273 -49.303  1.00  0.42           C  \\nATOM   1482  CB  GLN A 210     -12.268 -14.853 -49.611  1.00  0.40           C  \\nATOM   1483  O   GLN A 210     -11.886 -17.627 -50.469  1.00  0.41           O  \\nATOM   1484  CG  GLN A 210     -13.322 -14.337 -48.641  1.00  0.40           C  \\nATOM   1485  CD  GLN A 210     -14.171 -13.227 -49.232  1.00  0.46           C  \\nATOM   1486  NE2 GLN A 210     -13.703 -12.643 -50.330  1.00  0.38           N  \\nATOM   1487  OE1 GLN A 210     -15.240 -12.899 -48.708  1.00  0.51           O  \\nATOM   1488  N   GLN A 211     -11.252 -18.250 -48.516  1.00  0.44           N  \\nATOM   1489  CA  GLN A 211     -12.036 -19.481 -48.515  1.00  0.46           C  \\nATOM   1490  C   GLN A 211     -13.447 -19.234 -47.986  1.00  0.44           C  \\nATOM   1491  CB  GLN A 211     -11.344 -20.559 -47.681  1.00  0.41           C  \\nATOM   1492  O   GLN A 211     -13.627 -18.924 -46.807  1.00  0.43           O  \\nATOM   1493  CG  GLN A 211     -10.658 -21.634 -48.512  1.00  0.41           C  \\nATOM   1494  CD  GLN A 211     -10.029 -22.722 -47.662  1.00  0.46           C  \\nATOM   1495  NE2 GLN A 211      -9.154 -22.326 -46.744  1.00  0.39           N  \\nATOM   1496  OE1 GLN A 211     -10.329 -23.909 -47.827  1.00  0.50           O  \\nATOM   1497  N   ASP A 212     -14.427 -18.849 -48.854  1.00  0.41           N  \\nATOM   1498  CA  ASP A 212     -15.812 -19.269 -49.044  1.00  0.42           C  \\nATOM   1499  C   ASP A 212     -15.959 -20.777 -48.854  1.00  0.41           C  \\nATOM   1500  CB  ASP A 212     -16.311 -18.862 -50.431  1.00  0.39           C  \\nATOM   1501  O   ASP A 212     -15.566 -21.559 -49.723  1.00  0.40           O  \\nATOM   1502  CG  ASP A 212     -17.355 -17.760 -50.386  1.00  0.38           C  \\nATOM   1503  OD1 ASP A 212     -17.985 -17.560 -49.326  1.00  0.42           O  \\nATOM   1504  OD2 ASP A 212     -17.550 -17.087 -51.421  1.00  0.45           O  \\nATOM   1505  N   ARG A 213     -15.756 -21.297 -47.646  1.00  0.44           N  \\nATOM   1506  CA  ARG A 213     -16.341 -22.603 -47.360  1.00  0.44           C  \\nATOM   1507  C   ARG A 213     -17.846 -22.595 -47.608  1.00  0.44           C  \\nATOM   1508  CB  ARG A 213     -16.051 -23.018 -45.916  1.00  0.42           C  \\nATOM   1509  O   ARG A 213     -18.609 -22.042 -46.813  1.00  0.42           O  \\nATOM   1510  CG  ARG A 213     -14.646 -23.559 -45.702  1.00  0.40           C  \\nATOM   1511  CD  ARG A 213     -14.436 -24.031 -44.270  1.00  0.44           C  \\nATOM   1512  NE  ARG A 213     -13.101 -24.589 -44.076  1.00  0.40           N  \\nATOM   1513  NH1 ARG A 213     -13.334 -24.897 -41.799  1.00  0.31           N  \\nATOM   1514  NH2 ARG A 213     -11.379 -25.479 -42.844  1.00  0.26           N  \\nATOM   1515  CZ  ARG A 213     -12.607 -24.987 -42.907  1.00  0.38           C  \\nATOM   1516  N   GLY A 214     -18.289 -22.529 -48.707  1.00  0.38           N  \\nATOM   1517  CA  GLY A 214     -19.369 -23.268 -49.342  1.00  0.39           C  \\nATOM   1518  C   GLY A 214     -19.717 -24.555 -48.618  1.00  0.39           C  \\nATOM   1519  O   GLY A 214     -19.062 -25.581 -48.815  1.00  0.37           O  \\nATOM   1520  N   GLY A 215     -20.200 -24.495 -47.404  1.00  0.40           N  \\nATOM   1521  CA  GLY A 215     -21.018 -25.633 -47.014  1.00  0.42           C  \\nATOM   1522  C   GLY A 215     -22.497 -25.424 -47.279  1.00  0.41           C  \\nATOM   1523  O   GLY A 215     -23.127 -24.561 -46.665  1.00  0.40           O  \\nATOM   1524  N   ARG A 216     -22.981 -25.489 -48.450  1.00  0.39           N  \\nATOM   1525  CA  ARG A 216     -24.216 -26.189 -48.790  1.00  0.39           C  \\nATOM   1526  C   ARG A 216     -24.325 -27.507 -48.029  1.00  0.39           C  \\nATOM   1527  CB  ARG A 216     -24.292 -26.446 -50.296  1.00  0.37           C  \\nATOM   1528  O   ARG A 216     -23.691 -28.497 -48.399  1.00  0.38           O  \\nATOM   1529  CG  ARG A 216     -24.885 -25.292 -51.088  1.00  0.38           C  \\nATOM   1530  CD  ARG A 216     -25.001 -25.625 -52.569  1.00  0.44           C  \\nATOM   1531  NE  ARG A 216     -25.516 -24.495 -53.337  1.00  0.37           N  \\nATOM   1532  NH1 ARG A 216     -25.510 -25.603 -55.364  1.00  0.28           N  \\nATOM   1533  NH2 ARG A 216     -26.216 -23.426 -55.246  1.00  0.23           N  \\nATOM   1534  CZ  ARG A 216     -25.746 -24.511 -54.647  1.00  0.34           C  \\nATOM   1535  N   GLY A 217     -24.633 -27.499 -46.695  1.00  0.39           N  \\nATOM   1536  CA  GLY A 217     -25.359 -28.669 -46.228  1.00  0.41           C  \\nATOM   1537  C   GLY A 217     -26.832 -28.398 -45.983  1.00  0.39           C  \\nATOM   1538  O   GLY A 217     -27.184 -27.513 -45.201  1.00  0.38           O  \\nATOM   1539  N   ARG A 218     -27.615 -28.377 -46.945  1.00  0.33           N  \\nATOM   1540  CA  ARG A 218     -28.885 -29.087 -47.055  1.00  0.35           C  \\nATOM   1541  C   ARG A 218     -28.994 -30.181 -45.998  1.00  0.33           C  \\nATOM   1542  CB  ARG A 218     -29.045 -29.691 -48.452  1.00  0.31           C  \\nATOM   1543  O   ARG A 218     -28.134 -31.061 -45.917  1.00  0.32           O  \\nATOM   1544  CG  ARG A 218     -30.110 -29.012 -49.298  1.00  0.33           C  \\nATOM   1545  CD  ARG A 218     -30.184 -29.608 -50.697  1.00  0.40           C  \\nATOM   1546  NE  ARG A 218     -31.207 -28.954 -51.508  1.00  0.36           N  \\nATOM   1547  NH1 ARG A 218     -31.037 -30.427 -53.280  1.00  0.28           N  \\nATOM   1548  NH2 ARG A 218     -32.529 -28.689 -53.367  1.00  0.23           N  \\nATOM   1549  CZ  ARG A 218     -31.589 -29.358 -52.716  1.00  0.31           C  \\nATOM   1550  N   GLY A 219     -29.528 -29.813 -44.622  1.00  0.33           N  \\nATOM   1551  CA  GLY A 219     -30.471 -30.759 -44.048  1.00  0.38           C  \\nATOM   1552  C   GLY A 219     -29.967 -31.405 -42.772  1.00  0.34           C  \\nATOM   1553  O   GLY A 219     -29.086 -32.266 -42.811  1.00  0.33           O  \\nATOM   1554  N   GLY A 220     -29.851 -30.754 -41.619  1.00  0.34           N  \\nATOM   1555  CA  GLY A 220     -30.455 -31.113 -40.347  1.00  0.37           C  \\nATOM   1556  C   GLY A 220     -30.743 -29.915 -39.463  1.00  0.35           C  \\nATOM   1557  O   GLY A 220     -29.847 -29.119 -39.176  1.00  0.34           O  \\nATOM   1558  N   SER A 221     -31.663 -29.034 -39.837  1.00  0.31           N  \\nATOM   1559  CA  SER A 221     -32.626 -28.333 -38.994  1.00  0.35           C  \\nATOM   1560  C   SER A 221     -32.451 -28.707 -37.526  1.00  0.32           C  \\nATOM   1561  CB  SER A 221     -34.055 -28.642 -39.440  1.00  0.30           C  \\nATOM   1562  O   SER A 221     -32.636 -29.865 -37.148  1.00  0.31           O  \\nATOM   1563  OG  SER A 221     -34.990 -27.866 -38.710  1.00  0.31           O  \\nATOM   1564  N   GLY A 222     -31.247 -28.530 -36.883  1.00  0.34           N  \\nATOM   1565  CA  GLY A 222     -31.432 -28.304 -35.458  1.00  0.36           C  \\nATOM   1566  C   GLY A 222     -30.155 -28.471 -34.656  1.00  0.35           C  \\nATOM   1567  O   GLY A 222     -30.161 -28.327 -33.432  1.00  0.33           O  \\nATOM   1568  N   GLY A 223     -28.858 -28.676 -35.319  1.00  0.34           N  \\nATOM   1569  CA  GLY A 223     -27.707 -28.966 -34.479  1.00  0.36           C  \\nATOM   1570  C   GLY A 223     -26.508 -28.087 -34.785  1.00  0.35           C  \\nATOM   1571  O   GLY A 223     -25.556 -28.033 -34.004  1.00  0.33           O  \\nATOM   1572  N   GLY A 224     -26.531 -27.404 -35.988  1.00  0.35           N  \\nATOM   1573  CA  GLY A 224     -25.313 -26.692 -36.342  1.00  0.36           C  \\nATOM   1574  C   GLY A 224     -25.156 -25.376 -35.606  1.00  0.35           C  \\nATOM   1575  O   GLY A 224     -24.037 -24.961 -35.297  1.00  0.33           O  \\nATOM   1576  N   GLY A 225     -26.404 -24.795 -35.208  1.00  0.38           N  \\nATOM   1577  CA  GLY A 225     -26.358 -23.539 -34.476  1.00  0.39           C  \\nATOM   1578  C   GLY A 225     -25.790 -23.685 -33.077  1.00  0.38           C  \\nATOM   1579  O   GLY A 225     -24.970 -22.871 -32.647  1.00  0.37           O  \\nATOM   1580  N   GLN A 226     -26.146 -24.861 -32.465  1.00  0.40           N  \\nATOM   1581  CA  GLN A 226     -25.655 -25.100 -31.111  1.00  0.41           C  \\nATOM   1582  C   GLN A 226     -24.153 -25.370 -31.110  1.00  0.40           C  \\nATOM   1583  CB  GLN A 226     -26.400 -26.271 -30.469  1.00  0.38           C  \\nATOM   1584  O   GLN A 226     -23.424 -24.846 -30.266  1.00  0.39           O  \\nATOM   1585  CG  GLN A 226     -27.560 -25.848 -29.579  1.00  0.37           C  \\nATOM   1586  CD  GLN A 226     -28.215 -27.020 -28.872  1.00  0.43           C  \\nATOM   1587  NE2 GLN A 226     -29.057 -26.723 -27.887  1.00  0.36           N  \\nATOM   1588  OE1 GLN A 226     -27.967 -28.182 -29.208  1.00  0.46           O  \\nATOM   1589  N   GLN A 227     -23.649 -26.253 -32.078  1.00  0.44           N  \\nATOM   1590  CA  GLN A 227     -22.220 -26.545 -32.105  1.00  0.44           C  \\nATOM   1591  C   GLN A 227     -21.407 -25.287 -32.399  1.00  0.44           C  \\nATOM   1592  CB  GLN A 227     -21.911 -27.624 -33.143  1.00  0.42           C  \\nATOM   1593  O   GLN A 227     -20.390 -25.034 -31.750  1.00  0.43           O  \\nATOM   1594  CG  GLN A 227     -21.678 -29.005 -32.545  1.00  0.41           C  \\nATOM   1595  CD  GLN A 227     -21.363 -30.054 -33.595  1.00  0.46           C  \\nATOM   1596  NE2 GLN A 227     -21.006 -31.253 -33.146  1.00  0.38           N  \\nATOM   1597  OE1 GLN A 227     -21.441 -29.790 -34.799  1.00  0.47           O  \\nATOM   1598  N   GLN A 228     -21.820 -24.463 -33.382  1.00  0.43           N  \\nATOM   1599  CA  GLN A 228     -21.113 -23.215 -33.649  1.00  0.44           C  \\nATOM   1600  C   GLN A 228     -21.169 -22.282 -32.443  1.00  0.43           C  \\nATOM   1601  CB  GLN A 228     -21.698 -22.519 -34.879  1.00  0.42           C  \\nATOM   1602  O   GLN A 228     -20.162 -21.671 -32.077  1.00  0.43           O  \\nATOM   1603  CG  GLN A 228     -21.025 -22.913 -36.187  1.00  0.41           C  \\nATOM   1604  CD  GLN A 228     -21.617 -22.202 -37.389  1.00  0.45           C  \\nATOM   1605  NE2 GLN A 228     -21.169 -22.580 -38.582  1.00  0.39           N  \\nATOM   1606  OE1 GLN A 228     -22.472 -21.322 -37.248  1.00  0.48           O  \\nATOM   1607  N   GLN A 229     -22.348 -22.242 -31.825  1.00  0.42           N  \\nATOM   1608  CA  GLN A 229     -22.467 -21.450 -30.605  1.00  0.44           C  \\nATOM   1609  C   GLN A 229     -21.510 -21.952 -29.527  1.00  0.42           C  \\nATOM   1610  CB  GLN A 229     -23.905 -21.477 -30.085  1.00  0.41           C  \\nATOM   1611  O   GLN A 229     -20.833 -21.157 -28.872  1.00  0.42           O  \\nATOM   1612  CG  GLN A 229     -24.500 -20.095 -29.851  1.00  0.40           C  \\nATOM   1613  CD  GLN A 229     -25.942 -20.149 -29.381  1.00  0.45           C  \\nATOM   1614  NE2 GLN A 229     -26.500 -18.990 -29.046  1.00  0.38           N  \\nATOM   1615  OE1 GLN A 229     -26.549 -21.222 -29.321  1.00  0.47           O  \\nATOM   1616  N   GLN A 230     -21.492 -23.276 -29.322  1.00  0.45           N  \\nATOM   1617  CA  GLN A 230     -20.569 -23.851 -28.349  1.00  0.45           C  \\nATOM   1618  C   GLN A 230     -19.121 -23.536 -28.711  1.00  0.45           C  \\nATOM   1619  CB  GLN A 230     -20.768 -25.364 -28.248  1.00  0.43           C  \\nATOM   1620  O   GLN A 230     -18.318 -23.191 -27.841  1.00  0.45           O  \\nATOM   1621  CG  GLN A 230     -21.991 -25.771 -27.437  1.00  0.41           C  \\nATOM   1622  CD  GLN A 230     -22.165 -27.275 -27.353  1.00  0.45           C  \\nATOM   1623  NE2 GLN A 230     -23.170 -27.715 -26.602  1.00  0.39           N  \\nATOM   1624  OE1 GLN A 230     -21.404 -28.037 -27.958  1.00  0.49           O  \\nATOM   1625  N   GLN A 231     -18.785 -23.713 -30.031  1.00  0.45           N  \\nATOM   1626  CA  GLN A 231     -17.424 -23.400 -30.452  1.00  0.46           C  \\nATOM   1627  C   GLN A 231     -17.096 -21.930 -30.206  1.00  0.45           C  \\nATOM   1628  CB  GLN A 231     -17.226 -23.742 -31.930  1.00  0.43           C  \\nATOM   1629  O   GLN A 231     -16.022 -21.605 -29.696  1.00  0.45           O  \\nATOM   1630  CG  GLN A 231     -15.934 -24.492 -32.220  1.00  0.42           C  \\nATOM   1631  CD  GLN A 231     -15.789 -24.869 -33.683  1.00  0.46           C  \\nATOM   1632  NE2 GLN A 231     -14.615 -25.368 -34.053  1.00  0.40           N  \\nATOM   1633  OE1 GLN A 231     -16.725 -24.713 -34.473  1.00  0.48           O  \\nATOM   1634  N   TRP A 232     -18.028 -21.021 -30.515  1.00  0.33           N  \\nATOM   1635  CA  TRP A 232     -17.841 -19.599 -30.252  1.00  0.42           C  \\nATOM   1636  C   TRP A 232     -17.707 -19.335 -28.756  1.00  0.34           C  \\nATOM   1637  CB  TRP A 232     -19.006 -18.787 -30.823  1.00  0.35           C  \\nATOM   1638  O   TRP A 232     -16.817 -18.596 -28.326  1.00  0.34           O  \\nATOM   1639  CG  TRP A 232     -18.791 -18.322 -32.232  1.00  0.36           C  \\nATOM   1640  CD1 TRP A 232     -18.821 -19.086 -33.366  1.00  0.33           C  \\nATOM   1641  CD2 TRP A 232     -18.508 -16.985 -32.657  1.00  0.36           C  \\nATOM   1642  CE2 TRP A 232     -18.379 -17.012 -34.063  1.00  0.34           C  \\nATOM   1643  CE3 TRP A 232     -18.352 -15.766 -31.984  1.00  0.44           C  \\nATOM   1644  NE1 TRP A 232     -18.574 -18.304 -34.470  1.00  0.42           N  \\nATOM   1645  CH2 TRP A 232     -17.952 -14.684 -34.126  1.00  0.35           C  \\nATOM   1646  CZ2 TRP A 232     -18.100 -15.864 -34.809  1.00  0.36           C  \\nATOM   1647  CZ3 TRP A 232     -18.075 -14.625 -32.728  1.00  0.34           C  \\nATOM   1648  N   GLN A 233     -18.610 -19.933 -27.969  1.00  0.47           N  \\nATOM   1649  CA  GLN A 233     -18.523 -19.807 -26.518  1.00  0.48           C  \\nATOM   1650  C   GLN A 233     -17.167 -20.282 -26.005  1.00  0.47           C  \\nATOM   1651  CB  GLN A 233     -19.646 -20.596 -25.842  1.00  0.46           C  \\nATOM   1652  O   GLN A 233     -16.545 -19.619 -25.172  1.00  0.48           O  \\nATOM   1653  CG  GLN A 233     -20.979 -19.861 -25.807  1.00  0.44           C  \\nATOM   1654  CD  GLN A 233     -22.087 -20.684 -25.176  1.00  0.48           C  \\nATOM   1655  NE2 GLN A 233     -23.261 -20.081 -25.025  1.00  0.41           N  \\nATOM   1656  OE1 GLN A 233     -21.889 -21.852 -24.828  1.00  0.51           O  \\nATOM   1657  N   TRP A 234     -16.757 -21.410 -26.607  1.00  0.33           N  \\nATOM   1658  CA  TRP A 234     -15.468 -21.963 -26.204  1.00  0.42           C  \\nATOM   1659  C   TRP A 234     -14.328 -21.032 -26.601  1.00  0.33           C  \\nATOM   1660  CB  TRP A 234     -15.256 -23.345 -26.828  1.00  0.34           C  \\nATOM   1661  O   TRP A 234     -13.451 -20.732 -25.787  1.00  0.34           O  \\nATOM   1662  CG  TRP A 234     -14.535 -24.312 -25.937  1.00  0.35           C  \\nATOM   1663  CD1 TRP A 234     -15.054 -24.996 -24.873  1.00  0.32           C  \\nATOM   1664  CD2 TRP A 234     -13.161 -24.698 -26.032  1.00  0.35           C  \\nATOM   1665  CE2 TRP A 234     -12.914 -25.621 -24.991  1.00  0.33           C  \\nATOM   1666  CE3 TRP A 234     -12.113 -24.353 -26.896  1.00  0.44           C  \\nATOM   1667  NE1 TRP A 234     -14.083 -25.785 -24.300  1.00  0.41           N  \\nATOM   1668  CH2 TRP A 234     -10.653 -25.850 -25.653  1.00  0.33           C  \\nATOM   1669  CZ2 TRP A 234     -11.660 -26.204 -24.793  1.00  0.35           C  \\nATOM   1670  CZ3 TRP A 234     -10.866 -24.935 -26.697  1.00  0.34           C  \\nATOM   1671  N   GLN A 235     -14.292 -20.563 -27.804  1.00  0.50           N  \\nATOM   1672  CA  GLN A 235     -13.265 -19.643 -28.278  1.00  0.51           C  \\nATOM   1673  C   GLN A 235     -13.270 -18.348 -27.470  1.00  0.50           C  \\nATOM   1674  CB  GLN A 235     -13.463 -19.335 -29.763  1.00  0.48           C  \\nATOM   1675  O   GLN A 235     -12.213 -17.847 -27.084  1.00  0.50           O  \\nATOM   1676  CG  GLN A 235     -13.025 -20.463 -30.688  1.00  0.46           C  \\nATOM   1677  CD  GLN A 235     -13.305 -20.166 -32.149  1.00  0.49           C  \\nATOM   1678  NE2 GLN A 235     -12.736 -20.973 -33.038  1.00  0.42           N  \\nATOM   1679  OE1 GLN A 235     -14.027 -19.219 -32.477  1.00  0.54           O  \\nATOM   1680  N   TRP A 236     -14.508 -17.859 -27.234  1.00  0.38           N  \\nATOM   1681  CA  TRP A 236     -14.655 -16.664 -26.410  1.00  0.45           C  \\nATOM   1682  C   TRP A 236     -14.111 -16.902 -25.005  1.00  0.38           C  \\nATOM   1683  CB  TRP A 236     -16.123 -16.237 -26.337  1.00  0.38           C  \\nATOM   1684  O   TRP A 236     -13.352 -16.085 -24.479  1.00  0.39           O  \\nATOM   1685  CG  TRP A 236     -16.343 -14.924 -25.648  1.00  0.37           C  \\nATOM   1686  CD1 TRP A 236     -16.167 -13.676 -26.178  1.00  0.32           C  \\nATOM   1687  CD2 TRP A 236     -16.774 -14.729 -24.298  1.00  0.36           C  \\nATOM   1688  CE2 TRP A 236     -16.839 -13.336 -24.077  1.00  0.35           C  \\nATOM   1689  CE3 TRP A 236     -17.113 -15.599 -23.252  1.00  0.51           C  \\nATOM   1690  NE1 TRP A 236     -16.464 -12.716 -25.238  1.00  0.48           N  \\nATOM   1691  CH2 TRP A 236     -17.557 -13.664 -21.845  1.00  0.41           C  \\nATOM   1692  CZ2 TRP A 236     -17.231 -12.792 -22.851  1.00  0.52           C  \\nATOM   1693  CZ3 TRP A 236     -17.502 -15.055 -22.034  1.00  0.37           C  \\nATOM   1694  N   TRP A 237     -14.401 -18.058 -24.383  1.00  0.40           N  \\nATOM   1695  CA  TRP A 237     -13.901 -18.441 -23.066  1.00  0.47           C  \\nATOM   1696  C   TRP A 237     -12.379 -18.529 -23.066  1.00  0.40           C  \\nATOM   1697  CB  TRP A 237     -14.502 -19.781 -22.632  1.00  0.40           C  \\nATOM   1698  O   TRP A 237     -11.720 -18.005 -22.164  1.00  0.41           O  \\nATOM   1699  CG  TRP A 237     -14.429 -20.030 -21.155  1.00  0.39           C  \\nATOM   1700  CD1 TRP A 237     -15.165 -19.418 -20.178  1.00  0.34           C  \\nATOM   1701  CD2 TRP A 237     -13.568 -20.957 -20.487  1.00  0.37           C  \\nATOM   1702  CE2 TRP A 237     -13.836 -20.855 -19.104  1.00  0.36           C  \\nATOM   1703  CE3 TRP A 237     -12.595 -21.866 -20.925  1.00  0.49           C  \\nATOM   1704  NE1 TRP A 237     -14.813 -19.910 -18.943  1.00  0.46           N  \\nATOM   1705  CH2 TRP A 237     -12.217 -22.509 -18.609  1.00  0.39           C  \\nATOM   1706  CZ2 TRP A 237     -13.164 -21.628 -18.154  1.00  0.47           C  \\nATOM   1707  CZ3 TRP A 237     -11.928 -22.635 -19.978  1.00  0.36           C  \\nATOM   1708  N   GLN A 238     -11.765 -19.180 -24.093  1.00  0.51           N  \\nATOM   1709  CA  GLN A 238     -10.315 -19.265 -24.234  1.00  0.52           C  \\nATOM   1710  C   GLN A 238      -9.689 -17.876 -24.316  1.00  0.51           C  \\nATOM   1711  CB  GLN A 238      -9.942 -20.083 -25.472  1.00  0.49           C  \\nATOM   1712  O   GLN A 238      -8.669 -17.610 -23.676  1.00  0.51           O  \\nATOM   1713  CG  GLN A 238      -9.993 -21.588 -25.252  1.00  0.47           C  \\nATOM   1714  CD  GLN A 238      -9.233 -22.362 -26.313  1.00  0.51           C  \\nATOM   1715  NE2 GLN A 238      -8.825 -23.582 -25.979  1.00  0.43           N  \\nATOM   1716  OE1 GLN A 238      -9.016 -21.868 -27.424  1.00  0.53           O  \\nATOM   1717  N   GLN A 239     -10.375 -16.988 -25.089  1.00  0.54           N  \\nATOM   1718  CA  GLN A 239      -9.884 -15.619 -25.206  1.00  0.54           C  \\nATOM   1719  C   GLN A 239      -9.931 -14.901 -23.860  1.00  0.53           C  \\nATOM   1720  CB  GLN A 239     -10.698 -14.843 -26.243  1.00  0.51           C  \\nATOM   1721  O   GLN A 239      -8.972 -14.229 -23.475  1.00  0.53           O  \\nATOM   1722  CG  GLN A 239     -10.050 -14.792 -27.620  1.00  0.49           C  \\nATOM   1723  CD  GLN A 239     -10.909 -14.076 -28.646  1.00  0.51           C  \\nATOM   1724  NE2 GLN A 239     -10.411 -13.983 -29.874  1.00  0.43           N  \\nATOM   1725  OE1 GLN A 239     -12.011 -13.612 -28.337  1.00  0.54           O  \\nATOM   1726  N   GLN A 240     -11.079 -15.101 -23.130  1.00  0.54           N  \\nATOM   1727  CA  GLN A 240     -11.200 -14.517 -21.798  1.00  0.54           C  \\nATOM   1728  C   GLN A 240     -10.116 -15.048 -20.865  1.00  0.53           C  \\nATOM   1729  CB  GLN A 240     -12.583 -14.800 -21.211  1.00  0.51           C  \\nATOM   1730  O   GLN A 240      -9.484 -14.279 -20.137  1.00  0.53           O  \\nATOM   1731  CG  GLN A 240     -13.714 -14.067 -21.920  1.00  0.49           C  \\nATOM   1732  CD  GLN A 240     -13.600 -12.558 -21.804  1.00  0.52           C  \\nATOM   1733  NE2 GLN A 240     -14.139 -11.846 -22.788  1.00  0.43           N  \\nATOM   1734  OE1 GLN A 240     -13.033 -12.038 -20.838  1.00  0.53           O  \\nATOM   1735  N   GLN A 241      -9.919 -16.438 -20.857  1.00  0.53           N  \\nATOM   1736  CA  GLN A 241      -8.866 -17.033 -20.041  1.00  0.53           C  \\nATOM   1737  C   GLN A 241      -7.497 -16.470 -20.412  1.00  0.52           C  \\nATOM   1738  CB  GLN A 241      -8.866 -18.555 -20.191  1.00  0.50           C  \\nATOM   1739  O   GLN A 241      -6.704 -16.125 -19.534  1.00  0.52           O  \\nATOM   1740  CG  GLN A 241     -10.084 -19.234 -19.578  1.00  0.48           C  \\nATOM   1741  CD  GLN A 241     -10.093 -19.165 -18.062  1.00  0.51           C  \\nATOM   1742  NE2 GLN A 241     -11.219 -19.530 -17.459  1.00  0.43           N  \\nATOM   1743  OE1 GLN A 241      -9.096 -18.787 -17.438  1.00  0.52           O  \\nATOM   1744  N   GLN A 242      -7.241 -16.384 -21.790  1.00  0.52           N  \\nATOM   1745  CA  GLN A 242      -5.973 -15.821 -22.241  1.00  0.52           C  \\nATOM   1746  C   GLN A 242      -5.808 -14.382 -21.761  1.00  0.52           C  \\nATOM   1747  CB  GLN A 242      -5.870 -15.881 -23.766  1.00  0.50           C  \\nATOM   1748  O   GLN A 242      -4.736 -14.000 -21.287  1.00  0.52           O  \\nATOM   1749  CG  GLN A 242      -5.403 -17.229 -24.299  1.00  0.48           C  \\nATOM   1750  CD  GLN A 242      -5.323 -17.266 -25.813  1.00  0.51           C  \\nATOM   1751  NE2 GLN A 242      -5.007 -18.434 -26.362  1.00  0.43           N  \\nATOM   1752  OE1 GLN A 242      -5.545 -16.253 -26.485  1.00  0.53           O  \\nATOM   1753  N   GLN A 243      -6.911 -13.594 -21.908  1.00  0.53           N  \\nATOM   1754  CA  GLN A 243      -6.884 -12.218 -21.424  1.00  0.53           C  \\nATOM   1755  C   GLN A 243      -6.655 -12.170 -19.916  1.00  0.52           C  \\nATOM   1756  CB  GLN A 243      -8.185 -11.497 -21.780  1.00  0.50           C  \\nATOM   1757  O   GLN A 243      -5.868 -11.356 -19.428  1.00  0.52           O  \\nATOM   1758  CG  GLN A 243      -8.247 -11.019 -23.225  1.00  0.48           C  \\nATOM   1759  CD  GLN A 243      -9.542 -10.298 -23.550  1.00  0.51           C  \\nATOM   1760  NE2 GLN A 243      -9.697  -9.895 -24.806  1.00  0.43           N  \\nATOM   1761  OE1 GLN A 243     -10.396 -10.106 -22.679  1.00  0.53           O  \\nATOM   1762  N   GLN A 244      -7.385 -13.126 -19.125  1.00  0.52           N  \\nATOM   1763  CA  GLN A 244      -7.176 -13.220 -17.684  1.00  0.52           C  \\nATOM   1764  C   GLN A 244      -5.739 -13.618 -17.361  1.00  0.52           C  \\nATOM   1765  CB  GLN A 244      -8.151 -14.223 -17.065  1.00  0.49           C  \\nATOM   1766  O   GLN A 244      -5.114 -13.039 -16.470  1.00  0.52           O  \\nATOM   1767  CG  GLN A 244      -9.549 -13.664 -16.839  1.00  0.47           C  \\nATOM   1768  CD  GLN A 244     -10.464 -14.641 -16.125  1.00  0.50           C  \\nATOM   1769  NE2 GLN A 244     -11.672 -14.193 -15.801  1.00  0.41           N  \\nATOM   1770  OE1 GLN A 244     -10.088 -15.789 -15.868  1.00  0.51           O  \\nATOM   1771  N   TRP A 245      -5.133 -14.648 -18.110  1.00  0.43           N  \\nATOM   1772  CA  TRP A 245      -3.744 -15.054 -17.923  1.00  0.50           C  \\nATOM   1773  C   TRP A 245      -2.794 -13.899 -18.223  1.00  0.43           C  \\nATOM   1774  CB  TRP A 245      -3.408 -16.252 -18.815  1.00  0.43           C  \\nATOM   1775  O   TRP A 245      -1.847 -13.654 -17.473  1.00  0.44           O  \\nATOM   1776  CG  TRP A 245      -3.668 -17.582 -18.172  1.00  0.42           C  \\nATOM   1777  CD1 TRP A 245      -4.853 -18.032 -17.661  1.00  0.36           C  \\nATOM   1778  CD2 TRP A 245      -2.718 -18.632 -17.969  1.00  0.39           C  \\nATOM   1779  CE2 TRP A 245      -3.397 -19.691 -17.328  1.00  0.36           C  \\nATOM   1780  CE3 TRP A 245      -1.357 -18.781 -18.267  1.00  0.44           C  \\nATOM   1781  NE1 TRP A 245      -4.697 -19.300 -17.151  1.00  0.41           N  \\nATOM   1782  CH2 TRP A 245      -1.429 -21.004 -17.283  1.00  0.33           C  \\nATOM   1783  CZ2 TRP A 245      -2.760 -20.885 -16.980  1.00  0.35           C  \\nATOM   1784  CZ3 TRP A 245      -0.725 -19.969 -17.920  1.00  0.34           C  \\nATOM   1785  N   TRP A 246      -3.035 -13.148 -19.418  1.00  0.49           N  \\nATOM   1786  CA  TRP A 246      -2.218 -11.985 -19.747  1.00  0.56           C  \\nATOM   1787  C   TRP A 246      -2.290 -10.938 -18.640  1.00  0.48           C  \\nATOM   1788  CB  TRP A 246      -2.665 -11.372 -21.077  1.00  0.48           C  \\nATOM   1789  O   TRP A 246      -1.267 -10.380 -18.237  1.00  0.49           O  \\nATOM   1790  CG  TRP A 246      -1.814 -11.766 -22.246  1.00  0.48           C  \\nATOM   1791  CD1 TRP A 246      -1.497 -13.035 -22.644  1.00  0.42           C  \\nATOM   1792  CD2 TRP A 246      -1.166 -10.883 -23.167  1.00  0.43           C  \\nATOM   1793  CE2 TRP A 246      -0.473 -11.686 -24.099  1.00  0.41           C  \\nATOM   1794  CE3 TRP A 246      -1.105  -9.489 -23.295  1.00  0.46           C  \\nATOM   1795  NE1 TRP A 246      -0.691 -12.994 -23.758  1.00  0.43           N  \\nATOM   1796  CH2 TRP A 246       0.316  -9.774 -25.249  1.00  0.35           C  \\nATOM   1797  CZ2 TRP A 246       0.273 -11.140 -25.147  1.00  0.38           C  \\nATOM   1798  CZ3 TRP A 246      -0.362  -8.947 -24.338  1.00  0.35           C  \\nATOM   1799  N   TRP A 247      -3.586 -10.683 -18.105  1.00  0.49           N  \\nATOM   1800  CA  TRP A 247      -3.758  -9.748 -16.998  1.00  0.57           C  \\nATOM   1801  C   TRP A 247      -2.986 -10.213 -15.768  1.00  0.49           C  \\nATOM   1802  CB  TRP A 247      -5.241  -9.587 -16.655  1.00  0.49           C  \\nATOM   1803  O   TRP A 247      -2.291  -9.421 -15.127  1.00  0.49           O  \\nATOM   1804  CG  TRP A 247      -5.879  -8.376 -17.266  1.00  0.49           C  \\nATOM   1805  CD1 TRP A 247      -6.067  -8.125 -18.597  1.00  0.43           C  \\nATOM   1806  CD2 TRP A 247      -6.410  -7.246 -16.566  1.00  0.44           C  \\nATOM   1807  CE2 TRP A 247      -6.905  -6.348 -17.537  1.00  0.41           C  \\nATOM   1808  CE3 TRP A 247      -6.516  -6.906 -15.211  1.00  0.46           C  \\nATOM   1809  NE1 TRP A 247      -6.684  -6.907 -18.767  1.00  0.43           N  \\nATOM   1810  CH2 TRP A 247      -7.589  -4.822 -15.862  1.00  0.35           C  \\nATOM   1811  CZ2 TRP A 247      -7.498  -5.130 -17.195  1.00  0.38           C  \\nATOM   1812  CZ3 TRP A 247      -7.106  -5.693 -14.872  1.00  0.35           C  \\nATOM   1813  N   TRP A 248      -3.126 -11.602 -15.417  1.00  0.49           N  \\nATOM   1814  CA  TRP A 248      -2.391 -12.178 -14.296  1.00  0.56           C  \\nATOM   1815  C   TRP A 248      -0.887 -12.029 -14.496  1.00  0.49           C  \\nATOM   1816  CB  TRP A 248      -2.751 -13.656 -14.120  1.00  0.49           C  \\nATOM   1817  O   TRP A 248      -0.170 -11.617 -13.581  1.00  0.49           O  \\nATOM   1818  CG  TRP A 248      -3.704 -13.921 -12.993  1.00  0.48           C  \\nATOM   1819  CD1 TRP A 248      -5.046 -13.661 -12.971  1.00  0.40           C  \\nATOM   1820  CD2 TRP A 248      -3.385 -14.493 -11.721  1.00  0.42           C  \\nATOM   1821  CE2 TRP A 248      -4.584 -14.552 -10.977  1.00  0.40           C  \\nATOM   1822  CE3 TRP A 248      -2.200 -14.964 -11.139  1.00  0.47           C  \\nATOM   1823  NE1 TRP A 248      -5.581 -14.038 -11.761  1.00  0.42           N  \\nATOM   1824  CH2 TRP A 248      -3.459 -15.517  -9.132  1.00  0.33           C  \\nATOM   1825  CZ2 TRP A 248      -4.631 -15.063  -9.678  1.00  0.37           C  \\nATOM   1826  CZ3 TRP A 248      -2.250 -15.473  -9.846  1.00  0.35           C  \\nATOM   1827  N   TRP A 249      -0.395 -12.408 -15.754  1.00  0.49           N  \\nATOM   1828  CA  TRP A 249       1.024 -12.273 -16.063  1.00  0.55           C  \\nATOM   1829  C   TRP A 249       1.465 -10.816 -15.969  1.00  0.48           C  \\nATOM   1830  CB  TRP A 249       1.326 -12.822 -17.460  1.00  0.48           C  \\nATOM   1831  O   TRP A 249       2.502 -10.511 -15.375  1.00  0.49           O  \\nATOM   1832  CG  TRP A 249       2.496 -13.758 -17.507  1.00  0.48           C  \\nATOM   1833  CD1 TRP A 249       2.489 -15.100 -17.242  1.00  0.41           C  \\nATOM   1834  CD2 TRP A 249       3.847 -13.421 -17.834  1.00  0.43           C  \\nATOM   1835  CE2 TRP A 249       4.607 -14.609 -17.749  1.00  0.41           C  \\nATOM   1836  CE3 TRP A 249       4.490 -12.229 -18.193  1.00  0.46           C  \\nATOM   1837  NE1 TRP A 249       3.755 -15.617 -17.386  1.00  0.42           N  \\nATOM   1838  CH2 TRP A 249       6.583 -13.458 -18.358  1.00  0.33           C  \\nATOM   1839  CZ2 TRP A 249       5.979 -14.638 -18.009  1.00  0.36           C  \\nATOM   1840  CZ3 TRP A 249       5.856 -12.260 -18.452  1.00  0.34           C  \\nATOM   1841  N   TRP A 250       0.644  -9.902 -16.651  1.00  0.52           N  \\nATOM   1842  CA  TRP A 250       0.939  -8.477 -16.545  1.00  0.58           C  \\nATOM   1843  C   TRP A 250       0.970  -8.035 -15.086  1.00  0.51           C  \\nATOM   1844  CB  TRP A 250      -0.095  -7.655 -17.319  1.00  0.52           C  \\nATOM   1845  O   TRP A 250       1.891  -7.331 -14.664  1.00  0.52           O  \\nATOM   1846  CG  TRP A 250       0.443  -6.372 -17.879  1.00  0.50           C  \\nATOM   1847  CD1 TRP A 250       1.192  -6.217 -19.012  1.00  0.44           C  \\nATOM   1848  CD2 TRP A 250       0.276  -5.063 -17.326  1.00  0.45           C  \\nATOM   1849  CE2 TRP A 250       0.951  -4.160 -18.178  1.00  0.43           C  \\nATOM   1850  CE3 TRP A 250      -0.381  -4.565 -16.193  1.00  0.49           C  \\nATOM   1851  NE1 TRP A 250       1.501  -4.889 -19.197  1.00  0.44           N  \\nATOM   1852  CH2 TRP A 250       0.338  -2.325 -16.815  1.00  0.36           C  \\nATOM   1853  CZ2 TRP A 250       0.988  -2.786 -17.930  1.00  0.40           C  \\nATOM   1854  CZ3 TRP A 250      -0.342  -3.197 -15.948  1.00  0.36           C  \\nATOM   1855  N   TRP A 251      -0.052  -8.515 -14.289  1.00  0.50           N  \\nATOM   1856  CA  TRP A 251      -0.088  -8.209 -12.863  1.00  0.57           C  \\nATOM   1857  C   TRP A 251       1.137  -8.777 -12.153  1.00  0.50           C  \\nATOM   1858  CB  TRP A 251      -1.365  -8.763 -12.226  1.00  0.50           C  \\nATOM   1859  O   TRP A 251       1.779  -8.085 -11.359  1.00  0.50           O  \\nATOM   1860  CG  TRP A 251      -2.054  -7.800 -11.306  1.00  0.50           C  \\nATOM   1861  CD1 TRP A 251      -2.921  -6.802 -11.656  1.00  0.43           C  \\nATOM   1862  CD2 TRP A 251      -1.928  -7.740  -9.882  1.00  0.44           C  \\nATOM   1863  CE2 TRP A 251      -2.748  -6.682  -9.433  1.00  0.42           C  \\nATOM   1864  CE3 TRP A 251      -1.200  -8.481  -8.940  1.00  0.47           C  \\nATOM   1865  NE1 TRP A 251      -3.343  -6.127 -10.534  1.00  0.43           N  \\nATOM   1866  CH2 TRP A 251      -2.139  -7.086  -7.182  1.00  0.34           C  \\nATOM   1867  CZ2 TRP A 251      -2.861  -6.346  -8.082  1.00  0.37           C  \\nATOM   1868  CZ3 TRP A 251      -1.314  -8.145  -7.596  1.00  0.35           C  \\nATOM   1869  N   TRP A 252       1.486 -10.134 -12.395  1.00  0.46           N  \\nATOM   1870  CA  TRP A 252       2.668 -10.758 -11.810  1.00  0.54           C  \\nATOM   1871  C   TRP A 252       3.935 -10.018 -12.224  1.00  0.46           C  \\nATOM   1872  CB  TRP A 252       2.761 -12.229 -12.225  1.00  0.47           C  \\nATOM   1873  O   TRP A 252       4.802  -9.745 -11.391  1.00  0.47           O  \\nATOM   1874  CG  TRP A 252       2.978 -13.172 -11.080  1.00  0.47           C  \\nATOM   1875  CD1 TRP A 252       2.030 -13.664 -10.227  1.00  0.41           C  \\nATOM   1876  CD2 TRP A 252       4.226 -13.733 -10.661  1.00  0.42           C  \\nATOM   1877  CE2 TRP A 252       3.959 -14.559  -9.546  1.00  0.40           C  \\nATOM   1878  CE3 TRP A 252       5.545 -13.619 -11.121  1.00  0.45           C  \\nATOM   1879  NE1 TRP A 252       2.614 -14.499  -9.302  1.00  0.41           N  \\nATOM   1880  CH2 TRP A 252       6.246 -15.138  -9.354  1.00  0.31           C  \\nATOM   1881  CZ2 TRP A 252       4.965 -15.267  -8.884  1.00  0.34           C  \\nATOM   1882  CZ3 TRP A 252       6.544 -14.325 -10.461  1.00  0.34           C  \\nATOM   1883  N   TRP A 253       4.082  -9.669 -13.552  1.00  0.46           N  \\nATOM   1884  CA  TRP A 253       5.232  -8.901 -14.020  1.00  0.54           C  \\nATOM   1885  C   TRP A 253       5.263  -7.519 -13.376  1.00  0.46           C  \\nATOM   1886  CB  TRP A 253       5.206  -8.767 -15.545  1.00  0.47           C  \\nATOM   1887  O   TRP A 253       6.319  -7.053 -12.942  1.00  0.47           O  \\nATOM   1888  CG  TRP A 253       6.433  -9.296 -16.224  1.00  0.48           C  \\nATOM   1889  CD1 TRP A 253       6.756 -10.607 -16.439  1.00  0.42           C  \\nATOM   1890  CD2 TRP A 253       7.505  -8.524 -16.773  1.00  0.43           C  \\nATOM   1891  CE2 TRP A 253       8.445  -9.433 -17.306  1.00  0.41           C  \\nATOM   1892  CE3 TRP A 253       7.762  -7.149 -16.866  1.00  0.46           C  \\nATOM   1893  NE1 TRP A 253       7.965 -10.696 -17.090  1.00  0.41           N  \\nATOM   1894  CH2 TRP A 253       9.851  -7.661 -18.003  1.00  0.32           C  \\nATOM   1895  CZ2 TRP A 253       9.624  -9.011 -17.925  1.00  0.35           C  \\nATOM   1896  CZ3 TRP A 253       8.936  -6.731 -17.482  1.00  0.34           C  \\nATOM   1897  N   TRP A 254       4.048  -6.820 -13.297  1.00  0.48           N  \\nATOM   1898  CA  TRP A 254       3.955  -5.539 -12.603  1.00  0.57           C  \\nATOM   1899  C   TRP A 254       4.371  -5.680 -11.143  1.00  0.48           C  \\nATOM   1900  CB  TRP A 254       2.531  -4.982 -12.690  1.00  0.49           C  \\nATOM   1901  O   TRP A 254       5.096  -4.835 -10.613  1.00  0.49           O  \\nATOM   1902  CG  TRP A 254       2.424  -3.704 -13.467  1.00  0.50           C  \\nATOM   1903  CD1 TRP A 254       2.522  -3.553 -14.822  1.00  0.44           C  \\nATOM   1904  CD2 TRP A 254       2.203  -2.396 -12.932  1.00  0.45           C  \\nATOM   1905  CE2 TRP A 254       2.177  -1.497 -14.022  1.00  0.43           C  \\nATOM   1906  CE3 TRP A 254       2.023  -1.895 -11.635  1.00  0.47           C  \\nATOM   1907  NE1 TRP A 254       2.374  -2.228 -15.162  1.00  0.44           N  \\nATOM   1908  CH2 TRP A 254       1.805   0.338 -12.575  1.00  0.35           C  \\nATOM   1909  CZ2 TRP A 254       1.978  -0.125 -13.853  1.00  0.39           C  \\nATOM   1910  CZ3 TRP A 254       1.825  -0.529 -11.470  1.00  0.36           C  \\nATOM   1911  N   TRP A 255       4.086  -6.787 -10.408  1.00  0.45           N  \\nATOM   1912  CA  TRP A 255       4.506  -7.018  -9.030  1.00  0.55           C  \\nATOM   1913  C   TRP A 255       6.020  -7.181  -8.943  1.00  0.45           C  \\nATOM   1914  CB  TRP A 255       3.813  -8.256  -8.456  1.00  0.46           C  \\nATOM   1915  O   TRP A 255       6.651  -6.689  -8.004  1.00  0.46           O  \\nATOM   1916  CG  TRP A 255       2.544  -7.955  -7.716  1.00  0.47           C  \\nATOM   1917  CD1 TRP A 255       1.299  -7.777  -8.253  1.00  0.42           C  \\nATOM   1918  CD2 TRP A 255       2.398  -7.791  -6.302  1.00  0.43           C  \\nATOM   1919  CE2 TRP A 255       1.036  -7.517  -6.051  1.00  0.42           C  \\nATOM   1920  CE3 TRP A 255       3.289  -7.851  -5.221  1.00  0.48           C  \\nATOM   1921  NE1 TRP A 255       0.387  -7.514  -7.257  1.00  0.44           N  \\nATOM   1922  CH2 TRP A 255       1.434  -7.366  -3.723  1.00  0.35           C  \\nATOM   1923  CZ2 TRP A 255       0.542  -7.302  -4.762  1.00  0.38           C  \\nATOM   1924  CZ3 TRP A 255       2.796  -7.637  -3.939  1.00  0.36           C  \\nATOM   1925  N   TRP A 256       6.672  -7.892  -9.932  1.00  0.45           N  \\nATOM   1926  CA  TRP A 256       8.120  -8.064  -9.892  1.00  0.53           C  \\nATOM   1927  C   TRP A 256       8.833  -6.756 -10.219  1.00  0.45           C  \\nATOM   1928  CB  TRP A 256       8.559  -9.158 -10.870  1.00  0.45           C  \\nATOM   1929  O   TRP A 256       9.967  -6.535  -9.786  1.00  0.46           O  \\nATOM   1930  CG  TRP A 256       8.279 -10.551 -10.392  1.00  0.45           C  \\nATOM   1931  CD1 TRP A 256       7.092 -11.224 -10.467  1.00  0.39           C  \\nATOM   1932  CD2 TRP A 256       9.205 -11.439  -9.759  1.00  0.41           C  \\nATOM   1933  CE2 TRP A 256       8.510 -12.636  -9.477  1.00  0.40           C  \\nATOM   1934  CE3 TRP A 256      10.557 -11.339  -9.402  1.00  0.49           C  \\nATOM   1935  NE1 TRP A 256       7.224 -12.479  -9.919  1.00  0.44           N  \\nATOM   1936  CH2 TRP A 256      10.446 -13.602  -8.517  1.00  0.34           C  \\nATOM   1937  CZ2 TRP A 256       9.123 -13.727  -8.856  1.00  0.37           C  \\nATOM   1938  CZ3 TRP A 256      11.165 -12.425  -8.784  1.00  0.36           C  \\nATOM   1939  N   TRP A 257       8.127  -5.811 -10.879  1.00  0.42           N  \\nATOM   1940  CA  TRP A 257       8.784  -4.561 -11.248  1.00  0.51           C  \\nATOM   1941  C   TRP A 257       8.404  -3.441 -10.285  1.00  0.42           C  \\nATOM   1942  CB  TRP A 257       8.419  -4.165 -12.681  1.00  0.42           C  \\nATOM   1943  O   TRP A 257       8.917  -2.324 -10.389  1.00  0.42           O  \\nATOM   1944  CG  TRP A 257       9.070  -5.013 -13.732  1.00  0.43           C  \\nATOM   1945  CD1 TRP A 257       8.665  -6.246 -14.163  1.00  0.39           C  \\nATOM   1946  CD2 TRP A 257      10.245  -4.692 -14.482  1.00  0.40           C  \\nATOM   1947  CE2 TRP A 257      10.496  -5.777 -15.351  1.00  0.39           C  \\nATOM   1948  CE3 TRP A 257      11.110  -3.590 -14.505  1.00  0.43           C  \\nATOM   1949  NE1 TRP A 257       9.519  -6.711 -15.136  1.00  0.40           N  \\nATOM   1950  CH2 TRP A 257      12.410  -4.701 -16.236  1.00  0.30           C  \\nATOM   1951  CZ2 TRP A 257      11.578  -5.791 -16.234  1.00  0.32           C  \\nATOM   1952  CZ3 TRP A 257      12.187  -3.606 -15.384  1.00  0.33           C  \\nATOM   1953  N   TRP A 258       7.308  -3.618  -9.412  1.00  0.36           N  \\nATOM   1954  CA  TRP A 258       7.282  -2.664  -8.308  1.00  0.49           C  \\nATOM   1955  C   TRP A 258       8.494  -2.849  -7.401  1.00  0.38           C  \\nATOM   1956  CB  TRP A 258       5.993  -2.816  -7.496  1.00  0.39           C  \\nATOM   1957  O   TRP A 258       8.766  -3.958  -6.935  1.00  0.39           O  \\nATOM   1958  CG  TRP A 258       5.482  -1.531  -6.918  1.00  0.42           C  \\nATOM   1959  CD1 TRP A 258       5.958  -0.876  -5.816  1.00  0.41           C  \\nATOM   1960  CD2 TRP A 258       4.392  -0.747  -7.412  1.00  0.41           C  \\nATOM   1961  CE2 TRP A 258       4.262   0.372  -6.559  1.00  0.39           C  \\nATOM   1962  CE3 TRP A 258       3.511  -0.881  -8.494  1.00  0.46           C  \\nATOM   1963  NE1 TRP A 258       5.229   0.270  -5.595  1.00  0.43           N  \\nATOM   1964  CH2 TRP A 258       2.438   1.196  -7.822  1.00  0.34           C  \\nATOM   1965  CZ2 TRP A 258       3.286   1.352  -6.756  1.00  0.36           C  \\nATOM   1966  CZ3 TRP A 258       2.541   0.095  -8.688  1.00  0.35           C  \\nATOM   1967  N   PRO A 259       9.467  -1.978  -7.536  1.00  0.50           N  \\nATOM   1968  CA  PRO A 259      10.270  -1.958  -6.311  1.00  0.49           C  \\nATOM   1969  C   PRO A 259       9.419  -2.037  -5.046  1.00  0.50           C  \\nATOM   1970  CB  PRO A 259      11.005  -0.618  -6.392  1.00  0.48           C  \\nATOM   1971  O   PRO A 259       8.273  -1.580  -5.039  1.00  0.50           O  \\nATOM   1972  CG  PRO A 259      10.144   0.237  -7.265  1.00  0.48           C  \\nATOM   1973  CD  PRO A 259       9.336  -0.657  -8.161  1.00  0.50           C  \\nATOM   1974  N   PRO A 260       9.613  -3.051  -4.248  1.00  0.50           N  \\nATOM   1975  CA  PRO A 260       8.989  -2.877  -2.934  1.00  0.51           C  \\nATOM   1976  C   PRO A 260       8.742  -1.410  -2.588  1.00  0.51           C  \\nATOM   1977  CB  PRO A 260      10.008  -3.499  -1.975  1.00  0.49           C  \\nATOM   1978  O   PRO A 260       9.472  -0.531  -3.052  1.00  0.50           O  \\nATOM   1979  CG  PRO A 260      11.305  -3.459  -2.717  1.00  0.48           C  \\nATOM   1980  CD  PRO A 260      11.011  -3.419  -4.189  1.00  0.50           C  \\nATOM   1981  N   PRO A 261       7.439  -1.012  -2.305  1.00  0.51           N  \\nATOM   1982  CA  PRO A 261       7.384   0.337  -1.737  1.00  0.52           C  \\nATOM   1983  C   PRO A 261       8.668   0.723  -1.008  1.00  0.52           C  \\nATOM   1984  CB  PRO A 261       6.204   0.262  -0.764  1.00  0.50           C  \\nATOM   1985  O   PRO A 261       9.340  -0.139  -0.435  1.00  0.52           O  \\nATOM   1986  CG  PRO A 261       6.061  -1.192  -0.449  1.00  0.50           C  \\nATOM   1987  CD  PRO A 261       6.634  -1.986  -1.589  1.00  0.52           C  \\nATOM   1988  N   PRO A 262       9.254   1.873  -1.445  1.00  0.51           N  \\nATOM   1989  CA  PRO A 262      10.287   2.318  -0.506  1.00  0.52           C  \\nATOM   1990  C   PRO A 262       9.918   2.045   0.950  1.00  0.52           C  \\nATOM   1991  CB  PRO A 262      10.380   3.823  -0.773  1.00  0.49           C  \\nATOM   1992  O   PRO A 262       8.734   2.028   1.298  1.00  0.52           O  \\nATOM   1993  CG  PRO A 262       9.062   4.186  -1.377  1.00  0.49           C  \\nATOM   1994  CD  PRO A 262       8.486   2.965  -2.034  1.00  0.52           C  \\nATOM   1995  N   PRO A 263      10.725   1.310   1.623  1.00  0.54           N  \\nATOM   1996  CA  PRO A 263      10.510   1.254   3.071  1.00  0.57           C  \\nATOM   1997  C   PRO A 263       9.854   2.520   3.620  1.00  0.56           C  \\nATOM   1998  CB  PRO A 263      11.924   1.085   3.631  1.00  0.53           C  \\nATOM   1999  O   PRO A 263      10.022   3.602   3.052  1.00  0.55           O  \\nATOM   2000  CG  PRO A 263      12.822   1.649   2.578  1.00  0.53           C  \\nATOM   2001  CD  PRO A 263      12.138   1.517   1.247  1.00  0.55           C  \\nATOM   2002  N   PRO A 264       8.731   2.396   4.253  1.00  0.58           N  \\nATOM   2003  CA  PRO A 264       8.227   3.573   4.963  1.00  0.63           C  \\nATOM   2004  C   PRO A 264       9.344   4.501   5.436  1.00  0.60           C  \\nATOM   2005  CB  PRO A 264       7.477   2.972   6.154  1.00  0.57           C  \\nATOM   2006  O   PRO A 264      10.452   4.041   5.726  1.00  0.59           O  \\nATOM   2007  CG  PRO A 264       8.097   1.627   6.357  1.00  0.56           C  \\nATOM   2008  CD  PRO A 264       8.639   1.148   5.041  1.00  0.58           C  \\nATOM   2009  N   PRO A 265       9.228   5.778   5.078  1.00  0.57           N  \\nATOM   2010  CA  PRO A 265      10.172   6.688   5.732  1.00  0.61           C  \\nATOM   2011  C   PRO A 265      10.418   6.329   7.196  1.00  0.59           C  \\nATOM   2012  CB  PRO A 265       9.488   8.052   5.612  1.00  0.55           C  \\nATOM   2013  O   PRO A 265       9.526   5.799   7.864  1.00  0.57           O  \\nATOM   2014  CG  PRO A 265       8.035   7.738   5.455  1.00  0.54           C  \\nATOM   2015  CD  PRO A 265       7.905   6.375   4.837  1.00  0.57           C  \\nATOM   2016  N   PRO A 266      11.636   6.148   7.516  1.00  0.61           N  \\nATOM   2017  CA  PRO A 266      11.918   6.007   8.946  1.00  0.64           C  \\nATOM   2018  C   PRO A 266      11.006   6.872   9.815  1.00  0.62           C  \\nATOM   2019  CB  PRO A 266      13.375   6.461   9.065  1.00  0.58           C  \\nATOM   2020  O   PRO A 266      10.546   7.928   9.373  1.00  0.60           O  \\nATOM   2021  CG  PRO A 266      13.589   7.368   7.897  1.00  0.57           C  \\nATOM   2022  CD  PRO A 266      12.661   6.951   6.792  1.00  0.59           C  \\nATOM   2023  N   PRO A 267      10.332   6.307  10.706  1.00  0.68           N  \\nATOM   2024  CA  PRO A 267       9.595   7.108  11.686  1.00  0.71           C  \\nATOM   2025  C   PRO A 267      10.296   8.423  12.020  1.00  0.70           C  \\nATOM   2026  CB  PRO A 267       9.533   6.195  12.913  1.00  0.65           C  \\nATOM   2027  O   PRO A 267      11.526   8.503  11.962  1.00  0.66           O  \\nATOM   2028  CG  PRO A 267      10.707   5.282  12.770  1.00  0.63           C  \\nATOM   2029  CD  PRO A 267      11.013   5.124  11.308  1.00  0.64           C  \\nATOM   2030  N   PRO A 268       9.661   9.562  11.895  1.00  0.67           N  \\nATOM   2031  CA  PRO A 268      10.264  10.789  12.421  1.00  0.69           C  \\nATOM   2032  C   PRO A 268      11.061  10.553  13.702  1.00  0.68           C  \\nATOM   2033  CB  PRO A 268       9.055  11.690  12.689  1.00  0.64           C  \\nATOM   2034  O   PRO A 268      10.732   9.654  14.481  1.00  0.64           O  \\nATOM   2035  CG  PRO A 268       7.907  10.748  12.856  1.00  0.61           C  \\nATOM   2036  CD  PRO A 268       8.202   9.493  12.085  1.00  0.62           C  \\nATOM   2037  N   PRO A 269      12.323  10.964  13.744  1.00  0.67           N  \\nATOM   2038  CA  PRO A 269      13.052  10.935  15.014  1.00  0.68           C  \\nATOM   2039  C   PRO A 269      12.168  11.285  16.209  1.00  0.67           C  \\nATOM   2040  CB  PRO A 269      14.145  11.989  14.817  1.00  0.63           C  \\nATOM   2041  O   PRO A 269      11.219  12.061  16.074  1.00  0.64           O  \\nATOM   2042  CG  PRO A 269      13.597  12.919  13.783  1.00  0.61           C  \\nATOM   2043  CD  PRO A 269      12.653  12.151  12.903  1.00  0.62           C  \\nATOM   2044  N   PRO A 270      12.035  10.433  17.146  1.00  0.71           N  \\nATOM   2045  CA  PRO A 270      11.366  10.778  18.403  1.00  0.71           C  \\nATOM   2046  C   PRO A 270      11.536  12.249  18.777  1.00  0.71           C  \\nATOM   2047  CB  PRO A 270      12.051   9.871  19.428  1.00  0.67           C  \\nATOM   2048  O   PRO A 270      12.569  12.851  18.473  1.00  0.66           O  \\nATOM   2049  CG  PRO A 270      13.402   9.594  18.851  1.00  0.65           C  \\nATOM   2050  CD  PRO A 270      13.301   9.649  17.353  1.00  0.65           C  \\nATOM   2051  N   PRO A 271      10.436  13.002  18.843  1.00  0.71           N  \\nATOM   2052  CA  PRO A 271      10.591  14.354  19.385  1.00  0.72           C  \\nATOM   2053  C   PRO A 271      11.777  14.475  20.338  1.00  0.71           C  \\nATOM   2054  CB  PRO A 271       9.271  14.591  20.123  1.00  0.66           C  \\nATOM   2055  O   PRO A 271      11.929  13.656  21.248  1.00  0.66           O  \\nATOM   2056  CG  PRO A 271       8.650  13.237  20.240  1.00  0.64           C  \\nATOM   2057  CD  PRO A 271       9.352  12.307  19.293  1.00  0.64           C  \\nATOM   2058  N   GLN A 272      12.968  15.014  19.793  1.00  0.70           N  \\nATOM   2059  CA  GLN A 272      14.034  15.415  20.704  1.00  0.72           C  \\nATOM   2060  C   GLN A 272      13.475  15.782  22.076  1.00  0.69           C  \\nATOM   2061  CB  GLN A 272      14.821  16.592  20.126  1.00  0.65           C  \\nATOM   2062  O   GLN A 272      12.628  16.671  22.190  1.00  0.64           O  \\nATOM   2063  CG  GLN A 272      16.028  16.176  19.296  1.00  0.60           C  \\nATOM   2064  CD  GLN A 272      16.721  17.354  18.637  1.00  0.61           C  \\nATOM   2065  NE2 GLN A 272      17.681  17.065  17.765  1.00  0.53           N  \\nATOM   2066  OE1 GLN A 272      16.396  18.514  18.908  1.00  0.65           O  \\nATOM   2067  N   GLN A 273      13.329  14.712  23.032  1.00  0.75           N  \\nATOM   2068  CA  GLN A 273      13.126  15.037  24.439  1.00  0.77           C  \\nATOM   2069  C   GLN A 273      13.793  16.362  24.799  1.00  0.75           C  \\nATOM   2070  CB  GLN A 273      13.663  13.918  25.333  1.00  0.69           C  \\nATOM   2071  O   GLN A 273      15.015  16.493  24.703  1.00  0.68           O  \\nATOM   2072  CG  GLN A 273      12.590  12.962  25.834  1.00  0.64           C  \\nATOM   2073  CD  GLN A 273      13.164  11.778  26.589  1.00  0.63           C  \\nATOM   2074  NE2 GLN A 273      12.305  10.831  26.949  1.00  0.55           N  \\nATOM   2075  OE1 GLN A 273      14.371  11.715  26.847  1.00  0.68           O  \\nATOM   2076  N   GLN A 274      13.068  17.541  24.542  1.00  0.75           N  \\nATOM   2077  CA  GLN A 274      13.486  18.777  25.195  1.00  0.78           C  \\nATOM   2078  C   GLN A 274      14.107  18.496  26.560  1.00  0.77           C  \\nATOM   2079  CB  GLN A 274      12.302  19.734  25.343  1.00  0.70           C  \\nATOM   2080  O   GLN A 274      13.466  17.899  27.428  1.00  0.71           O  \\nATOM   2081  CG  GLN A 274      11.988  20.524  24.080  1.00  0.65           C  \\nATOM   2082  CD  GLN A 274      10.769  21.415  24.233  1.00  0.64           C  \\nATOM   2083  NE2 GLN A 274      10.413  22.121  23.165  1.00  0.58           N  \\nATOM   2084  OE1 GLN A 274      10.152  21.467  25.302  1.00  0.71           O  \\nATOM   2085  N   GLN A 275      15.493  18.333  26.574  1.00  0.77           N  \\nATOM   2086  CA  GLN A 275      16.188  18.401  27.856  1.00  0.80           C  \\nATOM   2087  C   GLN A 275      15.533  19.421  28.783  1.00  0.79           C  \\nATOM   2088  CB  GLN A 275      17.663  18.749  27.650  1.00  0.73           C  \\nATOM   2089  O   GLN A 275      15.283  20.560  28.383  1.00  0.74           O  \\nATOM   2090  CG  GLN A 275      18.525  18.526  28.885  1.00  0.68           C  \\nATOM   2091  CD  GLN A 275      19.998  18.784  28.627  1.00  0.67           C  \\nATOM   2092  NE2 GLN A 275      20.792  18.806  29.692  1.00  0.60           N  \\nATOM   2093  OE1 GLN A 275      20.419  18.960  27.479  1.00  0.72           O  \\nATOM   2094  N   PRO A 276      14.821  18.872  29.822  1.00  0.79           N  \\nATOM   2095  CA  PRO A 276      14.340  19.814  30.836  1.00  0.81           C  \\nATOM   2096  C   PRO A 276      15.327  20.947  31.105  1.00  0.82           C  \\nATOM   2097  CB  PRO A 276      14.166  18.935  32.078  1.00  0.77           C  \\nATOM   2098  O   PRO A 276      16.542  20.747  31.015  1.00  0.77           O  \\nATOM   2099  CG  PRO A 276      15.043  17.749  31.833  1.00  0.74           C  \\nATOM   2100  CD  PRO A 276      15.226  17.590  30.351  1.00  0.74           C  \\nATOM   2101  N   PRO A 277      14.854  22.155  30.966  1.00  0.80           N  \\nATOM   2102  CA  PRO A 277      15.706  23.279  31.361  1.00  0.83           C  \\nATOM   2103  C   PRO A 277      16.544  22.977  32.601  1.00  0.83           C  \\nATOM   2104  CB  PRO A 277      14.702  24.400  31.642  1.00  0.78           C  \\nATOM   2105  O   PRO A 277      16.114  22.216  33.471  1.00  0.79           O  \\nATOM   2106  CG  PRO A 277      13.429  23.696  31.986  1.00  0.75           C  \\nATOM   2107  CD  PRO A 277      13.421  22.362  31.296  1.00  0.74           C  \\nATOM   2108  N   PRO A 278      17.827  23.196  32.516  1.00  0.83           N  \\nATOM   2109  CA  PRO A 278      18.687  23.084  33.696  1.00  0.85           C  \\nATOM   2110  C   PRO A 278      18.042  23.660  34.954  1.00  0.85           C  \\nATOM   2111  CB  PRO A 278      19.930  23.887  33.303  1.00  0.81           C  \\nATOM   2112  O   PRO A 278      17.252  24.604  34.870  1.00  0.82           O  \\nATOM   2113  CG  PRO A 278      19.442  24.877  32.295  1.00  0.78           C  \\nATOM   2114  CD  PRO A 278      18.275  24.278  31.565  1.00  0.76           C  \\nATOM   2115  N   PRO A 279      18.045  22.934  36.014  1.00  0.86           N  \\nATOM   2116  CA  PRO A 279      17.588  23.460  37.302  1.00  0.87           C  \\nATOM   2117  C   PRO A 279      18.082  24.880  37.569  1.00  0.88           C  \\nATOM   2118  CB  PRO A 279      18.180  22.476  38.315  1.00  0.84           C  \\nATOM   2119  O   PRO A 279      19.167  25.255  37.118  1.00  0.86           O  \\nATOM   2120  CG  PRO A 279      19.381  21.904  37.634  1.00  0.80           C  \\nATOM   2121  CD  PRO A 279      19.142  21.919  36.152  1.00  0.78           C  \\nATOM   2122  N   PRO A 280      17.234  25.777  37.979  1.00  0.88           N  \\nATOM   2123  CA  PRO A 280      17.652  27.110  38.420  1.00  0.90           C  \\nATOM   2124  C   PRO A 280      18.880  27.073  39.327  1.00  0.90           C  \\nATOM   2125  CB  PRO A 280      16.429  27.630  39.179  1.00  0.87           C  \\nATOM   2126  O   PRO A 280      19.087  26.098  40.053  1.00  0.87           O  \\nATOM   2127  CG  PRO A 280      15.717  26.401  39.646  1.00  0.82           C  \\nATOM   2128  CD  PRO A 280      16.000  25.290  38.676  1.00  0.78           C  \\nATOM   2129  N   PRO A 281      19.830  27.920  39.116  1.00  0.90           N  \\nATOM   2130  CA  PRO A 281      20.969  28.067  40.024  1.00  0.91           C  \\nATOM   2131  C   PRO A 281      20.551  28.130  41.491  1.00  0.92           C  \\nATOM   2132  CB  PRO A 281      21.605  29.388  39.581  1.00  0.88           C  \\nATOM   2133  O   PRO A 281      19.455  28.603  41.805  1.00  0.88           O  \\nATOM   2134  CG  PRO A 281      20.486  30.160  38.958  1.00  0.84           C  \\nATOM   2135  CD  PRO A 281      19.496  29.186  38.385  1.00  0.80           C  \\nATOM   2136  N   PRO A 282      21.244  27.427  42.302  1.00  0.91           N  \\nATOM   2137  CA  PRO A 282      21.012  27.545  43.744  1.00  0.92           C  \\nATOM   2138  C   PRO A 282      20.910  28.996  44.208  1.00  0.92           C  \\nATOM   2139  CB  PRO A 282      22.238  26.861  44.356  1.00  0.89           C  \\nATOM   2140  O   PRO A 282      21.530  29.883  43.616  1.00  0.90           O  \\nATOM   2141  CG  PRO A 282      23.306  26.992  43.318  1.00  0.85           C  \\nATOM   2142  CD  PRO A 282      22.653  27.050  41.967  1.00  0.82           C  \\nATOM   2143  N   PRO A 283      19.972  29.274  45.038  1.00  0.90           N  \\nATOM   2144  CA  PRO A 283      19.924  30.597  45.665  1.00  0.91           C  \\nATOM   2145  C   PRO A 283      21.275  31.031  46.230  1.00  0.91           C  \\nATOM   2146  CB  PRO A 283      18.896  30.419  46.785  1.00  0.88           C  \\nATOM   2147  O   PRO A 283      22.083  30.187  46.625  1.00  0.88           O  \\nATOM   2148  CG  PRO A 283      18.916  28.957  47.094  1.00  0.84           C  \\nATOM   2149  CD  PRO A 283      19.328  28.214  45.856  1.00  0.80           C  \\nATOM   2150  N   PRO A 284      21.654  32.224  46.045  1.00  0.90           N  \\nATOM   2151  CA  PRO A 284      22.846  32.767  46.698  1.00  0.91           C  \\nATOM   2152  C   PRO A 284      22.888  32.466  48.195  1.00  0.91           C  \\nATOM   2153  CB  PRO A 284      22.731  34.273  46.447  1.00  0.88           C  \\nATOM   2154  O   PRO A 284      21.839  32.336  48.831  1.00  0.87           O  \\nATOM   2155  CG  PRO A 284      21.269  34.518  46.257  1.00  0.84           C  \\nATOM   2156  CD  PRO A 284      20.643  33.266  45.714  1.00  0.80           C  \\nATOM   2157  N   PRO A 285      24.011  32.054  48.688  1.00  0.89           N  \\nATOM   2158  CA  PRO A 285      24.168  31.895  50.135  1.00  0.90           C  \\nATOM   2159  C   PRO A 285      23.608  33.078  50.923  1.00  0.90           C  \\nATOM   2160  CB  PRO A 285      25.684  31.788  50.316  1.00  0.87           C  \\nATOM   2161  O   PRO A 285      23.587  34.204  50.420  1.00  0.86           O  \\nATOM   2162  CG  PRO A 285      26.261  32.520  49.147  1.00  0.83           C  \\nATOM   2163  CD  PRO A 285      25.305  32.406  47.994  1.00  0.80           C  \\nATOM   2164  N   PRO A 286      22.918  32.850  51.947  1.00  0.85           N  \\nATOM   2165  CA  PRO A 286      22.498  33.936  52.836  1.00  0.86           C  \\nATOM   2166  C   PRO A 286      23.640  34.888  53.186  1.00  0.85           C  \\nATOM   2167  CB  PRO A 286      22.000  33.199  54.082  1.00  0.82           C  \\nATOM   2168  O   PRO A 286      24.802  34.476  53.223  1.00  0.81           O  \\nATOM   2169  CG  PRO A 286      22.745  31.904  54.083  1.00  0.79           C  \\nATOM   2170  CD  PRO A 286      23.048  31.527  52.661  1.00  0.77           C  \\nATOM   2171  N   PRO A 287      23.459  36.124  53.091  1.00  0.83           N  \\nATOM   2172  CA  PRO A 287      24.441  37.090  53.590  1.00  0.84           C  \\nATOM   2173  C   PRO A 287      24.982  36.722  54.970  1.00  0.83           C  \\nATOM   2174  CB  PRO A 287      23.651  38.400  53.645  1.00  0.80           C  \\nATOM   2175  O   PRO A 287      24.272  36.112  55.773  1.00  0.78           O  \\nATOM   2176  CG  PRO A 287      22.226  37.980  53.816  1.00  0.78           C  \\nATOM   2177  CD  PRO A 287      22.047  36.632  53.178  1.00  0.76           C  \\nATOM   2178  N   PRO A 288      26.267  36.681  55.097  1.00  0.85           N  \\nATOM   2179  CA  PRO A 288      26.831  36.499  56.437  1.00  0.85           C  \\nATOM   2180  C   PRO A 288      26.017  37.202  57.520  1.00  0.84           C  \\nATOM   2181  CB  PRO A 288      28.228  37.114  56.318  1.00  0.82           C  \\nATOM   2182  O   PRO A 288      25.684  38.382  57.379  1.00  0.79           O  \\nATOM   2183  CG  PRO A 288      28.161  37.982  55.103  1.00  0.78           C  \\nATOM   2184  CD  PRO A 288      27.004  37.532  54.259  1.00  0.76           C  \\nATOM   2185  N   GLN A 289      25.273  36.419  58.304  1.00  0.83           N  \\nATOM   2186  CA  GLN A 289      24.751  37.011  59.531  1.00  0.83           C  \\nATOM   2187  C   GLN A 289      25.766  37.963  60.157  1.00  0.80           C  \\nATOM   2188  CB  GLN A 289      24.366  35.921  60.533  1.00  0.78           C  \\nATOM   2189  O   GLN A 289      26.870  37.551  60.520  1.00  0.75           O  \\nATOM   2190  CG  GLN A 289      23.149  35.106  60.117  1.00  0.73           C  \\nATOM   2191  CD  GLN A 289      22.747  34.079  61.158  1.00  0.71           C  \\nATOM   2192  NE2 GLN A 289      21.675  33.343  60.882  1.00  0.69           N  \\nATOM   2193  OE1 GLN A 289      23.394  33.947  62.202  1.00  0.76           O  \\nATOM   2194  N   GLN A 290      25.612  39.309  59.951  1.00  0.82           N  \\nATOM   2195  CA  GLN A 290      26.257  40.290  60.818  1.00  0.82           C  \\nATOM   2196  C   GLN A 290      26.048  39.944  62.289  1.00  0.80           C  \\nATOM   2197  CB  GLN A 290      25.725  41.694  60.528  1.00  0.77           C  \\nATOM   2198  O   GLN A 290      24.933  39.625  62.706  1.00  0.74           O  \\nATOM   2199  CG  GLN A 290      26.322  42.332  59.280  1.00  0.73           C  \\nATOM   2200  CD  GLN A 290      25.901  43.779  59.102  1.00  0.70           C  \\nATOM   2201  NE2 GLN A 290      26.791  44.591  58.542  1.00  0.66           N  \\nATOM   2202  OE1 GLN A 290      24.784  44.163  59.463  1.00  0.73           O  \\nATOM   2203  N   PRO A 291      27.065  39.438  62.915  1.00  0.81           N  \\nATOM   2204  CA  PRO A 291      26.896  39.394  64.369  1.00  0.80           C  \\nATOM   2205  C   PRO A 291      26.157  40.615  64.914  1.00  0.79           C  \\nATOM   2206  CB  PRO A 291      28.334  39.349  64.890  1.00  0.77           C  \\nATOM   2207  O   PRO A 291      26.519  41.751  64.599  1.00  0.74           O  \\nATOM   2208  CG  PRO A 291      29.150  39.997  63.818  1.00  0.76           C  \\nATOM   2209  CD  PRO A 291      28.444  39.813  62.505  1.00  0.76           C  \\nATOM   2210  N   TRP A 292      24.821  40.558  65.288  1.00  0.76           N  \\nATOM   2211  CA  TRP A 292      24.175  41.499  66.196  1.00  0.81           C  \\nATOM   2212  C   TRP A 292      25.165  42.024  67.231  1.00  0.74           C  \\nATOM   2213  CB  TRP A 292      22.985  40.839  66.898  1.00  0.70           C  \\nATOM   2214  O   TRP A 292      25.705  41.254  68.029  1.00  0.65           O  \\nATOM   2215  CG  TRP A 292      21.917  40.357  65.963  1.00  0.61           C  \\nATOM   2216  CD1 TRP A 292      21.789  39.103  65.434  1.00  0.56           C  \\nATOM   2217  CD2 TRP A 292      20.828  41.125  65.443  1.00  0.61           C  \\nATOM   2218  CE2 TRP A 292      20.078  40.272  64.604  1.00  0.57           C  \\nATOM   2219  CE3 TRP A 292      20.412  42.454  65.605  1.00  0.70           C  \\nATOM   2220  NE1 TRP A 292      20.685  39.045  64.616  1.00  0.73           N  \\nATOM   2221  CH2 TRP A 292      18.549  42.009  64.105  1.00  0.67           C  \\nATOM   2222  CZ2 TRP A 292      18.934  40.705  63.929  1.00  0.71           C  \\nATOM   2223  CZ3 TRP A 292      19.274  42.883  64.933  1.00  0.67           C  \\nATOM   2224  N   GLY A 293      25.908  42.995  66.795  1.00  0.79           N  \\nATOM   2225  CA  GLY A 293      26.584  43.819  67.783  1.00  0.79           C  \\nATOM   2226  C   GLY A 293      25.932  43.765  69.152  1.00  0.77           C  \\nATOM   2227  O   GLY A 293      24.719  43.944  69.275  1.00  0.72           O  \\nATOM   2228  N   VAL A 294      26.535  43.062  70.101  1.00  0.79           N  \\nATOM   2229  CA  VAL A 294      26.397  43.410  71.511  1.00  0.80           C  \\nATOM   2230  C   VAL A 294      26.390  44.928  71.671  1.00  0.79           C  \\nATOM   2231  CB  VAL A 294      27.527  42.789  72.362  1.00  0.75           C  \\nATOM   2232  O   VAL A 294      27.413  45.585  71.463  1.00  0.75           O  \\nATOM   2233  CG1 VAL A 294      27.299  43.067  73.847  1.00  0.70           C  \\nATOM   2234  CG2 VAL A 294      27.623  41.286  72.105  1.00  0.67           C  \\nATOM   2235  N   VAL A 295      25.171  45.495  71.666  1.00  0.78           N  \\nATOM   2236  CA  VAL A 295      25.048  46.765  72.374  1.00  0.80           C  \\nATOM   2237  C   VAL A 295      25.533  46.602  73.813  1.00  0.79           C  \\nATOM   2238  CB  VAL A 295      23.594  47.286  72.357  1.00  0.75           C  \\nATOM   2239  O   VAL A 295      24.997  45.786  74.567  1.00  0.77           O  \\nATOM   2240  CG1 VAL A 295      23.504  48.657  73.023  1.00  0.72           C  \\nATOM   2241  CG2 VAL A 295      23.067  47.348  70.925  1.00  0.68           C  \\nATOM   2242  N   PRO A 296      26.730  47.158  74.017  1.00  0.81           N  \\nATOM   2243  CA  PRO A 296      26.966  47.409  75.440  1.00  0.83           C  \\nATOM   2244  C   PRO A 296      25.904  48.311  76.065  1.00  0.84           C  \\nATOM   2245  CB  PRO A 296      28.338  48.089  75.455  1.00  0.80           C  \\nATOM   2246  O   PRO A 296      25.514  49.317  75.466  1.00  0.82           O  \\nATOM   2247  CG  PRO A 296      28.422  48.808  74.147  1.00  0.78           C  \\nATOM   2248  CD  PRO A 296      27.608  48.054  73.135  1.00  0.78           C  \\nATOM   2249  N   ILE A 297      25.249  47.797  76.973  1.00  0.88           N  \\nATOM   2250  CA  ILE A 297      24.445  48.610  77.879  1.00  0.89           C  \\nATOM   2251  C   ILE A 297      25.330  49.649  78.564  1.00  0.89           C  \\nATOM   2252  CB  ILE A 297      23.727  47.738  78.934  1.00  0.86           C  \\nATOM   2253  O   ILE A 297      26.482  49.366  78.903  1.00  0.86           O  \\nATOM   2254  CG1 ILE A 297      22.801  46.726  78.250  1.00  0.81           C  \\nATOM   2255  CG2 ILE A 297      22.948  48.615  79.919  1.00  0.78           C  \\nATOM   2256  CD1 ILE A 297      22.210  45.690  79.195  1.00  0.81           C  \",\"pdb\");\n\tviewer_17251214197132845.setStyle({\"resi\": 1},{\"cartoon\": {\"color\": \"#4900b6\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 2},{\"cartoon\": {\"color\": \"#c80037\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 3},{\"cartoon\": {\"color\": \"#a80057\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 4},{\"cartoon\": {\"color\": \"#8f0070\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 5},{\"cartoon\": {\"color\": \"#7d0082\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 6},{\"cartoon\": {\"color\": \"#70008f\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 7},{\"cartoon\": {\"color\": \"#9f0060\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 8},{\"cartoon\": {\"color\": \"#bc0043\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 9},{\"cartoon\": {\"color\": \"#cd0032\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 10},{\"cartoon\": {\"color\": \"#c4003b\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 11},{\"cartoon\": {\"color\": \"#d1002e\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 12},{\"cartoon\": {\"color\": \"#db0024\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 13},{\"cartoon\": {\"color\": \"#e3001c\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 14},{\"cartoon\": {\"color\": \"#e80017\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 15},{\"cartoon\": {\"color\": \"#ec0013\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 16},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 17},{\"cartoon\": {\"color\": \"#f3000c\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 18},{\"cartoon\": {\"color\": \"#ec0013\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 19},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 20},{\"cartoon\": {\"color\": \"#fa0005\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 21},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 22},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 23},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 24},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 25},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 26},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 27},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 28},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 29},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 30},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 31},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 32},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 33},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 34},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 35},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 36},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 37},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 38},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 39},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 40},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 41},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 42},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 43},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 44},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 45},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 46},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 47},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 48},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 49},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 50},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 51},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 52},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 53},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 54},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 55},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 56},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 57},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 58},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 59},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 60},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 61},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 62},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 63},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 64},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 65},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 66},{\"cartoon\": {\"color\": \"#fb0004\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 67},{\"cartoon\": {\"color\": \"#f80007\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 68},{\"cartoon\": {\"color\": \"#f80007\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 69},{\"cartoon\": {\"color\": \"#f2000d\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 70},{\"cartoon\": {\"color\": \"#eb0014\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 71},{\"cartoon\": {\"color\": \"#eb0014\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 72},{\"cartoon\": {\"color\": \"#f3000c\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 73},{\"cartoon\": {\"color\": \"#f3000c\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 74},{\"cartoon\": {\"color\": \"#f3000c\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 75},{\"cartoon\": {\"color\": \"#f3000c\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 76},{\"cartoon\": {\"color\": \"#f0000f\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 77},{\"cartoon\": {\"color\": \"#ef0010\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 78},{\"cartoon\": {\"color\": \"#f60009\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 79},{\"cartoon\": {\"color\": \"#fb0004\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 80},{\"cartoon\": {\"color\": \"#fb0004\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 81},{\"cartoon\": {\"color\": \"#fb0004\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 82},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 83},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 84},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 85},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 86},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 87},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 88},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 89},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 90},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 91},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 92},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 93},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 94},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 95},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 96},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 97},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 98},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 99},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 100},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 101},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 102},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 103},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 104},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 105},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 106},{\"cartoon\": {\"color\": \"#fe0001\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 107},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 108},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 109},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 110},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 111},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 112},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 113},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 114},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 115},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 116},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 117},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 118},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 119},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 120},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 121},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 122},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 123},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 124},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 125},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 126},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 127},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 128},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 129},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 130},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 131},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 132},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 133},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 134},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 135},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 136},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 137},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 138},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 139},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 140},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 141},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 142},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 143},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 144},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 145},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 146},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 147},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 148},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 149},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 150},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 151},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 152},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 153},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 154},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 155},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 156},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 157},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 158},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 159},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 160},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 161},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 162},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 163},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 164},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 165},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 166},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 167},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 168},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 169},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 170},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 171},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 172},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 173},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 174},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 175},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 176},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 177},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 178},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 179},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 180},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 181},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 182},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 183},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 184},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 185},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 186},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 187},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 188},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 189},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 190},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 191},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 192},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 193},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 194},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 195},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 196},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 197},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 198},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 199},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 200},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 201},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 202},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 203},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 204},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 205},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 206},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 207},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 208},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 209},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 210},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 211},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 212},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 213},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 214},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 215},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 216},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 217},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 218},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 219},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 220},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 221},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 222},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 223},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 224},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 225},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 226},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 227},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 228},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 229},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 230},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 231},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 232},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 233},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 234},{\"cartoon\": {\"color\": \"#f4000b\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 235},{\"cartoon\": {\"color\": \"#ec0013\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 236},{\"cartoon\": {\"color\": \"#e1001e\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 237},{\"cartoon\": {\"color\": \"#d5002a\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 238},{\"cartoon\": {\"color\": \"#c5003a\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 239},{\"cartoon\": {\"color\": \"#b1004e\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 240},{\"cartoon\": {\"color\": \"#9a0065\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 241},{\"cartoon\": {\"color\": \"#7e0081\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 242},{\"cartoon\": {\"color\": \"#5e00a1\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 243},{\"cartoon\": {\"color\": \"#3a00c5\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 244},{\"cartoon\": {\"color\": \"#1000ef\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 245},{\"cartoon\": {\"color\": \"#1200ed\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 246},{\"cartoon\": {\"color\": \"#1400eb\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 247},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 248},{\"cartoon\": {\"color\": \"#0e00f1\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 249},{\"cartoon\": {\"color\": \"#1000ef\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 250},{\"cartoon\": {\"color\": \"#1100ee\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 251},{\"cartoon\": {\"color\": \"#1100ee\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 252},{\"cartoon\": {\"color\": \"#0f00f0\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 253},{\"cartoon\": {\"color\": \"#0e00f1\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 254},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 255},{\"cartoon\": {\"color\": \"#0c00f3\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 256},{\"cartoon\": {\"color\": \"#0b00f4\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 257},{\"cartoon\": {\"color\": \"#0b00f4\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 258},{\"cartoon\": {\"color\": \"#3700c8\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 259},{\"cartoon\": {\"color\": \"#5d00a2\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 260},{\"cartoon\": {\"color\": \"#7d0082\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 261},{\"cartoon\": {\"color\": \"#980067\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 262},{\"cartoon\": {\"color\": \"#af0050\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 263},{\"cartoon\": {\"color\": \"#c3003c\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 264},{\"cartoon\": {\"color\": \"#d3002c\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 265},{\"cartoon\": {\"color\": \"#e0001f\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 266},{\"cartoon\": {\"color\": \"#eb0014\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 267},{\"cartoon\": {\"color\": \"#f3000c\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 268},{\"cartoon\": {\"color\": \"#f90006\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 269},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 270},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 271},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 272},{\"cartoon\": {\"color\": \"#ff0000\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 273},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 274},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 275},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 276},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 277},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 278},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 279},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 280},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 281},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 282},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 283},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 284},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 285},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 286},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 287},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 288},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 289},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 290},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 291},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 292},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 293},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 294},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.setStyle({\"resi\": 295},{\"cartoon\": {\"color\": \"#fd0002\"}});\n\tviewer_17251214197132845.zoomTo();\nviewer_17251214197132845.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_17251214197132845\"  style=\"position: relative; width: 1000px; height: 800px;\">\n",
              "        <p id=\"3dmolwarning_17251214197132845\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.3.0/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_17251214197132845 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_17251214197132845\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_17251214197132845 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17251214197132845\"),{backgroundColor:\"white\"});\n",
              "viewer_17251214197132845.zoomTo();\n",
              "\tviewer_17251214197132845.addModel(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   MET A   1      23.464 -21.164 -38.766  1.00  0.39           N  \\nATOM      2  CA  MET A   1      23.372 -20.209 -37.665  1.00  0.43           C  \\nATOM      3  C   MET A   1      22.209 -19.245 -37.878  1.00  0.41           C  \\nATOM      4  CB  MET A   1      24.679 -19.429 -37.519  1.00  0.37           C  \\nATOM      5  O   MET A   1      22.304 -18.321 -38.688  1.00  0.41           O  \\nATOM      6  CG  MET A   1      25.852 -20.278 -37.057  1.00  0.39           C  \\nATOM      7  SD  MET A   1      27.395 -19.304 -36.870  1.00  0.38           S  \\nATOM      8  CE  MET A   1      28.571 -20.635 -36.501  1.00  0.36           C  \\nATOM      9  N   ALA A   2      21.042 -19.728 -38.337  1.00  0.45           N  \\nATOM     10  CA  ALA A   2      19.997 -18.719 -38.490  1.00  0.47           C  \\nATOM     11  C   ALA A   2      19.503 -18.231 -37.131  1.00  0.45           C  \\nATOM     12  CB  ALA A   2      18.834 -19.275 -39.306  1.00  0.40           C  \\nATOM     13  O   ALA A   2      19.086 -19.032 -36.291  1.00  0.42           O  \\nATOM     14  N   SER A   3      20.351 -17.463 -36.437  1.00  0.43           N  \\nATOM     15  CA  SER A   3      20.108 -16.483 -35.383  1.00  0.45           C  \\nATOM     16  C   SER A   3      18.732 -15.843 -35.530  1.00  0.43           C  \\nATOM     17  CB  SER A   3      21.188 -15.400 -35.398  1.00  0.39           C  \\nATOM     18  O   SER A   3      18.432 -15.234 -36.559  1.00  0.41           O  \\nATOM     19  OG  SER A   3      20.797 -14.289 -34.610  1.00  0.43           O  \\nATOM     20  N   ASN A   4      17.695 -16.560 -35.306  1.00  0.47           N  \\nATOM     21  CA  ASN A   4      16.308 -16.142 -35.135  1.00  0.51           C  \\nATOM     22  C   ASN A   4      16.211 -14.793 -34.429  1.00  0.47           C  \\nATOM     23  CB  ASN A   4      15.519 -17.203 -34.364  1.00  0.42           C  \\nATOM     24  O   ASN A   4      16.559 -14.677 -33.252  1.00  0.45           O  \\nATOM     25  CG  ASN A   4      14.674 -18.076 -35.271  1.00  0.41           C  \\nATOM     26  ND2 ASN A   4      14.021 -19.077 -34.692  1.00  0.39           N  \\nATOM     27  OD1 ASN A   4      14.609 -17.854 -36.483  1.00  0.42           O  \\nATOM     28  N   ASP A   5      16.721 -13.756 -35.098  1.00  0.41           N  \\nATOM     29  CA  ASP A   5      16.711 -12.298 -35.047  1.00  0.44           C  \\nATOM     30  C   ASP A   5      15.283 -11.759 -35.004  1.00  0.41           C  \\nATOM     31  CB  ASP A   5      17.457 -11.714 -36.249  1.00  0.37           C  \\nATOM     32  O   ASP A   5      14.610 -11.685 -36.034  1.00  0.39           O  \\nATOM     33  CG  ASP A   5      18.856 -11.235 -35.904  1.00  0.36           C  \\nATOM     34  OD1 ASP A   5      19.156 -11.043 -34.706  1.00  0.40           O  \\nATOM     35  OD2 ASP A   5      19.664 -11.044 -36.838  1.00  0.41           O  \\nATOM     36  N   TYR A   6      14.344 -12.453 -34.202  1.00  0.46           N  \\nATOM     37  CA  TYR A   6      13.027 -11.846 -34.046  1.00  0.49           C  \\nATOM     38  C   TYR A   6      12.848 -11.285 -32.640  1.00  0.45           C  \\nATOM     39  CB  TYR A   6      11.926 -12.867 -34.345  1.00  0.41           C  \\nATOM     40  O   TYR A   6      12.625 -12.037 -31.689  1.00  0.43           O  \\nATOM     41  CG  TYR A   6      11.702 -13.104 -35.819  1.00  0.41           C  \\nATOM     42  CD1 TYR A   6      10.875 -12.265 -36.562  1.00  0.39           C  \\nATOM     43  CD2 TYR A   6      12.316 -14.168 -36.471  1.00  0.41           C  \\nATOM     44  CE1 TYR A   6      10.664 -12.481 -37.920  1.00  0.38           C  \\nATOM     45  CE2 TYR A   6      12.113 -14.394 -37.828  1.00  0.42           C  \\nATOM     46  OH  TYR A   6      11.082 -13.765 -39.887  1.00  0.28           O  \\nATOM     47  CZ  TYR A   6      11.287 -13.546 -38.543  1.00  0.34           C  \\nATOM     48  N   THR A   7      13.825 -10.482 -32.090  1.00  0.37           N  \\nATOM     49  CA  THR A   7      13.997  -9.788 -30.818  1.00  0.39           C  \\nATOM     50  C   THR A   7      14.228  -8.296 -31.044  1.00  0.37           C  \\nATOM     51  CB  THR A   7      15.172 -10.376 -30.016  1.00  0.34           C  \\nATOM     52  O   THR A   7      15.286  -7.893 -31.531  1.00  0.36           O  \\nATOM     53  CG2 THR A   7      14.711 -10.873 -28.650  1.00  0.33           C  \\nATOM     54  OG1 THR A   7      15.740 -11.471 -30.744  1.00  0.32           O  \\nATOM     55  N   GLN A   8      13.395  -7.471 -31.807  1.00  0.36           N  \\nATOM     56  CA  GLN A   8      13.733  -6.052 -31.793  1.00  0.40           C  \\nATOM     57  C   GLN A   8      12.521  -5.194 -32.145  1.00  0.36           C  \\nATOM     58  CB  GLN A   8      14.880  -5.763 -32.763  1.00  0.34           C  \\nATOM     59  O   GLN A   8      12.001  -5.272 -33.261  1.00  0.35           O  \\nATOM     60  CG  GLN A   8      16.260  -5.841 -32.125  1.00  0.35           C  \\nATOM     61  CD  GLN A   8      17.377  -5.517 -33.099  1.00  0.44           C  \\nATOM     62  NE2 GLN A   8      18.617  -5.609 -32.631  1.00  0.33           N  \\nATOM     63  OE1 GLN A   8      17.127  -5.187 -34.263  1.00  0.47           O  \\nATOM     64  N   GLN A   9      11.306  -5.225 -31.302  1.00  0.31           N  \\nATOM     65  CA  GLN A   9      10.516  -4.023 -31.551  1.00  0.38           C  \\nATOM     66  C   GLN A   9       9.299  -3.965 -30.632  1.00  0.32           C  \\nATOM     67  CB  GLN A   9      10.073  -3.965 -33.014  1.00  0.29           C  \\nATOM     68  O   GLN A   9       8.494  -4.898 -30.600  1.00  0.31           O  \\nATOM     69  CG  GLN A   9      11.061  -3.252 -33.927  1.00  0.31           C  \\nATOM     70  CD  GLN A   9      10.555  -3.122 -35.352  1.00  0.41           C  \\nATOM     71  NE2 GLN A   9      11.355  -2.500 -36.211  1.00  0.32           N  \\nATOM     72  OE1 GLN A   9       9.455  -3.578 -35.677  1.00  0.46           O  \\nATOM     73  N   ALA A  10       9.494  -3.398 -29.307  1.00  0.33           N  \\nATOM     74  CA  ALA A  10       8.569  -2.423 -28.733  1.00  0.37           C  \\nATOM     75  C   ALA A  10       8.372  -2.667 -27.240  1.00  0.33           C  \\nATOM     76  CB  ALA A  10       7.227  -2.472 -29.459  1.00  0.31           C  \\nATOM     77  O   ALA A  10       7.714  -3.633 -26.844  1.00  0.33           O  \\nATOM     78  N   THR A  11       9.561  -2.440 -26.426  1.00  0.34           N  \\nATOM     79  CA  THR A  11       9.285  -2.253 -25.006  1.00  0.37           C  \\nATOM     80  C   THR A  11      10.083  -1.077 -24.450  1.00  0.34           C  \\nATOM     81  CB  THR A  11       9.614  -3.525 -24.201  1.00  0.32           C  \\nATOM     82  O   THR A  11      11.312  -1.133 -24.377  1.00  0.33           O  \\nATOM     83  CG2 THR A  11       8.493  -3.861 -23.223  1.00  0.30           C  \\nATOM     84  OG1 THR A  11       9.789  -4.623 -25.105  1.00  0.29           O  \\nATOM     85  N   GLN A  12       9.820   0.205 -24.774  1.00  0.37           N  \\nATOM     86  CA  GLN A  12      10.113   1.430 -24.037  1.00  0.40           C  \\nATOM     87  C   GLN A  12       9.630   1.332 -22.593  1.00  0.38           C  \\nATOM     88  CB  GLN A  12       9.472   2.637 -24.724  1.00  0.35           C  \\nATOM     89  O   GLN A  12       8.430   1.202 -22.342  1.00  0.36           O  \\nATOM     90  CG  GLN A  12      10.242   3.133 -25.941  1.00  0.34           C  \\nATOM     91  CD  GLN A  12       9.622   4.369 -26.564  1.00  0.42           C  \\nATOM     92  NE2 GLN A  12      10.271   4.903 -27.593  1.00  0.33           N  \\nATOM     93  OE1 GLN A  12       8.568   4.840 -26.124  1.00  0.46           O  \\nATOM     94  N   SER A  13      10.281   0.657 -21.663  1.00  0.35           N  \\nATOM     95  CA  SER A  13      10.150   0.310 -20.251  1.00  0.38           C  \\nATOM     96  C   SER A  13      11.030   1.198 -19.379  1.00  0.35           C  \\nATOM     97  CB  SER A  13      10.510  -1.159 -20.025  1.00  0.32           C  \\nATOM     98  O   SER A  13      12.223   1.356 -19.649  1.00  0.34           O  \\nATOM     99  OG  SER A  13      11.558  -1.559 -20.890  1.00  0.30           O  \\nATOM    100  N   TYR A  14      10.477   2.322 -18.631  1.00  0.37           N  \\nATOM    101  CA  TYR A  14      10.507   3.233 -17.492  1.00  0.41           C  \\nATOM    102  C   TYR A  14      11.507   2.761 -16.443  1.00  0.38           C  \\nATOM    103  CB  TYR A  14       9.114   3.357 -16.866  1.00  0.35           C  \\nATOM    104  O   TYR A  14      11.336   1.694 -15.849  1.00  0.37           O  \\nATOM    105  CG  TYR A  14       8.139   4.143 -17.709  1.00  0.34           C  \\nATOM    106  CD1 TYR A  14       8.094   5.534 -17.643  1.00  0.32           C  \\nATOM    107  CD2 TYR A  14       7.260   3.497 -18.571  1.00  0.35           C  \\nATOM    108  CE1 TYR A  14       7.196   6.262 -18.416  1.00  0.34           C  \\nATOM    109  CE2 TYR A  14       6.357   4.215 -19.348  1.00  0.37           C  \\nATOM    110  OH  TYR A  14       5.441   6.310 -20.032  1.00  0.25           O  \\nATOM    111  CZ  TYR A  14       6.333   5.594 -19.264  1.00  0.31           C  \\nATOM    112  N   GLY A  15      12.741   3.096 -16.410  1.00  0.36           N  \\nATOM    113  CA  GLY A  15      13.912   3.183 -15.553  1.00  0.37           C  \\nATOM    114  C   GLY A  15      13.801   4.268 -14.498  1.00  0.36           C  \\nATOM    115  O   GLY A  15      13.613   5.441 -14.825  1.00  0.35           O  \\nATOM    116  N   ALA A  16      13.440   4.188 -13.190  1.00  0.38           N  \\nATOM    117  CA  ALA A  16      13.467   4.948 -11.943  1.00  0.41           C  \\nATOM    118  C   ALA A  16      14.853   4.906 -11.305  1.00  0.38           C  \\nATOM    119  CB  ALA A  16      12.420   4.411 -10.970  1.00  0.35           C  \\nATOM    120  O   ALA A  16      15.307   3.848 -10.862  1.00  0.38           O  \\nATOM    121  N   TYR A  17      15.722   5.703 -11.634  1.00  0.36           N  \\nATOM    122  CA  TYR A  17      16.863   6.147 -10.841  1.00  0.39           C  \\nATOM    123  C   TYR A  17      16.563   7.470 -10.146  1.00  0.36           C  \\nATOM    124  CB  TYR A  17      18.107   6.292 -11.724  1.00  0.34           C  \\nATOM    125  O   TYR A  17      15.899   8.341 -10.715  1.00  0.35           O  \\nATOM    126  CG  TYR A  17      18.678   4.975 -12.189  1.00  0.35           C  \\nATOM    127  CD1 TYR A  17      19.537   4.240 -11.376  1.00  0.32           C  \\nATOM    128  CD2 TYR A  17      18.360   4.463 -13.442  1.00  0.34           C  \\nATOM    129  CE1 TYR A  17      20.067   3.026 -11.801  1.00  0.34           C  \\nATOM    130  CE2 TYR A  17      18.884   3.250 -13.878  1.00  0.38           C  \\nATOM    131  OH  TYR A  17      20.257   1.339 -13.478  1.00  0.25           O  \\nATOM    132  CZ  TYR A  17      19.735   2.540 -13.052  1.00  0.31           C  \\nATOM    133  N   PRO A  18      16.857   7.768  -8.739  1.00  0.38           N  \\nATOM    134  CA  PRO A  18      16.977   8.990  -7.941  1.00  0.40           C  \\nATOM    135  C   PRO A  18      17.524  10.168  -8.745  1.00  0.38           C  \\nATOM    136  CB  PRO A  18      17.949   8.591  -6.828  1.00  0.36           C  \\nATOM    137  O   PRO A  18      18.588  10.059  -9.359  1.00  0.37           O  \\nATOM    138  CG  PRO A  18      18.765   7.483  -7.413  1.00  0.35           C  \\nATOM    139  CD  PRO A  18      17.968   6.829  -8.505  1.00  0.41           C  \\nATOM    140  N   THR A  19      16.652  10.984  -9.367  1.00  0.35           N  \\nATOM    141  CA  THR A  19      16.871  12.360  -9.799  1.00  0.37           C  \\nATOM    142  C   THR A  19      17.816  13.083  -8.843  1.00  0.35           C  \\nATOM    143  CB  THR A  19      15.543  13.134  -9.895  1.00  0.33           C  \\nATOM    144  O   THR A  19      17.571  13.128  -7.636  1.00  0.34           O  \\nATOM    145  CG2 THR A  19      15.242  13.533 -11.336  1.00  0.32           C  \\nATOM    146  OG1 THR A  19      14.479  12.307  -9.408  1.00  0.31           O  \\nATOM    147  N   GLN A  20      18.986  13.116  -9.146  1.00  0.41           N  \\nATOM    148  CA  GLN A  20      19.956  14.126  -8.735  1.00  0.44           C  \\nATOM    149  C   GLN A  20      19.353  15.526  -8.801  1.00  0.42           C  \\nATOM    150  CB  GLN A  20      21.210  14.053  -9.608  1.00  0.39           C  \\nATOM    151  O   GLN A  20      18.637  15.856  -9.748  1.00  0.40           O  \\nATOM    152  CG  GLN A  20      22.192  12.969  -9.184  1.00  0.40           C  \\nATOM    153  CD  GLN A  20      23.540  13.098  -9.869  1.00  0.47           C  \\nATOM    154  NE2 GLN A  20      24.533  12.374  -9.364  1.00  0.36           N  \\nATOM    155  OE1 GLN A  20      23.687  13.840 -10.845  1.00  0.51           O  \\nATOM    156  N   PRO A  21      18.981  16.251  -7.755  1.00  0.42           N  \\nATOM    157  CA  PRO A  21      18.885  17.711  -7.684  1.00  0.43           C  \\nATOM    158  C   PRO A  21      19.704  18.409  -8.767  1.00  0.42           C  \\nATOM    159  CB  PRO A  21      19.432  18.033  -6.291  1.00  0.39           C  \\nATOM    160  O   PRO A  21      20.926  18.245  -8.824  1.00  0.40           O  \\nATOM    161  CG  PRO A  21      19.959  16.733  -5.777  1.00  0.40           C  \\nATOM    162  CD  PRO A  21      19.642  15.660  -6.780  1.00  0.46           C  \\nATOM    163  N   GLY A  22      19.171  18.688  -9.986  1.00  0.38           N  \\nATOM    164  CA  GLY A  22      19.829  19.689 -10.810  1.00  0.43           C  \\nATOM    165  C   GLY A  22      19.621  19.469 -12.297  1.00  0.38           C  \\nATOM    166  O   GLY A  22      20.301  20.084 -13.122  1.00  0.36           O  \\nATOM    167  N   GLN A  23      18.497  18.863 -12.822  1.00  0.36           N  \\nATOM    168  CA  GLN A  23      18.354  18.906 -14.273  1.00  0.41           C  \\nATOM    169  C   GLN A  23      16.922  19.245 -14.675  1.00  0.37           C  \\nATOM    170  CB  GLN A  23      18.773  17.573 -14.894  1.00  0.33           C  \\nATOM    171  O   GLN A  23      15.974  18.605 -14.214  1.00  0.35           O  \\nATOM    172  CG  GLN A  23      20.275  17.433 -15.098  1.00  0.34           C  \\nATOM    173  CD  GLN A  23      20.642  16.245 -15.966  1.00  0.41           C  \\nATOM    174  NE2 GLN A  23      21.926  16.116 -16.284  1.00  0.34           N  \\nATOM    175  OE1 GLN A  23      19.779  15.447 -16.347  1.00  0.47           O  \\nATOM    176  N   GLY A  24      16.620  20.606 -14.994  1.00  0.42           N  \\nATOM    177  CA  GLY A  24      15.883  21.666 -15.664  1.00  0.45           C  \\nATOM    178  C   GLY A  24      14.860  21.146 -16.656  1.00  0.42           C  \\nATOM    179  O   GLY A  24      15.178  20.308 -17.502  1.00  0.40           O  \\nATOM    180  N   TYR A  25      13.531  20.962 -16.207  1.00  0.39           N  \\nATOM    181  CA  TYR A  25      12.142  20.936 -16.651  1.00  0.43           C  \\nATOM    182  C   TYR A  25      12.012  21.483 -18.068  1.00  0.39           C  \\nATOM    183  CB  TYR A  25      11.256  21.742 -15.697  1.00  0.37           C  \\nATOM    184  O   TYR A  25      12.233  22.673 -18.303  1.00  0.38           O  \\nATOM    185  CG  TYR A  25      11.113  21.120 -14.329  1.00  0.38           C  \\nATOM    186  CD1 TYR A  25      10.191  20.101 -14.099  1.00  0.34           C  \\nATOM    187  CD2 TYR A  25      11.899  21.549 -13.265  1.00  0.36           C  \\nATOM    188  CE1 TYR A  25      10.054  19.526 -12.840  1.00  0.34           C  \\nATOM    189  CE2 TYR A  25      11.771  20.980 -12.002  1.00  0.40           C  \\nATOM    190  OH  TYR A  25      10.716  19.405 -10.551  1.00  0.24           O  \\nATOM    191  CZ  TYR A  25      10.847  19.971 -11.799  1.00  0.31           C  \\nATOM    192  N   SER A  26      12.091  20.682 -19.070  1.00  0.42           N  \\nATOM    193  CA  SER A  26      11.567  21.030 -20.386  1.00  0.47           C  \\nATOM    194  C   SER A  26      10.193  20.410 -20.617  1.00  0.42           C  \\nATOM    195  CB  SER A  26      12.531  20.577 -21.484  1.00  0.39           C  \\nATOM    196  O   SER A  26      10.010  19.206 -20.423  1.00  0.41           O  \\nATOM    197  OG  SER A  26      13.418  19.585 -20.997  1.00  0.39           O  \\nATOM    198  N   GLN A  27       9.006  21.106 -20.239  1.00  0.44           N  \\nATOM    199  CA  GLN A  27       7.612  21.172 -20.664  1.00  0.47           C  \\nATOM    200  C   GLN A  27       7.401  20.425 -21.977  1.00  0.44           C  \\nATOM    201  CB  GLN A  27       7.164  22.627 -20.810  1.00  0.41           C  \\nATOM    202  O   GLN A  27       7.843  20.880 -23.035  1.00  0.43           O  \\nATOM    203  CG  GLN A  27       6.695  23.260 -19.507  1.00  0.42           C  \\nATOM    204  CD  GLN A  27       6.251  24.700 -19.680  1.00  0.47           C  \\nATOM    205  NE2 GLN A  27       5.793  25.314 -18.594  1.00  0.38           N  \\nATOM    206  OE1 GLN A  27       6.320  25.257 -20.780  1.00  0.52           O  \\nATOM    207  N   GLN A  28       7.378  19.093 -21.847  1.00  0.41           N  \\nATOM    208  CA  GLN A  28       6.654  18.400 -22.908  1.00  0.45           C  \\nATOM    209  C   GLN A  28       5.244  18.027 -22.460  1.00  0.41           C  \\nATOM    210  CB  GLN A  28       7.414  17.147 -23.347  1.00  0.39           C  \\nATOM    211  O   GLN A  28       5.064  17.404 -21.412  1.00  0.41           O  \\nATOM    212  CG  GLN A  28       8.127  17.297 -24.684  1.00  0.39           C  \\nATOM    213  CD  GLN A  28       8.934  16.069 -25.060  1.00  0.44           C  \\nATOM    214  NE2 GLN A  28       9.621  16.135 -26.195  1.00  0.37           N  \\nATOM    215  OE1 GLN A  28       8.941  15.069 -24.335  1.00  0.49           O  \\nATOM    216  N   SER A  29       4.224  18.899 -22.791  1.00  0.36           N  \\nATOM    217  CA  SER A  29       2.785  18.722 -22.950  1.00  0.40           C  \\nATOM    218  C   SER A  29       2.405  17.245 -22.945  1.00  0.36           C  \\nATOM    219  CB  SER A  29       2.304  19.376 -24.246  1.00  0.35           C  \\nATOM    220  O   SER A  29       2.845  16.483 -23.808  1.00  0.36           O  \\nATOM    221  OG  SER A  29       2.029  18.395 -25.232  1.00  0.37           O  \\nATOM    222  N   SER A  30       2.459  16.548 -21.905  1.00  0.38           N  \\nATOM    223  CA  SER A  30       1.714  15.318 -21.654  1.00  0.40           C  \\nATOM    224  C   SER A  30       0.264  15.448 -22.107  1.00  0.39           C  \\nATOM    225  CB  SER A  30       1.763  14.955 -20.170  1.00  0.37           C  \\nATOM    226  O   SER A  30      -0.488  16.265 -21.572  1.00  0.38           O  \\nATOM    227  OG  SER A  30       1.317  16.039 -19.372  1.00  0.36           O  \\nATOM    228  N   GLN A  31       0.017  15.512 -23.378  1.00  0.40           N  \\nATOM    229  CA  GLN A  31      -1.310  15.085 -23.806  1.00  0.44           C  \\nATOM    230  C   GLN A  31      -1.896  14.058 -22.842  1.00  0.41           C  \\nATOM    231  CB  GLN A  31      -1.258  14.506 -25.221  1.00  0.39           C  \\nATOM    232  O   GLN A  31      -1.203  13.130 -22.418  1.00  0.40           O  \\nATOM    233  CG  GLN A  31      -1.060  15.554 -26.308  1.00  0.40           C  \\nATOM    234  CD  GLN A  31      -1.790  15.211 -27.593  1.00  0.46           C  \\nATOM    235  NE2 GLN A  31      -2.222  16.236 -28.321  1.00  0.37           N  \\nATOM    236  OE1 GLN A  31      -1.962  14.036 -27.930  1.00  0.51           O  \\nATOM    237  N   PRO A  32      -2.691  14.342 -21.929  1.00  0.43           N  \\nATOM    238  CA  PRO A  32      -3.487  13.355 -21.195  1.00  0.43           C  \\nATOM    239  C   PRO A  32      -3.714  12.071 -21.989  1.00  0.43           C  \\nATOM    240  CB  PRO A  32      -4.810  14.084 -20.945  1.00  0.41           C  \\nATOM    241  O   PRO A  32      -3.975  12.125 -23.194  1.00  0.42           O  \\nATOM    242  CG  PRO A  32      -4.874  15.134 -22.008  1.00  0.40           C  \\nATOM    243  CD  PRO A  32      -3.480  15.403 -22.497  1.00  0.45           C  \\nATOM    244  N   TYR A  33      -2.993  10.900 -22.083  1.00  0.40           N  \\nATOM    245  CA  TYR A  33      -3.651   9.857 -22.862  1.00  0.43           C  \\nATOM    246  C   TYR A  33      -4.941  10.374 -23.488  1.00  0.40           C  \\nATOM    247  CB  TYR A  33      -3.948   8.637 -21.985  1.00  0.39           C  \\nATOM    248  O   TYR A  33      -5.856  10.797 -22.778  1.00  0.39           O  \\nATOM    249  CG  TYR A  33      -2.753   7.742 -21.764  1.00  0.39           C  \\nATOM    250  CD1 TYR A  33      -2.424   6.746 -22.681  1.00  0.39           C  \\nATOM    251  CD2 TYR A  33      -1.950   7.890 -20.638  1.00  0.39           C  \\nATOM    252  CE1 TYR A  33      -1.325   5.918 -22.480  1.00  0.38           C  \\nATOM    253  CE2 TYR A  33      -0.848   7.068 -20.427  1.00  0.41           C  \\nATOM    254  OH  TYR A  33       0.546   5.270 -21.149  1.00  0.23           O  \\nATOM    255  CZ  TYR A  33      -0.544   6.087 -21.352  1.00  0.36           C  \\nATOM    256  N   GLY A  34      -4.938  11.374 -24.324  1.00  0.35           N  \\nATOM    257  CA  GLY A  34      -6.107  11.621 -25.153  1.00  0.35           C  \\nATOM    258  C   GLY A  34      -7.363  10.953 -24.627  1.00  0.35           C  \\nATOM    259  O   GLY A  34      -7.389   9.737 -24.427  1.00  0.33           O  \\nATOM    260  N   GLN A  35      -8.036  11.223 -23.499  1.00  0.37           N  \\nATOM    261  CA  GLN A  35      -9.398  11.112 -24.012  1.00  0.39           C  \\nATOM    262  C   GLN A  35      -9.400  10.866 -25.518  1.00  0.38           C  \\nATOM    263  CB  GLN A  35     -10.199  12.373 -23.685  1.00  0.35           C  \\nATOM    264  O   GLN A  35      -9.112  11.774 -26.301  1.00  0.35           O  \\nATOM    265  CG  GLN A  35     -10.985  12.282 -22.384  1.00  0.35           C  \\nATOM    266  CD  GLN A  35     -11.720  13.567 -22.051  1.00  0.41           C  \\nATOM    267  NE2 GLN A  35     -12.636  13.494 -21.090  1.00  0.35           N  \\nATOM    268  OE1 GLN A  35     -11.467  14.616 -22.652  1.00  0.46           O  \\nATOM    269  N   GLN A  36      -8.705   9.918 -26.078  1.00  0.36           N  \\nATOM    270  CA  GLN A  36      -9.606   9.619 -27.186  1.00  0.38           C  \\nATOM    271  C   GLN A  36     -11.058   9.895 -26.804  1.00  0.36           C  \\nATOM    272  CB  GLN A  36      -9.447   8.164 -27.629  1.00  0.34           C  \\nATOM    273  O   GLN A  36     -11.579   9.305 -25.856  1.00  0.33           O  \\nATOM    274  CG  GLN A  36      -8.264   7.930 -28.558  1.00  0.34           C  \\nATOM    275  CD  GLN A  36      -8.219   6.516 -29.106  1.00  0.41           C  \\nATOM    276  NE2 GLN A  36      -7.229   6.238 -29.948  1.00  0.33           N  \\nATOM    277  OE1 GLN A  36      -9.065   5.680 -28.775  1.00  0.45           O  \\nATOM    278  N   SER A  37     -11.516  11.061 -26.547  1.00  0.33           N  \\nATOM    279  CA  SER A  37     -12.853  11.230 -27.106  1.00  0.35           C  \\nATOM    280  C   SER A  37     -13.167  10.140 -28.126  1.00  0.33           C  \\nATOM    281  CB  SER A  37     -12.992  12.606 -27.759  1.00  0.32           C  \\nATOM    282  O   SER A  37     -12.499  10.036 -29.157  1.00  0.31           O  \\nATOM    283  OG  SER A  37     -11.781  13.335 -27.659  1.00  0.32           O  \\nATOM    284  N   TYR A  38     -13.272   8.886 -27.807  1.00  0.36           N  \\nATOM    285  CA  TYR A  38     -14.283   8.171 -28.578  1.00  0.40           C  \\nATOM    286  C   TYR A  38     -15.341   9.129 -29.112  1.00  0.36           C  \\nATOM    287  CB  TYR A  38     -14.945   7.087 -27.722  1.00  0.35           C  \\nATOM    288  O   TYR A  38     -16.297   9.463 -28.407  1.00  0.35           O  \\nATOM    289  CG  TYR A  38     -14.188   5.781 -27.704  1.00  0.36           C  \\nATOM    290  CD1 TYR A  38     -14.429   4.801 -28.665  1.00  0.36           C  \\nATOM    291  CD2 TYR A  38     -13.233   5.524 -26.727  1.00  0.37           C  \\nATOM    292  CE1 TYR A  38     -13.735   3.596 -28.652  1.00  0.37           C  \\nATOM    293  CE2 TYR A  38     -12.534   4.322 -26.704  1.00  0.40           C  \\nATOM    294  OH  TYR A  38     -12.102   2.174 -27.651  1.00  0.25           O  \\nATOM    295  CZ  TYR A  38     -12.791   3.365 -27.669  1.00  0.34           C  \\nATOM    296  N   SER A  39     -15.047  10.215 -29.530  1.00  0.32           N  \\nATOM    297  CA  SER A  39     -15.905  10.903 -30.489  1.00  0.36           C  \\nATOM    298  C   SER A  39     -15.688  10.376 -31.903  1.00  0.33           C  \\nATOM    299  CB  SER A  39     -15.649  12.410 -30.452  1.00  0.31           C  \\nATOM    300  O   SER A  39     -15.155  11.085 -32.760  1.00  0.31           O  \\nATOM    301  OG  SER A  39     -14.353  12.712 -30.939  1.00  0.31           O  \\nATOM    302  N   GLY A  40     -15.639   9.035 -31.983  1.00  0.32           N  \\nATOM    303  CA  GLY A  40     -15.785   8.437 -33.301  1.00  0.36           C  \\nATOM    304  C   GLY A  40     -16.037   6.942 -33.253  1.00  0.33           C  \\nATOM    305  O   GLY A  40     -15.735   6.225 -34.209  1.00  0.31           O  \\nATOM    306  N   TYR A  41     -16.814   6.518 -32.355  1.00  0.34           N  \\nATOM    307  CA  TYR A  41     -17.569   5.302 -32.639  1.00  0.38           C  \\nATOM    308  C   TYR A  41     -19.042   5.482 -32.293  1.00  0.34           C  \\nATOM    309  CB  TYR A  41     -16.990   4.116 -31.862  1.00  0.32           C  \\nATOM    310  O   TYR A  41     -19.411   5.500 -31.116  1.00  0.33           O  \\nATOM    311  CG  TYR A  41     -15.855   3.420 -32.573  1.00  0.34           C  \\nATOM    312  CD1 TYR A  41     -16.102   2.404 -33.493  1.00  0.34           C  \\nATOM    313  CD2 TYR A  41     -14.534   3.776 -32.325  1.00  0.35           C  \\nATOM    314  CE1 TYR A  41     -15.059   1.758 -34.149  1.00  0.33           C  \\nATOM    315  CE2 TYR A  41     -13.483   3.138 -32.975  1.00  0.37           C  \\nATOM    316  OH  TYR A  41     -12.718   1.496 -34.530  1.00  0.16           O  \\nATOM    317  CZ  TYR A  41     -13.755   2.132 -33.884  1.00  0.31           C  \\nATOM    318  N   SER A  42     -19.813   6.378 -32.813  1.00  0.37           N  \\nATOM    319  CA  SER A  42     -21.127   6.042 -33.349  1.00  0.40           C  \\nATOM    320  C   SER A  42     -21.089   4.728 -34.122  1.00  0.37           C  \\nATOM    321  CB  SER A  42     -21.640   7.162 -34.256  1.00  0.36           C  \\nATOM    322  O   SER A  42     -20.632   4.688 -35.266  1.00  0.36           O  \\nATOM    323  OG  SER A  42     -22.856   7.694 -33.759  1.00  0.38           O  \\nATOM    324  N   GLN A  43     -20.645   3.684 -33.614  1.00  0.38           N  \\nATOM    325  CA  GLN A  43     -21.296   2.541 -34.247  1.00  0.42           C  \\nATOM    326  C   GLN A  43     -22.664   2.274 -33.627  1.00  0.38           C  \\nATOM    327  CB  GLN A  43     -20.416   1.295 -34.138  1.00  0.36           C  \\nATOM    328  O   GLN A  43     -22.770   2.050 -32.419  1.00  0.37           O  \\nATOM    329  CG  GLN A  43     -19.272   1.259 -35.142  1.00  0.37           C  \\nATOM    330  CD  GLN A  43     -18.445  -0.009 -35.045  1.00  0.44           C  \\nATOM    331  NE2 GLN A  43     -17.273  -0.002 -35.671  1.00  0.37           N  \\nATOM    332  OE1 GLN A  43     -18.855  -0.986 -34.410  1.00  0.48           O  \\nATOM    333  N   SER A  44     -23.721   2.852 -34.077  1.00  0.39           N  \\nATOM    334  CA  SER A  44     -25.005   2.204 -34.322  1.00  0.41           C  \\nATOM    335  C   SER A  44     -24.856   0.688 -34.384  1.00  0.39           C  \\nATOM    336  CB  SER A  44     -25.625   2.718 -35.622  1.00  0.37           C  \\nATOM    337  O   SER A  44     -24.339   0.150 -35.366  1.00  0.38           O  \\nATOM    338  OG  SER A  44     -24.814   2.380 -36.734  1.00  0.41           O  \\nATOM    339  N   THR A  45     -24.486   0.067 -33.382  1.00  0.41           N  \\nATOM    340  CA  THR A  45     -24.803  -1.356 -33.322  1.00  0.44           C  \\nATOM    341  C   THR A  45     -26.291  -1.568 -33.062  1.00  0.41           C  \\nATOM    342  CB  THR A  45     -23.982  -2.065 -32.228  1.00  0.40           C  \\nATOM    343  O   THR A  45     -26.791  -1.247 -31.982  1.00  0.41           O  \\nATOM    344  CG2 THR A  45     -22.604  -2.463 -32.746  1.00  0.39           C  \\nATOM    345  OG1 THR A  45     -23.824  -1.183 -31.110  1.00  0.35           O  \\nATOM    346  N   ASP A  46     -27.134  -1.407 -34.077  1.00  0.41           N  \\nATOM    347  CA  ASP A  46     -28.361  -2.176 -34.262  1.00  0.44           C  \\nATOM    348  C   ASP A  46     -28.161  -3.635 -33.857  1.00  0.41           C  \\nATOM    349  CB  ASP A  46     -28.833  -2.094 -35.715  1.00  0.39           C  \\nATOM    350  O   ASP A  46     -27.581  -4.420 -34.610  1.00  0.40           O  \\nATOM    351  CG  ASP A  46     -29.980  -1.118 -35.912  1.00  0.41           C  \\nATOM    352  OD1 ASP A  46     -30.658  -0.766 -34.923  1.00  0.43           O  \\nATOM    353  OD2 ASP A  46     -30.209  -0.699 -37.067  1.00  0.47           O  \\nATOM    354  N   THR A  47     -27.969  -3.955 -32.600  1.00  0.40           N  \\nATOM    355  CA  THR A  47     -28.007  -5.283 -31.998  1.00  0.42           C  \\nATOM    356  C   THR A  47     -29.436  -5.669 -31.629  1.00  0.40           C  \\nATOM    357  CB  THR A  47     -27.113  -5.353 -30.746  1.00  0.38           C  \\nATOM    358  O   THR A  47     -29.932  -5.292 -30.565  1.00  0.39           O  \\nATOM    359  CG2 THR A  47     -25.930  -6.289 -30.966  1.00  0.37           C  \\nATOM    360  OG1 THR A  47     -26.621  -4.042 -30.442  1.00  0.35           O  \\nATOM    361  N   SER A  48     -30.281  -5.701 -32.473  1.00  0.35           N  \\nATOM    362  CA  SER A  48     -31.546  -6.427 -32.509  1.00  0.39           C  \\nATOM    363  C   SER A  48     -31.317  -7.934 -32.566  1.00  0.36           C  \\nATOM    364  CB  SER A  48     -32.385  -5.984 -33.708  1.00  0.34           C  \\nATOM    365  O   SER A  48     -30.653  -8.431 -33.478  1.00  0.34           O  \\nATOM    366  OG  SER A  48     -31.628  -6.055 -34.905  1.00  0.37           O  \\nATOM    367  N   GLY A  49     -30.859  -8.641 -31.382  1.00  0.32           N  \\nATOM    368  CA  GLY A  49     -31.309 -10.024 -31.365  1.00  0.38           C  \\nATOM    369  C   GLY A  49     -30.167 -11.022 -31.338  1.00  0.32           C  \\nATOM    370  O   GLY A  49     -29.440 -11.167 -32.323  1.00  0.32           O  \\nATOM    371  N   TYR A  50     -29.429 -11.176 -30.168  1.00  0.38           N  \\nATOM    372  CA  TYR A  50     -28.889 -12.491 -29.840  1.00  0.43           C  \\nATOM    373  C   TYR A  50     -28.483 -12.564 -28.373  1.00  0.39           C  \\nATOM    374  CB  TYR A  50     -27.685 -12.815 -30.731  1.00  0.36           C  \\nATOM    375  O   TYR A  50     -27.722 -11.722 -27.890  1.00  0.38           O  \\nATOM    376  CG  TYR A  50     -28.057 -13.150 -32.155  1.00  0.37           C  \\nATOM    377  CD1 TYR A  50     -28.472 -14.433 -32.504  1.00  0.35           C  \\nATOM    378  CD2 TYR A  50     -27.994 -12.184 -33.154  1.00  0.36           C  \\nATOM    379  CE1 TYR A  50     -28.813 -14.746 -33.816  1.00  0.34           C  \\nATOM    380  CE2 TYR A  50     -28.334 -12.486 -34.469  1.00  0.38           C  \\nATOM    381  OH  TYR A  50     -29.079 -14.073 -36.089  1.00  0.23           O  \\nATOM    382  CZ  TYR A  50     -28.742 -13.768 -34.789  1.00  0.32           C  \\nATOM    383  N   GLY A  51     -29.357 -13.189 -27.555  1.00  0.37           N  \\nATOM    384  CA  GLY A  51     -29.601 -14.169 -26.509  1.00  0.40           C  \\nATOM    385  C   GLY A  51     -28.562 -14.135 -25.405  1.00  0.37           C  \\nATOM    386  O   GLY A  51     -27.359 -14.155 -25.677  1.00  0.37           O  \\nATOM    387  N   GLN A  52     -28.796 -13.404 -24.311  1.00  0.38           N  \\nATOM    388  CA  GLN A  52     -28.670 -13.396 -22.857  1.00  0.42           C  \\nATOM    389  C   GLN A  52     -27.678 -14.456 -22.385  1.00  0.39           C  \\nATOM    390  CB  GLN A  52     -30.032 -13.622 -22.199  1.00  0.36           C  \\nATOM    391  O   GLN A  52     -27.956 -15.654 -22.467  1.00  0.38           O  \\nATOM    392  CG  GLN A  52     -30.643 -12.362 -21.601  1.00  0.38           C  \\nATOM    393  CD  GLN A  52     -32.033 -12.593 -21.039  1.00  0.46           C  \\nATOM    394  NE2 GLN A  52     -32.666 -11.525 -20.565  1.00  0.37           N  \\nATOM    395  OE1 GLN A  52     -32.535 -13.721 -21.033  1.00  0.50           O  \\nATOM    396  N   SER A  53     -26.358 -14.115 -22.163  1.00  0.41           N  \\nATOM    397  CA  SER A  53     -25.495 -14.871 -21.262  1.00  0.44           C  \\nATOM    398  C   SER A  53     -25.186 -14.077 -19.997  1.00  0.41           C  \\nATOM    399  CB  SER A  53     -24.192 -15.255 -21.964  1.00  0.39           C  \\nATOM    400  O   SER A  53     -24.719 -12.939 -20.071  1.00  0.41           O  \\nATOM    401  OG  SER A  53     -23.403 -14.107 -22.223  1.00  0.39           O  \\nATOM    402  N   SER A  54     -25.875 -14.244 -18.897  1.00  0.38           N  \\nATOM    403  CA  SER A  54     -25.772 -14.091 -17.450  1.00  0.41           C  \\nATOM    404  C   SER A  54     -24.332 -14.261 -16.977  1.00  0.38           C  \\nATOM    405  CB  SER A  54     -26.674 -15.101 -16.738  1.00  0.37           C  \\nATOM    406  O   SER A  54     -23.789 -15.367 -17.009  1.00  0.38           O  \\nATOM    407  OG  SER A  54     -26.405 -16.418 -17.187  1.00  0.37           O  \\nATOM    408  N   TYR A  55     -23.457 -13.297 -16.959  1.00  0.42           N  \\nATOM    409  CA  TYR A  55     -22.185 -13.288 -16.245  1.00  0.45           C  \\nATOM    410  C   TYR A  55     -22.298 -12.507 -14.941  1.00  0.42           C  \\nATOM    411  CB  TYR A  55     -21.081 -12.688 -17.120  1.00  0.40           C  \\nATOM    412  O   TYR A  55     -22.583 -11.307 -14.952  1.00  0.42           O  \\nATOM    413  CG  TYR A  55     -20.553 -13.637 -18.168  1.00  0.40           C  \\nATOM    414  CD1 TYR A  55     -19.530 -14.535 -17.870  1.00  0.39           C  \\nATOM    415  CD2 TYR A  55     -21.076 -13.639 -19.456  1.00  0.40           C  \\nATOM    416  CE1 TYR A  55     -19.039 -15.410 -18.833  1.00  0.38           C  \\nATOM    417  CE2 TYR A  55     -20.592 -14.510 -20.427  1.00  0.42           C  \\nATOM    418  OH  TYR A  55     -19.094 -16.255 -21.064  1.00  0.28           O  \\nATOM    419  CZ  TYR A  55     -19.576 -15.391 -20.107  1.00  0.36           C  \\nATOM    420  N   SER A  56     -22.483 -13.153 -13.837  1.00  0.38           N  \\nATOM    421  CA  SER A  56     -22.520 -13.048 -12.382  1.00  0.41           C  \\nATOM    422  C   SER A  56     -21.132 -12.778 -11.811  1.00  0.39           C  \\nATOM    423  CB  SER A  56     -23.094 -14.325 -11.766  1.00  0.37           C  \\nATOM    424  O   SER A  56     -20.167 -13.460 -12.163  1.00  0.38           O  \\nATOM    425  OG  SER A  56     -22.305 -15.450 -12.114  1.00  0.37           O  \\nATOM    426  N   SER A  57     -20.639 -11.408 -11.702  1.00  0.35           N  \\nATOM    427  CA  SER A  57     -20.112 -10.768 -10.501  1.00  0.39           C  \\nATOM    428  C   SER A  57     -18.608 -10.987 -10.371  1.00  0.35           C  \\nATOM    429  CB  SER A  57     -20.822 -11.300  -9.255  1.00  0.34           C  \\nATOM    430  O   SER A  57     -18.134 -12.123 -10.427  1.00  0.35           O  \\nATOM    431  OG  SER A  57     -20.643 -12.700  -9.134  1.00  0.35           O  \\nATOM    432  N   TYR A  58     -17.799 -10.037 -11.060  1.00  0.39           N  \\nATOM    433  CA  TYR A  58     -16.605  -9.478 -10.437  1.00  0.42           C  \\nATOM    434  C   TYR A  58     -15.578  -9.077 -11.488  1.00  0.39           C  \\nATOM    435  CB  TYR A  58     -15.987 -10.484  -9.460  1.00  0.37           C  \\nATOM    436  O   TYR A  58     -15.122  -9.914 -12.271  1.00  0.39           O  \\nATOM    437  CG  TYR A  58     -16.666 -10.517  -8.113  1.00  0.39           C  \\nATOM    438  CD1 TYR A  58     -16.320  -9.610  -7.114  1.00  0.38           C  \\nATOM    439  CD2 TYR A  58     -17.653 -11.456  -7.834  1.00  0.40           C  \\nATOM    440  CE1 TYR A  58     -16.941  -9.638  -5.870  1.00  0.37           C  \\nATOM    441  CE2 TYR A  58     -18.281 -11.493  -6.594  1.00  0.41           C  \\nATOM    442  OH  TYR A  58     -18.538 -10.614  -4.390  1.00  0.26           O  \\nATOM    443  CZ  TYR A  58     -17.919 -10.581  -5.620  1.00  0.36           C  \\nATOM    444  N   GLY A  59     -15.759  -7.779 -11.960  1.00  0.37           N  \\nATOM    445  CA  GLY A  59     -14.685  -6.810 -12.109  1.00  0.40           C  \\nATOM    446  C   GLY A  59     -14.447  -6.400 -13.549  1.00  0.37           C  \\nATOM    447  O   GLY A  59     -14.358  -7.251 -14.436  1.00  0.36           O  \\nATOM    448  N   GLN A  60     -15.173  -5.375 -13.903  1.00  0.42           N  \\nATOM    449  CA  GLN A  60     -15.287  -4.356 -14.941  1.00  0.44           C  \\nATOM    450  C   GLN A  60     -15.349  -4.988 -16.328  1.00  0.42           C  \\nATOM    451  CB  GLN A  60     -14.115  -3.376 -14.862  1.00  0.40           C  \\nATOM    452  O   GLN A  60     -14.337  -5.466 -16.844  1.00  0.41           O  \\nATOM    453  CG  GLN A  60     -14.382  -2.167 -13.975  1.00  0.40           C  \\nATOM    454  CD  GLN A  60     -13.200  -1.218 -13.905  1.00  0.47           C  \\nATOM    455  NE2 GLN A  60     -13.413  -0.049 -13.311  1.00  0.37           N  \\nATOM    456  OE1 GLN A  60     -12.105  -1.534 -14.380  1.00  0.49           O  \\nATOM    457  N   SER A  61     -16.395  -5.753 -16.512  1.00  0.36           N  \\nATOM    458  CA  SER A  61     -17.000  -5.933 -17.827  1.00  0.39           C  \\nATOM    459  C   SER A  61     -17.801  -4.702 -18.240  1.00  0.37           C  \\nATOM    460  CB  SER A  61     -17.904  -7.166 -17.837  1.00  0.35           C  \\nATOM    461  O   SER A  61     -18.745  -4.310 -17.552  1.00  0.35           O  \\nATOM    462  OG  SER A  61     -18.072  -7.656 -19.157  1.00  0.34           O  \\nATOM    463  N   GLN A  62     -17.228  -3.737 -18.684  1.00  0.40           N  \\nATOM    464  CA  GLN A  62     -18.064  -3.009 -19.633  1.00  0.42           C  \\nATOM    465  C   GLN A  62     -18.938  -3.964 -20.441  1.00  0.40           C  \\nATOM    466  CB  GLN A  62     -17.201  -2.165 -20.572  1.00  0.39           C  \\nATOM    467  O   GLN A  62     -18.449  -4.649 -21.342  1.00  0.39           O  \\nATOM    468  CG  GLN A  62     -16.517  -0.988 -19.888  1.00  0.39           C  \\nATOM    469  CD  GLN A  62     -15.763  -0.101 -20.861  1.00  0.45           C  \\nATOM    470  NE2 GLN A  62     -14.988   0.837 -20.329  1.00  0.37           N  \\nATOM    471  OE1 GLN A  62     -15.876  -0.258 -22.081  1.00  0.49           O  \\nATOM    472  N   ASN A  63     -19.702  -4.757 -19.760  1.00  0.33           N  \\nATOM    473  CA  ASN A  63     -20.934  -5.261 -20.357  1.00  0.38           C  \\nATOM    474  C   ASN A  63     -21.987  -4.163 -20.483  1.00  0.34           C  \\nATOM    475  CB  ASN A  63     -21.485  -6.434 -19.543  1.00  0.32           C  \\nATOM    476  O   ASN A  63     -22.503  -3.675 -19.476  1.00  0.32           O  \\nATOM    477  CG  ASN A  63     -22.238  -7.435 -20.398  1.00  0.33           C  \\nATOM    478  ND2 ASN A  63     -22.824  -8.439 -19.756  1.00  0.24           N  \\nATOM    479  OD1 ASN A  63     -22.293  -7.307 -21.623  1.00  0.35           O  \\nATOM    480  N   THR A  64     -21.870  -3.357 -21.514  1.00  0.36           N  \\nATOM    481  CA  THR A  64     -22.938  -2.664 -22.227  1.00  0.38           C  \\nATOM    482  C   THR A  64     -23.978  -3.655 -22.740  1.00  0.36           C  \\nATOM    483  CB  THR A  64     -22.382  -1.843 -23.405  1.00  0.34           C  \\nATOM    484  O   THR A  64     -23.775  -4.297 -23.773  1.00  0.35           O  \\nATOM    485  CG2 THR A  64     -22.124  -0.397 -22.995  1.00  0.34           C  \\nATOM    486  OG1 THR A  64     -21.152  -2.429 -23.849  1.00  0.32           O  \\nATOM    487  N   GLY A  65     -24.482  -4.530 -21.794  1.00  0.35           N  \\nATOM    488  CA  GLY A  65     -25.790  -5.040 -22.173  1.00  0.37           C  \\nATOM    489  C   GLY A  65     -26.877  -4.698 -21.172  1.00  0.35           C  \\nATOM    490  O   GLY A  65     -26.783  -5.061 -19.997  1.00  0.34           O  \\nATOM    491  N   TYR A  66     -27.485  -3.564 -21.281  1.00  0.37           N  \\nATOM    492  CA  TYR A  66     -28.878  -3.200 -21.052  1.00  0.42           C  \\nATOM    493  C   TYR A  66     -29.819  -4.251 -21.628  1.00  0.38           C  \\nATOM    494  CB  TYR A  66     -29.185  -1.831 -21.667  1.00  0.36           C  \\nATOM    495  O   TYR A  66     -29.877  -4.441 -22.845  1.00  0.37           O  \\nATOM    496  CG  TYR A  66     -28.496  -0.684 -20.970  1.00  0.39           C  \\nATOM    497  CD1 TYR A  66     -29.014  -0.145 -19.795  1.00  0.37           C  \\nATOM    498  CD2 TYR A  66     -27.326  -0.136 -21.484  1.00  0.37           C  \\nATOM    499  CE1 TYR A  66     -28.384   0.914 -19.149  1.00  0.33           C  \\nATOM    500  CE2 TYR A  66     -26.687   0.923 -20.847  1.00  0.39           C  \\nATOM    501  OH  TYR A  66     -26.594   2.488 -19.047  1.00  0.21           O  \\nATOM    502  CZ  TYR A  66     -27.222   1.440 -19.682  1.00  0.32           C  \\nATOM    503  N   GLY A  67     -29.968  -5.309 -20.863  1.00  0.36           N  \\nATOM    504  CA  GLY A  67     -31.340  -5.791 -20.868  1.00  0.39           C  \\nATOM    505  C   GLY A  67     -31.754  -6.427 -19.555  1.00  0.37           C  \\nATOM    506  O   GLY A  67     -31.019  -7.243 -18.995  1.00  0.36           O  \\nATOM    507  N   THR A  68     -32.412  -5.583 -18.733  1.00  0.38           N  \\nATOM    508  CA  THR A  68     -33.538  -5.956 -17.883  1.00  0.43           C  \\nATOM    509  C   THR A  68     -33.852  -7.443 -18.024  1.00  0.38           C  \\nATOM    510  CB  THR A  68     -34.792  -5.130 -18.222  1.00  0.37           C  \\nATOM    511  O   THR A  68     -34.207  -7.908 -19.110  1.00  0.38           O  \\nATOM    512  CG2 THR A  68     -35.045  -4.057 -17.168  1.00  0.38           C  \\nATOM    513  OG1 THR A  68     -34.612  -4.499 -19.496  1.00  0.35           O  \\nATOM    514  N   GLN A  69     -33.106  -8.205 -17.252  1.00  0.37           N  \\nATOM    515  CA  GLN A  69     -34.019  -9.198 -16.697  1.00  0.42           C  \\nATOM    516  C   GLN A  69     -33.276 -10.197 -15.814  1.00  0.38           C  \\nATOM    517  CB  GLN A  69     -34.757  -9.935 -17.816  1.00  0.35           C  \\nATOM    518  O   GLN A  69     -32.248 -10.745 -16.218  1.00  0.37           O  \\nATOM    519  CG  GLN A  69     -36.011  -9.221 -18.300  1.00  0.37           C  \\nATOM    520  CD  GLN A  69     -36.754  -9.999 -19.370  1.00  0.45           C  \\nATOM    521  NE2 GLN A  69     -37.803  -9.398 -19.920  1.00  0.35           N  \\nATOM    522  OE1 GLN A  69     -36.388 -11.132 -19.699  1.00  0.49           O  \\nATOM    523  N   SER A  70     -33.538 -10.031 -14.518  1.00  0.39           N  \\nATOM    524  CA  SER A  70     -34.043 -10.866 -13.433  1.00  0.41           C  \\nATOM    525  C   SER A  70     -32.951 -11.777 -12.882  1.00  0.39           C  \\nATOM    526  CB  SER A  70     -35.227 -11.708 -13.911  1.00  0.37           C  \\nATOM    527  O   SER A  70     -32.268 -12.466 -13.643  1.00  0.38           O  \\nATOM    528  OG  SER A  70     -34.899 -12.410 -15.098  1.00  0.41           O  \\nATOM    529  N   THR A  71     -32.210 -11.420 -11.828  1.00  0.41           N  \\nATOM    530  CA  THR A  71     -31.822 -12.093 -10.593  1.00  0.42           C  \\nATOM    531  C   THR A  71     -31.884 -13.609 -10.761  1.00  0.41           C  \\nATOM    532  CB  THR A  71     -32.721 -11.667  -9.418  1.00  0.39           C  \\nATOM    533  O   THR A  71     -32.953 -14.168 -11.013  1.00  0.40           O  \\nATOM    534  CG2 THR A  71     -31.907 -10.993  -8.319  1.00  0.38           C  \\nATOM    535  OG1 THR A  71     -33.715 -10.750  -9.893  1.00  0.39           O  \\nATOM    536  N   PRO A  72     -30.739 -14.298 -11.001  1.00  0.45           N  \\nATOM    537  CA  PRO A  72     -30.875 -15.678 -10.530  1.00  0.46           C  \\nATOM    538  C   PRO A  72     -30.954 -15.777  -9.008  1.00  0.45           C  \\nATOM    539  CB  PRO A  72     -29.607 -16.356 -11.055  1.00  0.42           C  \\nATOM    540  O   PRO A  72     -30.071 -15.278  -8.306  1.00  0.44           O  \\nATOM    541  CG  PRO A  72     -28.778 -15.242 -11.608  1.00  0.42           C  \\nATOM    542  CD  PRO A  72     -29.574 -13.970 -11.526  1.00  0.48           C  \\nATOM    543  N   GLN A  73     -31.981 -15.903  -8.412  1.00  0.38           N  \\nATOM    544  CA  GLN A  73     -32.495 -16.788  -7.371  1.00  0.42           C  \\nATOM    545  C   GLN A  73     -31.833 -18.161  -7.440  1.00  0.38           C  \\nATOM    546  CB  GLN A  73     -34.013 -16.930  -7.488  1.00  0.36           C  \\nATOM    547  O   GLN A  73     -31.785 -18.780  -8.505  1.00  0.37           O  \\nATOM    548  CG  GLN A  73     -34.782 -16.260  -6.358  1.00  0.38           C  \\nATOM    549  CD  GLN A  73     -36.285 -16.403  -6.505  1.00  0.47           C  \\nATOM    550  NE2 GLN A  73     -37.018 -16.086  -5.443  1.00  0.37           N  \\nATOM    551  OE1 GLN A  73     -36.784 -16.795  -7.565  1.00  0.51           O  \\nATOM    552  N   GLY A  74     -30.768 -18.458  -6.498  1.00  0.39           N  \\nATOM    553  CA  GLY A  74     -30.801 -19.704  -5.749  1.00  0.42           C  \\nATOM    554  C   GLY A  74     -29.871 -20.763  -6.311  1.00  0.39           C  \\nATOM    555  O   GLY A  74     -30.138 -21.332  -7.371  1.00  0.38           O  \\nATOM    556  N   TYR A  75     -28.452 -20.712  -5.891  1.00  0.40           N  \\nATOM    557  CA  TYR A  75     -27.901 -22.002  -5.493  1.00  0.45           C  \\nATOM    558  C   TYR A  75     -27.187 -21.900  -4.150  1.00  0.40           C  \\nATOM    559  CB  TYR A  75     -26.935 -22.527  -6.559  1.00  0.37           C  \\nATOM    560  O   TYR A  75     -26.262 -21.100  -3.992  1.00  0.39           O  \\nATOM    561  CG  TYR A  75     -27.624 -23.094  -7.776  1.00  0.39           C  \\nATOM    562  CD1 TYR A  75     -28.066 -24.414  -7.799  1.00  0.36           C  \\nATOM    563  CD2 TYR A  75     -27.834 -22.310  -8.906  1.00  0.38           C  \\nATOM    564  CE1 TYR A  75     -28.700 -24.942  -8.919  1.00  0.36           C  \\nATOM    565  CE2 TYR A  75     -28.467 -22.827 -10.032  1.00  0.41           C  \\nATOM    566  OH  TYR A  75     -29.523 -24.658 -11.141  1.00  0.25           O  \\nATOM    567  CZ  TYR A  75     -28.896 -24.141 -10.029  1.00  0.32           C  \\nATOM    568  N   GLY A  76     -27.597 -22.509  -3.113  1.00  0.37           N  \\nATOM    569  CA  GLY A  76     -27.697 -23.594  -2.150  1.00  0.41           C  \\nATOM    570  C   GLY A  76     -26.707 -24.714  -2.409  1.00  0.37           C  \\nATOM    571  O   GLY A  76     -26.708 -25.314  -3.486  1.00  0.36           O  \\nATOM    572  N   SER A  77     -25.384 -24.633  -1.770  1.00  0.34           N  \\nATOM    573  CA  SER A  77     -25.070 -25.844  -1.019  1.00  0.38           C  \\nATOM    574  C   SER A  77     -23.572 -25.956  -0.753  1.00  0.34           C  \\nATOM    575  CB  SER A  77     -25.556 -27.083  -1.772  1.00  0.31           C  \\nATOM    576  O   SER A  77     -22.762 -25.838  -1.675  1.00  0.33           O  \\nATOM    577  OG  SER A  77     -24.489 -27.991  -1.988  1.00  0.39           O  \\nATOM    578  N   THR A  78     -22.959 -25.823   0.550  1.00  0.32           N  \\nATOM    579  CA  THR A  78     -22.631 -26.752   1.627  1.00  0.37           C  \\nATOM    580  C   THR A  78     -21.375 -27.550   1.289  1.00  0.32           C  \\nATOM    581  CB  THR A  78     -23.798 -27.718   1.905  1.00  0.31           C  \\nATOM    582  O   THR A  78     -21.349 -28.288   0.302  1.00  0.32           O  \\nATOM    583  CG2 THR A  78     -24.751 -27.144   2.947  1.00  0.32           C  \\nATOM    584  OG1 THR A  78     -24.520 -27.949   0.689  1.00  0.33           O  \\nATOM    585  N   GLY A  79     -19.984 -27.115   1.920  1.00  0.37           N  \\nATOM    586  CA  GLY A  79     -19.199 -28.107   2.636  1.00  0.41           C  \\nATOM    587  C   GLY A  79     -17.706 -27.965   2.406  1.00  0.37           C  \\nATOM    588  O   GLY A  79     -17.252 -27.901   1.261  1.00  0.36           O  \\nATOM    589  N   GLY A  80     -16.809 -27.439   3.451  1.00  0.37           N  \\nATOM    590  CA  GLY A  80     -15.840 -27.967   4.398  1.00  0.42           C  \\nATOM    591  C   GLY A  80     -14.561 -28.447   3.738  1.00  0.37           C  \\nATOM    592  O   GLY A  80     -14.593 -29.338   2.887  1.00  0.36           O  \\nATOM    593  N   TYR A  81     -13.363 -27.407   3.707  1.00  0.41           N  \\nATOM    594  CA  TYR A  81     -12.002 -27.844   3.996  1.00  0.47           C  \\nATOM    595  C   TYR A  81     -10.997 -26.743   3.677  1.00  0.41           C  \\nATOM    596  CB  TYR A  81     -11.662 -29.108   3.200  1.00  0.38           C  \\nATOM    597  O   TYR A  81     -11.024 -26.167   2.587  1.00  0.40           O  \\nATOM    598  CG  TYR A  81     -12.268 -30.367   3.772  1.00  0.40           C  \\nATOM    599  CD1 TYR A  81     -11.636 -31.061   4.801  1.00  0.36           C  \\nATOM    600  CD2 TYR A  81     -13.471 -30.864   3.284  1.00  0.38           C  \\nATOM    601  CE1 TYR A  81     -12.190 -32.223   5.330  1.00  0.36           C  \\nATOM    602  CE2 TYR A  81     -14.034 -32.025   3.805  1.00  0.41           C  \\nATOM    603  OH  TYR A  81     -13.939 -33.845   5.346  1.00  0.24           O  \\nATOM    604  CZ  TYR A  81     -13.387 -32.696   4.827  1.00  0.31           C  \\nATOM    605  N   GLY A  82     -10.084 -26.198   4.677  1.00  0.40           N  \\nATOM    606  CA  GLY A  82      -8.871 -26.125   5.477  1.00  0.46           C  \\nATOM    607  C   GLY A  82      -7.609 -26.042   4.640  1.00  0.40           C  \\nATOM    608  O   GLY A  82      -7.309 -26.954   3.866  1.00  0.39           O  \\nATOM    609  N   SER A  83      -7.158 -24.772   4.229  1.00  0.36           N  \\nATOM    610  CA  SER A  83      -5.714 -24.615   4.090  1.00  0.43           C  \\nATOM    611  C   SER A  83      -5.360 -23.251   3.508  1.00  0.37           C  \\nATOM    612  CB  SER A  83      -5.137 -25.721   3.206  1.00  0.34           C  \\nATOM    613  O   SER A  83      -5.955 -22.819   2.518  1.00  0.36           O  \\nATOM    614  OG  SER A  83      -5.635 -25.619   1.883  1.00  0.41           O  \\nATOM    615  N   SER A  84      -4.538 -22.357   4.291  1.00  0.38           N  \\nATOM    616  CA  SER A  84      -3.291 -21.696   4.660  1.00  0.45           C  \\nATOM    617  C   SER A  84      -2.886 -20.658   3.619  1.00  0.38           C  \\nATOM    618  CB  SER A  84      -2.170 -22.722   4.832  1.00  0.36           C  \\nATOM    619  O   SER A  84      -2.900 -20.938   2.418  1.00  0.37           O  \\nATOM    620  OG  SER A  84      -2.208 -23.687   3.795  1.00  0.41           O  \\nATOM    621  N   GLN A  85      -2.859 -19.264   4.056  1.00  0.40           N  \\nATOM    622  CA  GLN A  85      -2.139 -18.013   4.269  1.00  0.47           C  \\nATOM    623  C   GLN A  85      -1.433 -17.560   2.995  1.00  0.40           C  \\nATOM    624  CB  GLN A  85      -1.125 -18.163   5.405  1.00  0.37           C  \\nATOM    625  O   GLN A  85      -0.492 -18.210   2.535  1.00  0.39           O  \\nATOM    626  CG  GLN A  85      -1.694 -17.849   6.782  1.00  0.39           C  \\nATOM    627  CD  GLN A  85      -0.665 -17.989   7.888  1.00  0.48           C  \\nATOM    628  NE2 GLN A  85      -1.075 -17.705   9.120  1.00  0.38           N  \\nATOM    629  OE1 GLN A  85       0.489 -18.350   7.638  1.00  0.53           O  \\nATOM    630  N   SER A  86      -2.162 -16.552   2.220  1.00  0.43           N  \\nATOM    631  CA  SER A  86      -1.415 -15.487   1.559  1.00  0.48           C  \\nATOM    632  C   SER A  86      -2.185 -14.170   1.590  1.00  0.43           C  \\nATOM    633  CB  SER A  86      -1.103 -15.870   0.112  1.00  0.40           C  \\nATOM    634  O   SER A  86      -3.350 -14.117   1.192  1.00  0.42           O  \\nATOM    635  OG  SER A  86      -2.297 -16.095  -0.618  1.00  0.42           O  \\nATOM    636  N   SER A  87      -1.788 -13.187   2.464  1.00  0.43           N  \\nATOM    637  CA  SER A  87      -1.868 -11.743   2.656  1.00  0.47           C  \\nATOM    638  C   SER A  87      -1.691 -10.999   1.337  1.00  0.43           C  \\nATOM    639  CB  SER A  87      -0.814 -11.276   3.660  1.00  0.41           C  \\nATOM    640  O   SER A  87      -0.596 -10.981   0.771  1.00  0.42           O  \\nATOM    641  OG  SER A  87       0.464 -11.787   3.321  1.00  0.43           O  \\nATOM    642  N   GLN A  88      -2.660 -10.802   0.553  1.00  0.44           N  \\nATOM    643  CA  GLN A  88      -2.815  -9.916  -0.595  1.00  0.47           C  \\nATOM    644  C   GLN A  88      -3.487  -8.606  -0.192  1.00  0.45           C  \\nATOM    645  CB  GLN A  88      -3.622 -10.602  -1.699  1.00  0.41           C  \\nATOM    646  O   GLN A  88      -4.584  -8.612   0.369  1.00  0.43           O  \\nATOM    647  CG  GLN A  88      -2.769 -11.142  -2.839  1.00  0.40           C  \\nATOM    648  CD  GLN A  88      -3.584 -11.886  -3.880  1.00  0.46           C  \\nATOM    649  NE2 GLN A  88      -2.937 -12.272  -4.974  1.00  0.38           N  \\nATOM    650  OE1 GLN A  88      -4.784 -12.114  -3.701  1.00  0.50           O  \\nATOM    651  N   SER A  89      -2.814  -7.508  -0.082  1.00  0.44           N  \\nATOM    652  CA  SER A  89      -2.951  -6.058   0.005  1.00  0.48           C  \\nATOM    653  C   SER A  89      -3.941  -5.533  -1.030  1.00  0.44           C  \\nATOM    654  CB  SER A  89      -1.594  -5.380  -0.186  1.00  0.41           C  \\nATOM    655  O   SER A  89      -3.820  -5.833  -2.220  1.00  0.42           O  \\nATOM    656  OG  SER A  89      -1.079  -5.639  -1.481  1.00  0.42           O  \\nATOM    657  N   SER A  90      -5.283  -5.336  -0.752  1.00  0.35           N  \\nATOM    658  CA  SER A  90      -6.591  -4.707  -0.902  1.00  0.39           C  \\nATOM    659  C   SER A  90      -6.486  -3.385  -1.656  1.00  0.36           C  \\nATOM    660  CB  SER A  90      -7.234  -4.474   0.466  1.00  0.34           C  \\nATOM    661  O   SER A  90      -5.948  -2.407  -1.132  1.00  0.35           O  \\nATOM    662  OG  SER A  90      -6.329  -3.824   1.343  1.00  0.36           O  \\nATOM    663  N   TYR A  91      -6.244  -3.399  -3.008  1.00  0.42           N  \\nATOM    664  CA  TYR A  91      -6.712  -2.367  -3.925  1.00  0.47           C  \\nATOM    665  C   TYR A  91      -7.516  -2.976  -5.068  1.00  0.42           C  \\nATOM    666  CB  TYR A  91      -5.531  -1.569  -4.486  1.00  0.40           C  \\nATOM    667  O   TYR A  91      -7.012  -3.828  -5.804  1.00  0.41           O  \\nATOM    668  CG  TYR A  91      -4.976  -0.548  -3.522  1.00  0.41           C  \\nATOM    669  CD1 TYR A  91      -5.559   0.712  -3.401  1.00  0.41           C  \\nATOM    670  CD2 TYR A  91      -3.869  -0.840  -2.733  1.00  0.42           C  \\nATOM    671  CE1 TYR A  91      -5.050   1.657  -2.516  1.00  0.39           C  \\nATOM    672  CE2 TYR A  91      -3.353   0.098  -1.845  1.00  0.43           C  \\nATOM    673  OH  TYR A  91      -3.441   2.273  -0.866  1.00  0.26           O  \\nATOM    674  CZ  TYR A  91      -3.949   1.341  -1.743  1.00  0.36           C  \\nATOM    675  N   GLY A  92      -8.937  -3.115  -4.982  1.00  0.37           N  \\nATOM    676  CA  GLY A  92     -10.210  -3.101  -5.686  1.00  0.41           C  \\nATOM    677  C   GLY A  92     -10.478  -4.379  -6.458  1.00  0.38           C  \\nATOM    678  O   GLY A  92     -10.035  -4.525  -7.599  1.00  0.37           O  \\nATOM    679  N   GLN A  93     -10.371  -5.496  -5.841  1.00  0.40           N  \\nATOM    680  CA  GLN A  93     -11.100  -6.650  -6.355  1.00  0.42           C  \\nATOM    681  C   GLN A  93     -12.290  -6.991  -5.463  1.00  0.40           C  \\nATOM    682  CB  GLN A  93     -10.173  -7.860  -6.478  1.00  0.37           C  \\nATOM    683  O   GLN A  93     -12.126  -7.233  -4.265  1.00  0.38           O  \\nATOM    684  CG  GLN A  93      -9.669  -8.107  -7.894  1.00  0.37           C  \\nATOM    685  CD  GLN A  93      -8.668  -9.245  -7.970  1.00  0.42           C  \\nATOM    686  NE2 GLN A  93      -8.224  -9.564  -9.181  1.00  0.37           N  \\nATOM    687  OE1 GLN A  93      -8.296  -9.831  -6.949  1.00  0.47           O  \\nATOM    688  N   GLN A  94     -13.480  -6.432  -5.857  1.00  0.42           N  \\nATOM    689  CA  GLN A  94     -14.777  -7.038  -5.570  1.00  0.45           C  \\nATOM    690  C   GLN A  94     -14.728  -8.553  -5.750  1.00  0.42           C  \\nATOM    691  CB  GLN A  94     -15.860  -6.437  -6.467  1.00  0.40           C  \\nATOM    692  O   GLN A  94     -14.530  -9.045  -6.863  1.00  0.41           O  \\nATOM    693  CG  GLN A  94     -16.704  -5.371  -5.783  1.00  0.40           C  \\nATOM    694  CD  GLN A  94     -17.721  -4.740  -6.714  1.00  0.46           C  \\nATOM    695  NE2 GLN A  94     -18.441  -3.740  -6.217  1.00  0.38           N  \\nATOM    696  OE1 GLN A  94     -17.859  -5.147  -7.872  1.00  0.50           O  \\nATOM    697  N   SER A  95     -14.168  -9.240  -4.881  1.00  0.36           N  \\nATOM    698  CA  SER A  95     -14.410 -10.608  -4.437  1.00  0.39           C  \\nATOM    699  C   SER A  95     -15.901 -10.928  -4.421  1.00  0.37           C  \\nATOM    700  CB  SER A  95     -13.817 -10.835  -3.046  1.00  0.35           C  \\nATOM    701  O   SER A  95     -16.669 -10.300  -3.689  1.00  0.36           O  \\nATOM    702  OG  SER A  95     -14.786 -10.592  -2.041  1.00  0.35           O  \\nATOM    703  N   SER A  96     -16.554 -11.191  -5.535  1.00  0.40           N  \\nATOM    704  CA  SER A  96     -17.740 -12.040  -5.477  1.00  0.43           C  \\nATOM    705  C   SER A  96     -17.362 -13.504  -5.280  1.00  0.41           C  \\nATOM    706  CB  SER A  96     -18.573 -11.886  -6.750  1.00  0.39           C  \\nATOM    707  O   SER A  96     -17.058 -14.207  -6.247  1.00  0.40           O  \\nATOM    708  OG  SER A  96     -17.790 -12.158  -7.899  1.00  0.39           O  \\nATOM    709  N   TYR A  97     -16.608 -13.830  -4.306  1.00  0.37           N  \\nATOM    710  CA  TYR A  97     -16.806 -15.114  -3.643  1.00  0.40           C  \\nATOM    711  C   TYR A  97     -17.676 -14.958  -2.402  1.00  0.37           C  \\nATOM    712  CB  TYR A  97     -15.459 -15.735  -3.261  1.00  0.35           C  \\nATOM    713  O   TYR A  97     -17.390 -14.129  -1.535  1.00  0.35           O  \\nATOM    714  CG  TYR A  97     -14.851 -16.584  -4.352  1.00  0.36           C  \\nATOM    715  CD1 TYR A  97     -15.131 -17.946  -4.439  1.00  0.35           C  \\nATOM    716  CD2 TYR A  97     -13.996 -16.026  -5.296  1.00  0.37           C  \\nATOM    717  CE1 TYR A  97     -14.572 -18.732  -5.441  1.00  0.38           C  \\nATOM    718  CE2 TYR A  97     -13.431 -16.803  -6.302  1.00  0.40           C  \\nATOM    719  OH  TYR A  97     -13.168 -18.926  -7.360  1.00  0.30           O  \\nATOM    720  CZ  TYR A  97     -13.725 -18.152  -6.366  1.00  0.34           C  \\nATOM    721  N   PRO A  98     -18.974 -15.240  -2.351  1.00  0.41           N  \\nATOM    722  CA  PRO A  98     -19.700 -15.682  -1.157  1.00  0.42           C  \\nATOM    723  C   PRO A  98     -18.932 -16.729  -0.354  1.00  0.41           C  \\nATOM    724  CB  PRO A  98     -20.992 -16.271  -1.729  1.00  0.38           C  \\nATOM    725  O   PRO A  98     -18.699 -17.838  -0.842  1.00  0.39           O  \\nATOM    726  CG  PRO A  98     -20.689 -16.531  -3.169  1.00  0.38           C  \\nATOM    727  CD  PRO A  98     -19.458 -15.757  -3.545  1.00  0.44           C  \\nATOM    728  N   GLY A  99     -17.771 -16.322   0.139  1.00  0.36           N  \\nATOM    729  CA  GLY A  99     -17.399 -17.214   1.226  1.00  0.40           C  \\nATOM    730  C   GLY A  99     -15.967 -17.027   1.689  1.00  0.36           C  \\nATOM    731  O   GLY A  99     -15.027 -17.242   0.920  1.00  0.35           O  \\nATOM    732  N   TYR A 100     -15.772 -15.810   2.261  1.00  0.33           N  \\nATOM    733  CA  TYR A 100     -14.944 -15.878   3.460  1.00  0.38           C  \\nATOM    734  C   TYR A 100     -14.901 -14.531   4.171  1.00  0.33           C  \\nATOM    735  CB  TYR A 100     -13.522 -16.327   3.107  1.00  0.31           C  \\nATOM    736  O   TYR A 100     -14.564 -13.512   3.564  1.00  0.32           O  \\nATOM    737  CG  TYR A 100     -13.382 -17.820   2.938  1.00  0.32           C  \\nATOM    738  CD1 TYR A 100     -13.138 -18.645   4.034  1.00  0.31           C  \\nATOM    739  CD2 TYR A 100     -13.490 -18.409   1.683  1.00  0.32           C  \\nATOM    740  CE1 TYR A 100     -13.006 -20.021   3.882  1.00  0.32           C  \\nATOM    741  CE2 TYR A 100     -13.360 -19.784   1.519  1.00  0.35           C  \\nATOM    742  OH  TYR A 100     -12.988 -21.943   2.468  1.00  0.18           O  \\nATOM    743  CZ  TYR A 100     -13.118 -20.581   2.623  1.00  0.29           C  \\nATOM    744  N   GLY A 101     -15.967 -14.411   5.187  1.00  0.34           N  \\nATOM    745  CA  GLY A 101     -15.922 -14.175   6.621  1.00  0.37           C  \\nATOM    746  C   GLY A 101     -15.800 -12.707   6.981  1.00  0.34           C  \\nATOM    747  O   GLY A 101     -14.910 -12.012   6.486  1.00  0.32           O  \\nATOM    748  N   GLN A 102     -16.951 -11.842   6.835  1.00  0.41           N  \\nATOM    749  CA  GLN A 102     -17.277 -10.970   7.958  1.00  0.44           C  \\nATOM    750  C   GLN A 102     -16.128 -10.015   8.265  1.00  0.41           C  \\nATOM    751  CB  GLN A 102     -17.618 -11.797   9.199  1.00  0.37           C  \\nATOM    752  O   GLN A 102     -15.100 -10.425   8.807  1.00  0.39           O  \\nATOM    753  CG  GLN A 102     -19.076 -12.231   9.268  1.00  0.37           C  \\nATOM    754  CD  GLN A 102     -19.370 -13.114  10.465  1.00  0.43           C  \\nATOM    755  NE2 GLN A 102     -20.442 -13.895  10.378  1.00  0.35           N  \\nATOM    756  OE1 GLN A 102     -18.639 -13.096  11.461  1.00  0.48           O  \\nATOM    757  N   GLN A 103     -15.899  -8.981   7.436  1.00  0.46           N  \\nATOM    758  CA  GLN A 103     -15.599  -7.785   8.215  1.00  0.51           C  \\nATOM    759  C   GLN A 103     -16.878  -7.109   8.700  1.00  0.46           C  \\nATOM    760  CB  GLN A 103     -14.766  -6.802   7.390  1.00  0.43           C  \\nATOM    761  O   GLN A 103     -17.803  -6.884   7.917  1.00  0.44           O  \\nATOM    762  CG  GLN A 103     -14.213  -5.635   8.198  1.00  0.42           C  \\nATOM    763  CD  GLN A 103     -13.323  -4.722   7.377  1.00  0.48           C  \\nATOM    764  NE2 GLN A 103     -13.103  -3.508   7.870  1.00  0.38           N  \\nATOM    765  OE1 GLN A 103     -12.837  -5.104   6.308  1.00  0.50           O  \\nATOM    766  N   PRO A 104     -17.243  -7.348  10.017  1.00  0.46           N  \\nATOM    767  CA  PRO A 104     -18.233  -6.506  10.693  1.00  0.45           C  \\nATOM    768  C   PRO A 104     -18.334  -5.110  10.083  1.00  0.46           C  \\nATOM    769  CB  PRO A 104     -17.714  -6.437  12.131  1.00  0.42           C  \\nATOM    770  O   PRO A 104     -17.312  -4.475   9.811  1.00  0.44           O  \\nATOM    771  CG  PRO A 104     -16.299  -6.911  12.050  1.00  0.41           C  \\nATOM    772  CD  PRO A 104     -16.097  -7.584  10.723  1.00  0.45           C  \\nATOM    773  N   ALA A 105     -19.380  -4.724   9.227  1.00  0.44           N  \\nATOM    774  CA  ALA A 105     -20.069  -3.441   9.112  1.00  0.45           C  \\nATOM    775  C   ALA A 105     -19.602  -2.469  10.192  1.00  0.44           C  \\nATOM    776  CB  ALA A 105     -21.580  -3.639   9.196  1.00  0.42           C  \\nATOM    777  O   ALA A 105     -19.469  -2.846  11.359  1.00  0.43           O  \\nATOM    778  N   PRO A 106     -18.859  -1.426   9.929  1.00  0.51           N  \\nATOM    779  CA  PRO A 106     -18.782  -0.328  10.895  1.00  0.52           C  \\nATOM    780  C   PRO A 106     -20.156   0.213  11.285  1.00  0.51           C  \\nATOM    781  CB  PRO A 106     -17.966   0.735  10.156  1.00  0.47           C  \\nATOM    782  O   PRO A 106     -20.974   0.516  10.413  1.00  0.48           O  \\nATOM    783  CG  PRO A 106     -18.193   0.451   8.706  1.00  0.46           C  \\nATOM    784  CD  PRO A 106     -18.511  -1.008   8.550  1.00  0.50           C  \\nATOM    785  N   SER A 107     -20.939  -0.280  12.243  1.00  0.47           N  \\nATOM    786  CA  SER A 107     -21.821   0.512  13.094  1.00  0.49           C  \\nATOM    787  C   SER A 107     -21.891   1.960  12.622  1.00  0.46           C  \\nATOM    788  CB  SER A 107     -21.350   0.463  14.548  1.00  0.44           C  \\nATOM    789  O   SER A 107     -20.921   2.709  12.755  1.00  0.44           O  \\nATOM    790  OG  SER A 107     -20.013   0.921  14.659  1.00  0.50           O  \\nATOM    791  N   SER A 108     -22.617   2.336  11.611  1.00  0.49           N  \\nATOM    792  CA  SER A 108     -23.102   3.708  11.510  1.00  0.53           C  \\nATOM    793  C   SER A 108     -24.094   4.028  12.624  1.00  0.49           C  \\nATOM    794  CB  SER A 108     -23.757   3.948  10.149  1.00  0.45           C  \\nATOM    795  O   SER A 108     -25.136   3.379  12.739  1.00  0.47           O  \\nATOM    796  OG  SER A 108     -24.805   3.020   9.925  1.00  0.47           O  \\nATOM    797  N   THR A 109     -23.753   4.549  13.708  1.00  0.44           N  \\nATOM    798  CA  THR A 109     -24.119   5.528  14.725  1.00  0.49           C  \\nATOM    799  C   THR A 109     -25.239   6.436  14.224  1.00  0.44           C  \\nATOM    800  CB  THR A 109     -22.908   6.384  15.139  1.00  0.41           C  \\nATOM    801  O   THR A 109     -25.092   7.103  13.198  1.00  0.41           O  \\nATOM    802  CG2 THR A 109     -22.648   6.280  16.638  1.00  0.38           C  \\nATOM    803  OG1 THR A 109     -21.747   5.931  14.432  1.00  0.39           O  \\nATOM    804  N   SER A 110     -26.600   6.199  14.374  1.00  0.43           N  \\nATOM    805  CA  SER A 110     -27.590   6.934  15.155  1.00  0.49           C  \\nATOM    806  C   SER A 110     -27.446   8.439  14.954  1.00  0.43           C  \\nATOM    807  CB  SER A 110     -27.462   6.595  16.640  1.00  0.40           C  \\nATOM    808  O   SER A 110     -26.459   9.036  15.389  1.00  0.41           O  \\nATOM    809  OG  SER A 110     -26.165   6.105  16.934  1.00  0.47           O  \\nATOM    810  N   GLY A 111     -27.975   9.013  13.830  1.00  0.44           N  \\nATOM    811  CA  GLY A 111     -28.514  10.334  14.112  1.00  0.46           C  \\nATOM    812  C   GLY A 111     -29.794  10.630  13.353  1.00  0.44           C  \\nATOM    813  O   GLY A 111     -29.856  10.450  12.135  1.00  0.42           O  \\nATOM    814  N   SER A 112     -31.101  10.383  13.732  1.00  0.39           N  \\nATOM    815  CA  SER A 112     -32.245  11.125  14.250  1.00  0.43           C  \\nATOM    816  C   SER A 112     -32.598  12.300  13.344  1.00  0.39           C  \\nATOM    817  CB  SER A 112     -31.961  11.630  15.666  1.00  0.36           C  \\nATOM    818  O   SER A 112     -31.726  13.088  12.972  1.00  0.38           O  \\nATOM    819  OG  SER A 112     -33.145  12.114  16.274  1.00  0.42           O  \\nATOM    820  N   TYR A 113     -33.721  12.277  12.392  1.00  0.45           N  \\nATOM    821  CA  TYR A 113     -34.730  13.327  12.462  1.00  0.50           C  \\nATOM    822  C   TYR A 113     -34.283  14.565  11.694  1.00  0.44           C  \\nATOM    823  CB  TYR A 113     -35.023  13.695  13.920  1.00  0.42           C  \\nATOM    824  O   TYR A 113     -33.285  15.196  12.048  1.00  0.43           O  \\nATOM    825  CG  TYR A 113     -35.890  12.690  14.638  1.00  0.46           C  \\nATOM    826  CD1 TYR A 113     -37.278  12.729  14.522  1.00  0.44           C  \\nATOM    827  CD2 TYR A 113     -35.324  11.699  15.433  1.00  0.44           C  \\nATOM    828  CE1 TYR A 113     -38.081  11.806  15.184  1.00  0.41           C  \\nATOM    829  CE2 TYR A 113     -36.117  10.770  16.098  1.00  0.45           C  \\nATOM    830  OH  TYR A 113     -38.283   9.915  16.624  1.00  0.31           O  \\nATOM    831  CZ  TYR A 113     -37.493  10.831  15.968  1.00  0.37           C  \\nATOM    832  N   GLY A 114     -34.672  14.597  10.213  1.00  0.43           N  \\nATOM    833  CA  GLY A 114     -35.365  15.787   9.745  1.00  0.46           C  \\nATOM    834  C   GLY A 114     -35.636  15.772   8.252  1.00  0.44           C  \\nATOM    835  O   GLY A 114     -34.719  15.579   7.451  1.00  0.42           O  \\nATOM    836  N   SER A 115     -36.701  15.165   7.598  1.00  0.38           N  \\nATOM    837  CA  SER A 115     -37.686  15.834   6.755  1.00  0.46           C  \\nATOM    838  C   SER A 115     -37.205  17.219   6.335  1.00  0.38           C  \\nATOM    839  CB  SER A 115     -39.025  15.951   7.486  1.00  0.38           C  \\nATOM    840  O   SER A 115     -36.987  18.089   7.180  1.00  0.38           O  \\nATOM    841  OG  SER A 115     -39.315  17.305   7.791  1.00  0.48           O  \\nATOM    842  N   SER A 116     -36.412  17.359   5.327  1.00  0.40           N  \\nATOM    843  CA  SER A 116     -36.647  18.572   4.551  1.00  0.44           C  \\nATOM    844  C   SER A 116     -36.127  18.425   3.124  1.00  0.40           C  \\nATOM    845  CB  SER A 116     -35.983  19.775   5.222  1.00  0.37           C  \\nATOM    846  O   SER A 116     -34.983  18.015   2.914  1.00  0.39           O  \\nATOM    847  OG  SER A 116     -34.586  19.574   5.348  1.00  0.43           O  \\nATOM    848  N   SER A 117     -36.991  17.996   2.182  1.00  0.39           N  \\nATOM    849  CA  SER A 117     -37.388  18.770   1.011  1.00  0.46           C  \\nATOM    850  C   SER A 117     -36.311  19.777   0.622  1.00  0.40           C  \\nATOM    851  CB  SER A 117     -38.708  19.498   1.271  1.00  0.39           C  \\nATOM    852  O   SER A 117     -36.158  20.813   1.273  1.00  0.39           O  \\nATOM    853  OG  SER A 117     -38.653  20.219   2.489  1.00  0.48           O  \\nATOM    854  N   GLN A 118     -35.284  19.393   0.208  1.00  0.39           N  \\nATOM    855  CA  GLN A 118     -34.627  20.214  -0.804  1.00  0.44           C  \\nATOM    856  C   GLN A 118     -34.318  19.401  -2.057  1.00  0.40           C  \\nATOM    857  CB  GLN A 118     -33.342  20.828  -0.245  1.00  0.37           C  \\nATOM    858  O   GLN A 118     -33.629  18.381  -1.987  1.00  0.38           O  \\nATOM    859  CG  GLN A 118     -33.540  22.198   0.389  1.00  0.39           C  \\nATOM    860  CD  GLN A 118     -32.269  22.746   1.011  1.00  0.45           C  \\nATOM    861  NE2 GLN A 118     -32.405  23.803   1.805  1.00  0.37           N  \\nATOM    862  OE1 GLN A 118     -31.174  22.223   0.782  1.00  0.48           O  \\nATOM    863  N   SER A 119     -35.265  19.303  -3.038  1.00  0.38           N  \\nATOM    864  CA  SER A 119     -35.178  19.759  -4.422  1.00  0.44           C  \\nATOM    865  C   SER A 119     -33.792  20.313  -4.735  1.00  0.38           C  \\nATOM    866  CB  SER A 119     -36.238  20.825  -4.703  1.00  0.38           C  \\nATOM    867  O   SER A 119     -33.435  21.401  -4.278  1.00  0.38           O  \\nATOM    868  OG  SER A 119     -36.026  21.970  -3.896  1.00  0.45           O  \\nATOM    869  N   SER A 120     -32.862  19.750  -4.699  1.00  0.37           N  \\nATOM    870  CA  SER A 120     -31.609  20.086  -5.368  1.00  0.42           C  \\nATOM    871  C   SER A 120     -31.364  19.180  -6.571  1.00  0.38           C  \\nATOM    872  CB  SER A 120     -30.436  19.981  -4.393  1.00  0.36           C  \\nATOM    873  O   SER A 120     -31.435  17.955  -6.456  1.00  0.37           O  \\nATOM    874  OG  SER A 120     -30.295  18.654  -3.916  1.00  0.38           O  \\nATOM    875  N   SER A 121     -32.051  19.623  -7.647  1.00  0.37           N  \\nATOM    876  CA  SER A 121     -31.559  19.752  -9.015  1.00  0.42           C  \\nATOM    877  C   SER A 121     -30.591  18.625  -9.362  1.00  0.38           C  \\nATOM    878  CB  SER A 121     -30.871  21.103  -9.212  1.00  0.37           C  \\nATOM    879  O   SER A 121     -29.531  18.498  -8.747  1.00  0.37           O  \\nATOM    880  OG  SER A 121     -29.467  20.944  -9.310  1.00  0.39           O  \\nATOM    881  N   TYR A 122     -30.998  17.526  -9.191  1.00  0.43           N  \\nATOM    882  CA  TYR A 122     -30.358  16.730 -10.233  1.00  0.48           C  \\nATOM    883  C   TYR A 122     -29.977  17.599 -11.426  1.00  0.43           C  \\nATOM    884  CB  TYR A 122     -31.282  15.596 -10.687  1.00  0.42           C  \\nATOM    885  O   TYR A 122     -30.842  18.013 -12.202  1.00  0.42           O  \\nATOM    886  CG  TYR A 122     -31.402  14.473  -9.686  1.00  0.43           C  \\nATOM    887  CD1 TYR A 122     -30.453  13.454  -9.637  1.00  0.42           C  \\nATOM    888  CD2 TYR A 122     -32.463  14.428  -8.788  1.00  0.42           C  \\nATOM    889  CE1 TYR A 122     -30.560  12.416  -8.718  1.00  0.39           C  \\nATOM    890  CE2 TYR A 122     -32.580  13.394  -7.864  1.00  0.44           C  \\nATOM    891  OH  TYR A 122     -31.736  11.369  -6.924  1.00  0.24           O  \\nATOM    892  CZ  TYR A 122     -31.625  12.395  -7.836  1.00  0.35           C  \\nATOM    893  N   GLY A 123     -29.277  18.575 -11.205  1.00  0.34           N  \\nATOM    894  CA  GLY A 123     -28.641  19.597 -12.020  1.00  0.36           C  \\nATOM    895  C   GLY A 123     -27.540  19.051 -12.910  1.00  0.35           C  \\nATOM    896  O   GLY A 123     -26.700  18.269 -12.458  1.00  0.33           O  \\nATOM    897  N   GLN A 124     -27.926  18.387 -14.048  1.00  0.39           N  \\nATOM    898  CA  GLN A 124     -26.911  18.377 -15.097  1.00  0.41           C  \\nATOM    899  C   GLN A 124     -25.519  18.156 -14.512  1.00  0.39           C  \\nATOM    900  CB  GLN A 124     -26.946  19.684 -15.890  1.00  0.37           C  \\nATOM    901  O   GLN A 124     -25.188  18.704 -13.459  1.00  0.37           O  \\nATOM    902  CG  GLN A 124     -27.814  19.623 -17.139  1.00  0.38           C  \\nATOM    903  CD  GLN A 124     -27.807  20.921 -17.924  1.00  0.47           C  \\nATOM    904  NE2 GLN A 124     -27.548  20.828 -19.224  1.00  0.38           N  \\nATOM    905  OE1 GLN A 124     -28.033  22.000 -17.366  1.00  0.51           O  \\nATOM    906  N   PRO A 125     -24.752  17.099 -14.561  1.00  0.45           N  \\nATOM    907  CA  PRO A 125     -23.354  17.417 -14.260  1.00  0.46           C  \\nATOM    908  C   PRO A 125     -23.071  18.917 -14.284  1.00  0.45           C  \\nATOM    909  CB  PRO A 125     -22.583  16.698 -15.371  1.00  0.43           C  \\nATOM    910  O   PRO A 125     -23.317  19.579 -15.296  1.00  0.44           O  \\nATOM    911  CG  PRO A 125     -23.581  16.505 -16.466  1.00  0.42           C  \\nATOM    912  CD  PRO A 125     -24.959  16.617 -15.880  1.00  0.47           C  \\nATOM    913  N   GLN A 126     -23.361  19.594 -13.407  1.00  0.34           N  \\nATOM    914  CA  GLN A 126     -22.553  20.803 -13.282  1.00  0.37           C  \\nATOM    915  C   GLN A 126     -21.071  20.498 -13.484  1.00  0.35           C  \\nATOM    916  CB  GLN A 126     -22.772  21.456 -11.916  1.00  0.32           C  \\nATOM    917  O   GLN A 126     -20.512  19.633 -12.806  1.00  0.33           O  \\nATOM    918  CG  GLN A 126     -24.024  22.321 -11.843  1.00  0.34           C  \\nATOM    919  CD  GLN A 126     -24.137  23.079 -10.534  1.00  0.43           C  \\nATOM    920  NE2 GLN A 126     -25.106  23.985 -10.455  1.00  0.33           N  \\nATOM    921  OE1 GLN A 126     -23.361  22.853  -9.600  1.00  0.46           O  \\nATOM    922  N   SER A 127     -20.440  20.405 -14.615  1.00  0.37           N  \\nATOM    923  CA  SER A 127     -19.087  20.953 -14.606  1.00  0.40           C  \\nATOM    924  C   SER A 127     -18.726  21.511 -13.234  1.00  0.37           C  \\nATOM    925  CB  SER A 127     -18.946  22.048 -15.665  1.00  0.36           C  \\nATOM    926  O   SER A 127     -19.144  22.614 -12.877  1.00  0.36           O  \\nATOM    927  OG  SER A 127     -20.073  22.907 -15.653  1.00  0.41           O  \\nATOM    928  N   GLY A 128     -18.891  20.941 -12.239  1.00  0.33           N  \\nATOM    929  CA  GLY A 128     -18.338  21.516 -11.024  1.00  0.37           C  \\nATOM    930  C   GLY A 128     -16.861  21.220 -10.843  1.00  0.34           C  \\nATOM    931  O   GLY A 128     -16.428  20.075 -10.996  1.00  0.32           O  \\nATOM    932  N   SER A 129     -15.975  21.967 -11.547  1.00  0.35           N  \\nATOM    933  CA  SER A 129     -14.616  22.248 -11.093  1.00  0.39           C  \\nATOM    934  C   SER A 129     -13.939  20.988 -10.564  1.00  0.36           C  \\nATOM    935  CB  SER A 129     -14.625  23.325 -10.008  1.00  0.34           C  \\nATOM    936  O   SER A 129     -14.385  20.405  -9.574  1.00  0.34           O  \\nATOM    937  OG  SER A 129     -13.415  24.063 -10.022  1.00  0.34           O  \\nATOM    938  N   TYR A 130     -13.646  20.055 -11.373  1.00  0.43           N  \\nATOM    939  CA  TYR A 130     -12.431  19.330 -11.019  1.00  0.47           C  \\nATOM    940  C   TYR A 130     -11.608  20.109 -10.000  1.00  0.44           C  \\nATOM    941  CB  TYR A 130     -11.588  19.052 -12.267  1.00  0.41           C  \\nATOM    942  O   TYR A 130     -11.035  21.152 -10.322  1.00  0.42           O  \\nATOM    943  CG  TYR A 130     -12.156  17.972 -13.156  1.00  0.42           C  \\nATOM    944  CD1 TYR A 130     -11.863  16.629 -12.927  1.00  0.40           C  \\nATOM    945  CD2 TYR A 130     -12.986  18.291 -14.225  1.00  0.41           C  \\nATOM    946  CE1 TYR A 130     -12.383  15.631 -13.744  1.00  0.38           C  \\nATOM    947  CE2 TYR A 130     -13.511  17.301 -15.048  1.00  0.44           C  \\nATOM    948  OH  TYR A 130     -13.722  14.991 -15.612  1.00  0.25           O  \\nATOM    949  CZ  TYR A 130     -13.205  15.976 -14.800  1.00  0.34           C  \\nATOM    950  N   SER A 131     -11.954  20.373  -8.911  1.00  0.36           N  \\nATOM    951  CA  SER A 131     -10.972  20.531  -7.843  1.00  0.40           C  \\nATOM    952  C   SER A 131      -9.960  19.390  -7.850  1.00  0.36           C  \\nATOM    953  CB  SER A 131     -11.665  20.599  -6.482  1.00  0.35           C  \\nATOM    954  O   SER A 131     -10.339  18.217  -7.832  1.00  0.35           O  \\nATOM    955  OG  SER A 131     -11.734  19.315  -5.887  1.00  0.36           O  \\nATOM    956  N   GLN A 132      -8.930  19.471  -8.865  1.00  0.39           N  \\nATOM    957  CA  GLN A 132      -7.552  19.020  -8.707  1.00  0.41           C  \\nATOM    958  C   GLN A 132      -7.420  18.055  -7.533  1.00  0.40           C  \\nATOM    959  CB  GLN A 132      -6.615  20.213  -8.513  1.00  0.37           C  \\nATOM    960  O   GLN A 132      -7.634  18.438  -6.381  1.00  0.38           O  \\nATOM    961  CG  GLN A 132      -6.255  20.930  -9.808  1.00  0.38           C  \\nATOM    962  CD  GLN A 132      -5.336  22.116  -9.587  1.00  0.45           C  \\nATOM    963  NE2 GLN A 132      -4.995  22.812 -10.666  1.00  0.37           N  \\nATOM    964  OE1 GLN A 132      -4.935  22.405  -8.455  1.00  0.51           O  \\nATOM    965  N   GLN A 133      -7.888  16.887  -7.636  1.00  0.41           N  \\nATOM    966  CA  GLN A 133      -7.071  15.974  -6.843  1.00  0.45           C  \\nATOM    967  C   GLN A 133      -5.617  16.435  -6.798  1.00  0.41           C  \\nATOM    968  CB  GLN A 133      -7.154  14.553  -7.403  1.00  0.39           C  \\nATOM    969  O   GLN A 133      -5.048  16.812  -7.825  1.00  0.40           O  \\nATOM    970  CG  GLN A 133      -7.228  13.474  -6.331  1.00  0.40           C  \\nATOM    971  CD  GLN A 133      -7.369  12.079  -6.912  1.00  0.47           C  \\nATOM    972  NE2 GLN A 133      -6.526  11.159  -6.457  1.00  0.38           N  \\nATOM    973  OE1 GLN A 133      -8.230  11.831  -7.763  1.00  0.51           O  \\nATOM    974  N   PRO A 134      -5.081  16.977  -5.782  1.00  0.38           N  \\nATOM    975  CA  PRO A 134      -3.617  17.008  -5.756  1.00  0.39           C  \\nATOM    976  C   PRO A 134      -2.988  15.964  -6.675  1.00  0.38           C  \\nATOM    977  CB  PRO A 134      -3.284  16.717  -4.290  1.00  0.36           C  \\nATOM    978  O   PRO A 134      -3.383  14.795  -6.649  1.00  0.36           O  \\nATOM    979  CG  PRO A 134      -4.523  16.093  -3.734  1.00  0.36           C  \\nATOM    980  CD  PRO A 134      -5.673  16.425  -4.640  1.00  0.41           C  \\nATOM    981  N   SER A 135      -2.692  16.223  -8.005  1.00  0.36           N  \\nATOM    982  CA  SER A 135      -1.567  15.622  -8.712  1.00  0.37           C  \\nATOM    983  C   SER A 135      -0.575  14.993  -7.739  1.00  0.36           C  \\nATOM    984  CB  SER A 135      -0.856  16.665  -9.575  1.00  0.34           C  \\nATOM    985  O   SER A 135      -0.008  15.684  -6.890  1.00  0.35           O  \\nATOM    986  OG  SER A 135       0.340  17.102  -8.952  1.00  0.36           O  \\nATOM    987  N   TYR A 136      -0.828  14.101  -7.051  1.00  0.36           N  \\nATOM    988  CA  TYR A 136       0.382  13.373  -6.689  1.00  0.40           C  \\nATOM    989  C   TYR A 136       1.330  13.266  -7.877  1.00  0.36           C  \\nATOM    990  CB  TYR A 136       0.033  11.974  -6.172  1.00  0.35           C  \\nATOM    991  O   TYR A 136       0.969  12.716  -8.920  1.00  0.35           O  \\nATOM    992  CG  TYR A 136      -0.497  11.963  -4.759  1.00  0.36           C  \\nATOM    993  CD1 TYR A 136       0.371  11.927  -3.669  1.00  0.35           C  \\nATOM    994  CD2 TYR A 136      -1.865  11.989  -4.510  1.00  0.36           C  \\nATOM    995  CE1 TYR A 136      -0.113  11.915  -2.365  1.00  0.34           C  \\nATOM    996  CE2 TYR A 136      -2.360  11.978  -3.210  1.00  0.38           C  \\nATOM    997  OH  TYR A 136      -1.962  11.930  -0.857  1.00  0.19           O  \\nATOM    998  CZ  TYR A 136      -1.478  11.941  -2.146  1.00  0.31           C  \\nATOM    999  N   GLY A 137       2.125  14.316  -8.321  1.00  0.33           N  \\nATOM   1000  CA  GLY A 137       3.463  14.255  -8.887  1.00  0.35           C  \\nATOM   1001  C   GLY A 137       4.040  12.852  -8.905  1.00  0.34           C  \\nATOM   1002  O   GLY A 137       4.507  12.354  -7.878  1.00  0.32           O  \\nATOM   1003  N   GLY A 138       3.488  11.888  -9.423  1.00  0.35           N  \\nATOM   1004  CA  GLY A 138       4.268  10.699  -9.728  1.00  0.37           C  \\nATOM   1005  C   GLY A 138       5.363  10.950 -10.748  1.00  0.35           C  \\nATOM   1006  O   GLY A 138       5.118  11.558 -11.793  1.00  0.34           O  \\nATOM   1007  N   GLN A 139       6.542  11.443 -10.402  1.00  0.37           N  \\nATOM   1008  CA  GLN A 139       7.870  11.292 -10.987  1.00  0.40           C  \\nATOM   1009  C   GLN A 139       7.808  10.520 -12.301  1.00  0.37           C  \\nATOM   1010  CB  GLN A 139       8.812  10.588 -10.008  1.00  0.35           C  \\nATOM   1011  O   GLN A 139       7.463   9.336 -12.316  1.00  0.36           O  \\nATOM   1012  CG  GLN A 139       9.422  11.516  -8.967  1.00  0.35           C  \\nATOM   1013  CD  GLN A 139      10.391  10.803  -8.041  1.00  0.42           C  \\nATOM   1014  NE2 GLN A 139      11.057  11.564  -7.180  1.00  0.36           N  \\nATOM   1015  OE1 GLN A 139      10.538   9.578  -8.100  1.00  0.48           O  \\nATOM   1016  N   GLN A 140       7.600  11.084 -13.506  1.00  0.36           N  \\nATOM   1017  CA  GLN A 140       8.034  10.725 -14.852  1.00  0.38           C  \\nATOM   1018  C   GLN A 140       9.549  10.845 -14.991  1.00  0.36           C  \\nATOM   1019  CB  GLN A 140       7.340  11.603 -15.894  1.00  0.34           C  \\nATOM   1020  O   GLN A 140      10.109  11.930 -14.826  1.00  0.35           O  \\nATOM   1021  CG  GLN A 140       6.275  10.875 -16.701  1.00  0.34           C  \\nATOM   1022  CD  GLN A 140       5.538  11.789 -17.662  1.00  0.42           C  \\nATOM   1023  NE2 GLN A 140       4.689  11.207 -18.502  1.00  0.35           N  \\nATOM   1024  OE1 GLN A 140       5.730  13.009 -17.648  1.00  0.48           O  \\nATOM   1025  N   GLN A 141      10.346   9.771 -14.837  1.00  0.36           N  \\nATOM   1026  CA  GLN A 141      11.724   9.413 -15.154  1.00  0.39           C  \\nATOM   1027  C   GLN A 141      11.920   9.257 -16.660  1.00  0.36           C  \\nATOM   1028  CB  GLN A 141      12.123   8.123 -14.436  1.00  0.34           C  \\nATOM   1029  O   GLN A 141      11.228   8.464 -17.302  1.00  0.35           O  \\nATOM   1030  CG  GLN A 141      12.323   8.291 -12.936  1.00  0.34           C  \\nATOM   1031  CD  GLN A 141      12.810   7.021 -12.263  1.00  0.40           C  \\nATOM   1032  NE2 GLN A 141      13.015   7.085 -10.952  1.00  0.36           N  \\nATOM   1033  OE1 GLN A 141      12.998   5.989 -12.917  1.00  0.46           O  \\nATOM   1034  N   SER A 142      12.477  10.178 -17.463  1.00  0.35           N  \\nATOM   1035  CA  SER A 142      13.252  10.327 -18.690  1.00  0.38           C  \\nATOM   1036  C   SER A 142      14.511   9.468 -18.657  1.00  0.36           C  \\nATOM   1037  CB  SER A 142      13.630  11.792 -18.910  1.00  0.34           C  \\nATOM   1038  O   SER A 142      15.347   9.616 -17.763  1.00  0.35           O  \\nATOM   1039  OG  SER A 142      12.954  12.322 -20.038  1.00  0.35           O  \\nATOM   1040  N   TYR A 143      14.616   8.174 -18.988  1.00  0.34           N  \\nATOM   1041  CA  TYR A 143      15.577   7.110 -19.256  1.00  0.37           C  \\nATOM   1042  C   TYR A 143      16.277   7.330 -20.591  1.00  0.34           C  \\nATOM   1043  CB  TYR A 143      14.882   5.745 -19.250  1.00  0.32           C  \\nATOM   1044  O   TYR A 143      15.640   7.298 -21.647  1.00  0.33           O  \\nATOM   1045  CG  TYR A 143      14.389   5.320 -17.888  1.00  0.32           C  \\nATOM   1046  CD1 TYR A 143      15.251   4.730 -16.966  1.00  0.30           C  \\nATOM   1047  CD2 TYR A 143      13.061   5.508 -17.520  1.00  0.33           C  \\nATOM   1048  CE1 TYR A 143      14.800   4.335 -15.711  1.00  0.33           C  \\nATOM   1049  CE2 TYR A 143      12.599   5.118 -16.268  1.00  0.36           C  \\nATOM   1050  OH  TYR A 143      13.023   4.145 -14.129  1.00  0.26           O  \\nATOM   1051  CZ  TYR A 143      13.475   4.534 -15.371  1.00  0.30           C  \\nATOM   1052  N   GLY A 144      17.458   7.820 -20.694  1.00  0.33           N  \\nATOM   1053  CA  GLY A 144      18.668   8.028 -21.473  1.00  0.38           C  \\nATOM   1054  C   GLY A 144      19.852   7.230 -20.962  1.00  0.34           C  \\nATOM   1055  O   GLY A 144      20.203   7.316 -19.783  1.00  0.34           O  \\nATOM   1056  N   GLN A 145      19.922   5.754 -21.297  1.00  0.37           N  \\nATOM   1057  CA  GLN A 145      21.033   4.890 -21.681  1.00  0.40           C  \\nATOM   1058  C   GLN A 145      20.694   3.421 -21.446  1.00  0.37           C  \\nATOM   1059  CB  GLN A 145      22.298   5.266 -20.908  1.00  0.34           C  \\nATOM   1060  O   GLN A 145      20.273   3.042 -20.351  1.00  0.35           O  \\nATOM   1061  CG  GLN A 145      23.053   6.448 -21.501  1.00  0.35           C  \\nATOM   1062  CD  GLN A 145      24.348   6.745 -20.769  1.00  0.43           C  \\nATOM   1063  NE2 GLN A 145      25.148   7.650 -21.323  1.00  0.34           N  \\nATOM   1064  OE1 GLN A 145      24.628   6.166 -19.714  1.00  0.48           O  \\nATOM   1065  N   GLN A 146      20.258   2.444 -22.572  1.00  0.35           N  \\nATOM   1066  CA  GLN A 146      20.458   1.169 -23.253  1.00  0.39           C  \\nATOM   1067  C   GLN A 146      21.500   0.318 -22.533  1.00  0.35           C  \\nATOM   1068  CB  GLN A 146      20.879   1.395 -24.706  1.00  0.33           C  \\nATOM   1069  O   GLN A 146      22.691   0.636 -22.553  1.00  0.34           O  \\nATOM   1070  CG  GLN A 146      19.728   1.313 -25.700  1.00  0.34           C  \\nATOM   1071  CD  GLN A 146      20.164   1.587 -27.127  1.00  0.43           C  \\nATOM   1072  NE2 GLN A 146      19.276   1.321 -28.079  1.00  0.34           N  \\nATOM   1073  OE1 GLN A 146      21.288   2.035 -27.371  1.00  0.49           O  \\nATOM   1074  N   GLN A 147      21.133  -0.478 -21.416  1.00  0.37           N  \\nATOM   1075  CA  GLN A 147      21.851  -1.722 -21.157  1.00  0.42           C  \\nATOM   1076  C   GLN A 147      20.956  -2.934 -21.401  1.00  0.37           C  \\nATOM   1077  CB  GLN A 147      22.388  -1.745 -19.725  1.00  0.35           C  \\nATOM   1078  O   GLN A 147      19.853  -3.017 -20.858  1.00  0.36           O  \\nATOM   1079  CG  GLN A 147      23.650  -0.917 -19.529  1.00  0.36           C  \\nATOM   1080  CD  GLN A 147      24.264  -1.100 -18.153  1.00  0.43           C  \\nATOM   1081  NE2 GLN A 147      25.408  -0.463 -17.925  1.00  0.36           N  \\nATOM   1082  OE1 GLN A 147      23.715  -1.808 -17.303  1.00  0.49           O  \\nATOM   1083  N   SER A 148      20.929  -3.711 -22.625  1.00  0.36           N  \\nATOM   1084  CA  SER A 148      20.681  -5.043 -23.167  1.00  0.41           C  \\nATOM   1085  C   SER A 148      20.622  -6.089 -22.059  1.00  0.37           C  \\nATOM   1086  CB  SER A 148      21.763  -5.421 -24.179  1.00  0.35           C  \\nATOM   1087  O   SER A 148      21.613  -6.323 -21.365  1.00  0.36           O  \\nATOM   1088  OG  SER A 148      23.055  -5.216 -23.633  1.00  0.36           O  \\nATOM   1089  N   TYR A 149      19.595  -6.231 -21.311  1.00  0.37           N  \\nATOM   1090  CA  TYR A 149      19.323  -7.453 -20.563  1.00  0.41           C  \\nATOM   1091  C   TYR A 149      18.575  -8.464 -21.424  1.00  0.38           C  \\nATOM   1092  CB  TYR A 149      18.514  -7.142 -19.301  1.00  0.35           C  \\nATOM   1093  O   TYR A 149      17.497  -8.168 -21.945  1.00  0.37           O  \\nATOM   1094  CG  TYR A 149      18.702  -8.152 -18.195  1.00  0.35           C  \\nATOM   1095  CD1 TYR A 149      17.802  -9.202 -18.024  1.00  0.31           C  \\nATOM   1096  CD2 TYR A 149      19.778  -8.058 -17.319  1.00  0.33           C  \\nATOM   1097  CE1 TYR A 149      17.970 -10.134 -17.006  1.00  0.31           C  \\nATOM   1098  CE2 TYR A 149      19.955  -8.985 -16.297  1.00  0.37           C  \\nATOM   1099  OH  TYR A 149      19.219 -10.938 -15.138  1.00  0.21           O  \\nATOM   1100  CZ  TYR A 149      19.048 -10.018 -16.148  1.00  0.29           C  \\nATOM   1101  N   ASN A 150      19.271  -9.485 -21.970  1.00  0.36           N  \\nATOM   1102  CA  ASN A 150      19.002 -10.871 -22.341  1.00  0.39           C  \\nATOM   1103  C   ASN A 150      18.105 -11.561 -21.316  1.00  0.36           C  \\nATOM   1104  CB  ASN A 150      20.310 -11.646 -22.508  1.00  0.35           C  \\nATOM   1105  O   ASN A 150      18.497 -11.740 -20.162  1.00  0.35           O  \\nATOM   1106  CG  ASN A 150      20.486 -12.200 -23.908  1.00  0.35           C  \\nATOM   1107  ND2 ASN A 150      21.676 -12.712 -24.199  1.00  0.30           N  \\nATOM   1108  OD1 ASN A 150      19.560 -12.167 -24.723  1.00  0.36           O  \\nATOM   1109  N   PRO A 151      16.760 -11.499 -21.374  1.00  0.42           N  \\nATOM   1110  CA  PRO A 151      15.884 -12.315 -20.529  1.00  0.44           C  \\nATOM   1111  C   PRO A 151      16.315 -13.778 -20.472  1.00  0.42           C  \\nATOM   1112  CB  PRO A 151      14.516 -12.174 -21.201  1.00  0.39           C  \\nATOM   1113  O   PRO A 151      16.923 -14.285 -21.419  1.00  0.41           O  \\nATOM   1114  CG  PRO A 151      14.822 -11.810 -22.618  1.00  0.39           C  \\nATOM   1115  CD  PRO A 151      16.189 -11.191 -22.667  1.00  0.44           C  \\nATOM   1116  N   PRO A 152      16.325 -14.478 -19.337  1.00  0.35           N  \\nATOM   1117  CA  PRO A 152      16.603 -15.916 -19.330  1.00  0.36           C  \\nATOM   1118  C   PRO A 152      15.852 -16.668 -20.427  1.00  0.35           C  \\nATOM   1119  CB  PRO A 152      16.131 -16.360 -17.943  1.00  0.33           C  \\nATOM   1120  O   PRO A 152      14.654 -16.448 -20.622  1.00  0.33           O  \\nATOM   1121  CG  PRO A 152      15.379 -15.188 -17.403  1.00  0.32           C  \\nATOM   1122  CD  PRO A 152      15.619 -14.010 -18.303  1.00  0.37           C  \\nATOM   1123  N   GLN A 153      16.357 -16.999 -21.615  1.00  0.32           N  \\nATOM   1124  CA  GLN A 153      16.200 -18.243 -22.361  1.00  0.34           C  \\nATOM   1125  C   GLN A 153      15.714 -19.370 -21.455  1.00  0.32           C  \\nATOM   1126  CB  GLN A 153      17.516 -18.638 -23.032  1.00  0.31           C  \\nATOM   1127  O   GLN A 153      16.364 -19.697 -20.459  1.00  0.31           O  \\nATOM   1128  CG  GLN A 153      17.893 -17.757 -24.215  1.00  0.33           C  \\nATOM   1129  CD  GLN A 153      19.080 -18.293 -24.992  1.00  0.40           C  \\nATOM   1130  NE2 GLN A 153      19.364 -17.683 -26.138  1.00  0.32           N  \\nATOM   1131  OE1 GLN A 153      19.738 -19.248 -24.565  1.00  0.47           O  \\nATOM   1132  N   GLY A 154      14.334 -19.404 -20.986  1.00  0.33           N  \\nATOM   1133  CA  GLY A 154      13.882 -20.659 -20.407  1.00  0.36           C  \\nATOM   1134  C   GLY A 154      12.581 -20.528 -19.638  1.00  0.33           C  \\nATOM   1135  O   GLY A 154      12.543 -19.909 -18.573  1.00  0.32           O  \\nATOM   1136  N   TYR A 155      11.351 -20.274 -20.327  1.00  0.33           N  \\nATOM   1137  CA  TYR A 155      10.068 -20.963 -20.253  1.00  0.38           C  \\nATOM   1138  C   TYR A 155       9.404 -21.028 -21.623  1.00  0.34           C  \\nATOM   1139  CB  TYR A 155       9.139 -20.263 -19.256  1.00  0.33           C  \\nATOM   1140  O   TYR A 155       8.923 -20.015 -22.136  1.00  0.33           O  \\nATOM   1141  CG  TYR A 155       9.557 -20.430 -17.815  1.00  0.34           C  \\nATOM   1142  CD1 TYR A 155       9.222 -21.579 -17.102  1.00  0.33           C  \\nATOM   1143  CD2 TYR A 155      10.286 -19.441 -17.165  1.00  0.34           C  \\nATOM   1144  CE1 TYR A 155       9.604 -21.737 -15.774  1.00  0.33           C  \\nATOM   1145  CE2 TYR A 155      10.673 -19.589 -15.837  1.00  0.37           C  \\nATOM   1146  OH  TYR A 155      10.708 -20.890 -13.836  1.00  0.18           O  \\nATOM   1147  CZ  TYR A 155      10.328 -20.739 -15.151  1.00  0.32           C  \\nATOM   1148  N   GLY A 156       9.821 -21.783 -22.608  1.00  0.35           N  \\nATOM   1149  CA  GLY A 156       9.278 -22.837 -23.450  1.00  0.35           C  \\nATOM   1150  C   GLY A 156       8.645 -23.966 -22.659  1.00  0.35           C  \\nATOM   1151  O   GLY A 156       9.348 -24.824 -22.121  1.00  0.34           O  \\nATOM   1152  N   GLN A 157       7.878 -23.819 -21.648  1.00  0.34           N  \\nATOM   1153  CA  GLN A 157       6.921 -24.893 -21.406  1.00  0.37           C  \\nATOM   1154  C   GLN A 157       5.573 -24.584 -22.051  1.00  0.35           C  \\nATOM   1155  CB  GLN A 157       6.743 -25.127 -19.905  1.00  0.32           C  \\nATOM   1156  O   GLN A 157       4.898 -23.629 -21.664  1.00  0.33           O  \\nATOM   1157  CG  GLN A 157       7.817 -26.014 -19.289  1.00  0.33           C  \\nATOM   1158  CD  GLN A 157       7.570 -26.301 -17.820  1.00  0.40           C  \\nATOM   1159  NE2 GLN A 157       8.508 -26.996 -17.186  1.00  0.32           N  \\nATOM   1160  OE1 GLN A 157       6.545 -25.900 -17.259  1.00  0.45           O  \\nATOM   1161  N   GLN A 158       5.519 -24.564 -23.423  1.00  0.38           N  \\nATOM   1162  CA  GLN A 158       4.400 -25.169 -24.139  1.00  0.39           C  \\nATOM   1163  C   GLN A 158       3.572 -26.058 -23.215  1.00  0.38           C  \\nATOM   1164  CB  GLN A 158       4.903 -25.979 -25.335  1.00  0.36           C  \\nATOM   1165  O   GLN A 158       4.057 -27.084 -22.733  1.00  0.37           O  \\nATOM   1166  CG  GLN A 158       5.089 -25.154 -26.601  1.00  0.36           C  \\nATOM   1167  CD  GLN A 158       5.489 -25.998 -27.798  1.00  0.42           C  \\nATOM   1168  NE2 GLN A 158       5.511 -25.383 -28.975  1.00  0.35           N  \\nATOM   1169  OE1 GLN A 158       5.777 -27.191 -27.664  1.00  0.46           O  \\nATOM   1170  N   ASN A 159       2.912 -25.656 -22.255  1.00  0.37           N  \\nATOM   1171  CA  ASN A 159       1.796 -26.511 -21.862  1.00  0.42           C  \\nATOM   1172  C   ASN A 159       1.105 -27.125 -23.075  1.00  0.38           C  \\nATOM   1173  CB  ASN A 159       0.790 -25.725 -21.018  1.00  0.36           C  \\nATOM   1174  O   ASN A 159       0.495 -26.412 -23.875  1.00  0.36           O  \\nATOM   1175  CG  ASN A 159       1.093 -25.793 -19.534  1.00  0.37           C  \\nATOM   1176  ND2 ASN A 159       0.452 -24.927 -18.758  1.00  0.30           N  \\nATOM   1177  OD1 ASN A 159       1.896 -26.617 -19.089  1.00  0.38           O  \\nATOM   1178  N   GLN A 160       1.691 -28.038 -23.773  1.00  0.37           N  \\nATOM   1179  CA  GLN A 160       1.053 -29.229 -24.323  1.00  0.38           C  \\nATOM   1180  C   GLN A 160      -0.272 -29.516 -23.622  1.00  0.37           C  \\nATOM   1181  CB  GLN A 160       1.981 -30.439 -24.206  1.00  0.35           C  \\nATOM   1182  O   GLN A 160      -0.289 -30.008 -22.492  1.00  0.36           O  \\nATOM   1183  CG  GLN A 160       2.988 -30.554 -25.342  1.00  0.35           C  \\nATOM   1184  CD  GLN A 160       3.866 -31.786 -25.226  1.00  0.42           C  \\nATOM   1185  NE2 GLN A 160       4.806 -31.933 -26.153  1.00  0.34           N  \\nATOM   1186  OE1 GLN A 160       3.702 -32.597 -24.310  1.00  0.45           O  \\nATOM   1187  N   TYR A 161      -1.213 -28.657 -23.586  1.00  0.40           N  \\nATOM   1188  CA  TYR A 161      -2.479 -29.380 -23.594  1.00  0.45           C  \\nATOM   1189  C   TYR A 161      -2.593 -30.268 -24.828  1.00  0.41           C  \\nATOM   1190  CB  TYR A 161      -3.657 -28.402 -23.545  1.00  0.39           C  \\nATOM   1191  O   TYR A 161      -2.679 -29.770 -25.953  1.00  0.39           O  \\nATOM   1192  CG  TYR A 161      -4.097 -28.052 -22.145  1.00  0.40           C  \\nATOM   1193  CD1 TYR A 161      -5.019 -28.844 -21.465  1.00  0.39           C  \\nATOM   1194  CD2 TYR A 161      -3.593 -26.928 -21.499  1.00  0.39           C  \\nATOM   1195  CE1 TYR A 161      -5.429 -28.524 -20.175  1.00  0.38           C  \\nATOM   1196  CE2 TYR A 161      -3.996 -26.598 -20.209  1.00  0.42           C  \\nATOM   1197  OH  TYR A 161      -5.315 -27.080 -18.279  1.00  0.23           O  \\nATOM   1198  CZ  TYR A 161      -4.913 -27.401 -19.556  1.00  0.34           C  \\nATOM   1199  N   ASN A 162      -1.863 -31.325 -24.959  1.00  0.36           N  \\nATOM   1200  CA  ASN A 162      -2.461 -32.541 -25.499  1.00  0.40           C  \\nATOM   1201  C   ASN A 162      -3.973 -32.560 -25.294  1.00  0.37           C  \\nATOM   1202  CB  ASN A 162      -1.824 -33.780 -24.866  1.00  0.36           C  \\nATOM   1203  O   ASN A 162      -4.452 -32.843 -24.195  1.00  0.36           O  \\nATOM   1204  CG  ASN A 162      -0.980 -34.569 -25.848  1.00  0.37           C  \\nATOM   1205  ND2 ASN A 162      -0.410 -35.675 -25.384  1.00  0.31           N  \\nATOM   1206  OD1 ASN A 162      -0.842 -34.187 -27.013  1.00  0.39           O  \\nATOM   1207  N   SER A 163      -4.711 -31.724 -25.790  1.00  0.43           N  \\nATOM   1208  CA  SER A 163      -6.034 -32.242 -26.123  1.00  0.45           C  \\nATOM   1209  C   SER A 163      -5.936 -33.547 -26.906  1.00  0.43           C  \\nATOM   1210  CB  SER A 163      -6.826 -31.212 -26.930  1.00  0.41           C  \\nATOM   1211  O   SER A 163      -5.600 -33.542 -28.092  1.00  0.42           O  \\nATOM   1212  OG  SER A 163      -6.135 -30.866 -28.119  1.00  0.42           O  \\nATOM   1213  N   SER A 164      -5.301 -34.495 -26.385  1.00  0.35           N  \\nATOM   1214  CA  SER A 164      -5.715 -35.855 -26.713  1.00  0.37           C  \\nATOM   1215  C   SER A 164      -7.196 -36.069 -26.418  1.00  0.35           C  \\nATOM   1216  CB  SER A 164      -4.880 -36.873 -25.935  1.00  0.33           C  \\nATOM   1217  O   SER A 164      -7.595 -36.165 -25.256  1.00  0.33           O  \\nATOM   1218  OG  SER A 164      -5.064 -36.712 -24.539  1.00  0.35           O  \\nATOM   1219  N   SER A 165      -8.053 -35.404 -27.001  1.00  0.39           N  \\nATOM   1220  CA  SER A 165      -9.263 -36.155 -27.317  1.00  0.42           C  \\nATOM   1221  C   SER A 165      -8.928 -37.528 -27.891  1.00  0.39           C  \\nATOM   1222  CB  SER A 165     -10.134 -35.378 -28.305  1.00  0.37           C  \\nATOM   1223  O   SER A 165      -8.303 -37.628 -28.949  1.00  0.37           O  \\nATOM   1224  OG  SER A 165      -9.560 -35.393 -29.601  1.00  0.38           O  \\nATOM   1225  N   GLY A 166      -8.371 -38.351 -27.164  1.00  0.32           N  \\nATOM   1226  CA  GLY A 166      -8.633 -39.781 -27.156  1.00  0.34           C  \\nATOM   1227  C   GLY A 166      -9.698 -40.199 -28.152  1.00  0.32           C  \\nATOM   1228  O   GLY A 166     -10.893 -40.139 -27.853  1.00  0.31           O  \\nATOM   1229  N   GLY A 167      -9.353 -40.065 -29.439  1.00  0.34           N  \\nATOM   1230  CA  GLY A 167      -9.923 -41.090 -30.298  1.00  0.36           C  \\nATOM   1231  C   GLY A 167      -8.912 -42.134 -30.733  1.00  0.35           C  \\nATOM   1232  O   GLY A 167      -7.872 -41.799 -31.303  1.00  0.33           O  \\nATOM   1233  N   GLY A 168      -8.567 -43.035 -29.923  1.00  0.30           N  \\nATOM   1234  CA  GLY A 168      -8.580 -44.464 -30.192  1.00  0.35           C  \\nATOM   1235  C   GLY A 168      -8.280 -44.801 -31.640  1.00  0.30           C  \\nATOM   1236  O   GLY A 168      -9.045 -44.444 -32.538  1.00  0.30           O  \\nATOM   1237  N   GLY A 169      -6.923 -44.833 -31.988  1.00  0.31           N  \\nATOM   1238  CA  GLY A 169      -6.664 -45.958 -32.874  1.00  0.35           C  \\nATOM   1239  C   GLY A 169      -5.450 -45.751 -33.760  1.00  0.31           C  \\nATOM   1240  O   GLY A 169      -5.366 -44.758 -34.485  1.00  0.31           O  \\nATOM   1241  N   GLY A 170      -4.264 -46.025 -33.253  1.00  0.30           N  \\nATOM   1242  CA  GLY A 170      -3.505 -46.977 -34.049  1.00  0.35           C  \\nATOM   1243  C   GLY A 170      -2.458 -46.319 -34.928  1.00  0.31           C  \\nATOM   1244  O   GLY A 170      -2.784 -45.478 -35.768  1.00  0.30           O  \\nATOM   1245  N   GLY A 171      -1.270 -46.045 -34.319  1.00  0.32           N  \\nATOM   1246  CA  GLY A 171      -0.157 -46.600 -35.072  1.00  0.36           C  \\nATOM   1247  C   GLY A 171       0.822 -45.546 -35.553  1.00  0.32           C  \\nATOM   1248  O   GLY A 171       0.435 -44.601 -36.243  1.00  0.32           O  \\nATOM   1249  N   GLY A 172       1.863 -45.185 -34.747  1.00  0.31           N  \\nATOM   1250  CA  GLY A 172       3.173 -45.414 -35.337  1.00  0.36           C  \\nATOM   1251  C   GLY A 172       3.678 -44.234 -36.146  1.00  0.32           C  \\nATOM   1252  O   GLY A 172       3.172 -43.960 -37.236  1.00  0.32           O  \\nATOM   1253  N   GLY A 173       4.212 -43.196 -35.419  1.00  0.34           N  \\nATOM   1254  CA  GLY A 173       5.447 -42.689 -35.997  1.00  0.36           C  \\nATOM   1255  C   GLY A 173       5.946 -41.424 -35.324  1.00  0.34           C  \\nATOM   1256  O   GLY A 173       5.213 -40.439 -35.222  1.00  0.34           O  \\nATOM   1257  N   GLY A 174       6.756 -41.507 -34.232  1.00  0.35           N  \\nATOM   1258  CA  GLY A 174       7.976 -40.725 -34.110  1.00  0.37           C  \\nATOM   1259  C   GLY A 174       8.100 -39.644 -35.167  1.00  0.35           C  \\nATOM   1260  O   GLY A 174       8.516 -39.916 -36.295  1.00  0.34           O  \\nATOM   1261  N   GLY A 175       7.266 -38.772 -35.404  1.00  0.34           N  \\nATOM   1262  CA  GLY A 175       7.704 -37.620 -36.177  1.00  0.36           C  \\nATOM   1263  C   GLY A 175       8.292 -36.515 -35.320  1.00  0.35           C  \\nATOM   1264  O   GLY A 175       7.674 -36.082 -34.346  1.00  0.34           O  \\nATOM   1265  N   ASN A 176       9.640 -36.729 -34.946  1.00  0.34           N  \\nATOM   1266  CA  ASN A 176      10.494 -35.548 -35.002  1.00  0.38           C  \\nATOM   1267  C   ASN A 176       9.701 -34.299 -35.377  1.00  0.35           C  \\nATOM   1268  CB  ASN A 176      11.643 -35.763 -35.989  1.00  0.33           C  \\nATOM   1269  O   ASN A 176       9.218 -34.181 -36.505  1.00  0.34           O  \\nATOM   1270  CG  ASN A 176      12.813 -34.832 -35.736  1.00  0.34           C  \\nATOM   1271  ND2 ASN A 176      13.676 -34.680 -36.733  1.00  0.29           N  \\nATOM   1272  OD1 ASN A 176      12.939 -34.254 -34.653  1.00  0.35           O  \\nATOM   1273  N   TYR A 177       8.884 -33.852 -34.528  1.00  0.35           N  \\nATOM   1274  CA  TYR A 177       8.470 -32.465 -34.706  1.00  0.40           C  \\nATOM   1275  C   TYR A 177       9.674 -31.565 -34.959  1.00  0.36           C  \\nATOM   1276  CB  TYR A 177       7.699 -31.972 -33.478  1.00  0.35           C  \\nATOM   1277  O   TYR A 177      10.482 -31.331 -34.058  1.00  0.35           O  \\nATOM   1278  CG  TYR A 177       6.314 -32.559 -33.356  1.00  0.37           C  \\nATOM   1279  CD1 TYR A 177       5.238 -32.006 -34.046  1.00  0.36           C  \\nATOM   1280  CD2 TYR A 177       6.078 -33.667 -32.549  1.00  0.37           C  \\nATOM   1281  CE1 TYR A 177       3.959 -32.542 -33.935  1.00  0.36           C  \\nATOM   1282  CE2 TYR A 177       4.803 -34.211 -32.430  1.00  0.40           C  \\nATOM   1283  OH  TYR A 177       2.488 -34.178 -33.012  1.00  0.21           O  \\nATOM   1284  CZ  TYR A 177       3.752 -33.643 -33.126  1.00  0.32           C  \\nATOM   1285  N   GLY A 178      10.352 -31.855 -35.997  1.00  0.34           N  \\nATOM   1286  CA  GLY A 178      11.144 -30.850 -36.688  1.00  0.34           C  \\nATOM   1287  C   GLY A 178      10.506 -29.473 -36.670  1.00  0.34           C  \\nATOM   1288  O   GLY A 178       9.311 -29.333 -36.939  1.00  0.33           O  \\nATOM   1289  N   GLN A 179      10.590 -28.829 -35.611  1.00  0.38           N  \\nATOM   1290  CA  GLN A 179      10.562 -27.402 -35.915  1.00  0.41           C  \\nATOM   1291  C   GLN A 179      10.429 -27.162 -37.416  1.00  0.38           C  \\nATOM   1292  CB  GLN A 179      11.820 -26.714 -35.383  1.00  0.36           C  \\nATOM   1293  O   GLN A 179      11.386 -27.356 -38.168  1.00  0.37           O  \\nATOM   1294  CG  GLN A 179      11.640 -26.079 -34.011  1.00  0.37           C  \\nATOM   1295  CD  GLN A 179      12.911 -25.433 -33.492  1.00  0.44           C  \\nATOM   1296  NE2 GLN A 179      12.832 -24.837 -32.307  1.00  0.36           N  \\nATOM   1297  OE1 GLN A 179      13.956 -25.471 -34.150  1.00  0.49           O  \\nATOM   1298  N   ASP A 180       9.345 -27.572 -37.977  1.00  0.34           N  \\nATOM   1299  CA  ASP A 180       8.994 -26.987 -39.267  1.00  0.36           C  \\nATOM   1300  C   ASP A 180       8.251 -25.665 -39.087  1.00  0.34           C  \\nATOM   1301  CB  ASP A 180       8.143 -27.962 -40.085  1.00  0.32           C  \\nATOM   1302  O   ASP A 180       7.124 -25.643 -38.588  1.00  0.33           O  \\nATOM   1303  CG  ASP A 180       8.959 -28.775 -41.074  1.00  0.33           C  \\nATOM   1304  OD1 ASP A 180      10.060 -28.333 -41.466  1.00  0.37           O  \\nATOM   1305  OD2 ASP A 180       8.495 -29.867 -41.467  1.00  0.36           O  \\nATOM   1306  N   GLN A 181       8.842 -24.650 -38.521  1.00  0.37           N  \\nATOM   1307  CA  GLN A 181       8.709 -23.275 -38.989  1.00  0.41           C  \\nATOM   1308  C   GLN A 181       8.546 -23.223 -40.505  1.00  0.37           C  \\nATOM   1309  CB  GLN A 181       9.919 -22.442 -38.561  1.00  0.36           C  \\nATOM   1310  O   GLN A 181       9.530 -23.308 -41.243  1.00  0.37           O  \\nATOM   1311  CG  GLN A 181       9.847 -21.947 -37.123  1.00  0.38           C  \\nATOM   1312  CD  GLN A 181      10.996 -21.023 -36.763  1.00  0.47           C  \\nATOM   1313  NE2 GLN A 181      11.150 -20.743 -35.474  1.00  0.38           N  \\nATOM   1314  OE1 GLN A 181      11.739 -20.565 -37.637  1.00  0.51           O  \\nATOM   1315  N   SER A 182       7.731 -24.026 -40.943  1.00  0.32           N  \\nATOM   1316  CA  SER A 182       7.218 -24.014 -42.310  1.00  0.36           C  \\nATOM   1317  C   SER A 182       5.831 -24.642 -42.385  1.00  0.32           C  \\nATOM   1318  CB  SER A 182       8.173 -24.754 -43.247  1.00  0.30           C  \\nATOM   1319  O   SER A 182       5.670 -25.837 -42.128  1.00  0.31           O  \\nATOM   1320  OG  SER A 182       8.508 -26.027 -42.722  1.00  0.29           O  \\nATOM   1321  N   SER A 183       4.752 -23.978 -41.665  1.00  0.34           N  \\nATOM   1322  CA  SER A 183       3.337 -24.006 -42.020  1.00  0.37           C  \\nATOM   1323  C   SER A 183       2.469 -23.502 -40.872  1.00  0.35           C  \\nATOM   1324  CB  SER A 183       2.909 -25.421 -42.410  1.00  0.33           C  \\nATOM   1325  O   SER A 183       2.174 -24.248 -39.936  1.00  0.33           O  \\nATOM   1326  OG  SER A 183       3.192 -26.338 -41.367  1.00  0.34           O  \\nATOM   1327  N   MET A 184       2.687 -22.256 -40.365  1.00  0.39           N  \\nATOM   1328  CA  MET A 184       1.414 -21.585 -40.119  1.00  0.41           C  \\nATOM   1329  C   MET A 184       0.564 -21.548 -41.385  1.00  0.39           C  \\nATOM   1330  CB  MET A 184       1.645 -20.164 -39.603  1.00  0.37           C  \\nATOM   1331  O   MET A 184       0.843 -20.773 -42.302  1.00  0.38           O  \\nATOM   1332  CG  MET A 184       2.017 -20.100 -38.130  1.00  0.38           C  \\nATOM   1333  SD  MET A 184       2.204 -18.377 -37.527  1.00  0.34           S  \\nATOM   1334  CE  MET A 184       2.315 -18.674 -35.740  1.00  0.37           C  \\nATOM   1335  N   SER A 185       0.274 -22.587 -41.937  1.00  0.34           N  \\nATOM   1336  CA  SER A 185      -0.829 -22.718 -42.883  1.00  0.38           C  \\nATOM   1337  C   SER A 185      -2.140 -22.231 -42.274  1.00  0.35           C  \\nATOM   1338  CB  SER A 185      -0.978 -24.171 -43.336  1.00  0.33           C  \\nATOM   1339  O   SER A 185      -2.498 -22.625 -41.162  1.00  0.33           O  \\nATOM   1340  OG  SER A 185      -2.183 -24.731 -42.845  1.00  0.34           O  \\nATOM   1341  N   SER A 186      -2.491 -20.871 -42.316  1.00  0.35           N  \\nATOM   1342  CA  SER A 186      -3.699 -20.157 -42.717  1.00  0.38           C  \\nATOM   1343  C   SER A 186      -4.947 -20.813 -42.136  1.00  0.35           C  \\nATOM   1344  CB  SER A 186      -3.806 -20.098 -44.241  1.00  0.33           C  \\nATOM   1345  O   SER A 186      -5.376 -21.869 -42.607  1.00  0.34           O  \\nATOM   1346  OG  SER A 186      -3.980 -21.395 -44.784  1.00  0.36           O  \\nATOM   1347  N   GLY A 187      -5.169 -20.889 -40.908  1.00  0.34           N  \\nATOM   1348  CA  GLY A 187      -6.542 -20.982 -40.439  1.00  0.36           C  \\nATOM   1349  C   GLY A 187      -7.210 -19.630 -40.277  1.00  0.35           C  \\nATOM   1350  O   GLY A 187      -6.824 -18.840 -39.412  1.00  0.34           O  \\nATOM   1351  N   GLY A 188      -7.624 -18.958 -41.377  1.00  0.36           N  \\nATOM   1352  CA  GLY A 188      -8.792 -18.104 -41.529  1.00  0.38           C  \\nATOM   1353  C   GLY A 188      -9.821 -18.299 -40.432  1.00  0.36           C  \\nATOM   1354  O   GLY A 188     -10.522 -19.312 -40.404  1.00  0.35           O  \\nATOM   1355  N   GLY A 189      -9.627 -17.887 -39.334  1.00  0.38           N  \\nATOM   1356  CA  GLY A 189     -10.744 -17.675 -38.426  1.00  0.40           C  \\nATOM   1357  C   GLY A 189     -11.579 -16.460 -38.781  1.00  0.38           C  \\nATOM   1358  O   GLY A 189     -11.062 -15.344 -38.853  1.00  0.38           O  \\nATOM   1359  N   SER A 190     -12.602 -16.533 -39.626  1.00  0.35           N  \\nATOM   1360  CA  SER A 190     -13.835 -15.757 -39.698  1.00  0.38           C  \\nATOM   1361  C   SER A 190     -14.186 -15.151 -38.343  1.00  0.36           C  \\nATOM   1362  CB  SER A 190     -14.991 -16.629 -40.191  1.00  0.35           C  \\nATOM   1363  O   SER A 190     -14.634 -15.859 -37.438  1.00  0.36           O  \\nATOM   1364  OG  SER A 190     -16.139 -15.841 -40.454  1.00  0.36           O  \\nATOM   1365  N   GLY A 191     -13.503 -14.185 -37.973  1.00  0.39           N  \\nATOM   1366  CA  GLY A 191     -13.953 -13.237 -36.967  1.00  0.40           C  \\nATOM   1367  C   GLY A 191     -14.453 -11.932 -37.558  1.00  0.40           C  \\nATOM   1368  O   GLY A 191     -13.716 -11.241 -38.263  1.00  0.39           O  \\nATOM   1369  N   GLY A 192     -15.662 -11.963 -37.980  1.00  0.38           N  \\nATOM   1370  CA  GLY A 192     -16.556 -10.833 -38.178  1.00  0.40           C  \\nATOM   1371  C   GLY A 192     -15.956  -9.513 -37.734  1.00  0.38           C  \\nATOM   1372  O   GLY A 192     -15.573  -9.359 -36.572  1.00  0.37           O  \\nATOM   1373  N   GLY A 193     -15.233  -8.844 -38.776  1.00  0.37           N  \\nATOM   1374  CA  GLY A 193     -15.020  -7.460 -39.166  1.00  0.37           C  \\nATOM   1375  C   GLY A 193     -15.821  -6.476 -38.335  1.00  0.37           C  \\nATOM   1376  O   GLY A 193     -17.046  -6.406 -38.455  1.00  0.36           O  \\nATOM   1377  N   TYR A 194     -15.333  -6.140 -37.278  1.00  0.40           N  \\nATOM   1378  CA  TYR A 194     -15.768  -4.833 -36.798  1.00  0.46           C  \\nATOM   1379  C   TYR A 194     -14.579  -3.901 -36.601  1.00  0.40           C  \\nATOM   1380  CB  TYR A 194     -16.544  -4.973 -35.485  1.00  0.39           C  \\nATOM   1381  O   TYR A 194     -13.767  -4.103 -35.695  1.00  0.40           O  \\nATOM   1382  CG  TYR A 194     -17.908  -5.599 -35.650  1.00  0.42           C  \\nATOM   1383  CD1 TYR A 194     -18.997  -4.841 -36.074  1.00  0.42           C  \\nATOM   1384  CD2 TYR A 194     -18.111  -6.948 -35.381  1.00  0.43           C  \\nATOM   1385  CE1 TYR A 194     -20.256  -5.412 -36.225  1.00  0.39           C  \\nATOM   1386  CE2 TYR A 194     -19.365  -7.530 -35.529  1.00  0.43           C  \\nATOM   1387  OH  TYR A 194     -21.674  -7.327 -36.099  1.00  0.27           O  \\nATOM   1388  CZ  TYR A 194     -20.430  -6.756 -35.951  1.00  0.37           C  \\nATOM   1389  N   GLY A 195     -13.937  -3.349 -37.900  1.00  0.40           N  \\nATOM   1390  CA  GLY A 195     -13.421  -1.990 -37.948  1.00  0.43           C  \\nATOM   1391  C   GLY A 195     -12.814  -1.629 -39.290  1.00  0.40           C  \\nATOM   1392  O   GLY A 195     -12.080  -2.425 -39.880  1.00  0.39           O  \\nATOM   1393  N   ASN A 196     -13.521  -0.837 -40.126  1.00  0.40           N  \\nATOM   1394  CA  ASN A 196     -13.169   0.089 -41.197  1.00  0.46           C  \\nATOM   1395  C   ASN A 196     -11.851  -0.298 -41.862  1.00  0.41           C  \\nATOM   1396  CB  ASN A 196     -13.094   1.522 -40.665  1.00  0.39           C  \\nATOM   1397  O   ASN A 196     -10.793  -0.236 -41.235  1.00  0.40           O  \\nATOM   1398  CG  ASN A 196     -13.988   2.477 -41.432  1.00  0.40           C  \\nATOM   1399  ND2 ASN A 196     -13.832   3.770 -41.175  1.00  0.34           N  \\nATOM   1400  OD1 ASN A 196     -14.812   2.055 -42.248  1.00  0.42           O  \\nATOM   1401  N   GLN A 197     -11.883  -1.213 -42.735  1.00  0.43           N  \\nATOM   1402  CA  GLN A 197     -11.311  -1.139 -44.075  1.00  0.45           C  \\nATOM   1403  C   GLN A 197     -11.442   0.268 -44.652  1.00  0.43           C  \\nATOM   1404  CB  GLN A 197     -11.984  -2.151 -45.003  1.00  0.40           C  \\nATOM   1405  O   GLN A 197     -12.541   0.700 -45.006  1.00  0.42           O  \\nATOM   1406  CG  GLN A 197     -11.197  -3.443 -45.178  1.00  0.41           C  \\nATOM   1407  CD  GLN A 197     -11.895  -4.438 -46.086  1.00  0.47           C  \\nATOM   1408  NE2 GLN A 197     -11.324  -5.632 -46.208  1.00  0.39           N  \\nATOM   1409  OE1 GLN A 197     -12.940  -4.137 -46.672  1.00  0.51           O  \\nATOM   1410  N   ASP A 198     -10.521   1.154 -44.192  1.00  0.43           N  \\nATOM   1411  CA  ASP A 198      -9.974   2.056 -45.200  1.00  0.46           C  \\nATOM   1412  C   ASP A 198      -8.750   1.444 -45.879  1.00  0.44           C  \\nATOM   1413  CB  ASP A 198      -9.609   3.404 -44.574  1.00  0.41           C  \\nATOM   1414  O   ASP A 198      -7.672   1.380 -45.285  1.00  0.42           O  \\nATOM   1415  CG  ASP A 198     -10.742   4.413 -44.635  1.00  0.41           C  \\nATOM   1416  OD1 ASP A 198     -11.617   4.295 -45.520  1.00  0.44           O  \\nATOM   1417  OD2 ASP A 198     -10.759   5.336 -43.792  1.00  0.49           O  \\nATOM   1418  N   GLN A 199      -8.918   0.489 -46.765  1.00  0.44           N  \\nATOM   1419  CA  GLN A 199      -8.189   0.241 -48.004  1.00  0.46           C  \\nATOM   1420  C   GLN A 199      -8.206   1.471 -48.908  1.00  0.44           C  \\nATOM   1421  CB  GLN A 199      -8.777  -0.962 -48.743  1.00  0.41           C  \\nATOM   1422  O   GLN A 199      -9.190   1.718 -49.609  1.00  0.43           O  \\nATOM   1423  CG  GLN A 199      -8.384  -2.305 -48.142  1.00  0.42           C  \\nATOM   1424  CD  GLN A 199      -8.932  -3.482 -48.926  1.00  0.49           C  \\nATOM   1425  NE2 GLN A 199      -9.158  -4.598 -48.241  1.00  0.39           N  \\nATOM   1426  OE1 GLN A 199      -9.151  -3.389 -50.138  1.00  0.53           O  \\nATOM   1427  N   SER A 200      -7.630   2.521 -48.405  1.00  0.39           N  \\nATOM   1428  CA  SER A 200      -7.096   3.548 -49.294  1.00  0.43           C  \\nATOM   1429  C   SER A 200      -5.585   3.682 -49.139  1.00  0.39           C  \\nATOM   1430  CB  SER A 200      -7.766   4.895 -49.021  1.00  0.36           C  \\nATOM   1431  O   SER A 200      -5.096   4.061 -48.072  1.00  0.38           O  \\nATOM   1432  OG  SER A 200      -7.210   5.510 -47.871  1.00  0.41           O  \\nATOM   1433  N   GLY A 201      -4.790   2.812 -49.740  1.00  0.37           N  \\nATOM   1434  CA  GLY A 201      -3.552   3.307 -50.321  1.00  0.40           C  \\nATOM   1435  C   GLY A 201      -2.530   2.214 -50.568  1.00  0.37           C  \\nATOM   1436  O   GLY A 201      -2.179   1.467 -49.652  1.00  0.35           O  \\nATOM   1437  N   GLY A 202      -2.573   1.712 -51.799  1.00  0.35           N  \\nATOM   1438  CA  GLY A 202      -1.452   1.688 -52.724  1.00  0.38           C  \\nATOM   1439  C   GLY A 202      -0.647   0.404 -52.653  1.00  0.35           C  \\nATOM   1440  O   GLY A 202      -0.170   0.022 -51.582  1.00  0.34           O  \\nATOM   1441  N   GLY A 203      -1.051  -0.697 -53.438  1.00  0.36           N  \\nATOM   1442  CA  GLY A 203      -0.094  -1.287 -54.361  1.00  0.40           C  \\nATOM   1443  C   GLY A 203       0.873  -2.242 -53.687  1.00  0.37           C  \\nATOM   1444  O   GLY A 203       1.777  -1.814 -52.967  1.00  0.36           O  \\nATOM   1445  N   GLY A 204       0.347  -3.680 -53.302  1.00  0.38           N  \\nATOM   1446  CA  GLY A 204       1.208  -4.799 -53.651  1.00  0.41           C  \\nATOM   1447  C   GLY A 204       0.516  -6.144 -53.532  1.00  0.38           C  \\nATOM   1448  O   GLY A 204      -0.075  -6.454 -52.496  1.00  0.37           O  \\nATOM   1449  N   SER A 205      -0.245  -6.803 -54.585  1.00  0.34           N  \\nATOM   1450  CA  SER A 205      -0.352  -8.077 -55.288  1.00  0.39           C  \\nATOM   1451  C   SER A 205       0.200  -9.222 -54.445  1.00  0.35           C  \\nATOM   1452  CB  SER A 205       0.387  -8.016 -56.626  1.00  0.33           C  \\nATOM   1453  O   SER A 205       1.406  -9.291 -54.196  1.00  0.34           O  \\nATOM   1454  OG  SER A 205       0.411  -9.290 -57.246  1.00  0.38           O  \\nATOM   1455  N   GLY A 206      -0.625  -9.702 -53.341  1.00  0.38           N  \\nATOM   1456  CA  GLY A 206      -0.517 -11.087 -52.910  1.00  0.41           C  \\nATOM   1457  C   GLY A 206      -1.752 -11.584 -52.182  1.00  0.38           C  \\nATOM   1458  O   GLY A 206      -2.166 -10.996 -51.180  1.00  0.37           O  \\nATOM   1459  N   GLY A 207      -2.830 -12.198 -53.008  1.00  0.39           N  \\nATOM   1460  CA  GLY A 207      -3.893 -13.189 -53.057  1.00  0.41           C  \\nATOM   1461  C   GLY A 207      -4.133 -13.872 -51.724  1.00  0.40           C  \\nATOM   1462  O   GLY A 207      -3.319 -14.685 -51.281  1.00  0.39           O  \\nATOM   1463  N   TYR A 208      -4.694 -13.299 -50.706  1.00  0.41           N  \\nATOM   1464  CA  TYR A 208      -5.295 -13.985 -49.567  1.00  0.45           C  \\nATOM   1465  C   TYR A 208      -6.707 -14.453 -49.896  1.00  0.41           C  \\nATOM   1466  CB  TYR A 208      -5.321 -13.069 -48.340  1.00  0.39           C  \\nATOM   1467  O   TYR A 208      -7.580 -13.641 -50.212  1.00  0.40           O  \\nATOM   1468  CG  TYR A 208      -3.965 -12.855 -47.713  1.00  0.39           C  \\nATOM   1469  CD1 TYR A 208      -3.419 -13.800 -46.848  1.00  0.38           C  \\nATOM   1470  CD2 TYR A 208      -3.228 -11.707 -47.984  1.00  0.40           C  \\nATOM   1471  CE1 TYR A 208      -2.170 -13.606 -46.266  1.00  0.38           C  \\nATOM   1472  CE2 TYR A 208      -1.978 -11.503 -47.408  1.00  0.41           C  \\nATOM   1473  OH  TYR A 208      -0.222 -12.259 -45.980  1.00  0.29           O  \\nATOM   1474  CZ  TYR A 208      -1.458 -12.456 -46.553  1.00  0.35           C  \\nATOM   1475  N   GLY A 209      -6.849 -15.607 -50.552  1.00  0.39           N  \\nATOM   1476  CA  GLY A 209      -7.943 -16.542 -50.764  1.00  0.41           C  \\nATOM   1477  C   GLY A 209      -8.960 -16.535 -49.639  1.00  0.40           C  \\nATOM   1478  O   GLY A 209      -8.623 -16.828 -48.489  1.00  0.39           O  \\nATOM   1479  N   GLN A 210      -9.925 -15.575 -49.450  1.00  0.42           N  \\nATOM   1480  CA  GLN A 210     -11.282 -15.841 -48.985  1.00  0.44           C  \\nATOM   1481  C   GLN A 210     -11.702 -17.273 -49.303  1.00  0.42           C  \\nATOM   1482  CB  GLN A 210     -12.268 -14.853 -49.611  1.00  0.40           C  \\nATOM   1483  O   GLN A 210     -11.886 -17.627 -50.469  1.00  0.41           O  \\nATOM   1484  CG  GLN A 210     -13.322 -14.337 -48.641  1.00  0.40           C  \\nATOM   1485  CD  GLN A 210     -14.171 -13.227 -49.232  1.00  0.46           C  \\nATOM   1486  NE2 GLN A 210     -13.703 -12.643 -50.330  1.00  0.38           N  \\nATOM   1487  OE1 GLN A 210     -15.240 -12.899 -48.708  1.00  0.51           O  \\nATOM   1488  N   GLN A 211     -11.252 -18.250 -48.516  1.00  0.44           N  \\nATOM   1489  CA  GLN A 211     -12.036 -19.481 -48.515  1.00  0.46           C  \\nATOM   1490  C   GLN A 211     -13.447 -19.234 -47.986  1.00  0.44           C  \\nATOM   1491  CB  GLN A 211     -11.344 -20.559 -47.681  1.00  0.41           C  \\nATOM   1492  O   GLN A 211     -13.627 -18.924 -46.807  1.00  0.43           O  \\nATOM   1493  CG  GLN A 211     -10.658 -21.634 -48.512  1.00  0.41           C  \\nATOM   1494  CD  GLN A 211     -10.029 -22.722 -47.662  1.00  0.46           C  \\nATOM   1495  NE2 GLN A 211      -9.154 -22.326 -46.744  1.00  0.39           N  \\nATOM   1496  OE1 GLN A 211     -10.329 -23.909 -47.827  1.00  0.50           O  \\nATOM   1497  N   ASP A 212     -14.427 -18.849 -48.854  1.00  0.41           N  \\nATOM   1498  CA  ASP A 212     -15.812 -19.269 -49.044  1.00  0.42           C  \\nATOM   1499  C   ASP A 212     -15.959 -20.777 -48.854  1.00  0.41           C  \\nATOM   1500  CB  ASP A 212     -16.311 -18.862 -50.431  1.00  0.39           C  \\nATOM   1501  O   ASP A 212     -15.566 -21.559 -49.723  1.00  0.40           O  \\nATOM   1502  CG  ASP A 212     -17.355 -17.760 -50.386  1.00  0.38           C  \\nATOM   1503  OD1 ASP A 212     -17.985 -17.560 -49.326  1.00  0.42           O  \\nATOM   1504  OD2 ASP A 212     -17.550 -17.087 -51.421  1.00  0.45           O  \\nATOM   1505  N   ARG A 213     -15.756 -21.297 -47.646  1.00  0.44           N  \\nATOM   1506  CA  ARG A 213     -16.341 -22.603 -47.360  1.00  0.44           C  \\nATOM   1507  C   ARG A 213     -17.846 -22.595 -47.608  1.00  0.44           C  \\nATOM   1508  CB  ARG A 213     -16.051 -23.018 -45.916  1.00  0.42           C  \\nATOM   1509  O   ARG A 213     -18.609 -22.042 -46.813  1.00  0.42           O  \\nATOM   1510  CG  ARG A 213     -14.646 -23.559 -45.702  1.00  0.40           C  \\nATOM   1511  CD  ARG A 213     -14.436 -24.031 -44.270  1.00  0.44           C  \\nATOM   1512  NE  ARG A 213     -13.101 -24.589 -44.076  1.00  0.40           N  \\nATOM   1513  NH1 ARG A 213     -13.334 -24.897 -41.799  1.00  0.31           N  \\nATOM   1514  NH2 ARG A 213     -11.379 -25.479 -42.844  1.00  0.26           N  \\nATOM   1515  CZ  ARG A 213     -12.607 -24.987 -42.907  1.00  0.38           C  \\nATOM   1516  N   GLY A 214     -18.289 -22.529 -48.707  1.00  0.38           N  \\nATOM   1517  CA  GLY A 214     -19.369 -23.268 -49.342  1.00  0.39           C  \\nATOM   1518  C   GLY A 214     -19.717 -24.555 -48.618  1.00  0.39           C  \\nATOM   1519  O   GLY A 214     -19.062 -25.581 -48.815  1.00  0.37           O  \\nATOM   1520  N   GLY A 215     -20.200 -24.495 -47.404  1.00  0.40           N  \\nATOM   1521  CA  GLY A 215     -21.018 -25.633 -47.014  1.00  0.42           C  \\nATOM   1522  C   GLY A 215     -22.497 -25.424 -47.279  1.00  0.41           C  \\nATOM   1523  O   GLY A 215     -23.127 -24.561 -46.665  1.00  0.40           O  \\nATOM   1524  N   ARG A 216     -22.981 -25.489 -48.450  1.00  0.39           N  \\nATOM   1525  CA  ARG A 216     -24.216 -26.189 -48.790  1.00  0.39           C  \\nATOM   1526  C   ARG A 216     -24.325 -27.507 -48.029  1.00  0.39           C  \\nATOM   1527  CB  ARG A 216     -24.292 -26.446 -50.296  1.00  0.37           C  \\nATOM   1528  O   ARG A 216     -23.691 -28.497 -48.399  1.00  0.38           O  \\nATOM   1529  CG  ARG A 216     -24.885 -25.292 -51.088  1.00  0.38           C  \\nATOM   1530  CD  ARG A 216     -25.001 -25.625 -52.569  1.00  0.44           C  \\nATOM   1531  NE  ARG A 216     -25.516 -24.495 -53.337  1.00  0.37           N  \\nATOM   1532  NH1 ARG A 216     -25.510 -25.603 -55.364  1.00  0.28           N  \\nATOM   1533  NH2 ARG A 216     -26.216 -23.426 -55.246  1.00  0.23           N  \\nATOM   1534  CZ  ARG A 216     -25.746 -24.511 -54.647  1.00  0.34           C  \\nATOM   1535  N   GLY A 217     -24.633 -27.499 -46.695  1.00  0.39           N  \\nATOM   1536  CA  GLY A 217     -25.359 -28.669 -46.228  1.00  0.41           C  \\nATOM   1537  C   GLY A 217     -26.832 -28.398 -45.983  1.00  0.39           C  \\nATOM   1538  O   GLY A 217     -27.184 -27.513 -45.201  1.00  0.38           O  \\nATOM   1539  N   ARG A 218     -27.615 -28.377 -46.945  1.00  0.33           N  \\nATOM   1540  CA  ARG A 218     -28.885 -29.087 -47.055  1.00  0.35           C  \\nATOM   1541  C   ARG A 218     -28.994 -30.181 -45.998  1.00  0.33           C  \\nATOM   1542  CB  ARG A 218     -29.045 -29.691 -48.452  1.00  0.31           C  \\nATOM   1543  O   ARG A 218     -28.134 -31.061 -45.917  1.00  0.32           O  \\nATOM   1544  CG  ARG A 218     -30.110 -29.012 -49.298  1.00  0.33           C  \\nATOM   1545  CD  ARG A 218     -30.184 -29.608 -50.697  1.00  0.40           C  \\nATOM   1546  NE  ARG A 218     -31.207 -28.954 -51.508  1.00  0.36           N  \\nATOM   1547  NH1 ARG A 218     -31.037 -30.427 -53.280  1.00  0.28           N  \\nATOM   1548  NH2 ARG A 218     -32.529 -28.689 -53.367  1.00  0.23           N  \\nATOM   1549  CZ  ARG A 218     -31.589 -29.358 -52.716  1.00  0.31           C  \\nATOM   1550  N   GLY A 219     -29.528 -29.813 -44.622  1.00  0.33           N  \\nATOM   1551  CA  GLY A 219     -30.471 -30.759 -44.048  1.00  0.38           C  \\nATOM   1552  C   GLY A 219     -29.967 -31.405 -42.772  1.00  0.34           C  \\nATOM   1553  O   GLY A 219     -29.086 -32.266 -42.811  1.00  0.33           O  \\nATOM   1554  N   GLY A 220     -29.851 -30.754 -41.619  1.00  0.34           N  \\nATOM   1555  CA  GLY A 220     -30.455 -31.113 -40.347  1.00  0.37           C  \\nATOM   1556  C   GLY A 220     -30.743 -29.915 -39.463  1.00  0.35           C  \\nATOM   1557  O   GLY A 220     -29.847 -29.119 -39.176  1.00  0.34           O  \\nATOM   1558  N   SER A 221     -31.663 -29.034 -39.837  1.00  0.31           N  \\nATOM   1559  CA  SER A 221     -32.626 -28.333 -38.994  1.00  0.35           C  \\nATOM   1560  C   SER A 221     -32.451 -28.707 -37.526  1.00  0.32           C  \\nATOM   1561  CB  SER A 221     -34.055 -28.642 -39.440  1.00  0.30           C  \\nATOM   1562  O   SER A 221     -32.636 -29.865 -37.148  1.00  0.31           O  \\nATOM   1563  OG  SER A 221     -34.990 -27.866 -38.710  1.00  0.31           O  \\nATOM   1564  N   GLY A 222     -31.247 -28.530 -36.883  1.00  0.34           N  \\nATOM   1565  CA  GLY A 222     -31.432 -28.304 -35.458  1.00  0.36           C  \\nATOM   1566  C   GLY A 222     -30.155 -28.471 -34.656  1.00  0.35           C  \\nATOM   1567  O   GLY A 222     -30.161 -28.327 -33.432  1.00  0.33           O  \\nATOM   1568  N   GLY A 223     -28.858 -28.676 -35.319  1.00  0.34           N  \\nATOM   1569  CA  GLY A 223     -27.707 -28.966 -34.479  1.00  0.36           C  \\nATOM   1570  C   GLY A 223     -26.508 -28.087 -34.785  1.00  0.35           C  \\nATOM   1571  O   GLY A 223     -25.556 -28.033 -34.004  1.00  0.33           O  \\nATOM   1572  N   GLY A 224     -26.531 -27.404 -35.988  1.00  0.35           N  \\nATOM   1573  CA  GLY A 224     -25.313 -26.692 -36.342  1.00  0.36           C  \\nATOM   1574  C   GLY A 224     -25.156 -25.376 -35.606  1.00  0.35           C  \\nATOM   1575  O   GLY A 224     -24.037 -24.961 -35.297  1.00  0.33           O  \\nATOM   1576  N   GLY A 225     -26.404 -24.795 -35.208  1.00  0.38           N  \\nATOM   1577  CA  GLY A 225     -26.358 -23.539 -34.476  1.00  0.39           C  \\nATOM   1578  C   GLY A 225     -25.790 -23.685 -33.077  1.00  0.38           C  \\nATOM   1579  O   GLY A 225     -24.970 -22.871 -32.647  1.00  0.37           O  \\nATOM   1580  N   GLN A 226     -26.146 -24.861 -32.465  1.00  0.40           N  \\nATOM   1581  CA  GLN A 226     -25.655 -25.100 -31.111  1.00  0.41           C  \\nATOM   1582  C   GLN A 226     -24.153 -25.370 -31.110  1.00  0.40           C  \\nATOM   1583  CB  GLN A 226     -26.400 -26.271 -30.469  1.00  0.38           C  \\nATOM   1584  O   GLN A 226     -23.424 -24.846 -30.266  1.00  0.39           O  \\nATOM   1585  CG  GLN A 226     -27.560 -25.848 -29.579  1.00  0.37           C  \\nATOM   1586  CD  GLN A 226     -28.215 -27.020 -28.872  1.00  0.43           C  \\nATOM   1587  NE2 GLN A 226     -29.057 -26.723 -27.887  1.00  0.36           N  \\nATOM   1588  OE1 GLN A 226     -27.967 -28.182 -29.208  1.00  0.46           O  \\nATOM   1589  N   GLN A 227     -23.649 -26.253 -32.078  1.00  0.44           N  \\nATOM   1590  CA  GLN A 227     -22.220 -26.545 -32.105  1.00  0.44           C  \\nATOM   1591  C   GLN A 227     -21.407 -25.287 -32.399  1.00  0.44           C  \\nATOM   1592  CB  GLN A 227     -21.911 -27.624 -33.143  1.00  0.42           C  \\nATOM   1593  O   GLN A 227     -20.390 -25.034 -31.750  1.00  0.43           O  \\nATOM   1594  CG  GLN A 227     -21.678 -29.005 -32.545  1.00  0.41           C  \\nATOM   1595  CD  GLN A 227     -21.363 -30.054 -33.595  1.00  0.46           C  \\nATOM   1596  NE2 GLN A 227     -21.006 -31.253 -33.146  1.00  0.38           N  \\nATOM   1597  OE1 GLN A 227     -21.441 -29.790 -34.799  1.00  0.47           O  \\nATOM   1598  N   GLN A 228     -21.820 -24.463 -33.382  1.00  0.43           N  \\nATOM   1599  CA  GLN A 228     -21.113 -23.215 -33.649  1.00  0.44           C  \\nATOM   1600  C   GLN A 228     -21.169 -22.282 -32.443  1.00  0.43           C  \\nATOM   1601  CB  GLN A 228     -21.698 -22.519 -34.879  1.00  0.42           C  \\nATOM   1602  O   GLN A 228     -20.162 -21.671 -32.077  1.00  0.43           O  \\nATOM   1603  CG  GLN A 228     -21.025 -22.913 -36.187  1.00  0.41           C  \\nATOM   1604  CD  GLN A 228     -21.617 -22.202 -37.389  1.00  0.45           C  \\nATOM   1605  NE2 GLN A 228     -21.169 -22.580 -38.582  1.00  0.39           N  \\nATOM   1606  OE1 GLN A 228     -22.472 -21.322 -37.248  1.00  0.48           O  \\nATOM   1607  N   GLN A 229     -22.348 -22.242 -31.825  1.00  0.42           N  \\nATOM   1608  CA  GLN A 229     -22.467 -21.450 -30.605  1.00  0.44           C  \\nATOM   1609  C   GLN A 229     -21.510 -21.952 -29.527  1.00  0.42           C  \\nATOM   1610  CB  GLN A 229     -23.905 -21.477 -30.085  1.00  0.41           C  \\nATOM   1611  O   GLN A 229     -20.833 -21.157 -28.872  1.00  0.42           O  \\nATOM   1612  CG  GLN A 229     -24.500 -20.095 -29.851  1.00  0.40           C  \\nATOM   1613  CD  GLN A 229     -25.942 -20.149 -29.381  1.00  0.45           C  \\nATOM   1614  NE2 GLN A 229     -26.500 -18.990 -29.046  1.00  0.38           N  \\nATOM   1615  OE1 GLN A 229     -26.549 -21.222 -29.321  1.00  0.47           O  \\nATOM   1616  N   GLN A 230     -21.492 -23.276 -29.322  1.00  0.45           N  \\nATOM   1617  CA  GLN A 230     -20.569 -23.851 -28.349  1.00  0.45           C  \\nATOM   1618  C   GLN A 230     -19.121 -23.536 -28.711  1.00  0.45           C  \\nATOM   1619  CB  GLN A 230     -20.768 -25.364 -28.248  1.00  0.43           C  \\nATOM   1620  O   GLN A 230     -18.318 -23.191 -27.841  1.00  0.45           O  \\nATOM   1621  CG  GLN A 230     -21.991 -25.771 -27.437  1.00  0.41           C  \\nATOM   1622  CD  GLN A 230     -22.165 -27.275 -27.353  1.00  0.45           C  \\nATOM   1623  NE2 GLN A 230     -23.170 -27.715 -26.602  1.00  0.39           N  \\nATOM   1624  OE1 GLN A 230     -21.404 -28.037 -27.958  1.00  0.49           O  \\nATOM   1625  N   GLN A 231     -18.785 -23.713 -30.031  1.00  0.45           N  \\nATOM   1626  CA  GLN A 231     -17.424 -23.400 -30.452  1.00  0.46           C  \\nATOM   1627  C   GLN A 231     -17.096 -21.930 -30.206  1.00  0.45           C  \\nATOM   1628  CB  GLN A 231     -17.226 -23.742 -31.930  1.00  0.43           C  \\nATOM   1629  O   GLN A 231     -16.022 -21.605 -29.696  1.00  0.45           O  \\nATOM   1630  CG  GLN A 231     -15.934 -24.492 -32.220  1.00  0.42           C  \\nATOM   1631  CD  GLN A 231     -15.789 -24.869 -33.683  1.00  0.46           C  \\nATOM   1632  NE2 GLN A 231     -14.615 -25.368 -34.053  1.00  0.40           N  \\nATOM   1633  OE1 GLN A 231     -16.725 -24.713 -34.473  1.00  0.48           O  \\nATOM   1634  N   TRP A 232     -18.028 -21.021 -30.515  1.00  0.33           N  \\nATOM   1635  CA  TRP A 232     -17.841 -19.599 -30.252  1.00  0.42           C  \\nATOM   1636  C   TRP A 232     -17.707 -19.335 -28.756  1.00  0.34           C  \\nATOM   1637  CB  TRP A 232     -19.006 -18.787 -30.823  1.00  0.35           C  \\nATOM   1638  O   TRP A 232     -16.817 -18.596 -28.326  1.00  0.34           O  \\nATOM   1639  CG  TRP A 232     -18.791 -18.322 -32.232  1.00  0.36           C  \\nATOM   1640  CD1 TRP A 232     -18.821 -19.086 -33.366  1.00  0.33           C  \\nATOM   1641  CD2 TRP A 232     -18.508 -16.985 -32.657  1.00  0.36           C  \\nATOM   1642  CE2 TRP A 232     -18.379 -17.012 -34.063  1.00  0.34           C  \\nATOM   1643  CE3 TRP A 232     -18.352 -15.766 -31.984  1.00  0.44           C  \\nATOM   1644  NE1 TRP A 232     -18.574 -18.304 -34.470  1.00  0.42           N  \\nATOM   1645  CH2 TRP A 232     -17.952 -14.684 -34.126  1.00  0.35           C  \\nATOM   1646  CZ2 TRP A 232     -18.100 -15.864 -34.809  1.00  0.36           C  \\nATOM   1647  CZ3 TRP A 232     -18.075 -14.625 -32.728  1.00  0.34           C  \\nATOM   1648  N   GLN A 233     -18.610 -19.933 -27.969  1.00  0.47           N  \\nATOM   1649  CA  GLN A 233     -18.523 -19.807 -26.518  1.00  0.48           C  \\nATOM   1650  C   GLN A 233     -17.167 -20.282 -26.005  1.00  0.47           C  \\nATOM   1651  CB  GLN A 233     -19.646 -20.596 -25.842  1.00  0.46           C  \\nATOM   1652  O   GLN A 233     -16.545 -19.619 -25.172  1.00  0.48           O  \\nATOM   1653  CG  GLN A 233     -20.979 -19.861 -25.807  1.00  0.44           C  \\nATOM   1654  CD  GLN A 233     -22.087 -20.684 -25.176  1.00  0.48           C  \\nATOM   1655  NE2 GLN A 233     -23.261 -20.081 -25.025  1.00  0.41           N  \\nATOM   1656  OE1 GLN A 233     -21.889 -21.852 -24.828  1.00  0.51           O  \\nATOM   1657  N   TRP A 234     -16.757 -21.410 -26.607  1.00  0.33           N  \\nATOM   1658  CA  TRP A 234     -15.468 -21.963 -26.204  1.00  0.42           C  \\nATOM   1659  C   TRP A 234     -14.328 -21.032 -26.601  1.00  0.33           C  \\nATOM   1660  CB  TRP A 234     -15.256 -23.345 -26.828  1.00  0.34           C  \\nATOM   1661  O   TRP A 234     -13.451 -20.732 -25.787  1.00  0.34           O  \\nATOM   1662  CG  TRP A 234     -14.535 -24.312 -25.937  1.00  0.35           C  \\nATOM   1663  CD1 TRP A 234     -15.054 -24.996 -24.873  1.00  0.32           C  \\nATOM   1664  CD2 TRP A 234     -13.161 -24.698 -26.032  1.00  0.35           C  \\nATOM   1665  CE2 TRP A 234     -12.914 -25.621 -24.991  1.00  0.33           C  \\nATOM   1666  CE3 TRP A 234     -12.113 -24.353 -26.896  1.00  0.44           C  \\nATOM   1667  NE1 TRP A 234     -14.083 -25.785 -24.300  1.00  0.41           N  \\nATOM   1668  CH2 TRP A 234     -10.653 -25.850 -25.653  1.00  0.33           C  \\nATOM   1669  CZ2 TRP A 234     -11.660 -26.204 -24.793  1.00  0.35           C  \\nATOM   1670  CZ3 TRP A 234     -10.866 -24.935 -26.697  1.00  0.34           C  \\nATOM   1671  N   GLN A 235     -14.292 -20.563 -27.804  1.00  0.50           N  \\nATOM   1672  CA  GLN A 235     -13.265 -19.643 -28.278  1.00  0.51           C  \\nATOM   1673  C   GLN A 235     -13.270 -18.348 -27.470  1.00  0.50           C  \\nATOM   1674  CB  GLN A 235     -13.463 -19.335 -29.763  1.00  0.48           C  \\nATOM   1675  O   GLN A 235     -12.213 -17.847 -27.084  1.00  0.50           O  \\nATOM   1676  CG  GLN A 235     -13.025 -20.463 -30.688  1.00  0.46           C  \\nATOM   1677  CD  GLN A 235     -13.305 -20.166 -32.149  1.00  0.49           C  \\nATOM   1678  NE2 GLN A 235     -12.736 -20.973 -33.038  1.00  0.42           N  \\nATOM   1679  OE1 GLN A 235     -14.027 -19.219 -32.477  1.00  0.54           O  \\nATOM   1680  N   TRP A 236     -14.508 -17.859 -27.234  1.00  0.38           N  \\nATOM   1681  CA  TRP A 236     -14.655 -16.664 -26.410  1.00  0.45           C  \\nATOM   1682  C   TRP A 236     -14.111 -16.902 -25.005  1.00  0.38           C  \\nATOM   1683  CB  TRP A 236     -16.123 -16.237 -26.337  1.00  0.38           C  \\nATOM   1684  O   TRP A 236     -13.352 -16.085 -24.479  1.00  0.39           O  \\nATOM   1685  CG  TRP A 236     -16.343 -14.924 -25.648  1.00  0.37           C  \\nATOM   1686  CD1 TRP A 236     -16.167 -13.676 -26.178  1.00  0.32           C  \\nATOM   1687  CD2 TRP A 236     -16.774 -14.729 -24.298  1.00  0.36           C  \\nATOM   1688  CE2 TRP A 236     -16.839 -13.336 -24.077  1.00  0.35           C  \\nATOM   1689  CE3 TRP A 236     -17.113 -15.599 -23.252  1.00  0.51           C  \\nATOM   1690  NE1 TRP A 236     -16.464 -12.716 -25.238  1.00  0.48           N  \\nATOM   1691  CH2 TRP A 236     -17.557 -13.664 -21.845  1.00  0.41           C  \\nATOM   1692  CZ2 TRP A 236     -17.231 -12.792 -22.851  1.00  0.52           C  \\nATOM   1693  CZ3 TRP A 236     -17.502 -15.055 -22.034  1.00  0.37           C  \\nATOM   1694  N   TRP A 237     -14.401 -18.058 -24.383  1.00  0.40           N  \\nATOM   1695  CA  TRP A 237     -13.901 -18.441 -23.066  1.00  0.47           C  \\nATOM   1696  C   TRP A 237     -12.379 -18.529 -23.066  1.00  0.40           C  \\nATOM   1697  CB  TRP A 237     -14.502 -19.781 -22.632  1.00  0.40           C  \\nATOM   1698  O   TRP A 237     -11.720 -18.005 -22.164  1.00  0.41           O  \\nATOM   1699  CG  TRP A 237     -14.429 -20.030 -21.155  1.00  0.39           C  \\nATOM   1700  CD1 TRP A 237     -15.165 -19.418 -20.178  1.00  0.34           C  \\nATOM   1701  CD2 TRP A 237     -13.568 -20.957 -20.487  1.00  0.37           C  \\nATOM   1702  CE2 TRP A 237     -13.836 -20.855 -19.104  1.00  0.36           C  \\nATOM   1703  CE3 TRP A 237     -12.595 -21.866 -20.925  1.00  0.49           C  \\nATOM   1704  NE1 TRP A 237     -14.813 -19.910 -18.943  1.00  0.46           N  \\nATOM   1705  CH2 TRP A 237     -12.217 -22.509 -18.609  1.00  0.39           C  \\nATOM   1706  CZ2 TRP A 237     -13.164 -21.628 -18.154  1.00  0.47           C  \\nATOM   1707  CZ3 TRP A 237     -11.928 -22.635 -19.978  1.00  0.36           C  \\nATOM   1708  N   GLN A 238     -11.765 -19.180 -24.093  1.00  0.51           N  \\nATOM   1709  CA  GLN A 238     -10.315 -19.265 -24.234  1.00  0.52           C  \\nATOM   1710  C   GLN A 238      -9.689 -17.876 -24.316  1.00  0.51           C  \\nATOM   1711  CB  GLN A 238      -9.942 -20.083 -25.472  1.00  0.49           C  \\nATOM   1712  O   GLN A 238      -8.669 -17.610 -23.676  1.00  0.51           O  \\nATOM   1713  CG  GLN A 238      -9.993 -21.588 -25.252  1.00  0.47           C  \\nATOM   1714  CD  GLN A 238      -9.233 -22.362 -26.313  1.00  0.51           C  \\nATOM   1715  NE2 GLN A 238      -8.825 -23.582 -25.979  1.00  0.43           N  \\nATOM   1716  OE1 GLN A 238      -9.016 -21.868 -27.424  1.00  0.53           O  \\nATOM   1717  N   GLN A 239     -10.375 -16.988 -25.089  1.00  0.54           N  \\nATOM   1718  CA  GLN A 239      -9.884 -15.619 -25.206  1.00  0.54           C  \\nATOM   1719  C   GLN A 239      -9.931 -14.901 -23.860  1.00  0.53           C  \\nATOM   1720  CB  GLN A 239     -10.698 -14.843 -26.243  1.00  0.51           C  \\nATOM   1721  O   GLN A 239      -8.972 -14.229 -23.475  1.00  0.53           O  \\nATOM   1722  CG  GLN A 239     -10.050 -14.792 -27.620  1.00  0.49           C  \\nATOM   1723  CD  GLN A 239     -10.909 -14.076 -28.646  1.00  0.51           C  \\nATOM   1724  NE2 GLN A 239     -10.411 -13.983 -29.874  1.00  0.43           N  \\nATOM   1725  OE1 GLN A 239     -12.011 -13.612 -28.337  1.00  0.54           O  \\nATOM   1726  N   GLN A 240     -11.079 -15.101 -23.130  1.00  0.54           N  \\nATOM   1727  CA  GLN A 240     -11.200 -14.517 -21.798  1.00  0.54           C  \\nATOM   1728  C   GLN A 240     -10.116 -15.048 -20.865  1.00  0.53           C  \\nATOM   1729  CB  GLN A 240     -12.583 -14.800 -21.211  1.00  0.51           C  \\nATOM   1730  O   GLN A 240      -9.484 -14.279 -20.137  1.00  0.53           O  \\nATOM   1731  CG  GLN A 240     -13.714 -14.067 -21.920  1.00  0.49           C  \\nATOM   1732  CD  GLN A 240     -13.600 -12.558 -21.804  1.00  0.52           C  \\nATOM   1733  NE2 GLN A 240     -14.139 -11.846 -22.788  1.00  0.43           N  \\nATOM   1734  OE1 GLN A 240     -13.033 -12.038 -20.838  1.00  0.53           O  \\nATOM   1735  N   GLN A 241      -9.919 -16.438 -20.857  1.00  0.53           N  \\nATOM   1736  CA  GLN A 241      -8.866 -17.033 -20.041  1.00  0.53           C  \\nATOM   1737  C   GLN A 241      -7.497 -16.470 -20.412  1.00  0.52           C  \\nATOM   1738  CB  GLN A 241      -8.866 -18.555 -20.191  1.00  0.50           C  \\nATOM   1739  O   GLN A 241      -6.704 -16.125 -19.534  1.00  0.52           O  \\nATOM   1740  CG  GLN A 241     -10.084 -19.234 -19.578  1.00  0.48           C  \\nATOM   1741  CD  GLN A 241     -10.093 -19.165 -18.062  1.00  0.51           C  \\nATOM   1742  NE2 GLN A 241     -11.219 -19.530 -17.459  1.00  0.43           N  \\nATOM   1743  OE1 GLN A 241      -9.096 -18.787 -17.438  1.00  0.52           O  \\nATOM   1744  N   GLN A 242      -7.241 -16.384 -21.790  1.00  0.52           N  \\nATOM   1745  CA  GLN A 242      -5.973 -15.821 -22.241  1.00  0.52           C  \\nATOM   1746  C   GLN A 242      -5.808 -14.382 -21.761  1.00  0.52           C  \\nATOM   1747  CB  GLN A 242      -5.870 -15.881 -23.766  1.00  0.50           C  \\nATOM   1748  O   GLN A 242      -4.736 -14.000 -21.287  1.00  0.52           O  \\nATOM   1749  CG  GLN A 242      -5.403 -17.229 -24.299  1.00  0.48           C  \\nATOM   1750  CD  GLN A 242      -5.323 -17.266 -25.813  1.00  0.51           C  \\nATOM   1751  NE2 GLN A 242      -5.007 -18.434 -26.362  1.00  0.43           N  \\nATOM   1752  OE1 GLN A 242      -5.545 -16.253 -26.485  1.00  0.53           O  \\nATOM   1753  N   GLN A 243      -6.911 -13.594 -21.908  1.00  0.53           N  \\nATOM   1754  CA  GLN A 243      -6.884 -12.218 -21.424  1.00  0.53           C  \\nATOM   1755  C   GLN A 243      -6.655 -12.170 -19.916  1.00  0.52           C  \\nATOM   1756  CB  GLN A 243      -8.185 -11.497 -21.780  1.00  0.50           C  \\nATOM   1757  O   GLN A 243      -5.868 -11.356 -19.428  1.00  0.52           O  \\nATOM   1758  CG  GLN A 243      -8.247 -11.019 -23.225  1.00  0.48           C  \\nATOM   1759  CD  GLN A 243      -9.542 -10.298 -23.550  1.00  0.51           C  \\nATOM   1760  NE2 GLN A 243      -9.697  -9.895 -24.806  1.00  0.43           N  \\nATOM   1761  OE1 GLN A 243     -10.396 -10.106 -22.679  1.00  0.53           O  \\nATOM   1762  N   GLN A 244      -7.385 -13.126 -19.125  1.00  0.52           N  \\nATOM   1763  CA  GLN A 244      -7.176 -13.220 -17.684  1.00  0.52           C  \\nATOM   1764  C   GLN A 244      -5.739 -13.618 -17.361  1.00  0.52           C  \\nATOM   1765  CB  GLN A 244      -8.151 -14.223 -17.065  1.00  0.49           C  \\nATOM   1766  O   GLN A 244      -5.114 -13.039 -16.470  1.00  0.52           O  \\nATOM   1767  CG  GLN A 244      -9.549 -13.664 -16.839  1.00  0.47           C  \\nATOM   1768  CD  GLN A 244     -10.464 -14.641 -16.125  1.00  0.50           C  \\nATOM   1769  NE2 GLN A 244     -11.672 -14.193 -15.801  1.00  0.41           N  \\nATOM   1770  OE1 GLN A 244     -10.088 -15.789 -15.868  1.00  0.51           O  \\nATOM   1771  N   TRP A 245      -5.133 -14.648 -18.110  1.00  0.43           N  \\nATOM   1772  CA  TRP A 245      -3.744 -15.054 -17.923  1.00  0.50           C  \\nATOM   1773  C   TRP A 245      -2.794 -13.899 -18.223  1.00  0.43           C  \\nATOM   1774  CB  TRP A 245      -3.408 -16.252 -18.815  1.00  0.43           C  \\nATOM   1775  O   TRP A 245      -1.847 -13.654 -17.473  1.00  0.44           O  \\nATOM   1776  CG  TRP A 245      -3.668 -17.582 -18.172  1.00  0.42           C  \\nATOM   1777  CD1 TRP A 245      -4.853 -18.032 -17.661  1.00  0.36           C  \\nATOM   1778  CD2 TRP A 245      -2.718 -18.632 -17.969  1.00  0.39           C  \\nATOM   1779  CE2 TRP A 245      -3.397 -19.691 -17.328  1.00  0.36           C  \\nATOM   1780  CE3 TRP A 245      -1.357 -18.781 -18.267  1.00  0.44           C  \\nATOM   1781  NE1 TRP A 245      -4.697 -19.300 -17.151  1.00  0.41           N  \\nATOM   1782  CH2 TRP A 245      -1.429 -21.004 -17.283  1.00  0.33           C  \\nATOM   1783  CZ2 TRP A 245      -2.760 -20.885 -16.980  1.00  0.35           C  \\nATOM   1784  CZ3 TRP A 245      -0.725 -19.969 -17.920  1.00  0.34           C  \\nATOM   1785  N   TRP A 246      -3.035 -13.148 -19.418  1.00  0.49           N  \\nATOM   1786  CA  TRP A 246      -2.218 -11.985 -19.747  1.00  0.56           C  \\nATOM   1787  C   TRP A 246      -2.290 -10.938 -18.640  1.00  0.48           C  \\nATOM   1788  CB  TRP A 246      -2.665 -11.372 -21.077  1.00  0.48           C  \\nATOM   1789  O   TRP A 246      -1.267 -10.380 -18.237  1.00  0.49           O  \\nATOM   1790  CG  TRP A 246      -1.814 -11.766 -22.246  1.00  0.48           C  \\nATOM   1791  CD1 TRP A 246      -1.497 -13.035 -22.644  1.00  0.42           C  \\nATOM   1792  CD2 TRP A 246      -1.166 -10.883 -23.167  1.00  0.43           C  \\nATOM   1793  CE2 TRP A 246      -0.473 -11.686 -24.099  1.00  0.41           C  \\nATOM   1794  CE3 TRP A 246      -1.105  -9.489 -23.295  1.00  0.46           C  \\nATOM   1795  NE1 TRP A 246      -0.691 -12.994 -23.758  1.00  0.43           N  \\nATOM   1796  CH2 TRP A 246       0.316  -9.774 -25.249  1.00  0.35           C  \\nATOM   1797  CZ2 TRP A 246       0.273 -11.140 -25.147  1.00  0.38           C  \\nATOM   1798  CZ3 TRP A 246      -0.362  -8.947 -24.338  1.00  0.35           C  \\nATOM   1799  N   TRP A 247      -3.586 -10.683 -18.105  1.00  0.49           N  \\nATOM   1800  CA  TRP A 247      -3.758  -9.748 -16.998  1.00  0.57           C  \\nATOM   1801  C   TRP A 247      -2.986 -10.213 -15.768  1.00  0.49           C  \\nATOM   1802  CB  TRP A 247      -5.241  -9.587 -16.655  1.00  0.49           C  \\nATOM   1803  O   TRP A 247      -2.291  -9.421 -15.127  1.00  0.49           O  \\nATOM   1804  CG  TRP A 247      -5.879  -8.376 -17.266  1.00  0.49           C  \\nATOM   1805  CD1 TRP A 247      -6.067  -8.125 -18.597  1.00  0.43           C  \\nATOM   1806  CD2 TRP A 247      -6.410  -7.246 -16.566  1.00  0.44           C  \\nATOM   1807  CE2 TRP A 247      -6.905  -6.348 -17.537  1.00  0.41           C  \\nATOM   1808  CE3 TRP A 247      -6.516  -6.906 -15.211  1.00  0.46           C  \\nATOM   1809  NE1 TRP A 247      -6.684  -6.907 -18.767  1.00  0.43           N  \\nATOM   1810  CH2 TRP A 247      -7.589  -4.822 -15.862  1.00  0.35           C  \\nATOM   1811  CZ2 TRP A 247      -7.498  -5.130 -17.195  1.00  0.38           C  \\nATOM   1812  CZ3 TRP A 247      -7.106  -5.693 -14.872  1.00  0.35           C  \\nATOM   1813  N   TRP A 248      -3.126 -11.602 -15.417  1.00  0.49           N  \\nATOM   1814  CA  TRP A 248      -2.391 -12.178 -14.296  1.00  0.56           C  \\nATOM   1815  C   TRP A 248      -0.887 -12.029 -14.496  1.00  0.49           C  \\nATOM   1816  CB  TRP A 248      -2.751 -13.656 -14.120  1.00  0.49           C  \\nATOM   1817  O   TRP A 248      -0.170 -11.617 -13.581  1.00  0.49           O  \\nATOM   1818  CG  TRP A 248      -3.704 -13.921 -12.993  1.00  0.48           C  \\nATOM   1819  CD1 TRP A 248      -5.046 -13.661 -12.971  1.00  0.40           C  \\nATOM   1820  CD2 TRP A 248      -3.385 -14.493 -11.721  1.00  0.42           C  \\nATOM   1821  CE2 TRP A 248      -4.584 -14.552 -10.977  1.00  0.40           C  \\nATOM   1822  CE3 TRP A 248      -2.200 -14.964 -11.139  1.00  0.47           C  \\nATOM   1823  NE1 TRP A 248      -5.581 -14.038 -11.761  1.00  0.42           N  \\nATOM   1824  CH2 TRP A 248      -3.459 -15.517  -9.132  1.00  0.33           C  \\nATOM   1825  CZ2 TRP A 248      -4.631 -15.063  -9.678  1.00  0.37           C  \\nATOM   1826  CZ3 TRP A 248      -2.250 -15.473  -9.846  1.00  0.35           C  \\nATOM   1827  N   TRP A 249      -0.395 -12.408 -15.754  1.00  0.49           N  \\nATOM   1828  CA  TRP A 249       1.024 -12.273 -16.063  1.00  0.55           C  \\nATOM   1829  C   TRP A 249       1.465 -10.816 -15.969  1.00  0.48           C  \\nATOM   1830  CB  TRP A 249       1.326 -12.822 -17.460  1.00  0.48           C  \\nATOM   1831  O   TRP A 249       2.502 -10.511 -15.375  1.00  0.49           O  \\nATOM   1832  CG  TRP A 249       2.496 -13.758 -17.507  1.00  0.48           C  \\nATOM   1833  CD1 TRP A 249       2.489 -15.100 -17.242  1.00  0.41           C  \\nATOM   1834  CD2 TRP A 249       3.847 -13.421 -17.834  1.00  0.43           C  \\nATOM   1835  CE2 TRP A 249       4.607 -14.609 -17.749  1.00  0.41           C  \\nATOM   1836  CE3 TRP A 249       4.490 -12.229 -18.193  1.00  0.46           C  \\nATOM   1837  NE1 TRP A 249       3.755 -15.617 -17.386  1.00  0.42           N  \\nATOM   1838  CH2 TRP A 249       6.583 -13.458 -18.358  1.00  0.33           C  \\nATOM   1839  CZ2 TRP A 249       5.979 -14.638 -18.009  1.00  0.36           C  \\nATOM   1840  CZ3 TRP A 249       5.856 -12.260 -18.452  1.00  0.34           C  \\nATOM   1841  N   TRP A 250       0.644  -9.902 -16.651  1.00  0.52           N  \\nATOM   1842  CA  TRP A 250       0.939  -8.477 -16.545  1.00  0.58           C  \\nATOM   1843  C   TRP A 250       0.970  -8.035 -15.086  1.00  0.51           C  \\nATOM   1844  CB  TRP A 250      -0.095  -7.655 -17.319  1.00  0.52           C  \\nATOM   1845  O   TRP A 250       1.891  -7.331 -14.664  1.00  0.52           O  \\nATOM   1846  CG  TRP A 250       0.443  -6.372 -17.879  1.00  0.50           C  \\nATOM   1847  CD1 TRP A 250       1.192  -6.217 -19.012  1.00  0.44           C  \\nATOM   1848  CD2 TRP A 250       0.276  -5.063 -17.326  1.00  0.45           C  \\nATOM   1849  CE2 TRP A 250       0.951  -4.160 -18.178  1.00  0.43           C  \\nATOM   1850  CE3 TRP A 250      -0.381  -4.565 -16.193  1.00  0.49           C  \\nATOM   1851  NE1 TRP A 250       1.501  -4.889 -19.197  1.00  0.44           N  \\nATOM   1852  CH2 TRP A 250       0.338  -2.325 -16.815  1.00  0.36           C  \\nATOM   1853  CZ2 TRP A 250       0.988  -2.786 -17.930  1.00  0.40           C  \\nATOM   1854  CZ3 TRP A 250      -0.342  -3.197 -15.948  1.00  0.36           C  \\nATOM   1855  N   TRP A 251      -0.052  -8.515 -14.289  1.00  0.50           N  \\nATOM   1856  CA  TRP A 251      -0.088  -8.209 -12.863  1.00  0.57           C  \\nATOM   1857  C   TRP A 251       1.137  -8.777 -12.153  1.00  0.50           C  \\nATOM   1858  CB  TRP A 251      -1.365  -8.763 -12.226  1.00  0.50           C  \\nATOM   1859  O   TRP A 251       1.779  -8.085 -11.359  1.00  0.50           O  \\nATOM   1860  CG  TRP A 251      -2.054  -7.800 -11.306  1.00  0.50           C  \\nATOM   1861  CD1 TRP A 251      -2.921  -6.802 -11.656  1.00  0.43           C  \\nATOM   1862  CD2 TRP A 251      -1.928  -7.740  -9.882  1.00  0.44           C  \\nATOM   1863  CE2 TRP A 251      -2.748  -6.682  -9.433  1.00  0.42           C  \\nATOM   1864  CE3 TRP A 251      -1.200  -8.481  -8.940  1.00  0.47           C  \\nATOM   1865  NE1 TRP A 251      -3.343  -6.127 -10.534  1.00  0.43           N  \\nATOM   1866  CH2 TRP A 251      -2.139  -7.086  -7.182  1.00  0.34           C  \\nATOM   1867  CZ2 TRP A 251      -2.861  -6.346  -8.082  1.00  0.37           C  \\nATOM   1868  CZ3 TRP A 251      -1.314  -8.145  -7.596  1.00  0.35           C  \\nATOM   1869  N   TRP A 252       1.486 -10.134 -12.395  1.00  0.46           N  \\nATOM   1870  CA  TRP A 252       2.668 -10.758 -11.810  1.00  0.54           C  \\nATOM   1871  C   TRP A 252       3.935 -10.018 -12.224  1.00  0.46           C  \\nATOM   1872  CB  TRP A 252       2.761 -12.229 -12.225  1.00  0.47           C  \\nATOM   1873  O   TRP A 252       4.802  -9.745 -11.391  1.00  0.47           O  \\nATOM   1874  CG  TRP A 252       2.978 -13.172 -11.080  1.00  0.47           C  \\nATOM   1875  CD1 TRP A 252       2.030 -13.664 -10.227  1.00  0.41           C  \\nATOM   1876  CD2 TRP A 252       4.226 -13.733 -10.661  1.00  0.42           C  \\nATOM   1877  CE2 TRP A 252       3.959 -14.559  -9.546  1.00  0.40           C  \\nATOM   1878  CE3 TRP A 252       5.545 -13.619 -11.121  1.00  0.45           C  \\nATOM   1879  NE1 TRP A 252       2.614 -14.499  -9.302  1.00  0.41           N  \\nATOM   1880  CH2 TRP A 252       6.246 -15.138  -9.354  1.00  0.31           C  \\nATOM   1881  CZ2 TRP A 252       4.965 -15.267  -8.884  1.00  0.34           C  \\nATOM   1882  CZ3 TRP A 252       6.544 -14.325 -10.461  1.00  0.34           C  \\nATOM   1883  N   TRP A 253       4.082  -9.669 -13.552  1.00  0.46           N  \\nATOM   1884  CA  TRP A 253       5.232  -8.901 -14.020  1.00  0.54           C  \\nATOM   1885  C   TRP A 253       5.263  -7.519 -13.376  1.00  0.46           C  \\nATOM   1886  CB  TRP A 253       5.206  -8.767 -15.545  1.00  0.47           C  \\nATOM   1887  O   TRP A 253       6.319  -7.053 -12.942  1.00  0.47           O  \\nATOM   1888  CG  TRP A 253       6.433  -9.296 -16.224  1.00  0.48           C  \\nATOM   1889  CD1 TRP A 253       6.756 -10.607 -16.439  1.00  0.42           C  \\nATOM   1890  CD2 TRP A 253       7.505  -8.524 -16.773  1.00  0.43           C  \\nATOM   1891  CE2 TRP A 253       8.445  -9.433 -17.306  1.00  0.41           C  \\nATOM   1892  CE3 TRP A 253       7.762  -7.149 -16.866  1.00  0.46           C  \\nATOM   1893  NE1 TRP A 253       7.965 -10.696 -17.090  1.00  0.41           N  \\nATOM   1894  CH2 TRP A 253       9.851  -7.661 -18.003  1.00  0.32           C  \\nATOM   1895  CZ2 TRP A 253       9.624  -9.011 -17.925  1.00  0.35           C  \\nATOM   1896  CZ3 TRP A 253       8.936  -6.731 -17.482  1.00  0.34           C  \\nATOM   1897  N   TRP A 254       4.048  -6.820 -13.297  1.00  0.48           N  \\nATOM   1898  CA  TRP A 254       3.955  -5.539 -12.603  1.00  0.57           C  \\nATOM   1899  C   TRP A 254       4.371  -5.680 -11.143  1.00  0.48           C  \\nATOM   1900  CB  TRP A 254       2.531  -4.982 -12.690  1.00  0.49           C  \\nATOM   1901  O   TRP A 254       5.096  -4.835 -10.613  1.00  0.49           O  \\nATOM   1902  CG  TRP A 254       2.424  -3.704 -13.467  1.00  0.50           C  \\nATOM   1903  CD1 TRP A 254       2.522  -3.553 -14.822  1.00  0.44           C  \\nATOM   1904  CD2 TRP A 254       2.203  -2.396 -12.932  1.00  0.45           C  \\nATOM   1905  CE2 TRP A 254       2.177  -1.497 -14.022  1.00  0.43           C  \\nATOM   1906  CE3 TRP A 254       2.023  -1.895 -11.635  1.00  0.47           C  \\nATOM   1907  NE1 TRP A 254       2.374  -2.228 -15.162  1.00  0.44           N  \\nATOM   1908  CH2 TRP A 254       1.805   0.338 -12.575  1.00  0.35           C  \\nATOM   1909  CZ2 TRP A 254       1.978  -0.125 -13.853  1.00  0.39           C  \\nATOM   1910  CZ3 TRP A 254       1.825  -0.529 -11.470  1.00  0.36           C  \\nATOM   1911  N   TRP A 255       4.086  -6.787 -10.408  1.00  0.45           N  \\nATOM   1912  CA  TRP A 255       4.506  -7.018  -9.030  1.00  0.55           C  \\nATOM   1913  C   TRP A 255       6.020  -7.181  -8.943  1.00  0.45           C  \\nATOM   1914  CB  TRP A 255       3.813  -8.256  -8.456  1.00  0.46           C  \\nATOM   1915  O   TRP A 255       6.651  -6.689  -8.004  1.00  0.46           O  \\nATOM   1916  CG  TRP A 255       2.544  -7.955  -7.716  1.00  0.47           C  \\nATOM   1917  CD1 TRP A 255       1.299  -7.777  -8.253  1.00  0.42           C  \\nATOM   1918  CD2 TRP A 255       2.398  -7.791  -6.302  1.00  0.43           C  \\nATOM   1919  CE2 TRP A 255       1.036  -7.517  -6.051  1.00  0.42           C  \\nATOM   1920  CE3 TRP A 255       3.289  -7.851  -5.221  1.00  0.48           C  \\nATOM   1921  NE1 TRP A 255       0.387  -7.514  -7.257  1.00  0.44           N  \\nATOM   1922  CH2 TRP A 255       1.434  -7.366  -3.723  1.00  0.35           C  \\nATOM   1923  CZ2 TRP A 255       0.542  -7.302  -4.762  1.00  0.38           C  \\nATOM   1924  CZ3 TRP A 255       2.796  -7.637  -3.939  1.00  0.36           C  \\nATOM   1925  N   TRP A 256       6.672  -7.892  -9.932  1.00  0.45           N  \\nATOM   1926  CA  TRP A 256       8.120  -8.064  -9.892  1.00  0.53           C  \\nATOM   1927  C   TRP A 256       8.833  -6.756 -10.219  1.00  0.45           C  \\nATOM   1928  CB  TRP A 256       8.559  -9.158 -10.870  1.00  0.45           C  \\nATOM   1929  O   TRP A 256       9.967  -6.535  -9.786  1.00  0.46           O  \\nATOM   1930  CG  TRP A 256       8.279 -10.551 -10.392  1.00  0.45           C  \\nATOM   1931  CD1 TRP A 256       7.092 -11.224 -10.467  1.00  0.39           C  \\nATOM   1932  CD2 TRP A 256       9.205 -11.439  -9.759  1.00  0.41           C  \\nATOM   1933  CE2 TRP A 256       8.510 -12.636  -9.477  1.00  0.40           C  \\nATOM   1934  CE3 TRP A 256      10.557 -11.339  -9.402  1.00  0.49           C  \\nATOM   1935  NE1 TRP A 256       7.224 -12.479  -9.919  1.00  0.44           N  \\nATOM   1936  CH2 TRP A 256      10.446 -13.602  -8.517  1.00  0.34           C  \\nATOM   1937  CZ2 TRP A 256       9.123 -13.727  -8.856  1.00  0.37           C  \\nATOM   1938  CZ3 TRP A 256      11.165 -12.425  -8.784  1.00  0.36           C  \\nATOM   1939  N   TRP A 257       8.127  -5.811 -10.879  1.00  0.42           N  \\nATOM   1940  CA  TRP A 257       8.784  -4.561 -11.248  1.00  0.51           C  \\nATOM   1941  C   TRP A 257       8.404  -3.441 -10.285  1.00  0.42           C  \\nATOM   1942  CB  TRP A 257       8.419  -4.165 -12.681  1.00  0.42           C  \\nATOM   1943  O   TRP A 257       8.917  -2.324 -10.389  1.00  0.42           O  \\nATOM   1944  CG  TRP A 257       9.070  -5.013 -13.732  1.00  0.43           C  \\nATOM   1945  CD1 TRP A 257       8.665  -6.246 -14.163  1.00  0.39           C  \\nATOM   1946  CD2 TRP A 257      10.245  -4.692 -14.482  1.00  0.40           C  \\nATOM   1947  CE2 TRP A 257      10.496  -5.777 -15.351  1.00  0.39           C  \\nATOM   1948  CE3 TRP A 257      11.110  -3.590 -14.505  1.00  0.43           C  \\nATOM   1949  NE1 TRP A 257       9.519  -6.711 -15.136  1.00  0.40           N  \\nATOM   1950  CH2 TRP A 257      12.410  -4.701 -16.236  1.00  0.30           C  \\nATOM   1951  CZ2 TRP A 257      11.578  -5.791 -16.234  1.00  0.32           C  \\nATOM   1952  CZ3 TRP A 257      12.187  -3.606 -15.384  1.00  0.33           C  \\nATOM   1953  N   TRP A 258       7.308  -3.618  -9.412  1.00  0.36           N  \\nATOM   1954  CA  TRP A 258       7.282  -2.664  -8.308  1.00  0.49           C  \\nATOM   1955  C   TRP A 258       8.494  -2.849  -7.401  1.00  0.38           C  \\nATOM   1956  CB  TRP A 258       5.993  -2.816  -7.496  1.00  0.39           C  \\nATOM   1957  O   TRP A 258       8.766  -3.958  -6.935  1.00  0.39           O  \\nATOM   1958  CG  TRP A 258       5.482  -1.531  -6.918  1.00  0.42           C  \\nATOM   1959  CD1 TRP A 258       5.958  -0.876  -5.816  1.00  0.41           C  \\nATOM   1960  CD2 TRP A 258       4.392  -0.747  -7.412  1.00  0.41           C  \\nATOM   1961  CE2 TRP A 258       4.262   0.372  -6.559  1.00  0.39           C  \\nATOM   1962  CE3 TRP A 258       3.511  -0.881  -8.494  1.00  0.46           C  \\nATOM   1963  NE1 TRP A 258       5.229   0.270  -5.595  1.00  0.43           N  \\nATOM   1964  CH2 TRP A 258       2.438   1.196  -7.822  1.00  0.34           C  \\nATOM   1965  CZ2 TRP A 258       3.286   1.352  -6.756  1.00  0.36           C  \\nATOM   1966  CZ3 TRP A 258       2.541   0.095  -8.688  1.00  0.35           C  \\nATOM   1967  N   PRO A 259       9.467  -1.978  -7.536  1.00  0.50           N  \\nATOM   1968  CA  PRO A 259      10.270  -1.958  -6.311  1.00  0.49           C  \\nATOM   1969  C   PRO A 259       9.419  -2.037  -5.046  1.00  0.50           C  \\nATOM   1970  CB  PRO A 259      11.005  -0.618  -6.392  1.00  0.48           C  \\nATOM   1971  O   PRO A 259       8.273  -1.580  -5.039  1.00  0.50           O  \\nATOM   1972  CG  PRO A 259      10.144   0.237  -7.265  1.00  0.48           C  \\nATOM   1973  CD  PRO A 259       9.336  -0.657  -8.161  1.00  0.50           C  \\nATOM   1974  N   PRO A 260       9.613  -3.051  -4.248  1.00  0.50           N  \\nATOM   1975  CA  PRO A 260       8.989  -2.877  -2.934  1.00  0.51           C  \\nATOM   1976  C   PRO A 260       8.742  -1.410  -2.588  1.00  0.51           C  \\nATOM   1977  CB  PRO A 260      10.008  -3.499  -1.975  1.00  0.49           C  \\nATOM   1978  O   PRO A 260       9.472  -0.531  -3.052  1.00  0.50           O  \\nATOM   1979  CG  PRO A 260      11.305  -3.459  -2.717  1.00  0.48           C  \\nATOM   1980  CD  PRO A 260      11.011  -3.419  -4.189  1.00  0.50           C  \\nATOM   1981  N   PRO A 261       7.439  -1.012  -2.305  1.00  0.51           N  \\nATOM   1982  CA  PRO A 261       7.384   0.337  -1.737  1.00  0.52           C  \\nATOM   1983  C   PRO A 261       8.668   0.723  -1.008  1.00  0.52           C  \\nATOM   1984  CB  PRO A 261       6.204   0.262  -0.764  1.00  0.50           C  \\nATOM   1985  O   PRO A 261       9.340  -0.139  -0.435  1.00  0.52           O  \\nATOM   1986  CG  PRO A 261       6.061  -1.192  -0.449  1.00  0.50           C  \\nATOM   1987  CD  PRO A 261       6.634  -1.986  -1.589  1.00  0.52           C  \\nATOM   1988  N   PRO A 262       9.254   1.873  -1.445  1.00  0.51           N  \\nATOM   1989  CA  PRO A 262      10.287   2.318  -0.506  1.00  0.52           C  \\nATOM   1990  C   PRO A 262       9.918   2.045   0.950  1.00  0.52           C  \\nATOM   1991  CB  PRO A 262      10.380   3.823  -0.773  1.00  0.49           C  \\nATOM   1992  O   PRO A 262       8.734   2.028   1.298  1.00  0.52           O  \\nATOM   1993  CG  PRO A 262       9.062   4.186  -1.377  1.00  0.49           C  \\nATOM   1994  CD  PRO A 262       8.486   2.965  -2.034  1.00  0.52           C  \\nATOM   1995  N   PRO A 263      10.725   1.310   1.623  1.00  0.54           N  \\nATOM   1996  CA  PRO A 263      10.510   1.254   3.071  1.00  0.57           C  \\nATOM   1997  C   PRO A 263       9.854   2.520   3.620  1.00  0.56           C  \\nATOM   1998  CB  PRO A 263      11.924   1.085   3.631  1.00  0.53           C  \\nATOM   1999  O   PRO A 263      10.022   3.602   3.052  1.00  0.55           O  \\nATOM   2000  CG  PRO A 263      12.822   1.649   2.578  1.00  0.53           C  \\nATOM   2001  CD  PRO A 263      12.138   1.517   1.247  1.00  0.55           C  \\nATOM   2002  N   PRO A 264       8.731   2.396   4.253  1.00  0.58           N  \\nATOM   2003  CA  PRO A 264       8.227   3.573   4.963  1.00  0.63           C  \\nATOM   2004  C   PRO A 264       9.344   4.501   5.436  1.00  0.60           C  \\nATOM   2005  CB  PRO A 264       7.477   2.972   6.154  1.00  0.57           C  \\nATOM   2006  O   PRO A 264      10.452   4.041   5.726  1.00  0.59           O  \\nATOM   2007  CG  PRO A 264       8.097   1.627   6.357  1.00  0.56           C  \\nATOM   2008  CD  PRO A 264       8.639   1.148   5.041  1.00  0.58           C  \\nATOM   2009  N   PRO A 265       9.228   5.778   5.078  1.00  0.57           N  \\nATOM   2010  CA  PRO A 265      10.172   6.688   5.732  1.00  0.61           C  \\nATOM   2011  C   PRO A 265      10.418   6.329   7.196  1.00  0.59           C  \\nATOM   2012  CB  PRO A 265       9.488   8.052   5.612  1.00  0.55           C  \\nATOM   2013  O   PRO A 265       9.526   5.799   7.864  1.00  0.57           O  \\nATOM   2014  CG  PRO A 265       8.035   7.738   5.455  1.00  0.54           C  \\nATOM   2015  CD  PRO A 265       7.905   6.375   4.837  1.00  0.57           C  \\nATOM   2016  N   PRO A 266      11.636   6.148   7.516  1.00  0.61           N  \\nATOM   2017  CA  PRO A 266      11.918   6.007   8.946  1.00  0.64           C  \\nATOM   2018  C   PRO A 266      11.006   6.872   9.815  1.00  0.62           C  \\nATOM   2019  CB  PRO A 266      13.375   6.461   9.065  1.00  0.58           C  \\nATOM   2020  O   PRO A 266      10.546   7.928   9.373  1.00  0.60           O  \\nATOM   2021  CG  PRO A 266      13.589   7.368   7.897  1.00  0.57           C  \\nATOM   2022  CD  PRO A 266      12.661   6.951   6.792  1.00  0.59           C  \\nATOM   2023  N   PRO A 267      10.332   6.307  10.706  1.00  0.68           N  \\nATOM   2024  CA  PRO A 267       9.595   7.108  11.686  1.00  0.71           C  \\nATOM   2025  C   PRO A 267      10.296   8.423  12.020  1.00  0.70           C  \\nATOM   2026  CB  PRO A 267       9.533   6.195  12.913  1.00  0.65           C  \\nATOM   2027  O   PRO A 267      11.526   8.503  11.962  1.00  0.66           O  \\nATOM   2028  CG  PRO A 267      10.707   5.282  12.770  1.00  0.63           C  \\nATOM   2029  CD  PRO A 267      11.013   5.124  11.308  1.00  0.64           C  \\nATOM   2030  N   PRO A 268       9.661   9.562  11.895  1.00  0.67           N  \\nATOM   2031  CA  PRO A 268      10.264  10.789  12.421  1.00  0.69           C  \\nATOM   2032  C   PRO A 268      11.061  10.553  13.702  1.00  0.68           C  \\nATOM   2033  CB  PRO A 268       9.055  11.690  12.689  1.00  0.64           C  \\nATOM   2034  O   PRO A 268      10.732   9.654  14.481  1.00  0.64           O  \\nATOM   2035  CG  PRO A 268       7.907  10.748  12.856  1.00  0.61           C  \\nATOM   2036  CD  PRO A 268       8.202   9.493  12.085  1.00  0.62           C  \\nATOM   2037  N   PRO A 269      12.323  10.964  13.744  1.00  0.67           N  \\nATOM   2038  CA  PRO A 269      13.052  10.935  15.014  1.00  0.68           C  \\nATOM   2039  C   PRO A 269      12.168  11.285  16.209  1.00  0.67           C  \\nATOM   2040  CB  PRO A 269      14.145  11.989  14.817  1.00  0.63           C  \\nATOM   2041  O   PRO A 269      11.219  12.061  16.074  1.00  0.64           O  \\nATOM   2042  CG  PRO A 269      13.597  12.919  13.783  1.00  0.61           C  \\nATOM   2043  CD  PRO A 269      12.653  12.151  12.903  1.00  0.62           C  \\nATOM   2044  N   PRO A 270      12.035  10.433  17.146  1.00  0.71           N  \\nATOM   2045  CA  PRO A 270      11.366  10.778  18.403  1.00  0.71           C  \\nATOM   2046  C   PRO A 270      11.536  12.249  18.777  1.00  0.71           C  \\nATOM   2047  CB  PRO A 270      12.051   9.871  19.428  1.00  0.67           C  \\nATOM   2048  O   PRO A 270      12.569  12.851  18.473  1.00  0.66           O  \\nATOM   2049  CG  PRO A 270      13.402   9.594  18.851  1.00  0.65           C  \\nATOM   2050  CD  PRO A 270      13.301   9.649  17.353  1.00  0.65           C  \\nATOM   2051  N   PRO A 271      10.436  13.002  18.843  1.00  0.71           N  \\nATOM   2052  CA  PRO A 271      10.591  14.354  19.385  1.00  0.72           C  \\nATOM   2053  C   PRO A 271      11.777  14.475  20.338  1.00  0.71           C  \\nATOM   2054  CB  PRO A 271       9.271  14.591  20.123  1.00  0.66           C  \\nATOM   2055  O   PRO A 271      11.929  13.656  21.248  1.00  0.66           O  \\nATOM   2056  CG  PRO A 271       8.650  13.237  20.240  1.00  0.64           C  \\nATOM   2057  CD  PRO A 271       9.352  12.307  19.293  1.00  0.64           C  \\nATOM   2058  N   GLN A 272      12.968  15.014  19.793  1.00  0.70           N  \\nATOM   2059  CA  GLN A 272      14.034  15.415  20.704  1.00  0.72           C  \\nATOM   2060  C   GLN A 272      13.475  15.782  22.076  1.00  0.69           C  \\nATOM   2061  CB  GLN A 272      14.821  16.592  20.126  1.00  0.65           C  \\nATOM   2062  O   GLN A 272      12.628  16.671  22.190  1.00  0.64           O  \\nATOM   2063  CG  GLN A 272      16.028  16.176  19.296  1.00  0.60           C  \\nATOM   2064  CD  GLN A 272      16.721  17.354  18.637  1.00  0.61           C  \\nATOM   2065  NE2 GLN A 272      17.681  17.065  17.765  1.00  0.53           N  \\nATOM   2066  OE1 GLN A 272      16.396  18.514  18.908  1.00  0.65           O  \\nATOM   2067  N   GLN A 273      13.329  14.712  23.032  1.00  0.75           N  \\nATOM   2068  CA  GLN A 273      13.126  15.037  24.439  1.00  0.77           C  \\nATOM   2069  C   GLN A 273      13.793  16.362  24.799  1.00  0.75           C  \\nATOM   2070  CB  GLN A 273      13.663  13.918  25.333  1.00  0.69           C  \\nATOM   2071  O   GLN A 273      15.015  16.493  24.703  1.00  0.68           O  \\nATOM   2072  CG  GLN A 273      12.590  12.962  25.834  1.00  0.64           C  \\nATOM   2073  CD  GLN A 273      13.164  11.778  26.589  1.00  0.63           C  \\nATOM   2074  NE2 GLN A 273      12.305  10.831  26.949  1.00  0.55           N  \\nATOM   2075  OE1 GLN A 273      14.371  11.715  26.847  1.00  0.68           O  \\nATOM   2076  N   GLN A 274      13.068  17.541  24.542  1.00  0.75           N  \\nATOM   2077  CA  GLN A 274      13.486  18.777  25.195  1.00  0.78           C  \\nATOM   2078  C   GLN A 274      14.107  18.496  26.560  1.00  0.77           C  \\nATOM   2079  CB  GLN A 274      12.302  19.734  25.343  1.00  0.70           C  \\nATOM   2080  O   GLN A 274      13.466  17.899  27.428  1.00  0.71           O  \\nATOM   2081  CG  GLN A 274      11.988  20.524  24.080  1.00  0.65           C  \\nATOM   2082  CD  GLN A 274      10.769  21.415  24.233  1.00  0.64           C  \\nATOM   2083  NE2 GLN A 274      10.413  22.121  23.165  1.00  0.58           N  \\nATOM   2084  OE1 GLN A 274      10.152  21.467  25.302  1.00  0.71           O  \\nATOM   2085  N   GLN A 275      15.493  18.333  26.574  1.00  0.77           N  \\nATOM   2086  CA  GLN A 275      16.188  18.401  27.856  1.00  0.80           C  \\nATOM   2087  C   GLN A 275      15.533  19.421  28.783  1.00  0.79           C  \\nATOM   2088  CB  GLN A 275      17.663  18.749  27.650  1.00  0.73           C  \\nATOM   2089  O   GLN A 275      15.283  20.560  28.383  1.00  0.74           O  \\nATOM   2090  CG  GLN A 275      18.525  18.526  28.885  1.00  0.68           C  \\nATOM   2091  CD  GLN A 275      19.998  18.784  28.627  1.00  0.67           C  \\nATOM   2092  NE2 GLN A 275      20.792  18.806  29.692  1.00  0.60           N  \\nATOM   2093  OE1 GLN A 275      20.419  18.960  27.479  1.00  0.72           O  \\nATOM   2094  N   PRO A 276      14.821  18.872  29.822  1.00  0.79           N  \\nATOM   2095  CA  PRO A 276      14.340  19.814  30.836  1.00  0.81           C  \\nATOM   2096  C   PRO A 276      15.327  20.947  31.105  1.00  0.82           C  \\nATOM   2097  CB  PRO A 276      14.166  18.935  32.078  1.00  0.77           C  \\nATOM   2098  O   PRO A 276      16.542  20.747  31.015  1.00  0.77           O  \\nATOM   2099  CG  PRO A 276      15.043  17.749  31.833  1.00  0.74           C  \\nATOM   2100  CD  PRO A 276      15.226  17.590  30.351  1.00  0.74           C  \\nATOM   2101  N   PRO A 277      14.854  22.155  30.966  1.00  0.80           N  \\nATOM   2102  CA  PRO A 277      15.706  23.279  31.361  1.00  0.83           C  \\nATOM   2103  C   PRO A 277      16.544  22.977  32.601  1.00  0.83           C  \\nATOM   2104  CB  PRO A 277      14.702  24.400  31.642  1.00  0.78           C  \\nATOM   2105  O   PRO A 277      16.114  22.216  33.471  1.00  0.79           O  \\nATOM   2106  CG  PRO A 277      13.429  23.696  31.986  1.00  0.75           C  \\nATOM   2107  CD  PRO A 277      13.421  22.362  31.296  1.00  0.74           C  \\nATOM   2108  N   PRO A 278      17.827  23.196  32.516  1.00  0.83           N  \\nATOM   2109  CA  PRO A 278      18.687  23.084  33.696  1.00  0.85           C  \\nATOM   2110  C   PRO A 278      18.042  23.660  34.954  1.00  0.85           C  \\nATOM   2111  CB  PRO A 278      19.930  23.887  33.303  1.00  0.81           C  \\nATOM   2112  O   PRO A 278      17.252  24.604  34.870  1.00  0.82           O  \\nATOM   2113  CG  PRO A 278      19.442  24.877  32.295  1.00  0.78           C  \\nATOM   2114  CD  PRO A 278      18.275  24.278  31.565  1.00  0.76           C  \\nATOM   2115  N   PRO A 279      18.045  22.934  36.014  1.00  0.86           N  \\nATOM   2116  CA  PRO A 279      17.588  23.460  37.302  1.00  0.87           C  \\nATOM   2117  C   PRO A 279      18.082  24.880  37.569  1.00  0.88           C  \\nATOM   2118  CB  PRO A 279      18.180  22.476  38.315  1.00  0.84           C  \\nATOM   2119  O   PRO A 279      19.167  25.255  37.118  1.00  0.86           O  \\nATOM   2120  CG  PRO A 279      19.381  21.904  37.634  1.00  0.80           C  \\nATOM   2121  CD  PRO A 279      19.142  21.919  36.152  1.00  0.78           C  \\nATOM   2122  N   PRO A 280      17.234  25.777  37.979  1.00  0.88           N  \\nATOM   2123  CA  PRO A 280      17.652  27.110  38.420  1.00  0.90           C  \\nATOM   2124  C   PRO A 280      18.880  27.073  39.327  1.00  0.90           C  \\nATOM   2125  CB  PRO A 280      16.429  27.630  39.179  1.00  0.87           C  \\nATOM   2126  O   PRO A 280      19.087  26.098  40.053  1.00  0.87           O  \\nATOM   2127  CG  PRO A 280      15.717  26.401  39.646  1.00  0.82           C  \\nATOM   2128  CD  PRO A 280      16.000  25.290  38.676  1.00  0.78           C  \\nATOM   2129  N   PRO A 281      19.830  27.920  39.116  1.00  0.90           N  \\nATOM   2130  CA  PRO A 281      20.969  28.067  40.024  1.00  0.91           C  \\nATOM   2131  C   PRO A 281      20.551  28.130  41.491  1.00  0.92           C  \\nATOM   2132  CB  PRO A 281      21.605  29.388  39.581  1.00  0.88           C  \\nATOM   2133  O   PRO A 281      19.455  28.603  41.805  1.00  0.88           O  \\nATOM   2134  CG  PRO A 281      20.486  30.160  38.958  1.00  0.84           C  \\nATOM   2135  CD  PRO A 281      19.496  29.186  38.385  1.00  0.80           C  \\nATOM   2136  N   PRO A 282      21.244  27.427  42.302  1.00  0.91           N  \\nATOM   2137  CA  PRO A 282      21.012  27.545  43.744  1.00  0.92           C  \\nATOM   2138  C   PRO A 282      20.910  28.996  44.208  1.00  0.92           C  \\nATOM   2139  CB  PRO A 282      22.238  26.861  44.356  1.00  0.89           C  \\nATOM   2140  O   PRO A 282      21.530  29.883  43.616  1.00  0.90           O  \\nATOM   2141  CG  PRO A 282      23.306  26.992  43.318  1.00  0.85           C  \\nATOM   2142  CD  PRO A 282      22.653  27.050  41.967  1.00  0.82           C  \\nATOM   2143  N   PRO A 283      19.972  29.274  45.038  1.00  0.90           N  \\nATOM   2144  CA  PRO A 283      19.924  30.597  45.665  1.00  0.91           C  \\nATOM   2145  C   PRO A 283      21.275  31.031  46.230  1.00  0.91           C  \\nATOM   2146  CB  PRO A 283      18.896  30.419  46.785  1.00  0.88           C  \\nATOM   2147  O   PRO A 283      22.083  30.187  46.625  1.00  0.88           O  \\nATOM   2148  CG  PRO A 283      18.916  28.957  47.094  1.00  0.84           C  \\nATOM   2149  CD  PRO A 283      19.328  28.214  45.856  1.00  0.80           C  \\nATOM   2150  N   PRO A 284      21.654  32.224  46.045  1.00  0.90           N  \\nATOM   2151  CA  PRO A 284      22.846  32.767  46.698  1.00  0.91           C  \\nATOM   2152  C   PRO A 284      22.888  32.466  48.195  1.00  0.91           C  \\nATOM   2153  CB  PRO A 284      22.731  34.273  46.447  1.00  0.88           C  \\nATOM   2154  O   PRO A 284      21.839  32.336  48.831  1.00  0.87           O  \\nATOM   2155  CG  PRO A 284      21.269  34.518  46.257  1.00  0.84           C  \\nATOM   2156  CD  PRO A 284      20.643  33.266  45.714  1.00  0.80           C  \\nATOM   2157  N   PRO A 285      24.011  32.054  48.688  1.00  0.89           N  \\nATOM   2158  CA  PRO A 285      24.168  31.895  50.135  1.00  0.90           C  \\nATOM   2159  C   PRO A 285      23.608  33.078  50.923  1.00  0.90           C  \\nATOM   2160  CB  PRO A 285      25.684  31.788  50.316  1.00  0.87           C  \\nATOM   2161  O   PRO A 285      23.587  34.204  50.420  1.00  0.86           O  \\nATOM   2162  CG  PRO A 285      26.261  32.520  49.147  1.00  0.83           C  \\nATOM   2163  CD  PRO A 285      25.305  32.406  47.994  1.00  0.80           C  \\nATOM   2164  N   PRO A 286      22.918  32.850  51.947  1.00  0.85           N  \\nATOM   2165  CA  PRO A 286      22.498  33.936  52.836  1.00  0.86           C  \\nATOM   2166  C   PRO A 286      23.640  34.888  53.186  1.00  0.85           C  \\nATOM   2167  CB  PRO A 286      22.000  33.199  54.082  1.00  0.82           C  \\nATOM   2168  O   PRO A 286      24.802  34.476  53.223  1.00  0.81           O  \\nATOM   2169  CG  PRO A 286      22.745  31.904  54.083  1.00  0.79           C  \\nATOM   2170  CD  PRO A 286      23.048  31.527  52.661  1.00  0.77           C  \\nATOM   2171  N   PRO A 287      23.459  36.124  53.091  1.00  0.83           N  \\nATOM   2172  CA  PRO A 287      24.441  37.090  53.590  1.00  0.84           C  \\nATOM   2173  C   PRO A 287      24.982  36.722  54.970  1.00  0.83           C  \\nATOM   2174  CB  PRO A 287      23.651  38.400  53.645  1.00  0.80           C  \\nATOM   2175  O   PRO A 287      24.272  36.112  55.773  1.00  0.78           O  \\nATOM   2176  CG  PRO A 287      22.226  37.980  53.816  1.00  0.78           C  \\nATOM   2177  CD  PRO A 287      22.047  36.632  53.178  1.00  0.76           C  \\nATOM   2178  N   PRO A 288      26.267  36.681  55.097  1.00  0.85           N  \\nATOM   2179  CA  PRO A 288      26.831  36.499  56.437  1.00  0.85           C  \\nATOM   2180  C   PRO A 288      26.017  37.202  57.520  1.00  0.84           C  \\nATOM   2181  CB  PRO A 288      28.228  37.114  56.318  1.00  0.82           C  \\nATOM   2182  O   PRO A 288      25.684  38.382  57.379  1.00  0.79           O  \\nATOM   2183  CG  PRO A 288      28.161  37.982  55.103  1.00  0.78           C  \\nATOM   2184  CD  PRO A 288      27.004  37.532  54.259  1.00  0.76           C  \\nATOM   2185  N   GLN A 289      25.273  36.419  58.304  1.00  0.83           N  \\nATOM   2186  CA  GLN A 289      24.751  37.011  59.531  1.00  0.83           C  \\nATOM   2187  C   GLN A 289      25.766  37.963  60.157  1.00  0.80           C  \\nATOM   2188  CB  GLN A 289      24.366  35.921  60.533  1.00  0.78           C  \\nATOM   2189  O   GLN A 289      26.870  37.551  60.520  1.00  0.75           O  \\nATOM   2190  CG  GLN A 289      23.149  35.106  60.117  1.00  0.73           C  \\nATOM   2191  CD  GLN A 289      22.747  34.079  61.158  1.00  0.71           C  \\nATOM   2192  NE2 GLN A 289      21.675  33.343  60.882  1.00  0.69           N  \\nATOM   2193  OE1 GLN A 289      23.394  33.947  62.202  1.00  0.76           O  \\nATOM   2194  N   GLN A 290      25.612  39.309  59.951  1.00  0.82           N  \\nATOM   2195  CA  GLN A 290      26.257  40.290  60.818  1.00  0.82           C  \\nATOM   2196  C   GLN A 290      26.048  39.944  62.289  1.00  0.80           C  \\nATOM   2197  CB  GLN A 290      25.725  41.694  60.528  1.00  0.77           C  \\nATOM   2198  O   GLN A 290      24.933  39.625  62.706  1.00  0.74           O  \\nATOM   2199  CG  GLN A 290      26.322  42.332  59.280  1.00  0.73           C  \\nATOM   2200  CD  GLN A 290      25.901  43.779  59.102  1.00  0.70           C  \\nATOM   2201  NE2 GLN A 290      26.791  44.591  58.542  1.00  0.66           N  \\nATOM   2202  OE1 GLN A 290      24.784  44.163  59.463  1.00  0.73           O  \\nATOM   2203  N   PRO A 291      27.065  39.438  62.915  1.00  0.81           N  \\nATOM   2204  CA  PRO A 291      26.896  39.394  64.369  1.00  0.80           C  \\nATOM   2205  C   PRO A 291      26.157  40.615  64.914  1.00  0.79           C  \\nATOM   2206  CB  PRO A 291      28.334  39.349  64.890  1.00  0.77           C  \\nATOM   2207  O   PRO A 291      26.519  41.751  64.599  1.00  0.74           O  \\nATOM   2208  CG  PRO A 291      29.150  39.997  63.818  1.00  0.76           C  \\nATOM   2209  CD  PRO A 291      28.444  39.813  62.505  1.00  0.76           C  \\nATOM   2210  N   TRP A 292      24.821  40.558  65.288  1.00  0.76           N  \\nATOM   2211  CA  TRP A 292      24.175  41.499  66.196  1.00  0.81           C  \\nATOM   2212  C   TRP A 292      25.165  42.024  67.231  1.00  0.74           C  \\nATOM   2213  CB  TRP A 292      22.985  40.839  66.898  1.00  0.70           C  \\nATOM   2214  O   TRP A 292      25.705  41.254  68.029  1.00  0.65           O  \\nATOM   2215  CG  TRP A 292      21.917  40.357  65.963  1.00  0.61           C  \\nATOM   2216  CD1 TRP A 292      21.789  39.103  65.434  1.00  0.56           C  \\nATOM   2217  CD2 TRP A 292      20.828  41.125  65.443  1.00  0.61           C  \\nATOM   2218  CE2 TRP A 292      20.078  40.272  64.604  1.00  0.57           C  \\nATOM   2219  CE3 TRP A 292      20.412  42.454  65.605  1.00  0.70           C  \\nATOM   2220  NE1 TRP A 292      20.685  39.045  64.616  1.00  0.73           N  \\nATOM   2221  CH2 TRP A 292      18.549  42.009  64.105  1.00  0.67           C  \\nATOM   2222  CZ2 TRP A 292      18.934  40.705  63.929  1.00  0.71           C  \\nATOM   2223  CZ3 TRP A 292      19.274  42.883  64.933  1.00  0.67           C  \\nATOM   2224  N   GLY A 293      25.908  42.995  66.795  1.00  0.79           N  \\nATOM   2225  CA  GLY A 293      26.584  43.819  67.783  1.00  0.79           C  \\nATOM   2226  C   GLY A 293      25.932  43.765  69.152  1.00  0.77           C  \\nATOM   2227  O   GLY A 293      24.719  43.944  69.275  1.00  0.72           O  \\nATOM   2228  N   VAL A 294      26.535  43.062  70.101  1.00  0.79           N  \\nATOM   2229  CA  VAL A 294      26.397  43.410  71.511  1.00  0.80           C  \\nATOM   2230  C   VAL A 294      26.390  44.928  71.671  1.00  0.79           C  \\nATOM   2231  CB  VAL A 294      27.527  42.789  72.362  1.00  0.75           C  \\nATOM   2232  O   VAL A 294      27.413  45.585  71.463  1.00  0.75           O  \\nATOM   2233  CG1 VAL A 294      27.299  43.067  73.847  1.00  0.70           C  \\nATOM   2234  CG2 VAL A 294      27.623  41.286  72.105  1.00  0.67           C  \\nATOM   2235  N   VAL A 295      25.171  45.495  71.666  1.00  0.78           N  \\nATOM   2236  CA  VAL A 295      25.048  46.765  72.374  1.00  0.80           C  \\nATOM   2237  C   VAL A 295      25.533  46.602  73.813  1.00  0.79           C  \\nATOM   2238  CB  VAL A 295      23.594  47.286  72.357  1.00  0.75           C  \\nATOM   2239  O   VAL A 295      24.997  45.786  74.567  1.00  0.77           O  \\nATOM   2240  CG1 VAL A 295      23.504  48.657  73.023  1.00  0.72           C  \\nATOM   2241  CG2 VAL A 295      23.067  47.348  70.925  1.00  0.68           C  \\nATOM   2242  N   PRO A 296      26.730  47.158  74.017  1.00  0.81           N  \\nATOM   2243  CA  PRO A 296      26.966  47.409  75.440  1.00  0.83           C  \\nATOM   2244  C   PRO A 296      25.904  48.311  76.065  1.00  0.84           C  \\nATOM   2245  CB  PRO A 296      28.338  48.089  75.455  1.00  0.80           C  \\nATOM   2246  O   PRO A 296      25.514  49.317  75.466  1.00  0.82           O  \\nATOM   2247  CG  PRO A 296      28.422  48.808  74.147  1.00  0.78           C  \\nATOM   2248  CD  PRO A 296      27.608  48.054  73.135  1.00  0.78           C  \\nATOM   2249  N   ILE A 297      25.249  47.797  76.973  1.00  0.88           N  \\nATOM   2250  CA  ILE A 297      24.445  48.610  77.879  1.00  0.89           C  \\nATOM   2251  C   ILE A 297      25.330  49.649  78.564  1.00  0.89           C  \\nATOM   2252  CB  ILE A 297      23.727  47.738  78.934  1.00  0.86           C  \\nATOM   2253  O   ILE A 297      26.482  49.366  78.903  1.00  0.86           O  \\nATOM   2254  CG1 ILE A 297      22.801  46.726  78.250  1.00  0.81           C  \\nATOM   2255  CG2 ILE A 297      22.948  48.615  79.919  1.00  0.78           C  \\nATOM   2256  CD1 ILE A 297      22.210  45.690  79.195  1.00  0.81           C  \",\"pdb\");\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 1},{\"cartoon\": {\"color\": \"#4900b6\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 2},{\"cartoon\": {\"color\": \"#c80037\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 3},{\"cartoon\": {\"color\": \"#a80057\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 4},{\"cartoon\": {\"color\": \"#8f0070\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 5},{\"cartoon\": {\"color\": \"#7d0082\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 6},{\"cartoon\": {\"color\": \"#70008f\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 7},{\"cartoon\": {\"color\": \"#9f0060\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 8},{\"cartoon\": {\"color\": \"#bc0043\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 9},{\"cartoon\": {\"color\": \"#cd0032\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 10},{\"cartoon\": {\"color\": \"#c4003b\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 11},{\"cartoon\": {\"color\": \"#d1002e\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 12},{\"cartoon\": {\"color\": \"#db0024\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 13},{\"cartoon\": {\"color\": \"#e3001c\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 14},{\"cartoon\": {\"color\": \"#e80017\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 15},{\"cartoon\": {\"color\": \"#ec0013\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 16},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 17},{\"cartoon\": {\"color\": \"#f3000c\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 18},{\"cartoon\": {\"color\": \"#ec0013\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 19},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 20},{\"cartoon\": {\"color\": \"#fa0005\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 21},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 22},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 23},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 24},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 25},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 26},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 27},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 28},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 29},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 30},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 31},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 32},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 33},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 34},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 35},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 36},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 37},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 38},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 39},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 40},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 41},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 42},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 43},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 44},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 45},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 46},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 47},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 48},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 49},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 50},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 51},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 52},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 53},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 54},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 55},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 56},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 57},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 58},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 59},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 60},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 61},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 62},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 63},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 64},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 65},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 66},{\"cartoon\": {\"color\": \"#fb0004\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 67},{\"cartoon\": {\"color\": \"#f80007\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 68},{\"cartoon\": {\"color\": \"#f80007\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 69},{\"cartoon\": {\"color\": \"#f2000d\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 70},{\"cartoon\": {\"color\": \"#eb0014\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 71},{\"cartoon\": {\"color\": \"#eb0014\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 72},{\"cartoon\": {\"color\": \"#f3000c\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 73},{\"cartoon\": {\"color\": \"#f3000c\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 74},{\"cartoon\": {\"color\": \"#f3000c\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 75},{\"cartoon\": {\"color\": \"#f3000c\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 76},{\"cartoon\": {\"color\": \"#f0000f\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 77},{\"cartoon\": {\"color\": \"#ef0010\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 78},{\"cartoon\": {\"color\": \"#f60009\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 79},{\"cartoon\": {\"color\": \"#fb0004\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 80},{\"cartoon\": {\"color\": \"#fb0004\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 81},{\"cartoon\": {\"color\": \"#fb0004\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 82},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 83},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 84},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 85},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 86},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 87},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 88},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 89},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 90},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 91},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 92},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 93},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 94},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 95},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 96},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 97},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 98},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 99},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 100},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 101},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 102},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 103},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 104},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 105},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 106},{\"cartoon\": {\"color\": \"#fe0001\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 107},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 108},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 109},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 110},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 111},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 112},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 113},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 114},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 115},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 116},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 117},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 118},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 119},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 120},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 121},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 122},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 123},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 124},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 125},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 126},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 127},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 128},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 129},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 130},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 131},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 132},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 133},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 134},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 135},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 136},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 137},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 138},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 139},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 140},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 141},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 142},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 143},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 144},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 145},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 146},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 147},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 148},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 149},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 150},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 151},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 152},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 153},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 154},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 155},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 156},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 157},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 158},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 159},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 160},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 161},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 162},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 163},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 164},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 165},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 166},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 167},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 168},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 169},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 170},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 171},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 172},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 173},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 174},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 175},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 176},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 177},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 178},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 179},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 180},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 181},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 182},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 183},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 184},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 185},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 186},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 187},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 188},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 189},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 190},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 191},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 192},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 193},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 194},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 195},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 196},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 197},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 198},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 199},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 200},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 201},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 202},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 203},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 204},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 205},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 206},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 207},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 208},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 209},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 210},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 211},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 212},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 213},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 214},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 215},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 216},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 217},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 218},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 219},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 220},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 221},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 222},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 223},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 224},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 225},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 226},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 227},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 228},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 229},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 230},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 231},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 232},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 233},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 234},{\"cartoon\": {\"color\": \"#f4000b\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 235},{\"cartoon\": {\"color\": \"#ec0013\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 236},{\"cartoon\": {\"color\": \"#e1001e\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 237},{\"cartoon\": {\"color\": \"#d5002a\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 238},{\"cartoon\": {\"color\": \"#c5003a\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 239},{\"cartoon\": {\"color\": \"#b1004e\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 240},{\"cartoon\": {\"color\": \"#9a0065\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 241},{\"cartoon\": {\"color\": \"#7e0081\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 242},{\"cartoon\": {\"color\": \"#5e00a1\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 243},{\"cartoon\": {\"color\": \"#3a00c5\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 244},{\"cartoon\": {\"color\": \"#1000ef\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 245},{\"cartoon\": {\"color\": \"#1200ed\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 246},{\"cartoon\": {\"color\": \"#1400eb\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 247},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 248},{\"cartoon\": {\"color\": \"#0e00f1\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 249},{\"cartoon\": {\"color\": \"#1000ef\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 250},{\"cartoon\": {\"color\": \"#1100ee\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 251},{\"cartoon\": {\"color\": \"#1100ee\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 252},{\"cartoon\": {\"color\": \"#0f00f0\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 253},{\"cartoon\": {\"color\": \"#0e00f1\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 254},{\"cartoon\": {\"color\": \"#0d00f2\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 255},{\"cartoon\": {\"color\": \"#0c00f3\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 256},{\"cartoon\": {\"color\": \"#0b00f4\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 257},{\"cartoon\": {\"color\": \"#0b00f4\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 258},{\"cartoon\": {\"color\": \"#3700c8\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 259},{\"cartoon\": {\"color\": \"#5d00a2\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 260},{\"cartoon\": {\"color\": \"#7d0082\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 261},{\"cartoon\": {\"color\": \"#980067\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 262},{\"cartoon\": {\"color\": \"#af0050\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 263},{\"cartoon\": {\"color\": \"#c3003c\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 264},{\"cartoon\": {\"color\": \"#d3002c\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 265},{\"cartoon\": {\"color\": \"#e0001f\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 266},{\"cartoon\": {\"color\": \"#eb0014\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 267},{\"cartoon\": {\"color\": \"#f3000c\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 268},{\"cartoon\": {\"color\": \"#f90006\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 269},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 270},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 271},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 272},{\"cartoon\": {\"color\": \"#ff0000\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 273},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 274},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 275},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 276},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 277},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 278},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 279},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 280},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 281},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 282},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 283},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 284},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 285},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 286},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 287},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 288},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 289},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 290},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 291},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 292},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 293},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 294},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.setStyle({\"resi\": 295},{\"cartoon\": {\"color\": \"#fd0002\"}});\n",
              "\tviewer_17251214197132845.zoomTo();\n",
              "viewer_17251214197132845.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_6f5c42f8-89c9-4232-9f55-ee0a03d3b0c0\", \"A.html\", 213139)"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "2024-08-31 16:23:39.723 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.724 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.725 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.729 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
            "2024-08-31 16:23:39.732 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 600x100 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAfoAAABlCAYAAAClUiFrAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAAAWWElEQVR4nO3dfVTV9R0H8Pfl4V4gLgIaXECGCAFGpinpIUNGY4PcfNjOSVuGsNTWtJOD5UO5hmmgh7QsjkXDBO2wbGu4Y0ia2dRJuZ0hOI8yTEDTBrbOCh95vJ/9ofdyH34XuQRX+fl+ncO5/H7fp8/3w7184PrDn0ZEBERERKRKbjc7ACIiIho8LPREREQqxkJPRESkYiz0REREKsZCT0REpGIs9ERERCrGQk9ERKRiLPREREQq5jEYk7a1taGjo2MwpiYiIlI1rVYLLy+vAZtvwAt9W1sbIkeNQsv58wM9NRERkeoZDAY0NTUNWLEf8ELf0dGBlvPncfbkcfj5+l4/K4Dpf9o1/4e7pmPLR5s+5v+dVxSOrc+JUt9ex9uu04e1Hc0nRutjo9Gir02bZbxGo/J4y2PT50YHa1k+OpzH2LOmoz5K89uuYVQa21ubzbx28yms6XCfCsd2cUFhvn7E5TDHSvtU2P8N89bzPBK7NtiPVfpaO5rXtk3pdWa0ec2Yu14/b+ytr8LrzUEfpfnEZi3FWGxf032Jy8F817681rk1t6HnuGe4aTys1rROsfU5uzFW3yp64ri2pv1YsY1L4dg2HqOjvViuef2cbcrFHIVFH4s203nbp6QRNmuZPrEYZ/Oss3i0yLFtm8IeeuJS2N/1eE3xGG3jNK8jFn1t92nbV2GcQh9TTErjLfdidDDOuq9Fm8WaHRCUtbSgo6Pj1i30Jn56Pfz0+utHCt/8vkuht+qr8I3HdP6Ghd7JtftT6B0UrIEr9H3p299Cr7Av27G9/hBgM29f1rzRPpXmc7R2fwu9o8KsOJ/CGCfmsy/0CmO/S6FX3IPNa6aXwnzDQu/kfDcu9KIcx43i6kOh762gOizaGtu+9uPtC71C8dYor+10obctbkpF0sWFXpTigu2j2Mdj0Wa3BwdrGi0ebQuyfR+x6Ou4zXEfx4+O2qQPfRTnEeu2gcaL8YiIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxTwGa+ILFy8CItePpOdz0ynTJ2L5aNPHcrzdsfU5Uerb63jbdfqwtqP5xGh9bDRa9LVps4zXaFQeb3ls+tzoYC3LR4fzGHvWdNRHaX7bNYxKY3trs5nXbj6FNR3uU+HYLi4ozNePuBzmWGmfCvu/Yd56nkdi1wb7sUpfa0fz2rYpvc6MNq8Zc9fr54299VV4vTnoozSf2KylGIvta7ovcTmY79qX1zq35jb0HPcMN42H1ZrWKbY+ZzfG6ltFTxzX1rQfK7ZxKRzbxmN0tBfLNa+fs025mKOw6GPRZjpv+5Q0wmYt0ycW42yedRaPFjm2bVPYQ09cCvu7Hq8pHqNtnOZ1xKKv7T5t+yqMU+hjiklpvOVejA7GWfe1aLNYs6OnSA6YAS/0IgJfX1+Ex8QP9NRERESq5+vra/HD9Hc34IVeo9Hg0qVLOHv2LPz8/AZ6elJw4cIFhIeHM+cuxJy7HnPuesy565lyrtFoBmzOQXvr3s/Pj08MF2POXY85dz3m3PWY86GNF+MRERGpGAs9ERGRig14odfpdMjNzYVOpxvoqckB5tz1mHPXY85djzl3vcHIuUYG8tI+IiIiuqXwrXsiIiIVY6EnIiJSMRZ6IiIiFWOhJyIiUrF+FfpNmzZh1KhR8PLywuTJk/GPf/yj1/5/+tOfEBcXBy8vL4wdOxaVlZX9CvZ25kzOi4uLkZSUhICAAAQEBCA1NfWGXyOy5+zz3GT79u3QaDSYNWvW4AaoQs7m/Ntvv8XixYsREhICnU6HmJgYfn9xkrM537hxI2JjY+Ht7Y3w8HBkZ2ejra3NRdEObQcPHsT06dMRGhoKjUaDv/zlLzccs3//fkyYMAE6nQ7R0dEoLS11fmFx0vbt20Wr1cqWLVvk+PHjsnDhQvH395fz588r9q+qqhJ3d3cpKCiQEydOyG9/+1vx9PSUY8eOObv0bcvZnD/22GOyadMmqampkbq6OsnKypJhw4bJuXPnXBz50OVszk2ampokLCxMkpKSZObMma4JViWczXl7e7skJCTItGnT5NChQ9LU1CT79++X2tpaF0c+dDmb87KyMtHpdFJWViZNTU2yZ88eCQkJkezsbBdHPjRVVlbKypUrpby8XADIjh07eu3f2NgoPj4+kpOTIydOnJDCwkJxd3eX3bt3O7Wu04V+0qRJsnjxYvNxd3e3hIaGytq1axX7z549W3784x9bnZs8ebL88pe/dHbp25azObfV1dUler1etm7dOlghqk5/ct7V1SUPPPCAbN68WTIzM1noneRszt98800ZPXq0dHR0uCpE1XE254sXL5aHHnrI6lxOTo5MmTJlUONUo74U+mXLlkl8fLzVuTlz5khaWppTazn11n1HRweqq6uRmppqPufm5obU1FR89tlnimM+++wzq/4AkJaW5rA/WetPzm1duXIFnZ2dCAwMHKwwVaW/OV+9ejWCgoIwf/58V4SpKv3J+c6dO5GYmIjFixcjODgY99xzD/Lz89Hd3e2qsIe0/uT8gQceQHV1tfnt/cbGRlRWVmLatGkuifl2M1D106mb2nz99dfo7u5GcHCw1fng4GD8+9//VhzT0tKi2L+lpcWpQG9X/cm5reXLlyM0NNTuCUPK+pPzQ4cO4e2330Ztba0LIlSf/uS8sbERn3zyCebOnYvKykqcOnUKixYtQmdnJ3Jzc10R9pDWn5w/9thj+Prrr/Hggw9CRNDV1YWnnnoKzz//vCtCvu04qp8XLlzA1atX4e3t3ad5eNW9yq1btw7bt2/Hjh074OXldbPDUaWLFy8iIyMDxcXFGDFixM0O57ZhNBoRFBSE3//+95g4cSLmzJmDlStXoqio6GaHplr79+9Hfn4+3njjDRw5cgTl5eXYtWsX1qxZc7NDo1449Rv9iBEj4O7ujvPnz1udP3/+PAwGg+IYg8HgVH+y1p+cm6xfvx7r1q3Dxx9/jHvvvXcww1QVZ3Pe0NCA06dPY/r06eZzRqMRAODh4YH6+npERUUNbtBDXH+e5yEhIfD09IS7u7v53JgxY9DS0oKOjg5otdpBjXmo60/OX3jhBWRkZGDBggUAgLFjx+Ly5ct48sknsXLlSri58XfHgeSofvr5+fX5t3nAyd/otVotJk6ciH379pnPGY1G7Nu3D4mJiYpjEhMTrfoDwN69ex32J2v9yTkAFBQUYM2aNdi9ezcSEhJcEapqOJvzuLg4HDt2DLW1teaPGTNmICUlBbW1tQgPD3dl+ENSf57nU6ZMwalTp8w/VAHAyZMnERISwiLfB/3J+ZUrV+yKuekHLeFtUwbcgNVP564TvPbnGDqdTkpLS+XEiRPy5JNPir+/v7S0tIiISEZGhqxYscLcv6qqSjw8PGT9+vVSV1cnubm5/PM6Jzmb83Xr1olWq5X3339fmpubzR8XL168WVsYcpzNuS1ede88Z3P+xRdfiF6vl6efflrq6+uloqJCgoKC5KWXXrpZWxhynM15bm6u6PV6effdd6WxsVE++ugjiYqKktmzZ9+sLQwpFy9elJqaGqmpqREA8sorr0hNTY2cOXNGRERWrFghGRkZ5v6mP69bunSp1NXVyaZNm1zz53UiIoWFhfK9731PtFqtTJo0SQ4fPmxuS05OlszMTKv+f/zjHyUmJka0Wq3Ex8fLrl27+rPsbc2ZnEdERAgAu4/c3FzXBz6EOfs8t8RC3z/O5vzTTz+VyZMni06nk9GjR0teXp50dXW5OOqhzZmcd3Z2yqpVqyQqKkq8vLwkPDxcFi1aJN98843rAx+C/vrXvyp+bzblODMzU5KTk+3GjB8/XrRarYwePVpKSkqcXpe3qSUiIlIxXjlBRESkYiz0REREKsZCT0REpGIs9ERERCrGQk9ERKRiLPREREQqxkJPRESkYiz0RN9RVlYWZs2aZT7+/ve/j1//+tcuj2P//v3QaDT49ttvXb42Ed26WOhJlbKysqDRaKDRaKDVahEdHY3Vq1ejq6tr0NcuLy/v8928XF2cR40ahY0bNyq2nT59GhqNxuGtdktLS805dXNzw8iRI/GLX/wCX331lbnPgQMH8NBDDyEwMBA+Pj646667kJmZiY6ODocxHT16FDNmzEBQUBC8vLwwatQozJkzx2peIuo/FnpSrfT0dDQ3N+Pzzz/Hb37zG6xatQovv/yyYt/eCpGzAgMDodfrB2y+W4mfnx+am5tx7tw5FBcX48MPP0RGRgYA4MSJE0hPT0dCQgIOHjyIY8eOobCwEFqtFt3d3Yrz/fe//8UPfvADBAYGYs+ePairq0NJSQlCQ0Nx+fLlQdtHZ2fnoM1NdKthoSfV0ul0MBgMiIiIwK9+9SukpqZi586dAHrebs/Ly0NoaChiY2MBAGfPnsXs2bPh7++PwMBAzJw5E6dPnzbP2d3djZycHPj7+2P48OFYtmyZ3V27bN+6b29vx/LlyxEeHg6dTofo6Gi8/fbbOH36NFJSUgAAAQEB0Gg0yMrKAnDtLmJr165FZGQkvL29MW7cOLz//vtW61RWViImJgbe3t5ISUmxinOwaDQaGAwGhIaG4uGHH8YzzzyDjz/+GFevXsVHH30Eg8GAgoIC3HPPPYiKikJ6ejqKi4sd3lKzqqoKra2t2Lx5M+677z5ERkYiJSUFr776KiIjI839jh8/jp/85Cfw8/ODXq9HUlISGhoaAFzL1erVqzFy5EjodDqMHz8eu3fvNo81vVPx3nvvITk5GV5eXigrKwMAbN68GWPGjIGXlxfi4uLwxhtvDGL2iG4OFnq6bXh7e1v95r5v3z7U19dj7969qKioQGdnJ9LS0qDX6/G3v/0NVVVV8PX1RXp6unnchg0bUFpaii1btuDQoUP43//+hx07dvS67rx58/Duu+/i9ddfR11dHd566y34+voiPDwcf/7znwEA9fX1aG5uxmuvvQYAWLt2LbZt24aioiIcP34c2dnZePzxx3HgwAEA134g+dnPfobp06ejtrYWCxYswIoVKwYjbb3y9vaG0WhEV1cXDAYDmpubcfDgwT6PNxgM6Orqwo4dOxze5vTLL7/E1KlTodPp8Mknn6C6uhpPPPGE+Z9hXnvtNWzYsAHr16/Hv/71L6SlpWHGjBn4/PPPreZZsWIFlixZgrq6OqSlpaGsrAy/+93vkJeXh7q6OuTn5+OFF17A1q1b+58QolvRd7sXD9GtyfLucUajUfbu3Ss6nU6effZZc3twcLC0t7ebx7zzzjsSGxsrRqPRfK69vV28vb1lz549IiISEhIiBQUF5vbOzk4ZOXKk1Z3qkpOTZcmSJSIiUl9fLwBk7969inGa7mZlefevtrY28fHxkU8//dSq7/z58+XnP/+5iIg899xzcvfdd1u1L1++3G4uWxEREfLqq68qtjU1NQkAqampUWwvKSmRYcOGmY9PnjwpMTExkpCQICIiXV1dkpWVJQDEYDDIrFmzpLCwUFpbWx3GIyLy/PPPi4eHhwQGBkp6eroUFBSYb5Nq2mtkZKR0dHQojg8NDZW8vDyrc/fff78sWrTIal8bN2606hMVFSV/+MMfrM6tWbNGEhMTe42XaKjhb/SkWhUVFfD19YWXlxcefvhhzJkzB6tWrTK3jx07Flqt1nx89OhRnDp1Cnq9Hr6+vvD19UVgYCDa2trQ0NCA1tZWNDc3Y/LkyeYxHh4eSEhIcBhDbW0t3N3dkZyc3Oe4T506hStXruCHP/yhOQ5fX19s27bN/HZ1XV2dVRwAkJiY2Oc1+qu1tRW+vr7w8fFBbGwsgoODzW+Du7u7o6SkBOfOnUNBQQHCwsKQn5+P+Ph4NDc3O5wzLy8PLS0tKCoqQnx8PIqKihAXF4djx44BuJbDpKQkeHp62o29cOEC/vOf/2DKlClW56dMmYK6ujqrc5Zfp8uXL6OhoQHz58+3yvFLL71kzjGRWnjc7ACIBktKSgrefPNNaLVahIaGwsPD+ul+xx13WB1funQJEydONBcuS3feeWe/YnD0b9O9uXTpEgBg165dCAsLs2rT6XT9imOg6PV6HDlyBG5ubggJCVHcX1hYGDIyMpCRkYE1a9YgJiYGRUVFePHFFx3OO3z4cDzyyCN45JFHkJ+fj/vuuw/r16/H1q1b+5VDJZZfb1OOi4uL7X5gcnd3H5D1iG4VLPSkWnfccQeio6P73H/ChAl47733EBQUBD8/P8U+ISEh+Pvf/46pU6cCALq6ulBdXY0JEyYo9h87diyMRiMOHDiA1NRUu3bTOwqWV6Xffffd0Ol0+OKLLxy+EzBmzBjzhYUmhw8fvvEmvyM3NzenchoQEICQkBCnrqDXarWIiooyj7n33nuxdetWdHZ22v1W7+fnh9DQUFRVVVnlqqqqCpMmTXK4RnBwMEJDQ9HY2Ii5c+f2OTaioYiFnui6uXPn4uWXX8bMmTPNV3GfOXMG5eXlWLZsGUaOHIklS5Zg3bp1uOuuuxAXF4dXXnml17+BHzVqFDIzM/HEE0/g9ddfx7hx43DmzBl89dVXmD17NiIiIqDRaFBRUYFp06bB29sber0ezz77LLKzs2E0GvHggw+itbUVVVVV8PPzQ2ZmJp566ils2LABS5cuxYIFC1BdXY3S0tI+7fPLL7+0+1v5iIgI8+f19fV2Y+Lj428471tvvYXa2lr89Kc/RVRUFNra2rBt2zYcP34chYWFimMqKiqwfft2PProo4iJiYGI4IMPPkBlZSVKSkoAAE8//TQKCwvx6KOP4rnnnsOwYcNw+PBhTJo0CbGxsVi6dClyc3MRFRWF8ePHo6SkBLW1tYrvzFh68cUX8cwzz2DYsGFIT09He3s7/vnPf+Kbb75BTk7ODfdLNGTc7IsEiAaD5cV4zrQ3NzfLvHnzZMSIEaLT6WT06NGycOFC8wVlnZ2dsmTJEvHz8xN/f3/JycmRefPmObwYT0Tk6tWrkp2dLSEhIaLVaiU6Olq2bNlibl+9erUYDAbRaDSSmZkpItcuINy4caPExsaKp6en3HnnnZKWliYHDhwwj/vggw8kOjpadDqdJCUlyZYtW/p0MR4Au4933nnHfNGa0sfZs2ftLsazdeTIEXn88cclMjJSdDqdDB8+XKZOnSo7d+50OKahoUEWLlwoMTEx4u3tLf7+/nL//fdLSUmJVb+jR4/Kj370I/Hx8RG9Xi9JSUnS0NAgIiLd3d2yatUqCQsLE09PTxk3bpx8+OGH5rG9XWRYVlYm48ePF61WKwEBATJ16lQpLy93GC/RUKQRcfA3LURERDTk8ap7IiIiFWOhJyIiUjEWeiIiIhVjoSciIlIxFnoiIiIVY6EnIiJSMRZ6IiIiFWOhJyIiUjEWeiIiIhVjoSciIlIxFnoiIiIVY6EnIiJSsf8DMIiDYEQUuXYAAAAASUVORK5CYII=\n"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Downloading the results`\n",
        "\n",
        "if os.path.exists(\"d\"):\n",
        "    !rm -rf \"d\"\n",
        "if os.path.exists(\"text_dir\"):\n",
        "    !rm -rf \"text_dir\"\n",
        "\n",
        "!mkdir text_dir\n",
        "!mkdir d\n",
        "c = PPA(Sequence , model3)\n",
        "if Sequence != None and Sequence !=\"\" :\n",
        "  if ID == None:\n",
        "    c.name = \"Your peptide\"\n",
        "  with open(\"seq.txt\",\"w\") as w:\n",
        "          w.write(\"                                                 <<<<<<  LLPS prediction results  >>>>>>\\n\\n\\n\")\n",
        "          w.write(\"[Sequence id] : \" +c.name+\"\\n\\n\")\n",
        "          w.write(\"[Sequence length] : \"+str(len(c.seq))+\"\\n\\n\")\n",
        "          w.write(\"[Sequence_of_peptide] : \" +c.seq+\"\\n\\n\")\n",
        "          w.write(\"[Predicted_score] : \" +score +\"\\n\\n\")\n",
        "          w.close()\n",
        "\n",
        "\n",
        "\n",
        "  if c.name != None:\n",
        "    img_name = \"Info_of_sequenceID:\"+c.name+\".png\"\n",
        "    fig1.write_image(img_name)\n",
        "  else:\n",
        "    name = np.random.randint(1000000)\n",
        "    img_name = \"Info_of_sequenceID:\"+str(name)+\".png\"\n",
        "    fig1.write_image(img_name)\n",
        "\n",
        "  shutil.move(img_name, \"d\")\n",
        "  shutil.move(\"seq.txt\", \"text_dir\")\n",
        "  text_files_path = \"text_dir\"\n",
        "  png_files_path = \"d\"\n",
        "\n",
        "  zip_output_path = 'zipped_files.zip'\n",
        "\n",
        "  text_files = [f for f in os.listdir(text_files_path) if f.endswith('.txt')]\n",
        "  png_files = [f for f in os.listdir(png_files_path) if f.endswith('.png')]\n",
        "\n",
        "  with zipfile.ZipFile(zip_output_path, 'w') as zipf:\n",
        "    for text_file in text_files:\n",
        "      zipf.write(os.path.join(text_files_path, text_file), arcname=text_file)\n",
        "\n",
        "    for png_file in png_files:\n",
        "      zipf.write(os.path.join(png_files_path, png_file), arcname=png_file)\n",
        "\n",
        "  print(\"Files zipped successfully!\")\n",
        "\n",
        "\n",
        "else:\n",
        "  h = pd.DataFrame(D)\n",
        "  h.to_json(\"h.json\")\n",
        "  with open(\"results.txt\",\"w\") as w:\n",
        "    w.write(\"                                                 <<<<<<  LLPS prediction results  >>>>>>\\n\\n\\n\")\n",
        "    w.write(\"Number of sequences : \"+str(len(D[\"seq\"]))+\"\\n\\n\")\n",
        "    for i in range(len(D[\"id\"])):\n",
        "      w.write(\" [Sequence {i}]\".format(i = i)+\"\\n\\n\")\n",
        "      w.write(\"   Sequence id : \"+D[\"id\"][i]+\"\\n\\n\")\n",
        "      w.write(\"   Sequence length : \"+str(len(D[\"seq\"][i]))+\"\\n\\n\")\n",
        "      w.write(\"   Sequence : \" + D[\"seq\"][i]+\"\\n\\n\")\n",
        "      w.write(\"   Predicted Score : \" + str(D[\"score\"][i])+\"\\n\\n\")\n",
        "      w.write(\"-\"*150+\"\\n\\n\")\n",
        "    w.close()\n",
        "    shutil.move(\"h.json\", \"d\")\n",
        "    shutil.move(\"results.txt\", \"text_dir\")\n",
        "    text_files_path = \"text_dir\"\n",
        "    png_files_path = \"d\"\n",
        "    zip_output_path = 'zipped_files.zip'\n",
        "\n",
        "    text_files = [f for f in os.listdir(text_files_path) if f.endswith('.txt')]\n",
        "    png_files = [f for f in os.listdir(png_files_path) if f.endswith('.json')]\n",
        "\n",
        "    with zipfile.ZipFile(zip_output_path, 'w') as zipf:\n",
        "      for text_file in text_files:\n",
        "        zipf.write(os.path.join(text_files_path, text_file), arcname=text_file)\n",
        "\n",
        "      for png_file in png_files:\n",
        "        zipf.write(os.path.join(png_files_path, png_file), arcname=png_file)\n",
        "\n",
        "    print(\"Files zipped successfully!\")\n",
        "\n",
        "files.download(\"zipped_files.zip\")\n",
        "\n",
        "\n"
      ],
      "metadata": {
        "cellView": "form",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "id": "3XzZgNVFM0G-",
        "outputId": "fb571425-96e6-4855-9bbc-28e5c7b287f6"
      },
      "execution_count": 11,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Files zipped successfully!\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_19925040-a63b-4599-85f0-a9c8c610fc6d\", \"zipped_files.zip\", 137060)"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "##NCBI BLAST+\n",
        "To determine the BLAST database, you can either provide a FASTA file path to create your own database or use the URL link to NCBI's protein databases available on the NCBI page: https://ftp.ncbi.nlm.nih.gov/blast/db/\n",
        "\n",
        "The DrLLPS database has already been provided. By setting the Blast_Database to DrLLPS, the BLAST search will be conducted against the DrLLPS database.\n",
        "\n",
        "To load your sequences, you can either input a single sequence or upload a FASTA format file containing your sequences into the designated box\n",
        "\n",
        "as example :\n",
        "  **MAVRVAKMTEREKLLLKKYGNRRLYDTKRSAYVTLSQVADIIKQGVDVEVVDAKTQEDVTAFILTQIVLEQARRKNALLPAELLHLIIRYGEDLLQEFFQKYLEQTIHGYIQFKKAMDEQFSKWIDMQLNYSDMAHKTLSGLTRFPNIFSPPPGSPGTKKDKG**\n",
        "\n",
        "or\n",
        "\n",
        "**Your_file.fasta**\n",
        "\n"
      ],
      "metadata": {
        "id": "tDV8YK1JM457"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#@title BLAST+ Tool\n",
        "\n",
        "\n",
        "import os\n",
        "#@markdown - Specify the sequence or the path of your FASTA format file in this box : like  `MAVRVAKMTEREKLL`  or `Your_file.fasta`\n",
        "\n",
        "Sequence = \"MAVRVAKMTEREKLLLKKYGNRRLYDTKRSAYVTLSQVADIIKQGVDVEVVDAKTQEDVTAFILTQIVLEQARRKNALLPAELLHLIIRYGEDLLQEFFQKYLEQTIHGYIQFKKAMDEQFSKWIDMQLNYSDMAHKTLSGLTRFPNIFSPPPGSPGTKKDKG\" #@param {type:\"string\"}\n",
        "#@markdown - Specify the fasta file or URL of your Database here :                    like  `https://ftp.ncbi.nlm.nih.gov/blast/db/swissprot.tar.gz`  or `Your_Database.fasta`\n",
        "\n",
        "Blast_Database = \"DrLLPS\" #@param [\"DrLLPS\",\"https://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/swissprot.gz\"] {allow-input : true}\n",
        "Blast_result_format = \"1\" #@param [1,2,3,4,5,6,7]\n",
        "Blast_result_file_name = \"Blast_result.txt\" #@param {type:\"string\"}\n",
        "E_value_threshold = \"100\" #@param {type:\"string\"}\n",
        "Max_target_Sequences = 6 #@param {type:\"integer\"}\n",
        "\n",
        "if Blast_Database == \"DrLLPS\":\n",
        "  Blast_Database = \"DrLLPS.fasta\"\n",
        "\n",
        "import sys\n",
        "from os import path\n",
        "\n",
        "if path.exists(\"/content/bioinformatics_stockholm/\"):\n",
        "  pass\n",
        "else:\n",
        "  !git clone https://github.com/Intertangler/bioinformatics_stockholm\n",
        "sys.path.insert(0,'/content')\n",
        "\n",
        "import importlib.util\n",
        "spec = importlib.util.spec_from_file_location(\"msa\", \"/content/bioinformatics_stockholm/msa/msa.py\")\n",
        "msa = importlib.util.module_from_spec(spec)\n",
        "spec.loader.exec_module(msa)\n",
        "\n",
        "spec = importlib.util.spec_from_file_location(\"miniblast\", \"/content/bioinformatics_stockholm/Miniblast/miniblast.py\")\n",
        "miniblast = importlib.util.module_from_spec(spec)\n",
        "spec.loader.exec_module(miniblast)\n",
        "\n",
        "def Fasta(fasta_path):\n",
        "  from command_runner import command_runner\n",
        "  command = \"makeblastdb -in {k} -dbtype prot -out blastdb\".format(k =fasta_path )\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  print(\"-- Blast Database has been created -- \\n\")\n",
        "\n",
        "\n",
        "if \".fasta\" in Blast_Database or \".txt\" in Blast_Database:\n",
        "  if os.path.exists(Blast_Database):\n",
        "    Fasta(Blast_Database)\n",
        "  with open(Blast_Database) as w:\n",
        "    j = w.read().split(\">\")[1:]\n",
        "  Dict = {i[:i.index(\"\\n\")].split(\" \")[0] if \"\\n\" in i else i : edit(i[i.index(\"\\n\"):]) if \"\\n\" in i else None for i in j}\n",
        "\n",
        "else:\n",
        "  command = \"wget {k}\".format(k = Blast_Database)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  print(\"-- Blast Database has been downloaded -- \\n\")\n",
        "  exit_code, output = command_runner(\"gunzip {l}\".format(l = Blast_Database.split(\"/\")[-1]), shell=True, live_output=True)\n",
        "  Fasta(Blast_Database.split(\"/\")[-1][:-3])\n",
        "  db = miniblast.read_database_fasta(Blast_Database.split(\"/\")[-1][:-3])\n",
        "  Dict = {i.id :\"\".join(list(i.seq)) for i in db}\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "cellView": "form",
        "id": "zf3DjMfIM32y",
        "outputId": "c573de52-edac-445e-d036-39a89cb5fc5d"
      },
      "execution_count": 12,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Cloning into 'bioinformatics_stockholm'...\n",
            "remote: Enumerating objects: 296, done.\u001b[K\n",
            "remote: Total 296 (delta 0), reused 0 (delta 0), pack-reused 296 (from 1)\u001b[K\n",
            "Receiving objects: 100% (296/296), 42.10 MiB | 32.69 MiB/s, done.\n",
            "Resolving deltas: 100% (127/127), done.\n",
            "\n",
            "\n",
            "Building a new DB, current time: 08/31/2024 16:25:29\n",
            "New DB name:   /content/blastdb\n",
            "New DB title:  DrLLPS.fasta\n",
            "Sequence type: Protein\n",
            "Keep MBits: T\n",
            "Maximum file size: 1000000000B\n",
            "Adding sequences from FASTA; added 1272 sequences in 0.0482361 seconds.\n",
            "\n",
            "\n",
            "-- Blast Database has been created -- \n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Blast Result\n",
        "\n",
        "blosum62 = {\n",
        "    ('W', 'F'): 1, ('L', 'R'): -2, ('S', 'P'): -1, ('V', 'T'): 0,\n",
        "    ('Q', 'Q'): 5, ('N', 'A'): -2, ('Z', 'Y'): -2, ('W', 'R'): -3,\n",
        "    ('Q', 'A'): -1, ('S', 'D'): 0, ('H', 'H'): 8, ('S', 'H'): -1,\n",
        "    ('H', 'D'): -1, ('L', 'N'): -3, ('W', 'A'): -3, ('Y', 'M'): -1,\n",
        "    ('G', 'R'): -2, ('Y', 'I'): -1, ('Y', 'E'): -2, ('B', 'Y'): -3,\n",
        "    ('Y', 'A'): -2, ('V', 'D'): -3, ('B', 'S'): 0, ('Y', 'Y'): 7,\n",
        "    ('G', 'N'): 0, ('E', 'C'): -4, ('Y', 'Q'): -1, ('Z', 'Z'): 4,\n",
        "    ('V', 'A'): 0, ('C', 'C'): 9, ('M', 'R'): -1, ('V', 'E'): -2,\n",
        "    ('T', 'N'): 0, ('P', 'P'): 7, ('V', 'I'): 3, ('V', 'S'): -2,\n",
        "    ('Z', 'P'): -1, ('V', 'M'): 1, ('T', 'F'): -2, ('V', 'Q'): -2,\n",
        "    ('K', 'K'): 5, ('P', 'D'): -1, ('I', 'H'): -3, ('I', 'D'): -3,\n",
        "    ('T', 'R'): -1, ('P', 'L'): -3, ('K', 'G'): -2, ('M', 'N'): -2,\n",
        "    ('P', 'H'): -2, ('F', 'Q'): -3, ('Z', 'G'): -2, ('X', 'L'): -1,\n",
        "    ('T', 'M'): -1, ('Z', 'C'): -3, ('X', 'H'): -1, ('D', 'R'): -2,\n",
        "    ('B', 'W'): -4, ('X', 'D'): -1, ('Z', 'K'): 1, ('F', 'A'): -2,\n",
        "    ('Z', 'W'): -3, ('F', 'E'): -3, ('D', 'N'): 1, ('B', 'K'): 0,\n",
        "    ('X', 'X'): -1, ('F', 'I'): 0, ('B', 'G'): -1, ('X', 'T'): 0,\n",
        "    ('F', 'M'): 0, ('B', 'C'): -3, ('Z', 'I'): -3, ('Z', 'V'): -2,\n",
        "    ('S', 'S'): 4, ('L', 'Q'): -2, ('W', 'E'): -3, ('Q', 'R'): 1,\n",
        "    ('N', 'N'): 6, ('W', 'M'): -1, ('Q', 'C'): -3, ('W', 'I'): -3,\n",
        "    ('S', 'C'): -1, ('L', 'A'): -1, ('S', 'G'): 0, ('L', 'E'): -3,\n",
        "    ('W', 'Q'): -2, ('H', 'G'): -2, ('S', 'K'): 0, ('Q', 'N'): 0,\n",
        "    ('N', 'R'): 0, ('H', 'C'): -3, ('Y', 'N'): -2, ('G', 'Q'): -2,\n",
        "    ('Y', 'F'): 3, ('C', 'A'): 0, ('V', 'L'): 1, ('G', 'E'): -2,\n",
        "    ('G', 'A'): 0, ('K', 'R'): 2, ('E', 'D'): 2, ('Y', 'R'): -2,\n",
        "    ('M', 'Q'): 0, ('T', 'I'): -1, ('C', 'D'): -3, ('V', 'F'): -1,\n",
        "    ('T', 'A'): 0, ('T', 'P'): -1, ('B', 'P'): -2, ('T', 'E'): -1,\n",
        "    ('V', 'N'): -3, ('P', 'G'): -2, ('M', 'A'): -1, ('K', 'H'): -1,\n",
        "    ('V', 'R'): -3, ('P', 'C'): -3, ('M', 'E'): -2, ('K', 'L'): -2,\n",
        "    ('V', 'V'): 4, ('M', 'I'): 1, ('T', 'Q'): -1, ('I', 'G'): -4,\n",
        "    ('P', 'K'): -1, ('M', 'M'): 5, ('K', 'D'): -1, ('I', 'C'): -1,\n",
        "    ('Z', 'D'): 1, ('F', 'R'): -3, ('X', 'K'): -1, ('Q', 'D'): 0,\n",
        "    ('X', 'G'): -1, ('Z', 'L'): -3, ('X', 'C'): -2, ('Z', 'H'): 0,\n",
        "    ('B', 'L'): -4, ('B', 'H'): 0, ('F', 'F'): 6, ('X', 'W'): -2,\n",
        "    ('B', 'D'): 4, ('D', 'A'): -2, ('S', 'L'): -2, ('X', 'S'): 0,\n",
        "    ('F', 'N'): -3, ('S', 'R'): -1, ('W', 'D'): -4, ('V', 'Y'): -1,\n",
        "    ('W', 'L'): -2, ('H', 'R'): 0, ('W', 'H'): -2, ('H', 'N'): 1,\n",
        "    ('W', 'T'): -2, ('T', 'T'): 5, ('S', 'F'): -2, ('W', 'P'): -4,\n",
        "    ('L', 'D'): -4, ('B', 'I'): -3, ('L', 'H'): -3, ('S', 'N'): 1,\n",
        "    ('B', 'T'): -1, ('L', 'L'): 4, ('Y', 'K'): -2, ('E', 'Q'): 2,\n",
        "    ('Y', 'G'): -3, ('Z', 'S'): 0, ('Y', 'C'): -2, ('G', 'D'): -1,\n",
        "    ('B', 'V'): -3, ('E', 'A'): -1, ('Y', 'W'): 2, ('E', 'E'): 5,\n",
        "    ('Y', 'S'): -2, ('C', 'N'): -3, ('V', 'C'): -1, ('T', 'H'): -2,\n",
        "    ('P', 'R'): -2, ('V', 'G'): -3, ('T', 'L'): -1, ('V', 'K'): -2,\n",
        "    ('K', 'Q'): 1, ('R', 'A'): -1, ('I', 'R'): -3, ('T', 'D'): -1,\n",
        "    ('P', 'F'): -4, ('I', 'N'): -3, ('K', 'I'): -3, ('M', 'D'): -3,\n",
        "    ('V', 'W'): -3, ('W', 'W'): 11, ('M', 'H'): -2, ('P', 'N'): -2,\n",
        "    ('K', 'A'): -1, ('M', 'L'): 2, ('K', 'E'): 1, ('Z', 'E'): 4,\n",
        "    ('X', 'N'): -1, ('Z', 'A'): -1, ('Z', 'M'): -1, ('X', 'F'): -1,\n",
        "    ('K', 'C'): -3, ('B', 'Q'): 0, ('X', 'B'): -1, ('B', 'M'): -3,\n",
        "    ('F', 'C'): -2, ('Z', 'Q'): 3, ('X', 'Z'): -1, ('F', 'G'): -3,\n",
        "    ('B', 'E'): 1, ('X', 'V'): -1, ('F', 'K'): -3, ('B', 'A'): -2,\n",
        "    ('X', 'R'): -1, ('D', 'D'): 6, ('W', 'G'): -2, ('Z', 'F'): -3,\n",
        "    ('S', 'Q'): 0, ('W', 'C'): -2, ('W', 'K'): -3, ('H', 'Q'): 0,\n",
        "    ('L', 'C'): -1, ('W', 'N'): -4, ('S', 'A'): 1, ('L', 'G'): -4,\n",
        "    ('W', 'S'): -3, ('S', 'E'): 0, ('H', 'E'): 0, ('S', 'I'): -2,\n",
        "    ('H', 'A'): -2, ('S', 'M'): -1, ('Y', 'L'): -1, ('Y', 'H'): 2,\n",
        "    ('Y', 'D'): -3, ('E', 'R'): 0, ('X', 'P'): -2, ('G', 'G'): 6,\n",
        "    ('G', 'C'): -3, ('E', 'N'): 0, ('Y', 'T'): -2, ('Y', 'P'): -3,\n",
        "    ('T', 'K'): -1, ('A', 'A'): 4, ('P', 'Q'): -1, ('T', 'C'): -1,\n",
        "    ('V', 'H'): -3, ('T', 'G'): -2, ('I', 'Q'): -3, ('Z', 'T'): -1,\n",
        "    ('C', 'R'): -3, ('V', 'P'): -2, ('P', 'E'): -1, ('M', 'C'): -1,\n",
        "    ('K', 'N'): 0, ('I', 'I'): 4, ('P', 'A'): -1, ('M', 'G'): -3,\n",
        "    ('T', 'S'): 1, ('I', 'E'): -3, ('P', 'M'): -2, ('M', 'K'): -1,\n",
        "    ('I', 'A'): -1, ('P', 'I'): -3, ('R', 'R'): 5, ('X', 'M'): -1,\n",
        "    ('L', 'I'): 2, ('X', 'I'): -1, ('Z', 'B'): 1, ('X', 'E'): -1,\n",
        "    ('Z', 'N'): 0, ('X', 'A'): 0, ('B', 'R'): -1, ('B', 'N'): 3,\n",
        "    ('F', 'D'): -3, ('X', 'Y'): -1, ('Z', 'R'): 0, ('F', 'H'): -1,\n",
        "    ('B', 'F'): -3, ('F', 'L'): 0, ('X', 'Q'): -1, ('B', 'B'): 4\n",
        "}\n",
        "\n",
        "Blosum62 = {}\n",
        "for i in blosum62:\n",
        "  Blosum62[i] = blosum62[i]\n",
        "  Blosum62[i[::-1]] = blosum62[i]\n",
        "if \".\" not in Sequence:\n",
        "  with open(\"sequence.txt\",\"w\") as w:\n",
        "    w.write(\">seq1\\n{o}\".format(o = Sequence))\n",
        "    w.close()\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = \"sequence.txt\",d = \"blastdb\"\n",
        "  ,f = Blast_result_format,o = Blast_result_file_name , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  files.download(Blast_result_file_name)\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = \"sequence.txt\",d = \"blastdb\"\n",
        "  ,f = 6 ,o = \"results_dict.txt\" , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "\n",
        "  with open(\"results_dict.txt\") as w:\n",
        "    y = w.read()\n",
        "  idx = [i.split(\"\\t\")[1] for i in y.split(\"\\nseq\")]\n",
        "  for i in idx:\n",
        "    target_sequence = Dict[i]\n",
        "    aln = pairwise2.align.localds(Sequence,target_sequence, match_dict = Blosum62, open = -3, extend = -1)[0]\n",
        "    print(\"Subject_Id : \"+i+\"\\n\")\n",
        "    print(pairwise2.format_alignment(align1=aln[0], align2=aln[1], score=aln[2], begin=aln[3], end=aln[4], full_sequences=True))\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "if \".\" in Sequence:\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = Sequence,d = \"blastdb\"\n",
        "  ,f = Blast_result_format,o = Blast_result_file_name , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  files.download(Blast_result_file_name)"
      ],
      "metadata": {
        "cellView": "form",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 887
        },
        "id": "GWl1oETONCIj",
        "outputId": "0f924f18-a928-499a-ca7b-2c70b7a7abd0"
      },
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_bb7e1f99-dc01-43a7-81c1-77853b9ed511\", \"Blast_result.txt\", 3233)"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Subject_Id : seq612\n",
            "\n",
            "--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MAVRVAKMTEREKLLLKKYG--NRRL------YDTKRSAY----VTLSQ--VADIIKQGVD-V-EVVDAKTQEDVTAF-ILTQIVLEQARRKNALLPAEL--L----HLIIRYGE-DLLQEFFQKYLEQTIHGYIQFKKAMDEQFSKWIDMQLNYSDMAHKT-LSGLTRFPNIFSPPPGSP-GTKKDKG---------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |.|.....|.   ...|  .||.      |.....||    .||.|  .. ..|...| . ||      ..| .. .| ..|||  |    ..|..|  .    |..|.  | |  | .. |     .|....||....|....|..|.|...| |... |..||.  ||.|.....| | ...|                                                                                                                \n",
            "MAAESALQVVEKLQARLAANPDPKKLLKYLKKLSILPITVDILVETGVGKTVNSFRKHEQVGNFARDLVAQWKKLVPVERNSEAEDQDFEKNNSRKRPRDALQREEELEGNYQESWKPSGSRSYSPEHRQKKHKKLSEPERPHKVAHSHEKRDERKRCHKVSPPYSSDPESSDYGHVQSPPPSSPHQMYTDLSRSPEEDQEPIISHQKPGKVHSNTFQDRLGVSHLGEQGKGAVSHHKQHRSSHKEKHPADAREDEKISAVSREKSHKASSKEESRRLLSGDSAKEKLPSSVVKKDKDREGSSLKKKFSPALDVASDNHFKKPKHKDSEKAKSDKNKQSVDGVDSGRGTGDPLPKAKEKVPNHLKAQEGKVRTNADGKSAGPLHPKAEETDVDDEFERPTMSFESYLSYDQPRKKKKKVVKTSSTALGEKGLKKKDSKSTSKNLNSAQKLPKVNENKSEKLQPAGAEPTRPRKVPTDVLPALPDIPLPAIHANYRPLPSLELIPSFQPKRKAFSSPQEE---EEAGFTGRRMNSKMQVYSGSKCAYLPKMMTLHQQCIR-VLKNNIDSIFEV------GGV-PYSVL-EPVLE--R----CTPDQLYRIEECNHVLIE--ETD--Q-LW-K-----VHCHRDFKEERPEEYESWREMYLRLQD-AREQRLRLLTN--NIRSAHANKPKG-RQAKMAFVNSVAKPPRDVRRRQEKFGTGGAAVPEKVRIKPAPYTTGSSHVPASNSSSNFHSSPEELAYDGPSTSSAHLAPVASSSVSYDPRKPAVKKIAPMMAKTIKAFKNRFSRR\n",
            "  Score=141\n",
            "\n",
            "Subject_Id : seq760\n",
            "\n",
            "--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MAVRV---AKMTEREKLLLKKY-GNRRLYDTKRSAYVTLSQVADIIKQGVDVEVV-DAKTQE-DVTAFILTQIV-LE-QA-RR-KNA-LLPAELLHLIIRYGEDLLQE----FFQKYLEQTIHGY--I-QFKKAMDE-QFSKWIDMQLNYSDMAHKTLSGLTRFPNIFSPPPGSPGTKK--DKG----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                            ||.   |..|...  |.|.| ..|||.|   ||   |..| ||...|  |.|. |.|..| |.....| |.. .| .| .| ..| ......|.| .|..|..|||    ..|.|.|.. ||.  . |.||..|| . .|..|....| |.|  |...| |.   |.    .|..||  .|                                                                                                                                                                                                                                                                                                                                                                                                     \n",
            "MNDQTQFTERALTILTLAQKLASDHQHPQLQPIHILAAFIETPEDGSVPYLQNLIEKGRYDYDLFKKVVNRNLVRIPQQQPAPAEITPSYALGKVLQDAAKIQKQQKDSFIAQDHILFALFNDSSIQQIFKEAQVDIEAIKQQALELRGNTRIDSRGADTNTPLEYLSKYAIDMTEQARQGKLDPVIGREEEIRSTIRVLARRIKSNPCLIGEPGIGKTAIIEGVAQRIIDDDVPTILQGAKLFSLDLAALTAGAKYKGDFEERFKGVLKEIEESKTLIVLFIDEIHMLMGNGKDDAANILKPALSRGQLKVIGATTNNEYRSIVEKDGAFERRFQKIEVAEPSVRQTVAILRGLQPKYEIHHGVRILDSALVTAAQ--LAKRYLPYRRLPD---SA---LDLV-DISCAG--VAVARDSKPEELDSKERQL-QLIQVEIKALERDEDADSTTKDRLKL-ARQKEASLQEELEPLRQRYNEEK-HGHEELTQAKKKLDELE-NKALDAERRY-DTA--TAADL-RY---FA----IPDIKKQIEKLEDQVAEEERRAGANSMIQNVVDSDTISETAARLTGIPVKKLSESENEKLIHMERDLSSEVVGQMDAIKAVSNAVRLSRSGLANPRQPASFLFLGLSGSGKTELAKKVAGFLFNDEDMMIRVDCSELSEKYAVSKLLGTTAGYVGYDEGGFLTNQLQYKPYSVLLFDEVEKAHPDVLTVMLQMLDDGRITSGQGKTIDCSNCIVIMTSNLGAEFINSQQGSKIQESTKNLVMGAVRQHFRPEFLNRISSIVIFNKLSRKAIHKIVDIRLKEIEERFEQNDKHYKLNLTQEAKDFLAKYGYSDDMGARPLNRLIQNEILNKLALRILKNEIKDKETVNVVLKKGKSRDENVPEEAEECLEVLPNHEATIGADTLGDDDNEDSMEIDDDLD\n",
            "  Score=174\n",
            "\n",
            "Subject_Id : seq306\n",
            "\n",
            "---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MAVRVAK-MT--EREKLL--LKKY-G-----NRRLYDTKR--SAYVTLSQVADIIK-QGVD--V----EVVD-AKTQEDVTAF-ILTQI-VLEQARRKNALLPAELLHLIIRYG--EDLLQEFFQKYLEQTIHG---YI---QFKKAMD---EQFSKWIDM--QLNYSD---MAHKTLSGLT-RF-PNIF---SPP---P--G-SP-------GTKKDKG----------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                       ..||. ..  ||...|  .|.| |     |.|....|.  .|| .|...|   | |||.  |    ..|| .|..|..... ..|.. |.   |..|.  ......... ..  |........|. .|...|   |.   | .|  |   .|....|..  |..|..   .|.....|.. .| |..|   .||   |  | .|       |..|. |                                                                                                          \n",
            "MGSSHHHHHHHHASENLYFQSADITDKTAEQLENLNIQDDQKQAATGSESQSVENSSASLYVGDLEPSVSEAHLYDIFSPIGSVSSIRVCRDAITKTSLGYAYVNFNDHEAGRKAIEQLNYTPIKGRLCRIMWSQRDPSLRKKGSGNIFIKNLHPDIDNKALYDTFSVFGDILSSKIATDENGKSKGFGFVHFEEEGAAKEAIDALNGMLLNGQEIYVAPHLSRKERDSQLEETKLYVGRAQKKNERMHVLKKQYEAY-RLEKMA---KYQGVNLFVKNLDDSVDDEKLEEEFAPYGTITSAKVM---RTENG--KSKGFGFVC-FSTPEEATKAITEKN-QQIVAGKPLYVAIAQ-RK--DVRRSQLAQQIQARNQMRYQQATAAAAAAAAGMPGQFMPPMFYGVMPPRGVPFNGPNPQQMNPMGGMPKN-GMPPQFRNGPVYGVPPQGGFPRNANDNNQFYQQKQRQALGEQLYKKVSAKTSNEEAAGKITGMILDLPPQEVFPLLESDELFEQHYKEASAAYESFKKEQEQQTEQA\n",
            "  Score=155\n",
            "\n",
            "Subject_Id : seq1067\n",
            "\n",
            "---------------------------------------------------------------------------------------------------------------------------MAVRVAK-M-TEREKLLLKK-YGN---RRLY---DTKRSA-YVTLSQV-ADII--KQGVDVEVVDAKTQEDVTAFILTQIVLEQARRKNALLPAELLH-LIIRYGEDLLQEFFQKYLEQTIHGYIQFKKAMDEQFSKWIDMQLNYSDMAH-KTL-SGLT--RFPNIFSP--PPGSPGTKKD-----KG-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                              |... | ..|...|.|| .||   ..|.   |.|  | |.|.| | .||.  |...| |  ..|...  ..|....   |....|.|.   ..|. .... |    ||.   |...  |  .. .|..|.|..   .....|... . |.. |..|  .|...|..  |..|....||     ||                                                                                                                                                                                                                                         \n",
            "MGSSHHHHHHHHASENLYFQSADITDKTAEQLENLNIQDDQKQAATGSESQSVENSSASLYVGDLEPSVSEAHLYDIFSPIGSVSSIRVCRDAITKTSLGYAYVNFNDHEAGRKAIEQLNYTPIKGRLCRIMWSQRDPSLRKKGSGNIFIKNLHPDIDNK--ALYDTFS-VFGDILSSKIATD-E--NGKSKG--FGFVHFE---EEGAAKEAI---DALNGMLLN-G----QEI---YVAP--H--LS-RKERDSQLE---ETKAHYTNL-YVKNINSETTDEQFQELFAKFGPIVSASLEKDADGKLKGFGFVNYEKHEDAVKAVEALNDSELNGEKLYVGRAQKKNERMHVLKKQYEAYRLEKMAKYQGQLAQQIQARNQMRYQQATAAAAAAAAGMPGQFMPPMFYGVMPPRGVPFNGPNPQQMNPMGGMPKNGMPPQFRNGPVYGVPPQGGFPRNANDNNQFYQQKQRQALGEQLYKKVSAKTSNEEAAGKITGMILDLPPQEVFPLLESDELFEQHYKEASAAYESFKKEQEQQTEQA\n",
            "  Score=147\n",
            "\n",
            "Subject_Id : seq398\n",
            "\n",
            "-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MAVRVAKMTER-------EKL-LLK-K--YGNRRLYDTKRSAYVTLSQVADIIKQGVDVEVVDAKTQED--V-----TAFILTQIVLE----QARRKNALLPAELLHLIIRYGEDL-L----QEFFQ-KYLE--QT-IH-G-----YIQFKKAMD-EQFSKWID--MQLNYSDMAH--KTLSG---LTRFPNI---FSPPPGSPGT-K---KD-KG--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |.|.||.       ||. ..| |  ..|  | |....|.||    |...|...||| ||  |.||  |     |.||  ....|    .| .|.|...|| |.   ...||. .    .|... |..|  .| |. |     ... ||  | |...| .|  |. | |..|.  |...|   .... ||   |......|.. |   || ||                                                                                                                                                                                                                                            \n",
            "MVESLTVENQEHNVQPPSVTSAGDSYSTLATDLPLPSTNDIIESRDQLTESDLDEAINATENFAQELSSQRKSSKLKGHKKKNQGQIKANRDRDTIVKLSSSVGETEEASTRDAISHDLERKDDVIEIATDTINDATESPTQIPIDVNVVIKETSTNNVAEGTENVPPIKESTGIEVGNSPITRRKKNKKKKTTNRRGRNSSNPADTTDLSKQSTLDSILVGIEEYLQEDGSKNEDIKVNIVQDEPVNVEKMDIRTRNESSDKTFDIDVPNKDNVDETSSKSENNINEEKAEHTLPREENEILNVNEGNAASFKHQLEPHGLEAGDENGQASTKDVESESLTKNGFNFKENESKHLKAGEKQQTESDRDGISPSVLAKNQKETEIGKEDHVFEQKDKEDEKCRKELSVNHENNMSHNFNAAGSDSIIPPETERETYDDETMGPTKRISDNEKNLQHGTNDISVEVEKEEEEEEEEEENSTFSKVKKENVTGEQEAVRNNEVSGTEEESTSKGEEIMGGDEKQSEAGEKSSIIEIEGSANSAKISKDNLVLEDEAEAPTQENKPTEVVGEIDIPDAPRDDVEIVEAVEKNIIPEDLEVAKEDQEGEQVKLDEPVKAMKDDKIAMRGAESISEDMKKKQEGTAELSNEKAKKEVDETARESAEGVEVEKSKTPESPKVVKRCTSGRPEDLQINERDPEVLKEDVRVPDEDVKPEIATTIENSEEEDPKSQRVQISTEQAETTQKDMGDVGSTTSFKEEEKPKRFEITQEGDKITGKDTNHEHGEATEAASENSKASDVGTAEKYIEPSSESVKKDTEEDAEVENSEKTEFIKVKAELEN--L-DAPKEAEVT----AELNKENEDVE-VD--TEEDAEVENSEKTEFI--KVKAELGNLDA-PKEAEVTAE-LN---KENEDVEVAATSKEDIETKCSEPAETPIEDGTCTEAEVS-KK--DAEAVTK-EDENME-N-SKIAEALKDVTGDQEIDDI-NISDEFQRTVELPELEKQDIKDNKGEDKELEVEETEKETSLPDLVVEENITEEKNEIKQEEEEVSQLDFNETESISKEAPNNDENGFEDQSTRENPKKASADDIFKDILDETNEFLEQLKIVDDSELNALLQSLDAKDSTTQTTEQSKKNNDKPQDVITTSEIRKLNEKEPVYIYTSLAGGGFHMIPRTNRLSTILTANRIPFTYRDLGTDDEARKVWKTFSKGRSLPGVVRGHNDLIGNWEEIEEANEDYKLRELIYDTI\n",
            "  Score=149\n",
            "\n",
            "Subject_Id : seq398\n",
            "\n",
            "-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MAVRVAKMTER-------EKL-LLK-K--YGNRRLYDTKRSAYVTLSQVADIIKQGVDVEVVDAKTQED--V-----TAFILTQIVLE----QARRKNALLPAELLHLIIRYGEDL-L----QEFFQ-KYLE--QT-IH-G-----YIQFKKAMD-EQFSKWID--MQLNYSDMAH--KTLSG---LTRFPNI---FSPPPGSPGT-K---KD-KG--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |.|.||.       ||. ..| |  ..|  | |....|.||    |...|...||| ||  |.||  |     |.||  ....|    .| .|.|...|| |.   ...||. .    .|... |..|  .| |. |     ... ||  | |...| .|  |. | |..|.  |...|   .... ||   |......|.. |   || ||                                                                                                                                                                                                                                            \n",
            "MVESLTVENQEHNVQPPSVTSAGDSYSTLATDLPLPSTNDIIESRDQLTESDLDEAINATENFAQELSSQRKSSKLKGHKKKNQGQIKANRDRDTIVKLSSSVGETEEASTRDAISHDLERKDDVIEIATDTINDATESPTQIPIDVNVVIKETSTNNVAEGTENVPPIKESTGIEVGNSPITRRKKNKKKKTTNRRGRNSSNPADTTDLSKQSTLDSILVGIEEYLQEDGSKNEDIKVNIVQDEPVNVEKMDIRTRNESSDKTFDIDVPNKDNVDETSSKSENNINEEKAEHTLPREENEILNVNEGNAASFKHQLEPHGLEAGDENGQASTKDVESESLTKNGFNFKENESKHLKAGEKQQTESDRDGISPSVLAKNQKETEIGKEDHVFEQKDKEDEKCRKELSVNHENNMSHNFNAAGSDSIIPPETERETYDDETMGPTKRISDNEKNLQHGTNDISVEVEKEEEEEEEEEENSTFSKVKKENVTGEQEAVRNNEVSGTEEESTSKGEEIMGGDEKQSEAGEKSSIIEIEGSANSAKISKDNLVLEDEAEAPTQENKPTEVVGEIDIPDAPRDDVEIVEAVEKNIIPEDLEVAKEDQEGEQVKLDEPVKAMKDDKIAMRGAESISEDMKKKQEGTAELSNEKAKKEVDETARESAEGVEVEKSKTPESPKVVKRCTSGRPEDLQINERDPEVLKEDVRVPDEDVKPEIATTIENSEEEDPKSQRVQISTEQAETTQKDMGDVGSTTSFKEEEKPKRFEITQEGDKITGKDTNHEHGEATEAASENSKASDVGTAEKYIEPSSESVKKDTEEDAEVENSEKTEFIKVKAELEN--L-DAPKEAEVT----AELNKENEDVE-VD--TEEDAEVENSEKTEFI--KVKAELGNLDA-PKEAEVTAE-LN---KENEDVEVAATSKEDIETKCSEPAETPIEDGTCTEAEVS-KK--DAEAVTK-EDENME-N-SKIAEALKDVTGDQEIDDI-NISDEFQRTVELPELEKQDIKDNKGEDKELEVEETEKETSLPDLVVEENITEEKNEIKQEEEEVSQLDFNETESISKEAPNNDENGFEDQSTRENPKKASADDIFKDILDETNEFLEQLKIVDDSELNALLQSLDAKDSTTQTTEQSKKNNDKPQDVITTSEIRKLNEKEPVYIYTSLAGGGFHMIPRTNRLSTILTANRIPFTYRDLGTDDEARKVWKTFSKGRSLPGVVRGHNDLIGNWEEIEEANEDYKLRELIYDTI\n",
            "  Score=149\n",
            "\n",
            "Subject_Id : seq1043\n",
            "\n",
            "--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MAVR-VAKM--TE-R-EKLLL----KKYGNRRLYDTKR--SAYVTLSQVADIIK-QGVD--V----EVVD-AKTQEDVTAF-ILTQI-VLEQARRKNALLPAELLHLIIRYG--EDLLQEFFQKYLEQTIHG---YI---QFKKAMD---EQFSKWIDM--QLNYSD---MAHKTLSGLT-RF-PNIF---SPP---P--G-SP-------GTKKDKG----------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                   ||. |...  .| . |||..    ||  |.|....|.  .|| .|...|   | |||.  |    ..|| .|..|..... ..|.. |.   |..|.  ......... ..  |........|. .|...|   |.   | .|  |   .|....|..  |..|..   .|.....|.. .| |..|   .||   |  | .|       |..|. |                                                                                                          \n",
            "MGSSHHHHHHHHASENLYFQSADITDKTAEQLENLNIQDDQKQAATGSESQSVENSSASLYVGDLEPSVSEAHLYDIFSPIGSVSSIRVCRDAITKTSLGYAYVNFNDHEAGRKAIEQLNYTPIKGRLCRIMWSQRDPSLRKKGSGNIFIKNLHPDIDNKALYDTFSVFGDILSSKIATDENGKSKGFGFVHFEEEGAAKEAIDALNGMLLNGQEIYVAPHLSRKERDSQLEETKAHYTNLYVKNINSETTDEQFQELFAKFGPIVSASLEKDADGKLKGFGFVNYEKHEDAVKAVEALNDSELNGEKLYVGRAQKK--NERMHVLKKQYEAY-RLEKMA---KYQGVNLFVKNLDDSVDDEKLEEEFAPYGTITSAKVM---RTENG--KSKGFGFVC-FSTPEEATKAITEKN-QQIVAGKPLYVAIAQ-RK--DVRRSQLAQQIQARNQMRYQQATAAAAAAAAGMPGQFMPPMFYGVMPPRGVPFNGPNPQQMNPMGGMPKN-GMPPQFRNGPVYGVPPQGGFPRNANDNNQFYQQKQRQALGEQLYKKVSAKTSNEEAAGKITGMILDLPPQEVFPLLESDELFEQHYKEASAAYESFKKEQEQQTEQA\n",
            "  Score=157\n",
            "\n"
          ]
        }
      ]
    }
  ]
}